| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMM17B](https://www.ncbi.nlm.nih.gov/gene?term=TIMM17B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 301 (min len 4) | Possible functional associations between TBP and TIMM17B are their interaction through the EGLN3 protein, which is involved in cellular response to low oxygen levels, and their co-localization in the mitochondrial membrane, where TIMM17B is an essential component of the TIM23 complex for protein translocation and TBP is involved in transcription activation.|
| [NOSIP](https://www.ncbi.nlm.nih.gov/gene?term=NOSIP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between NOSIP and WDR7 may involve ribosome biogenesis, as WDR7 is a ribosome biogenesis factor and NOSIP negatively regulates nitric oxide production by inducing NOS1 and NOS3 translocation to actin cytoskeleton. However, further research would be needed to determine if there is a direct interaction or shared pathway between these two genes in this context.|
| [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EBP](https://www.ncbi.nlm.nih.gov/gene?term=EBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no known direct functional associations between ATP11B and EBP in the literature. ATP11B is involved in the maintenance of lipid asymmetry in endosomes, while EBP catalyzes the conversion of Delta(8)-sterols to their corresponding Delta(7)-isomers.|
| [FAM63A](https://www.ncbi.nlm.nih.gov/gene?term=FAM63A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 4) | Possible functional associations between FAM63A and TSPAN4 include involvement in immune response and cell signaling pathways. FAM63A may play a regulatory role in protein turnover and associate with spliceosomes and snRNPs, while TSPAN4 mediates signal transduction downstream of transmembrane receptors, regulates innate and adaptive immunity, cell adhesion, osteoclast maturation, platelet activation, and vascular development. However, more information is needed to determine a specific biological association between these two genes.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP7](https://www.ncbi.nlm.nih.gov/gene?term=MAP7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between CDKN2A and MAP7 are currently unknown. CDKN2A acts as a negative regulator of cell proliferation by inhibiting cyclin-dependent kinases, while MAP7 is a microtubule-stabilizing protein involved in epithelial cell polarization and intercellular contact formation.|
| [PGM1](https://www.ncbi.nlm.nih.gov/gene?term=PGM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 4) | Possible functional associations between PGM1 and SLC25A14 may be related to glucose metabolism and mitochondrial function, as PGM1 participates in the breakdown and synthesis of glucose, and SLC25A14 participates in the mitochondrial proton leak. The BioGRID protein interaction network suggests a possible indirect interaction through shared interaction partners.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM3](https://www.ncbi.nlm.nih.gov/gene?term=ICAM3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between ARHGEF12 and ICAM3 is through their interaction with the actin cytoskeleton and cell adhesion pathways. ARHGEF12 regulates RhoA GTPase and may play a role in regulating cell migration and invasion, while ICAM3 is involved in leukocyte adhesion and apoptotic neutrophil phagocytosis by macrophages. The shortest path between these two genes in the BioGRID protein interaction network involves EZR, which is a known mediator of cytoskeleton remodeling and cell adhesion.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB10](https://www.ncbi.nlm.nih.gov/gene?term=PSMB10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 37.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are currently no known functional associations between ARHGEF12 and PSMB10 in specific biological terms. ARHGEF12 is involved in the regulation of RhoA GTPase, while PSMB10 is part of the proteasome complex involved in antigen processing. The shortest path between the two genes involves PSMB10 interacting with HIV-1 TAT protein.|
| [TWF2](https://www.ncbi.nlm.nih.gov/gene?term=TWF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZMYM2](https://www.ncbi.nlm.nih.gov/gene?term=ZMYM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 37.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association between TWF2 and ZMYM2 is their interaction in a BHC histone deacetylase complex. TWF2 regulates actin polymerization and plays a role in clathrin-mediated endocytosis, while ZMYM2 is involved in negative regulation of transcription.|
| [PAK1](https://www.ncbi.nlm.nih.gov/gene?term=PAK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TERF2IP](https://www.ncbi.nlm.nih.gov/gene?term=TERF2IP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 37.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no clear functional association between PAK1 and TERF2IP in the literature. PAK1 is involved in intracellular signaling pathways related to cytoskeleton dynamics, cell adhesion, proliferation, and apoptosis, while TERF2IP is a component of the shelterin complex that regulates telomere function and acts as a transcription regulator.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGM1](https://www.ncbi.nlm.nih.gov/gene?term=PGM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.9</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations: Both HSPD1 and PGM1 have been shown to interact with ALDH2, suggesting a potential role in maintaining mitochondrial function and glucose metabolism. HSPD1 is involved in protein import and folding in mitochondria, while PGM1 participates in glucose breakdown and synthesis.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.2</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | ARNT2 and CEBPD both function as transcription factors and have been found to interact in the BioGRID protein interaction network. ARNT2 is involved in the development of various organs and recognizes xenobiotic response elements, while CEBPD plays a role in immune and inflammatory responses. Their interaction may suggest a possible role in regulating gene expression related to these processes.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAMBP](https://www.ncbi.nlm.nih.gov/gene?term=STAMBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.6</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CDKN2A and STAMBP may involve negative regulation of cell proliferation and signal transduction for cell growth, as well as involvement in regulating cellular receptor-mediated endocytosis and ubiquitin-dependent sorting of receptors to lysosomes. However, no direct biological process linking these two genes has been reported in the literature.|
| [PSMD4](https://www.ncbi.nlm.nih.gov/gene?term=PSMD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNRD1](https://www.ncbi.nlm.nih.gov/gene?term=TXNRD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.5</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | PSMD4 and TXNRD1 are connected in the BioGRID protein interaction network through several proteins and pathways involved in DNA damage repair, cell cycle progression, and apoptosis. It is possible that PSMD4's role in protein degradation through the 26S proteasome could affect TXNRD1's ability to enhance the transcriptional activity of the estrogen receptor ESR2 or mediate cell death. Further research is needed to fully understand their functional associations.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MMP1](https://www.ncbi.nlm.nih.gov/gene?term=MMP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.4</br>**STRING score:** 390</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 186 (min len 4) | There is no clear functional association between CDKN2A and MMP1 based on the provided information. CDKN2A inhibits cell proliferation by binding to CDK4 and CDK6, while MMP1 is a collagenase. The shortest path in the BioGRID interaction network suggests that they may interact indirectly via BSG.|
| [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NSA2](https://www.ncbi.nlm.nih.gov/gene?term=NSA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.1</br>**STRING score:** 164</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 618 (min len 4) | Possible functional association: EML3 regulates mitotic spindle assembly, microtubule-kinetochore attachment, and chromosome separation whereas NSA2 is involved in the biogenesis of the 60S ribosomal subunit. There might not be any direct functional association between these two genes, but interaction partners such as ZRANB1, YWHAG, DYNLL1, and NEK6 could potentially connect these two gene products through specific biological pathways involved in mitotic spindle assembly or ribosomal subunit biogenesis.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.0</br>**STRING score:** 261</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association between MYC and SPDEF in TCGA-BRCA might involve regulation of transcription and growth-related genes. MYC activates transcription of growth-related genes and binds DNA, while SPDEF acts as a transcriptional activator for SERPINB5 promoter and interacts with the DNA-binding domain of the androgen receptor, potentially regulating prostate gland and/or prostate cancer development.|
| [LSM5](https://www.ncbi.nlm.nih.gov/gene?term=LSM5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR2K](https://www.ncbi.nlm.nih.gov/gene?term=POLR2K[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.9</br>**STRING score:** 545</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | LSM5 and POLR2K are both involved in RNA splicing and transcription. There are several potential functional associations between these genes, including their involvement in spliceosome assembly and RNA polymerase complex formation. They may also interact through the protein interaction network identified in BioGRID.|
