| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RAB11FIP2](https://www.ncbi.nlm.nih.gov/gene?term=RAB11FIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: RAB11FIP2 is involved in membrane trafficking and required for the transport of NPC1L1 to the plasma membrane, while TXNDC9 negatively impacts protein folding. Therefore, it is possible that RAB11FIP2 and TXNDC9 may have a role in regulating the transport and folding of specific proteins at the plasma membrane.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between ARHGEF12 and HSPD1 include their interaction through several proteins in the BioGRID network and their roles in regulating RhoA GTPase and mitochondrial protein import and macromolecular assembly, respectively. However, further experimental evidence is needed to determine any direct or indirect functional connections between these two genes.|
| [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional associations: VAPB participates in the endoplasmic reticulum unfolded protein response and interacts with HCV RNA-directed RNA polymerase, while WDR7 is involved in nucleolar processing of pre-18S ribosomal RNA and is part of the small subunit (SSU) processome. There is no clear biological overlap between the two genes and no known functional association.|
| [PHKA1](https://www.ncbi.nlm.nih.gov/gene?term=PHKA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between PHKA1 and WDR7 may involve ribosome biogenesis and RNA processing, as WDR7 is involved in nucleolar processing of pre-18S ribosomal RNA, and PHKA1 is a regulatory subunit of phosphorylase b kinase which may play a role in RNA folding and modifications through binding with calmodulin. The shortest path between these two genes in the BioGRID protein interaction network also suggests potential direct interaction between them.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.5</br>**STRING score:** 981</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | Possible functional associations between CREB1 and TBP can be their involvement in transcription regulation. CREB1 is a transcription factor that stimulates transcription, while TBP is a general transcription factor that is involved in the initiation of RNA polymerase II-dependent transcription. Their interaction in the BioGRID protein interaction network and involvement in RNA polymerase III-mediated transcription may suggest a potential functional association in the regulation of gene expression.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIAA0100](https://www.ncbi.nlm.nih.gov/gene?term=KIAA0100[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear biological pathway connecting ARNT2 and KIAA0100, so there are no obvious functional associations between these two genes. ARNT2 is involved in transcription regulation and postnatal brain growth, while KIAA0100 plays a role in lipid transport and interacts with the coxsackievirus.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional association: MYCBP2 is an E3 ubiquitin-protein ligase involved in the ubiquitination and subsequent proteasomal degradation of TSC2/tuberin, and it interacts with HSPD1, which is involved in facilitating correct folding of mitochondrial proteins and preventing misfolding under stress conditions. It is possible that HSPD1 plays a role in regulating the stability or folding of MYCBP2 or its targets.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There are no known functional associations between HSPA4 and SLC1A4. HSPA4 is a crucial regulator of mitosis, required for spindle assembly and cytokinesis, while SLC1A4 is a sodium-dependent amino acid transporter.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 245</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | There is not enough information to determine a direct functional association between ARHGEF12 and VAPB. ARHGEF12 is involved in the regulation of RhoA GTPase, while VAPB participates in the endoplasmic reticulum unfolded protein response and cellular calcium homeostasis regulation. However, they do interact in the BioGRID protein interaction network, possibly indicating a shared pathway or process.|
| [RAB11FIP2](https://www.ncbi.nlm.nih.gov/gene?term=RAB11FIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on their respective functions, RAB11FIP2 and TBP do not have any evident functional association. RAB11FIP2 is involved in membrane trafficking and phagocytosis, while TBP is a general transcription factor essential for RNA polymerase II-dependent transcription initiation.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.8</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | DLD and SLC1A4 interact in several ways, suggesting a functional association. DLD is involved in the glycine cleavage system and three alpha-ketoacid dehydrogenase complexes, while SLC1A4 is a neutral amino acid transporter for several amino acids. It is possible that DLD provides substrates for SLC1A4's transport function.|
| [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | COPB2 and SMC3 may be functionally associated in Golgi vesicular trafficking and spindle pole assembly during mitosis. COPB2 is involved in Golgi membrane binding and essential for Golgi budding, while SMC3 is a central component of cohesin complex required for chromosome cohesion during the cell cycle and spindle pole assembly during mitosis.|
| [APPBP2](https://www.ncbi.nlm.nih.gov/gene?term=APPBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 467</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association: APPBP2 is a substrate-recognition component of a CRL2 E3 ubiquitin-protein ligase complex while CLTC is a major protein of the polyhedral coat of coated pits and vesicles. These two genes may be involved in intracellular protein transport and autophagosome formation, potentially through their interactions in the TACC3/ch-TOG/clathrin complex.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.4</br>**STRING score:** 196</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 90 (min len 3) | Possible functional associations between HSPD1 and SMC3 include mitochondrial protein import, macromolecular assembly, and chromosome cohesion during the cell cycle. HSPD1 is a chaperonin involved in mitochondrial protein import, while SMC3 is a central component of cohesin complexes involved in chromosome cohesion during the cell cycle. While they do not directly interact, their association with these cellular processes could suggest possible indirect functional crosstalk.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.3</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 176 (min len 4) | One possible functional association between ABL1 and ATP11B is their involvement in membrane dynamics. ABL1 plays a role in cytoskeleton remodeling and endocytosis, while ATP11B catalyzes the transport of phospholipids between intracellular membranes, potentially contributing to the maintenance of membrane lipid asymmetry in endosome compartments. However, further investigation is needed to establish a direct functional link between these two genes in specific biological processes.|
| [CCP110](https://www.ncbi.nlm.nih.gov/gene?term=CCP110[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.9</br>**STRING score:** 607</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context, there is no clear functional association between CCP110 and CDK2 in TCGA-BRCA. CCP110 is involved in centrosome duplication, ciliogenesis, spindle formation, and cytokinesis. CDK2 controls the cell cycle and is essential for meiosis, but dispensable for mitosis. It phosphorylates various proteins involved in DNA synthesis, replication, repair, and gene transcription.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MMP2](https://www.ncbi.nlm.nih.gov/gene?term=MMP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.3</br>**STRING score:** 754</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between HIF1A and MMP2 in TCGA-BRCA include regulation of inflammation and promotion of tumor vascularization and invasion. HIF1A is involved in pro-inflammatory cytokine expression and is required for glycolysis induction in monocytes infected with SARS-CoV-2, while MMP2 promotes vascular invasion and interacts with integrin alpha-V/beta-3. These processes may contribute to tumorigenic properties of breast cancer cells.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.1</br>**STRING score:** 447</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between CREB1 and HSPA4 include their involvement in mitosis and cytokinesis regulation and spindle assembly. CREB1 also plays a role in transcription activation and adipose cell differentiation, while HSPA4 is a component of the chromosomal passenger complex and a kinase involved in mediating phosphorylation events.  However, there is currently no known direct biological relationship between these two genes.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.1</br>**STRING score:** 387</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 438 (min len 4) | There are no obvious functional associations between ARNT2 and TIMELESS, as they are involved in different biological processes. ARNT2 is a transcription factor involved in development and xenobiotic response, while TIMELESS is involved in DNA replication, repair, and the circadian clock.|
| [UBE3B](https://www.ncbi.nlm.nih.gov/gene?term=UBE3B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.9</br>**STRING score:** 353</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between UBE3B and UBQLN2 include their roles in the regulation of protein degradation mechanisms such as the ubiquitin-proteasome system, autophagy, and the endoplasmic reticulum-associated protein degradation pathway. They may also be involved in protein-protein interactions and transcriptional regulation. There is no direct evidence of a functional association between these two genes in TCGA-BRCA.|
| [IKZF1](https://www.ncbi.nlm.nih.gov/gene?term=IKZF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.6</br>**STRING score:** 411</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between IKZF1 and TBP based on their known biological roles. IKZF1 is a transcription regulator that plays a role in lymphocyte development and interacts with chromatin-remodeling complexes, while TBP is a general transcription factor that functions at the core of the TFIID complex and is involved in the initiation of RNA polymerase II-dependent transcription.|
| [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.4</br>**STRING score:** 742</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between POLE2 and SMC4 could be involvement in DNA replication and repair, as POLE2 is an accessory subunit of DNA polymerase epsilon complex and SMC4 is a central component of the condensin complex required for chromatin condensation during mitosis. They may interact through the protein interaction network to facilitate these processes.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.0</br>**STRING score:** 999</br>**BioGRID count:** 29</br>**BioGRID shortest paths:** 1 (min len 2) | EED and SUZ12 are components of the PRC2/EED-EZH2 complex, which methylates histones to repress the transcription of target genes. As they both play a role in the same complex, it is likely that they functionally interact to regulate gene expression through epigenetic modifications.|
| [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is currently no known functional association between SUZ12 and TBC1D15 in specific biological terms. SUZ12 is a component of the PRC2 complex, which methylates histones and leads to transcriptional repression of target genes, while TBC1D15 acts as a GTPase activating protein for RAB7A.|
