| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RAB11FIP2](https://www.ncbi.nlm.nih.gov/gene?term=RAB11FIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no clear functional associations between RAB11FIP2 and TXNDC9 based on their known roles in phagocytosis, vesicle trafficking, and protein folding. However, both genes are involved in regulating the transport of molecules to the plasma membrane, and may interact indirectly through other proteins in the network.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional association:      ARHGEF12 may regulate the activity of RhoA GTPase, while HSPD1 may be involved in mitochondrial protein import and assembly. There may be a functional association between the two genes in regulating mitochondrial function and/or stress response.|
| [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | A possible functional association between VAPB and WDR7 is in ribosome biogenesis. WDR7 is a ribosome biogenesis factor and VAPB participates in the endoplasmic reticulum unfolded protein response, which is involved in regulating RNA transcription and translation. However, more research is needed to fully understand any specific biological interactions between the two genes.|
| [PHKA1](https://www.ncbi.nlm.nih.gov/gene?term=PHKA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between PHKA1 and WDR7 in TCGA-BRCA cancer could involve nucleolar processing of pre-18S ribosomal RNA and optimal pre-ribosomal RNA transcription, as both genes are involved in ribosome biogenesis and function. However, further investigation would be necessary to determine any specific interactions or relationships between these two genes in cancer.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.5</br>**STRING score:** 981</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | Possible functional associations between CREB1 and TBP include their involvement in transcriptional regulation. CREB1 is a phosphorylation-dependent transcription factor that stimulates transcription by binding to DNA response elements, while TBP is a TATA-binding protein that plays a major role in the initiation of RNA polymerase II-dependent transcription. The protein interaction network suggests that CREB1 and TBP may interact through multiple pathways involving other proteins such as CREBBP, EP300, TP53, NR3C1, and GTF2A1.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIAA0100](https://www.ncbi.nlm.nih.gov/gene?term=KIAA0100[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no clear functional associations between ARNT2 and KIAA0100 based on their known roles in biology and the provided context. ARNT2 is a transcription factor involved in development and xenobiotic responses, while KIAA0100 is involved in lipid transport and interacts with a viral protease.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional associations: HSPD1 is involved in mitochondrial protein import and macromolecular assembly, while MYCBP2 is an atypical E3 ubiquitin-protein ligase that plays a key role in neural development, regulating axon growth, and axon navigation. There is no known direct biological association between these two genes.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: TCGA-BRCA cancer type shows the shortest path between HSPA4 and SLC1A4 in the BioGRID protein interaction network, suggesting they may interact in cellular processes related to mitosis and cytokinesis. HSPA4 is a component of the chromosomal passenger complex, while SLC1A4 is a sodium-dependent neutral amino-acid transporter. It is possible that SLC1A4 may play a role in providing amino acids necessary for HSPA4 function during mitosis and cytokinesis.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 245</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of functional associations between ARHGEF12 and VAPB based on the provided context and known biology. ARHGEF12 is involved in regulation of RhoA GTPase and may interact with IGF1R beta subunit, while VAPB participates in endoplasmic reticulum unfolded protein response and cellular calcium homeostasis regulation.|
| [RAB11FIP2](https://www.ncbi.nlm.nih.gov/gene?term=RAB11FIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no known direct functional associations between RAB11FIP2 and TBP. RAB11FIP2 is involved in regulating vesicle transport and cell polarity, while TBP is a general transcription factor involved in RNA polymerase II-dependent transcription initiation.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.8</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between DLD and SLC1A4 include DLD's role as a component of the pyruvate dehydrogenase complex and SLC1A4's function as a sodium-dependent neutral amino acid transporter. These two genes may be involved in regulating amino acid metabolism and transportation in cells.|
| [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between COPB2 and SMC3 include their involvement in protein trafficking, Golgi apparatus organization, spindle pole assembly during mitosis, and chromosome cohesion and movement. Both genes are essential components of large protein complexes required for these processes.|
| [APPBP2](https://www.ncbi.nlm.nih.gov/gene?term=APPBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 467</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association: APPBP2 and CLTC both play a role in intracellular protein transport and interact with microtubules. It is possible that they may function together in the transport of specific proteins or organelles within cells.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.4</br>**STRING score:** 196</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 90 (min len 3) | Possible functional associations between HSPD1 and SMC3 in TCGA-BRCA include involvement in protein folding and mitochondrial import (HSPD1) and chromosome cohesion during cell cycle and spindle pole assembly (SMC3). It is possible that dysfunction of these processes may contribute to cancer development, but specific mechanistic links between HSPD1 and SMC3 in this context are unclear.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.3</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 176 (min len 4) | Possible functional associations between ABL1 and ATP11B are currently unknown. ABL1 is a non-receptor tyrosine-protein kinase involved in many cellular processes such as cytoskeleton remodeling and DNA damage response, while ATP11B is a component of a P4-ATPase flippase complex involved in maintaining membrane lipid asymmetry. There is currently no evidence to connect these two genes in a specific biological process.|
| [CCP110](https://www.ncbi.nlm.nih.gov/gene?term=CCP110[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.9</br>**STRING score:** 607</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between CCP110 and CDK2 is their involvement in promoting correct spindle formation and regulation of genome stability. CCP110 negative regulates ciliogenesis, while CDK2 is involved in controlling the cell cycle, coordination of cyclin B/CDK1, and homologous recombination-dependent repair via phosphorylation of BRCA2. There is no known direct interaction between CCP110 and CDK2.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MMP2](https://www.ncbi.nlm.nih.gov/gene?term=MMP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.3</br>**STRING score:** 754</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between HIF1A and MMP2 include regulation of inflammation and angiogenesis, as well as promoting the replication of human coronavirus SARS-CoV-2 in monocytes. HIF1A is involved in hypoxia sensing and regulates the expression of target genes, while MMP2 is involved in extracellular matrix remodeling and is responsible for degrading various ECM components.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.1</br>**STRING score:** 447</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known functional association between CREB1 and HSPA4 based on their respective roles in transcription and mitosis regulation, unless new data emerges. CREB1 is a phosphorylation-dependent transcription factor that stimulates transcription upon binding to the DNA cAMP response element, while HSPA4 is a serine/threonine-protein kinase component of the chromosomal passenger complex, which is a key regulator of mitosis.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.1</br>**STRING score:** 387</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 438 (min len 4) | Possible functional associations between ARNT2 and TIMELESS might be regulating DNA replication and repair, and the circadian clock. ARNT2 plays a role in transcriptional regulation and is involved in the development of the hypothalamo-pituitary axis, while TIMELESS is required to stabilize replication forks during DNA replication and for DNA repair. They both interact with DNA and transcription factors, and may have a role in gene expression regulation.|
| [UBE3B](https://www.ncbi.nlm.nih.gov/gene?term=UBE3B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.9</br>**STRING score:** 353</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between UBE3B and UBQLN2 are their roles in the regulation of protein degradation mechanisms, as both genes play important roles in pathways such as ubiquitin-proteasome system and autophagy. UBE3B is a E3 ubiquitin-protein ligase involved in targeting substrates for degradation, while UBQLN2 mediates the proteasomal targeting of misfolded or accumulated proteins and plays a role in autophagy and endoplasmic reticulum-associated protein degradation pathways. Their interaction in the context of these pathways should be further explored.|
| [IKZF1](https://www.ncbi.nlm.nih.gov/gene?term=IKZF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.6</br>**STRING score:** 411</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between IKZF1 and TBP based on their known roles in biology. IKZF1 is a transcription regulator of hematopoietic cell differentiation, while TBP is a general transcription factor that functions at the core of the TFIID complex. The shortest path between the two in the BioGRID protein interaction network is a direct interaction.|
| [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.4</br>**STRING score:** 742</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association between POLE2 and SMC4 is their involvement in DNA replication and repair. POLE2 is an accessory component of the DNA polymerase epsilon complex, while SMC4 is a central component of the condensin complex required for chromatin condensation during mitosis. Both genes are involved in DNA processes and could potentially interact in these functions.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.0</br>**STRING score:** 999</br>**BioGRID count:** 29</br>**BioGRID shortest paths:** 1 (min len 2) | EED and SUZ12 are both components of the PRC2 complex, which methylates histone proteins and leads to transcriptional repression of target genes. As such, they likely work together to regulate gene expression through epigenetic modifications.|
| [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no clear functional association between SUZ12 and TBC1D15 in terms of their known biological roles. SUZ12 is a component of the polycomb repressive complex 2 involved in histone methylation and transcriptional repression, while TBC1D15 acts as a GTPase activating protein for RAB7A involved in intracellular trafficking.|
