| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35F2](https://www.ncbi.nlm.nih.gov/gene?term=SLC35F2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | AKT1 and SLC35F2 do not appear to have a direct functional association in TCGA-BRCA. AKT1 is involved in regulating many cellular processes, including metabolism, proliferation, and survival, while SLC35F2 is a putative solute transporter. The shortest path between the two genes in the BioGRID protein interaction network involves intermediate proteins CTBP1 and TP53.|
| [FBXO21](https://www.ncbi.nlm.nih.gov/gene?term=FBXO21[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between FBXO21 and VAPB could involve modulation of the endoplasmic reticulum unfolded protein response (UPR) and regulation of cellular calcium homeostasis. FBXO21 is involved in substrate recognition for the SCF E3 ubiquitin ligase complex, while VAPB participates in UPR and interacts with HCV proteins. The shortest paths in the BioGRID protein interaction network suggest 4 other genes that may also be involved in UPR modulation.|
| [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 98 (min len 4) | Possible functional association between TMEM5 and TRAPPC3 is their involvement in vesicular transport from endoplasmic reticulum to Golgi, with TMEM5 acting as a component of the complex FKTN/fukutin, FKRP and RXYLT1/TMEM5 forming a complex that participates in the biosynthesis of phosphorylated O-mannosyl trisaccharide which is required for binding laminin G-like domain-containing extracellular proteins with high affinity, whereas TRAPPC3 is a component of the multisubunit transport protein particle (TRAPP) complex.|
| [GNA15](https://www.ncbi.nlm.nih.gov/gene?term=GNA15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK9](https://www.ncbi.nlm.nih.gov/gene?term=MAPK9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 104 (min len 4) | Possible functional associations between GNA15 and MAPK9 include their involvement in transmembrane signaling cascades and the JNK signaling pathway. GNA15 is a guanine nucleotide-binding protein involved in modulating transmembrane signaling, while MAPK9 encodes a member of the JNK family of protein kinases involved in signaling cascades activated in response to environmental stress. The interaction between these two genes might suggest that they work together to transmit signals between the cell membrane and nucleus.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 4) | Possible functional association between ELOVL6 and PARP2 could be related to the repair of DNA damage, as PARP2 plays a key role in DNA repair and ELOVL6 is involved in the production of membrane lipids and lipid mediators, which are important for maintaining cellular integrity. The shortest path between these two genes in the BioGRID protein interaction network involves several other genes (CERS2, EGFR, LCP2, ORF7b, ESR2, and TRIM25) that may also participate in related processes such as lipid metabolism and signaling.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 56 (min len 4) | There is no known direct functional association between ELOVL6 and LIG1.	ELOVL6 is involved in the elongation of long-chain fatty acids, while LIG1 is responsible for repairing nicks in double-stranded DNA during DNA repair, replication, and recombination.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 3) | There are no clear functional associations between ACLY and RAE1 based on their known biological roles. ACLY is involved in citrate cleavage and subsequent cholesterol and fatty acid synthesis, while RAE1 functions primarily in mRNA export and cytoskeletal attachment. The only known interaction between these genes is through protein interaction, where ACLY may interact with RAE1 through through CUL3, UBC, FN1, CUL7, or OBSL1.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.5</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | There is no clear indication of any functional association between FKBP4 and HK1 in the provided context. However, FKBP4 plays a role in intracellular trafficking of steroid hormone receptors and regulates microtubule dynamics, while HK1 catalyzes the initial step of glycolysis and acts as a pattern recognition receptor for bacterial peptidoglycan.|
| [ATP1B1](https://www.ncbi.nlm.nih.gov/gene?term=ATP1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XPO7](https://www.ncbi.nlm.nih.gov/gene?term=XPO7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ATP1B1 and XPO7 are not clear from the provided information. ATP1B1 encodes for a subunit of the sodium/potassium-transporting ATPase involved in cell adhesion and epithelial cell polarity, while XPO7 mediates the nuclear export of proteins with broad substrate specificity.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | LYN and NCOA3 are linked through several protein interactions involved in immune responses, cytokine signaling, and integrin signaling. Their proximity in the shortest path suggests possible functional associations in these processes. However, their specific roles in regulating gene expression and chromatin remodeling indicate potential differences in their cellular functions.|
| [SCCPDH](https://www.ncbi.nlm.nih.gov/gene?term=SCCPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.3</br>**STRING score:** 272</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between SCCPDH and SQSTM1 could be in the autophagy pathway. SCCPDH is involved in mannosyl transfer reactions, while SQSTM1 acts as an autophagy receptor required for selective macroautophagy (aggrephagy). It is possible that SCCPDH may contribute to the synthesis of glycans required for SQSTM1-mediated autophagy.|
| [MAP7](https://www.ncbi.nlm.nih.gov/gene?term=MAP7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBKS](https://www.ncbi.nlm.nih.gov/gene?term=RBKS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.9</br>**STRING score:** 247</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no obvious functional associations between MAP7 and RBKS, as they are involved in different biological processes. MAP7 is a microtubule-stabilizing protein involved in cell polarization and differentiation, while RBKS is a ribose kinase involved in nucleotide synthesis and the pentose phosphate pathway. |
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX4](https://www.ncbi.nlm.nih.gov/gene?term=STX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.1</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | COG7 and STX4 are functionally associated in the Golgi transport process. COG7 is required for normal Golgi function while STX4 mediates docking of transport vesicles, necessary for translocation of SLC2A4 from intracellular vesicles to the plasma membrane. The shortest path in the BioGRID protein interaction network between the two genes involves multiple interactions with other proteins, potentially indicating a complex interplay of these genes in the transport process.|
| [EIF3E](https://www.ncbi.nlm.nih.gov/gene?term=EIF3E[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.1</br>**STRING score:** 178</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 68 (min len 3) | EIF3E and SRP14 are not directly functionally associated, but they both play important roles in translation initiation and co-translational targeting of proteins to the endoplasmic reticulum, respectively. EIF3E is a component of the eukaryotic translation initiation factor 3 (eIF-3) complex, which is required for several steps in the initiation of protein synthesis, while SRP14 is a component of the signal recognition particle (SRP) complex, which mediates the cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum.|
| [IKBKG](https://www.ncbi.nlm.nih.gov/gene?term=IKBKG[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKB2](https://www.ncbi.nlm.nih.gov/gene?term=NFKB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.1</br>**STRING score:** 980</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between IKBKG and NFKB2 is in the regulation of NF-kappa-B signaling pathway. IKBKG mediates activation of NF-kappa-B and interacts with SARS-CoV-2 virus protein, while NFKB2 is a subunit of the NF-kappa-B complex and regulates gene expression related to inflammation, immunity, differentiation, cell growth, tumorigenesis, and apoptosis.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF5](https://www.ncbi.nlm.nih.gov/gene?term=EIF5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known functional association between APP and EIF5, as they do not directly interact in the BioGRID protein interaction network or in the provided context.   APP is primarily involved in neuronal function and has been implicated in Alzheimer's disease pathogenesis. EIF5 is involved in translation initiation.|
| [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPIC](https://www.ncbi.nlm.nih.gov/gene?term=PPIC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.0</br>**STRING score:** 315</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: MLEC may play a role in protein N-glycosylation and interacts with the oligosaccharyltransferase complex while PPIC helps in protein folding and interacts with PCNA. They are connected in the BioGRID protein interaction network through a shortest path of five proteins with roles in various cellular processes. No direct functional associations between MLEC and PPIC have been reported in the literature.|
| [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PXN](https://www.ncbi.nlm.nih.gov/gene?term=PXN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.7</br>**STRING score:** 743</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: IQGAP1 and PXN both play important roles in regulating actin cytoskeleton dynamics and assembly, and are involved in cell adhesion to the extracellular matrix. They may interact in the formation of focal adhesions and contribute to cell cycle progression.|
| [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.4</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 92 (min len 3) | Possible functional associations between SKP1 and SQSTM1 could involve regulation of cell cycle progression and protein degradation through the SCF (SKP1-CUL1-F-box protein) ubiquitin ligase complex and autophagy, respectively. Additionally, SKP1 may participate in Wnt signaling, while SQSTM1 may play a role in endosome organization and NFE2L2/NRF2 pathway activation.|
