| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35F2](https://www.ncbi.nlm.nih.gov/gene?term=SLC35F2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | AKT1 and SLC35F2 have been found to be connected in the BioGRID protein interaction network via CTBP1 and TP53. AKT1 is a serine/threonine-protein kinase that regulates various biological processes including metabolism, proliferation, and cell survival. SLC35F2 is a putative solute transporter that interacts with SH3 domain-containing proteins. There is no known functional association between these two genes in specific biological terms.|
| [FBXO21](https://www.ncbi.nlm.nih.gov/gene?term=FBXO21[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are no apparent functional associations between FBXO21 and VAPB based on their known biology. FBXO21 is involved in substrate-recognition as a component of the SCF-type E3 ubiquitin ligase complex, while VAPB participates in ER unfolded protein response and cellular calcium homeostasis regulation. The BioGRID protein interaction network shows that the shortest path between the two genes involves several intermediary proteins such as TP53, RAF1 and TRIM67.|
| [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 98 (min len 4) | There's no clear indication of any direct functional association between TMEM5 and TRAPPC3. TMEM5 is involved in the biosynthesis of phosphorylated O-mannosyl trisaccharide, which is required for binding laminin G-like domain-containing extracellular proteins with high affinity, while TRAPPC3 is involved in vesicular transport from endoplasmic reticulum to Golgi as a component of the multisubunit transport protein particle (TRAPP) complex.|
| [GNA15](https://www.ncbi.nlm.nih.gov/gene?term=GNA15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK9](https://www.ncbi.nlm.nih.gov/gene?term=MAPK9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 104 (min len 4) | Possible functional associations between GNA15 and MAPK9 include their involvement in transmembrane signaling and phosphorylation of transcription factors. They may also be part of the JNK signaling pathway and interact with scaffolding proteins and other components of this pathway.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 4) | Possible functional associations between ELOVL6 and PARP2 include involvement in lipid metabolism and DNA repair. ELOVL6 catalyzes the elongation of long-chain fatty acids while PARP2 plays a key role in DNA repair. There may be a connection between altered lipid metabolism and DNA damage in cancer, but more research is needed to determine if there is a specific biological relationship between these two genes.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 56 (min len 4) | There is no clear biological functional association between ELOVL6 and LIG1. ELOVL6 is involved in fatty acid elongation, while LIG1 regulates signaling by receptor tyrosine kinases.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 3) | There are no known functional associations between ACLY and RAE1 in specific biological terms. ACLY catalyzes citrate cleavage for de novo synthesis of cholesterol and fatty acids, while RAE1 is involved in mRNA export and may play a role in attaching cytoplasmic mRNPs to the cytoskeleton and mitotic spindle formation.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.5</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | Possible functional associations between FKBP4 and HK1 may involve glucose metabolism, as HK1 is a key enzyme in the glycolysis pathway and FKBP4 may play a regulatory role in microtubule dynamics. The BioGRID protein interaction network suggests these genes may interact through the EGFR and MAPT/TAU pathways.|
| [ATP1B1](https://www.ncbi.nlm.nih.gov/gene?term=ATP1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XPO7](https://www.ncbi.nlm.nih.gov/gene?term=XPO7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations: ATP1B1 is involved in cell adhesion and establishing epithelial cell polarity, while XPO7 mediates the nuclear export of proteins with broad substrate specificity. They may work together in regulating the localization of certain proteins involved in cell adhesion or polarity, or in targeting specific proteins for export out of the nucleus.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | LYN and NCOA3 are functionally associated through their shortest path in the BioGRID protein interaction network. LYN is a non-receptor tyrosine-protein kinase involved in regulating immune responses, while NCOA3 is a nuclear receptor coactivator that stimulates transcriptional activities in a hormone-dependent fashion. In the context of TCGA-BRCA, their association may potentially play a role in the regulation of gene expression in breast cancer cells.|
| [SCCPDH](https://www.ncbi.nlm.nih.gov/gene?term=SCCPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.3</br>**STRING score:** 272</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: SQSTM1 is involved in autophagy and SCCPDH is involved in critical mannosyl transfer reactions. It is possible that SCCPDH plays a role in modulating autophagy through its involvement in the synthesis of dolichol-phosphate-mannose.|
| [MAP7](https://www.ncbi.nlm.nih.gov/gene?term=MAP7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBKS](https://www.ncbi.nlm.nih.gov/gene?term=RBKS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.9</br>**STRING score:** 247</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between MAP7 and RBKS may involve their roles in cytoskeletal organization and regulation of intracellular signaling pathways. MAP7 is involved in microtubule stabilization and formation of intercellular contacts, while RBKS catalyzes phosphorylation of ribose for nucleotide synthesis and the pentose phosphate pathway. There is no direct evidence of interaction between MAP7 and RBKS in current literature.|
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX4](https://www.ncbi.nlm.nih.gov/gene?term=STX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.1</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | COG7 and STX4 are functionally associated in vesicle trafficking and membrane fusion at the Golgi complex and plasma membrane. COG7 is a subunit of the conserved oligomeric Golgi complex, while STX4 is a plasma membrane t-SNARE that mediates docking of transport vesicles and facilitates fusion with the plasma membrane. The shortest path between COG7 and STX4 in the BioGRID protein interaction network involves several intermediates, suggesting a complex interaction network between these genes and other players in the vesicle trafficking machinery.|
| [EIF3E](https://www.ncbi.nlm.nih.gov/gene?term=EIF3E[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.1</br>**STRING score:** 178</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 68 (min len 3) | Possible functional association between EIF3E and SRP14 is their involvement in protein translation and targeting. EIF3E is a component of the eukaryotic translation initiation factor 3 (eIF-3) complex, while SRP14 is a component of the signal recognition particle (SRP) complex that mediates the cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum (ER). Therefore, EIF3E and SRP14 might interact in the process of protein translation and targeting to the ER.|
| [IKBKG](https://www.ncbi.nlm.nih.gov/gene?term=IKBKG[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKB2](https://www.ncbi.nlm.nih.gov/gene?term=NFKB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.1</br>**STRING score:** 980</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: IKBKG mediates NF-kappa-B activation and interacts with NFKB2 in the protein interaction network. NFKB2 is a component of the NF-kappa-B complex with distinct preferences for different kappa-B sites in DNA. These genes may work together to regulate transcriptional activity in response to various stimuli, including microbial infection and tumorigenesis.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF5](https://www.ncbi.nlm.nih.gov/gene?term=EIF5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known functional association between APP and EIF5. APP is involved in processes such as binding to cytoplasmic proteins and microtubules, while EIF5 is involved in the hydrolysis of GTP bound to the 40S ribosomal initiation complex.|
| [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPIC](https://www.ncbi.nlm.nih.gov/gene?term=PPIC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.0</br>**STRING score:** 315</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no evidence of a direct functional association between MLEC and PPIC. MLEC potentially plays a role in protein N-glycosylation, while PPIC is involved in protein folding and may interact with various receptors, including EGFR/ERBB family.|
| [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PXN](https://www.ncbi.nlm.nih.gov/gene?term=PXN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.7</br>**STRING score:** 743</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between IQGAP1 and PXN could be their role in regulating the actin cytoskeleton dynamics and assembly at sites of cell adhesion to the extracellular matrix. Both genes are involved in focal adhesion and may interact in promoting neurite outgrowth and cell cycle progression after DNA replication arrest.|
| [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.4</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 92 (min len 3) | SKP1 and SQSTM1 are involved in protein degradation pathways. SKP1 acts as an adapter in the SCF ubiquitin ligase complex, while SQSTM1 is an autophagy receptor. Their shortest path in the BioGRID protein interaction network suggests they may function together in ubiquitination and autophagy pathways.|
