| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CNPY3](https://www.ncbi.nlm.nih.gov/gene?term=CNPY3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFAP2A](https://www.ncbi.nlm.nih.gov/gene?term=TFAP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TFAP2A is a transcription factor involved in developmental processes, whereas CNPY3 is known as a TLR-specific co-chaperone required for innate and adaptive immune responses. There may be a regulatory relationship between these genes in the control of immune-related developmental processes.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: CD44 and NET1 are both involved in cell migration and adhesion, with CD44 serving as a receptor for extracellular matrix components and activating downstream signaling pathways including RHOA, while NET1 acts as a guanine nucleotide exchange factor for RhoA. It is possible that their interaction in the shortest path between them in the BioGRID protein interaction network is related to their shared involvement in these processes.|
| [LPAR2](https://www.ncbi.nlm.nih.gov/gene?term=LPAR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LRPAP1](https://www.ncbi.nlm.nih.gov/gene?term=LRPAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 814 (min len 4) | There are no known functional associations between LPAR2 and LRPAP1. LPAR2 is a receptor for lysophosphatidic acid that plays a key role in phospholipase C-beta signaling pathway, while LRPAP1 is a molecular chaperone for LDL receptor-related proteins that regulate their ligand binding activity along the secretory pathway.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLA2G15](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | The shortest path between IGF1R and PLA2G15 in the BioGRID network suggests a possible functional association. IGF1R encodes a receptor for insulin-like growth factor 1 while PLA2G15 encodes for a dual-activity enzyme involved in phospholipid breakdown. PLA2G15 has been implicated in pulmonary surfactant clearance and IGF1R has been shown to promote cell growth and proliferation, potentially linking these two genes in a pathway involved in cell growth and maintenance.|
| [S100A13](https://www.ncbi.nlm.nih.gov/gene?term=S100A13[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSKU](https://www.ncbi.nlm.nih.gov/gene?term=TSKU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 4) | Possible functional associations between S100A13 and TSKU include involvement in the regulation of developmental events, wound healing, cholesterol homeostasis, and neural stem cell differentiation, potentially through interactions with multiple signaling pathways such as Wnt and TGFB1. The shortest path in the BioGRID protein interaction network suggests that these genes may interact through the proteins SYT1, ORF3a, RPN2, FBXO7, UXS1, and ZRANB1.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCTA](https://www.ncbi.nlm.nih.gov/gene?term=TCTA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 45 (min len 4) | Possible functional associations between CD44 and TCTA may be in cell adhesion and migration, as CD44 plays a role in these processes and TCTA may be required for cellular fusion during osteoclastogenesis. However, more research is needed to determine if and how these genes interact in specific biological terms.|
| [PLA2G15](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPAG7](https://www.ncbi.nlm.nih.gov/gene?term=SPAG7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 4) | Possible functional associations between PLA2G15 and SPAG7 may relate to their involvement in cellular processes such as phospholipid homeostasis and ribosomal subunit assembly, respectively. However, based on the limited information provided, it is unclear how these processes may intersect or interact.|
| [CIAPIN1](https://www.ncbi.nlm.nih.gov/gene?term=CIAPIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between CIAPIN1 and MAP2K5 based on the provided context. CIAPIN1 is involved in Fe-S protein assembly and RNR activation, while MAP2K5 acts as a scaffold for MAPK signaling. However, they do interact via HSPA8 in the BioGRID protein interaction network.|
| [CNPY3](https://www.ncbi.nlm.nih.gov/gene?term=CNPY3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSKU](https://www.ncbi.nlm.nih.gov/gene?term=TSKU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 4) | Possible functional association: CNPY3 and TSKU may have a role in regulating immune response and wound healing since CNPY3 interacts with Toll-like receptors and TSKU is involved in various developmental events and wound healing processes. The shortest path in the BioGRID protein interaction network suggests that these genes may interact through the protein ORF3a.|
| [CAB39](https://www.ncbi.nlm.nih.gov/gene?term=CAB39[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN6](https://www.ncbi.nlm.nih.gov/gene?term=PTPN6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: PTPN6 modulates signaling by tyrosine phosphorylated cell surface receptors such as EGFR, while CAB39 is a component of a complex that binds and activates STK11/LKB1. EGFR is part of the shortest path between CAB39 and PTPN6 in the BioGRID protein interaction network. Therefore, it is possible that PTPN6 and CAB39 are involved in regulating EGFR signaling through STK11/LKB1 activation.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between EPN2 and NRAS could be involvement in clathrin-mediated endocytosis and/or regulation of Ras signaling pathways. EPN2 plays a role in the formation of clathrin-coated invaginations and endocytosis, while NRAS is a member of the Ras family of small GTPases that regulate signaling pathways involved in cell proliferation, differentiation, and survival.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between ELAVL1 and PAN2 in the context of TCGA-PRAD. ELAVL1 is an RNA-binding protein that stabilizes mRNAs, while PAN2 is a catalytic subunit of the poly(A)-nuclease deadenylation complex involved in mRNA turnover.|
| [ALDOA](https://www.ncbi.nlm.nih.gov/gene?term=ALDOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations between ALDOA and CD44 include their interactions within the protein interaction network and their involvement in cell migration and adhesion, as well as signal transduction pathways and regulation of cytoskeleton reorganization. ALDOA is involved in glycolysis and may also function as a scaffolding protein, while CD44 plays a role in cell-cell interactions, activation and homing of T-lymphocytes, hematopoiesis, inflammation, and response to bacterial infection.|
| [AP1B1](https://www.ncbi.nlm.nih.gov/gene?term=AP1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.1</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 62 (min len 3) | There is no clear evidence of a functional association between AP1B1 and ELAVL1 in biological terms based on their known roles in protein sorting and mRNA stabilization, respectively. However, given their interaction via protein-protein networks, it is possible that they may interact or affect each other's functions indirectly.|
| [CSNK2A2](https://www.ncbi.nlm.nih.gov/gene?term=CSNK2A2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.1</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | CSNK2A2 and TBC1D15 are not directly linked in terms of biological function, as CSNK2A2 is involved in numerous cellular processes, such as cell cycle progression and apoptosis regulation, while TBC1D15 acts as a GTPase activating protein for RAB7A. However, the shortest path between them in the BioGRID protein interaction network suggests potential indirect interactions or functional associations through intermediates such as MEPCE, TERF2, UBC, WWOX, and SQSTM1.|
| [AP1B1](https://www.ncbi.nlm.nih.gov/gene?term=AP1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CIAPIN1](https://www.ncbi.nlm.nih.gov/gene?term=CIAPIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between AP1B1 and CIAPIN1 based on the provided context and the shortest protein interaction network path. AP1B1 is a subunit of clathrin-associated adaptor protein complex 1 involved in protein sorting, while CIAPIN1 plays a role in cytosolic iron-sulfur protein assembly, and has anti-apoptotic effects.|
| [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SIRT3](https://www.ncbi.nlm.nih.gov/gene?term=SIRT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.9</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between HMGCS1 and SIRT3 is in their roles in regulating cellular energy metabolism. HMGCS1 catalyzes the first step in the mevalonate pathway, which produces cholesterol, while SIRT3 deacetylates key lysine residues in mitochondrial target proteins, including those involved in energy metabolism. There is also a direct protein-protein interaction between these two genes demonstrated by the shortest path in the BioGRID interaction network.|
