| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [SLC25A22](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A22[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There do not seem to be any direct functional associations between SLC25A22 and WDR7 based on their individual roles in mitochondrial glutamate transport and ribosome biogenesis, respectively. However, both genes have been shown to interact with proteins involved in insulin secretion and RNA processing.|
| [CHMP4A](https://www.ncbi.nlm.nih.gov/gene?term=CHMP4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are currently no known functional associations between CHMP4A and MAPK1IP1L. However, CHMP4A is involved in endosomal sorting required for transport and MVB formation, while MAPK1IP1L may regulate transcriptional activity and interacts with ZNF202.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | RFC2 and WDR7 are functionally associated through protein-protein interactions involved in DNA replication and ribosome biogenesis, respectively. RFC2 is involved in DNA polymerase activity while WDR7 is a ribosome biogenesis factor. The shortest path between the two genes was found through protein interactions with MYC, KRAS, and other proteins involved in cell survival and cell cycle regulation.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF451](https://www.ncbi.nlm.nih.gov/gene?term=ZNF451[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | ZNF451 is a SUMO-protein ligase, while RFC2 is a subunit of the RFC complex involved in DNA replication. They are connected through a protein interaction network and RFC2 also interacts with PRKAR1A. There is no apparent direct functional association between these two genes in specific biological terms.|
| [ADH5](https://www.ncbi.nlm.nih.gov/gene?term=ADH5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There isn't enough information to determine any possible functional associations between ADH5 and WDR7 in specific biological terms. However, ADH5 catalyzes the oxidation of long-chain primary alcohols and detoxifies carcinogenic metabolites, while WDR7 is involved in ribosome biogenesis by processing pre-18S ribosomal RNA.|
| [CSRP1](https://www.ncbi.nlm.nih.gov/gene?term=CSRP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no clear functional association between CSRP1 and HPRT1. CSRP1 is involved in neuronal development and interacts with proteins involved in transcription and DNA repair, while HPRT1 plays a central role in purine nucleotide generation through the salvage pathway.|
| [ARNT2](https://www.ncbi.nlm.nih.gov/gene?term=ARNT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SDHB](https://www.ncbi.nlm.nih.gov/gene?term=SDHB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between ARNT2 and SDHB is that they both play a role in cellular metabolism. ARNT2 is a transcription factor involved in xenobiotic response, while SDHB is a subunit of the mitochondrial succinate dehydrogenase complex in the respiratory chain. The shortest path of protein interaction between these two genes involves CLPP, which is a mitochondrial protease involved in protein quality control.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFP36](https://www.ncbi.nlm.nih.gov/gene?term=ZFP36[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ECH1 and ZFP36 are unclear based solely on the provided information. ECH1 is involved in isomerization of 3-trans,5-cis-dienoyl-CoA to 2-trans,4-trans-dienoyl-CoA, while ZFP36 interacts with HTLV-1 TAX to inhibit its ability to transactivate viral long terminal repeat promoter.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF318](https://www.ncbi.nlm.nih.gov/gene?term=ZNF318[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: RFC2 and ZNF318 may interact in cell survival and transcriptional regulation during spermatogenesis and meiotic division. The shortest path between these genes in the protein interaction network involves proteins that are also involved in transcriptional regulation, such as BRD4 and MYC.|
| [MSRA](https://www.ncbi.nlm.nih.gov/gene?term=MSRA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MSRA and NCAPD2 in TCGA-PRAD could involve the regulation of mitotic chromosome condensation and DNA condensation. MSRA may interact with enzymes responsible for fatty acid elongation, while NCAPD2 is a regulatory subunit of the condensin complex, which plays a role in the conversion of interphase chromatin into mitotic-like condensed chromosomes.|
| [NOP16](https://www.ncbi.nlm.nih.gov/gene?term=NOP16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | NOP16 and PAN2 are not functionally associated, as NOP16 is involved in ATP/GTP synthesis in the citric acid cycle, while PAN2 is a deadenylase involved in mRNA degradation and HIF1A regulation. There is no clear overlap in the processes where these genes are involved.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.2</br>**STRING score:** 332</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | ELAVL1 and WDR7 are not known to have any direct functional association. ELAVL1 is an RNA-binding protein that increases the stability of target mRNAs and is involved in embryonic stem cell differentiation and regulating the nuclear export of p53/TP53 mRNA. WDR7 is a ribosome biogenesis factor involved in nucleolar processing of pre-18S ribosomal RNA.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 424</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TRIB1 may regulate myeloid cell differentiation by altering the expression of CEBPA in a COP1-dependent manner, and APOE may be involved in viral attachment and infectivity, including with hepatitis C virus (HCV) envelope glycoprotein E2. Given the involvement of immune cells in antiviral defense, it's possible that TRIB1 and APOE may have a combined role in regulating the response to viral infections in myeloid cells.|
| [PIK3C2B](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C2B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no immediate evidence of functional association between PIK3C2B and RFC2 based on the protein interaction network and gene function. PIK3C2B is involved in phosphorylation of PtdIns and PtdIns4P, potentially involved in EGFR and PDGFR signaling. RFC2 is involved in DNA elongation via ATP binding.|
| [GNAI1](https://www.ncbi.nlm.nih.gov/gene?term=GNAI1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 532</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between GNAI1 and RFC2 could involve the regulation of DNA replication, repair, and/or cell survival. GNAI1 is involved in signaling pathways downstream of G protein-coupled receptors, while RFC2 is an accessory subunit of the DNA polymerase delta/epsilon complex. The shortest path found in the BioGRID protein interaction network suggests physical interaction between the two proteins.|
| [PRKACA](https://www.ncbi.nlm.nih.gov/gene?term=PRKACA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: There is no known functional association between PRKACA and SRP14.          PRKACA: Phosphorylates and activates various protein targets involved in cellular processes such as sperm capacitation and platelet activation. It also plays a role in regulating the cAMP-dependent protein kinase holoenzyme.          SRP14: Is a component of the signal recognition particle complex which helps to target secretory and membrane proteins to the endoplasmic reticulum during translation. It forms a heterodimer with SRP9 and binds RNA.|
| [DNMT3A](https://www.ncbi.nlm.nih.gov/gene?term=DNMT3A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.9</br>**STRING score:** 184</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations: Both DNMT3A and ELAVL1 are involved in transcriptional regulation. DNMT3A is a DNA methyltransferase that plays a role in establishing DNA methylation patterns during development, while ELAVL1 is an RNA-binding protein that stabilizes mRNA molecules. While there is no clear direct relationship between these two genes, they may have some overlap in their downstream targets and could potentially be involved in regulating common biological pathways.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 157</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between APOE and EPN2 include endocytosis and viral infection. APOE is involved in HCV infection, while EPN2 plays a role in clathrin-coated invaginations and endocytosis. They may work together to facilitate HCV attachment and entry into cells via endocytosis.|
| [AP1B1](https://www.ncbi.nlm.nih.gov/gene?term=AP1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSTA3](https://www.ncbi.nlm.nih.gov/gene?term=TSTA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 295</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | AP1B1 and TSTA3 are not known to have any direct functional associations, but both are involved in important cellular processes. AP1B1 is a subunit of the clathrin-associated adaptor protein complex 1, which plays a role in protein sorting in the Golgi/trans-Golgi network and/or endosomes. TSTA3 catalyzes the conversion of GDP-4-dehydro-6-deoxy-D-mannose to GDP-fucose, which is important for glycosylation of proteins and lipids.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR7](https://www.ncbi.nlm.nih.gov/gene?term=WDR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 150</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | There is no obvious functional association between CYCS and WDR7 based on their known roles in apoptosis and ribosome biogenesis, respectively. However, the protein interaction network suggests a possible direct physical interaction or indirect functional association between these two genes through other intermediate proteins.|
| [SYF2](https://www.ncbi.nlm.nih.gov/gene?term=SYF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 378</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 649 (min len 4) | Possible functional association between SYF2 and TBC1D15 is in the regulation of protein degradation through interaction with CUL3, as evidenced by the shortest path in the BioGRID protein interaction network. SYF2 is involved in pre-mRNA splicing as part of the spliceosome, while TBC1D15 acts as a GTPase activating protein for RAB7A.|
