| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CORO1A](https://www.ncbi.nlm.nih.gov/gene?term=CORO1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.1</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | There is no clear functional association between CORO1A and PHGDH based on their known biological processes. CORO1A is involved in cytoskeletal organization and binds actin, while PHGDH is involved in serine biosynthesis and oxidation of various metabolites.|
| [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STUB1](https://www.ncbi.nlm.nih.gov/gene?term=STUB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 158.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association between CPNE3 and STUB1 is in the regulation of protein degradation. CPNE3 may play a role in ERBB2-mediated tumor cell migration while STUB1 targets misfolded chaperone substrates towards proteasomal degradation. They may interact through the ubiquitination pathway.|
| [HN1L](https://www.ncbi.nlm.nih.gov/gene?term=HN1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCAND1](https://www.ncbi.nlm.nih.gov/gene?term=SCAND1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 72 (min len 4) | There are no known functional associations between HN1L and SCAND1 based on their biological roles or protein interaction network. HN1L appears to be involved in endolysosomal trafficking of human coronavirus SARS-CoV-2, while SCAND1 may regulate transcriptional activity and interacts with ZNF202.|
| [DAG1](https://www.ncbi.nlm.nih.gov/gene?term=DAG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMF1](https://www.ncbi.nlm.nih.gov/gene?term=PMF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: None evident based on available information.   DAG1 acts as a receptor for viruses and mycobacteria, while PMF1 is part of the MIS12 complex required for chromosome alignment and kinetochore formation. There is no direct evidence of functional association between these two genes in specific biological processes or pathways.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 143.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 100 (min len 4) | There is no clear functional association between CEBPD and POLG2 based on their known biological roles. CEBPD is a transcription activator involved in immune and inflammatory responses, while POLG2 is a mitochondrial polymerase processivity subunit that promotes processive DNA synthesis.|
| [DUSP14](https://www.ncbi.nlm.nih.gov/gene?term=DUSP14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 143.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear functional association between DUSP14 and TPM1 based on their known roles in MAP kinase inactivation and actin filament binding. However, the protein interaction network shows potential for physical interaction between the two proteins via EGFR, RHOBTB2, ESR2, APEX1, or CAPZA2.|
| [HLA-DRA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DRA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between HLA-DRA and PHGDH could be their involvement in the immune response against cancer cells. HLA-DRA plays a role in presenting antigens to T-cells for recognition and activation, while PHGDH is involved in the biosynthesis of L-serine, an amino acid important for immune function.|
| [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STUB1](https://www.ncbi.nlm.nih.gov/gene?term=STUB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.8</br>**STRING score:** 948</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: STUB1 is known to negatively regulate TGF-beta signaling by modulating the basal level of SMAD3 via ubiquitin-mediated degradation. Both genes are involved in protein degradation and can regulate TGF-beta signaling, suggesting a potential functional association in this pathway.|
| [CCDC85B](https://www.ncbi.nlm.nih.gov/gene?term=CCDC85B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 203 (min len 4) | Possible functional associations between CCDC85B and NET1 may be related to their interactions with ESR1, a transcription factor involved in hormone signaling. CCDC85B has also been implicated in viral replication, while NET1 is involved in transcriptional regulation through complex formation with RFX proteins.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HINT1](https://www.ncbi.nlm.nih.gov/gene?term=HINT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 714 (min len 4) | CEBPD and HINT1 interact in the protein network through the BTRC protein. CEBPD is a transcription activator involved in immune and inflammatory responses and enhances IL6 transcription, while HINT1 exhibits adenosine 5'-monophosphoramidase activity and modulates transcriptional activation and proteasomal degradation by interacting with various proteins. There is no clear functional association between these two genes in specific biological terms.|
| [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.6</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 342 (min len 4) | Possible functional association between HMGCS1 and NR1H2 is in regulating cholesterol metabolism. HMGCS1 catalyzes the synthesis of HMG-CoA, a precursor for cholesterol synthesis, while NR1H2 regulates cholesterol uptake and metabolism. Both genes also interact with SIRT1, a protein involved in metabolism and aging.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.7</br>**STRING score:** 183</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | A possible functional association between HSPB1 and PTPN1 could be in regulating cellular stress responses. HSPB1 is a molecular chaperone that helps maintain protein folding and plays a role in stress resistance, while PTPN1 is a phosphatase involved in the endoplasmic reticulum unfolded protein response and cellular signaling pathways.|
| [RALGDS](https://www.ncbi.nlm.nih.gov/gene?term=RALGDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPRED2](https://www.ncbi.nlm.nih.gov/gene?term=SPRED2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.2</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between RALGDS and SPRED2 may include negative regulation of Ras signaling pathways and downstream activation of MAP kinases, and inhibition of fibroblast growth factor-induced retinal lens fiber differentiation. It is possible that RALGDS activates R-Ras, H-Ras, K-Ras, and Rap and interacts with TRAF3, while SPRED2 recruits and translocates NF1 to the cell membrane, interacts with RAS, and may interact with ZDHHC13 and ZDHHC17. The shortest path between RALGDS and SPRED2 in the BioGRID protein interaction network is through KRAS.|
| [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.8</br>**STRING score:** 285</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: PNKP plays a key role in DNA damage repair while PGRMC1 is involved in heme synthesis and homeostasis; both processes are important for cellular survival and functioning. It is possible that PGRMC1 regulates the activity or expression of PNKP in some way.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.6</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 45 (min len 3) | Possible functional associations between PHGDH and SRC could be related to cancer development and progression, as both genes have been implicated in these processes. PHGDH is involved in the biosynthesis of serine, which is important for cancer cell growth and proliferation, while SRC is a tyrosine kinase that regulates cell signaling pathways and has been shown to be involved in tumor invasion and metastasis. The protein interaction network between PHGDH and SRC suggests a potential pathway for their interaction in cancer cells.|
| [HIST1H2BK](https://www.ncbi.nlm.nih.gov/gene?term=HIST1H2BK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUV39H1](https://www.ncbi.nlm.nih.gov/gene?term=SUV39H1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.4</br>**STRING score:** 655</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their interaction in the BioGRID network, HIST1H2BK and SUV39H1 may have a functional association in regulating transcriptional repression through H3 'Lys-9' trimethylation, with SUV39H1 playing a key role in metabolic sensing and silencing rDNA in response to cellular stress. HIST1H2BK, on the other hand, has broad antibacterial activity and may contribute to the formation of the functional antimicrobial barrier of the colonic epithelium and the bactericidal activity of amniotic fluid.|
| [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT5B](https://www.ncbi.nlm.nih.gov/gene?term=STAT5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.8</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between RHEB and STAT5B could be related to the regulation of apoptosis and transcriptional activation in hematopoietic/erythroid differentiation. RHEB activates the protein kinase activity of mTORC1 while STAT5B carries out a dual function of signal transduction and activation of transcription. The shortest path between the two genes in the BioGRID protein interaction network involves several intermediate proteins, including TRIM25, EGFR, and NR3C1, suggesting that there may be complex regulatory processes linking RHEB and STAT5B.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS6](https://www.ncbi.nlm.nih.gov/gene?term=RPS6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.5</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between PRKAG2 and RPS6 in TCGA-LAML are related to cell growth, proliferation, and metabolism. PRKAG2 is an energy sensor protein kinase that regulates energy metabolism and inhibits cell growth and proliferation, while RPS6 is a ribosomal protein that plays a role in translation and controlling cell growth and proliferation. The interaction between PRKAG2 and RPS6 may be involved in modulating these processes in TCGA-LAML.|
| [DNAJB2](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both DNAJB2 and SRP14 are involved in protein quality control and may function together in ER-associated protein degradation.          DNAJB2 functions as a co-chaperone, regulating the substrate binding and promoting the degradation of misfolded proteins, while SRP14 is a component of the SRP complex that mediates the cotranslational targeting of secretory and membrane proteins to the ER. Both genes are involved in important biological processes related to protein folding and degradation in the ER.|
| [CHMP4A](https://www.ncbi.nlm.nih.gov/gene?term=CHMP4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.0</br>**STRING score:** 217</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no obvious functional association between CHMP4A and PHGDH. CHMP4A is involved in endosomal sorting required for transport complex III while PHGDH catalyzes the reversible oxidation of 3-phospho-D-glycerate to 3-phosphonooxypyruvate in the L-serine biosynthesis pathway.|
