| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CORO1A](https://www.ncbi.nlm.nih.gov/gene?term=CORO1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.1</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CORO1A and PHGDH may involve regulation of cell motility and metabolism. CORO1A is involved in cytoskeletal organization and plasma membrane dynamics, while PHGDH catalyzes reactions involved in serine and malate metabolism. The shortest path in the BioGRID protein interaction network suggests they may interact indirectly through other proteins involved in cell division and proliferation.|
| [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STUB1](https://www.ncbi.nlm.nih.gov/gene?term=STUB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 158.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | CPNE3 is involved in ERBB2-mediated tumor cell migration, while STUB1 is an E3 ubiquitin-protein ligase that targets misfolded chaperone substrates for degradation. The shortest path in the protein interaction network suggests a possible direct interaction between the two genes, but further experimentation will be needed to confirm any functional association between them.|
| [HN1L](https://www.ncbi.nlm.nih.gov/gene?term=HN1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCAND1](https://www.ncbi.nlm.nih.gov/gene?term=SCAND1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 72 (min len 4) | There is currently no known functional association between the genes HN1L and SCAND1. HN1L is involved in endolysosomal trafficking of SARS-CoV-2, while SCAND1 may regulate transcriptional activity and interact with ZNF202.|
| [DAG1](https://www.ncbi.nlm.nih.gov/gene?term=DAG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMF1](https://www.ncbi.nlm.nih.gov/gene?term=PMF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between DAG1 and PMF1 is their involvement in cell cycle regulation and mitosis. DAG1 acts as a receptor for Lassa virus and lymphocytic choriomeningitis virus glycoprotein, while PMF1 is part of the MIS12 complex required for proper chromosome segregation during mitosis and interacts with NFE2L2 for transcriptional regulation of SSAT.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 143.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 100 (min len 4) | There is no clear functional association between CEBPD and POLG2 in terms of their known biological roles. CEBPD is a transcriptional activator involved in immune and inflammatory responses, while POLG2 is a mitochondrial polymerase processivity subunit. The protein interaction network indicates a potential indirect interaction through PARP1, SMAD3/4, RELA, HDAC1/3, which could suggest a link between CEBPD-mediated transcriptional regulation and mitochondrial DNA maintenance.|
| [DUSP14](https://www.ncbi.nlm.nih.gov/gene?term=DUSP14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 143.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Based on their protein interaction network, DUSP14 and TPM1 may have a functional association in regulating cytoskeleton organization and smooth muscle contraction. DUSP14 is involved in the inactivation of MAP kinases, while TPM1 plays a central role, in association with the troponin complex, in calcium-dependent regulation of vertebrate striated muscle contraction and stabilizing cytoskeleton actin filaments.|
| [HLA-DRA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DRA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no known direct functional association between HLA-DRA and PHGDH. HLA-DRA is an antigen-presenting protein, while PHGDH catalyzes key steps in serine biosynthesis and oxaloacetate production.|
| [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STUB1](https://www.ncbi.nlm.nih.gov/gene?term=STUB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.8</br>**STRING score:** 948</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | One possible functional association between SMAD3 and STUB1 is the regulation of TGF-beta signaling. SMAD3 is an intracellular signal transducer and transcriptional modulator activated by TGF-beta, while STUB1 negatively regulates TGF-beta signaling by modulating the basal level of SMAD3 via ubiquitin-mediated degradation. Additionally, STUB1 is an E3 ubiquitin-protein ligase that can target misfolded chaperone substrates towards proteasomal degradation.|
| [CCDC85B](https://www.ncbi.nlm.nih.gov/gene?term=CCDC85B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 203 (min len 4) | Possible functional associations between CCDC85B and NET1 in TCGA-LAML cancer: CCDC85B may play a role in hepatitis delta virus genomic replication, while NET1 acts as a guanine nucleotide exchange factor for RhoA GTPase and is involved in activation of the SAPK/JNK pathway. There is currently no known direct biological process connecting these two genes.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HINT1](https://www.ncbi.nlm.nih.gov/gene?term=HINT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 714 (min len 4) | CEBPD and HINT1 have a direct protein interaction according to the shortest path in the BioGRID network. CEBPD is a transcriptional activator involved in immune and inflammatory responses, while HINT1 exhibits adenosine monophosphoramidase activity and modulates proteasomal degradation and protein SUMOylation.  The functional association between CEBPD and HINT1 in a biological process needs further investigation.|
| [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.6</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 342 (min len 4) | Possible functional associations between HMGCS1 and NR1H2 include a regulatory role of NR1H2 in the cholesterol uptake process through the ubiquitination of LDL receptors, which HMGCS1 is involved in the synthesis of cholesterol precursors. HMGCS1 also plays a role in mevalonate production, which may impact the regulation of lipid metabolism modulated by NR1H2.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.7</br>**STRING score:** 183</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no clear functional associations between HSPB1 and PTPN1 based on the limited information provided. HSPB1 is involved in stress resistance and acts as a molecular chaperone, while PTPN1 is a tyrosine-protein phosphatase that regulates the unfolded protein response and may play a role in signal transduction cascades.|
| [RALGDS](https://www.ncbi.nlm.nih.gov/gene?term=RALGDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPRED2](https://www.ncbi.nlm.nih.gov/gene?term=SPRED2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.2</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: RALGDS activates RalA or RalB GTPases and interacts with Ras, while SPRED2 negatively regulates Ras signaling and interacts with Ras. Both genes play a role in regulating intracellular transport, and SPRED2 recruits and translocates NF1 to the cell membrane to inhibit active GTP-bound Ras. The shortest path in the BioGRID protein interaction network between these genes is via KRAS. These genes may be involved in regulating Ras signaling and intracellular transport processes.|
| [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.8</br>**STRING score:** 285</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear biological association between PGRMC1 and PNKP based on their reported functions, except for both genes having roles in DNA damage repair. PGRMC1 is involved in heme homeostasis and interacts with cytochrome P450 enzymes, while PNKP has catalytic activities in the non-homologous end-joining and base excision repair pathways.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.6</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 45 (min len 3) | Possible functional associations between PHGDH and SRC in TCGA-LAML cancer may involve regulation of cellular metabolism and oncogenic signaling pathways. PHGDH catalyzes the first step of phosphorylated L-serine biosynthesis and also oxidizes 2-hydroxyglutarate and (S)-malate, while SRC is a non-receptor protein tyrosine kinase involved in neurite elongation and interacts with HCV non-structural protein 5A. The BioGRID protein interaction network shows a shortest path between PHGDH and SRC, suggesting a potential direct physical interaction between these two proteins.|
| [HIST1H2BK](https://www.ncbi.nlm.nih.gov/gene?term=HIST1H2BK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUV39H1](https://www.ncbi.nlm.nih.gov/gene?term=SUV39H1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.4</br>**STRING score:** 655</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between HIST1H2BK and SUV39H1 include epigenetic regulation of gene expression through histone methylation and chromatin organization. Both genes are involved in establishing and maintaining constitutive heterochromatin regions, with SUV39H1 specifically trimethylating histone H3 at lysine 9 to facilitate recruitment of HP1 proteins. HIST1H2BK is also involved in antibacterial defense and may contribute to the formation of the antimicrobial barrier in the colonic epithelium.|
| [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT5B](https://www.ncbi.nlm.nih.gov/gene?term=STAT5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.8</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between RHEB and STAT5B in TCGA-LAML cancer may include regulation of apoptosis and hematopoietic/erythroid differentiation. RHEB activates the protein kinase activity of mTORC1, while STAT5B carries out signal transduction and activation of transcription in response to cytokines and growth factors. The shortest path between RHEB and STAT5B in the BioGRID protein interaction network involves several intermediate proteins, including TRIM25, NR3C1, EGFR, and TRIM67.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS6](https://www.ncbi.nlm.nih.gov/gene?term=RPS6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.5</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between PRKAG2 and RPS6 include regulation of cell growth and proliferation through selective mRNA translation, and control of cellular energy metabolism via AMPK signaling and ribosomal subunit synthesis. It is unclear if these genes directly interact, but their roles in these biological processes suggest potential crosstalk between metabolic and translational pathways.|
| [DNAJB2](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between DNAJB2 and SRP14 could relate to protein folding and targeting to the endoplasmic reticulum (ER). DNAJB2 functions as a co-chaperone regulating the substrate binding and activating the ATPase activity of HSP70 family chaperones, while SRP14 is a component of the signal recognition particle (SRP) complex that mediates the cotranslational targeting of secretory and membrane proteins to the ER.|
| [CHMP4A](https://www.ncbi.nlm.nih.gov/gene?term=CHMP4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.0</br>**STRING score:** 217</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association between CHMP4A and PHGDH could be their involvement in intracellular vesicle trafficking and metabolism, respectively, as both genes play a role in cellular processes that require membrane curvature and fission.|
