| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CHIC2](https://www.ncbi.nlm.nih.gov/gene?term=CHIC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 132 (min len 4) | Possible functional associations between CHIC2 and STK10 include regulation of cell cycle and cell migration. CHIC2 is a regulatory component of the cyclin D1-CDK4 complex that phosphorylates and inhibits the retinoblastoma protein family, while STK10 is a serine/threonine kinase involved in regulation of lymphocyte migration and acts as a negative regulator of MAP3K1/MEKK1. There is a shortest path between CHIC2 and STK10 in the BioGRID protein interaction network.|
| [KIAA0753](https://www.ncbi.nlm.nih.gov/gene?term=KIAA0753[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALGDS](https://www.ncbi.nlm.nih.gov/gene?term=RALGDS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 165.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 214 (min len 4) | Possible functional associations between KIAA0753 and RALGDS involve their roles in intracellular transport, centrosomal localization, and potentially cilium assembly. KIAA0753 positively regulates CEP63 centrosomal localization and promotes the centrosomal localization of CDK2, while RALGDS functions as a guanine nucleotide exchange factor (GEF) activating either RalA or RalB GTPases and plays a role in intracellular transport. No direct biological interactions have been reported between these two genes.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between LYN and PRKAG2 is regulation of cellular energy metabolism and cell growth/proliferation, as LYN plays a role in response to growth factors and cytokines, while PRKAG2 is an energy sensor protein kinase that regulates energy-producing and consuming processes. There are no direct protein-protein interactions reported in the BioGRID network, but the shortest path between these genes includes proteins involved in gene expression regulation and actin cytoskeleton remodeling, potentially suggesting indirect interactions.|
| [ACBD3](https://www.ncbi.nlm.nih.gov/gene?term=ACBD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 157.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional association: ACBD3 plays a role in viral replication while CRKL may mediate intracellular signal transduction. There is no known direct biological pathway linking these two genes.|
| [FDFT1](https://www.ncbi.nlm.nih.gov/gene?term=FDFT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between FDFT1 and MLEC could be related to protein N-glycosylation and sterol biosynthesis, as MLEC is involved in N-glycan recognition and FDFT1 catalyzes the condensation of two farnesyl pyrophosphates to form squalene, a precursor for sterol biosynthesis. There is no direct interaction between these two genes, but their products may have indirect effects on each other through shared metabolic pathways.|
| [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear functional association between HMGCS1 and PUM2 based on the provided information. HMGCS1 encodes a protein involved in cholesterol synthesis, while PUM2 is an RNA-binding protein with various post-transcriptional regulatory roles. The shortest path in the BioGRID protein interaction network may hint at a potential interaction, but further research is needed to confirm any functional relationship.|
| [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | LSM6 is involved in pre-mRNA splicing and RNA degradation, while PGRMC1 is a progesterone-binding protein and a heme chaperone. There are no known direct functional associations between these two genes based on their described functions, but the BioGRID protein interaction network shows a shortest path between them involving proteins involved in DNA replication, repair, and phosphorylation, as well as receptor tyrosine kinase EGFR and small GTPase KRAS.|
| [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: NRIP1 is a coactivator for ESR1, which interacts with PIK3CA, a key enzyme in the PI3K/AKT signaling pathway. This suggests that NRIP1 may contribute to the regulation of this pathway through its interaction with ESR1 and subsequently PIK3CA.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: HIF1A is involved in hypoxia response and inflammatory response to microbe infections, while HLA-DMA is involved in antigen processing and presentation through MHC class II molecules. There is no known direct functional association between these two genes.|
| [DUSP6](https://www.ncbi.nlm.nih.gov/gene?term=DUSP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HLA-DRA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DRA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between DUSP6 and HLA-DRA in TCGA-LAML could involve DUSP6 regulating MAPK1/MAPK3 activity and expression of AP1 transcription factors, which could impact the presentation of antigenic peptides by HLA-DRA on APCs for recognition by CD4-positive T cells. The shortest path between DUSP6 and HLA-DRA in the BioGRID protein interaction network involves protein interactions with MGRN1, TMEM97, and HSPA8.|
| [ELAC2](https://www.ncbi.nlm.nih.gov/gene?term=ELAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP8](https://www.ncbi.nlm.nih.gov/gene?term=RRP8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 139.7</br>**STRING score:** 211</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: ELAC2 is involved in tRNA maturation and associates with mitochondrial DNA complexes, while RRP8 is an essential component in the eNoSC complex that mediates silencing of rDNA in response to intracellular energy status and acts by recruiting histone-modifying enzymes. There is no specific evidence for a direct functional association between ELAC2 and RRP8.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.7</br>**STRING score:** 303</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association between DDX42 and RAE1 is their interaction in the RNA export pathway. DDX42 is a component of the splicing factor complex while RAE1 is an mRNA export factor involved in nucleocytoplasmic transport. They may work together to ensure efficient RNA transport from the nucleus to the cytoplasm.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.2</br>**STRING score:** 217</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 3) | Possible functional associations between NR3C1 and PGRMC1 include: PGRMC1 being a component of a progesterone-binding protein complex and NR3C1 being a progesterone receptor, and PGRMC1 regulating heme synthesis and NR3C1 interacting with co-chaperones involved in protein folding. However, there is no direct interaction reported between these two genes in the literature.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.5</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 79 (min len 4) | There are no obvious functional associations between ELOVL6 and TBP. ELOVL6 is involved in the elongation and synthesis of fatty acids, while TBP is a transcription factor required for RNA polymerase II-mediated transcription initiation.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.0</br>**STRING score:** 401</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | One possible functional association between ACLY and TP53 is in the regulation of cell growth and division. ACLY is involved in fatty acid and cholesterol synthesis, while TP53 acts as a tumor suppressor and negatively regulates cell division. Both genes may play a role in coordinating the metabolic and cell cycle processes necessary for proper cell growth and proliferation.|
| [IARS2](https://www.ncbi.nlm.nih.gov/gene?term=IARS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL19](https://www.ncbi.nlm.nih.gov/gene?term=MRPL19[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.9</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 67 (min len 3) | There are no clear functional associations between IARS2 and MRPL19 based on the provided context. IARS2 is involved in activating MAP kinases and phosphorylating PIK3R1 in response to ligand-mediated activation of FGFR1, while MRPL19 is part of the mitochondrial large ribosomal subunit.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 129.1</br>**STRING score:** 966</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CCNA2 and KIF2C include their involvement in chromosome segregation during mitosis and their interaction with microtubules and kinetochores. CCNA2 is a cyclin that controls the G1/S and G2/M transition phases of the cell cycle while KIF2C functions as a microtubule depolymerizing activity in mitotic cells and plays a role in chromosome congression and attachment.|
| [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZMIZ1](https://www.ncbi.nlm.nih.gov/gene?term=ZMIZ1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 126.4</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | RASA1 and ZMIZ1 may be functionally associated through the AR signaling pathway, as ZMIZ1 is a transcriptional coactivator of AR and RASA1 directly interacts with tyrosine-phosphorylated AR. Additionally, ZMIZ1 is involved in the regulation of postmitotic positioning of pyramidal neurons, while RASA1 is an inhibitory regulator of the Ras-cyclic AMP pathway.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ARL4C](https://www.ncbi.nlm.nih.gov/gene?term=ARL4C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 125.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between APP and ARL4C include their roles in intracellular vesicular transport and regulation of cholesterol secretion pathways. However, based on the limited information provided, it is difficult to make specific predictions about their interactions or shared biological processes.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C2B](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C2B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 125.4</br>**STRING score:** 209</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CDK6 and PIK3C2B may involve their roles in EGF and PDGF signaling cascades, as PIK3C2B phosphorylates PtdIns and PtdIns4P with a preference for PtdIns and may be involved in these pathways. CDK6, on the other hand, promotes G1/S transition, interacts with D-type G1 cyclins, and is required for cell proliferation within certain cell types, potentially suggesting a role in cell growth and division processes that may intersect with PIK3C2B's signaling functions.|
| [BRPF1](https://www.ncbi.nlm.nih.gov/gene?term=BRPF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 123.7</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between BRPF1 and TBP could be in the regulation of transcription. BRPF1 is a scaffold subunit of various histone acetyltransferase (HAT) complexes, while TBP is a subunit of the TFIID basal transcription factor complex. Both play essential roles in the initiation of RNA polymerase II-dependent transcription.|
| [ARID5B](https://www.ncbi.nlm.nih.gov/gene?term=ARID5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.3</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ARID5B and PIK3CA in TCGA-LAML cancer type may include involvement in adipogenesis, liver development, and smooth muscle cell differentiation and proliferation. ARID5B acts as a transcription coactivator while PIK3CA is a phosphoinositide-3-kinase that plays a role in cellular signaling involved in cell growth, survival, proliferation, motility, and morphology. The shortest path between these genes in the BioGRID protein interaction network involves HDAC1, KIAA1429, and FASN.|
| [AURKB](https://www.ncbi.nlm.nih.gov/gene?term=AURKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.6</br>**STRING score:** 892</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between AURKB and MELK may involve regulating cell cycle and proliferation, with AURKB playing a role in mitosis, spindle assembly, and cytokinesis, and MELK being involved in cell cycle regulation and self-renewal of stem cells. Their interaction in the BioGRID protein interaction network may suggest their collaboration in these processes.|
| [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.1</br>**STRING score:** 160</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association: PMPCB and UBL5 both interact with the molecular chaperone complex TRiC. PMPCB is the catalytic subunit of the essential mitochondrial processing protease (MPP), while UBL5 is involved in the folding of actin and tubulin. They may work together to regulate protein folding and turnover, potentially impacting cellular metabolism and stability.|
