| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CHIC2](https://www.ncbi.nlm.nih.gov/gene?term=CHIC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 132 (min len 4) | CHIC2 is a regulatory component of the cyclin D1-CDK4 complex involved in cell cycle progression while STK10 is a serine/threonine-protein kinase involved in regulation of lymphocyte migration and negative regulation of MAP3K1/MEKK1. There is a possible functional association between CHIC2 and STK10 through the protein interaction network where CHIC2 can be found on the shortest path linking the two genes.|
| [KIAA0753](https://www.ncbi.nlm.nih.gov/gene?term=KIAA0753[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALGDS](https://www.ncbi.nlm.nih.gov/gene?term=RALGDS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 165.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 214 (min len 4) | Possible functional association: KIAA0753 may play a role in centriole duplication and cilium assembly, while RALGDS functions as a guanine nucleotide exchange factor and plays a role in intracellular transport.      They may be functionally associated in regulating intracellular transport and/or ciliary/centrosomal processes.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association(s) between LYN and PRKAG2: LYN is a protein kinase that plays important roles in signaling pathways regulating immune responses, cell differentiation, proliferation, and survival. PRKAG2 is a subunit of AMP-activated protein kinase that regulates cellular energy metabolism. There is no clear biological context or known direct interaction between these two genes.|
| [ACBD3](https://www.ncbi.nlm.nih.gov/gene?term=ACBD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 157.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations: ACBD3 plays a role in viral RNA replication and interacts with various picornaviruses, while CRKL mediates intracellular signal transduction. There is no obvious biological overlap between these genes based on their known functions.|
| [FDFT1](https://www.ncbi.nlm.nih.gov/gene?term=FDFT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional association: MLEC may play a role in the early steps of protein N-glycosylation that FDFT1 may need for its function in the sterol biosynthesis pathway.  FDFT1 catalyzes the condensation of 2 farnesyl pyrophosphates to form squalene, the first committed enzyme in the sterol biosynthesis pathway, while MLEC interacts with the oligosaccharyltransferase (OST) complex involved in protein N-glycosylation.|
| [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association between HMGCS1 and PUM2 could be in regulation of cell proliferation and differentiation. HMGCS1 is involved in cholesterol synthesis, while PUM2 acts as a post-transcriptional repressor of various mRNAs and regulates gene expression at the post-transcriptional level, suggesting a potential connection in cellular metabolic and regulatory pathways.|
| [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between LSM6 and PGRMC1 could be related to mRNA processing and heme homeostasis, as LSM6 plays a role in pre-mRNA splicing and is involved in RNA processing, while PGRMC1 is an intracellular heme chaperone that regulates heme synthesis. There is currently no known direct interaction between LSM6 and PGRMC1 in these specific processes.|
| [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | NRIP1 modulates transcriptional activation by steroid receptors such as ESR1 and modulates transcriptional repression by nuclear hormone receptors. PIK3CA is a phosphoinositide-3-kinase that phosphorylates phosphatidylinositol to produce 3-phosphoinositides, which play a key role in activating signaling cascades involved in cell growth, survival, proliferation, motility, and morphology. There is a possible functional association between NRIP1 and PIK3CA through a protein interaction network involving ESR1 and HDAC1, suggesting a role in transcriptional regulation and cell signaling.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between HIF1A and HLA-DMA is their involvement in the immune response. HIF1A plays a role in inducing cytokines and promoting inflammation, while HLA-DMA is involved in antigen presentation by MHC class II molecules on immune cells.|
| [DUSP6](https://www.ncbi.nlm.nih.gov/gene?term=DUSP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HLA-DRA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DRA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | The shortest path in the BioGRID protein interaction network between DUSP6 and HLA-DRA is through MGRN1, TMEM97, and HSPA8. DUSP6 inactivates MAP kinases, promotes cell differentiation, and is important for resolving acute pain. HLA-DRA is an MHC class II molecule that presents antigenic peptides to CD4-positive T cells for immune response and is involved in central immune tolerance. No specific functional associations can be inferred between these genes based on available information.|
| [ELAC2](https://www.ncbi.nlm.nih.gov/gene?term=ELAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP8](https://www.ncbi.nlm.nih.gov/gene?term=RRP8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 139.7</br>**STRING score:** 211</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear evidence of a direct functional association between ELAC2 and RRP8, but both genes play critical roles in RNA processing and ribosome assembly. ELAC2 is a mitochondrial tRNA phosphodiesterase, while RRP8 is an essential component of the energy-dependent nucleolar silencing complex (eNoSC) involved in response to intracellular energy status.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.7</br>**STRING score:** 303</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association between DDX42 and RAE1 could be in nucleocytoplasmic transport, given RAE1's role as an mRNA export factor and DDX42's interaction with nuclear protein TP53BP2. DDX42's involvement in RNA helicase activity and splicing factor complex formation may also have indirect effects on RAE1's functions in mRNA transport and cytoskeleton attachment.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.2</br>**STRING score:** 217</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 3) | Possible functional association: NR3C1 and PGRMC1 may have a role in hormone signaling and/or regulation of gene expression in TCGA-LAML cancer cells. NR3C1 is a steroid hormone receptor that regulates transcriptional activation and repression, while PGRMC1 is a progesterone-binding protein that interacts with cytochrome P450 (CYP) enzymes involved in hormone metabolism and is involved in intracellular heme homeostasis.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.5</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 79 (min len 4) | Possible functional associations between ELOVL6 and TBP could involve lipid metabolism and transcription regulation, respectively. ELOVL6 is involved in the elongation of fatty acids, while TBP is a basal transcription factor that initiates RNA polymerase II-dependent transcription.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.0</br>**STRING score:** 401</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ACLY and TP53 do not have any known direct or indirect functional associations from the provided context. ACLY is involved in citrate cleavage while TP53 is a tumor suppressor involved in apoptosis, cell cycle regulation, and transcriptional regulation.|
| [IARS2](https://www.ncbi.nlm.nih.gov/gene?term=IARS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL19](https://www.ncbi.nlm.nih.gov/gene?term=MRPL19[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.9</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 67 (min len 3) | There is no apparent functional association between IARS2 and MRPL19 based on their known biology and the provided context. IARS2 is involved in a signaling pathway linked to activation of MAP kinases and phosphorylation of PIK3R1, while MRPL19 is a component of the mitochondrial large ribosomal subunit and acts as a chemokine ligand.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 129.1</br>**STRING score:** 966</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: CCNA2 and KIF2C are both involved in cell cycle regulation during mitosis and chromosome segregation. They may interact through BRCA1, ANAPC4, KRAS, HIST1H2BG, and BRD4 in the BioGRID protein interaction network.|
| [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZMIZ1](https://www.ncbi.nlm.nih.gov/gene?term=ZMIZ1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 126.4</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between RASA1 and ZMIZ1 could be in the regulation of transcriptional activity, as both genes act as transcriptional coactivators and have been reported to interact with AR and NOTCH1. RASA1 has also been shown to inhibit the Ras-cyclic AMP pathway, while ZMIZ1 is involved in thymocyte and T cell development, and regulates the postmitotic positioning of pyramidal neurons in the developing cerebral cortex.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ARL4C](https://www.ncbi.nlm.nih.gov/gene?term=ARL4C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 125.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context and known biology, there is no clear functional association between APP and ARL4C. APP is involved in neuronal cell degeneration, axon degeneration, microtubule association, and interacts with various proteins. ARL4C is a small GTP-binding protein involved in intracellular vesicular transport.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C2B](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C2B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 125.4</br>**STRING score:** 209</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: CDK6 is involved in cell cycle control and differentiation, while PIK3C2B is involved in intracellular signaling pathways. CDK6 phosphorylates PIK3C2B, suggesting a potential regulatory role in the PI3K signaling pathway.|
| [BRPF1](https://www.ncbi.nlm.nih.gov/gene?term=BRPF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 123.7</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | BRPF1 and TBP are both involved in transcriptional regulation. BRPF1 is a scaffold subunit of histone acetyltransferase (HAT) complexes and positively regulates the transcription of RUNX1 and RUNX2, while TBP is a TATA-box-binding protein that plays a major role in the initiation of RNA polymerase II-dependent transcription. There is a possible functional association between these two genes as they are both involved in transcription.|
| [ARID5B](https://www.ncbi.nlm.nih.gov/gene?term=ARID5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.3</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: PIK3CA is involved in the PI3K/AKT signaling cascade, which ARID5B acts as a coactivator of HNF4A in liver. These two genes may work together to regulate gene expression in liver development and adipogenesis. The shortest path in the BioGRID protein interaction network suggests that PIK3CA may interact with HDAC1, KIAA1429, and FASN, which may indirectly affect ARID5B's role in transcription coactivation and adipogenesis.|
| [AURKB](https://www.ncbi.nlm.nih.gov/gene?term=AURKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.6</br>**STRING score:** 892</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations: AURKB is a serine/threonine-protein kinase that is involved in regulating mitosis and cytokinesis, while MELK is also a serine/threonine-protein kinase that plays a role in cell cycle regulation, stem cell self-renewal, and apoptosis. There is no direct evidence of physical interaction between these two genes, but they may both be involved in regulating cell proliferation and/or carcinogenesis.|
| [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.1</br>**STRING score:** 160</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between PMPCB and UBL5 include their involvement in protein folding and regulation of cellular processes such as mitochondrial protein processing and telomere maintenance, respectively. These genes may also interact with other proteins to regulate kinase activity.|
