| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ABHD6](https://www.ncbi.nlm.nih.gov/gene?term=ABHD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 4) | Possible functional associations between ABHD6 and POLE2 include involvement in DNA repair and chromosomal DNA replication, as well as potential regulation of endocannabinoid signaling pathways and BMP catabolism. However, there is currently no clear evidence of a direct biological link between these two genes.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTFR1](https://www.ncbi.nlm.nih.gov/gene?term=MTFR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: IDE is involved in viral infection as an entry receptor for varicella-zoster virus, while MTFR1 plays a role in mitochondrial aerobic respiration and organization, as well as interacts with proteins involved in fission and trafficking. There are no known direct functional associations between these two genes.|
| [GTDC1](https://www.ncbi.nlm.nih.gov/gene?term=GTDC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRCP](https://www.ncbi.nlm.nih.gov/gene?term=PRCP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 4) | Possible functional association: GTDC1 is involved in transcriptional activation and interacts with MAP2K3 and MAP2K6, while PRCP cleaves peptides at an acidic pH. There are no known direct functional associations between these two genes.|
| [FAM63A](https://www.ncbi.nlm.nih.gov/gene?term=FAM63A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFP36](https://www.ncbi.nlm.nih.gov/gene?term=ZFP36[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 235 (min len 4) | There is no clear functional association between FAM63A and ZFP36, based on the provided context. FAM63A is involved in protein turnover and associates with spliceosomes, while ZFP36 negatively regulates HTLV-1 TAX-dependent transactivation of viral long terminal repeat promoter and inhibits HTLV-1 TAX.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EDEM1](https://www.ncbi.nlm.nih.gov/gene?term=EDEM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.8</br>**STRING score:** 728</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between ATF6 and EDEM1 include their involvement in the endoplasmic reticulum-associated degradation (ERAD) process, where EDEM1 extracts misfolded glycoproteins and targets them for degradation, and ATF6 acts as a transcription factor that initiates the unfolded protein response (UPR) during endoplasmic reticulum stress and activates genes involved in the UPR. The shortest path between these two genes in the BioGRID protein interaction network involves several other proteins, suggesting a complex interplay between them in various cellular processes.|
| [MAPK9](https://www.ncbi.nlm.nih.gov/gene?term=MAPK9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN1](https://www.ncbi.nlm.nih.gov/gene?term=RPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | MAPK9 and RPN1 do not have a direct or well-established functional association. MAPK9 (JNK2) is a protein kinase involved in cellular stress response and signaling pathways, while RPN1 is a subunit of the oligosaccharyl transferase (OST) complex involved in protein N-glycosylation in the endoplasmic reticulum.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.6</br>**STRING score:** 172</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between LYN and RGS2 could include regulating immune responses and G protein-coupled receptor signaling cascades. LYN is a tyrosine kinase that regulates immune responses, while RGS2 inhibits signal transduction by G protein alpha subunits. The shortest path between these two proteins involves several other proteins, suggesting that they may be involved in a larger network of interactions.|
| [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between MYL9 and TBC1D15 could be related to the regulation of cell contractility and cellular transport through interaction with RAB7A, respectively. However, further experimental evidence is needed to determine any direct biological interactions or pathways between these genes.|
| [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: DYNLT3 is involved in linking the dynein motor protein to cargos for intracellular transport and progression through mitosis, while HSPA4 is a component of the chromosomal passenger complex (CPC) that regulates mitosis and cytokinesis. As both proteins are involved in mitosis and cytokinesis, there may be a functional association between them in regulating these cellular processes.|
| [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between NPC1 and TRIB1 are currently unknown. NPC1 acts as an endosomal entry receptor for ebolavirus, while TRIB1 is an adapter protein involved in protein degradation and regulates myeloid cell differentiation by altering the expression of CEBPA in a COP1-dependent manner.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.0</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between IKBKB and NRAS may exist in the NF-kappa-B signaling pathway, as IKBKB plays a crucial role in activating NF-kappa-B, and NRAS is involved in cellular signaling pathways, including the Ras signaling pathway. Their interaction in this process could potentially impact immune response, growth control, or protection against apoptosis.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.1</br>**STRING score:** 997</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations: CCNB2 and PLK1 are both involved in cell cycle regulation, with CCNB2 controlling the transition from G2 to M phase and PLK1 regulating events throughout M phase. They interact with each other in the BioGRID protein interaction network, suggesting they may cooperate in some aspects of cell cycle progression.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.9</br>**STRING score:** 721</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: Both ICAM1 and SYK are involved in immune response. ICAM1 plays a role in cell adhesion and can be degraded by viral E3 ubiquitin ligase MIR2 during Kaposi's sarcoma-associated herpesvirus/HHV-8 infection, while SYK is a non-receptor tyrosine kinase that mediates signal transduction downstream of classical immunoreceptors like the B-cell receptor (BCR) and regulates innate and adaptive immunity, cell adhesion, osteoclast maturation, platelet activation and vascular development. The shortest path between ICAM1 and SYK in the BioGRID protein interaction network includes several intermediate genes and may suggest indirect functional associations rather than direct physical interactions between ICAM1 and SYK.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.1</br>**STRING score:** 249</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations: APP and LGMN both play a role in protein degradation, with APP being involved in lysosomal protein degradation and LGMN specifically required for lysosomal protein degradation in renal proximal tubules. There is no direct evidence of functional associations between these two genes based on the provided information.|
| [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK3](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.5</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: KIF2C and MAPKAPK3 may be involved in chromosome segregation and cytokine production respectively, suggesting a possible role in the regulation of immune response in cancer. KIF2C is also involved in microtubule turnover and attachment to kinetochore, while MAPKAPK3 regulates transcriptional activity and phosphorylates substrates such as TTP/ZFP36 and HSP27/HSPB1, implying a role in RNA stabilization and stress response.|
| [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.7</br>**STRING score:** 730</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between NOTCH1 and NRAS include NOTCH1's role in regulating cell-fate determination and angiogenesis, while NRAS is involved in Ras protein signaling. These two genes may intersect in pathways related to cell differentiation and proliferation.|
| [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF274](https://www.ncbi.nlm.nih.gov/gene?term=ZNF274[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.3</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear biological association between BRCA1 and ZNF274. BRCA1 is involved in DNA repair, centrosomal microtubule nucleation, and transcriptional regulation, while ZNF274 is a transcription repressor that specifically binds to the 3'-end of zinc-finger coding genes and recruits chromatin-modifying proteins.|
| [C5](https://www.ncbi.nlm.nih.gov/gene?term=C5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.6</br>**STRING score:** 234</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between C5 and ICAM1 include their involvement in local inflammatory processes, chemotaxis and migration of immune cells, and interaction with microbial infections. C5 is a mediator of inflammation and a potent chemokine, while ICAM1 facilitates immune cell adhesion and viral infections in some contexts. The shortest path between these genes in the BioGRID network suggests that they may act in a signaling pathway involving the NR3C1 and RAF1 proteins.|
| [CDC45](https://www.ncbi.nlm.nih.gov/gene?term=CDC45[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GABPB1](https://www.ncbi.nlm.nih.gov/gene?term=GABPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.3</br>**STRING score:** 164</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | CDC45 and GABPB1 may be functionally associated in DNA replication processes, as CDC45 is required for initiation of chromosomal DNA replication and interacts with other factors involved in replication, while GABPB1 is necessary for transcriptional activity and interacts with HCFC1 to regulate gene expression. However, without further experimental evidence, it is unclear if there is a direct interaction or co-regulation between these genes in this context.|
| [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.1</br>**STRING score:** 270</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between TBPL1 and TP53 could be related to cellular transcription and regulation of cell cycle. TBPL1 is involved in mediating transcription of various genes through a specific core promoter element, while TP53 acts as a tumor suppressor and regulates transcription of genes required for cell division. Their interaction may influence regulation of cellular processes related to transcription and cell cycle control.|
