| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ABHD6](https://www.ncbi.nlm.nih.gov/gene?term=ABHD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 4) | There is no clear functional association between ABHD6 and POLE2. ABHD6 is a lipase involved in lysophospholipid degradation and endocannabinoid signaling, while POLE2 is an accessory subunit of DNA polymerase epsilon complex involved in DNA replication and repair. The shortest path between these genes in the BioGRID protein interaction network involves several intermediary proteins and no direct interaction between ABHD6 and POLE2 has been reported.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTFR1](https://www.ncbi.nlm.nih.gov/gene?term=MTFR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between IDE and MTFR1 could be related to mitochondrial respiration, organization, and fission. IDE can act as an entry receptor for varicella-zoster virus, while MTFR1 plays a role in regulating mitochondrial function and interacts with PEX19 and FIS1. There seems to be no direct functional association mentioned between the two genes.|
| [GTDC1](https://www.ncbi.nlm.nih.gov/gene?term=GTDC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRCP](https://www.ncbi.nlm.nih.gov/gene?term=PRCP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 4) | Possible functional associations between GTDC1 and PRCP remain unknown based solely on the provided context. GTDC1 is involved in transcriptional activation and MAP kinase signaling, while PRCP is involved in peptide cleavage.|
| [FAM63A](https://www.ncbi.nlm.nih.gov/gene?term=FAM63A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFP36](https://www.ncbi.nlm.nih.gov/gene?term=ZFP36[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 235 (min len 4) | There is no clear functional association between FAM63A and ZFP36 based on the provided context. FAM63A is a hydrolase involved in protein turnover and associates with spliceosomes and snRNPs, while ZFP36 negatively regulates HTLV-1 TAX-dependent transactivation of viral LTR promoter and interacts with HTLV-1 TAX.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EDEM1](https://www.ncbi.nlm.nih.gov/gene?term=EDEM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.8</br>**STRING score:** 728</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between ATF6 and EDEM1 could involve their involvement in endoplasmic reticulum stress response and protein degradation pathways. ATF6 activates transcription of genes involved in the unfolded protein response, while EDEM1 targets misfolded glycoproteins for degradation in an N-glycan-independent manner. They both interact with other proteins involved in these processes, such as SEL1L and DERL 2/3.|
| [MAPK9](https://www.ncbi.nlm.nih.gov/gene?term=MAPK9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN1](https://www.ncbi.nlm.nih.gov/gene?term=RPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Based on the shortest path in the BioGRID protein interaction network, MAPK9 and RPN1 may be functionally associated in the context of cancer. MAPK9 is involved in the JNK signaling pathway and can interact with various proteins, including ATF2 and TP53, while RPN1 is a subunit of the oligosaccharyl transferase complex that mediates protein N-glycosylation. It is possible that MAPK9 may regulate the glycosylation of proteins through its interaction with RPN1.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.6</br>**STRING score:** 172</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | One possible functional association between LYN and RGS2 is their involvement in regulating signaling pathways. LYN acts as a negative regulator while RGS2 inhibits G protein-coupled receptor signaling cascades by increasing the GTPase activity of G protein alpha subunits. They may work together to fine-tune signaling events in immune responses, hematopoiesis, and blood pressure regulation.|
| [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between MYL9 and TBC1D15 could involve regulation of cell contractile activity and GTPase signaling, respectively. MYL9 is involved in regulation of both smooth muscle and nonmuscle cell contractile activity while TBC1D15 acts as a GTPase activating protein for RAB7A. There is no clear biological evidence suggesting any direct link between the two proteins.|
| [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: Both DYNLT3 and HSPA4 are involved in mitosis and are part of protein complexes that regulate intracellular transport and cell division. DYNLT3 is part of the cytoplasmic dynein 1 complex, which links dynein to cargos and adapter proteins, while HSPA4 is part of the chromosomal passenger complex, which regulates mitosis and cytokinesis. There is no known direct interaction between these two genes, but they may have indirect functional associations through their involvement in regulating key processes during cell division.|
| [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 153.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TRIB1 interacts with COP1, which is involved in protein degradation. NPC1 acts as an endosomal entry receptor for ebolavirus. There doesn't seem to be a clear biological connection between these two genes.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.0</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between IKBKB and NRAS include involvement in the NF-kappa-B signaling pathway and Ras protein binding. IKBKB plays an essential role in activating the NF-kappa-B pathway by phosphorylating inhibitors of NF-kappa-B, while NRAS binds GDP/GTP and has intrinsic GTPase activity. The shortest path between these two genes in the BioGRID protein interaction network suggests they may indirectly interact through CTNNB1, AKT1, PRKCI, PRKCD, or PLK1.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.1</br>**STRING score:** 997</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Both CCNB2 and PLK1 are involved in regulating the cell cycle, with CCNB2 controlling the transition to mitosis and PLK1 regulating various processes during M phase. Their interaction may suggest a potential role of PLK1 in regulating CCNB2 activity during mitosis.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.9</br>**STRING score:** 721</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between ICAM1 and SYK include regulation of innate and adaptive immunity, cell adhesion, and platelet activation. ICAM1 is involved in immune responses to microbial infections and interacts with Coxsackievirus A21 capsid proteins, while SYK is a non-receptor tyrosine kinase that mediates signal transduction downstream of various transmembrane receptors. The BioGRID protein interaction network suggests potential interactions between ICAM1 and SYK via multiple intermediate proteins.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.1</br>**STRING score:** 249</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | There is a possible functional association between APP and LGMN in lysosomal protein degradation and EGFR degradation, as both genes are involved in these processes. APP also plays a role in neuronal cell degeneration and binds to various cytoplasmic proteins, while LGMN has strict specificity for hydrolysis of asparaginyl bonds and may be involved in MHC class II antigen presentation.|
| [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK3](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.5</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: KIF2C and MAPKAPK3 both have roles in chromosome segregation during mitosis and may interact with each other through protein-protein interactions such as TRIM67 and APP. MAPKAPK3 also plays a role in regulating cytokine production and inflammatory response.|
| [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.7</br>**STRING score:** 730</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: NRAS is a member of the Ras protein family and is involved in intracellular signaling pathways that regulate cell growth and differentiation. NOTCH1 codes for a receptor that plays an important role in cell fate determination and is involved in angiogenesis, neurogenesis, and immune cell differentiation. Possible cross-talk between these two genes could be involved in the regulation of cell proliferation and differentiation in cancer cells.|
| [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF274](https://www.ncbi.nlm.nih.gov/gene?term=ZNF274[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.3</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: The BioGRID protein interaction network shows a shortest path between BRCA1 and ZNF274, suggesting physical interaction between these two proteins. However, BRCA1 is primarily involved in DNA repair and transcriptional regulation, while ZNF274 is a transcriptional repressor, so the functional association between these genes is unclear.|
| [C5](https://www.ncbi.nlm.nih.gov/gene?term=C5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.6</br>**STRING score:** 234</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between C5 and ICAM1 are related to inflammation and immune responses. C5 plays a role in complement system activation, while ICAM1 is involved in leukocyte migration and infection response. The shortest path in the BioGRID protein interaction network suggests a potential direct interaction between C5 and ICAM1 through NR3C1 and RAF1, respectively.|
| [CDC45](https://www.ncbi.nlm.nih.gov/gene?term=CDC45[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GABPB1](https://www.ncbi.nlm.nih.gov/gene?term=GABPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.3</br>**STRING score:** 164</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: Both CDC45 and GABPB1 are involved in DNA replication and transcriptional regulation, respectively. However, there is no direct or clear molecular mechanism that connects these two genes in specific biological terms.|
| [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.1</br>**STRING score:** 270</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | One possible functional association between TBPL1 and TP53 is that TBPL1 may be involved in the regulation of TP53 expression. Both genes are involved in transcriptional regulation, with TBPL1 mediating transcription of ribosomal proteins and NF1, while TP53 acts as a tumor suppressor and regulates a set of genes required for cell division.|
