| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CCDC85B](https://www.ncbi.nlm.nih.gov/gene?term=CCDC85B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are currently no known functional associations between ACLY and CCDC85B in specific biological terms. ACLY is involved in citrate cleavage for cholesterol and fatty acid synthesis, while CCDC85B plays a role in hepatitis delta virus genomic replication and interacts with HDAG.|
| [RELB](https://www.ncbi.nlm.nih.gov/gene?term=RELB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | RELB is involved in the NF-kappa-B pathway and is a transcriptional activator. SLC35A1 transports CMP-sialic acid and exchanges it for other molecules. There are no known direct functional associations between these two genes, but they may indirectly interact through the NF-kappa-B signaling pathway.|
| [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional association between MRPL18 and TUBB6 could be their involvement in mitochondrial protein synthesis and cytoskeletal organization respectively. MRPL18 plays a role in the formation of mitochondrial ribosomes whereas TUBB6 is a major component of microtubules. The identified shortest path of interaction between the two genes in the BioGRID protein interaction network suggests that they may have some functional relationship in bladder cancer.|
| [MAN2B1](https://www.ncbi.nlm.nih.gov/gene?term=MAN2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 132 (min len 4) | Possible functional association: MAN2B1 and SLC35A1 are both involved in glycoprotein metabolism and transport. They may play a role in the glycosylation of proteins and/or transport of glycoproteins to the Golgi apparatus for further processing and secretion. The identified protein interaction network suggests a potential direct interaction between the two proteins mediated by EGFR.|
| [NIPSNAP1](https://www.ncbi.nlm.nih.gov/gene?term=NIPSNAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association: NIPSNAP1 and TUBB6 are indirectly associated through interactions with common binding partners involved in endocytosis and microtubule formation, respectively. NIPSNAP1 is involved in receptor-mediated endocytosis and may indirectly regulate slow endocytic recycling of proteins, while TUBB6 is a major constituent of microtubules, which are involved in various cellular processes such as cell division and intracellular transport.|
| [BAG3](https://www.ncbi.nlm.nih.gov/gene?term=BAG3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP88](https://www.ncbi.nlm.nih.gov/gene?term=NUP88[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | There is no clear biological basis for a functional association between BAG3 and NUP88. BAG3 is involved in regulating the release of client/substrate proteins from HSP70/HSC70 chaperones and has anti-apoptotic activity, while NUP88 is a component of the nuclear pore complex that interacts with other proteins in the complex.|
| [PTK2B](https://www.ncbi.nlm.nih.gov/gene?term=PTK2B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: PTK2B and TRAPPC3 both play roles in intracellular transport and membrane trafficking, with PTK2B regulating cytoskeleton reorganization and TRAPPC3 being a component of the TRAPP complex involved in vesicular transport. These genes may interact in regulating these processes in specific cell types or under certain physiological conditions.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.0</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CEBPD and IKBKE have a shortest path of protein interaction in the BioGRID network. CEBPD is a transcription activator involved in immune and inflammatory responses, while IKBKE is a serine/threonine kinase regulating viral infection, TNFA, and cytokines signaling. They might play a role in regulating inflammatory responses to viral infection or TNFA signaling.|
| [ENOPH1](https://www.ncbi.nlm.nih.gov/gene?term=ENOPH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2L6](https://www.ncbi.nlm.nih.gov/gene?term=UBE2L6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between ENOPH1 and UBE2L6 is the ubiquitination and subsequent proteasomal degradation of FLT3, as UBE2L6 catalyzes the covalent attachment of ubiquitin to proteins and interacts with FLT3, which is also known to interact with ENOPH1. ENOPH1 is involved in the enolization of DK-MTP-1-P, leading to the formation of acireductone DHK-MTPene.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 62 (min len 3) | Possible functional associations between HDAC2 and TUBB6 may involve the regulation of gene transcription and cell cycle progression. HDAC2 is a histone deacetylase that represses gene transcription by removing acetyl groups from histones, while TUBB6 encodes for the beta subunit of microtubules, which have a crucial role in mitosis and cell division. The protein interaction network suggests that HDAC2 may indirectly regulate the expression of TUBB6 through intermediate proteins such as HDAC1 and BRCA1.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.6</br>**STRING score:** 461</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | There is no clear functional association between ACLY and ETFB. ACLY is involved in citrate cleavage and fatty acid synthesis while ETFB is involved in electron transfer and fatty acid oxidation in mitochondria. They may interact indirectly through other molecules in metabolic pathways.|
| [RNH1](https://www.ncbi.nlm.nih.gov/gene?term=RNH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.5</br>**STRING score:** 276</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no evident functional associations between RNH1 and UBQLN2 based on their biological functions and interactions. RNH1 inhibits ribonucleases, while UBQLN2 plays a role in protein degradation and autophagy. The shortest path between the two in the BioGRID protein interaction network is a direct interaction.|
| [COG2](https://www.ncbi.nlm.nih.gov/gene?term=COG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YKT6](https://www.ncbi.nlm.nih.gov/gene?term=YKT6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.1</br>**STRING score:** 711</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association between COG2 and YKT6 in TCGA-BLCA is their involvement in intracellular trafficking and vesicle fusion. COG2 is a component of the conserved oligomeric Golgi complex, while YKT6 is a vesicular soluble NSF attachment protein receptor involved in vesicle docking and fusion. The identified shortest path suggests that they may interact through a SNARE complex to regulate Golgi morphology and localization.|
| [HYOU1](https://www.ncbi.nlm.nih.gov/gene?term=HYOU1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.5</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 48 (min len 3) | The shortest path between HYOU1 and TUBB6 in the BioGRID protein interaction network suggests a potential physical interaction between these two genes. HYOU1 is involved in cytoprotective mechanisms triggered by oxygen deprivation and participates in protein folding, while TUBB6 is a major component of microtubules. It is possible that HYOU1 may participate in the regulation of microtubule dynamics through its interaction with TUBB6.|
