| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CBR3](https://www.ncbi.nlm.nih.gov/gene?term=CBR3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.8</br>**STRING score:** 250</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are no apparent functional associations between CBR3 and ETFB based on the available information. CBR3 catalyzes the NADPH-dependent reduction of carbonyl compounds while ETFB accepts electrons from mitochondrial dehydrogenases in fatty acid oxidation and amino acid metabolism.|
| [FAS](https://www.ncbi.nlm.nih.gov/gene?term=FAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no clear functional associations between FAS and TUBB6 based on the available information. FAS is involved in fatty acid synthesis, while TUBB6 is a major component of microtubules. Both genes have roles in cellular processes but do not directly interact based on the BioGRID network.|
| [PRPF4](https://www.ncbi.nlm.nih.gov/gene?term=PRPF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no visible functional association between PRPF4 and ZNF131 based on the given information. PRPF4 is involved in pre-mRNA splicing, while ZNF131 plays a role in transcriptional regulation as a repressor of ESR1/ER-alpha signaling and interacts with receptors such as LRP1 and LDLR.|
| [CEP57](https://www.ncbi.nlm.nih.gov/gene?term=CEP57[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B2](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: None can be inferred based on the available information.  CEP57 is involved in microtubule attachment to centrosomes and has mitogenic activity. PAFAH1B2 is an enzyme that hydrolyzes platelet-activating factors and may play a role in male germ cell meiosis. No known biological process or pathway links these two genes together.|
| [TSPAN3](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF586](https://www.ncbi.nlm.nih.gov/gene?term=ZNF586[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 4) | Possible functional association: TSPAN3 regulates cell proliferation and migration, while ZNF586 is involved in transcriptional regulation and proteasome complex formation. It is possible that TSPAN3 interacts with ZNF586 to regulate gene expression or protein degradation. However, further experimentation is needed to confirm this hypothesis.|
| [HEATR1](https://www.ncbi.nlm.nih.gov/gene?term=HEATR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF451](https://www.ncbi.nlm.nih.gov/gene?term=ZNF451[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no direct biological evidence to suggest a functional association between HEATR1 and ZNF451. HEATR1 is involved in ribosome biogenesis, while ZNF451 is involved in the SUMO-protein ligase pathway and plays a role in DNA damage response and transcriptional regulation.|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGCR](https://www.ncbi.nlm.nih.gov/gene?term=HMGCR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is currently no known direct functional association between AKAP8 and HMGCR. AKAP8 is involved in mitosis and cell cycle regulation, while HMGCR plays a critical role in cellular cholesterol homeostasis. The shortest path between AKAP8 and HMGCR in the BioGRID protein interaction network includes KIF20A, EP300, and TRIM67.|
| [CHERP](https://www.ncbi.nlm.nih.gov/gene?term=CHERP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 41 (min len 3) | There do not seem to be any direct functional associations between CHERP and HSPA4 based on their known roles. CHERP is involved in calcium homeostasis, growth, and proliferation, while HSPA4 is a key regulator of mitosis and cytokinesis. However, they have several intermediate protein interactions based on the BioGRID network.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | ACLY and FBXO11 have no known direct functional association in specific biological processes. ACLY catalyzes citrate cleavage into oxaloacetate and acetyl-CoA, while FBXO11 targets proteins for proteasomal degradation. The possible indirect functional association would depend on the specific downstream targets of FBXO11 and their relationship with the metabolic functions of ACLY.|
| [DHX29](https://www.ncbi.nlm.nih.gov/gene?term=DHX29[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | DHX29 and PTK2 are functionally associated through protein-protein interactions in the BioGRID network. DHX29 is involved in translation initiation, specifically binding to the 40S ribosome, while PTK2 is a protein tyrosine kinase that plays a role in various cellular processes, including cell adhesion, spreading, and migration. It is possible that DHX29 may regulate PTK2 activity in translation initiation or cellular processes that involve PTK2.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between HSPA4 and MAPKAPK5 include their involvement in the regulation of mitosis, cytokinesis, and spindle assembly. HSPA4 is an essential component of the chromosomal passenger complex that ensures correct chromosome alignment and segregation, while MAPKAPK5 is a tumor suppressor serine/threonine-protein kinase involved in mTORC1 signaling and post-transcriptional regulation. There is currently no known direct interaction between the two genes, but they are both key regulators of important cellular processes.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ISOC1](https://www.ncbi.nlm.nih.gov/gene?term=ISOC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.3</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | There is no clear functional association between ACLY and ISOC1, as they have different roles in metabolic and cellular processes. ACLY is involved in fatty acid synthesis, while ISOC1 is known to inactivate metalloproteinases. However, the shortest path in the BioGRID protein interaction network suggests potential indirect interactions between the two genes through intermediate proteins.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VDAC1](https://www.ncbi.nlm.nih.gov/gene?term=VDAC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: ELAVL1 is an RNA-binding protein that stabilizes target mRNAs, while VDAC1 forms a channel through the mitochondrial outer membrane. ELAVL1 has been shown to interact with proteins involved in mRNA processing, such as IGF2BP1, and could potentially be involved in regulating the stability or translation of VDAC1 mRNA. Further investigation is needed to establish a direct functional association between these two genes.|
| [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPEN](https://www.ncbi.nlm.nih.gov/gene?term=SPEN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.0</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | Possible functional associations between BRCA1 and SPEN may be related to transcriptional regulation and DNA damage repair, as both genes have been implicated in these processes. However, there is currently no direct evidence for a specific interaction or functional association between these two genes.|
| [CHMP4A](https://www.ncbi.nlm.nih.gov/gene?term=CHMP4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is currently no known functional association between CHMP4A and ETFB in specific biological terms. CHMP4A is involved in the endosomal sorting required for transport complex III (ESCRT-III) and ETFB is involved in mitochondrial fatty acid oxidation and amino acid metabolism.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YTHDF2](https://www.ncbi.nlm.nih.gov/gene?term=YTHDF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.2</br>**STRING score:** 671</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: Based on the provided context, G3BP1 and YTHDF2 are involved in microbial infection and RNA processing. It is possible that YTHDF2 interacts with G3BP1 in the context of viral RNA binding and processing.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.3</br>**STRING score:** 408</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between DNM1L and SLC25A46 based on their individual roles. DNM1L is involved in mitochondrial fission regulation, while SLC25A46 mediates mitochondrial organization and lipid homeostasis.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YKT6](https://www.ncbi.nlm.nih.gov/gene?term=YKT6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.7</br>**STRING score:** 232</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear indication of a functional association between CDC42 and YKT6 in the provided context and their known biology. CDC42 is involved in a variety of cellular processes such as cell polarity, migration, and phagocytosis, while YKT6 is a v-SNARE involved in vesicle fusion and transport. The only reported interaction between the two is a single protein-protein interaction in the BioGRID network.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 177</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between LIG1 and PIK3C3 could involve regulation and degradation of receptor tyrosine kinases and involvement in the PI3K complex and autophagy.|
| [UBE2A](https://www.ncbi.nlm.nih.gov/gene?term=UBE2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 508</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between UBE2A and UBQLN2 is their involvement in the ubiquitin-proteasome system (UPS) and protein degradation pathways. UBE2A catalyzes the attachment of ubiquitin to target proteins, while UBQLN2 plays a role in the proteasomal targeting of misfolded or accumulated proteins. They may work together to regulate protein turnover and degradation in cells.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.1</br>**STRING score:** 658</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between CFLAR and MYC in the context of TCGA-BLCA may involve CFLAR's role in apoptosis regulation and MYC's role in transcriptional activation of growth-related genes and angiogenesis. Their interaction in the BioGRID protein interaction network through various proteins involved in cellular processes such as TNFRSF6 mediated apoptosis, caspase-8 recruitment and processing, and RNA polymerase II pause release suggests a possible pathway of interaction that may require further investigation.|
| [PAFAH1B1](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCM1](https://www.ncbi.nlm.nih.gov/gene?term=PCM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.0</br>**STRING score:** 451</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | Possible functional associations: PAFAH1B1 and PCM1 are both involved in neuronal migration and microtubule-dependent processes. The shortest path in the BioGRID protein interaction network suggests a potential physical interaction between these two genes, possibly through DISC1 or TNIK.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP133](https://www.ncbi.nlm.nih.gov/gene?term=NUP133[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.8</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 61 (min len 3) | ELAVL1 is an RNA-binding protein that stabilizes target mRNAs, while NUP133 is involved in RNA export and poly(A)+ RNA transport. Based on their respective functions and an indirect association through SENP2, DDX3X, SEH1L, SPAST, and ZMYM2, it is possible that these genes may have a role in regulating mRNA stability and transport.|
