| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CBR3](https://www.ncbi.nlm.nih.gov/gene?term=CBR3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.8</br>**STRING score:** 250</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear functional association between CBR3 and ETFB based on their known roles; CBR3 is involved in the reduction of carbonyl compounds while ETFB is involved in electron transfer. However, both genes are involved in metabolic processes, and therefore could potentially interact indirectly through shared pathways.|
| [FAS](https://www.ncbi.nlm.nih.gov/gene?term=FAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between FAS and TUBB6 could be related to microtubule stabilization and apoptotic signaling. FAS is a receptor involved in initiating apoptosis, while TUBB6 is the major constituent of microtubules, which are important for cell division and transportation within cells. The interaction between the two could potentially affect stability of microtubules involved in apoptotic signaling.|
| [PRPF4](https://www.ncbi.nlm.nih.gov/gene?term=PRPF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between PRPF4 and ZNF131 are unclear based on the provided information. PRPF4 is involved in pre-mRNA splicing and is part of the U4/U6-U5 tri-snRNP complex. ZNF131 plays a role in transcriptional regulation and interacts with various receptors.|
| [CEP57](https://www.ncbi.nlm.nih.gov/gene?term=CEP57[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B2](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear evidence of a functional association between CEP57 and PAFAH1B2 based on their known functions and interactions. CEP57 is involved in microtubule attachment to centrosomes and nuclear translocation of growth factors, while PAFAH1B2 is the alpha2 catalytic subunit of the PAF acetylhydrolase enzyme with a role in male germ cell meiosis and spermiogenesis.|
| [TSPAN3](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF586](https://www.ncbi.nlm.nih.gov/gene?term=ZNF586[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 4) | Possible functional association between TSPAN3 and ZNF586 is their involvement in regulating gene expression and protein degradation respectively. TSPAN3 regulates myelination and interacts with integrins, while ZNF586 is a component of the 19S proteasome regulatory particle complex and involved in transcriptional regulation. Their interaction through the shortest path in the BioGRID protein interaction network suggests a possible connection in regulating cellular processes involving gene expression and protein degradation.|
| [HEATR1](https://www.ncbi.nlm.nih.gov/gene?term=HEATR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF451](https://www.ncbi.nlm.nih.gov/gene?term=ZNF451[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between HEATR1 and ZNF451 include involvement in ribosome biogenesis and protein modification, respectively. They may interact as part of the small subunit (SSU) processome during ribosomal RNA processing, and ZNF451 may play a role in MCM4 sumoylation. However, further research is needed to determine any direct biological connections between these genes.|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGCR](https://www.ncbi.nlm.nih.gov/gene?term=HMGCR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: AKAP8 and HMGCR may be involved in cell cycle regulation and chromatin condensation through their interactions with other proteins. AKAP8 is also involved in transcriptional regulation and Toll-like receptor signaling, while HMGCR plays a critical role in cellular cholesterol homeostasis.|
| [CHERP](https://www.ncbi.nlm.nih.gov/gene?term=CHERP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 41 (min len 3) | There is no clear functional association between CHERP and HSPA4 in the provided context, as they are not involved in overlapping biological processes. CHERP is involved in calcium homeostasis, growth and proliferation, while HSPA4 is a key regulator of mitosis and cytokinesis.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations: ACLY is involved in citrate cleavage and fatty acid synthesis, while FBXO11 is a substrate recognition component of an E3 ubiquitin-protein ligase complex. There are no known direct functional associations between these two genes in specific biological terms.|
| [DHX29](https://www.ncbi.nlm.nih.gov/gene?term=DHX29[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | DHX29 and PTK2 are functionally associated in the TCGA-BLCA cancer type, as they are part of the same protein interaction network. DHX29 is involved in translation initiation and is part of the 43S pre-initiation complex, while PTK2 is a tyrosine kinase that interacts with various proteins and is involved in signal transduction pathways.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | HSPA4 and MAPKAPK5 have a possible functional association in mitosis and cytokinesis. HSPA4 acts as a regulator of mitosis by ensuring correct chromosome alignment and segregation, whereas MAPKAPK5 is involved in post-transcriptional regulation and mediates phosphorylation of HSP27/HSPB1 in response to PKA/PRKACA stimulation, inducing F-actin rearrangement. The shortest path between these two genes in the BioGrid protein interaction network involves different proteins, all related to mitosis and cytokinesis.|
| [ACLY](https://www.ncbi.nlm.nih.gov/gene?term=ACLY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ISOC1](https://www.ncbi.nlm.nih.gov/gene?term=ISOC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.3</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between ACLY and ISOC1 could involve regulation of cholesterol and fatty acid synthesis as well as matrix metalloproteinase activity. However, there is no direct biological process linking these two genes.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VDAC1](https://www.ncbi.nlm.nih.gov/gene?term=VDAC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between ELAVL1 and VDAC1. ELAVL1 is an RNA-binding protein involved in stabilizing target mRNAs, while VDAC1 forms ion channels in the mitochondrial and plasma membranes.|
| [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPEN](https://www.ncbi.nlm.nih.gov/gene?term=SPEN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.0</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | There are no known direct functional associations between BRCA1 and SPEN in specific biological terms. However, BRCA1 is involved in DNA repair and cell cycle regulation, while SPEN is a corepressor that regulates different pathways such as the Notch pathway.|
| [CHMP4A](https://www.ncbi.nlm.nih.gov/gene?term=CHMP4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between CHMP4A and ETFB in TCGA-BLCA. CHMP4A is involved in endosomal sorting and membrane fission while ETFB is a component of the mitochondrial respiratory chain involved in fatty acid oxidation and amino acid metabolism.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YTHDF2](https://www.ncbi.nlm.nih.gov/gene?term=YTHDF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.2</br>**STRING score:** 671</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | There is no strong evidence for a direct functional association between G3BP1 and YTHDF2. G3BP1 plays a role in innate immunity and stress granule formation, while YTHDF2 is involved in viral gene expression and RNA decay. However, both genes have been implicated in the regulation of RNA, albeit through different mechanisms.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.3</br>**STRING score:** 408</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of a functional association between DNM1L and SLC25A46. DNM1L is involved in mitochondrial fission, while SLC25A46 controls mitochondrial organization. The two genes may indirectly regulate mitochondrial dynamics through separate mechanisms.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YKT6](https://www.ncbi.nlm.nih.gov/gene?term=YKT6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.7</br>**STRING score:** 232</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between CDC42 and YKT6 could be related to vesicle transport processes, as YKT6 is involved in SNARE complex-mediated vesicle docking and fusion, and CDC42 plays a role in regulating cellular responses such as cell migration and the actin cytoskeleton, which can impact vesicle transport. However, further experimental studies would be needed to confirm any direct functional relationship between these two genes.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 177</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations: LIG1 is involved in DNA repair, replication, and recombination, while PIK3C3 is a component of the PI3K complex involved in intracellular signaling, including autophagy. It is possible that PIK3C3 may play a role in DNA damage response, and LIG1 may have a role in autophagy. However, there is no known direct functional association between these two genes.|
| [UBE2A](https://www.ncbi.nlm.nih.gov/gene?term=UBE2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 508</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: UBQLN2 may play a role in regulating the degradation of proteins targeted by ubiquitin, such as those processed by UBE2A. UBE2A accepts ubiquitin and catalyzes its covalent attachment to other proteins, and may play a role in transcription regulation and post-replication repair of DNA.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.1</br>**STRING score:** 658</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | There are several possible functional associations between CFLAR and MYC in cancer. CFLAR is involved in apoptosis regulation, while MYC is a transcription factor that activates genes related to cell growth and proliferation. The shortest path between them in the BioGRID protein interaction network suggests potential interactions via SF3B3, CASP8, TRAF2, RAF1, and TP53.|
| [PAFAH1B1](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCM1](https://www.ncbi.nlm.nih.gov/gene?term=PCM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.0</br>**STRING score:** 451</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | Possible functional associations between PAFAH1B1 and PCM1 are related to microtubule organization, cytoskeleton dynamics, and centrosome assembly. PAFAH1B1 is involved in regulating the activity of dynein and microtubule-dependent processes, while PCM1 is essential for correct localization and function of several centrosomal proteins and anchoring microtubules to the centrosome. Both genes are involved in cilium biogenesis.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP133](https://www.ncbi.nlm.nih.gov/gene?term=NUP133[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.8</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 61 (min len 3) | There is no clear functional association between ELAVL1 and NUP133 based on their known biological functions. ELAVL1 is an RNA-binding protein that stabilizes mRNAs, while NUP133 is a component of the nuclear pore complex involved in RNA transport.|
