| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCM1](https://www.ncbi.nlm.nih.gov/gene?term=PCM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | One possible functional association between MELK and PCM1 is that they have a direct protein interaction, as indicated by the shortest path in the BioGRID network. MELK is a serine/threonine kinase involved in cell cycle regulation, apoptosis, and splicing regulation, while PCM1 is a transcriptional repressor that binds CpG islands in methylated promoters. The two genes may play a role in regulating gene expression and cell cycle progression.|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: BAD promotes cell death by binding to anti-apoptotic proteins, while PTPN12 regulates cellular signaling cascades by dephosphorylating proteins such as ERBB2 and PTK2B/PYK2. There is no known direct interaction between these two genes, but they both play roles in regulating cellular signaling pathways.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR61](https://www.ncbi.nlm.nih.gov/gene?term=WDR61[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | LYN is a negative regulator of the B-cell response and plays a role in regulating cell proliferation, survival, differentiation, migration, and adhesion. WDR61 is a component of the PAF1 complex, which has multiple functions during transcription by RNA polymerase II and is implicated in the regulation of development and maintenance of embryonic stem cell pluripotency.  There is no clear biological function association between LYN and WDR61, as they are involved in distinct cellular processes. However, the shortest path between them in the BioGRID protein interaction network suggests potential indirect interactions through intermediate proteins such as PTPN6, HBS1L, TRIM28, LARP7, and KIAA1429, which serve as potential points of crosstalk between their respective signaling pathways.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B2](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between DLD and PAFAH1B2 include involvement in mitochondrial metabolism and male germ cell meiosis. DLD is a component of multiple dehydrogenase complexes involved in the breakdown of amino acids and sugars, while PAFAH1B2 is a subunit of the cytosolic PAF acetylhydrolase enzyme. The shortest path between the two genes in the BioGRID protein interaction network involves several intermediary proteins, suggesting potential protein-protein interactions in signaling or other processes.|
| [CLPX](https://www.ncbi.nlm.nih.gov/gene?term=CLPX[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAI14](https://www.ncbi.nlm.nih.gov/gene?term=RAI14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between CLPX and RAI14 may involve actin regulation or sperm polarity and adhesion, since RAI14 plays a role in these processes and CLPX can enhance the DNA-binding activity of TFAM and is required for maintaining a normal mitochondrial nucleoid structure. However, further research is needed to determine any specific interactions between these two genes.|
| [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNRD1](https://www.ncbi.nlm.nih.gov/gene?term=TXNRD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between SRP14 and TXNRD1 include a role in enhancing the transcriptional activity of ESR2 and regulation of cell death. SRP14, as part of the SRP complex, is involved in cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum, while TXNRD1 is involved in redox regulation and has been shown to interact with ESR1 and ESR2. The shortest path between these genes in the BioGRID protein interaction network involves several other proteins, suggesting possible indirect functional associations.|
| [GSTM2](https://www.ncbi.nlm.nih.gov/gene?term=GSTM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GTDC1](https://www.ncbi.nlm.nih.gov/gene?term=GTDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 4) | Possible functional association between GSTM2 and GTDC1 is their involvement in cellular detoxification processes and regulation of gene expression. GSTM2 is involved in the conjugation of glutathione to electrophiles while GTDC1 is a probable transcriptional activator and interacts with MAP2K3 and MAP2K6, suggesting a potential role in MAPK signaling pathway and gene expression regulation. The shortest path in the BioGRID protein interaction network may suggest a physical interaction between GSTM2 and GTDC1 mediated by some common binding partners.|
| [ABHD4](https://www.ncbi.nlm.nih.gov/gene?term=ABHD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMP2](https://www.ncbi.nlm.nih.gov/gene?term=TIMP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | It is difficult to reason about any functional associations between ABHD4 and TIMP2 based solely on their known biochemical activities and protein interactions. ABHD4 is involved in the biosynthesis of endocannabinoids and forms cohesin complexes with SMC proteins, while TIMP2 irreversibly inhibits metalloproteinases. However, further investigation is needed to determine if and how these genes may functionally interact in the context of TCGA-BLCA cancer.|
| [JUN](https://www.ncbi.nlm.nih.gov/gene?term=JUN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIE](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.3</br>**STRING score:** 302</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | JUN and NFKBIE are both involved in transcriptional regulation and have been found to interact in the BioGRID protein interaction network. JUN is a transcription factor that binds to the AP-1 consensus motif and heterodimerizes with proteins of the FOS family to form an AP-1 transcription complex. NFKBIE inhibits NF-kappa-B and interacts with RELA and other nuclear factors. There may be functional associations between JUN and NFKBIE in the regulation of gene expression and inflammatory responses.|
| [IFRD2](https://www.ncbi.nlm.nih.gov/gene?term=IFRD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA3](https://www.ncbi.nlm.nih.gov/gene?term=RPA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between IFRD2 and RPA3 include involvement in DNA metabolism, such as DNA replication, DNA repair, and checkpoint activation. IFRD2 plays a role in cell proliferation, differentiation, and apoptosis, while RPA3 is a single-stranded DNA-binding protein that stabilizes and prevents reannealing of DNA intermediates.|
| [COG4](https://www.ncbi.nlm.nih.gov/gene?term=COG4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX7](https://www.ncbi.nlm.nih.gov/gene?term=SNX7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.0</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on the shortest path in the BioGRID protein interaction network and their known functions, COG4 and SNX7 may have a functional association in Golgi-to-ER retrograde transport and SNARE-pin assembly. COG4 is a subunit of the COG complex which mediates the interaction of SNX7 with SCFD1, and SNX7 is involved in intracellular trafficking and autophagosome assembly.|
| [ENOPH1](https://www.ncbi.nlm.nih.gov/gene?term=ENOPH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.7</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is currently no clear functional association between ENOPH1 and MRPL18 based on their described roles in the literature. ENOPH1 is involved in the enolization of biochemical intermediates, while MRPL18 is a mitochondrial import factor and a component of the mitochondrial large ribosomal subunit.|
| [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.4</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between EML3 and REEP5 could be related to microtubule dynamics and membrane remodeling, respectively. EML3 regulates mitotic spindle assembly and chromosome separation, while REEP5 plays a crucial role in the organization and function of the sarcoplasmic reticulum in cardiomyocytes. As both genes interact with proteins involved in cytoskeleton and membrane biology, they could potentially be involved in processes such as vesicle trafficking, organelle quality control, or cell migration.|
| [AXIN1](https://www.ncbi.nlm.nih.gov/gene?term=AXIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MMP1](https://www.ncbi.nlm.nih.gov/gene?term=MMP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.8</br>**STRING score:** 197</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 57 (min len 4) | Possible functional associations between AXIN1 and MMP1 are not evident based on their known biology and protein interaction network. AXIN1 is involved in regulating CTNNB1 levels, modulating Wnt-signaling, enhancing TGF-beta signaling, and controlling cell growth, apoptosis, and development. MMP1 is involved in collagen cleavage and can interact with HIV-1 Tat protein.|
| [HEATR1](https://www.ncbi.nlm.nih.gov/gene?term=HEATR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.2</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | HEATR1 is involved in ribosome biogenesis, while MBNL1 mediates pre-mRNA alternative splicing regulation. They do not have any direct or obvious functional associations based on their known roles.|
| [CAPN1](https://www.ncbi.nlm.nih.gov/gene?term=CAPN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JUN](https://www.ncbi.nlm.nih.gov/gene?term=JUN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.7</br>**STRING score:** 218</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between CAPN1 and JUN could involve calcium signaling and regulation of transcription. CAPN1 is a calcium-regulated protease involved in cytoskeletal remodeling and signal transduction, while JUN is a transcription factor that recognizes and binds to DNA. The shortest path between these two genes in the BioGRID protein interaction network involves several other proteins known to be involved in transcriptional regulation, including NFKBIA, HIF1A, NFE2L1, STAT3, and TP53.|
| [CDC25B](https://www.ncbi.nlm.nih.gov/gene?term=CDC25B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP1](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.1</br>**STRING score:** 295</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations: CDC25B is involved in mitotic progression and directly dephosphorylates CDK1, while TP53BP1 is involved in double-strand break repair and interacts with BRCA1. There may be a role for CDC25B in regulating cell cycle checkpoints related to DNA damage repair, possibly involving TP53BP1 and BRCA1. The BioGRID protein interaction network also suggests potential interactions between CDC25B and TP53BP1 via ESR1, CHEK1, YWHAZ, CDK1, and CCNB1.|
| [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between PLSCR1 and SYK include regulation of immune response and cell proliferation. PLSCR1 is involved in calcium-induced phospholipid scrambling, cytokine-regulated cell proliferation and differentiation, and antiviral response via IFN signaling. SYK is a non-receptor tyrosine kinase that mediates signal transduction downstream of transmembrane receptors involved in immune response, platelet activation, and vascular development. The shortest path between PLSCR1 and SYK in the BioGRID protein interaction network suggests potential direct physical interactions between these proteins.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.6</br>**STRING score:** 439</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations: AKT1 plays a role in several cellular processes, including metabolism, proliferation, cell survival, and growth, while RAB4A is involved in protein transport and vesicular traffic. One possible association between these genes is in the regulation of VEGFR2 endosomal trafficking, which involves both AKT1 and RAB4A.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP1](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.2</br>**STRING score:** 305</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between CIRBP and TP53BP1 based on the provided context. CIRBP plays a role in the genotoxic stress response, translation, and stress granule formation, while TP53BP1 is involved in DNA damage response, repair, and telomere maintenance. The BioGRID interaction network shows a direct interaction between these two proteins.|
