| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCM1](https://www.ncbi.nlm.nih.gov/gene?term=PCM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between MELK and PCM1 include regulation of cell cycle progression, cell proliferation, and splicing regulation. MELK phosphorylates and inhibits BCL2L14, which may affect mammary carcinogenesis by mediating inhibition of the pro-apoptotic function of BCL2L14. PCM1 is required for centrosome assembly and function and cilium biogenesis, and interacts with several centrosomal proteins. There may be a role for MELK in regulating PCM1-mediated processes related to centrosome and cilia function.|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: Both genes are involved in regulating cellular signaling pathways. While BAD promotes cell death and affects the level of heterodimerization of anti-apoptotic proteins, PTPN12 dephosphorylates a range of proteins, including cellular tyrosine kinases, and regulates signaling cascades. It is possible that PTPN12 could regulate BAD's activity through dephosphorylation.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR61](https://www.ncbi.nlm.nih.gov/gene?term=WDR61[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between LYN and WDR61 include regulation of gene transcription and maintenance of embryonic stem cell pluripotency. LYN is a tyrosine kinase involved in immune response, while WDR61 is a component of the PAF1 complex involved in transcriptional regulation. The protein interaction network suggests potential interactions between LYN and WDR61 through several intermediary proteins.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B2](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | There are no known direct functional associations between DLD and PAFAH1B2. DLD is involved in various metabolic processes such as the glycine cleavage system and pyruvate dehydrogenase complex, while PAFAH1B2 is a catalytic subunit of the cytosolic type I platelet-activating factor acetylhydrolase.|
| [CLPX](https://www.ncbi.nlm.nih.gov/gene?term=CLPX[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAI14](https://www.ncbi.nlm.nih.gov/gene?term=RAI14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | CLPX is an ATP-dependent specificity component of the Clp protease complex and a chaperone function that enhances DNA-binding activity. RAI14 plays a role in actin regulation at the ectoplasmic specialization and associates with actin. The possible functional association between CLPX and RAI14 is through their interaction with actin, which may implicate a role in sperm polarity and normal spermatid adhesion.|
| [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNRD1](https://www.ncbi.nlm.nih.gov/gene?term=TXNRD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations: SRP14 is a component of the signal recognition particle complex, while TXNRD1 enhances the transcriptional activity of the estrogen receptor. There is no known direct biological process linking these two genes.|
| [GSTM2](https://www.ncbi.nlm.nih.gov/gene?term=GSTM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GTDC1](https://www.ncbi.nlm.nih.gov/gene?term=GTDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 4) | There is no clear evidence of specific functional associations between GSTM2 and GTDC1. GSTM2 is involved in the conjugation of glutathione to electrophiles, while GTDC1 is a probable transcriptional activator that interacts with MAP kinases and tubulins. The shortest path in the BioGRID protein interaction network between the two genes involves common interactors such as APP and PARK2.|
| [ABHD4](https://www.ncbi.nlm.nih.gov/gene?term=ABHD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMP2](https://www.ncbi.nlm.nih.gov/gene?term=TIMP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TIMP2 inhibits the activity of metalloproteinases, such as collagenases, and ABHD4 plays a role in the biosynthesis of N-acyl ethanolamines, including the endocannabinoid anandamide. These genes are not directly functionally associated in a biological process.|
| [JUN](https://www.ncbi.nlm.nih.gov/gene?term=JUN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIE](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.3</br>**STRING score:** 302</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | JUN and NFKBIE have a functional association in the regulation of gene transcription. JUN is part of the AP-1 transcription factor complex and can bind to the promoter site of NFKBIE, which inhibits NF-kappa-B activity by sequestering it in the cytoplasm.|
| [IFRD2](https://www.ncbi.nlm.nih.gov/gene?term=IFRD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA3](https://www.ncbi.nlm.nih.gov/gene?term=RPA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between IFRD2 and RPA3 in TCGA-BLCA could involve DNA damage response, replication, and repair pathways. IFRD2 is involved in cell proliferation, differentiation, and apoptosis, while RPA3 is a component of the RPA complex that binds and stabilizes single-stranded DNA intermediates during DNA replication or damage. Together, they may play a role in maintaining genomic integrity in bladder cancer.|
| [COG4](https://www.ncbi.nlm.nih.gov/gene?term=COG4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX7](https://www.ncbi.nlm.nih.gov/gene?term=SNX7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.0</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between COG4 and SNX7 may involve Golgi morphology and intracellular trafficking, as COG4 is required for normal Golgi function and SNX7 is involved in endocytosis regulation and intracellular trafficking. The shortest path between these two genes in the BioGRID protein interaction network involves LAMTOR1, which is known to regulate lysosomal positioning, and nsp16, which is a viral protein involved in RNA capping.|
| [ENOPH1](https://www.ncbi.nlm.nih.gov/gene?term=ENOPH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.7</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no obvious functional association between ENOPH1 and MRPL18 in specific biological terms based on their known roles in the literature and the BioGRID protein interaction network. ENOPH1 is involved in the biosynthesis of a critical precursor molecule for methionine salvage pathways, while MRPL18 plays a role in mitochondrial ribosomal subunit formation.|
| [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.4</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between EML3 and REEP5 in TCGA-BLCA cancer may involve regulation of spindle assembly and microtubule organization, as well as interaction with tubulin and mitotic kinases. Both genes may play a role in chromosome alignment during metaphase.|
| [AXIN1](https://www.ncbi.nlm.nih.gov/gene?term=AXIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MMP1](https://www.ncbi.nlm.nih.gov/gene?term=MMP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.8</br>**STRING score:** 197</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 57 (min len 4) | One possible functional association between AXIN1 and MMP1 is the regulation of extracellular matrix (ECM) remodeling. AXIN1 is involved in the Wnt signaling pathway and the degradation of inhibitory SMAD7, while MMP1 cleaves collagens in the ECM. These processes are connected in the regulation of tissue development and homeostasis, suggesting a potential relationship between these two genes in these biological processes.|
| [HEATR1](https://www.ncbi.nlm.nih.gov/gene?term=HEATR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.2</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between HEATR1 and MBNL1 include their involvement in small subunit ribosomal biogenesis and alternative pre-mRNA splicing regulation respectively. However, based on the given information, no direct biological interaction or specific association between these two genes has been identified.|
| [CAPN1](https://www.ncbi.nlm.nih.gov/gene?term=CAPN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JUN](https://www.ncbi.nlm.nih.gov/gene?term=JUN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.7</br>**STRING score:** 218</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | CAPN1 and JUN can be functionally associated through their interactions in the BioGRID protein interaction network. CAPN1 is a calcium-regulated protease involved in cytoskeletal remodeling and signal transduction, while JUN is a transcription factor involved in activation-induced cell death of T cells and steroidogenic gene expression. The possible functional association between these genes could be related to their roles in regulating cellular processes involved in cancer progression, such as cell proliferation and apoptosis.|
| [CDC25B](https://www.ncbi.nlm.nih.gov/gene?term=CDC25B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP1](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.1</br>**STRING score:** 295</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | CDC25B and TP53BP1 are involved in different biological processes, but they both play a role in the cell cycle and DNA damage response. There may be a functional association in promoting DNA repair through non-homologous end joining mediated by TP53BP1, with CDC25B potentially being involved in the regulation of CDK1 activity during this process.|
| [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | PLSCR1 and SYK both have roles in immune response, with PLSCR1 regulating cytokine-regulated cell proliferation and differentiation and SYK mediating signal transduction downstream of transmembrane receptors in innate and adaptive immunity. The shortest path between these two genes in the BioGRID protein interaction network includes interactions involving SRC, EGFR, and CRKL, suggesting a potential involvement in cell adhesion and possibly cancer development.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.6</br>**STRING score:** 439</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | AKT1 is a serine/threonine kinase involved in regulating a range of cellular processes including metabolism, growth, and survival. RAB4A is a small GTPase that plays a role in vesicular transport and VEGFR2 signaling. The shortest path between AKT1 and RAB4A in the BioGRID protein interaction network suggests a possible functional association in a pathway involving RAF1, TSC1, ARFIP2, NOTCH1, and TRIM13.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP1](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.2</br>**STRING score:** 305</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both genes are involved in the response to DNA damage. CIRBP plays a protective role in the genotoxic stress response and TP53BP1 plays a key role in the repair of double-strand DNA breaks. The shortest path between them in the BioGRID protein interaction network suggests they may directly interact in this process.|
