| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 187 (min len 4) | EPN2 and TSPAN4 are potentially functionally associated in endocytosis and immune processes. EPN2 plays a role in clathrin-coated invaginations and endocytosis, while TSPAN4 is a non-receptor tyrosine kinase involved in signal transduction downstream of transmembrane receptors, including immunoreceptors such as the B-cell receptor. Both genes interact with ITSN1, and EPN2 binds AP-2 and clathrin, while TSPAN4 forms signaling complexes with activated receptors at the plasma membrane via interaction between its SH2 domains and receptor-tyrosine phosphorylated ITAM domains.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LAMA3](https://www.ncbi.nlm.nih.gov/gene?term=LAMA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.5</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HSPA8 is a molecular chaperone involved in protein quality control, while LAMA3 is a component of laminin-5 involved in cell adhesion and differentiation. These two genes may have a role in regulating protein folding and cell adhesion processes in TCGA-CESC cancer.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 222 (min len 4) | NVL and TSPAN4 appear to be functionally associated through a protein interaction network involving HNRNPH1. NVL is involved in ribosomal subunit biogenesis and pre-rRNA processing pathways, whereas TSPAN4 is a non-receptor tyrosine kinase that mediates signal transduction downstream of transmembrane receptors, regulating innate and adaptive immunity, cell adhesion, and vascular development. The possible functional association between these genes may involve coordinating gene expression and signal transduction events that regulate cellular differentiation, proliferation, and/or growth.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 91 (min len 4) | LYPLA1 and TSPAN4 are not known to have a direct functional association. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while TSPAN4 is a non-receptor tyrosine kinase involved in signal transduction downstream of various transmembrane receptors, regulating several biological processes including immunity, cell adhesion, osteoclast maturation, platelet activation, and vascular development.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 103.9</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional association: IKBKB phosphorylates SFN, indicating a potential regulatory relationship in signaling pathways.   IKBKB is a serine kinase that plays an essential role in the NF-kappa-B signaling pathway, while SFN is a p53-regulated inhibitor of G2/M progression. SFN is also involved in a complex with various proteins, including those involved in vesicle trafficking and RNA metabolism.|
| [PRKCH](https://www.ncbi.nlm.nih.gov/gene?term=PRKCH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNPS1](https://www.ncbi.nlm.nih.gov/gene?term=RNPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: RNPS1 is involved in RNA splicing and PRKCH can inhibit the kinase activity of protein kinases, including PRKCE. It is possible that PRKCH may regulate RNPS1-mediated splicing through the inhibition of PRKCE. Further research would be needed to confirm this possible association.|
| [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | One possible functional association between LGMN and SFN is their shared involvement in protein degradation pathways, with LGMN known for its role in lysosomal protein degradation and SFN playing a role in proteasomal degradation. The identified protein interaction network suggests potential direct physical interactions between the two proteins.|
| [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPIA](https://www.ncbi.nlm.nih.gov/gene?term=RPIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | PSME2 is involved in immunoproteasome assembly and antigen processing, while RPIA participates in the pentose phosphate pathway and has various protein-protein interactions. They have a possible functional association through the BioGRID protein interaction network, which shows they potentially interact through several other proteins.|
| [FAM57A](https://www.ncbi.nlm.nih.gov/gene?term=FAM57A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDLIM1](https://www.ncbi.nlm.nih.gov/gene?term=PDLIM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 199 (min len 4) | Possible functional associations between FAM57A and PDLIM1 include involvement in cytoskeletal organization, cell migration, and maintenance of cell polarity. FAM57A has been linked to RB1-E2F1 complex disruption during viral infections while PDLIM1 acts as an adapter for bringing other proteins to the cytoskeleton and interacts with various actin-binding proteins. The shortest path in the BioGRID protein interaction network between FAM57A and PDLIM1 involves CUL3, suggesting potential involvement in protein degradation pathways.|
| [ATP6V1F](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1F[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between ATP6V1F and LIG1 based on their known biological processes, which involve vacuolar ATPase activity and DNA repair/replication/recombination, respectively. However, they interact indirectly through DDB1 and MOV10 in the BioGRID protein interaction network.|
| [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.4</br>**STRING score:** 999</br>**BioGRID count:** 6</br>**BioGRID shortest paths:** 1 (min len 2) | SFN interacts with TP53 in a protein interaction network. SFN is involved in G2/M progression and can be degraded by COP1. TP53 is a tumor suppressor that regulates cell cycle, induces apoptosis and interacts with PPP1R13B/ASPP1 or TP53BP2/ASPP2 to activate pro-apoptotic activity. Possible functional associations between SFN and TP53 may involve their roles in cell cycle regulation and apoptosis.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.2</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 46 (min len 3) | MBNL1 and TP53 have no known functional association. MBNL1 mediates pre-mRNA splicing regulation, while TP53 acts as a tumor suppressor and induces growth arrest or apoptosis depending on the physiological circumstances and cell type.|
| [POLB](https://www.ncbi.nlm.nih.gov/gene?term=POLB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.2</br>**STRING score:** 701</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional association between POLB and RPA1 is their involvement in DNA repair. POLB is a repair polymerase that plays a key role in base-excision repair, while RPA1 binds and stabilizes single-stranded DNA intermediates that form during DNA replication or upon DNA stress, preventing their reannealing, and recruits and activates different proteins and complexes involved in DNA metabolism. The BioGRID protein interaction network shows that there are several shortest paths between these two genes via other DNA repair proteins such as XRCC1, PNKP, WRN, PCNA, PARP1.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.8</br>**STRING score:** 231</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | The possible functional association between HSPA8 and LPGAT1 is unclear from the available context. HSPA8 is a molecular chaperone involved in several cellular processes, including proteome protection and protein quality control, mitochondrial import, autophagy, and transcriptional repression. LPGAT1 is a lysophospholipid acyltransferase involved in fatty acyl chain remodeling of glycerophospholipids in the endoplasmic reticulum membrane.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.1</br>**STRING score:** 287</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between GLOD4 and HSPB1 include stress response and actin organization, as well as regulation of protein phosphorylation and transport. GLOD4 inhibits PTK2/FAK1 signaling while HSPB1 acts as a molecular chaperone and interacts with other heat shock proteins. There is no direct evidence of functional associations between these two genes, although they may contribute to cellular processes related to protein homeostasis and stress response.|
| [EAPP](https://www.ncbi.nlm.nih.gov/gene?term=EAPP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EGR1](https://www.ncbi.nlm.nih.gov/gene?term=EGR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.9</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 291 (min len 4) | Possible functional associations between EAPP and EGR1 could involve the regulation of cell survival, proliferation, and cell death, as well as the modulation of inflammatory processes and tissue damage after ischemia. Both genes also play roles in the cell cycle and apoptosis.|
| [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATP6V1F](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1F[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.5</br>**STRING score:** 219</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between ATP11B and ATP6V1F could be related to the maintenance of membrane lipid asymmetry and pH regulation of intracellular compartments, as ATP11B is involved in flippase complex that transports aminophospholipids, phosphatidylserines, and phosphatidylethanolamines from the outer to the inner leaflet of membranes, while ATP6V1F is a subunit of the V1 complex of V-ATPase responsible for acidifying intracellular compartments.|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOCS2](https://www.ncbi.nlm.nih.gov/gene?term=SOCS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.9</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | DUSP4 and SOCS2 have a possible functional association in regulating signal transduction pathways. DUSP4 dephosphorylates MAP kinases, while SOCS2 is a negative regulator in the growth hormone/IGF1 signaling pathway and interacts with IGF1R. The shortest path between these two genes in the BioGRID protein interaction network involves the protein AMBRA1.|
