| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 121.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 187 (min len 4) | Possible functional associations between EPN2 and TSPAN4 include their interaction through various proteins such as FCHO1 and ORF9c. EPN2 plays a role in endocytosis and binds AP-2 and clathrin, while TSPAN4 mediates signal transduction downstream of transmembrane receptors and is involved in innate and adaptive immunity, cell adhesion, osteoclast maturation, platelet activation, and vascular development.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LAMA3](https://www.ncbi.nlm.nih.gov/gene?term=LAMA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.5</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between HSPA8 and LAMA3 is cell adhesion and differentiation. HSPA8 is involved in protein quality control and chaperone-mediated autophagy, while LAMA3 is a subunit of laminin-5 that is involved in cell adhesion and differentiation of keratinocytes.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 222 (min len 4) | NVL and TSPAN4 have a shortest path in the BioGRID protein interaction network, suggesting a functional association. NVL is involved in ribosome biogenesis and telomerase activity while TSPAN4 mediates signal transduction downstream of transmembrane receptors, regulating several biological processes including immunity, cell adhesion, and vascular development, among others. It is possible that NVL may play a role in regulating TSPAN4-mediated signaling pathways or vice versa, but further research is needed to confirm any functional association between these two genes.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 91 (min len 4) | Possible functional association: TSPAN4 is involved in signal transduction downstream of transmembrane receptors, while LYPLA1 acts as an acyl-protein thioesterase and hydrolyzes fatty acids from proteins. These genes may interact in regulating protein palmitoylation and depalmitoylation, which may affect intracellular signaling processes mediated by TSPAN4 in cancer cells.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 103.9</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between IKBKB and SFN could be their involvement in immune response and viral infections. IKBKB plays a key role in the NF-kappa-B signaling pathway, which activates transcription of immune response genes, while SFN acts as an exoribonuclease and interacts with a viral protein. The shortest protein interaction path suggests they may also be involved in regulating apoptosis and DNA damage response.|
| [PRKCH](https://www.ncbi.nlm.nih.gov/gene?term=PRKCH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNPS1](https://www.ncbi.nlm.nih.gov/gene?term=RNPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between PRKCH and RNPS1 could involve splicing regulation and apoptosis. PRKCH is involved in the regulation of cell differentiation, tight junction integrity, and foam cell formation, while RNPS1 is an auxiliary component of splicing-dependent multiprotein exon junction complex and participates in mRNA 3'-end cleavage, UPF2-dependent nonsense-mediated decay, and increase of mRNA abundance and translational efficiency. The interaction between PRKCH and RNPS1 through PLEKHA4 may modulate splicing regulation and apoptotic processes.|
| [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: LGMN may interact with SFN in the lysosomal/endosomal system for the processing of proteins for MHC class II antigen presentation.   LGMN is a protease involved in lysosomal protein degradation, and SFN is an exoribonuclease that preferentially degrades DNA and RNA oligonucleotides. However, based on the protein interaction network, LGMN may interact with SFN in the lysosomal/endosomal system, suggesting a possible role in the processing of proteins for MHC class II antigen presentation.|
| [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPIA](https://www.ncbi.nlm.nih.gov/gene?term=RPIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | PSME2 and RPIA have no known functional associations based on the provided context. PSME2 is involved in immunoproteasome assembly and antigen processing while RPIA is involved in the non-oxidative branch of the pentose phosphate pathway and has various protein interactions.|
| [FAM57A](https://www.ncbi.nlm.nih.gov/gene?term=FAM57A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDLIM1](https://www.ncbi.nlm.nih.gov/gene?term=PDLIM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 199 (min len 4) | Possible functional associations between FAM57A and PDLIM1 are in regulation of cell migration and cytoskeletal organization, as both genes are involved in these processes through interactions with cytoskeletal proteins and kinase activity. FAM57A may also have a role in regulating RB1 activity during viral infections.|
| [ATP6V1F](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1F[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no known direct functional associations between ATP6V1F and LIG1. ATP6V1F codes for a subunit of the vacuolar ATPase, which is involved in acidification of intracellular compartments. LIG1 codes for a negative regulator of receptor tyrosine kinases.|
| [SFN](https://www.ncbi.nlm.nih.gov/gene?term=SFN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.4</br>**STRING score:** 999</br>**BioGRID count:** 6</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between SFN and TP53 may involve regulation of cell cycle progression and induction of apoptosis. Interaction between the two genes may play a role in TP53-dependent transcriptional repression leading to apoptosis and cell-cycle regulation.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.2</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 46 (min len 3) | One possible functional association between MBNL1 and TP53 is the regulation of alternative splicing of specific pre-mRNA targets. MBNL1 mediates pre-mRNA splicing regulation, while TP53 acts as a tumor suppressor and negatively regulates cell division by controlling a set of genes required for this process.|
| [POLB](https://www.ncbi.nlm.nih.gov/gene?term=POLB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.2</br>**STRING score:** 701</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between POLB and RPA1 are in DNA repair processes, as both genes have been shown to interact with proteins involved in base-excision and nucleotide excision repair. POLB has 5'-deoxyribose-5-phosphate lyase activity and is involved in gap-filling DNA synthesis, while RPA1 is part of the replication protein A complex and binds and stabilizes single-stranded DNA intermediates during DNA replication and stress.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.8</br>**STRING score:** 231</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | There are currently no known functional associations between HSPA8 and LPGAT1. HSPA8 is involved in protein quality control, mitochondrial import, transcriptional repression, and protein degradation processes, while LPGAT1 is involved in fatty acid remodeling of phospholipids and can acylate monoacylglycerols.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.1</br>**STRING score:** 287</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between GLOD4 and HSPB1 based on the provided context.  GLOD4 inhibits PTK2/FAK1 phosphorylation and signaling, while HSPB1 functions as a molecular chaperone involved in stress resistance and actin organization.|
| [EAPP](https://www.ncbi.nlm.nih.gov/gene?term=EAPP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EGR1](https://www.ncbi.nlm.nih.gov/gene?term=EGR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.9</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 291 (min len 4) | Possible functional associations between EAPP and EGR1 include regulating cell survival, proliferation, and cell death, activating expression of p53/TP53 and TGFB1 to prevent tumor formation, and regulating inflammatory processes and development of tissue damage after ischemia. They may also regulate the expression of core-clock genes and biosynthesis of luteinizing hormone in the pituitary. The shortest path between the two genes in the BioGRID protein interaction network involves PSMA3, which plays a role in proteasome-mediated degradation, suggesting potential involvement in protein regulation as well.|
| [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATP6V1F](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1F[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.5</br>**STRING score:** 219</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no known functional association between ATP11B and ATP6V1F. ATP11B is involved in lipid transport, specifically maintaining membrane lipid asymmetry, while ATP6V1F is a subunit of the V1 complex of vacuolar(H+)-ATPase responsible for acidification of intracellular compartments and in some cell types, the extracellular environment.|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOCS2](https://www.ncbi.nlm.nih.gov/gene?term=SOCS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.9</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: DUSP4 is a mitogen-activated protein kinase (MAPK) phosphatase, and SOCS2 is a negative regulator in the growth hormone/IGF1 signaling pathway. They may function together in regulating cellular proliferation and differentiation processes. The BioGRID protein interaction network shows that they are connected through AMBRA1 in the shortest path.|
