| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [OXCT1](https://www.ncbi.nlm.nih.gov/gene?term=OXCT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLA2G4A](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | OXCT1 and PLA2G4A may be functionally associated through the generation of acetyl-CoA, which is produced from acetoacetyl-CoA by OXCT1 and used in the citric acid cycle for energy production. PLA2G4A, which has phospholipase activity, may play a role in providing the fatty acyl chains for acetyl-CoA production. The shortest path between the two genes in the BioGRID protein interaction network involves PARK2.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | There are no obvious functional associations between CDK2 and MBNL1. CDK2 is a kinase involved in cell cycle regulation, while MBNL1 mediates pre-mRNA alternative splicing regulation. The shortest path between the two genes in the BioGRID protein interaction network involves intermediate proteins and not a direct physical interaction.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 127.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There aren't any obvious functional associations between MBNL1 and NCAPD2 based on their known processes, as they both have distinct roles in alternative splicing and chromosome condensation, respectively. However, it's possible that they may interact indirectly through shared interactions with other proteins or regulatory pathways.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between GLOD4 and SMARCC1 could be that they both play a role in transcriptional regulation and chromatin remodeling. Specifically, GLOD4 may inhibit LPXN nuclear accumulation and limit SRF-dependent gene transcription, while SMARCC1 is involved in the activity of genes essential for dendrite growth and regulating the switch from neural stem/progenitor cells to postmitotic neurons. However, without further experimental evidence, any functional association between these two genes remains speculative.|
| [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KLHL9](https://www.ncbi.nlm.nih.gov/gene?term=KLHL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between KAT6A and KLHL9 could be related to mitotic progression and cytokinesis. KAT6A is a histone acetyltransferase that may act as a transcriptional coactivator, while KLHL9 is a substrate-specific adapter of a BCR E3 ubiquitin-protein ligase complex required for mitotic progression and cytokinesis. The shortest path between KAT6A and KLHL9 in the BioGRID protein interaction network also involves a protein (HNRNPL) that is involved in splicing and mRNA processing, suggesting potential associations with gene expression.|
| [OXCT1](https://www.ncbi.nlm.nih.gov/gene?term=OXCT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between OXCT1 and YME1L1 could involve regulation of mitochondrial function and metabolism. OXCT1 is a key enzyme in ketone body catabolism, while YME1L1 is an ATP-dependent metalloprotease that regulates mitochondrial morphology. These genes may work together to maintain normal mitochondrial structure and metabolism.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX11](https://www.ncbi.nlm.nih.gov/gene?term=SNX11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 4) | Possible functional associations between CENPO and SNX11 include roles in mitotic progression and chromosome segregation, as well as membrane trafficking in endosomes. The interaction between CENPO and SNX11 via NECAB and DCAF10 suggests a possible role in protein sorting and membrane trafficking in centrosomes.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | CRKL and TOPBP1 have different biological functions, and there is no known specific functional association between these two genes. CRKL is involved in intracellular signaling and interacts with various phosphorylated proteins, while TOPBP1 is required for DNA replication, the rescue of stalled replication forks, and checkpoint control.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between CRKL and MBNL1 could be in alternative splicing regulation and signal transduction. MBNL1 mediates pre-mRNA alternative splicing while CRKL interacts with tyrosine-phosphorylated proteins, mediating the transduction of intracellular signals. The shortest path between the two genes in the BioGRID protein interaction network suggests a potential physical interaction between them in alternative splicing regulation.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | There is no known direct biological association between IKBKB and PAN2. IKBKB acts as a serine kinase in the NF-kappa-B signaling pathway, while PAN2 is a retinol dehydrogenase with high activity towards various retinols and interacts with human cytomegalovirus. The shortest path between these genes in the BioGRID protein interaction network is through various intermediates.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.1</br>**STRING score:** 321</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | Possible functional associations between PARP1 and PIK3CA include regulating cell growth, survival, proliferation, and signaling cascades involved in DNA repair and apoptosis. PARP1 interacts with PIK3CA through a protein interaction network, suggesting a potential biological relationship in regulating cellular processes.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 105.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: PIK3R4 is a subunit of the PI3K complex, which plays a role in membrane trafficking pathways, including autophagy. PARP1 is involved in DNA repair and apoptosis. There may be a connection between autophagy and DNA repair/apoptosis, but further investigation is needed to determine the exact functional association between these two genes.|
| [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZMYM2](https://www.ncbi.nlm.nih.gov/gene?term=ZMYM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.9</br>**STRING score:** 514</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 61 (min len 3) | Possible functional associations between SUZ12 and ZMYM2 relate to their involvement in gene regulation and chromatin modification. SUZ12 is a component of the PRC2 complex involved in histone methylation and transcriptional repression, while ZMYM2 is involved in the negative regulation of transcription. The shortest path between these genes in the BioGRID interaction network suggests they may interact through multiple proteins, including SETDB1 and EED, both of which are involved in chromatin regulation.|
| [NUP62](https://www.ncbi.nlm.nih.gov/gene?term=NUP62[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNPS1](https://www.ncbi.nlm.nih.gov/gene?term=RNPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.3</br>**STRING score:** 615</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional associations between NUP62 and RNPS1 include their involvement in pre- and post-splicing multiprotein mRNP complexes, with RNPS1 being a component of the exon junction complex and ASAP complex, while NUP62 plays a role in nuclear pore complex assembly. They may also be involved in mRNA metabolism and splicing modulation. However, their exact biological relationship requires further investigation.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.0</br>**STRING score:** 157</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between GLOD4 and PIH1D1 based on the provided context, as they appear to be involved in distinct cellular processes. GLOD4 inhibits PTK2/FAK1 phosphorylation while PIH1D1 is involved in assembly of small nucleolar ribonucleoprotein particles and enhances pre-rRNA transcription.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 985</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | BIRC2 and CFLAR have a functional association in the regulation of apoptosis. BIRC2 is a multi-functional protein that regulates apoptosis, innate immune signaling, and cell proliferation. CFLAR is an apoptosis regulator protein that blocks caspase-8 recruitment in the death-inducing signaling complex. Their protein interaction network supports this functional association, with the shortest path between BIRC2 and CFLAR including intermediates in apoptosis regulation, such as caspases and RIP kinases.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 310</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations between LYN and PTPN12 could involve regulation of cellular signaling cascades, such as integrin signaling, immune responses, and cytokine release. LYN acts primarily as a negative regulator, while PTPN12 functions as a protein phosphatase that dephosphorylates tyrosine kinases, including LYN. The shortest path between the two genes in the BioGRID protein interaction network suggests potential interactions in multiple biological processes.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.5</br>**STRING score:** 693</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: PARP1 plays a role in DNA repair and promotes apoptosis after DNA damage, while TOPBP1 is required for DNA replication, rescues stalled replication forks, and controls checkpoints. It is possible that TOPBP1 recruits PARP1 for DNA repair in certain contexts.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP62](https://www.ncbi.nlm.nih.gov/gene?term=NUP62[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.5</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There are no clear or direct functional associations between LIG1 and NUP62. LIG1 acts as a negative feedback regulator of receptor tyrosine kinases, while NUP62 is an essential component of the nuclear pore complex involved in nucleocytoplasmic transport and mitotic cell cycle progression.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.3</br>**STRING score:** 437</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 72 (min len 3) | CDC42 and HIF1A may be functionally associated through their roles in cell migration, as CDC42 regulates this process and HIF1A induces expression of cytokines that promote inflammation and immune cell recruitment in response to hypoxia. Their interaction may play a role in the response to microbial infections, such as in the case of SARS-CoV-2, which may use this pathway to promote its replication in monocytes.|
| [GNAS](https://www.ncbi.nlm.nih.gov/gene?term=GNAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.7</br>**STRING score:** 683</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | It is possible for GNAS and TP53 to have a functional association in cell cycle regulation and apoptosis. GNAS is involved in downstream signaling pathways while TP53 acts as a tumor suppressor and induces apoptosis, suggesting a potential indirect influence of GNAS on TP53 activity.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.4</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between IKBKB and TBP include regulation of gene expression as TBP is a transcription factor and IKBKB plays a role in NF-kappa-B signaling pathway, which regulates the transcription of many genes involved in immune response, growth control, or protection against apoptosis. The shortest path in the BioGRID protein interaction network shows potential direct interactions between IKBKB and TBP through intermediate proteins such as NFKBIB, CUEDC2, RELA, TP53, and JUN.|
| [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations: NET1 and PLK1 are connected in the BioGRID protein interaction network via several intermediate proteins involved in cell division and spindle assembly. PLK1 is a key regulator of mitosis, including centrosome maturation and spindle assembly, while NET1 has been implicated in initiation of chromosome replication and regulation of transcription complex formation.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 306</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between GLOD4 and VAT1 could involve regulation of mitochondrial function or calcium signaling. GLOD4 may inhibit LPXN nuclear accumulation, while VAT1 plays a part in calcium-regulated keratinocyte activation and negatively regulates mitochondrial fusion in cooperation with MFN1-2. The shortest path between the two genes suggests protein interactions mediated by SOD1, BIRC3, SAR1B, SOX2, and TRIM67.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.9</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association between IKBKB and PTPN12 is that PTPN12 regulates signaling via ERBB2 and PTK2B/PYK2, and IKBKB phosphorylates IKK-related kinases TBK1 and IKBKE, which can participate in cytokine-stimulated activation of ERBB2. This suggests that both genes may play a role in regulating cytokine signaling and cell growth in cancer.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.6</br>**STRING score:** 999</br>**BioGRID count:** 8</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HIF1A is involved in the response to hypoxia and infection, whereas SMAD3 is a transcriptional modulator activated by TGF-beta. They regulate different biological processes, but could potentially interact in the regulation of cell growth and differentiation, as well as in inflammatory responses. However, no direct biological association between these two genes has been reported in the literature.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DNMT1](https://www.ncbi.nlm.nih.gov/gene?term=DNMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.1</br>**STRING score:** 263</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | ABL1 and DNMT1 may be functionally associated in regulating DNA damage response and apoptosis, as both genes are involved in these processes. The shortest path in the BioGRID protein interaction network suggests that ABL1 may directly phosphorylate RAD9A, a mediator of DNA repair, while DNMT1 is responsible for maintaining DNA methylation patterns that coordinate with methylation of histones.|
| [ZMYM2](https://www.ncbi.nlm.nih.gov/gene?term=ZMYM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF589](https://www.ncbi.nlm.nih.gov/gene?term=ZNF589[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.1</br>**STRING score:** 233</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear biological basis for a functional association between ZMYM2 and ZNF589, based on their known functions, protein interactions, and involvement in different biological processes. ZMYM2 is involved in transcriptional regulation and may interact with histone deacetylase complexes, while ZNF589 is involved in hematopoietic stem/progenitor cell differentiation and binds to DNA as a transcriptional repressor.|
