| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [OXCT1](https://www.ncbi.nlm.nih.gov/gene?term=OXCT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLA2G4A](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | OXCT1 and PLA2G4A have no known direct functional association. OXCT1 is involved in the catabolism of ketone bodies while PLA2G4A is involved in membrane lipid remodeling and biosynthesis of lipid mediators of the inflammatory response. The shortest path between them in the BioGRID protein interaction network is through PARK2.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | Possible functional associations between CDK2 and MBNL1 may involve regulation of alternative splicing of pre-mRNA targets involved in controlling the cell cycle and cell proliferation. CDK2 is a serine/threonine-protein kinase that controls the cell cycle and is involved in orchestrating a fine balance between cellular proliferation, cell death, and DNA repair. MBNL1 mediates the alternative splicing regulation of specific pre-mRNA targets and acts as an activator or repressor of splicing. Both genes may have a role in coordinating control of the cell cycle and alternative splicing processes.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 127.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no clear functional association between MBNL1 and NCAPD2 based on their known roles in pre-mRNA splicing regulation and chromosome condensation, respectively. However, both genes are involved in essential biological processes, such as gene expression and cell division, that may indirectly affect each other's function.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between GLOD4 and SMARCC1 are involvement in chromatin remodeling and transcriptional regulation. SMARCC1 is a component of the SWI/SNF chromatin remodeling complexes and GLOD4 interacts with transcription factors, suggesting a potential role for these genes in regulating gene expression.|
| [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KLHL9](https://www.ncbi.nlm.nih.gov/gene?term=KLHL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: KLHL9 may regulate the activity of KAT6A by ubiquitinating it. KAT6A is a histone acetyltransferase that can modify the chromatin structure, affecting gene expression and cell cycle regulation, which may in turn impact KLHL9-mediated mitotic progression and cytokinesis.|
| [OXCT1](https://www.ncbi.nlm.nih.gov/gene?term=OXCT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no known direct functional association between OXCT1 and YME1L1. OXCT1 is involved in ketone body catabolism while YME1L1 is an ATP-dependent metalloprotease important for maintaining mitochondrial morphology and proteostasis. However, both genes have roles in maintaining mitochondrial function, and disrupted mitochondrial function is associated with a variety of diseases, including cancer.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX11](https://www.ncbi.nlm.nih.gov/gene?term=SNX11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 4) | Possible functional association between CENPO and SNX11 is their involvement in mitotic progression and chromosome segregation. CENPO is a component of the CENPA-CAD complex, which is involved in these processes, while SNX11 is a phosphoinositide-binding protein involved in protein sorting and membrane trafficking in endosomes. However, further experimental evidence is needed to confirm any direct interaction or cooperation between these two genes.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CRKL and TOPBP1 include their roles in DNA replication and checkpoint control. CRKL may mediate intracellular signal transduction and interact with tyrosine-phosphorylated proteins, while TOPBP1 is required for DNA replication, binds to DNA breaks, and regulates E2F1 activity during G1/S transition and after DNA damage. There is currently no direct evidence of a physical interaction between these two proteins, but both are involved in similar molecular pathways.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is currently no known functional association between CRKL and MBNL1. CRKL is involved in intracellular signaling and interacts with several proteins, while MBNL1 mediates pre-mRNA alternative splicing regulation and binds to RNA structures.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between IKBKB and PAN2 could be related to mRNA turnover. IKBKB is involved in the NF-kappa-B signaling pathway and phosphorylates several components of the pathway, while PAN2 is a deadenylase involved in general and miRNA-mediated mRNA turnover. They may be involved in regulating the stability of specific mRNAs, such as those involved in immune response or growth control.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.1</br>**STRING score:** 321</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | Possible functional associations: PARP1 is involved in DNA repair and apoptosis while PIK3CA is a lipid kinase that plays a role in cellular signaling, growth, and survival. There may be a functional association through the PI3K/AKT signaling pathway, which is known to regulate apoptosis and DNA repair.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 105.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: PIK3R4 is a regulatory subunit of the PI3K complex, which is involved in multiple membrane trafficking pathways including autophagy. PARP1 interacts with AIFM1 and promotes DNA damage-induced apoptosis. There is no direct biological connection between these two proteins, but they may indirectly influence autophagy and apoptosis pathways.|
| [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZMYM2](https://www.ncbi.nlm.nih.gov/gene?term=ZMYM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.9</br>**STRING score:** 514</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 61 (min len 3) | Possible functional associations between SUZ12 and ZMYM2 include their involvement in transcriptional regulation and chromatin remodeling, as they are both associated with components of the Polycomb repressive complex 2 (PRC2) and a BHC histone deacetylase complex, respectively. The shortest path between these two genes in the BioGRID protein interaction network suggests these genes may physically interact through proteins such as SETDB1, EED, and RBBP4, and potentially play a role in modulating chromatin state to control gene expression.|
| [NUP62](https://www.ncbi.nlm.nih.gov/gene?term=NUP62[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNPS1](https://www.ncbi.nlm.nih.gov/gene?term=RNPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.3</br>**STRING score:** 615</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional associations between NUP62 and RNPS1 could involve their roles in mRNA splicing and transport. NUP62 is an essential component of the nuclear pore complex and involved in nucleocytoplasmic transport, while RNPS1 is part of pre- and post-splicing multiprotein mRNP complexes and the exonic splicing enhancer complex. They may interact in regulating mRNA splicing and transport.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.0</br>**STRING score:** 157</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between GLOD4 and PIH1D1 may involve the regulation of gene transcription and protein complex biogenesis. GLOD4 may attenuate LPXN nuclear accumulation and SRF-dependent gene transcription, while PIH1D1 is involved in the assembly of C/D box snoRNP particles and recruits the SWI/SNF complex to the core promoter of rRNA genes, as well as positively regulating mTORC1 complex assembly and activity.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 985</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | BIRC2 and CFLAR have a functional association as they both regulate apoptosis and modulate inflammatory signaling. BIRC2 also regulates the NF-kappa-B signaling pathway and acts as an E3 ubiquitin-protein ligase, while CFLAR acts as an inhibitor of TNFRSF6-mediated apoptosis. They both interact with caspases and interact indirectly through TRAF2 in the BioGRID protein interaction network.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 310</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | LYN and PTPN12 are both involved in regulating cellular signaling cascades. They may potentially interact with each other in the regulation of immune responses, hematopoiesis, and integrin signaling. LYN is a non-receptor tyrosine-protein kinase that acts downstream of immune receptors, while PTPN12 is a phosphatase that dephosphorylates tyrosine kinases and regulates signaling via kinase pathways.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.5</br>**STRING score:** 693</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PARP1 and TOPBP1 include their roles in DNA damage response, DNA repair, and DNA replication. PARP1 is involved in repairing DNA damage through the process of poly-ADP-ribosylation, while TOPBP1 is required for DNA replication and plays a role in the rescue of stalled replication forks and checkpoint control. Both genes are involved in maintaining genome stability and may work together in regulating DNA repair and replication processes.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP62](https://www.ncbi.nlm.nih.gov/gene?term=NUP62[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.5</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between LIG1 and NUP62 include their role in DNA repair and replication, as well as their involvement in nucleocytoplasmic transport. LIG1 is a DNA ligase that seals nicks, while NUP62 is an essential component of the nuclear pore complex that regulates centrosome segregation and spindle orientation.   However, their physical interaction in the BioGRID protein network is limited and indirect, suggesting that they may not have a direct functional relationship.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.3</br>**STRING score:** 437</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 72 (min len 3) | Possible functional associations between CDC42 and HIF1A involve regulating cellular responses, cell migration, filopodia and podosomes formation, phagocytosis, and dendritic spine structural plasticity. They may also be involved in inducing glycolysis and pro-inflammatory responses in monocytes during infection with SARS-CoV-2. It is unclear if there is a direct interaction or pathway between these two genes.|
| [GNAS](https://www.ncbi.nlm.nih.gov/gene?term=GNAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.7</br>**STRING score:** 683</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between GNAS and TP53 could be related to regulation of cell growth and proliferation, as GNAS is involved in immune cell development and signaling, while TP53 acts as a tumor suppressor and regulates cell cycle. However, without further experimental evidence, the exact nature of their association remains unclear.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.4</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | There are no clear functional associations between IKBKB and TBP based on their known biological roles. IKBKB is a serine kinase involved in the NF-kappa-B signaling pathway, while TBP is a general transcription factor involved in RNA polymerase II-dependent transcription. The shortest path between the two in the BioGRID protein interaction network involves intermediate proteins such as NFKBIB and CUEDC2.|
| [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | NET1 and PLK1 both play roles in regulating cell division during the M phase of the cell cycle. NET1 acts as a guanine nucleotide exchange factor for RhoA GTPase, while PLK1 regulates centrosome maturation, spindle assembly, and cytokinesis. The interaction between them in the protein interaction network suggests they may be involved in the same signaling pathway during cell division.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 306</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Based on their protein interaction network, GLOD4 and VAT1 may have a functional association in mitochondrial fusion and calcium regulation in keratinocyte activation. GLOD4 may inhibit LPXN nuclear accumulation, while VAT1 possesses ATPase activity and negatively regulates mitochondrial fusion.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.9</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between IKBKB and PTPN12 are that IKBKB phosphorylates PTPN12, which dephosphorylates cellular tyrosine kinases, such as ERBB2 and PTK2B/PYK2, and regulates signaling via ERBB2 and PTK2B/PYK2. Both genes are involved in cellular signaling pathways, with IKBKB playing an essential role in the NF-kappa-B pathway and PTPN12 regulating signaling cascades through tyrosine kinase dephosphorylation.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.6</br>**STRING score:** 999</br>**BioGRID count:** 8</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both HIF1A and SMAD3 are involved in inflammatory responses, with HIF1A inducing pro-inflammatory cytokines and facilitating replication of human coronavirus SARS-CoV-2, and SMAD3 being activated by TGF-beta to regulate transcription of genes involved in various inflammation-related processes. However, there is not enough context to suggest a direct functional association between these two genes.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DNMT1](https://www.ncbi.nlm.nih.gov/gene?term=DNMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.1</br>**STRING score:** 263</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | ABL1 and DNMT1 may have a functional association in DNA damage response and apoptosis as both genes are involved in these processes. ABL1 is a kinase that regulates multiple signaling cascades, including DNA repair, while DNMT1 maintains methylation patterns essential for epigenetic inheritance and associates with chromatin during G2 and M phases to maintain DNA methylation independently of replication. Additionally, ABL1 phosphorylates RAD9A, a mediator of DNA repair, which is also a substrate of DNMT1.|
| [ZMYM2](https://www.ncbi.nlm.nih.gov/gene?term=ZMYM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF589](https://www.ncbi.nlm.nih.gov/gene?term=ZNF589[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.1</br>**STRING score:** 233</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ZMYM2 and ZNF589 could involve their interaction in BHC histone deacetylase complex that regulates transcriptional repression. ZMYM2 can form homodimers and interact with FOXP1 and FOXP2, while ZNF589 may play a role in hematopoietic stem/progenitor cell differentiation and interacts with TRIM28.|
