| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RNH1](https://www.ncbi.nlm.nih.gov/gene?term=RNH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 137.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between RNH1 and SYK could be in the regulation of redox homeostasis and signal transduction downstream of transmembrane receptors, specifically in B-cell receptor (BCR) and T-cell receptor signaling. SYK mediates several biological processes including innate and adaptive immunity, cell adhesion, osteoclast maturation, platelet activation, and vascular development, while RNH1 inhibits ribonucleases and forms high-affinity heterodimers with RNASE1, ANG, and RNASE2.|
| [CLTB](https://www.ncbi.nlm.nih.gov/gene?term=CLTB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FCHO1](https://www.ncbi.nlm.nih.gov/gene?term=FCHO1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.1</br>**STRING score:** 852</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | CLTB and FCHO1 are involved in clathrin-mediated endocytosis. They interact with each other and with other proteins such as AP2A2, DAB2, and clathrin to regulate protein uptake and membrane trafficking. There is a possible functional association between these two genes in the regulation of BMP signaling and T-cell activation.|
| [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Based on the provided BioGRID protein interaction network and context, RBM34 and SLC25A46 may have a functional association in regulating mitochondrial organization and lipid homeostasis. RBM34 also interacts with the EGFR and promotes its dimerization, potentially suggesting a role in cell growth and signaling.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ECH1 and PTPRF may involve regulation of metabolic pathways and signaling processes. ECH1 is a homohexamer involved in isomerization of metabolites while PTPRF has enzymatic activity and interacts with signaling proteins. The connection could be related to the modulation of metabolic signaling pathways in cancer processes.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOX4](https://www.ncbi.nlm.nih.gov/gene?term=SOX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.2</br>**STRING score:** 235</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between NR3C1 and SOX4 in TCGA-CESC cancer could be related to the regulation of gene expression and transcriptional activation. NR3C1 is a transcription factor that binds to DNA and interacts with coactivator complexes to activate gene expression, while SOX4 is a transcriptional activator that binds to enhancer motifs to modulate gene expression. It is possible that these genes interact to regulate gene expression in cervical cancer.|
| [CLTB](https://www.ncbi.nlm.nih.gov/gene?term=CLTB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIPA](https://www.ncbi.nlm.nih.gov/gene?term=LIPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CLTB and LIPA include their interaction in the BioGRID protein network via IGF2R and TRIM67, as well as their involvement in endocytosis and lipid metabolism processes. CLTB plays a role in Clathrin-coated pit formation and interacts with proteins involved in endocytosis, while LIPA catalyzes the deacylation of lipids in endocytosed low density lipoproteins.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC27A3](https://www.ncbi.nlm.nih.gov/gene?term=SLC27A3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 129.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between IDE and SLC27A3 could be related to lipid metabolism as both genes are involved in fatty acid processing. IDE has been shown to interact with a fatty acid transporter while SLC27A3 has been characterized as an acyl-CoA ligase, suggesting a potential interaction in the transport and utilization of fatty acids.|
| [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPOX](https://www.ncbi.nlm.nih.gov/gene?term=PPOX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: There is no known direct functional association between FOXO3 and PPOX.   FOXO3: Transcriptional activator that regulates processes such as apoptosis and autophagy, and also acts as a key regulator of regulatory T-cells differentiation and chondrogenic commitment of skeletal progenitor cells in response to lipid availability.  PPOX: Catalyzes the 6-electron oxidation of protoporphyrinogen-IX to form protoporphyrin-IX and acts as a monomer homodimer.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 4) | Possible functional associations between ATMIN and PROS1 are unclear based on the provided information, as their interaction in the BioGRID network is indirect and their known functions do not overlap significantly. ATMIN is a transcription factor involved in DNA damage response and ATM regulation, while PROS1 is a plasma protein with anticoagulant activity.|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between MNAT1 and SLC35A1 is that they are involved in transcriptional regulation by RNA polymerase II. MNAT1 is a component of the CDK-activating kinase (CAK) complex, which activates RNA polymerase II by phosphorylating its C-terminal domain (CTD), while SLC35A1 transports CMP-sialic acid, a molecule involved in RNA processing, between the cytosol and Golgi apparatus.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 123.9</br>**STRING score:** 284</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | DDX42 and SLC35A1 do not have a clear functional association based on the provided context and their known biological roles. DDX42 is an ATP-dependent RNA helicase involved in RNA splicing, while SLC35A1 is a transporter protein involved in nucleotide sugar transport.|
| [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.0</br>**STRING score:** 345</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations: MAP2K5 and PTPN1 interact in the protein interaction network, suggesting a potential role in modulating cellular signaling pathways. MAP2K5 may play a role in protecting cells from stress-induced apoptosis and cardiac development, while PTPN1 regulates the endoplasmic reticulum unfolded protein response and cell signaling cascades.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SIRT3](https://www.ncbi.nlm.nih.gov/gene?term=SIRT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.9</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: Both MTX2 and SIRT3 are involved in mitochondrial processes. MTX2 is involved in mitochondrial protein transport while SIRT3 is a NAD-dependent deacetylase that regulates cellular energy metabolism and promotes mtDNA transcription. Given their related mitochondrial roles, it is possible that these two genes may interact in regulatory pathways involving mitochondrial functions. However, there is currently no known direct interaction between MTX2 and SIRT3 based on the BioGRID protein interaction network.|
| [LRRC16A](https://www.ncbi.nlm.nih.gov/gene?term=LRRC16A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MOK](https://www.ncbi.nlm.nih.gov/gene?term=MOK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.1</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | LRRC16A and MOK do not appear to have a direct functional association. LRRC16A is involved in actin polymerization and cell migration, while MOK is able to phosphorylate substrates and negatively regulates cilium length. The shortest path in their interaction network involves HUWE1 as an intermediate protein.|
| [CDKN1A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | Two possible functional associations between CDKN1A and KIF20A are their roles in cell cycle regulation and cytokinesis, respectively. CDKN1A inhibits cell cycle progression and DNA synthesis, while KIF20A is required for CPC-mediated cytokinesis and may be involved in Golgi membrane transport along microtubules. The shortest path between these genes in the BioGRID protein interaction network includes TP53 and PARP1, both known to be involved in DNA damage response and cell cycle regulation.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.7</br>**STRING score:** 405</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no known functional associations between CYCS and EIF4EBP1 in specific biological processes. CYCS plays a role in apoptosis, while EIF4EBP1 regulates translation initiation.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.6</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between CDK2 and EML3 involve their roles in regulating cell cycle progression, cell proliferation, and DNA repair. CDK2 functions as a serine/threonine-protein kinase that is essential for meiosis and modulates cell cycle progression, while EML3 regulates mitotic spindle assembly and chromosome separation during cell division. Both genes are involved in regulating DNA replication and repair, and may interact through shared protein interaction partners, such as TUBG1.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK2](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.9</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between CRKL and MAPKAPK2 include regulation of cytokine production, cell migration, chromatin remodeling, and DNA damage response. However, no direct interactions have been reported between these two genes in the literature.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.7</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | CDK2 and MAPKAPK5 are functionally associated through a protein interaction network involving TP53, HSPD1, FASN, HMGB1, and RPL13. CDK2 is a serine/threonine-protein kinase involved in controlling the cell cycle, while MAPKAPK5 is a tumor suppressor serine/threonine-protein kinase involved in mTORC1 signaling and post-transcriptional regulation.|
| [CAT](https://www.ncbi.nlm.nih.gov/gene?term=CAT[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA1A](https://www.ncbi.nlm.nih.gov/gene?term=HSPA1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 109.2</br>**STRING score:** 542</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | CAT and HSPA1A have a short interaction through multiple proteins, such as ABL2, ENO1 and PEX5. Although there is no direct biological functional association known between CAT and HSPA1A, these genes are involved in anti-oxidant and stress response pathways, respectively.|
| [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.1</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: MAPK13 is a serine/threonine kinase involved in the regulation of cellular responses. PTPRF is a protein tyrosine phosphatase with enzymatic activity. They interact indirectly through KRAS and may participate in the regulation of insulin secretion and epidermal keratinocyte differentiation and apoptosis. Further experimentation is necessary to determine a more specific functional relationship between these genes.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 107.5</br>**STRING score:** 157</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between LIG1 and PAN2 include regulation of mRNA turnover and stability, potentially through LIG1's interactions with PCNA and PAN2's involvement in deadenylation. Both genes are involved in DNA repair and metabolism, but their specific functions do not directly overlap.|
| [BNIP3L](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 107.3</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 83 (min len 4) | Possible functional associations: Based on the BioGRID interaction network and context, BNIP3L and XBP1 may be involved in the regulation of apoptosis and the unfolded protein response in the endoplasmic reticulum. BNIP3L induces apoptosis and is involved in mitochondrial quality control, while XBP1 is a stress-inducible transcriptional activator that up-regulates target genes encoding ER chaperones and ER-associated degradation components to maintain ER homeostasis under stress.|
