| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RNH1](https://www.ncbi.nlm.nih.gov/gene?term=RNH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 137.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between RNH1 and SYK could be related to innate and adaptive immunity, particularly in B-cell and T-cell receptor signaling and the immune response to fungal, bacterial, and viral pathogens. They may also play a role in inflammatory processes and platelet activation. The shortest path between the two genes in the BioGRID protein interaction network involves several proteins associated with immune responses, suggesting a potential interaction in this context.|
| [CLTB](https://www.ncbi.nlm.nih.gov/gene?term=CLTB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FCHO1](https://www.ncbi.nlm.nih.gov/gene?term=FCHO1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.1</br>**STRING score:** 852</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | CLTB and FCHO1 are involved in the process of clathrin-mediated endocytosis, with FCHO1 recruiting proteins for the formation of functional clathrin-coated pits and CLTB being a major protein of these pits. Based on their protein interactions, it is likely that FCHO1 may interact with CLTB through adaptor protein complexes such as AP-2 for the regulation of this process.|
| [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between RBM34 and SLC25A46 could be in the regulation of mitochondrial organization and cristae biogenesis, as both genes are involved in these processes according to the provided context. RBM34 may interact with EGFR and promote its dimerization, while SLC25A46 may associate with the EMC and regulate mitochondrial lipid homeostasis.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 132.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no clear functional association between ECH1 and PTPRF based on their known biological functions and protein interactions. ECH1 plays a role in isomerization of fatty acid metabolites, while PTPRF is a phosphatase involved in cellular signaling and interacts with other proteins such as GRIP1 and INSR.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOX4](https://www.ncbi.nlm.nih.gov/gene?term=SOX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.2</br>**STRING score:** 235</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association: Both NR3C1 and SOX4 are transcriptional regulators that bind to DNA motifs and interact with other proteins to regulate gene expression. NR3C1 has been shown to interact with the SMARCA4 complex, which is involved in chromatin remodeling, and with coactivators such as EP300, which is a histone acetyltransferase. SOX4, on the other hand, has been implicated in the regulation of IL17A production in T cells and skeletal myoblast differentiation. Possible crosstalk between these two genes may be related to their roles in transcriptional regulation, but further investigation would be needed to determine the specifics.|
| [CLTB](https://www.ncbi.nlm.nih.gov/gene?term=CLTB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIPA](https://www.ncbi.nlm.nih.gov/gene?term=LIPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CLTB and LIPA could involve the transport of lipoproteins inside cells, as CLTB is involved in endocytosis and LIPA catalyzes the deacylation of triacylglycerol and core lipids of endocytosed low-density lipoproteins. The BioGRID protein interaction network indicates that CLTB and LIPA might interact via IGF2R and TRIM67.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC27A3](https://www.ncbi.nlm.nih.gov/gene?term=SLC27A3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 129.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between IDE and SLC27A3 may involve their roles in lipid metabolism. SLC27A3 is an acyl-CoA ligase involved in the formation of fatty acyl-CoA, while IDE has been shown to regulate lipid metabolism. Additionally, both genes have been reported to interact with viral proteins, suggesting possible connections to immune response and infection.|
| [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPOX](https://www.ncbi.nlm.nih.gov/gene?term=PPOX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FOXO3 and PPOX might be related to oxidative stress response and heme biosynthesis, respectively. However, without further experimental evidence, it is difficult to determine if and how these two processes might intersect.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 4) | Possible functional associations between ATMIN and PROS1 based on the BioGRID protein interaction network and known biology: ATMIN may play a role in regulating the activity of ATM, which is involved in DNA damage response, and interacts with PROS1, an anticoagulant plasma protein. It is unclear if there is a direct biological link between these two genes.|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 128.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no clear functional associations between MNAT1 and SLC35A1 based on their known processes. MNAT1 stabilizes and activates cyclin-associated kinases and RNA polymerase II, while SLC35A1 transports CMP-sialic acid into the Golgi apparatus.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35A1](https://www.ncbi.nlm.nih.gov/gene?term=SLC35A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 123.9</br>**STRING score:** 284</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no apparent functional associations between DDX42 and SLC35A1 based on their known biological roles and the provided context. DDX42 is involved in RNA helicase activity and splicing factor complex formation, while SLC35A1 transports CMP-sialic acid. The shortest path between them in the BioGRID protein interaction network may be coincidental.|
| [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.0</br>**STRING score:** 345</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association: MAP2K5 and PTPN1 interact in a protein network. MAP2K5 is involved in MAPK signaling and stress response, while PTPN1 is a phosphatase involved in regulating protein kinases and cell signaling. These genes may play a role in modulating cell reorganization and response to stress.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SIRT3](https://www.ncbi.nlm.nih.gov/gene?term=SIRT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.9</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between MTX2 and SIRT3 in TCGA-CESC cancer type may involve mitochondrial transport, organization, and metabolism. MTX2 is involved in mitochondrial protein transport and interacts with SAM and MICOS complexes needed for proper mitochondrial organization. SIRT3 is a deacetylase that regulates mitochondrial protein activity, oxidative phosphorylation, and energy metabolism.|
| [LRRC16A](https://www.ncbi.nlm.nih.gov/gene?term=LRRC16A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MOK](https://www.ncbi.nlm.nih.gov/gene?term=MOK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.1</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | LRRC16A and MOK have no known functional associations based on the provided information. LRRC16A plays a role in regulating actin polymerization and cell migration, while MOK negatively regulates cilium length in a signaling-dependent manner and interacts with bacterial proteins.|
| [CDKN1A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | Possible functional associations: CDKN1A inhibits cell cycle progression and is involved in DNA damage response while KIF20A is a mitotic kinesin required for cytokinesis. These two genes may be functionally associated in regulating cell cycle progression and division. The shortest path between them in the BioGRID protein interaction network suggests that TP53, PARP1, GNB2L1, DDB1, and HDAC1 may also be involved in this functional association.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.7</br>**STRING score:** 405</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between CYCS and EIF4EBP1 based on the provided information. CYCS is involved in apoptosis regulation via the mitochondrial pathway, while EIF4EBP1 regulates translation initiation by interacting with eIF4E and mTORC1 signaling.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.6</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | There is no clear functional association between CDK2 and EML3 based on their roles and interactions. CDK2 is a serine/threonine protein kinase involved in cell cycle control, while EML3 regulates mitotic spindle assembly and chromosome separation. However, they do have a common interaction partner, TUBG1, which is involved in microtubule formation.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK2](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.9</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between CRKL and MAPKAPK2 could be related to cell signaling and stress response pathways. CRKL is involved in mediating intracellular signals and interacts with tyrosine-phosphorylated proteins, while MAPKAPK2 is a stress-activated kinase involved in cytokine production, cell cycle control, and chromatin remodeling. Their shortest path in the BioGRID protein interaction network involves proteins related to cell adhesion and movement.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.7</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | CDK2 and MAPKAPK5 may have a functional association in the regulation of the cell cycle; CDK2 is involved in the control of the cell cycle and DNA synthesis, while MAPKAPK5 is a tumor suppressor serine/threonine-protein kinase that mediates post-transcriptional regulation of MYC. The protein interaction network suggests that there may be multiple pathways connecting CDK2 and MAPKAPK5.|
| [CAT](https://www.ncbi.nlm.nih.gov/gene?term=CAT[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA1A](https://www.ncbi.nlm.nih.gov/gene?term=HSPA1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 109.2</br>**STRING score:** 542</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional associations between CAT and HSPA1A include their interactions in protein networks involved in detoxifying xenobiotics and endogenous organic acids, as well as their involvement in microbial infection and protein complexes. They may also have roles in regulating transcriptional activity during the heat shock response and stabilizing other proteins in the cell.|
| [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.1</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between MAPK13 and PTPRF include regulation of insulin secretion in pancreatic beta cells through down-regulation of PRKD1 by MAPK13 and interaction between PTPRF and INSR, a receptor involved in insulin signaling. Both genes also play a role in cytoskeletal remodeling and may be involved in epidermal keratinocyte differentiation, apoptosis, and skin tumor development.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 107.5</br>**STRING score:** 157</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | There are no known functional associations between LIG1 and PAN2. LIG1 acts as a negative regulator of receptor tyrosine kinase signaling and interacts with EGFR/ERBB1, ERBB2, ERBB3, and ERBB4. PAN2 functions as a retinol dehydrogenase and has been shown to interact with human cytomegalovirus (HHV-5) immediate early protein IE1.|
| [BNIP3L](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 107.3</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 83 (min len 4) | Possible functional association: Both BNIP3L and XBP1 are involved in cellular stress response pathways, with BNIP3L regulating mitochondrial quality control and apoptosis, and XBP1 inducing the unfolded protein response and contributing to cell proliferation and survival. It is possible that their pathways may intersect in regulating cellular homeostasis, particularly in response to stressors such as hypoxia or nutrient deprivation. However, further investigation is needed to determine the specific nature of their interaction.|
