| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 207.3</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 3) | Possible functional associations: CD44 is involved in cell signaling, adhesion, and migration, while CLTC plays a role in clathrin-mediated endocytosis and mitotic spindle organization. They may interact in the regulation of cell adhesion and migration through the formation of the TACC3/ch-TOG/clathrin complex.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCMT1](https://www.ncbi.nlm.nih.gov/gene?term=PCMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 201.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is no clear functional association between GOLT1B and PCMT1 based on their known biological roles and the provided context. GOLT1B is involved in ER-Golgi transport while PCMT1 repairs damaged proteins via methyl esterification. However, it is possible that PCMT1 may catalyze modifications on GOLT1B or its interacting proteins. Further investigation is needed to determine any potential functional connections.|
| [NR2F6](https://www.ncbi.nlm.nih.gov/gene?term=NR2F6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 183.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 504 (min len 4) | There is no clear functional association between NR2F6 and REEP5 based on the limited information provided. NR2F6 is a transcription factor involved in the regulation of hormonal responses and immune tolerance, while REEP5 plays a role in the organization and function of the sarcoplasmic reticulum in cardiomyocytes. The protein interaction network suggests that the two genes may interact indirectly through other proteins.|
| [FYN](https://www.ncbi.nlm.nih.gov/gene?term=FYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GDPD5](https://www.ncbi.nlm.nih.gov/gene?term=GDPD5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 178.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FYN and GDPD5 are unknown.   FYN is a non-receptor tyrosine kinase involved in regulation of cell growth, survival, adhesion, and immune response, while GDPD5 is involved in glycerophosphodiester cleavage and regulation of neurite formation. These genes seem to operate in different biological pathways and there is no known functional relationship between them.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between GOLT1B and PSMD10 include involvement in the regulation of the proteasome and apoptosis as well as interaction with tumor suppressors such as RB1 and p53/TP53. These genes may also be involved in the targeting and degradation of proteins involved in cell cycle regulation and oncogenic signaling.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.9</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | CD44 and GRN are not directly functionally associated in any known biological process, but they both play important roles in cell adhesion, migration, and inflammation. CD44 is a cell-surface receptor that interacts with extracellular matrix components while GRN stabilizes CTSD and facilitates its lysosomal delivery, both with potential implications in lysosomal function and proteolysis.|
| [CASC3](https://www.ncbi.nlm.nih.gov/gene?term=CASC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTF2](https://www.ncbi.nlm.nih.gov/gene?term=MTF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | CASC3 and MTF2 may have a functional association in the regulation of gene expression. CASC3 is involved in pre-mRNA splicing and mRNA metabolism, while MTF2 is a polycomb group protein involved in the regulation of transcriptional networks and histone methylation. The shortest path between these genes in the BioGRID protein interaction network involves multiple proteins, suggesting complex interactions between them.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF395](https://www.ncbi.nlm.nih.gov/gene?term=ZNF395[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no clear functional associations between ICAM1 and ZNF395, as their interactions seem to be limited to a single protein interaction pathway. ICAM1 is involved in immune response, while ZNF395 plays a role in papillomavirus gene transcription.|
| [KAT6B](https://www.ncbi.nlm.nih.gov/gene?term=KAT6B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: KAT6B is a histone acetyltransferase involved in both positive and negative regulation of transcription. NVL is involved in telomerase activity and pre-rRNA processing. There is no known direct functional association between these two genes based on their roles in different cellular processes.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 361</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations: based on the protein interaction network and context, it is possible that GOLT1B and NOLC1 may play a role in ribosomal processing and modification, nucleologenesis, and neural crest specification in cells.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YTHDF2](https://www.ncbi.nlm.nih.gov/gene?term=YTHDF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 301</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between MYC and YTHDF2 could be in regulating gene expression. MYC is a transcription factor that activates the transcription of growth-related genes and binds to VEGFA promoter. YTHDF2 regulates RNA stability and acts by binding to methylated viral RNAs.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.0</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations between CDK2 and USP1: CDK2 phosphorylates USP1, which negatively regulates DNA damage repair by deubiquitinating monoubiquitinated FANCD2 and PCNA-mediated translesion synthesis. CDK2 is also involved in DNA replication, cell cycle control, and centrosome duplication, while USP1 plays a role in DNA damage repair. They may function together in balancing cellular proliferation, cell death, and DNA repair.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSIP1](https://www.ncbi.nlm.nih.gov/gene?term=PSIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 162.6</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations between NVL and PSIP1 are unclear based on the provided context as they do not seem to be directly involved in the same biological processes. NVL is involved in pre-rRNA processing and ribosomal subunit biogenesis while PSIP1 is involved in transcriptional coactivation and lens epithelial cell gene regulation. However, both genes have been shown to interact with proteins involved in RNA processing and may have indirect functional associations.|
| [ILK](https://www.ncbi.nlm.nih.gov/gene?term=ILK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.8</br>**STRING score:** 932</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between ILK and SHC1. ILK is a receptor-proximal protein kinase regulating integrin-mediated signal transduction, whereas SHC1 is a signaling adapter that couples activated growth factor receptors to signaling pathways. These genes are involved in different biological processes, which suggest they may not directly interact.|
| [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.7</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | Possible functional association between CRK and STAT3 is their interaction in the regulation of cell migration, adhesion, and signaling transduction pathway. CRK is involved in mediating attachment-induced MAPK8 activation, membrane ruffling and cell migration in a Rac-dependent manner, while STAT3 mediates cellular responses to growth factors, activates key genes for G1 to S phase progression, and regulates cell cycle and inflammation.|
| [ARL4C](https://www.ncbi.nlm.nih.gov/gene?term=ARL4C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.4</br>**STRING score:** 153</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 71 (min len 3) | ARL4C and CDC42 are both small GTPase proteins involved in vesicular transport, cell polarization, and actin cytoskeleton organization. They interact indirectly through protein partners in the BioGRID interaction network, suggesting they may have roles in similar cellular processes. However, without further experimental evidence, the functional association between the two genes in specific biological terms is unclear.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.4</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 124 (min len 3) | Possible functional associations: Both CLTC and PSMD2 are involved in protein processing and degradation. CLTC is a component of the TACC3/ch-TOG/clathrin complex involved in stabilizing kinetochore fibers during mitosis, while PSMD2 is a subunit of the 19S proteasome regulatory particle complex involved in the degradation of ubiquitinated proteins. There is no known direct biological interaction between these two genes.|
| [PEX11A](https://www.ncbi.nlm.nih.gov/gene?term=PEX11A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB3](https://www.ncbi.nlm.nih.gov/gene?term=TRIB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.3</br>**STRING score:** 338</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 4) | Possible functional association: PEX11A is involved in peroxisomal proliferation, while TRIB3 is a regulator of the integrated stress response and can inhibit insulin signaling. Both genes have interactions with GSK3B. One possible biological process where these genes may be functionally associated is in regulating cellular stress responses and metabolism through peroxisomal and insulin signaling pathways.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.2</br>**STRING score:** 993</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CD44 helps cells sense and respond to changes in the tissue microenvironment, and interacts with the RhoGTPases RAC1 and RHOA for signaling pathways promoting calcium mobilization and actin-mediated cytoskeleton reorganization, which are essential for cell migration and adhesion. RHOA, on the other hand, serves as a target for microbial infection and interacts with human respiratory syncytial virus protein F for virus-induced syncytium formation. While there is no direct evidence of functional association between CD44 and RHOA, their interaction in the BioGRID protein interaction network suggests a possible role in regulating cell migration and adhesion.|
| [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC5A6](https://www.ncbi.nlm.nih.gov/gene?term=SLC5A6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 156.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no obvious functional associations between SLC25A46 and SLC5A6 based on their known roles in mitochondrial organization and vitamin transport, respectively. However, they may interact indirectly through their involvement in mitochondrial lipid homeostasis and fission.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.1</br>**STRING score:** 422</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 3) | CDC42 and ICAM1 are functionally associated in cancer. CDC42 regulates cell migration and organization of the F-actin cytoskeleton while ICAM1 is involved in cell adhesion and leukocyte extravasation. The shortest path between these genes in the BioGRID protein interaction network suggests a possible direct interaction through several other proteins.|
| [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 936</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 90 (min len 3) | SMC3 and SMC4 have a shortest path of protein interaction via cohesin complex and condensin complex. SMC3 is involved in chromosome cohesion, spindle pole assembly, and DNA repair, while SMC4 is involved in condensation of chromatin during mitosis. These two genes may function together in regulating chromosome condensation and segregation during cell division.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.7</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: CASP3 is involved in apoptosis and cleavage of various proteins, including USP1's interaction partner PCNA. USP1 is a negative regulator of DNA damage repair and is involved in PCNA-mediated translesion synthesis. There may be a role for the interaction of these genes in regulating DNA damage repair and apoptosis.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.1</br>**STRING score:** 880</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 138 (min len 3) | Possible functional associations between CCT8 and PSMD2 in TCGA-KIRC may include regulation of protein folding and degradation, as both genes are involved in these processes. CCT8 is a component of the TRiC chaperone complex that assists in protein folding, while PSMD2 is a subunit of the 19S proteasome regulatory particle complex involved in protein degradation.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.8</br>**STRING score:** 937</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | CCT8 and PSMG1 are functionally associated through a protein interaction network and both play roles in protein folding. CCT8 is involved in chaperone complex-mediated folding of proteins and regulatory processes, while PSMG1 promotes assembly of the 20S proteasome and interactions with proteasome subunits.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.3</br>**STRING score:** 672</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | MYC is a transcription factor involved in the activation of growth-related genes and angiogenesis, as well as somatic reprogramming and splicing regulation. SMARCC1 is part of chromatin remodeling complexes that alter DNA-nucleosome topology, playing a role in neural development and regulation of dendrite growth. Based on their roles, MYC and SMARCC1 could potentially interact to regulate gene expression in cancer development.|
| [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.1</br>**STRING score:** 903</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 58 (min len 3) | Possible functional associations between NCAPD2 and SMC3 involve their roles in chromosome organization and segregation during cell division. NCAPD2 is a subunit of the condensin complex which plays a role in chromosome condensation, while SMC3 is a subunit of the cohesin complex which is involved in maintaining sister chromatid cohesion. These two complexes work together to ensure proper chromosome segregation during cell division.|
| [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.7</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 46 (min len 3) | Possible functional associations between RHOA and SMC4 include their involvement in cell division and chromosome condensation. RHOA also has roles in microbial infection and virus-induced syncytium formation, while SMC4 is a central component of the condensin complex. There is currently insufficient information to suggest a direct biological interaction between these two genes.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.0</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | Possible functional association: CLTC plays a role in autophagosome formation and also interacts with NPC1, which acts as an endosomal entry receptor for ebolavirus. This suggests a potential involvement of CLTC in viral entry pathways, possibly through interactions with NPC1.|
| [ARHGEF2](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.5</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their roles in cell cycle regulation and cancer, there may be a functional association between ARHGEF2 and NOTCH1 in tumor progression and invasion. However, there is no direct biological evidence for interaction or synergistic action between these two genes.|
| [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKAP](https://www.ncbi.nlm.nih.gov/gene?term=IKBKAP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.3</br>**STRING score:** 456</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | Possible functional associations between HNRNPA2B1 and IKBKAP include regulation of gene expression through RNA splicing (HNRNPA2B1) and tRNA modification (IKBKAP). The shortest path between these two genes in the BioGRID protein interaction network suggests that they may interact in the context of protein degradation and/or RNA transport processes.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.6</br>**STRING score:** 196</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 3) | DDX42 and HNRNPA2B1 both participate in RNA processing and splicing. Their interaction may be involved in regulating RNA transport and localization in the cell, as well as in the survival of cells counteracting apoptosis-stimulating activity of TP53BP2.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.5</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between NVL and SMC3 include their involvement in nucleolar and ribosomal biogenesis and regulation, as well as their participation in chromosome cohesion during cell cycle and DNA repair. There is no clear evidence or known interaction between these two genes in specific biological processes.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.3</br>**STRING score:** 975</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | RFC2 and RPA2 have a functional association in DNA replication and DNA damage response. RFC2 is involved in the elongation of primed DNA templates with activator proteins, while RPA2 stabilizes single-stranded DNA intermediates and recruits repair factors to sites of DNA damage.|
| [NOS3](https://www.ncbi.nlm.nih.gov/gene?term=NOS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.7</br>**STRING score:** 392</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between NOS3 and SHC1 is in the regulation of endothelial cell migration and sprouting angiogenesis. NOS3 plays a role in nitric oxide synthesis pathway, while SHC1 participates in signaling downstream of angiopoietin receptor TEK/TIE2.|
| [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.6</br>**STRING score:** 994</br>**BioGRID count:** 15</br>**BioGRID shortest paths:** 1 (min len 2) | PSME1 and PSME2 form a heterodimeric ring required for efficient antigen processing and immunoproteasome assembly. The PA28 activator complex enhances peptide generation by altering proteasome cleavage pattern. As PSME1 can form homoheptamers, there may be potential for these proteins to interact in heptameric or hexameric complexes with other immunoproteasome proteins.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.5</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between GOLT1B and TXNDC9 are in vesicle transport and protein folding, respectively. GOLT1B may aid in the fusion of ER-derived transport vesicles with the Golgi complex, while TXNDC9 negatively impacts protein folding by diminishing the ATPase activity of the chaperonin TCP1 complex. The two genes may play a role in the proper folding and transport of proteins within the cell.|
| [GFPT1](https://www.ncbi.nlm.nih.gov/gene?term=GFPT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.4</br>**STRING score:** 284</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no clear functional association between GFPT1 and GOLT1B based on provided context and knowledge of biology. GFPT1 regulates the flux of glucose into the hexosamine pathway and may have a role in regulating circadian expression of clock genes, while GOLT1B may be involved in fusion of ER-derived transport vesicles with the Golgi complex.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.0</br>**STRING score:** 820</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between ICAM1 and NRAS include their involvement in cell adhesion and immune responses. ICAM1 plays a role in cell-cell adhesion and immune system activation, while NRAS is a member of the Ras family of GTPases involved in cell signaling pathways, including immune responses. Their interaction in the shortest path suggests a potential role in Ras-mediated signaling pathways.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIT1](https://www.ncbi.nlm.nih.gov/gene?term=NIT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.7</br>**STRING score:** 267</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There may not be a direct functional association between ECH1 and NIT1, as they have different roles. ECH1 is involved in fatty acid metabolism, while NIT1 is involved in the repair of metabolites and has tumor suppressor properties. However, both genes play roles in cellular growth and apoptosis, suggesting a potential indirect functional association.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.5</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association between CDK2 and TGFBR2 in TCGA-KIRC is that CDK2 phosphorylates TGFBR2, leading to the regulation of TGF-beta signaling pathway that is involved in cell proliferation and differentiation. TGFBR2 also plays a role in blocking activation of SMAD proteins by TGF-beta ligands.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.9</br>**STRING score:** 626</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association: CDKN1B inhibits the kinase activity of CDK2 bound to cyclin A, while CASP3 cleaves and activates caspase-6, -7 and -9 as well as XRCC4 and phospholipid scramblase proteins. However, it is unclear how these two genes may functionally interact within the context of TCGA-KIRC.|
