| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 207.3</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 3) | Possible functional association: CD44 is a cell-surface receptor involved in cell adhesion and migration, while Clathrin (CLTC) is a major protein of the coat of coated pits and vesicles. They may potentially interact in endocytosis, as CD44 has been shown to interact with EGFR, which can be internalized via clathrin-coated pits and vesicles.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCMT1](https://www.ncbi.nlm.nih.gov/gene?term=PCMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 201.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between GOLT1B and PCMT1 could be related to the regulation of protein repair and/or vesicle trafficking within the Golgi complex or ER-derived transport. GOLT1B may play a role in vesicle fusion, while PCMT1 initiates repair of damaged proteins through catalyzing methyl esterification of specific residues. However, further experimental evidence is needed to determine their specific roles in any potential functional associations.|
| [NR2F6](https://www.ncbi.nlm.nih.gov/gene?term=NR2F6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 183.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 504 (min len 4) | There is no clear functional association between NR2F6 and REEP5 based on available context and their known biological functions. NR2F6 is a transcription factor involved in transcriptional repression and hormonal response modulation, while REEP5 plays an essential role in heart function and development by regulating the organization and function of the sarcoplasmic reticulum in cardiomyocytes.|
| [FYN](https://www.ncbi.nlm.nih.gov/gene?term=FYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GDPD5](https://www.ncbi.nlm.nih.gov/gene?term=GDPD5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 178.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FYN and GDPD5 could be in cell adhesion and motility processes, since FYN regulates these processes by phosphorylating CTNNB1 and CTNND1, while GDPD5 promotes neurite formation and may contribute to osmotic regulation of glycerophosphocholine. The BioGRID protein interaction network shows a shortest path between FYN and GDPD5 through HSPA8 and SIAH1, but the specific context of this interaction is unclear.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between GOLT1B and PSMD10 is their role in protein degradation through the proteasome pathway, with PSMD10 as a regulatory subunit and GOLT1B possibly involved in vesicle fusion. There is no known direct biological relationship between the two genes.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.9</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between CD44 and GRN could involve cell migration, cell adhesion, and response to changes in the tissue microenvironment. CD44 is a cell-surface receptor that plays a role in these processes, while GRN stabilizes CTSD and interacts with it to maintain its aspartic-type peptidase activity, which may be relevant for cell migration. However, further experiment would be needed to confirm any functional associations between these two genes.|
| [CASC3](https://www.ncbi.nlm.nih.gov/gene?term=CASC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTF2](https://www.ncbi.nlm.nih.gov/gene?term=MTF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CASC3 and MTF2: CASC3 is involved in pre-mRNA splicing and RNA metabolism, while MTF2 regulates gene expression through histone modifications. They may indirectly interact through their roles in RNA processing and chromatin modification.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF395](https://www.ncbi.nlm.nih.gov/gene?term=ZNF395[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between ICAM1 and ZNF395 based on the given context. ICAM1 plays a role in viral infection and interaction with capsid proteins, while ZNF395 is involved in papillomavirus gene transcription and interacts with a repression-mediating binding site.|
| [KAT6B](https://www.ncbi.nlm.nih.gov/gene?term=KAT6B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | One possible functional association between KAT6B and NVL is in the regulation of gene expression at the level of chromatin and RNA processing. KAT6B is a histone acetyltransferase involved in both positive and negative transcriptional regulation, while NVL participates in telomerase holoenzyme assembly and pre-rRNA processing, spatiotemporally regulating 60S ribosomal subunit biogenesis in the nucleolus. The shortest path in the BioGRID protein interaction network between KAT6B and NVL involves two intermediates, UBE2I and ESR2, suggesting a potential regulatory connection between the two genes.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 361</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between GOLT1B and NOLC1 are in the regulation of RNA polymerase I and nucleologenesis, as NOLC1 connects RNA polymerase I with enzymes responsible for ribosomal processing and modification and may play a role in maintaining the fundamental structure of the nucleolus, while GOLT1B is involved in the fusion of ER-derived transport vesicles with the Golgi complex. However, more research is needed to determine if there are any direct interactions or functional associations between the two genes in specific biological processes.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YTHDF2](https://www.ncbi.nlm.nih.gov/gene?term=YTHDF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 301</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between MYC and YTHDF2 may involve regulation of transcription and mRNA stability, as MYC is a transcription factor and YTHDF2 interacts with RNA-binding proteins. MYC may also play a role in promoting angiogenesis, which could potentially be affected by YTHDF2's role in viral gene expression and virion production.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.0</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations between CDK2 and USP1 include their involvement in DNA damage repair and regulation of cell cycle progression through phosphorylation and deubiquitination processes. CDK2 promotes cell cycle progression while USP1 is a negative regulator of DNA damage repair. They interact indirectly through other proteins, such as PCNA.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSIP1](https://www.ncbi.nlm.nih.gov/gene?term=PSIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 162.6</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations between NVL and PSIP1 are unclear based on the provided context. NVL is involved in telomerase activity and pre-rRNA processing, while PSIP1 is a transcriptional coactivator and a cofactor for lentiviral integration. They interact indirectly through intermediate proteins in a protein interaction network.|
| [ILK](https://www.ncbi.nlm.nih.gov/gene?term=ILK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.8</br>**STRING score:** 932</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between ILK and SHC1 based on the provided context and known biology. ILK is involved in integrin-mediated signal transduction and cell motility, while SHC1 acts as a signaling adapter for growth factor receptors and is involved in regulation of endothelial cell migration and sprouting angiogenesis.|
| [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.7</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | CRK and STAT3 have a direct protein-protein interaction via several intermediate proteins. CRK plays a role in cell adhesion, spreading, and migration, while STAT3 is involved in mediating cellular responses to growth factors and regulating inflammatory and cell cycle pathways. It is possible that the interaction between CRK and STAT3 could affect these biological processes.|
| [ARL4C](https://www.ncbi.nlm.nih.gov/gene?term=ARL4C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.4</br>**STRING score:** 153</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 71 (min len 3) | There is no clear functional association between ARL4C and CDC42 based on the provided information. ARL4C is involved in intracellular transport and cholesterol secretion, while CDC42 regulates cell polarity, migration, and phagocytosis. The shortest path between these two proteins in the BioGRID interaction network involves intermediate proteins not known to be functionally related to ARL4C or CDC42.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.4</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 124 (min len 3) | Possible functional associations between CLTC and PSMD2 are their roles in autophagosome formation and protein degradation through the proteasome pathway, respectively. They interact indirectly through other proteins in the protein interaction network, possibly contributing to the regulation of protein turnover and maintenance of cellular homeostasis.|
| [PEX11A](https://www.ncbi.nlm.nih.gov/gene?term=PEX11A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB3](https://www.ncbi.nlm.nih.gov/gene?term=TRIB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.3</br>**STRING score:** 338</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 4) | PEX11A is involved in peroxisomal membrane dynamics and division, while TRIB3 is a regulator of the integrated stress response and a negative feedback inhibitor of the ATF4-dependent transcription. There is no known direct functional association between these two genes in specific biological terms based on the provided context.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.2</br>**STRING score:** 993</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CD44 and RHOA interact with each other. CD44 plays a role in cell-cell interactions, adhesion, and migration, while RHOA is involved in cytoskeleton reorganization and is targeted by some microorganisms. There could be a relationship between these genes in regulating cell migration and adhesion processes.|
| [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC5A6](https://www.ncbi.nlm.nih.gov/gene?term=SLC5A6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 156.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between SLC25A46 and SLC5A6 based on the limited context provided. SLC25A46 is involved in mitochondrial organization and lipid homeostasis, while SLC5A6 is a sodium-dependent multivitamin transporter.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.1</br>**STRING score:** 422</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 3) | Possible functional associations: CDC42 regulates cell migration and filopodia formation, while ICAM1 plays a role in cell adhesion and interacts with viral proteins. There is no direct evidence for functional associations between CDC42 and ICAM1, but they are connected in a protein interaction network through several intermediate proteins involved in cell polarization processes, structural plasticity, and phagocytosis.|
| [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 936</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 90 (min len 3) | SMC3 and SMC4 are functionally associated as central components of complexes involved in chromosome cohesion, movement, and modification. SMC3 and SMC4 heterodimerize in cohesin and condensin complexes and their shortest path in BioGRID protein interaction network involves shared interactions with RAD21, suggesting their roles in DNA replication, repair, and cell division are interconnected.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.7</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CASP3 and USP1 may involve DNA damage repair and apoptosis regulation. CASP3 is involved in the execution phase of apoptosis and cleavage of many proteins, while USP1 is a negative regulator of DNA damage repair by deubiquitinating monoubiquitinated FANCD2 and PCNA. There may also be an interaction between these two genes through the common interaction partner TRAF3.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.1</br>**STRING score:** 880</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 138 (min len 3) | Possible functional association between CCT8 and PSMD2 is their involvement in protein folding and degradation through the TRiC chaperone complex and the 19S proteasome regulatory particle complex, respectively. It is possible that CCT8 and PSMD2 work together to ensure proper folding and degradation of specific proteins, and their interaction through the BioGRID network may suggest a potential role for both genes in regulating ciliogenesis and telomere maintenance.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.8</br>**STRING score:** 937</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between CCT8 and PSMG1 include their roles in protein folding, with CCT8 being a component of the chaperonin-containing T-complex (TRiC) and PSMG1 promoting assembly of the 20S proteasome. They may also be involved in ciliogenesis and telomere maintenance. Based on the BioGRID protein interaction network, they may interact through other proteins, such as PSMA3 or CUL1, in assembly of the proteasome.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.3</br>**STRING score:** 672</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | MYC and SMARCC1 are both involved in transcriptional regulation through chromatin remodeling. Possible functional associations between these two genes include MYC regulating the expression of SMARCC1 or SMARCC1 influencing the activity of MYC through SWI/SNF complexes.|
| [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.1</br>**STRING score:** 903</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 58 (min len 3) | Possible functional association between NCAPD2 and SMC3 is their involvement in chromosome organization and segregation during mitosis and meiosis. NCAPD2 is a regulatory subunit of the condensin complex, which plays a role in mitotic chromosome condensation, while SMC3 is a central component of cohesin, which is required for chromosome cohesion during cell division. The BioGRID protein interaction network shows multiple interactions between NCAPD2 and SMC3 through other proteins, suggesting a possible functional relationship.|
| [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.7</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 46 (min len 3) | Possible functional associations between RHOA and SMC4 include their interaction in the formation of the condensin complex during chromosome condensation, regulation of gene expression via the NFKBIA pathway, and involvement in microbial infection. RHOA is also involved in facilitating virus-induced syncytium formation.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.0</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | Possible functional associations between CLTC and NPC1 are involvement in endocytosis and/or autophagy. CLTC is a component of the clathrin coat involved in endocytosis, and NPC1 has been shown to be involved in autophagy regulation. The protein interaction network suggests a potential physical association between these two proteins.|
| [ARHGEF2](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.5</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear indication of a functional association between ARHGEF2 and NOTCH1 based on the provided context. However, ARHGEF2 is involved in cell motility, cancer, and innate immune response, while NOTCH1 is involved in cell-fate determination, angiogenesis, and neuronal differentiation.|
| [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKAP](https://www.ncbi.nlm.nih.gov/gene?term=IKBKAP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.3</br>**STRING score:** 456</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | There is no clear functional association between HNRNPA2B1 and IKBKAP based on their known biological roles. HNRNPA2B1 is primarily involved in RNA transport and processing, while IKBKAP is involved in tRNA modification and neuronal development. However, they both have roles in binding to other proteins, suggesting potential indirect interactions.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.6</br>**STRING score:** 196</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 3) | Possible functional association between DDX42 and HNRNPA2B1 is their role in splicing factor SF3B complex, which is involved in RNA splicing. DDX42 is an ATP-dependent RNA helicase that promotes RNA strand separation, while HNRNPA2B1 is a regulatory protein that binds to RNA motifs and transports them out of the nucleus. Both genes are also involved in regulating gene expression and play a role in cellular survival.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.5</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between NVL and SMC3 include their involvement in telomerase activity and ribosomal subunit biogenesis. NVL participates in the assembly of the telomerase holoenzyme and regulates 60S ribosomal subunit biogenesis, while SMC3 is a central component of the cohesin complex, which plays a role in spindle pole assembly during mitosis and in chromosomes movement. There is no direct evidence of functional associations between these two genes.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.3</br>**STRING score:** 975</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | RFC2 and RPA2 both function in DNA replication and repair. RFC2 is an accessory protein that is part of a heterotetrameric complex involved in DNA elongation, while RPA2 stabilizes single-stranded DNA intermediates and recruits proteins for DNA metabolism. Based on their involvement in similar biological processes, it is possible that RFC2 and RPA2 may have functional associations in DNA replication and repair.|
| [NOS3](https://www.ncbi.nlm.nih.gov/gene?term=NOS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.7</br>**STRING score:** 392</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: NOS3 and SHC1 are both involved in cellular signaling pathways and may interact in the regulation of cell proliferation and survival.   NOS3 plays a role in the maintenance of germ cell lineage and suppresses apoptotic pathways, while SHC1 couples activated growth factor receptors to signaling pathways and is involved in the regulation of endothelial cell migration and sprouting angiogenesis. The shortest path between these genes in the BioGRID protein interaction network suggests potential physical interactions between them, which may modulate their respective functions.|
| [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.6</br>**STRING score:** 994</br>**BioGRID count:** 15</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PSME1 and PSME2 include their involvement in immunoproteasome assembly and efficient antigen processing, as well as their ability to form a heterodimer and hexameric ring. Both genes are required for the generation of class I binding peptides, and PSME1 can also form homoheptamers.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.5</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between GOLT1B and TXNDC9 include involvement in protein folding, as TXNDC9 negatively impacts protein folding and GOLT1B may be involved in vesicular transport between the ER and Golgi complex. The shortest path between these genes in the BioGRID protein interaction network also suggests possible physical interactions between their encoded proteins.|
| [GFPT1](https://www.ncbi.nlm.nih.gov/gene?term=GFPT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.4</br>**STRING score:** 284</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no direct evidence for a functional association between GFPT1 and GOLT1B. GFPT1 controls glucose flux into the hexosamine pathway and regulates the expression of clock genes BMAL1 and CRY1. GOLT1B may be involved in ER/Golgi fusion.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.0</br>**STRING score:** 820</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between ICAM1 and NRAS include their involvement in cellular signaling pathways related to immune response and infection. ICAM1 plays a role in cell-cell adhesion and immune functions, while NRAS is a GTPase involved in signal transduction pathways regulating cell growth and differentiation. The context provided suggests potential interactions with viral infection and modulation of immune response.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIT1](https://www.ncbi.nlm.nih.gov/gene?term=NIT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.7</br>**STRING score:** 267</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | It is not clear from the context and known biology what possible functional associations exist between ECH1 and NIT1. ECH1 is involved in fatty acid metabolism while NIT1 plays a role in metabolite repair and has tumor suppressor properties.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.5</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association: CDK2 and TGFBR2 may be involved in regulating cell cycle progression and growth through their interactions in the BioGRID protein interaction network and the involvement of CDK2 in controlling the cell cycle and TGFBR2 in blocking activation of SMAD2 and SMAD3.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.9</br>**STRING score:** 626</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between CASP3 and CDKN1B include involvement in the regulation of cell cycle progression and apoptosis. CASP3 is a protease that plays a major role in apoptosis, while CDKN1B is an important regulator of cell cycle progression. The shortest path in the BioGRID protein interaction network between these two genes involves several proteins, suggesting possible interactions between them.|
