| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 207.3</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 3) | CD44 is a cell-surface receptor that plays a role in cell-cell interactions, cell adhesion, and migration by binding to extracellular matrix components such as hyaluronan, collagen, and growth factors. CLTC is a major component of clathrin-coated vesicles and is involved in intracellular trafficking and early autophagosome formation. A possible functional association between these two genes could be through their interactions with EGFR, which is a key player in cell signaling pathways related to cell adhesion and migration, indicating a potential role in coordinating cellular responses in the tissue microenvironment.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCMT1](https://www.ncbi.nlm.nih.gov/gene?term=PCMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 201.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | GOLT1B is likely involved in the fusion of ER-derived transport vesicles with the Golgi complex, while PCMT1 plays a role in repairing damaged proteins by catalyzing methyl esterification of specific residues. A possible functional association between these two genes could be that GOLT1B aids in the transport of PCMT1 and/or its substrates within the cell, enabling PCMT1 to effectively repair damaged proteins.|
| [NR2F6](https://www.ncbi.nlm.nih.gov/gene?term=NR2F6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 183.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 504 (min len 4) | NR2F6 is a transcription factor involved in transcriptional repression, modulating hormonal responses, and development of the forebrain circadian clock, while REEP5 regulates the organization and function of the sarcoplasmic reticulum in cardiomyocytes, playing an essential role in heart function and development. A possible functional association between these two genes could be in their involvement in maintaining peripheral immunological tolerance and autoimmunity. However, further studies are needed to confirm any direct functional associations between NR2F6 and REEP5 in specific biological processes.|
| [FYN](https://www.ncbi.nlm.nih.gov/gene?term=FYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GDPD5](https://www.ncbi.nlm.nih.gov/gene?term=GDPD5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 178.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | FYN is a non-receptor tyrosine-protein kinase involved in processes like cell growth, survival, adhesion, and motility, while GDPD5 is a glycerophosphodiester phosphodiesterase that promotes neurite formation and motor neuron differentiation. Although both genes have distinct roles in cellular processes, there may be a functional association between them in the context of cell adhesion and motility, as FYN regulates these processes through phosphorylation and GDPD5 is involved in neurite formation and differentiation.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | GOLT1B is involved in the fusion of ER-derived transport vesicles with the Golgi complex, while PSMD10 is a proto-oncoprotein involved in negative regulation of tumor suppressors RB1 and p53/TP53, as well as the regulation of NF-kappa-B. The functional association between GOLT1B and PSMD10 may involve protein transport and degradation processes, specifically in the context of cellular protein homeostasis and tumorigenesis regulation.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.9</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | CD44 is a cell-surface receptor that plays a role in cell-cell interactions, cell adhesion, and migration by interacting with extracellular matrix components and promoting signaling pathways for cell migration and adhesion. GRN, or progranulin, is involved in stabilizing proteins and facilitating lysosomal delivery via interactions with other proteins. Although there are no direct functional associations between CD44 and GRN, they both participate in signaling pathways and protein interactions that influence cellular functions such as cell migration and extracellular matrix interactions.|
| [CASC3](https://www.ncbi.nlm.nih.gov/gene?term=CASC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTF2](https://www.ncbi.nlm.nih.gov/gene?term=MTF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | CASC3 is involved in pre-mRNA splicing and exon-exon junction regulation, while MTF2 is associated with gene regulation through histone modification and recruitment of the PRC2 complex. Although both genes have roles in gene expression regulation, there is no direct functional association between CASC3 and MTF2 in specific biological terms.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF395](https://www.ncbi.nlm.nih.gov/gene?term=ZNF395[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | ICAM1 is involved in cell adhesion and immune responses, and is known to interact with viral proteins during infections. ZNF395 plays a role in the transcription of papillomavirus genes and interacts with human papillomavirus type 8. Functional associations between these two genes may involve viral infection processes, specifically in immune evasion and gene expression regulation during infections.|
| [KAT6B](https://www.ncbi.nlm.nih.gov/gene?term=KAT6B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | KAT6B plays a role in transcription regulation, particularly in RUNX2-dependent transcriptional activation and cerebral cortex development. NVL is involved in telomerase holoenzyme assembly, pre-rRNA processing pathways, and ribosomal subunit biogenesis. Although both genes have a role in nucleic acid-related processes, there is no direct functional association in specific biological terms between KAT6B and NVL.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 361</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between GOLT1B and NOLC1 might involve ER-Golgi transport and nucleologenesis. GOLT1B is likely involved in the fusion of ER-derived transport vesicles with the Golgi complex. On the other hand, NOLC1 plays a role in regulating RNA polymerase I, ribosomal processing, and nucleolar structure maintenance. It is unclear how these functions could be directly associated, as they involve distinct cellular processes.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YTHDF2](https://www.ncbi.nlm.nih.gov/gene?term=YTHDF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.6</br>**STRING score:** 301</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | MYC is a transcription factor involved in activating growth-related genes and regulating embryonic stem cell self-renewal, while YTHDF2 is associated with RNA stability, promoting viral gene expression, and recruiting the CCR4-NOT complex. A possible functional association between these two genes could involve the regulation of gene expression through mRNA processing, as MYC is involved in transcriptional activation and YTHDF2 participates in RNA stability and degradation regulation. However, direct interactions between MYC and YTHDF2 have not been reported in the literature.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.0</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | CDK2 is a serine/threonine-protein kinase involved in the control of the cell cycle, promoting the E2F transcriptional program and initiation of DNA synthesis, while USP1 is a negative regulator of DNA damage repair that deubiquitinates monoubiquitinated FANCD2 and PCNA. The possible functional association between the two genes may lie in their shared involvement in the DNA damage response and cell cycle regulation. Both CDK2 and USP1 play roles in maintaining the balance between cellular proliferation and DNA repair processes.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSIP1](https://www.ncbi.nlm.nih.gov/gene?term=PSIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 162.6</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | NVL and PSIP1 may have functional associations in cellular processes such as ribosome biogenesis and stress responses. NVL plays a crucial role in telomerase activity and pre-RNA processing, while PSIP1 functions as a transcriptional coactivator in neuroepithelial stem cell differentiation and cellular stress response. Their interaction could potentially influence gene regulation and cellular responses to stress.|
| [ILK](https://www.ncbi.nlm.nih.gov/gene?term=ILK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.8</br>**STRING score:** 932</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ILK and SHC1 are both involved in signaling pathways that regulate cell adhesion, motility, and integrin-mediated signal transduction. ILK plays a role in inside-out integrin signaling and cell architecture, while SHC1 functions as a signaling adapter coupling activated growth factor receptors to downstream signaling pathways. The functional association between these two genes could involve their roles in integrin signaling and the regulation of cell adhesion and motility.|
| [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.7</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | There is a possible functional association between CRK and STAT3 in regulating cell motility, adhesion, and signaling pathways related to cell growth. CRK is involved in cell adhesion, spreading, migration, and phagocytosis, while STAT3 functions as a signal transducer and transcription activator mediating cellular responses to various growth factors and cytokines. The two genes may interact indirectly through their involvement in signaling pathways, such as PDGFRA, KIT, and EPHA3 for CRK and IL6, FGFRs, and LEP for STAT3.|
| [ARL4C](https://www.ncbi.nlm.nih.gov/gene?term=ARL4C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.4</br>**STRING score:** 153</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 71 (min len 3) | ARL4C and CDC42 are both small GTP-binding proteins involved in cellular responses and transport processes. ARL4C may play a role in transport pathways linked to cholesterol secretion, while CDC42 is implicated in epithelial cell polarization, cell migration, and regulation of spindle microtubules during cell division. Possible functional associations between ARL4C and CDC42 might involve vesicle transport or intracellular trafficking and the regulation of the actin cytoskeleton.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.4</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 124 (min len 3) | CLTC is involved in the formation of clathrin-coated vesicles, stabilization of kinetochore fibers, and early autophagosome formation, while PSMD2 is a component of the 19S proteasome regulatory particle complex and plays a role in protein degradation. A possible functional association between CLTC and PSMD2 could be related to their participation in intracellular protein trafficking and degradation processes. Both genes may contribute to maintaining cellular homeostasis by ensuring proper protein recognition, transport, and degradation.|
| [PEX11A](https://www.ncbi.nlm.nih.gov/gene?term=PEX11A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB3](https://www.ncbi.nlm.nih.gov/gene?term=TRIB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.3</br>**STRING score:** 338</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 4) | PEX11A is involved in peroxisomal proliferation, regulation of peroxisomes division, and mediating the binding of coatomer proteins to the peroxisomal membrane. TRIB3 is an inactive protein kinase that acts as a regulator of the integrated stress response and is involved in ER stress-related cell death and insulin signaling.  A possible functional association between PEX11A and TRIB3 could be their involvement in intracellular stress responses, as PEX11A is involved in peroxisome maintenance and TRIB3 regulates cellular stress. However, direct functional connections between the two genes remain unknown.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.2</br>**STRING score:** 993</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CD44 is a cell-surface receptor involved in cell-cell interactions, cell adhesion, and migration through extracellular matrix interactions and signal transduction. RHOA is a RhoGTPase involved in actin-mediated cytoskeleton reorganization and cell migration. The functional association between CD44 and RHOA lies in their cooperation in signaling pathways that regulate cell adhesion and migration by orchestrating cytoskeleton organization.|
| [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC5A6](https://www.ncbi.nlm.nih.gov/gene?term=SLC5A6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 156.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | SLC25A46 is a transmembrane protein involved in regulating mitochondrial organization, assembly of the MICOS complex, and mitochondrial lipid homeostasis. On the other hand, SLC5A6 is a sodium-dependent multivitamin transporter that transports essential vitamins and plays a role in maintaining intestinal mucosa integrity and transport across the blood-brain barrier. While both genes encode membrane transport proteins, there is no clear functional association between the two genes in specific biological processes, as they have distinct roles in different cellular compartments and functions.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.1</br>**STRING score:** 422</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 3) | CDC42 is a plasma membrane-associated small GTPase involved in cell polarization, migration, phagocytosis, and formation of filopodia and dendritic spines. Meanwhile, ICAM1 plays a role in cell adhesion and is associated with microbial infections. A possible functional association between these two genes could involve their contribution to cell migration and cytoskeletal organization, where CDC42 regulates F-actin cytoskeleton and ICAM1 regulates cell-cell and cell-matrix interactions.|
| [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.7</br>**STRING score:** 936</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 90 (min len 3) | SMC3 and SMC4 are both involved in essential chromosomal processes, with SMC3 being a central component of the cohesin complex, responsible for chromosome cohesion during cell division, and SMC4 being a core component of the condensin complex, required for condensing interphase chromatin into mitotic-like chromosomes. There might be a possible functional association between these two genes in coordinating chromosome dynamics during cell division, facilitating correct segregation and movement of chromosomes.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.7</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CASP3 is a major effector caspase involved in the execution phase of apoptosis, while USP1 is a negative regulator of DNA damage repair and is involved in PCNA-mediated translesion synthesis. A possible functional association between these two genes could be related to their roles in cellular stress response, where CASP3 promotes apoptosis, and USP1 regulates DNA repair mechanisms. These genes may function in a coordinated manner to maintain cellular homeostasis during stress conditions.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 152.1</br>**STRING score:** 880</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 138 (min len 3) | CCT8 is a component of the chaperonin-containing T-complex (TRiC), which assists in protein folding and plays a role in ciliogenesis. PSMD2 is part of the 19S proteasome regulatory particle complex, involved in protein degradation. While both genes have distinct functions in protein folding and degradation, there is no direct functional association between them in specific biological terms.|
| [CCT8](https://www.ncbi.nlm.nih.gov/gene?term=CCT8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.8</br>**STRING score:** 937</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | CCT8 is a component of the chaperonin-containing T-complex (TRiC), which assists in protein folding and plays a role in ciliogenesis and telomere maintenance. PSMG1 is a chaperone protein involved in the assembly of the 20S proteasome. Although both genes function in protein folding and assembly, they participate in distinct biological processes - CCT8 in ciliogenesis and telomere maintenance, and PSMG1 in proteasome assembly.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.3</br>**STRING score:** 672</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | MYC is a transcription factor that regulates growth-related genes and is involved in angiogenesis, somatic reprogramming, and embryonic stem cell self-renewal. SMARCC1 is a component of the SWI/SNF chromatin remodeling complexes, which play a role in gene activation and repression by altering DNA-nucleosome topology. Although both genes are involved in transcription regulation, no direct functional association between MYC and SMARCC1 in specific biological processes has been reported.|
| [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.1</br>**STRING score:** 903</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 58 (min len 3) | NCAPD2 is a regulatory subunit of the condensin complex involved in chromatin condensation, while SMC3 is a central component of the cohesin complex required for chromosome cohesion during the cell cycle. Both genes are functionally associated with chromosome dynamics and structure maintenance, playing crucial roles in various processes such as DNA replication, repair, mitosis, and neurogenesis.|
| [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.7</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 46 (min len 3) | Possible functional associations between RHOA and SMC4 include cellular organization and chromosome structure regulation during cell division. RHOA regulates actin cytoskeleton and is involved in microbial interaction, while SMC4 is a central component of the condensin complex responsible for converting interphase chromatin into mitotic-like condensed chromosomes. Their functional association can potentially impact cell division and chromosome integrity.|
| [CLTC](https://www.ncbi.nlm.nih.gov/gene?term=CLTC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.0</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | CLTC is involved in coated pit and vesicle formation, stabilization of kinetochore fibers, and early autophagosome formation, while NPC1 acts as an endosomal entry receptor for ebolavirus. The functional association between these genes could be related to endocytosis and vesicle trafficking, as CLTC plays a role in vesicle formation and NPC1 is involved in endosomal entry of pathogens.|
| [ARHGEF2](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.5</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ARHGEF2 is involved in the activation of Rho-GTPases and contributes to processes such as antigen presentation, cell motility, and polarization. NOTCH1 functions as a receptor regulating cell fate determination, angiogenesis, and hematopoietic cell differentiation. A possible functional association between these two genes could be seen in the regulation of cell differentiation and motility, although no direct interaction has been reported yet.|
| [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKAP](https://www.ncbi.nlm.nih.gov/gene?term=IKBKAP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.3</br>**STRING score:** 456</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | HNRNPA2B1 is involved in mRNA processing and transport, while also playing a role in the transport of HIV-1 genomic RNA. IKBKAP is a component of the elongator complex, essential for tRNA modifications and neuronal migration in the developing brain. Although both genes interact with some common proteins (e.g., TBK1, STING1, and SRC), no direct functional association between HNRNPA2B1 and IKBKAP is evident based on their specific roles in mRNA and tRNA processing, respectively.|
| [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.6</br>**STRING score:** 196</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 3) | DDX42 and HNRNPA2B1 are involved in RNA processing and regulation. DDX42 is an ATP-dependent RNA helicase that unwinds RNA secondary structures and participates in the survival of cells by interacting with TP53BP2. HNRNPA2B1, on the other hand, is involved in the transport of HIV-1 genomic RNA and has various interactions with other cellular factors. Therefore, a possible functional association between DDX42 and HNRNPA2B1 could be their involvement in RNA processing and regulation, particularly in RNA strand separation and transport.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.5</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | NVL and SMC3 are involved in distinct cellular processes, where NVL plays a role in telomerase holoenzyme assembly, pre-rRNA processing, and ribosomal subunit biogenesis, while SMC3 is a central component of the cohesin complex, essential for chromosome cohesion during cell division, DNA repair, and spindle pole assembly. There is no direct functional association between NVL and SMC3; however, they are connected through intermediate proteins in protein interaction networks, such as CUL3, RPL14, HNRNPU, MECP2, and ESR2.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 148.3</br>**STRING score:** 975</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | RFC2 and RPA2 are both involved in DNA replication processes. RFC2 plays a role in the elongation of primed DNA templates, functioning as an accessory protein for DNA polymerases delta and epsilon, while RPA2 is part of the replication protein A complex, stabilizing single-stranded DNA intermediates during replication or DNA stress. A possible functional association between these genes may be their involvement in coordinating DNA replication and repair mechanisms.|
| [NOS3](https://www.ncbi.nlm.nih.gov/gene?term=NOS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.7</br>**STRING score:** 392</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | NOS3 is involved in the production of nitric oxide and regulation of L-arginine synthesis and citrulline recycling, while SHC1 is a signaling adapter that couples activated growth factor receptors to signaling pathways in various cellular processes. A possible functional association between these two genes could be through their interaction with other signaling proteins, such as HSP90AA1, APP, ACTN4, or ACTB, which may play a role in regulating cell signaling pathways related to nitric oxide production and growth factor receptor activation.|
| [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.6</br>**STRING score:** 994</br>**BioGRID count:** 15</br>**BioGRID shortest paths:** 1 (min len 2) | PSME1 and PSME2 are both implicated in immunoproteasome assembly and are required for efficient antigen processing. They form a heterodimer, which results in a hexameric ring, known as the PA28 activator complex, enhancing the generation of class I binding peptides by altering the cleavage pattern of the proteasome.|
| [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.5</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | GOLT1B is involved in the fusion of ER-derived transport vesicles with the Golgi complex, while TXNDC9 negatively impacts protein folding by diminishing chaperonin TCP1 complex ATPase activity. A possible functional association between these two genes could be related to protein trafficking and folding processes within the cell, as GOLT1B's role in vesicle fusion may indirectly influence the availability or function of protein substrates for TXNDC9.|
| [GFPT1](https://www.ncbi.nlm.nih.gov/gene?term=GFPT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GOLT1B](https://www.ncbi.nlm.nih.gov/gene?term=GOLT1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.4</br>**STRING score:** 284</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | GFPT1 regulates the glucose flux into the hexosamine pathway and is involved in glycosylation of proteins, while GOLT1B is involved in the fusion of ER-derived transport vesicles with the Golgi complex. A possible functional association between these two genes could be related to post-translational modification and processing of proteins. Specifically, GFPT1 could impact the availability of precursors for glycosylation, while GOLT1B might assist in protein transport and maturation within the Golgi apparatus.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.0</br>**STRING score:** 820</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | ICAM1 is involved in cell adhesion and immune response, functioning as a receptor on various cell types, and plays a role in microbial infection response. NRAS is a small GTPase involved in cellular signal transduction and regulation of cell growth and survival. Although there is no direct functional association between ICAM1 and NRAS, they may interact indirectly through intermediary proteins, potentially impacting cell growth, survival, and immune response regulation.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIT1](https://www.ncbi.nlm.nih.gov/gene?term=NIT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.7</br>**STRING score:** 267</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ECH1 is involved in isomerization of fatty acid CoA esters, while NIT1 catalyzes the hydrolysis of a harmful metabolite, deaminated glutathione, and has tumor suppressor properties. Both genes are involved in metabolic processes; however, their functions do not seem to be directly related in terms of specific biological processes.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 146.5</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | CDK2 is a serine/threonine-protein kinase that plays a crucial role in cell cycle control, DNA replication, and repair, while TGFBR2 is a receptor that binds to TGF-beta ligands controlling cell growth and differentiation. Functional associations between these two genes could be related to regulation of cell cycle progression, particularly at the G1-S transition, and the modulation of the balance between cellular proliferation, cell death, and DNA repair.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.9</br>**STRING score:** 626</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | CASP3 is a major effector caspase involved in the execution phase of apoptosis, while CDKN1B plays a crucial role in cell cycle progression regulation and G1 arrest. There may be functional associations between these two genes in terms of cell death regulation, with CASP3 mediating apoptosis and CDKN1B controlling cell cycle progression to prevent uncontrolled cell growth and division.|
