| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 189.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HEBP1 and NPC1 include the binding and removal of potentially toxic porphyrinogens and metalloporphyrins, as well as acting as an endosomal entry receptor for ebolavirus. However, their specific biological functions do not suggest any direct interaction or shared pathway between them.|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 181.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HEBP1 and HSPD1 include involvement in protein folding and mitochondrial functions. HEBP1 may bind free porphyrinogens and porphyrins, while HSPD1 is a chaperonin that facilitates the correct folding of imported proteins and prevents misfolding under stress conditions. The shortest path in the BioGRID protein interaction network also suggests a direct interaction between these two genes through several intermediate proteins.|
| [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 178.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: Based on the shortest path in the BioGRID network, CTNNAL1 and PSMD2 are indirectly connected through proteins involved in regulating Rho signaling and protein degradation pathways. CTNNAL1 may scaffold the Rho guanine nucleotide exchange factor ARHGEF1 while PSMD2 is a component of the 19S regulatory particle complex involved in the 26S proteasome. This suggests a possible role for both genes in regulating protein degradation in cancer and highlights a potential link between Rho signaling and proteasomal degradation pathways.|
| [EIF4A3](https://www.ncbi.nlm.nih.gov/gene?term=EIF4A3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 3) | Possible functional associations between EIF4A3 and RPN2 in TCGA-KIRC cancer type may involve protein interaction in the mRNA splicing-dependent exon junction complex (EJC) and possibly in protein N-glycosylation through their respective roles in the EJC core complex and the oligosaccharyltransferase (OST) complex.|
| [PLEKHJ1](https://www.ncbi.nlm.nih.gov/gene?term=PLEKHJ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Although the exact functional association between PLEKHJ1 and PSMD2 is not known, PSMD2 is a component of the 19S proteasome regulatory particle complex involved in protein degradation, while PLEKHJ1 is a component of the MCM2-7 complex essential for DNA replication initiation. They may play a role in maintaining cellular homeostasis through regulation of protein levels during DNA replication.|
| [MYO10](https://www.ncbi.nlm.nih.gov/gene?term=MYO10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 168.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations: MYO10 and PCNA both play important roles in DNA replication and repair, with PCNA serving as an auxiliary protein for DNA polymerase and MYO10 potentially involved in regulating VASP retention in spine heads to induce spine formation and expansion. However, there is currently no known direct interaction between these two genes in the literature or in the BioGRID protein interaction network.|
| [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between EML3 and NPC1 based on the provided context. EML3 is involved in spindle assembly and chromosome separation during mitosis while NPC1 acts as a receptor for ebolavirus and may play a role in endosomal trafficking.|
| [MRPS16](https://www.ncbi.nlm.nih.gov/gene?term=MRPS16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: NPC1 interacts with ebolavirus glycoprotein and MRPS16 is a component of the mitochondrial small ribosomal subunit. While there is no direct evidence for a functional link between NPC1 and MRPS16, it is possible that altered mitochondrial function could have downstream effects on viral infection.|
| [ENOSF1](https://www.ncbi.nlm.nih.gov/gene?term=ENOSF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PYGL](https://www.ncbi.nlm.nih.gov/gene?term=PYGL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ENOSF1 and PYGL include their potential involvement in glucose homeostasis and glycogen metabolism. ENOSF1 plays a role in the catabolism of carbohydrates and interacts with RAP2A, while PYGL catalyzes the rate-limiting step in glycogen catabolism and interacts with PPP1R3B to regulate its activity. The shortest path in the BioGRID interaction network indicates that these two genes may interact indirectly through MYC and PARK2.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their known functions, PARP1 is involved in DNA repair and apoptosis while PSMD2 is a component of the proteasome complex involved in protein degradation. There is no known functional association between these two genes.|
| [NOSIP](https://www.ncbi.nlm.nih.gov/gene?term=NOSIP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.3</br>**STRING score:** 311</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between NOSIP and PIH1D1 could be the regulation of mTORC1 complex through PIH1D1 recruitment of the SWI/SNF complex to rRNA genes and NOSIP negative regulation of nitric oxide production, affecting mTORC1 signaling pathway. Both genes are involved in protein complex assembly and interactions with RNA processing machinery, suggesting potential co-regulation of protein synthesis and RNA processing.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.1</br>**STRING score:** 375</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Both CFLAR and PARP1 are involved in apoptosis regulation. Possible functional associations between these two genes could be their involvement in the DNA damage-induced apoptosis pathway, where CFLAR inhibits TNFRSF6-mediated apoptosis and PARP1 irreversibly binds to DNA breaks, leading to apoptosis.|
| [KCTD5](https://www.ncbi.nlm.nih.gov/gene?term=KCTD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.9</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between KCTD5 and SQSTM1 could involve the regulation of protein degradation and turnover, specifically through interactions with E3 ligase complexes and autophagy. SQSTM1 is involved in selective macroautophagy and acts as a bridge between polyubiquitinated cargo and autophagosomes, while KCTD5 may act as a substrate adapter in E3 ligase complexes.|
| [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.7</br>**STRING score:** 841</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 84 (min len 3) | Possible functional associations between GRWD1 and RRP1B include their involvement in chromatin dynamics and gene expression regulation. GRWD1 is a histone binding protein that regulates chromatin dynamics and minichromosome maintenance, while RRP1B interacts with viral RNA-directed RNA polymerase, potentially playing a role in gene expression during viral infection. The identified protein interaction network suggests that these genes may be involved in DNA replication and repair processes through their interactions with Dot1L and CUL3.|
| [PSMF1](https://www.ncbi.nlm.nih.gov/gene?term=PSMF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.1</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association between PSMF1 and SRP14: involved in protein degradation and co-translational targeting of secretory and membrane proteins to the endoplasmic reticulum, respectively. They may be involved in the regulation of protein homeostasis in cancer cells.|
| [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence for a functional association between HMG20B and HSPD1. HMG20B is involved in cell cycle regulation and chromatin remodeling, while HSPD1 is a mitochondrial chaperonin implicated in protein folding and assembly.|
| [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCRN1](https://www.ncbi.nlm.nih.gov/gene?term=SCRN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known direct functional association between RPN2 and SCRN1, but they both have diverse biological roles. RPN2 is involved in protein N-glycosylation and is a subunit of the OST complex, while SCRN1 plays a role in regulating exocytosis in mast cells and is a component of a base excision repair complex.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.5</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between CEBPA and PHGDH include regulation of cell cycle and metabolism. CEBPA directly enhances ribosomal DNA transcription and induces G1 cell cycle arrest, while PHGDH catalyzes the reversible oxidation of 3-phospho-D-glycerate to 3-phosphonooxypyruvate, the first step of the phosphorylated L-serine biosynthesis pathway. The shortest path between these genes in the BioGRID network involves MYC, NR3C1, AR, ESR1, and CDK2, which are involved in various processes including transcriptional regulation and cell proliferation.|
| [CRTAP](https://www.ncbi.nlm.nih.gov/gene?term=CRTAP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations could be in the protein glycosylation process, as RPN2 is a subunit of the oligosaccharyl transferase complex responsible for N-glycosylation, while CRTAP interacts with the GATOR2 complex that negatively regulates the mTOR pathway, which affects protein synthesis and secretion. However, further investigation is required to determine any direct functional associations between these two genes.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.8</br>**STRING score:** 932</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | PARP1 and PCNA are functionally associated in DNA repair and damage response pathways. PARP1 interacts with PCNA to promote DNA repair and apoptosis, while PCNA acts as a loading platform to recruit PARP1 and other DDR proteins to coordinate DNA replication with repair.|
| [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYO10](https://www.ncbi.nlm.nih.gov/gene?term=MYO10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.8</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between CTTN and MYO10 could involve regulation of actin cytoskeleton organization, cell migration, and/or focal adhesion assembly and turnover. The shortest path in the BioGRID protein interaction network between these two genes also suggests a potential physical interaction between CTTN and MYO10.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.2</br>**STRING score:** 593</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 186 (min len 3) | Possible functional associations between HDAC2 and HDAC6 are their involvement in regulating gene expression through chromatin structure modification and deacetylation of histones and non-histone proteins like microtubules. They are part of the NuRD and SIN3 HDAC complexes that remodel chromatin and can interact together in a protein network involved in transcriptional corepression of multiple genes.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLA2G15](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.4</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between HSPD1 and PLA2G15 include their involvement in mitochondrial protein import and glycerophospholipid homeostasis, respectively. HSPD1 acts as a chaperonin for proper folding and assembly of proteins in the mitochondrial matrix while PLA2G15 has dual phospholipase and O-acyltransferase activities for phospholipid degradation and remodeling in acidic cellular compartments. There may not be direct biological interactions between the two proteins, but their respective functions could play important roles in maintaining cellular homeostasis and metabolism.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.6</br>**STRING score:** 362</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between ECH1 and PSMD2 could be related to protein degradation processes mediated by the 26S proteasome complex. ECH1 is involved in isomerization of fatty acid CoA esters, while PSMD2 is a subunit of the 19S proteasome regulatory particle complex.|
| [HERC6](https://www.ncbi.nlm.nih.gov/gene?term=HERC6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB8](https://www.ncbi.nlm.nih.gov/gene?term=PSMB8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.0</br>**STRING score:** 268</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 141 (min len 4) | Possible functional associations between HERC6 and PSMB8 include their involvement in antigen processing and viral infections. HERC6 is an E3 ubiquitin-protein ligase that interacts with influenza A virus nucleoprotein NP and with RNA-directed RNA polymerase subunits PB1 and PB2. PSMB8, on the other hand, plays a key role in antigen processing to generate class I binding peptides and is involved in the HIV-1 TAT protein interaction. The shortest path between these genes in the BioGRID protein interaction network suggests a potential indirect interaction through other proteins involved in protein degradation and processing.|
