| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 189.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HEBP1 and NPC1 may include involvement in heme/porphyrin metabolism and endosomal entry for certain microbial infections, respectively. It is unclear whether these two genes directly interact biologically, but their roles suggest potential convergence in certain cellular pathways.|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 181.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HEBP1 and HSPD1 could be related to protein folding and assembly, including mitochondrial protein import and macromolecular assembly. Both genes have been shown to play a role in facilitating the correct folding of proteins and preventing misfolding and aggregation. The shortest path in the BioGRID protein interaction network between HEBP1 and HSPD1 includes chaperones HSPA1A and HSPA8.|
| [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 178.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear functional association between CTNNAL1 and PSMD2. CTNNAL1 may modulate the Rho pathway signaling, while PSMD2 is involved in the 19S proteasome regulatory particle complex and binds to TNFR1.|
| [EIF4A3](https://www.ncbi.nlm.nih.gov/gene?term=EIF4A3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 78 (min len 3) | EIF4A3 and RPN2 are involved in different biological processes. EIF4A3 plays a key role in pre-mRNA splicing, while RPN2 is a subunit of the oligosaccharyl transferase complex involved in protein N-glycosylation. There is no known functional association between these two genes in specific biological terms.|
| [PLEKHJ1](https://www.ncbi.nlm.nih.gov/gene?term=PLEKHJ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: PLEKHJ1 is involved in DNA replication and cell division, while PSMD2 is a component of the proteasome regulatory particle complex. There is no clear biological process linking these two genes.|
| [MYO10](https://www.ncbi.nlm.nih.gov/gene?term=MYO10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 168.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | There is no clear evidence for functional associations between MYO10 and PCNA in specific biological terms. MYO10 is involved in filopodia extension and motility, while PCNA is involved in DNA replication, damage response, and repair. However, both genes are part of a protein interaction network, and there may be indirect functional associations through their shared interactors.|
| [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no clear functional associations between EML3 and NPC1 in specific biological terms. EML3 is involved in regulating mitotic spindle assembly and chromosome separation, while NPC1 acts as an endosomal entry receptor for ebolavirus. The shortest path between EML3 and NPC1 in the BioGRID protein interaction network involves TRIM67 as an intermediate protein.|
| [MRPS16](https://www.ncbi.nlm.nih.gov/gene?term=MRPS16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: none apparent from the provided context and known functions of MRPS16 and NPC1.  MRPS16 is a component of mitochondrial ribosomes, while NPC1 acts as an endosomal entry receptor for ebolavirus and interacts with ebolavirus glycoprotein.|
| [ENOSF1](https://www.ncbi.nlm.nih.gov/gene?term=ENOSF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PYGL](https://www.ncbi.nlm.nih.gov/gene?term=PYGL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ENOSF1 and PYGL could involve regulation of sugar metabolism, as ENOSF1 is involved in catabolizing L-fucose and PYGL plays a role in glycogen catabolism. However, without further evidence, it is difficult to determine a specific biological process where these two genes may interact.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PARP1 and PSMD2 are involvement in DNA repair and protein degradation, respectively. PARP1 binds irreversibly to DNA breaks and interferes with DNA repair, while PSMD2 is a component of the proteasome 19S regulatory particle complex involved in protein degradation. There may be a role for PSMD2 in degrading proteins damaged by PARP1-mediated DNA damage.|
| [NOSIP](https://www.ncbi.nlm.nih.gov/gene?term=NOSIP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.3</br>**STRING score:** 311</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: PIH1D1 is involved in the assembly and activity of the mTORC1 complex, which is also negatively regulated by NOSIP. This suggests a possible regulatory interaction between NOSIP and PIH1D1 in the mTORC1 pathway. NOSIP also interacts with NOS1 and NOS3, while PIH1D1 interacts with TELO2 and the R2TP complex, suggesting potential roles in protein complex assembly and regulation.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 150.1</br>**STRING score:** 375</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | One possible functional association between CFLAR and PARP1 is their involvement in the regulation of apoptosis. CFLAR acts as an inhibitor of TNFRSF6 mediated apoptosis, while PARP1 plays a role in promoting DNA damage-induced apoptosis. The shortest protein interaction path between these two genes involves intermediate proteins such as CASP8, CASP3, and TP53, all of which are also involved in regulating apoptosis.|
| [KCTD5](https://www.ncbi.nlm.nih.gov/gene?term=KCTD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.9</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional association between KCTD5 and SQSTM1 is related to ubiquitination and autophagy processes. KCTD5 interacts with E3 ligase complexes and SQSTM1 serves as an autophagy receptor for selective macroautophagy and mediates the interaction between polyubiquitinated cargo and autophagosomes.|
| [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.7</br>**STRING score:** 841</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 84 (min len 3) | Possible functional associations between GRWD1 and RRP1B could involve chromatin dynamics, replication origin regulation, and mRNA transcription. GRWD1 regulates chromatin dynamics and minichromosome maintenance loading at replication origins through interaction with CDC6 and CDT1. RRP1B facilitates influenza virus mRNA transcription and interacts with RNA-directed RNA polymerase subunits. It is unclear if these two genes directly interact, but they may play complementary roles in maintaining chromatin and regulating mRNA transcription.|
| [PSMF1](https://www.ncbi.nlm.nih.gov/gene?term=PSMF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 142.1</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between PSMF1 and SRP14 include their involvement in protein degradation (PSMF1) and protein targeting to the endoplasmic reticulum (SRP14) - two interconnected cellular processes. The interactions between PSMF1 and SRP14 in the BioGRID protein interaction network may suggest a role for PSMF1 in regulating the activity of SRP14 during protein targeting.|
| [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between HMG20B and HSPD1. HMG20B is involved in cell cycle regulation and gene expression, while HSPD1 is involved in mitochondrial protein import and chaperoning.|
| [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCRN1](https://www.ncbi.nlm.nih.gov/gene?term=SCRN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between RPN2 and SCRN1, as they have distinct roles in different cellular processes - RPN2 is involved in protein N-glycosylation while SCRN1 is involved in exocytosis and DNA repair. However, the fact that they are the only two genes in the shortest path of the BioGRID protein interaction network suggests that they may physically interact in some way.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.5</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional association: It is possible that CEBPA and PHGDH are involved in the regulation of ribosomal DNA transcription and cell cycle arrest.   CEBPA is known to directly enhance ribosomal DNA transcription and induce G1 cell cycle arrest, while PHGDH catalyzes the first step of the phosphorylated L-serine biosynthesis pathway. There may be a regulation mechanism involving these two genes in the control of cell cycle progression and protein synthesis.|
| [CRTAP](https://www.ncbi.nlm.nih.gov/gene?term=CRTAP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both CRTAP and RPN2 are involved in protein processing and modification. CRTAP is necessary for efficient 3-hydroxylation of fibrillar collagen prolyl residues, while RPN2 is a subunit of the oligosaccharyl transferase (OST) complex that catalyzes protein N-glycosylation. It is possible that these two genes can function together in the same pathways or processes related to protein modification and quality control.|
| [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.8</br>**STRING score:** 932</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | PARP1 and PCNA both play key roles in DNA damage response and repair pathways. They interact directly via a protein interaction network, suggesting they may work together to coordinate DNA repair processes.|
| [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYO10](https://www.ncbi.nlm.nih.gov/gene?term=MYO10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.8</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between CTTN and MYO10 could involve regulating the actin cytoskeleton, cell shape, and cell migration, as well as playing a role in intracellular protein transport and endocytosis. The shortest path between CTTN and MYO10 in the BioGRID protein interaction network suggests that they may interact through other proteins such as ACTR2, ARPC2, HDAC6, BTRC, or CUL3.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.2</br>**STRING score:** 593</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 186 (min len 3) | HDAC2 and HDAC6 both have protein deacetylase activity, but they are involved in different biological processes. HDAC2 is mainly involved in regulating gene expression through histone deacetylation, while HDAC6 has a role in microtubule-dependent cell motility, autophagy, and protein degradation. Based on their different functions, it is not immediately clear what their functional association would be, if any.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLA2G15](https://www.ncbi.nlm.nih.gov/gene?term=PLA2G15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.4</br>**STRING score:** 304</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between HSPD1 and PLA2G15 include involvement in mitochondrial protein import and chaperone-mediated protein folding, as well as potential roles in glycerophospholipid homeostasis and remodeling. However, without further experimental evidence, the exact nature of any functional associations between these genes remains unclear.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.6</br>**STRING score:** 362</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between ECH1 and PSMD2 may involve the regulation of protein degradation via the 26S proteasome, due to PSMD2’s role in this pathway and the identified shortest path between the two genes in the BioGRID protein interaction network. ECH1 may also be involved in fatty acid metabolism by converting 3-trans,5-cis-dienoyl-CoA to 2-trans,4-trans-dienoyl-CoA.|
| [HERC6](https://www.ncbi.nlm.nih.gov/gene?term=HERC6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB8](https://www.ncbi.nlm.nih.gov/gene?term=PSMB8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.0</br>**STRING score:** 268</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 141 (min len 4) | Possible functional associations between HERC6 and PSMB8 could be in the process of antigen processing and presentation, where PSMB8 plays a role as a proteasome subunit involved in antigen processing and HERC6 may regulate protein degradation through its E3 ubiquitin ligase activity. They could also potentially have a role in the inflammatory response pathway, as PSMB8 is involved in it, while HERC6 has been shown to interact with influenza A virus nucleoprotein and with RNA-directed RNA polymerase subunits PB1 and PB2.|
