| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EIF4A3](https://www.ncbi.nlm.nih.gov/gene?term=EIF4A3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional association: EIF4A3 is involved in pre-mRNA splicing and assembly of the exon junction complex, while PAFAH1B3 is a subunit of cytosolic type I platelet-activating factor (PAF) acetylhydrolase. There is no known biological process linking the two genes.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRR7](https://www.ncbi.nlm.nih.gov/gene?term=PRR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 101 (min len 4) | Possible functional associations: CDK6 is involved in cell cycle control and proliferation, while PRR7 is involved in synapse-to-nucleus signaling and regulation of transcription factor JUN. These genes may have a potential role in regulating cell differentiation and proliferation processes in neuronal cells. However, without further experimental evidence, the exact nature and extent of their functional association remains unclear.|
| [CHIC2](https://www.ncbi.nlm.nih.gov/gene?term=CHIC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between CHIC2 and RPS5 in specific biological terms. CHIC2 is a regulatory component of the cyclin D1-CDK4 complex, important for cell cycle regulation, while RPS5 is a component of the small ribosomal subunit, involved in protein synthesis.|
| [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.7</br>**STRING score:** 369</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between CXCR4 and FOXO3 is their involvement in apoptosis. CXCR4 acts as a promoter of virus-mediated fusion and interacts with Staphylococcus aureus protein SSL10, while FOXO3 acts as a transcriptional activator that regulates autophagy, participates in post-transcriptional regulation of MYC, and promotes mtDNA transcription.|
| [MVP](https://www.ncbi.nlm.nih.gov/gene?term=MVP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAD51C](https://www.ncbi.nlm.nih.gov/gene?term=RAD51C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between MVP and RAD51C are in the homologous recombination repair (HRR) pathway of DNA repair. MVP may play a role in nucleo-cytoplasmic transport and down-regulates IFNG-mediated STAT1 signaling and subsequent activation of JAK while RAD51C is essential for HRR pathway of double-stranded DNA breaks arising during DNA replication or induced by DNA-damaging agents. The BioGRID protein interaction network shows a shortest path between MVP and RAD51C via KRAS.|
| [CSRP1](https://www.ncbi.nlm.nih.gov/gene?term=CSRP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DECR1](https://www.ncbi.nlm.nih.gov/gene?term=DECR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CSRP1 and DECR1 could be related to regulation of energy metabolism, particularly linked to beta-oxidation of fatty acids in mitochondria. Additionally, CSRP1's role in neuronal development and interaction with ASCC1, ASCC2, and TRIP4 could suggest a regulatory function in DECR1 expression.|
| [ERBB3](https://www.ncbi.nlm.nih.gov/gene?term=ERBB3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GALE](https://www.ncbi.nlm.nih.gov/gene?term=GALE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear biological association between ERBB3 and GALE. ERBB3 encodes a tyrosine-protein kinase that plays a role in cell surface receptor signaling, while GALE encodes an enzyme involved in the Leloir pathway of galactose catabolism and glycoprotein/lipid synthesis.|
| [ADCK3](https://www.ncbi.nlm.nih.gov/gene?term=ADCK3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ITGAE](https://www.ncbi.nlm.nih.gov/gene?term=ITGAE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 122 (min len 4) | Possible functional associations: ADCK3 is involved in the biosynthesis of coenzyme Q, while ITGAE is a receptor for E-cadherin. There is no clear biological association between these two genes based on their functions. However, according to the protein interaction network, there could be some indirect interactions through the intermediate proteins such as IRAK1, EGFR, SH3GLB1, TPD52L2, and ARHGEF16.|
| [GRB7](https://www.ncbi.nlm.nih.gov/gene?term=GRB7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | GRB7 and ICAM1 interact in the BioGRID protein interaction network via several intermediate proteins. GRB7 acts as an adapter protein that modulates signaling from receptor kinases and regulates cell proliferation and migration. ICAM1 is involved in immune response and is targeted by various pathogens. There is no clear functional association between these genes based on current knowledge.|
| [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUCB2](https://www.ncbi.nlm.nih.gov/gene?term=NUCB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between FOXO3 and NUCB2 may include regulation of metabolism and energy homeostasis, as both genes have been shown to play a role in these processes. FOXO3 is a transcriptional activator that regulates autophagy and mitochondrial transcription, while NUCB2 is an anorexigenic peptide that regulates food intake and blood pressure. However, there is no evidence of a direct interaction or pathway between these two genes.|
| [FHL2](https://www.ncbi.nlm.nih.gov/gene?term=FHL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCYL3](https://www.ncbi.nlm.nih.gov/gene?term=SCYL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between FHL2 and SCYL3 are currently unknown. FHL2 negatively regulates transcriptional repressors and apoptotic functions, while SCYL3 may play a role in regulating cell adhesion and migration. There is no clear biological process that connects the functions of these two genes.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VDAC1](https://www.ncbi.nlm.nih.gov/gene?term=VDAC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.0</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | CFLAR is an apoptosis regulator protein that can reduce TNFRSF-triggered apoptosis, while VDAC1 forms a channel through the mitochondrial outer membrane involved in cell volume regulation and apoptosis. The shortest path between them in the BioGRID protein interaction network suggests possible functional associations for apoptosis regulation and the permeability transition pore complex formation.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.5</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ATF6 and CCNB1 could involve ATF6's role in activating transcription during endoplasmic reticulum (ER) stress and CCNB1's essential function in controlling the cell cycle at the G2/M transition. This could suggest a potential relationship between ER stress response and cell cycle control in TCGA-COAD.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KTN1](https://www.ncbi.nlm.nih.gov/gene?term=KTN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HSPA8 and KTN1 may be involved in kinesin-driven vesicle motility and integrin-based adhesion complexes. HSPA8 is a molecular chaperone involved in protein quality control, mitochondrial import, and autophagy. KTN1 is a receptor for kinesin and forms parallel homodimers in both membrane-bound and cytosolic forms. There may be a role for HSPA8 in regulating KTN1 function in these processes.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIPARP](https://www.ncbi.nlm.nih.gov/gene?term=TIPARP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.9</br>**STRING score:** 150</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known direct functional association between FKBP4 and TIPARP. FKBP4 is involved in regulating neuronal growth cones and microtubule dynamics, as well as playing a role in the intracellular trafficking of steroid hormone receptors. TIPARP functions as a negative regulator of AHR and mediates mono-ADP-ribosylation of glutamate, aspartate, and cysteine residues on target proteins.|
| [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.6</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear functional association between HNRNPA2B1 and PTPRC. HNRNPA2B1 is involved in RNA transport and splicing, while PTPRC acts as a receptor for human cytomegalovirus protein UL11. The shortest path between these two genes in the BioGRID protein interaction network involves intermediate proteins that are not biologically connected to HNRNPA2B1 or PTPRC.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SORBS3](https://www.ncbi.nlm.nih.gov/gene?term=SORBS3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.1</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | HDAC2 and SORBS3 have a shortest path in the BioGRID protein interaction network, indicating a possible functional association between them. HDAC2 is a histone deacetylase involved in transcriptional regulation and chromatin remodeling, while SORBS3 is a protein involved in actin stress fiber formation and cell spreading. The association between the two genes may be related to the regulation of gene expression or cell morphology.|
| [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 197</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | CXCR4 and NFKBIB are involved in protein-protein interactions and are both related to immune response and infection. It is possible that CXCR4's role as a coreceptor for viruses and its interaction with Staphylococcus aureus protein SSL10 could have a functional association with NFKBIB's role in inhibiting NF-kappa-B and its interaction with NKIRAS1/2.|
| [ERBB2](https://www.ncbi.nlm.nih.gov/gene?term=ERBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.6</br>**STRING score:** 565</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 179 (min len 3) | There is no strong indication of a direct functional association between ERBB2 and HSPA8. ERBB2 is involved in transcriptional regulation, enhances protein synthesis and cell growth, and is part of protein complexes with other factors involved in signaling and nuclear transport. HSPA8 is a molecular chaperone with multiple roles in protein quality control, mitochondrial import, transcriptional repression, and autophagy. The shortest path between ERBB2 and HSPA8 in the BioGRID protein interaction network involves several intermediate proteins.|
