| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF3E](https://www.ncbi.nlm.nih.gov/gene?term=EIF3E[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.7</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between CPNE3 and EIF3E include regulation of translation initiation and mRNA decay, with CPNE3 potentially modulating EIF3E activity. CPNE3 may also play a role in ERBB2-mediated tumor cell migration, while EIF3E is part of the eukaryotic translation initiation factor 3 complex required for several steps in protein synthesis.|
| [CDK4](https://www.ncbi.nlm.nih.gov/gene?term=CDK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 37.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CDK4 and PAFAH1B3 may involve the regulation of cell cycle progression and development of brain, as CDK4 is a Ser/Thr-kinase component involved in G1/S transition and PAFAH1B3 is a catalytic subunit of a heterotetrameric enzyme involved in brain development. They may interact through a protein interaction network and play a role in modulating each other's enzymatic activity.|
| [CD58](https://www.ncbi.nlm.nih.gov/gene?term=CD58[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CD58 is involved in T-lymphocyte interactions with cells and could potentially prime response by both CD2+ and LFA-3+ cells. CFLAR acts as an inhibitor of TNFRSF6-mediated apoptosis. There is no known functional association between CD58 and CFLAR in specific biological terms.|
| [AKR7A2](https://www.ncbi.nlm.nih.gov/gene?term=AKR7A2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 190 (min len 4) | There is no clear functional association between AKR7A2 and DENND2D based on the given information. AKR7A2 catalyzes the reduction of succinic semialdehyde to gamma-hydroxybutyrate, while DENND2D functions as a guanine nucleotide exchange factor for RAB9A and RAB9B. Possible connections could be explored through additional research into shared biological pathways or interaction partners.|
| [FOSL1](https://www.ncbi.nlm.nih.gov/gene?term=FOSL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT1](https://www.ncbi.nlm.nih.gov/gene?term=STAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.0</br>**STRING score:** 321</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association: FOSL1 and STAT1 may interact in DNA damage tolerance and repair processes, as FOSL1 is involved in regulating translesion synthesis and STAT1 is activated in response to cellular stress signals and cytokines. They may also be involved in antiviral responses, as STAT1 mediates cellular responses to interferons and FOSL1 interacts with the BAF chromatin-remodeling complex. Further research is needed to determine the extent and specific nature of their functional associations.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYPL1](https://www.ncbi.nlm.nih.gov/gene?term=SYPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 47 (min len 4) | There is no apparent functional association between DHDDS and SYPL1 based on their known biological processes and interactions. DHDDS is involved in the biosynthesis of dolichol phosphate, while SYPL1 is responsible for the oxidation of protoporphyrin-IX.|
| [CD58](https://www.ncbi.nlm.nih.gov/gene?term=CD58[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLR4](https://www.ncbi.nlm.nih.gov/gene?term=TLR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.4</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 115 (min len 4) | Possible functional associations: CD58 has been shown to interact with TLR4 through intermediate proteins such as HSPA8 and DNAJA1. CD58 is involved in T-lymphocyte interactions with target cells, while TLR4 is involved in the innate immune response to bacterial lipopolysaccharide and in some cases, inflammation triggered by free fatty acids. There may be a functional association between CD58 and TLR4 in immune-related processes.|
| [CEBPZ](https://www.ncbi.nlm.nih.gov/gene?term=CEBPZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUSAP1](https://www.ncbi.nlm.nih.gov/gene?term=NUSAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | CEBPZ and NUSAP1 may have a functional association in mitotic spindle organization and microtubule stabilization during cell division. CEBPZ stimulates transcription and interacts with proteins involved in the regulation of cell cycle, while NUSAP1 is a microtubule-associated protein that promotes microtubule bundling and stability. The shortest path revealed by BioGRID interaction network suggests multiple direct interactions between these two genes.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.8</br>**STRING score:** 990</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between AURKA and UBE2C is their role in mitosis. AURKA is a mitotic kinase that plays a critical role in various mitotic events, including centrosome duplication, separation, chromosomal alignment, and cytokinesis, while UBE2C acts as an essential factor of the anaphase promoting complex/cyclosome (APC/C), a cell cycle-regulated ubiquitin ligase that controls progression through mitosis. AURKA can also phosphorylate UBE2C to promote the degradation of APC/C substrates by the proteasome.|
| [FPGS](https://www.ncbi.nlm.nih.gov/gene?term=FPGS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | FPGS catalyzes folate conversion to polyglutamate derivatives important for purine, pyrimidine, and amino acid synthesis while SYK regulates immune response, cell signaling, and platelet activation. The shortest protein interaction path between the two genes involves EGFR, suggesting possible involvement in downstream pathways downstream of receptor tyrosine kinases.|
| [RPA3](https://www.ncbi.nlm.nih.gov/gene?term=RPA3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.3</br>**STRING score:** 761</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between RPA3 and TIMELESS include their roles in DNA replication, maintenance of genome stability, and DNA repair. RPA3 functions as a part of the heterotrimeric replication protein A complex, which binds and stabilizes single-stranded DNA intermediates during DNA replication or stress, while TIMELESS plays an important role in the control of DNA replication, maintenance of replication fork stability, and cell survival after DNA damage or replication stress. Both proteins may be involved in the ATR-CHEK1 pathway in the replication checkpoint induced by hydroxyurea or ultraviolet light, and TIMELESS may promote homologous recombination repair via its interaction with PARP1.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.2</br>**STRING score:** 207</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 57 (min len 4) | Possible functional associations: ELOVL6 is involved in fatty acid synthesis and elongation, while LSM6 plays a role in pre-mRNA splicing and RNA processing. There is no clear biological process linking these two genes, although they do interact via the EGFR protein.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.8</br>**STRING score:** 363</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between CDK6 and CREB1 could be related to cell cycle control and transcription regulation. CDK6 controls G1/S transition and promotes cell proliferation, while CREB1 is a transcription factor that stimulates transcription upon binding to the DNA cAMP response element. There could be a possible involvement of these two genes in the regulation of cell proliferation through transcriptional control.|
| [BUB1B](https://www.ncbi.nlm.nih.gov/gene?term=BUB1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.7</br>**STRING score:** 999</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional associations between BUB1B and CDK1 include their involvement in mitotic checkpoint regulation and cell cycle control. BUB1B inhibits the activity of APC/C by blocking the binding of CDC20 to APC/C, while CDK1 forms a complex with cyclin B to regulate mitotic entry. The interaction between these two genes may contribute to proper chromosome segregation during mitosis.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.4</br>**STRING score:** 989</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between CCNA2 and CCNB2 include their role in controlling the cell cycle at different stages by forming serine/threonine kinase holoenzyme complexes with CDK1/CDK2 and conferring substrate specificity. CCNA2 and CCNB2 interact with each other through CDK1, suggesting they may work together in regulating the cell cycle transitions.|
| [CEBPZ](https://www.ncbi.nlm.nih.gov/gene?term=CEBPZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR1C](https://www.ncbi.nlm.nih.gov/gene?term=POLR1C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.3</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | There is no clear functional association between CEBPZ and POLR1C based on their known biology. CEBPZ stimulates transcription from the HSP70 promoter and interacts with a variety of proteins involved in different cellular processes, while POLR1C is a common component of RNA polymerase I and III complexes involved in synthesizing ribosomal RNA precursors and small RNAs.|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CSNK2A2](https://www.ncbi.nlm.nih.gov/gene?term=CSNK2A2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between BCL7B and CSNK2A2, as they are not known to directly interact. BCL7B is involved in apoptosis regulation and Wnt signaling, while CSNK2A2 is a constitutive kinase involved in numerous cellular processes, including transcription regulation and viral infection.|
| [ASH2L](https://www.ncbi.nlm.nih.gov/gene?term=ASH2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.9</br>**STRING score:** 153</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional association: ASH2L and CDK6 may be involved in regulating transcription and cell cycle progression. They both have interactions with other proteins involved in these processes, such as MLL complexes (ASH2L) and cyclins (CDK6).      ASH2L is a transcriptional regulator and a component of various histone methyltransferase complexes that regulate transcription through recruitment of those complexes to gene promoters. CDK6 is a serine/threonine-protein kinase involved in the control of the cell cycle and differentiation, promoting the G1/S transition. Both genes have interactions with proteins involved in transcription and cell cycle regulation, suggesting possible functional associations in these processes.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.4</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between DHDDS and SMC4 include involvement in chromatin condensation during mitosis and protein glycosylation in the endoplasmic reticulum. DHDDS synthesizes precursors for sugar carriers used in protein glycosylation, while SMC4 is a central component of the condensin complex involved in chromatin condensation during mitosis. It is unclear if there is a direct functional association between these two genes in specific biological terms.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.3</br>**STRING score:** 926</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | ORC1 and TFDP1 are functionally associated in the BioGRID protein interaction network, with a shortest path indicating that they are involved in the regulation of cell cycle-dependent genes. ORC1 serves a crucial role in the urea cycle by connecting the cytosolic and intramitochondrial reactions, while TFDP1 stimulates E2F-dependent transcription and can interact with RB family members to inhibit E2F transactivation.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 774</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CCNB2 and ORC1 are unlikely, as CCNB2 is involved in the control of the cell cycle during mitosis, while ORC1 plays a crucial role in the urea cycle and in forming a heterohexamer structure with TIMM9 and TIMM10/TIM10A. These two genes have distinct functions in different biological processes.|
| [ISOC1](https://www.ncbi.nlm.nih.gov/gene?term=ISOC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM5](https://www.ncbi.nlm.nih.gov/gene?term=LSM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 427</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 501 (min len 4) | Possible functional associations between ISOC1 and LSM5 could involve pre-mRNA splicing and inhibition of metalloproteinases. ISOC1 complexes with metalloproteinases and inactivates them, while LSM5 plays a role in pre-mRNA splicing as a component of the U4/U6-U5 tri-snRNP complex. However, without further experimental evidence, the specific biological processes linking these genes cannot be determined.|
| [CDCA4](https://www.ncbi.nlm.nih.gov/gene?term=CDCA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | CDCA4 and TIMELESS are both involved in DNA replication and maintenance of genome stability. There is a direct interaction between the two proteins via TRIM28 and PPP2R1A, suggesting a possible functional association in ensuring proper regulation of DNA replication.|
