| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF3E](https://www.ncbi.nlm.nih.gov/gene?term=EIF3E[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.7</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations: CPNE3 may play a role in tumor cell migration in response to growth factor stimulation, while EIF3E is involved in translation initiation and mRNA decay. There is currently no known direct biological association between these two genes.|
| [CDK4](https://www.ncbi.nlm.nih.gov/gene?term=CDK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 37.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CDK4 and PAFAH1B3 may involve regulating cell cycle and growth. CDK4 is a serine/threonine kinase that plays a role in G1/S transition by phosphorylating and inhibiting retinoblastoma protein family members. PAFAH1B3 is a catalytic subunit of the cytosolic type I platelet-activating factor acetylhydrolase that plays a role in modulating PAF activity. There may not be a direct biological relationship between these two genes, but their interaction through protein networks suggests a possible association.|
| [CD58](https://www.ncbi.nlm.nih.gov/gene?term=CD58[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CFLAR may regulate CD58-mediated T-lymphocyte interactions. CD58 is also involved in immune response to microbial infection, including human cytomegalovirus.   CD58 and CFLAR may have a role in regulating apoptosis pathways, including interactions with TNFRSF6. CFLAR may inhibit TNFRSF6-mediated apoptosis and regulate caspase-8 recruitment, while CD58 may prime response by both CD2+ and LFA-3+ cells and interact with cytomegalovirus protein UL148.|
| [AKR7A2](https://www.ncbi.nlm.nih.gov/gene?term=AKR7A2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 190 (min len 4) | Possible functional association: DENND2D, a GEF, may activate RAB proteins and thus contribute to intracellular protein trafficking. AKR7A2 could protect the liver from the toxic and carcinogenic effects of hepatocarcinogens like AFB1. In the context of colorectal cancer (TCGA-COAD), AKR7A2 and DENND2D are not known to be directly associated.|
| [FOSL1](https://www.ncbi.nlm.nih.gov/gene?term=FOSL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT1](https://www.ncbi.nlm.nih.gov/gene?term=STAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.0</br>**STRING score:** 321</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | FOSL1 and STAT1 are functionally associated in DNA repair and transcriptional regulation. FOSL1 is a component of the DNA polymerase delta complex involved in DNA repair and translesion synthesis, while STAT1 is a transcription factor that mediates cellular responses to cytokines and growth factors. The shortest path between them in the BioGRID protein interaction network suggests they may physically interact indirectly via JUN, HDAC1, CREBBP, EP300, and PSMC3.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYPL1](https://www.ncbi.nlm.nih.gov/gene?term=SYPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 47 (min len 4) | Possible functional associations between DHDDS and SYPL1 are unclear based on the provided information. DHDDS is involved in the biosynthesis of dolichol phosphate, which is utilized as a sugar carrier in protein glycosylation in the endoplasmic reticulum. SYPL1 catalyzes the oxidation of protoporphyrinogen-IX to form protoporphyrin-IX monomer.|
| [CD58](https://www.ncbi.nlm.nih.gov/gene?term=CD58[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLR4](https://www.ncbi.nlm.nih.gov/gene?term=TLR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.4</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 115 (min len 4) | CD58 and TLR4 may have a functional association in the innate immune response to bacterial lipopolysaccharide (LPS), as TLR4 is involved in mediating this response and CD58 may prime T lymphocytes for response to antigen-presenting cells. Both genes are also involved in microbial infection responses.|
| [CEBPZ](https://www.ncbi.nlm.nih.gov/gene?term=CEBPZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUSAP1](https://www.ncbi.nlm.nih.gov/gene?term=NUSAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between CEBPZ and NUSAP1 could be in the regulation of mitosis and microtubule organization during cell division. CEBPZ stimulates transcription and interacts with proteins involved in cell signaling and DNA binding, while NUSAP1 is a microtubule-associated protein that bundles and stabilizes microtubules.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.8</br>**STRING score:** 990</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and UBE2C have a functional association in the context of mitosis and cell cycle progression. AURKA is a kinase that regulates key mitotic events, while UBE2C acts as an essential factor of the anaphase promoting complex/cyclosome (APC/C), promoting mitotic exit through substrate degradation via ubiquitination. AURKA has been shown to directly interact with some components of the APC/C complex, suggesting a possible functional link between these two genes in regulating mitotic progression.|
| [FPGS](https://www.ncbi.nlm.nih.gov/gene?term=FPGS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYK](https://www.ncbi.nlm.nih.gov/gene?term=SYK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on the provided information, there does not seem to be a clear functional association between FPGS and SYK. FPGS is involved in folate conversion and MTX metabolism, while SYK mediates signal transduction downstream of various receptors and regulates immune responses and cell processes. The shortest pathway between the two genes in the BioGRID interaction network involves EGFR.|
| [RPA3](https://www.ncbi.nlm.nih.gov/gene?term=RPA3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.3</br>**STRING score:** 761</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: RPA3 and TIMELESS are involved in DNA replication, maintenance of replication fork stability, and DNA repair. RPA3 plays a role in the recruitment of repair factors to DNA damage sites, while TIMELESS stabilizes replication forks and influences both CHEK1 phosphorylation and the intra-S phase checkpoint in response to genotoxic stress. They may interact in the context of the DNA damage response and/or replication checkpoint pathway.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.2</br>**STRING score:** 207</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 57 (min len 4) | Possible functional association: ELOVL6 catalyzes the elongation of long-chain fatty acids while LSM6 plays a role in pre-mRNA splicing. While no direct interaction has been reported, both genes are involved in RNA processing and could potentially be involved in the regulation of lipid metabolism through post-transcriptional mechanisms.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.8</br>**STRING score:** 363</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between CDK6 and CREB1 may include regulation of cell cycle progression and transcriptional activation. CDK6 is involved in the control of cell cycle and differentiation, promoting G1/S transition and phosphorylating pRB/RB1. CREB1 is a phosphorylation-dependent transcription factor that stimulates transcription upon binding to the DNA cAMP response element (CRE) and is involved in different cellular processes, including differentiation. The shortest paths between CDK6 and CREB1 in the BioGRID protein interaction network suggest potential physical interactions between these two proteins through intermediaries such as SIRT1, POGZ, and AR.|
| [BUB1B](https://www.ncbi.nlm.nih.gov/gene?term=BUB1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.7</br>**STRING score:** 999</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional association: BUB1B is an essential component of the mitotic checkpoint, required for normal mitosis progression, while CDK1 is involved in the regulation of the cell cycle and acts as a receptor for hepatitis C virus (HCV) in hepatocytes. BUB1B negatively regulates PLK1 activity in interphase cells and suppresses centrosome amplification, while CDK1 inhibits the formation and nuclear import of the cyclin B1-CDK1 complex and leads to cell cycle arrest. There could be a functional association between these two genes in the regulation of the cell cycle and in their involvement in viral infections.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.4</br>**STRING score:** 989</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | These two genes, CCNA2 and CCNB2, are likely functionally associated through their role in regulating the cell cycle, specifically at the G2/M transition. CCNA2 forms complexes with cyclin-dependent protein kinases CDK1 and CDK2, while CCNB2 interacts with CDK1 to form maturation promoting factor (MPF) and control substrate specificity.|
| [CEBPZ](https://www.ncbi.nlm.nih.gov/gene?term=CEBPZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR1C](https://www.ncbi.nlm.nih.gov/gene?term=POLR1C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.3</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | CEBPZ and POLR1C have a shortest path of protein interaction in the bioGRID network and belong to different complexes, RNA polymerases I and III, that synthesize rRNA precursors and small RNAs, respectively. There are no known direct functional associations between these genes in specific biological processes, but both genes are involved in transcriptional regulation and RNA synthesis.|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CSNK2A2](https://www.ncbi.nlm.nih.gov/gene?term=CSNK2A2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no obvious functional association between BCL7B and CSNK2A2. BCL7B is a positive regulator of apoptosis and Wnt signaling, while CSNK2A2 is a serine/threonine-protein kinase that phosphorylates various substrates involved in cellular processes like cell cycle progression, transcription, and viral infection.|
| [ASH2L](https://www.ncbi.nlm.nih.gov/gene?term=ASH2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.9</br>**STRING score:** 153</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | ASH2L is a transcriptional regulator involved in methylation and dimethylation of histone H3, whereas CDK6 is a serine/threonine-protein kinase involved in the control of the cell cycle and differentiation. Possible functional associations between these two genes could be in regulating cell proliferation and differentiation, as CDK6 can phosphorylate pRB/RB1 and NPM1, and ASH2L is involved in methylation and dimethylation that affects gene expression regulation. The shortest path between ASH2L and CDK6 in the BioGRID protein interaction network includes several histone methyltransferase complexes, suggesting possible functional interactions between ASH2L and CDK6 in regulating chromatin structure and gene expression.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.4</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between DHDDS and SMC4 could involve regulation of chromatin condensation and potentially even cell division, as SMC4 is a component of the condensin complex involved in these processes and DHDDS interacts with NPC2, which promotes cholesterol trafficking and has been implicated in cell cycle progression. DHDDS also plays a role in protein glycosylation and synthesizes long-chain polyprenols, while SMC4 is a central component of the condensin complex.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.3</br>**STRING score:** 926</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | ORC1 and TFDP1 are functionally associated as they participate in DNA replication and cell cycle regulation, respectively. ORC1 is a component of the origin recognition complex and is required to assemble the pre-replication complex necessary to initiate DNA replication. TFDP1 stimulates E2F-dependent transcription and forms heterodimers with E2F family members, which are involved in regulating cell cycle progression.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 774</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CCNB2 and ORC1: CCNB2 interacts with CDK1 during G2/M transition and ORC1 is required for the assembly of pre-replication complex necessary for DNA replication. This suggests that CCNB2 and ORC1 may have a role in coordinating cell cycle progression and DNA replication.|
| [ISOC1](https://www.ncbi.nlm.nih.gov/gene?term=ISOC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM5](https://www.ncbi.nlm.nih.gov/gene?term=LSM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 427</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 501 (min len 4) | Possible functional association between ISOC1 and LSM5: Both genes are involved in pre-mRNA splicing and are part of the U4/U6-U5 tri-snRNP complex. ISOC1 also interacts with metalloproteinases and irreversibly inactivates them.|
| [CDCA4](https://www.ncbi.nlm.nih.gov/gene?term=CDCA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Both CDCA4 and TIMELESS are involved in the regulation of DNA replication and genome stability. They may interact through the stabilization of replication forks and the control of DNA synthesis, possibly playing a role in the ATR-CHEK1 pathway.|
