| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BMP4](https://www.ncbi.nlm.nih.gov/gene?term=BMP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between BMP4 and CPNE3 is their involvement in tumor cell migration in response to growth factor stimulation. BMP4 is a growth factor that activates the canonical BMP signaling cascade, while CPNE3 is a calcium-dependent phospholipid-binding protein that interacts with ERBB2 and SHC1.|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between BCL7B and PSMD10 may include regulation of cell cycle progression, apoptosis, and the Wnt signaling pathway. PSMD10 may also indirectly regulate the nuclear export of the NF-kappa-B component RELA through its interaction with the protein. The shortest path in the BioGRID protein interaction network between these genes involves PSMD3 and PSMC5, both components of the 26S proteasome, suggesting a potential role in protein degradation pathways.|
| [PLOD3](https://www.ncbi.nlm.nih.gov/gene?term=PLOD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLD4](https://www.ncbi.nlm.nih.gov/gene?term=POLD4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 129 (min len 4) | There is no clear functional association between PLOD3 and POLD4 based on the provided context. PLOD3 is involved in collagen biosynthesis while POLD4 plays a role in DNA damage tolerance by regulating translesion synthesis.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | CCNB1 and PNKP have multiple interactions in the BioGRID protein interaction network, suggesting a potential functional association. CCNB1 plays a role in cell cycle control and interacts with PNKP during mitosis to form an endocytotic complex, while PNKP helps repair DNA damage. It is possible that PNKP interacts with CCNB1 to regulate the repair of DNA damage during cell division.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A13](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between AKR1A1 and SLC25A13 include their involvement in xenobiotic metabolism and mitochondrial function, respectively. Their proximity in the shortest path of the BioGRID protein interaction network suggests they may interact directly in these processes.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIPSNAP1](https://www.ncbi.nlm.nih.gov/gene?term=NIPSNAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CDK6 and NIPSNAP1 include their involvement in receptor-mediated endocytosis and regulation of membrane tubulation. CDK6 promotes G1/S transition and controls entrance into the cell cycle, while NIPSNAP1 may regulate slow endocytic recycling of endocytosed proteins back to the plasma membrane and is involved in neurite outgrowth.|
| [SNX11](https://www.ncbi.nlm.nih.gov/gene?term=SNX11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 4) | Possible functional association: SNX11 and TSPAN4 are connected in a protein interaction network and may be involved in protein sorting and membrane trafficking in endosomes, as well as regulating platelet activation and vascular development through signaling complexes with activated receptors.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 4) | Possible functional associations between DHDDS and TSPAN4 could involve regulation of protein glycosylation in the endoplasmic reticulum and activation of platelets by collagen, respectively. These two genes may also interact directly via the protein interaction network, potentially through HNRNPH1 or HNRNPL, to form complexes involved in specific biological processes.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WFS1](https://www.ncbi.nlm.nih.gov/gene?term=WFS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There are no clear functional associations between EED and WFS1 based on their known biological functions and protein interactions. EED is involved in transcriptional repression through histone modification and recognition, while WFS1 modulates calcium homeostasis and the ER stress response.|
| [TESK1](https://www.ncbi.nlm.nih.gov/gene?term=TESK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBR7](https://www.ncbi.nlm.nih.gov/gene?term=UBR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 148 (min len 4) | There is no clear indication of a direct functional association between TESK1 and UBR7. TESK1 regulates cytoskeleton dynamics and cell motility, while UBR7 is involved in protein degradation via the N-end rule pathway.|
| [BMP4](https://www.ncbi.nlm.nih.gov/gene?term=BMP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.9</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1408 (min len 4) | Possible functional associations between BMP4 and MYL9 could involve BMP4 regulating smooth muscle and non-muscle cell contractile activity via phosphorylation of MYL9. BMP4 also plays essential roles in neurogenesis, vascular development, angiogenesis, and osteogenesis, while MYL9 is implicated in cytokinesis, receptor capping, and cell locomotion.|
| [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.5</br>**STRING score:** 981</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 3) | Possible functional associations: PTK2 plays a role in cell adhesion, migration, growth, and survival through its interactions with various signaling molecules. RHOA is a small GTPase that regulates the actin cytoskeleton and cell adhesion, migration, and division. These genes may be functionally associated in the regulation of cell adhesion and migration.|
| [CPSF4](https://www.ncbi.nlm.nih.gov/gene?term=CPSF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.8</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 449 (min len 4) | Possible functional associations between CPSF4 and EML3 may involve regulating mRNA processing and mitotic spindle assembly, respectively. However, there is no known direct biological interaction between these two genes based on the provided context and protein interaction network.|
| [MRPS16](https://www.ncbi.nlm.nih.gov/gene?term=MRPS16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPS2](https://www.ncbi.nlm.nih.gov/gene?term=MRPS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.3</br>**STRING score:** 999</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | The two genes MRPS16 and MRPS2 are both involved in the formation and stability of the mitochondrial small ribosomal subunit. There may be a functional association between these two genes in regulating mitochondrial translation and protein synthesis. The BioGRID protein interaction network shows a direct physical interaction between MRPS16 and MRPS2.|
| [CSRP1](https://www.ncbi.nlm.nih.gov/gene?term=CSRP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | CSRP1 and PTK2 have been shown to interact multiple times in the BioGRID protein interaction network. CSRP1 has been shown to play a role in neuronal development and interact with ASCC1, ASCC2, and TRIP4, while PTK2 has been shown to have a variety of interactions with other proteins related to cell signaling, adhesion, and proliferation. It is possible that CSRP1 and PTK2 may be involved in similar pathways related to cellular signaling and proliferation.|
| [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.6</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between PNKP and RFC5 include their involvement in DNA repair pathways. PNKP plays a key role in the non-homologous end-joining (NHEJ) and base excision repair (BER) pathways, while RFC5 is an accessory protein required for the elongation of primed DNA templates by DNA polymerases delta and epsilon.|
