| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BMP4](https://www.ncbi.nlm.nih.gov/gene?term=BMP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CPNE3](https://www.ncbi.nlm.nih.gov/gene?term=CPNE3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: BMP4 is involved in developmental processes including neurogenesis and angiogenesis, while CPNE3 plays a role in ERBB2-mediated tumor cell migration. It is possible that BMP4 and CPNE3 are involved in regulating tumor angiogenesis in TCGA-COAD through ERBB2 signaling.|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear biological process linking BCL7B and PSMD10, but they have a protein interaction through PSMD3 and PSMC5 in the 26S proteasome complex. BCL7B is a regulator of apoptosis and Wnt signaling, while PSMD10 is involved in negative regulation of tumor suppressors RB1 and p53/TP53.|
| [PLOD3](https://www.ncbi.nlm.nih.gov/gene?term=PLOD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLD4](https://www.ncbi.nlm.nih.gov/gene?term=POLD4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 129 (min len 4) | There is no clear functional association between PLOD3 and POLD4. PLOD3 is involved in hydroxylation and glycosylation of Lys residues in procollagen, while POLD4 plays a role in high fidelity genome replication and repair as a component of the DNA polymerase delta complex.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | CCNB1 is involved in the control of the cell cycle and interacts with other proteins during mitosis, while PNKP plays a crucial role in the repair of DNA damage. The shortest path between them in the BioGRID protein interaction network suggests possible functional associations through protein interactions during mitosis and DNA repair.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A13](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between AKR1A1 and SLC25A13 include drug metabolism and mitochondrial transport, as AKR1A1 is involved in detoxification and activation of procarcinogens while SLC25A13 is a mitochondrial antiporter involved in the malate-aspartate shuttle. The shortest path in the BioGRID protein interaction network involves several intermediary proteins, indicating potential indirect interactions between AKR1A1 and SLC25A13.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIPSNAP1](https://www.ncbi.nlm.nih.gov/gene?term=NIPSNAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | CDK6 and NIPSNAP1 have several possible functional associations based on the BioGRID protein interaction network, including involvement in cell cycle progression and endocytic recycling. NIPSNAP1 may act as a downstream effector of Rab proteins on endosome membranes, while CDK6 is a critical regulator of the G1/S transition in the cell cycle and promotes the proliferation of specific cell types. It is possible that CDK6 and NIPSNAP1 may work together to regulate endocytic recycling or other membrane trafficking events during cell proliferation and differentiation.|
| [SNX11](https://www.ncbi.nlm.nih.gov/gene?term=SNX11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 4) | Possible functional associations between SNX11 and TSPAN4 include their involvement in membrane trafficking and signaling downstream of cell surface receptors, respectively. SNX11 is involved in protein sorting and membrane trafficking in endosomes, while TSPAN4 mediates signal transduction downstream of a variety of transmembrane receptors, including classical immunoreceptors like the B-cell receptor. There may be interplay between these two genes in regulating immune responses and cellular processes involving membrane trafficking.|
| [DHDDS](https://www.ncbi.nlm.nih.gov/gene?term=DHDDS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 4) | DHDDS is involved in the biosynthesis of dolichol phosphate, which is utilized as a sugar carrier in protein glycosylation. TSPAN4 is a non-receptor tyrosine kinase that mediates signal transduction downstream of transmembrane receptors. There is no clear biological connection between these two genes based on their known functions.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WFS1](https://www.ncbi.nlm.nih.gov/gene?term=WFS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear functional association between EED and WFS1 based on their known biological processes and protein interaction network. EED encodes a Polycomb group protein involved in histone methylation and transcriptional repression, while WFS1 plays a role in regulating cellular calcium homeostasis and the ER stress response.|
| [TESK1](https://www.ncbi.nlm.nih.gov/gene?term=TESK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBR7](https://www.ncbi.nlm.nih.gov/gene?term=UBR7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 148 (min len 4) | There is no known direct functional association between TESK1 and UBR7. TESK1 is a kinase involved in regulating the cytoskeleton, while UBR7 acts as an E3 ubiquitin-protein ligase in the N-end rule pathway. The shortest path between these two genes in the BioGRID protein interaction network involves several intermediary proteins.|
| [BMP4](https://www.ncbi.nlm.nih.gov/gene?term=BMP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.9</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1408 (min len 4) | BMP4 is a growth factor involved in many developmental processes and regulates canonical BMP signaling by phosphorylating SMAD1/5/8. MYL9 plays a role in regulating both smooth muscle and nonmuscle cell contractile activity via its phosphorylation. There is a possible functional association between these genes in regulating cell locomotion and cytokinesis.|
| [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.5</br>**STRING score:** 981</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 53 (min len 3) | Possible functional association between PTK2 and RHOA could be their involvement in cell migration and adhesion signaling pathways. PTK2 is a focal adhesion kinase that is activated by integrins and regulates cell adhesion, migration, and survival, while RHOA is a small GTPase that regulates actin cytoskeleton remodeling and cell motility. The shortest path between PTK2 and RHOA in the BioGRID protein interaction network suggests that they may interact through several shared protein partners, including TRIP6, EGFR, PTEN, CXCR4, and others.|
| [CPSF4](https://www.ncbi.nlm.nih.gov/gene?term=CPSF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.8</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 449 (min len 4) | There is no clear biological process connecting CPSF4 and EML3. CPSF4 is involved in pre-mRNA 3'-end formation and can interact with viral NS1 to prevent host mRNA export. EML3 is involved in regulating mitotic spindle assembly and chromosome separation.|
| [MRPS16](https://www.ncbi.nlm.nih.gov/gene?term=MRPS16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPS2](https://www.ncbi.nlm.nih.gov/gene?term=MRPS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.3</br>**STRING score:** 999</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | These two genes, MRPS16 and MRPS2, both encode for small ribosomal subunit components involved in mitochondrial translation. There is no clear biological reason for functional association between these genes beyond their shared role in ribosomal assembly.|
| [CSRP1](https://www.ncbi.nlm.nih.gov/gene?term=CSRP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | CSRP1 and PTK2 are involved in protein interaction networks. CSRP1 has been shown to play a role in neuronal development while PTK2 is involved in intracellular signaling. No specific functional associations between the two genes have been established, but they may interact through the protein interaction network in specific biological processes such as cell migration, cytoskeleton organization, and gene transcription.|
| [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.6</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations: PNKP is involved in DNA damage repair and can interact with XRCC4. RFC5 is involved in DNA polymerase elongation and interacts with PCNA and RAD17. There is no known direct functional association between these two genes based on the provided context.|
