| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAD9A](https://www.ncbi.nlm.nih.gov/gene?term=RAD9A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 13.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CYCS and RAD9A could involve DNA repair and apoptosis regulation. CYCS is involved in apoptosis by regulating mitochondrial membrane permeability and releasing cytochrome c into the cytosol. RAD9A is involved in long-patch base excision repair and checkpoint responses to DNA damage. Both genes have been found to interact with BCL2L1, which is also involved in regulating apoptosis.|
| [GDPD5](https://www.ncbi.nlm.nih.gov/gene?term=GDPD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTHFD2](https://www.ncbi.nlm.nih.gov/gene?term=MTHFD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: MTHFD2 and GDPD5 both interact with PRDX1, suggesting a potential role in redox regulation. GDPD5 also functions in glycerophosphodiester and glycosylphosphatidylinositol anchor cleavage, while MTHFD2 is involved in one-carbon metabolism and nucleotide biosynthesis.|
| [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are no known direct functional associations between PAFAH1B3 and TBPL1. PAFAH1B3 is involved in the hydrolysis of PAF and plays an important role in brain development, while TBPL1 is a component of a specialized transcription system for ribosomal protein genes.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 486 (min len 4) | Possible functional associations between CYCS and GMEB2 could involve the regulation of apoptosis and gene expression. CYCS plays a role in apoptosis by releasing from the mitochondria, and GMEB2 is a trans-acting factor that binds to gene promoters, including the TAT and transferrin receptor promoters.|
| [GFOD1](https://www.ncbi.nlm.nih.gov/gene?term=GFOD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between GFOD1 and NOLC1 include regulating mitochondrial respiration, nucleolar organization, and ribosomal processing and modification. GFOD1 has been shown to interact with PEX11G, COPB2, COPA, PEX19, and FIS1, while NOLC1 interacts with RNA polymerase I, DKC1/NAP57, NOP58, and fibrillarin. However, without further experimentation, the specific biological terms linking these two genes remain unclear.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | JMJD6 and PAK4 may have a functional association in the regulation of gene transcription, as JMJD6 interacts with BRD4 and PAK4 plays a role in cell-cycle progression and phosphorylation of the cell-cycle regulatory protein CDKN1A. The shortest path in the BioGRID protein interaction network connecting these two genes involves several other proteins, suggesting a complex regulatory network.|
| [CTNND1](https://www.ncbi.nlm.nih.gov/gene?term=CTNND1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | CTNND1 and CYCS may be functionally associated through their involvement in apoptosis. CTNND1 regulates cell-cell adhesion and downstream cytoskeletal dynamics, while CYCS plays a role in the release of cytochrome c into the cytosol, triggering the activation of caspases and accelerating apoptosis. The shortest path between these two genes in the BioGRID protein interaction network suggests potential associations with EGFR, VCP, MYC, EGLN3, and HSPA8.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional associations between IGF1R and RPA2 may involve DNA damage response and repair, as RPA2 is involved in stabilizing and activating proteins during DNA metabolism, while IGF1R interacts with TP53 and other proteins involved in DNA repair and damage.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPARD](https://www.ncbi.nlm.nih.gov/gene?term=PPARD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | One possible functional association between CYCS and PPARD is that they both interact with the retinoid X receptor and CRY1/CRY2 in a ligand-dependent manner. CYCS plays a role in apoptosis and can trigger the activation of caspases, while PPARD is a ligand-activated transcription factor that regulates energy metabolism and the beta-oxidation pathway of fatty acids.|
| [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAT2A](https://www.ncbi.nlm.nih.gov/gene?term=MAT2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between MAPK1IP1L and MAT2A include regulation of transcriptional activity and formation of S-adenosylmethionine, respectively. While they interact in the BioGRID protein interaction network, there is no clear biological process connecting them.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GATA2](https://www.ncbi.nlm.nih.gov/gene?term=GATA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.0</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between CDK6 and GATA2 might include regulation of cell cycle and differentiation, as CDK6 is involved in controlling the cell cycle and promoting proliferation of specific cell types, while GATA2 is a transcriptional activator that regulates gene expression in endothelial cells. These genes might interact in processes such as regulation of gene expression and control of cell differentiation.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLE1](https://www.ncbi.nlm.nih.gov/gene?term=TLE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 278</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their known functions, there are no clear possible functional associations between CDK1 and TLE1. CDK1 is involved in cell cycle regulation and HCV infection, while TLE1 is a transcriptional corepressor that regulates a variety of signaling pathways.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 628</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CYCS and SRC include involvement in apoptotic pathways and regulation of protein tyrosine kinase activity. They may interact in biological processes related to mitochondrial membrane permeability, caspase activation, and neurite elongation.|
| [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.2</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between COL4A1 and TOP2A based on the available information. COL4A1 is involved in anti-angiogenic activity, while TOP2A is a key enzyme that alters DNA topology by binding to double-stranded DNA molecules. The shortest path in the BioGRID protein interaction network suggests that VHL and KIAA1429 may act as intermediaries between COL4A1 and TOP2A.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFB3](https://www.ncbi.nlm.nih.gov/gene?term=TGFB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.1</br>**STRING score:** 365</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 190 (min len 4) | Possible functional associations between IGF1R and TGFB3 include regulation of cell differentiation and embryogenesis, particularly in processes such as secondary palate development. The shortest path in the BioGRID protein interaction network suggests that they can interact through intermediary proteins like ELAVL1 and CSK.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 868</br>**BioGRID count:** 8</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: NR3C1 and NRIP1 both play roles in modulating transcriptional activation and repression by nuclear hormone receptors, and have been shown to interact with each other. Additionally, both genes have been implicated in the regulation of the circadian clock and have roles in the development of various tissues.|
| [CCNF](https://www.ncbi.nlm.nih.gov/gene?term=CCNF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VPS72](https://www.ncbi.nlm.nih.gov/gene?term=VPS72[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between CCNF and VPS72 include involvement in the regulation of cell cycle progression and maintenance of genome stability, as well as protein-protein interactions within the NuA4 histone acetyltransferase complex and the SRCAP complex. CCNF is a substrate recognition component of a SCF E3 ubiquitin-protein ligase complex and mediates the ubiquitination and subsequent proteasomal degradation of target proteins, while VPS72 specifically chaperones H2AZ1 and deposits it into nucleosomes.|
| [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOSL1](https://www.ncbi.nlm.nih.gov/gene?term=FOSL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.9</br>**STRING score:** 768</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear biological association between COX8A and FOSL1. COX8A is a component of the mitochondrial electron transport chain while FOSL1 plays a role in DNA replication and repair. The shortest path between these genes in the BioGRID protein interaction network is COX8A-PDCD11-FOSL1.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.9</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Based on their protein interaction network, a possible functional association between IGF1R and PROS1 is their involvement in the regulation of coagulation and fibrinolysis. IGF1R is a receptor involved in insulin and IGF1 regulation, while PROS1 is a plasma protein involved in anticoagulation.|
| [ACOT9](https://www.ncbi.nlm.nih.gov/gene?term=ACOT9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.8</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | ACOT9 and PAFAH1B3 have a shortest path of protein interaction in the BioGRID network. ACOT9 encodes for an enzyme that hydrolyzes acyl-CoAs to free fatty acids and CoASH, while PAFAH1B3 encodes for a subunit of the cytosolic type I platelet-activating factor acetylhydrolase heterotetrameric enzyme that catalyzes the hydrolysis of the acetyl group of PAF and its analogs. There are no known direct biological processes linking these genes.|
| [CCNF](https://www.ncbi.nlm.nih.gov/gene?term=CCNF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the context and their known functions, there do not appear to be any direct functional associations between CCNF and SLC25A4. CCNF is a component of the SCF E3 ubiquitin-protein ligase complex involved in cell cycle regulation and genome stability, while SLC25A4 is an ADP:ATP antiporter involved in mitochondrial energy production and regulation of mitophagy.|
