| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAD9A](https://www.ncbi.nlm.nih.gov/gene?term=RAD9A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 13.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between CYCS and RAD9A in TCGA-METABRIC cancer could be related to the role of RAD9A in DNA repair after damage and CYCS in apoptosis induction following damage. CYCS could induce apoptosis in cells with damaged DNA that could not be properly repaired by RAD9A, leading to elimination of these cells.|
| [GDPD5](https://www.ncbi.nlm.nih.gov/gene?term=GDPD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTHFD2](https://www.ncbi.nlm.nih.gov/gene?term=MTHFD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations:  GDPD5 is involved in glycerophosphodiester cleavage and regulation of osmotic pressure, while MTHFD2 is involved in dehydrogenase activity. These genes may have a functional association in cellular metabolism or regulation of signaling pathways, potentially through their protein interaction via HSPA8. However, further investigation is necessary to determine the specific biological context in which they interact.|
| [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: None can be inferred based on available information.  PAFAH1B3 is the alpha1 catalytic subunit of the cytosolic type I platelet-activating factor (PAF) acetylhydrolase, which catalyzes hydrolysis of the acetyl group at the sn-2 position of PAF and its analogs during brain development. TBPL1 is part of a specialized transcription system that mediates transcription of most ribosomal proteins and may also mediate transcription of NF1.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 486 (min len 4) | Possible functional association between CYCS and GMEB2 is their interaction through NR2C2 in the BioGRID protein interaction network, which may play a role in regulating gene expression. CYCS is involved in apoptosis by triggering caspase activation through cytochrome c release, while GMEB2 is a transcription factor that binds to glucocorticoid modulatory elements and is necessary for parvovirus replication.|
| [GFOD1](https://www.ncbi.nlm.nih.gov/gene?term=GFOD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There may be a functional association between GFOD1 and NOLC1 in mitochondrial respiration and nucleolar processing. GFOD1 plays a role in mitochondrial respiration and organization, while NOLC1 is involved in connecting RNA polymerase I with enzymes responsible for ribosomal processing and modification in the nucleolus.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: JMJD6, a lysyl-hydroxylase and arginine demethylase, is involved in RNA splicing, histone modifications, and hematopoietic differentiation, while PAK4, a serine/threonine protein kinase, regulates cytoskeleton, cell motility, and cell survival. There is no clear indication of direct functional associations between these genes based on their known biological processes.|
| [CTNND1](https://www.ncbi.nlm.nih.gov/gene?term=CTNND1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between CTNND1 and CYCS in TCGA-METABRIC cancer type could involve apoptosis and regulation of gene transcription. CTNND1 regulates gene transcription while CYCS plays a role in apoptosis. The shortest path between these two genes in the BioGRID protein interaction network involves several intermediaries, which suggests indirect interactions between CTNND1 and CYCS.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional associations between IGF1R and RPA2 may be related to DNA damage response and repair, as both genes are involved in these processes. Specifically, IGF1R is implicated in regulating the cellular response to DNA damage, while RPA2 is a key component of DNA replication and repair machinery. The shortest protein interaction path between IGF1R and RPA2 involves other genes known to be involved in DNA damage and repair, suggesting a possible functional link between the two genes in these processes.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPARD](https://www.ncbi.nlm.nih.gov/gene?term=PPARD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CYCS and PPARD may include the regulation of energy metabolism in adipose tissues and peroxisomal beta-oxidation of fatty acids, as they both form heterodimers with the retinoid X receptor and interact with cryptochrome proteins in a ligand-dependent manner. However, there is no direct evidence of a functional interaction between these two genes based on the provided context. CYCS plays a role in apoptosis by releasing cytochrome c into the cytosol, while PPARD acts as a transcription activator for the acyl-CoA oxidase gene and decreases expression of NPC1L1 upon activation.|
| [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAT2A](https://www.ncbi.nlm.nih.gov/gene?term=MAT2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: MAT2A regulates the formation of the co-factor S-adenosylmethionine, which is involved in MAPK1IP1L-mediated transcriptional regulation and may influence cancer growth.|
| [CDK6](https://www.ncbi.nlm.nih.gov/gene?term=CDK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GATA2](https://www.ncbi.nlm.nih.gov/gene?term=GATA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.0</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional association: CDK6 phosphorylates NPM1 and GATA2 interacts with BRD3. Both genes are involved in transcriptional regulation and cell cycle control, suggesting a possible role in the proliferation and differentiation of hematopoietic cells. However, further experimental validation is needed to confirm this potential relationship.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLE1](https://www.ncbi.nlm.nih.gov/gene?term=TLE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 278</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between CDK1 and TLE1 include CDK1's role in regulating cell cycle progression and TLE1's role as a transcriptional corepressor, which could potentially interact with CDK1-regulated genes. There is no direct evidence of interaction between CDK1 and TLE1 in the provided information.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 628</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional association: Both CYCS and SRC are involved in apoptotic pathways and have been shown to interact in the BioGRID protein interaction network.   Gene functions: CYCS plays a critical role in releasing cytochrome c from the mitochondria, which triggers apoptotic pathways. SRC is a non-receptor protein tyrosine kinase that is involved in various signaling pathways and has been shown to play a role in neurite elongation.|
| [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.2</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no known direct functional associations between COL4A1 and TOP2A, but they are found to be indirectly connected in the BioGRID protein interaction network. COL4A1 is involved in inhibiting angiogenesis and tumor formation, whereas TOP2A is a decatenating enzyme that alters DNA topology.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFB3](https://www.ncbi.nlm.nih.gov/gene?term=TGFB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.1</br>**STRING score:** 365</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 190 (min len 4) | Possible functional associations between IGF1R and TGFB3 include their interaction in the BioGRID protein network via ELAVL1 and their roles in regulating cell differentiation and embryogenesis. IGF1R binds IGF1 and is involved in hybrid receptor formation with INSR, while TGFB3 is required in various processes such as secondary palate development and is regulated by LAP and integrins.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 868</br>**BioGRID count:** 8</br>**BioGRID shortest paths:** 1 (min len 2) | NR3C1 and NRIP1 are known to interact and may have a functional association in modulating transcriptional activation and repression by nuclear hormone receptors and in regulating the circadian clock gene expression. They interact with several other proteins involved in various biological processes such as chaperones, coactivator complexes, and transcription factors.|
| [CCNF](https://www.ncbi.nlm.nih.gov/gene?term=CCNF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VPS72](https://www.ncbi.nlm.nih.gov/gene?term=VPS72[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional association between CCNF and VPS72 is involvement in transcriptional regulation through chromatin remodeling, as both genes are components of protein complexes involved in this process (SCF(CCNF) and NuA4 complex containing VPS72). CCNF is also involved in the regulation of cell cycle progression and genome stability through ubiquitination and degradation of target proteins, while VPS72 specifically chaperones and deposits histone H2AZ1 into nucleosomes.|
| [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOSL1](https://www.ncbi.nlm.nih.gov/gene?term=FOSL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.9</br>**STRING score:** 768</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is currently no known functional association between COX8A and FOSL1. COX8A plays a role in the mitochondrial electron transport chain, while FOSL1 is involved in DNA damage tolerance and regulation of translesion synthesis.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.9</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations: Both IGF1R and PROS1 are involved in regulating cellular signaling and hemostasis, respectively. However, there is no direct evidence for specific functional associations between these two genes.|
| [ACOT9](https://www.ncbi.nlm.nih.gov/gene?term=ACOT9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.8</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between ACOT9 and PAFAH1B3 is their role in lipid metabolism. ACOT9 encodes an enzyme involved in hydrolysis of acyl-CoAs to free fatty acids and CoASH, while PAFAH1B3 encodes the alpha1 subunit of PAF-AH enzyme involved in hydrolization of PAF and its analogs. There is no evidence of a direct biological relationship between the two genes.|
| [CCNF](https://www.ncbi.nlm.nih.gov/gene?term=CCNF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: CCNF is a component of the SCF(CCNF) E3 ubiquitin-protein ligase complex which mediates the degradation of target proteins involved in cell cycle progression and genome stability, while SLC25A4 is an ADP:ATP antiporter involved in mitochondrial energy output, uncoupling, and permeability transition. There is no known direct functional association between CCNF and SLC25A4 in specific biological terms.|
