| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.8</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: BAD promotes cell death while COX8A is a component of the mitochondrial electron transport chain. It is possible that BAD may affect mitochondrial metabolism through interactions with COX8A or other components of the electron transport chain.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CCNH and MAPKAPK5 could involve regulation of cell cycle control and RNA transcription through phosphorylation of CDK7 and the C-terminal domain of POLR2A, as well as post-transcriptional regulation of MYC via phosphorylation of FOXO3. Additionally, MAPKAPK5 may play a role in mTORC1 signaling and F-actin rearrangement. The shortest path between CCNH and MAPKAPK5 in the BioGRID protein interaction network involves TP53 and ZRANB1, suggesting potential involvement in tumor suppression and DNA repair.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between CYCS and DYNLT3 is related to their involvement in apoptosis and mitosis. CYCS plays a role in apoptosis by releasing cytochrome c, which triggers the activation of caspase-9, while DYNLT3 is required for efficient progression through mitosis. There is no evidence of direct physical interaction or regulation between these two genes.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CYCS and DNAJB6 in TCGA-METABRIC cancer type include regulation of apoptosis and inhibition of protein aggregation. CYCS plays a role in apoptosis by releasing cytochrome c into the cytosol, while DNAJB6 inhibits huntingtin aggregation and interacts with histone deacetylases. The shortest path between these genes in the BioGRID protein interaction network involves a series of intermediate protein interactions including with HSPA8, NLE1, EGLN3, TRIM28, and LRRK2.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no clear biological connection between ARHGEF12 and HMGCS1 based on current knowledge and available context. ARHGEF12 plays a role in RhoA GTPase regulation, while HMGCS1 catalyzes acetyl-CoA condensation, which is involved in cholesterol synthesis.|
| [HOXA5](https://www.ncbi.nlm.nih.gov/gene?term=HOXA5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 295 (min len 4) | Possible functional associations between HOXA5 and LGMN could be in the regulation of cell proliferation through their roles in EGFR degradation and lysosomal protein degradation. HOXA5 is a transcription factor involved in providing cells with specific positional identities, while LGMN is a protein involved in the processing of proteins for MHC class II antigen presentation. The shortest path in the BioGRID protein interaction network between HOXA5 and LGMN involves the autophagy-related protein ATG7, which is involved in the regulation of protein degradation processes.|
| [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM15B](https://www.ncbi.nlm.nih.gov/gene?term=RBM15B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between PAK4 and RBM15B may involve their roles in regulating cellular processes. PAK4 is a serine/threonine protein kinase involved in cytoskeleton regulation, cell migration, and proliferation. RBM15B is an RNA-binding protein that plays a key role in RNA methylation and splicing. Both genes may be involved in regulating the efficiency of mRNA splicing and RNA processing. However, without further research, the exact nature of their functional association cannot be determined.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no clear functional association between DNM1L and LSM6 based on current knowledge. DNM1L is involved in mitochondrial division and fission, while LSM6 is a component of the U4/U6-U5 tri-snRNP complex involved in pre-mRNA splicing.|
| [ALDOC](https://www.ncbi.nlm.nih.gov/gene?term=ALDOC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 117 (min len 4) | Possible functional association: There is no clearly established functional association between ALDOC and PROS1 based on the given context and network analysis.   Gene functions: ALDOC is involved in transmembrane signaling processes and regulates phospholipase C activity. PROS1 is an anticoagulant plasma protein that acts as a cofactor in the degradation of coagulation factors and helps to prevent coagulation.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 521</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 59 (min len 3) | There is no clear functional association between DNM1L and RHOA based on the provided information. DNM1L is involved in mitochondrial fission and interacts with MFF, while RHOA interacts with human respiratory syncytial virus (HRSV) protein F and serves as a target for the yopT cysteine peptidase from Yersinia pestis.|
| [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.9</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | PCNA is involved in DNA replication and repair, while PIK3C3 is part of the PI3K complex involved in autophagy. There is no obvious functional association between these two genes in specific biological terms.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations:  CYCS is involved in apoptosis by releasing cytochrome c into the cytosol, while DLD is a component of the alpha-ketoacid dehydrogenase complexes and is required for lysine succinylation of histones. There is no known direct functional association between these two genes.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 733</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CYCS and FIS1 are related to mitochondrial fission and apoptosis. CYCS is involved in apoptosis by releasing cytochrome c into the cytosol, which triggers caspase activation, while FIS1 is involved in mitochondrial fission and clustering, and can induce cytochrome c release from the mitochondrion to the cytosol. The shortest path between them in the BioGRID protein interaction network includes proteins associated with mitochondrial fission, suggesting a possible functional association in this process.|
| [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 462</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | CDKN1B and PCNA are functionally associated in DNA replication control. CDKN1B inhibits CDK2 activity and therefore, cell cycle progression, while PCNA is an auxiliary protein of DNA polymerase delta and epsilon, facilitating DNA replication. The shortest path in the BioGRID protein interaction network suggests that CDKN1B may interact with PCNA through various intermediary proteins involved in the control of cell cycle progression.|
| [C2CD5](https://www.ncbi.nlm.nih.gov/gene?term=C2CD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between C2CD5 and TOPBP1 are unclear based on the provided information. C2CD5 is involved in glucose transport and interacts with the MCU protein, while TOPBP1 is involved in DNA replication and interacts with various DNA repair proteins and chromatin remodeling complexes.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 355</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional association between DNM1L and SACM1L is related to the regulation of mitochondrial functions. DNM1L is involved in mitochondrial division and interacts with MFF, while SACM1L acts as a phosphoinositide phosphatase and interacts with TMEM39A and SEC23A/24A to regulate vesicular trafficking between the endoplasmic reticulum and the Golgi complex.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HOXA5](https://www.ncbi.nlm.nih.gov/gene?term=HOXA5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.2</br>**STRING score:** 307</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | There is no clear functional association between EGFR and HOXA5 based on their known biological roles. EGFR is a receptor involved in various cellular processes, including development, cell proliferation, and migration, while HOXA5 is a transcription factor involved in embryonic development and positional identity along the anterior-posterior axis. The shortest paths between these genes in the protein interaction network involve intermediate proteins involved in various signaling pathways, suggesting possible indirect interactions.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.9</br>**STRING score:** 283</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | Possible functional associations between IGF1R and TOP2A include regulation of cell growth and division, as well as DNA topology and maintenance. IGF1R is involved in growth factor signaling, while TOP2A is a key enzyme in altering DNA topology, such as during DNA replication and transcription. The shortest path in the BioGRID protein interaction network suggests a physical interaction between the two proteins, potentially supporting a collaborative role in regulating cellular processes.|
| [ALDOA](https://www.ncbi.nlm.nih.gov/gene?term=ALDOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BNIP3L](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.7</br>**STRING score:** 200</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between ALDOA and BNIP3L in the context of cancer and cellular metabolism. ALDOA is a key enzyme in glycolysis and gluconeogenesis, while BNIP3L plays a role in mitochondrial quality control and can induce apoptosis. These genes may have a role in regulating metabolic processes that impact cell survival and thereby potentially affect cancer progression.|
| [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 831</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 43 (min len 3) | Possible functional associations between HAT1 and PCNA include their involvement in DNA replication, repair, and damage response pathways. HAT1 acetylates histones, which is important for chromatin remodeling during DNA replication, while PCNA is an auxiliary protein for DNA polymerase that is essential for processivity and recruits DDR proteins to coordinate DNA repair with DNA replication. The BioGRID interaction network suggests that these two proteins may physically interact via other proteins such as BAP1 and G3BP1.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between EVL and PAK4 include regulation of cytoskeleton dynamics, cell migration, and cell survival. EVL enhances actin nucleation and polymerization, while PAK4 plays a role in cytoskeleton regulation, cell motility, and apoptosis. Their interaction via BioGRID protein interaction network suggests a potential involvement in downstream signaling pathways.|
| [KDM3A](https://www.ncbi.nlm.nih.gov/gene?term=KDM3A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: None can be inferred based solely on protein-protein interactions and gene function descriptions. Both genes are involved in different biological processes (histone demethylation and phosphatidylinositol metabolism).   KDM3A is a histone demethylase that specifically demethylates 'Lys-9' of histone H3, involved in hormone-dependent transcriptional activation, spermatogenesis, and obesity resistance. SACM1L is a phosphoinositide phosphatase that hydrolyzes phosphatidylinositol 4-phosphate and interacts with TMEM39A, SEC23A, and SEC24A, and PLEKHA3 and VAPA and/or VAPB to form a ternary complex.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context, it is difficult to reason about any possible functional associations between DNM1L and IQGAP1. DNM1L plays a role in mitochondrial fission while IQGAP1 is involved in regulating the actin cytoskeleton and promoting neurite outgrowth.|
| [CRYZ](https://www.ncbi.nlm.nih.gov/gene?term=CRYZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.2</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association between CRYZ and CYCS is involvement in apoptosis. CRYZ may enhance the stability of mRNA coding for BCL2, an anti-apoptotic protein, whereas CYCS plays a role in apoptosis by triggering the activation of caspase-9 upon release from mitochondria. The shortest path between the two genes in the BioGRID protein interaction network involves several proteins, some of which are also involved in apoptosis and mitochondrial function.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.1</br>**STRING score:** 464</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CCNB1 and RPA2 may be functionally associated in DNA replication and damage response. CCNB1 is essential for control of the cell cycle at the G2/M transition and forms a complex with CDC2 kinase, while RPA2 is part of the replication protein A complex that binds and stabilizes single-stranded DNA intermediates during DNA replication and recruits different proteins involved in DNA metabolism.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between HDAC2 and SACM1L could involve chromatin remodeling and transcriptional regulation. HDAC2 is a histone deacetylase that catalyzes the deacetylation of lysine residues on the core histones and plays a role in transcriptional repression. SACM1L is a phosphoinositide phosphatase that hydrolyzes phosphatidylinositol. The shortest path between HDAC2 and SACM1L in the protein interaction network involves several other proteins that are involved in chromatin remodeling and transcriptional regulation.|
