| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.8</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between BAD and COX8A could involve their mutual regulation of cellular apoptosis and mitochondrial respiration, respectively. BAD promotes cell death by competing for binding to anti-apoptotic proteins, while COX8A is a component of the cytochrome c oxidase complex in the electron transport chain, essential in ATP production. The shortest path in the protein interaction network suggests these genes may be indirectly associated through intermediates such as VCP, NDUFA12, ATAD3A, IPO9, CYC1.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 12.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no specific biological terms that suggest a functional association between CCNH and MAPKAPK5 in TCGA-METABRIC data as they have very different functions related to cell cycle regulation and protein kinase activity respectively. CCNH regulates CDK7, a catalytic subunit of the CDK-activating kinase, while MAPKAPK5 acts as a tumor suppressor serine/threonine-protein kinase involved in mTORC1 signaling and post-transcriptional regulation.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between CYCS and DYNLT3 is involvement in mitosis and intracellular transportation of cargos along microtubules. CYCS plays a role in apoptosis and binds to Apaf-1 to trigger caspase-9 activation, while DYNLT3 acts as a non-catalytic accessory component of cytoplasmic dynein 1 complex, required for efficient progression through mitosis and transport of cargos.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional association: DNAJB6 may interact with CYCS during apoptosis and play a role in preventing protein aggregation. CYCS plays a role in initiating apoptosis by releasing cytochrome c from the mitochondria, which activates caspases and promotes cell death.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGCS1](https://www.ncbi.nlm.nih.gov/gene?term=HMGCS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is currently no clear functional association between ARHGEF12 and HMGCS1. ARHGEF12 is implicated in the regulation of RhoA GTPase and may interact with IGF1R beta subunit, while HMGCS1 is involved in cholesterol synthesis through the condensation of acetyl-CoA and acetoacetyl-CoA.|
| [HOXA5](https://www.ncbi.nlm.nih.gov/gene?term=HOXA5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 295 (min len 4) | Based on the BioGRID protein interaction network and provided context, a possible functional association between HOXA5 and LGMN is their involvement in lysosomal protein degradation. HOXA5 is a transcription factor involved in the developmental regulatory system, while LGMN is an enzyme that cleaves asparaginyl bonds and plays a role in the regulation of cell proliferation via EGFR degradation.|
| [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM15B](https://www.ncbi.nlm.nih.gov/gene?term=RBM15B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between PAK4 and RBM15B could involve PAK4's role in cytoskeleton regulation, cell migration, growth, and proliferation, which could affect RBM15B's regulation of alternative splicing of mRNAs and N6-methyladenosine (m6A) methylation of RNAs. The shortest path in the BioGRID protein interaction network suggests that there may be a direct interaction between PAK4 and RBM15B via YWHAG, TERF1, SRPK2, HECW2, NTRK1.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: DNM1L is involved in mitochondrial fission while LSM6 plays a role in pre-mRNA splicing as a component of the spliceosome complex. There is no known direct biological association between these two genes.|
| [ALDOC](https://www.ncbi.nlm.nih.gov/gene?term=ALDOC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PROS1](https://www.ncbi.nlm.nih.gov/gene?term=PROS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 117 (min len 4) | Possible functional association: ALDOC and PROS1 may have a role in regulating coagulation and fibrinolysis. ALDOC may interact with phospholipase C and transduce FFAR4 signaling, while PROS1 acts as a cofactor to activated protein C in the degradation of coagulation factors. The shortest path between these genes in the BioGRID protein interaction network involves common interaction partners, including NPM1 and HSPA1A, which may suggest potential functional interactions.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.0</br>**STRING score:** 521</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 59 (min len 3) | Possible functional associations between DNM1L and RHOA include their involvement in actin-mediated processes such as cell migration and cytoskeletal organization. DNM1L is a GTPase involved in mitochondrial fission, while RHOA is a small GTPase that regulates actin cytoskeleton dynamics, suggesting a possible functional link between mitochondrial dynamics and cytoskeletal remodeling.|
| [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.9</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between PCNA and PIK3C3 could involve DNA replication control and DNA damage response pathways. PCNA is an auxiliary protein of DNA polymerase involved in the control of DNA replication, whereas PIK3C3 is a core component of the PI3K complex involved in regulating the autophagy pathway and SARS-CoV-2 replication. However, the specific biological mechanisms linking these two genes are currently unknown.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.6</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between CYCS and DLD include involvement in apoptosis and mitochondrial metabolism. CYCS plays a role in apoptosis by releasing from the mitochondria, while DLD is an E3 component of three alpha-ketoacid dehydrogenase complexes involved in mitochondrial metabolism. The shortest path in the BioGRID protein interaction network between these two genes involves multiple proteins that also participate in various mitochondrial processes.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 733</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CYCS and FIS1 could involve mitochondrial dynamics and apoptosis. CYCS plays a role in apoptosis by releasing cytochrome c into the cytosol, while FIS1 is involved in mitochondrial fragmentation and clustering, and can induce cytochrome c release, ultimately leading to apoptosis. The shortest path between CYCS and FIS1 in the BioGRID protein interaction network suggests that they may interact or functionally associate in mitochondrial fission.|
| [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 462</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | CDKN1B is an important regulator of cell cycle progression, while PCNA is an auxiliary protein involved in DNA replication and repair. Both genes are involved in cell proliferation and DNA repair processes, suggesting a possible functional association in maintaining genomic stability during cell division. The protein interaction network suggests that CDKN1B and PCNA interact indirectly through a network of other proteins involved in the regulation of cell cycle progression and DNA replication.|
| [C2CD5](https://www.ncbi.nlm.nih.gov/gene?term=C2CD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.4</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: C2CD5 is involved in glucose transport and interacts with MCU, while TOPBP1 is involved in DNA replication, checkpoint control, and interacts with E2F1. There may not be an immediate biological link between the two genes based on their known functions and interactions.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.3</br>**STRING score:** 355</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional association between DNM1L and SACM1L is their interaction with the endoplasmic reticulum-mitochondrial interface during mitochondrial fission. DNM1L inhibits peroxisomal division and helps in oligomerization to form membrane-associated tubules, while SACM1L is a phosphoinositide phosphatase that catalyzes the hydrolysis of phosphatidylinositol 4-phosphate.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HOXA5](https://www.ncbi.nlm.nih.gov/gene?term=HOXA5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.2</br>**STRING score:** 307</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | There are no known direct functional associations between EGFR and HOXA5. EGFR is involved in cell signaling, growth, and differentiation, while HOXA5 is a transcription factor involved in developmental regulation. The shortest path between these genes in the BioGRID network involves intermediate proteins MAPK14, HEATR3, CTTN, CTBP2, and FAM98A.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.9</br>**STRING score:** 283</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | Possible functional association: IGF1R and TOP2A interact directly in the BioGRID protein interaction network, suggesting a potential role in shared cellular processes.   IGF1R is involved in binding insulin-like growth factors and regulating cell proliferation, differentiation, and survival. TOP2A is a key enzyme in altering DNA topology and regulating DNA replication and transcription. These genes may potentially interact in processes such as cell growth and cytoskeletal dynamics.|
| [ALDOA](https://www.ncbi.nlm.nih.gov/gene?term=ALDOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BNIP3L](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.7</br>**STRING score:** 200</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ALDOA and BNIP3L include their involvement in apoptosis and mitochondrial quality control processes. ALDOA plays a key role in glycolysis and can function as a scaffolding protein, while BNIP3L induces apoptosis and participates in mitochondrial protein catabolic process. The shortest path in the BioGRID protein interaction network between these genes involves proteins ORF7b, GABARAPL1, and ORF9c.|
| [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCNA](https://www.ncbi.nlm.nih.gov/gene?term=PCNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 831</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 43 (min len 3) | Possible functional associations between HAT1 and PCNA include their involvement in DNA replication and damage response. HAT1 is a histone acetyltransferase that interacts with histones H4 and H2A, while PCNA is an auxiliary protein of DNA polymerase delta and epsilon that plays a key role in DNA damage response by coordinating DNA replication with DNA repair and DNA damage tolerance pathways. The shortest path between these genes in the BioGRID protein interaction network suggests that they may directly interact in these processes.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK4](https://www.ncbi.nlm.nih.gov/gene?term=PAK4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: EVL and PAK4 are involved in cytoskeleton regulation, cell migration, and growth. There is a shortest path between EVL and PAK4 in the BioGRID protein interaction network, suggesting a potential physical interaction between the two proteins.|
| [KDM3A](https://www.ncbi.nlm.nih.gov/gene?term=KDM3A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.4</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations:   KDM3A is involved in histone demethylation and hormone-dependent transcriptional activation, while SACM1L is a phosphoinositide phosphatase involved in the hydrolysis of phosphatidylinositol phosphates. There is no known direct functional association between these genes, but they may have indirect interactions through their involvement in cellular signaling and gene expression regulation.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between DNM1L and IQGAP1 may include their roles in cytoskeletal organization and cell cycle regulation. DNM1L is involved in mitochondrial fission and interacts with proteins such as MFF and FIS1, while IQGAP1 binds CDC42 and calmodulin to regulate actin dynamics and may contribute to cell cycle progression after DNA replication arrest. Their interaction may potentially impact these processes.|
| [CRYZ](https://www.ncbi.nlm.nih.gov/gene?term=CRYZ[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.2</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association: CRYZ may enhance the stability of mRNA coding for BCL2, which is regulated by CYCS in apoptosis, suggesting a potential interaction in the regulation of apoptosis.   CRYZ binds NADP and may act in the detoxification of xenobiotics. CYCS plays a role in apoptosis by triggering caspase activation. There is no direct functional association observed between CRYZ and CYCS in existing literature.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA2](https://www.ncbi.nlm.nih.gov/gene?term=RPA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.1</br>**STRING score:** 464</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both CCNB1 and RPA2 are involved in DNA replication and cellular response to DNA damage. CCNB1 interacts with CDC2 to form the serine/threonine kinase holoenzyme complex necessary for proper cell cycle progression, while RPA2 is a component of the heterotrimeric replication protein A complex that stabilizes single-stranded DNA intermediates and recruits different proteins involved in DNA metabolism. They could potentially interact during DNA replication or repair processes.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.0</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between HDAC2 and SACM1L include regulation of transcription, cell cycle progression, and developmental events. HDAC2 is involved in histone deacetylation and acts as a component of various protein complexes, while SACM1L is a phosphoinositide phosphatase that catalyzes the hydrolysis of phosphatidylinositol. It is unclear if there is a direct interaction or relationship between these two genes based on the given information.|
