| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ISOC1](https://www.ncbi.nlm.nih.gov/gene?term=ISOC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | EGFR and ISOC1 have a short path in the BioGRID protein interaction network, indicating possible functional association. EGFR is a receptor involved in cell signaling pathways, particularly in cell proliferation. ISOC1 is known to Complex with metalloproteinases, specifically inhibiting their activity.       Based on their known functions, it is possible that EGFR and ISOC1 may be involved in regulating cell proliferation and extracellular matrix remodeling. However, further studies are required to establish any functional relationship between the two genes.|
| [CPSF4](https://www.ncbi.nlm.nih.gov/gene?term=CPSF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF3E](https://www.ncbi.nlm.nih.gov/gene?term=EIF3E[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 11.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between CPSF4 and EIF3E could be related to mRNA processing and translation initiation. CPSF4 is involved in pre-mRNA 3'-end formation, while EIF3E is a component of the eukaryotic translation initiation factor 3 (eIF-3) complex, which is required for several steps in the initiation of protein synthesis. The shortest path between these two genes in the BioGRID protein interaction network involves several intermediary proteins that may mediate their physical interactions.|
| [CCND1](https://www.ncbi.nlm.nih.gov/gene?term=CCND1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDH3](https://www.ncbi.nlm.nih.gov/gene?term=CDH3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.8</br>**STRING score:** 407</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 468 (min len 4) | There is no clear functional association between CCND1 and CDH3 based on the provided information. CCND1 is involved in regulating the cell cycle and phosphorylating members of the RB protein family, while CDH3 is a calcium-dependent cell adhesion protein. However, they share common interacting partners in the BioGRID protein interaction network such as CTNNB1, suggesting a potential indirect association based on shared biological pathways.|
| [GHR](https://www.ncbi.nlm.nih.gov/gene?term=GHR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ID2](https://www.ncbi.nlm.nih.gov/gene?term=ID2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: GHR and ID2 have a protein interaction via TCEB1. GHR is involved in growth hormone signaling and interacts with JAK2 while ID2 is a transcriptional regulator involved in cellular processes such as growth, differentiation, and neoplastic transformation. Based on this information, it is possible that GHR and ID2 might have a role in regulating cellular growth and differentiation through their interaction.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional associations between HSPD1 and MYCBP2 include HSPD1's role in mitochondrial protein import and macromolecular assembly, while MYCBP2 functions as an atypical E3 ubiquitin-protein ligase and is involved in neural development and axon navigation. These genes may have indirect interactions through MYC, a transcription factor that interacts with both HSPD1 and MYCBP2.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FAT1](https://www.ncbi.nlm.nih.gov/gene?term=FAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 10.0</br>**STRING score:** 634</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 37 (min len 3) | Possible functional associations between EGFR and FAT1 include modulation of cell-cell contact, directed cell migration, and signaling downstream of EGFR in cell proliferation. It is also possible for these genes to interact in the formation of receptor complexes and downstream signaling pathways. EGFR acts as a receptor for hepatitis C virus, while FAT1 is involved in cellular polarity.|
| [CD320](https://www.ncbi.nlm.nih.gov/gene?term=CD320[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between CD320 and HNRNPA2B1 in TCGA-METABRIC may involve their interactions in mRNA transport and processing. CD320 plays a role in B cell differentiation and proliferation, while HNRNPA2B1 is involved in the transport of HIV-1 genomic RNA. The shortest path between these two genes in the BioGRID protein interaction network suggests their potential interaction in the regulation of mRNA splicing and nucleocytoplasmic transport.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP88](https://www.ncbi.nlm.nih.gov/gene?term=NUP88[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known functional association between EGFR and NUP88. EGFR is involved in cell growth and differentiation, while NUP88 is a component of the nuclear pore complex involved in nucleocytoplasmic transport.|
| [CD320](https://www.ncbi.nlm.nih.gov/gene?term=CD320[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSC22D3](https://www.ncbi.nlm.nih.gov/gene?term=TSC22D3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TSC22D3 may regulate the transcriptional activity of MYOD1, a protein involved in myogenic differentiation, which is inhibited by TSC22D3. CD320, on the other hand, plays a role in promoting B cell responses and differentiation. There is no clear biological connection between the two genes based on the provided context.|
| [GALE](https://www.ncbi.nlm.nih.gov/gene?term=GALE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: None apparent based on context and known biology.       GALE catalyzes interconversion of UDP-sugar molecules, important in synthesis of glycoproteins and glycolipids. UBL5 is a component of the molecular chaperone complex TRiC, which mediates folding of WRAP53/TCAB1 and plays a role in folding of actin and tubulin. The shortest path between GALE and UBL5 in the BioGRID protein interaction network includes intermediates KRAS and ZRANB1.|
| [ARFIP2](https://www.ncbi.nlm.nih.gov/gene?term=ARFIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNAI1](https://www.ncbi.nlm.nih.gov/gene?term=GNAI1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 9.3</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ARFIP2 and GNAI1 could be related to autophagy and cytokinesis, as ARFIP2 is involved in autophagy regulation and GNAI1 is required for normal cytokinesis during mitosis. However, there is no direct evidence of a direct interaction between the two genes.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ERBB3](https://www.ncbi.nlm.nih.gov/gene?term=ERBB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.9</br>**STRING score:** 997</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and ERBB3 are both receptor tyrosine kinases, known to form heterodimers and activate downstream signaling pathways involved in cell proliferation, differentiation, and survival. The shortest path between these two genes in the BioGRID protein interaction network suggests a direct physical interaction. It is possible that EGFR-ERBB3 heterodimers modulate downstream signaling in specific biological contexts, such as cancer or development.|
| [POLR2I](https://www.ncbi.nlm.nih.gov/gene?term=POLR2I[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.5</br>**STRING score:** 625</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between POLR2I and RPA1 based on the BioGRID protein interaction network and provided context are related to DNA replication, transcription, and DNA damage response. POLR2I is a subunit of RNA polymerase II complex that synthesizes mRNA precursors and many functional non-coding RNAs, while RPA1 is part of the heterotrimeric replication protein A complex that binds and stabilizes single-stranded DNA intermediates. They may interact in processes such as DNA repair, replication, and transcription through the protein-protein interactions identified in the shortest path between them.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GATA2](https://www.ncbi.nlm.nih.gov/gene?term=GATA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.5</br>**STRING score:** 302</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | Possible functional associations between EGFR and GATA2 include regulation of endothelin-1 gene expression in endothelial cells and cell signaling, specifically in downstream EGFR signaling pathways involved in cell proliferation and migration. The BioGRID protein interaction network suggests potential direct interactions between EGFR and GATA2, although more research is needed to determine the functional significance of these interactions.|
| [CNOT4](https://www.ncbi.nlm.nih.gov/gene?term=CNOT4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.4</br>**STRING score:** 679</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Both CNOT4 and TP53 are involved in protein ubiquitination and degradation, and play a role in regulating cell growth and apoptosis. The shortest path between them in the BioGRID network suggests they may interact in regulation of mitochondrial quality control and autophagy.|
| [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [S100A4](https://www.ncbi.nlm.nih.gov/gene?term=S100A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | HDAC6 and S100A4 may have a functional association in promoting cell motility and invasion, as both genes are involved in regulating the dynamics of microtubules and actin filaments through deacetylation of tubulin and modulation of non-muscle myosin heavy chain, respectively. HDAC6 also promotes autophagosome-lysosome fusion and completion of autophagy through the deacetylation of CTTN, which is regulated by S100A4.|
| [COASY](https://www.ncbi.nlm.nih.gov/gene?term=COASY[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YKT6](https://www.ncbi.nlm.nih.gov/gene?term=YKT6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.1</br>**STRING score:** 286</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between COASY and YKT6 could be related to vesicle trafficking and transport processes between the endoplasmic reticulum and Golgi. COASY, a bifunctional enzyme, is involved in CoA biosynthesis regulation, while YKT6 is a vesicular soluble NSF attachment protein receptor (v-SNARE) mediating vesicle docking and fusion to specific acceptor cellular compartments.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.0</br>**STRING score:** 182</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 136 (min len 3) | Possible functional associations: Both EGFR and NPC1 have been reported to be involved in viral infections and endosomal entry. EGFR is a receptor for hepatitis C virus and promotes the formation of receptor complexes essential for HCV entry, while NPC1 acts as an endosomal entry receptor for ebolavirus.  It is possible that EGFR and NPC1 may play roles in the same biological pathway related to viral entry and infection. The BioGRID protein interaction network suggests several possible short paths between EGFR and NPC1, involving various intermediate proteins, which may warrant further investigation.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 8.0</br>**STRING score:** 647</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 134 (min len 3) | EGFR and SMAD3 are both involved in cell signaling pathways. EGFR is a receptor protein that binds to ligands such as growth factors and triggers downstream signaling events, while SMAD3 is a transcriptional regulator that is activated by TGF-beta signaling. It is possible that they may interact in processes such as cell growth and differentiation regulation. The protein interaction network suggests a potential association between EGFR and SMAD3 through intermediates such as CTNNB1 and AR.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GHR](https://www.ncbi.nlm.nih.gov/gene?term=GHR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 7.8</br>**STRING score:** 260</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional association: Both EGFR and GHR are involved in cell signaling pathways, with EGFR activating downstream signaling pathways for cell proliferation and GHR regulating growth hormone signaling. There may be overlapping or complementary roles in the regulation of cell growth and development.|
| [CCND1](https://www.ncbi.nlm.nih.gov/gene?term=CCND1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK1](https://www.ncbi.nlm.nih.gov/gene?term=PAK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 7.8</br>**STRING score:** 390</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between CCND1 and PAK1 include regulation of cell cycle progression and cytoskeletal network, as well as involvement in tumor cell invasion and transcription regulation. The shortest path between CCND1 and PAK1 in the BioGRID protein interaction network suggests potential direct physical interaction between the two proteins.|
