| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BCL7B plays a role in apoptosis regulation, Wnt signaling, cell cycle progression, nuclear structure maintenance, and stem cell differentiation, while NR1H2 is involved in ligand-dependent transcriptional activation, cholesterol uptake regulation, liver metabolism, and phospholipid remodeling. A specific functional association between these two genes is not evident based on the provided information. However, they may interact indirectly through protein interaction networks like BioGRID suggests via SMARCA4.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | NR1H2 is a nuclear receptor involved in lipid metabolism, cholesterol uptake, and inflammation regulation. SMAD3 is an intracellular signal transducer and transcription modulator activated by TGF-beta and involved in chondrogenesis, osteogenesis, and wound healing. The functional association between these two genes could be related to their roles in regulating inflammation and cellular responses to stress.|
| [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | NCOA3 and PUM2 might have functional associations in the regulation of transcription and mRNA stability. NCOA3 functions as a nuclear receptor coactivator, involved in chromatin remodeling and transcriptional activation of various nuclear receptors, while PUM2 is an RNA-binding protein that represses gene expression via mRNA deadenylation and translational inhibition. The interaction between these two genes could occur during post-transcriptional regulation of gene expression and influence processes like cell proliferation, differentiation, and immune response.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.5</br>**STRING score:** 997</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | CCNA2 and ORC1 both play crucial roles in cell cycle regulation, with CCNA2 controlling G1/S and G2/M transition phases through interactions with CDK1 or CDK2, while ORC1 is a component of the origin recognition complex (ORC) involved in DNA replication. There could be a possible functional association between these two genes in the regulation and coordination of DNA replication and cell cycle progression.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | IKBKB is a serine kinase that plays an essential role in the NF-kappa-B signaling pathway, which is involved in immune response, growth control, and apoptosis protection. RPN2 is a subunit of the oligosaccharyl transferase (OST) complex that is responsible for protein N-glycosylation in the endoplasmic reticulum. There might be a possible functional association between these genes through cellular stress responses, which can activate the IKBKB-mediated NF-kappa-B pathway and require proper protein folding facilitated by RPN2's role in N-glycosylation.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 989</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and BRCA1 are functionally associated as both play crucial roles in cell cycle progression, DNA damage repair, and maintaining genomic stability. AURKA contributes to various mitotic events, regulates checkpoint-response pathways, and microtubule dynamics. BRCA1 functions as an E3 ubiquitin-protein ligase mediating DNA damage response, transcriptional regulation, and maintaining centrosomal microtubule nucleation.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX17](https://www.ncbi.nlm.nih.gov/gene?term=SNX17[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between AKR1A1 and SNX17 include their involvement in detoxification and endosomal recycling processes, respectively. AKR1A1 catalyzes the reduction of various carbonyl-containing compounds, playing a role in detoxifying toxic aldehydes and activating procarcinogens. SNX17 is a critical regulator of endosomal recycling of numerous surface proteins, such as integrins and signaling receptors. However, based on currently available knowledge, there is no direct biological link or shared biological process between these two genes.|
| [COPZ1](https://www.ncbi.nlm.nih.gov/gene?term=COPZ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | COPZ1 is involved in protein transport within the cell and is a part of the coatomer complex, while NR1H2 is a nuclear receptor that regulates cholesterol uptake and lipid metabolism. There might be a possible functional association between these two genes related to lipid synthesis, transport, and metabolism within the cell, as well as the regulation of endoplasmic reticulum membrane dynamics.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BIRC5 (Survivin) is a multifunctional protein involved in promoting cell proliferation, preventing apoptosis, and regulating chromosome alignment and segregation during mitosis. CENPO is a component of the CENPA-CAD complex, which plays a crucial role in assembling kinetochore proteins, mitotic progression, and chromosome segregation. Both genes participate in mitotic processes, specifically chromosome arrangement and segregation, suggesting a possible functional association in cell division regulation.|
| [CHEK2](https://www.ncbi.nlm.nih.gov/gene?term=CHEK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | CHEK2 is involved in DNA damage response pathways, typically phosphorylating and interacting with various proteins, including BRCA1. HPRT1 plays a critical role in purine nucleotide generation via the purine salvage pathway. While both genes participate in crucial cellular processes, there is no direct functional association between CHEK2 and HPRT1 in the context of specific biological terms.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.1</br>**STRING score:** 350</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Both CCNB1 and LIG1 play crucial roles in the cell cycle and DNA replication processes. CCNB1 is essential for controlling cell cycle progression at the G2/M transition while forming a complex with CDC2. In contrast, LIG1 is a DNA ligase involved in DNA repair, replication, and recombination while interacting with PCNA. A possible functional association between these genes could be that they both participate in DNA replication and cell cycle progression, ensuring the accurate division and copying of genetic material.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.9</br>**STRING score:** 503</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 597 (min len 4) | ORC1 and UBE2C both play crucial roles in cell cycle regulation. ORC1 is involved in the initiation of DNA replication by forming the pre-replication complex with other ORC subunits. UBE2C functions as a ubiquitin-conjugating enzyme, promoting the degradation of specific cell cycle regulatory proteins through the anaphase-promoting complex (APC/C). They may have a functional association in controlling cell cycle progression, particularly during the transition from the G1 phase to the S phase and mitotic exit.|
| [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 582</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | KIF20A and LIG1 may have functional associations in cell division and DNA repair processes. KIF20A is a mitotic kinesin involved in cytokinesis and vesicle transport along microtubules, while LIG1 is a DNA ligase crucial for DNA repair, replication, and recombination. Their interaction could play a role in coordinating chromosome and DNA dynamics during cell division.|
| [CCND3](https://www.ncbi.nlm.nih.gov/gene?term=CCND3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 473</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CCND3 and NOTCH1 both play crucial roles in cell cycle regulation, differentiation, and development. CCND3 is involved in cell cycle progression during the G1/S transition by forming a complex with CDK4, while NOTCH1 is a receptor involved in cell fate determination and regulating differentiation, proliferation, and apoptosis. There may be functional associations between these two genes in coordinating cell cycle progression and cell differentiation pathways.|
| [CDC20](https://www.ncbi.nlm.nih.gov/gene?term=CDC20[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.7</br>**STRING score:** 984</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | CDC20 and KIF2C both play crucial roles in mitosis, with CDC20 being required for the ubiquitin ligase activity of the anaphase-promoting complex/cyclosome (APC/C) and KIF2C functioning in chromosome segregation and congression. Their functional association could involve the coordination of mitotic processes, such as microtubule dynamics, kinetochore attachments, and chromosome segregation during cell division.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.4</br>**STRING score:** 501</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CCNB2 is essential for controlling cell cycle progression at the G2/M (mitosis) transition by forming a serine/threonine kinase holoenzyme complex with CDK1. RFC5 is involved in DNA replication, being part of a heterotetramer complex that interacts with PCNA for DNA polymerase delta and epsilon elongation. The potential functional association between CCNB2 and RFC5 could be related to cell cycle regulation and DNA replication, where an accurate timing of progression ensures proper cell replication and division.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.7</br>**STRING score:** 948</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | RFC2 and TIMELESS are involved in DNA replication and maintenance of genome stability. RFC2 interacts with accessory proteins like PCNA, while TIMELESS plays a role in replication fork stabilization and DNA repair. A possible functional association between these genes may be related to their roles in DNA replication and repair processes, with both contributing to maintaining genomic integrity and preventing replication stress.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.6</br>**STRING score:** 365</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA is a mitotic serine/threonine kinase that regulates cell cycle progression, spindle assembly, and p53/TP53 pathway, while IKBKB is a serine kinase involved in the NF-kappa-B signaling pathway that regulates immune response, growth control, and protection against apoptosis. Both genes are involved in regulating crucial cellular processes, such as cell cycle progression and apoptosis. Although there is no direct functional association between AURKA and IKBKB, they both have a role in modulating cell survival and regulatory pathways.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.2</br>**STRING score:** 999</br>**BioGRID count:** 49</br>**BioGRID shortest paths:** 1 (min len 2) | The CCNB1 and CDK1 genes associate to form the serine/threonine kinase holoenzyme complex, also known as maturation promoting factor (MPF), which is essential for the G2/M transition in the cell cycle. The cyclin subunit from CCNB1 imparts substrate specificity to the complex, and the interaction between CCNB1 and CDK1 helps regulate cell cycle progression, especially during mitosis.|
| [TIMP2](https://www.ncbi.nlm.nih.gov/gene?term=TIMP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.8</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | TIMP2 is involved in binding and inactivating metalloproteinases, thereby regulating extracellular matrix degradation and tissue remodeling. TPM1, on the other hand, plays a crucial role in stabilizing actin filaments in muscle and non-muscle cells and modulating muscle contraction. The functional association between these two genes may be indirectly related through their involvement in cell morphology, cellular adhesion, and tissue integrity in the context of cancer progression.|
| [DNAJC15](https://www.ncbi.nlm.nih.gov/gene?term=DNAJC15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.5</br>**STRING score:** 252</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | DNAJC15 is a negative regulator of the mitochondrial respiratory chain and a component of the TIM23 translocase complex, whereas PUM2 is a sequence-specific RNA-binding protein acting as a post-transcriptional repressor. Both genes are involved in the regulation of cellular processes; however, they have distinct functionalities with DNAJC15 involved in mitochondrial function, and PUM2 influencing mRNA stability and translation. The functional association between these two genes is not evident based on their specific roles.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.3</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | AURKA is a mitotic serine/threonine kinase that plays a critical role in various mitotic events, including spindle formation, centrosome duplication, and chromosome alignment, as well as regulating the p53/TP53 pathway for oncogenic transformation. CENPO is a component of the CENPA-CAD complex, which is involved in kinetochore assembly, mitotic progression, and chromosome segregation. A possible functional association between AURKA and CENPO could be their involvement in mitotic events, particularly in the formation and regulation of the mitotic spindle and chromosome segregation.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.2</br>**STRING score:** 725</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | BIRC5 and NCAPD2 may have functional associations in mitosis and cell proliferation. BIRC5 is involved in chromosome alignment, segregation, and cytokinesis through its association with the chromosome passage protein complex. On the other hand, NCAPD2 is a regulatory subunit of the condensin complex, which plays a crucial role in chromosome condensation during mitosis. Both genes play critical roles in maintaining genomic stability during cell division.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 459</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CENPO is involved in the assembly of kinetochore proteins, mitotic progression, and chromosome segregation, playing a role in the incorporation of CENPA into centromeres. PLK1 is a serine/threonine-protein kinase that regulates several essential functions throughout the M phase of the cell cycle, including centrosome maturation, spindle assembly, and chromosome segregation. Both genes play a role in mitosis, particularly in chromosome segregation and kinetochore assembly, indicating a possible functional association between CENPO and PLK1 in these specific biological processes.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCP2](https://www.ncbi.nlm.nih.gov/gene?term=SCP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | AKR1A1 is involved in the detoxification of endogenous and xenobiotic aldehydes, while SCP2 mediates lipid transfer and regulates steroidogenesis. A possible functional association between these two genes could be their involvement in the metabolism and detoxification of various compounds, including harmful oxidative byproducts, within cellular processes.|
| [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | MYCBP2 is an atypical E3 ubiquitin-protein ligase involved in neural development, axon growth regulation, and Wallerian axon degeneration. NCOA3 is a nuclear receptor coactivator involved in hormone-dependent transcriptional activities and chromatin remodeling. A possible functional association between these two genes could be related to transcriptional regulation, as MYCBP2 can also function as a facilitator or regulator of transcriptional activation by MYC.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.3</br>**STRING score:** 608</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | CCNA2 is a cyclin that controls cell cycle transitions, particularly the G1/S and G2/M phases, through the activation of CDK1 and CDK2 protein kinases. TIMELESS plays a crucial role in DNA replication, replication fork stability, genome maintenance, and circadian clock regulation. The potential functional association between CCNA2 and TIMELESS could involve their role in cell cycle regulation and response to DNA damage, as both genes have roles in the maintenance of genome stability and DNA repair mechanisms.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 392 (min len 4) | NR1H2 is a nuclear receptor that regulates cholesterol uptake, liver metabolism, lipogenesis, and anti-inflammatory response in hepatocytes. RHEB is a GTPase that activates the protein kinase activity of mTORC1, regulating apoptosis and stimulating cell growth. A possible functional association between these two genes could be related to the regulation of lipid metabolism and cellular processes that impact inflammation and cell growth.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNA11](https://www.ncbi.nlm.nih.gov/gene?term=GNA11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 586</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | AKT1 and GNA11 are both involved in transmembrane signaling systems, with AKT1 regulating processes such as cell survival, metabolism, and proliferation, while GNA11 acts as an activator of phospholipase C. A possible functional association between these genes could be in signal transduction pathways where AKT1 mediates cell growth and survival signals, and GNA11 participates in the corresponding modulation of signaling systems.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 619</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CCNB2 and TIMELESS may have functional associations in cell cycle regulation and DNA replication. CCNB2 is essential for the control of the G2/M transition in the cell cycle and forms a complex with CDK1. TIMELESS plays a crucial role in DNA replication, maintaining replication fork stability, and DNA repair.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.2</br>**STRING score:** 981</br>**BioGRID count:** 6</br>**BioGRID shortest paths:** 1 (min len 2) | G3BP1 plays a crucial role in innate immunity and stress granule formation, while HDAC6 is responsible for deacetylation of lysine residues in histones, microtubule-dependent cell motility, and degradation of misfolded proteins. Although there is no direct functional association shown in the provided context, both genes may be involved in stress response processes, as G3BP1 is involved in stress granule formation and HDAC6 in protein degradation.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.7</br>**STRING score:** 592</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | BIRC5 and ORC1 are functionally associated in the regulation of cell proliferation and cell cycle progression. BIRC5 plays a dual role in promoting cell proliferation and preventing apoptosis, while ORC1 is a component of the origin recognition complex (ORC) involved in DNA replication initiation. The interaction of these genes may have an impact on mitotic spindle formation and chromosome segregation during mitosis, which is critical for maintaining genome stability and preventing tumorigenesis.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | BIRC5 plays a dual role in promoting cell proliferation and preventing apoptosis, and is essential for chromosome alignment and segregation during mitosis and cytokinesis. USP22 is a histone deubiquitinating component of the SAGA complex, involved in transcription regulation and cell cycle progression. A possible functional association between these two genes could be related to their involvement in cell cycle progression and regulation of cell proliferation.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GADD45A](https://www.ncbi.nlm.nih.gov/gene?term=GADD45A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | BAG6 is involved in mediating ricin-induced apoptosis and interacts with L. pneumophila proteins, while GADD45A regulates p38 MAPKs in T-cells and plays a role in DNA excision repair. There is a potential functional association between these two genes in apoptosis regulation, as BAG6 mediates the cell death process and GADD45A is involved in DNA repair and cell cycle regulation.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 545</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | CCNB1 is involved in cell cycle control at the G2/M transition, forming a complex with CDC2 to promote mitosis. RFC5 functions as an accessory protein for DNA polymerases delta and epsilon, aiding in DNA elongation. The functional association between these two genes may lie in their roles in cell cycle regulation and DNA replication processes during cell division, with CCNB1 regulating mitotic entry and RFC5 ensuring precise DNA replication.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUV39H1](https://www.ncbi.nlm.nih.gov/gene?term=SUV39H1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.5</br>**STRING score:** 436</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | There might be a functional association between PLK1 and SUV39H1 in the regulation of chromatin structure and epigenetic regulation. PLK1 is a serine/threonine-protein kinase involved in cell cycle progression, centrosome maturation, spindle assembly, and mitotic exit. SUV39H1 is a histone methyltransferase that trimethylates 'Lys-9' of histone H3, creating a specific tag for epigenetic transcriptional repression and is mainly involved in heterochromatin regulation.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 996</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | CDK1 is involved in cell cycle regulation and acts as a receptor for hepatitis C virus (HCV), while UBE2C functions in protein ubiquitination and is an essential factor of the anaphase promoting complex/cyclosome (APC/C). There may be a functional association between these two genes in the context of cell cycle progression and mitotic exit, as both CDK1 and UBE2C play important roles in the regulation and control of these processes.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 254</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CDKN2A and IQGAP1 are both involved in cell cycle regulation and actin cytoskeleton dynamics. CDKN2A functions as a negative regulator of cell proliferation by interacting with CDK4 and CDK6, while IQGAP1 regulates actin cytoskeleton dynamics and may also play a role in cell cycle progression. The functional association between these two genes may involve coordinating cell cycle regulation and actin cytoskeleton organization during cellular processes like cell division and neurite outgrowth.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 557</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CASP3 is a major effector caspase involved in apoptosis, responsible for cleaving various proteins, whereas CXCR4 is a chemokine receptor essential for cell migration, HIV-1 entry, and immune responses. A possible functional association between these genes could be in the regulation of apoptotic cell clearance and immune response coordination. CASP3 may mediate the exposure of phosphatidylserine on apoptotic cells, while CXCR4 signaling can regulate immune cell trafficking to the site of apoptosis.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZW10](https://www.ncbi.nlm.nih.gov/gene?term=ZW10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 641</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | LYPLA1 acts as an acyl-protein thioesterase, involved in depalmitoylating proteins, and as a lysophospholipase, hydrolyzing various lysophospholipids. ZW10 is an essential component of the mitotic checkpoint and plays a role in regulation of membrane traffic between the Golgi and the endoplasmic reticulum. There is no direct functional association between these two genes, but they may interact indirectly through protein pathways related to membrane trafficking and lysophospholipid metabolism.|
