| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between BCL7B and NR1H2, as they have distinct biological roles. BCL7B positively regulates apoptosis and is involved in cell cycle progression, while NR1H2 is a nuclear receptor involved in regulating cholesterol uptake and inducing lipid metabolism. The shortest path in their protein interaction network goes through SMARCA4, which is involved in chromatin remodeling.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between NR1H2 and SMAD3 may involve regulation of lipid metabolism and inflammation, as NR1H2 plays a role in regulating cholesterol uptake and lipid metabolism while SMAD3 is involved in regulating wound healing and modulating chemotaxis of monocytes. The shortest path in the BioGRID protein interaction network between NR1H2 and SMAD3 involves several intermediary proteins, suggesting potential functional crosstalk between these genes in the regulation of shared biological processes.|
| [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between NCOA3 and PUM2 include regulation of gene expression, specifically post-transcriptional regulation, as well as mediation of different cellular pathways. NCOA3 acts as a nuclear receptor coactivator and histone acetyltransferase, and interacts with proteins involved in the NF-kappa-B pathway and self-renewal/pluripotency markers, while PUM2 acts as a sequence-specific RNA-binding protein that mediates deadenylation-independent repression and cytoplasmic sensing of viral infection. The shortest path between NCOA3 and PUM2 in the BioGRID protein interaction network involves several intermediates, including ESR1/2, ELAVL1, TP53, and ALG13. It is possible that these genes interact in the regulation of gene expression or cellular pathways mediated by these intermediates.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.5</br>**STRING score:** 997</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | There are no apparent functional associations between CCNA2 and ORC1 based on their known biological roles. CCNA2 is a cyclin involved in cell cycle regulation and interacts with CDK1 and CDK2, while ORC1 is involved in mitochondrial transport and the urea cycle.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between IKBKB and RPN2 include their involvement in protein glycosylation and immune response through the NF-kappa-B signaling pathway. These genes are also part of the oligosaccharyl transferase complex and are required for maximal enzyme activity.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 989</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and BRCA1 both play critical roles in regulating cell cycle progression and DNA damage repair. Possible functional associations between these genes include their involvement in centrosomal microtubule nucleation, homologous recombination repair, and ubiquitination pathways.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX17](https://www.ncbi.nlm.nih.gov/gene?term=SNX17[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | AKR1A1 and SNX17 are not known to have a direct functional association. AKR1A1 is involved in the detoxification of xenobiotics and drugs, while SNX17 regulates endosomal recycling of various surface proteins. However, both genes could play a role in the metabolism and/or transport of certain substances in the head and neck cancer cells.|
| [COPZ1](https://www.ncbi.nlm.nih.gov/gene?term=COPZ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is a single protein interaction link between COPZ1 and NR1H2. COPZ1 is involved in Golgi-to-ER transport of proteins, while NR1H2 is a nuclear receptor that regulates cholesterol uptake and liver metabolism. There is no clear biological process that links these two genes, but they may play a role in intracellular trafficking and lipid metabolism.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: BIRC5 and CENPO are both involved in chromosome segregation and mitotic progression, and they physically interact in the BioGRID protein interaction network.   BIRC5 (Survivin) plays a crucial dual role in promoting cell proliferation and preventing apoptosis, and is a component of the chromosome passage complex (CPC) which is essential for chromosome alignment and segregation. CENPO (Centromere protein O) is also involved in chromosome segregation and mitotic progression as a component of the CENPA-CAD complex, which recruits kinetochore proteins to centromeres. The physical interaction between BIRC5 and CENPO may suggest a functional association in regulating mitotic progression and chromosome segregation.|
| [CHEK2](https://www.ncbi.nlm.nih.gov/gene?term=CHEK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There is no clear evidence of functional association between CHEK2 and HPRT1. CHEK2 is involved in various cellular processes, including DNA damage response and cell cycle regulation. HPRT1 plays a central role in nucleotide metabolism as a key enzyme in the purine salvage pathway.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.1</br>**STRING score:** 350</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations: CCNB1 regulates the cell cycle at G2/M transition, while LIG1 acts as a negative feedback regulator of signaling by receptor tyrosine kinases. They do not have any known biological association based on the provided context and their distinct functions.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.9</br>**STRING score:** 503</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 597 (min len 4) | Possible functional associations between ORC1 and UBE2C are their involvement in cell cycle regulation and mitosis, as UBE2C is a component of the APC/C complex that controls progression through mitosis and ORC1 is involved in DNA replication initiation. Additionally, UBE2C may have a role in ubiquitination of ORC1 or its interacting partners, leading to their degradation and regulation of DNA replication initiation.|
| [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 582</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between KIF20A and LIG1 in TCGA-HNSC cancer type may involve regulation of cell cycle progression, cytokinesis, and signal transduction pathways involved in growth factor receptor signaling. KIF20A is required for CPC-mediated cytokinesis and interacts with RAB6A/B for Golgi membrane transport, while LIG1 acts as a feedback negative regulator of signaling by receptor tyrosine kinases such as EGFR.|
| [CCND3](https://www.ncbi.nlm.nih.gov/gene?term=CCND3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 473</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: CCND3 and NOTCH1 are both involved in cell cycle regulation, with CCND3 promoting cell cycle progression and NOTCH1 regulating cell fate determination. It is possible that changes in the expression or activity of one gene may affect the functioning of the other gene in processes such as proliferation or differentiation.|
| [CDC20](https://www.ncbi.nlm.nih.gov/gene?term=CDC20[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.7</br>**STRING score:** 984</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional association between CDC20 and KIF2C: CDC20 plays a role in regulating mitotic progression and may target KIF2C for degradation through the APC/C complex. KIF2C is involved in chromosome segregation and regulation of microtubule turnover at the kinetochore.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.4</br>**STRING score:** 501</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CCNB2 and RFC5 include their involvement in DNA replication and cell cycle progression. CCNB2 is essential for the control of the cell cycle at the G2/M transition, while RFC5 is a subunit of the elongation complex that is required for DNA polymerase activity. The shortest path between CCNB2 and RFC5 in the BioGRID protein interaction network suggests direct physical interaction between these two proteins in vivo.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.7</br>**STRING score:** 948</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations: RFC2 and TIMELESS are involved in DNA replication and stability, and likely have a direct physical interaction as part of a protein complex. RFC2 binds to PCNA and assists in DNA polymerase function, while TIMELESS aids in replication fork stability and activating DNA polymerases. They may collaborate to regulate the fork progression and DNA synthesis under both normal and stress conditions.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.6</br>**STRING score:** 365</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and IKBKB are not known to have direct functional associations in specific biological terms. AURKA is involved in regulating cell cycle progression and microtubule formation, while IKBKB plays an essential role in the NF-kappa-B signaling pathway and activation of immune response genes. The shortest path between these two genes in the BioGRID protein interaction network is a direct interaction.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.2</br>**STRING score:** 999</br>**BioGRID count:** 49</br>**BioGRID shortest paths:** 1 (min len 2) | CCNB1 and CDK1 have a strong functional association as they form a complex that plays a crucial role in regulating cell cycle progression at G2/M checkpoint. They interact with each other to form a serine/threonine kinase holoenzyme complex known as maturation promoting factor (MPF) that initiates mitosis.|
| [TIMP2](https://www.ncbi.nlm.nih.gov/gene?term=TIMP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.8</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TIMP2 inhibits metalloproteinase activity while TPM1 plays a role in muscle contraction and cytoskeleton stabilization; there is no known direct biological connection between the genes.|
| [DNAJC15](https://www.ncbi.nlm.nih.gov/gene?term=DNAJC15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.5</br>**STRING score:** 252</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between DNAJC15 and PUM2 could be in the regulation of mitochondrial energetics and metabolism. DNAJC15 acts as a negative regulator of mitochondrial respiratory chain and interacts with complex I. PUM2 mediates post-transcriptional regulation of key mitotic, DNA repair and DNA replication factors. It is possible that PUM2 regulates genes involved in mitochondrial energetics, in which DNAJC15 is also involved in, possibly through a direct or indirect interaction.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.3</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: AURKA is involved in mitotic events, including spindle formation and centrosome duplication, while CENPO is part of the CENPA-CAD complex involved in kinetochore assembly and chromosome segregation. They may function together in mitotic progression and chromosome segregation.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.2</br>**STRING score:** 725</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | BIRC5 and NCAPD2 are involved in chromosome alignment and segregation during mitosis and cytokinesis. They may have a functional association in ensuring accurate mitotic chromosome condensation and centromere recruitment during early mitosis.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 459</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CENPO and PLK1 are functionally associated in mitosis and chromosome segregation. CENPO is a component of the CENPA-CAD complex involved in kinetochore assembly, while PLK1 phosphorylates multiple proteins involved in mitotic progression, including CENPO. There is evidence of protein-protein interaction between CENPO and PLK1 in the BioGRID network, suggesting they may interact in a regulatory complex.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCP2](https://www.ncbi.nlm.nih.gov/gene?term=SCP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between AKR1A1 and SCP2 include their involvement in detoxification processes and potential roles in chromatin organization during meiotic prophase. The shortest path between these two genes in the BioGRID protein interaction network suggests potential direct or indirect interactions through CAV1, TP53, ACO2, PARK2, or ZRANB1.|
| [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association between MYCBP2 and NCOA3 is related to transcriptional regulation. MYCBP2 acts as a facilitator or regulator of transcriptional activation by MYC, while NCOA3 is a nuclear receptor coactivator that directly binds nuclear receptors and stimulates the transcriptional activities in a hormone-dependent fashion.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.3</br>**STRING score:** 608</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Both CCNA2 and TIMELESS are involved in regulating DNA replication and maintenance of genome stability. They are functionally associated through their protein interactions on the BioGRID network, potentially playing a role in stabilizing replication forks during DNA replication.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 392 (min len 4) | Possible functional association between NR1H2 and RHEB could be their involvement in regulating lipid metabolism and inflammation. NR1H2 modulates lipid homeostasis and has anti-inflammatory properties while RHEB activates mTORC1 signaling, which is involved in regulating lipid metabolism and inflammation. The shortest path between these two genes in the BioGRID interaction network includes TSC2, a known regulator of mTORC1 signaling, suggesting a possible functional interaction between NR1H2 and RHEB in this process.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNA11](https://www.ncbi.nlm.nih.gov/gene?term=GNA11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 586</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between AKT1 and GNA11 could be related to cell proliferation, survival, and growth, as both genes are involved in regulating these processes through various downstream signaling pathways. The shortest path in the protein interaction network suggests that ESR1, VCP, HSPA8, GNB1, and CD81 may be intermediates connecting AKT1 and GNA11, indicating a potential indirect functional association between the two genes.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 619</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CCNB2 and TIMELESS are functionally associated as they both play a role in the control of DNA replication and maintenance of genome stability. CCNB2 is essential for the control of the cell cycle at the G2/M transition and interacts with CDK1 to form a kinase complex. TIMELESS regulates DNA replication processes, stabilizes replication forks, and influences DNA repair through interaction with PARP1.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.2</br>**STRING score:** 981</br>**BioGRID count:** 6</br>**BioGRID shortest paths:** 1 (min len 2) | There are currently no known functional associations between G3BP1 and HDAC6, based on their respective roles in stress granule formation and histone deacetylation. G3BP1 plays a role in stress granule formation and in innate immunity, while HDAC6 is involved in the deacetylation of lysine residues on the N-terminal part of the core histones and also functions in microtubule-dependent cell motility and cilia disassembly.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.7</br>**STRING score:** 592</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | BIRC5 is involved in promoting cell proliferation and preventing apoptosis, while ORC1 plays a crucial role in the urea cycle and mitochondrial function. There is no clear biological connection between these two genes based on their known functions.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | BIRC5 and USP22 are not directly linked in a functional association in the provided context. BIRC5 has roles in promoting cell proliferation, preventing apoptosis, and spindle formation during mitosis, while USP22 is involved in histone deubiquitination and transcription regulation.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GADD45A](https://www.ncbi.nlm.nih.gov/gene?term=GADD45A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of any functional association between BAG6 and GADD45A. BAG6 mediates cell apoptosis and interacts with bacterial proteins, while GADD45A acts as a regulator of p38 MAPKs and stimulates DNA excision repair.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 545</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional associations between CCNB1 and RFC5 include their interaction in the DNA replication and repair process, where RFC5 forms a complex with PCNA and may stimulate CCNB1 activity. CCNB1 plays a critical role in the control of the cell cycle at the G2/M transition, while RFC5 is essential for elongation of primed DNA templates by DNA polymerase delta and epsilon.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUV39H1](https://www.ncbi.nlm.nih.gov/gene?term=SUV39H1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.5</br>**STRING score:** 436</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between PLK1 and SUV39H1 could involve the regulation of cell cycle, specifically in the transition from G2 to M phase. PLK1 plays a key role in mitosis exit and cytokinesis, while SUV39H1 specifically trimethylates 'Lys-9' of histone H3, contributing to the formation of constitutive heterochromatin in pericentric and telomere regions.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 996</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association between CDK1 and UBE2C in TCGA-HNSC is their involvement in regulating cell cycle progression and mitotic exit. CDK1 is a key regulator of the G2/M transition, while UBE2C is an essential factor of the APC/C complex that promotes mitotic exit by targeting APC/C substrates for degradation. The provided context also suggests that CDK1 can interact with UBE2C through FZR1 and other intermediates in the protein interaction network.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 254</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: CDKN2A may regulate caspase-independent cell death and interact with C1QBP, while IQGAP1 plays a role in regulating the actin cytoskeleton and binds to calmodulin. These genes may be involved in regulating cell cycle progression and potentially neurite outgrowth. The shortest path between them in the protein interaction network suggests a direct physical association between the two proteins.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 557</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear biological indication of a functional association between CASP3 and CXCR4. CASP3 is a Thiol protease involved in the execution phase of apoptosis and CXCR4 is a receptor that acts as a coreceptor for HIV-1 and interacts with Staphylococcus aureus protein SSL10.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZW10](https://www.ncbi.nlm.nih.gov/gene?term=ZW10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 641</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no direct functional association between LYPLA1 and ZW10 based on the protein interaction network. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while ZW10 is essential for mitotic checkpoint and regulation of membrane traffic.|
