| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | As there is a shortest path between BCL7B and NR1H2 in the BioGRID protein interaction network, it is possible that these genes functionally interact. BCL7B plays roles in cell cycle progression, nuclear structure maintenance, and stem cell differentiation, while NR1H2 is involved in cholesterol uptake, phospholipid remodeling in the endoplasmic reticulum, and lipid metabolism regulation. However, without further experimental evidence, the specific biological processes in which they may interact cannot be determined.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between NR1H2 and SMAD3 may involve regulation of TGF-beta-mediated transcription and modulation of inflammatory response, as suggested by their respective functions in cholesterol uptake and phospholipid remodeling, and in wound healing and regulation of chondrogenesis and osteogenesis. The shortest path in the BioGRID network suggests that NR1H2 and SMAD3 may have indirect interactions through a number of transcriptional co-activators and co-repressors, including NCOA6, HDAC4, SMARCA4, NCOA3, and RARA.|
| [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association: NCOA3 and PUM2 may have a role in regulating gene expression. NCOA3 is a coactivator of various nuclear receptors, histone acetyltransferase, and involved in the coactivation of the NF-kappa-B pathway. PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor and mediates deadenylation-independent repression by promoting accessibility of miRNAs. The shortest path between these genes in the BioGRID protein interaction network involves several other proteins.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.5</br>**STRING score:** 997</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: CCNA2 and ORC1 are involved in DNA replication. CCNA2 regulates the G1/S and G2/M cell cycle transitions by forming a protein complex with cyclin-dependent kinases. ORC1 is a component of the origin recognition complex and is required for pre-replication complex assembly and DNA replication initiation. They interact with each other and other proteins, such as CDC6 and KAT7/HBO1, in a cell cycle-dependent manner.|
| [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN2](https://www.ncbi.nlm.nih.gov/gene?term=RPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | There is a shortest path of protein-protein interactions between IKBKB and RPN2. IKBKB is a serine kinase involved in the NF-kappa-B signaling pathway, while RPN2 is a subunit of the oligosaccharyl transferase complex. Thus, there may be a functional association between these genes in regulating protein glycosylation and immune response.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BRCA1](https://www.ncbi.nlm.nih.gov/gene?term=BRCA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 989</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and BRCA1 are functionally associated in DNA damage repair and cell cycle regulation. AURKA is a mitotic kinase involved in spindle assembly, checkpoint signaling, and phosphorylation of BRCA1. BRCA1 is an E3 ubiquitin ligase that mediates the formation of 'Lys-6'-linked polyubiquitin chains and is critical for DNA repair and maintaining genomic stability.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX17](https://www.ncbi.nlm.nih.gov/gene?term=SNX17[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between AKR1A1 and SNX17 could be in the detoxification of xenobiotics and drugs. AKR1A1 is involved in the metabolism of various xenobiotics and drugs, while SNX17 is critical in the recycling of numerous surface proteins, including signaling receptors and channels.|
| [COPZ1](https://www.ncbi.nlm.nih.gov/gene?term=COPZ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between COPZ1 and NR1H2 could involve regulation of cholesterol uptake and lipid metabolism through MYLIP-dependent ubiquitination of LDLR, VLDLR, LRP8, and DLDLR, as well as modulation of SRC kinase membrane compartmentalization and limiting the synthesis of lipid inflammatory mediators. COPZ1 is involved in Golgi-to-ER transport of dilysine-tagged proteins, while NR1H2 is a nuclear receptor that regulates genes involved in liver metabolism and phospholipid remodeling in endoplasmic reticulum membranes. Their interaction through the BioGRID protein interaction network shortest path consists of CUL3, a component of the cullin family of E3 ubiquitin ligases.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BIRC5 and CENPO function in different stages of mitosis, with BIRC5 being involved in the regulation of mitotic spindle formation and preventing apoptosis, and CENPO being part of the CENPA-CAD complex involved in the assembly of kinetochore proteins and chromosome segregation. The interaction between the two genes, as indicated by the shortest path in the BioGRID network, suggests a possible role of BIRC5 in directing the movement of the CPC complex to different locations, including the kinetochore, where CENPO is found.|
| [CHEK2](https://www.ncbi.nlm.nih.gov/gene?term=CHEK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.2</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There are no direct functional associations between CHEK2 and HPRT1, as they are not connected in the BioGRID interaction network. CHEK2 is involved in cell cycle regulation and DNA damage response, while HPRT1 plays a central role in the purine salvage pathway, generating purine nucleotides.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.1</br>**STRING score:** 350</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between CCNB1 and LIG1 include a role for CCNB1 in regulating DNA repair through interaction with LIG1, as well as a potential role for LIG1 in DNA replication and recombination. Both genes are involved in critical cellular processes related to the cell cycle and DNA maintenance.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.9</br>**STRING score:** 503</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 597 (min len 4) | ORC1 and UBE2C may functionally associate with each other as they have a shortest path of protein interactions in the BioGRID network. ORC1 is involved in DNA replication initiation, while UBE2C is a component of the APC/C complex that promotes mitotic exit, suggesting their potential roles in cell cycle regulation.|
| [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 582</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between KIF20A and LIG1 in TCGA-HNSC include a potential role in DNA replication and recombination, as well as potential involvement in cytokinesis mediated by the chromosome passenger complex. KIF20A may also be involved in retrograde RAB6-regulated transport of Golgi membranes and associated vesicles along microtubules.|
| [CCND3](https://www.ncbi.nlm.nih.gov/gene?term=CCND3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.2</br>**STRING score:** 473</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CCND3 and NOTCH1 may be functionally associated in the regulation of cell-cycle progression and differentiation. CCND3 is a regulatory component of the cyclin D3-CDK4 complex that phosphorylates and inhibits members of the retinoblastoma protein family, whereas NOTCH1 is a receptor for membrane-bound ligands involved in cell-fate determination, differentiation, and proliferation. The shortest path between CCND3 and NOTCH1 in the BioGRID protein interaction network suggests multiple indirect interactions through RB1, APP, HGS, and MKRN2.|
| [CDC20](https://www.ncbi.nlm.nih.gov/gene?term=CDC20[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.7</br>**STRING score:** 984</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | These genes may have functional associations in mitosis and chromosome segregation, as CDC20 is required for APC/C activity and KIF2C regulates microtubule turnover and chromosome congression. The interaction between CDC20 and KIF2C may be through AURKA and FBXO5, which are known to interact with both genes according to the BioGRID network.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.4</br>**STRING score:** 501</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | One possible functional association between CCNB2 and RFC5 is their involvement in DNA replication and cell cycle control.  CCNB2 is a cyclin subunit that forms a complex with CDK1 to regulate progression through mitosis, while RFC5 is an accessory protein involved in elongation of primed DNA templates and interacts with PCNA.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.7</br>**STRING score:** 948</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | RFC2 and TIMELESS are both involved in DNA replication and maintenance of genome stability. The shortest path between these two genes in the BioGRID protein interaction network suggests that they may interact via PCNA and other proteins involved in the regulation of DNA replication. Therefore, it is possible that these two genes may function together to regulate DNA replication and maintenance.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.6</br>**STRING score:** 365</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between AURKA and IKBKB in TCGA-HNSC could involve regulation of cell cycle progression, centrosome duplication and separation, and cytokinesis via AURKA, as well as activation of the NF-kappa-B signaling pathway and the transcription of immune response and growth control genes via IKBKB. AURKA and IKBKB may also have a role in mediating TNF-dependent inactivation of pro-apoptotic transcription factors and preventing overproduction of inflammatory mediators.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.2</br>**STRING score:** 999</br>**BioGRID count:** 49</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: CCNB1 and CDK1 both play essential roles in the control of the cell cycle. CCNB1 is a cyclin subunit that imparts substrate specificity to the complex while CDK1 is a serine/threonine kinase. They interact with each other to form a holoenzyme complex known as maturation promoting factor (MPF) that regulates progression through the G2/M phase of the cell cycle.|
| [TIMP2](https://www.ncbi.nlm.nih.gov/gene?term=TIMP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.8</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between TIMP2 and TPM1 could involve the regulation of cell migration and invasion, as TIMP2 inhibits the activity of metalloproteinases involved in these processes, while TPM1 has been implicated in stabilizing cytoskeleton actin filaments that contribute to cell motility. However, more research is needed to establish a direct connection between these genes in the context of cancer progression.|
| [DNAJC15](https://www.ncbi.nlm.nih.gov/gene?term=DNAJC15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.5</br>**STRING score:** 252</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between DNAJC15 and PUM2 are unclear, as their roles in different biological processes don't indicate an obvious interaction. DNAJC15 is a negative regulator of the mitochondrial respiratory chain, while PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.3</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between AURKA and CENPO is their involvement in mitosis and chromosome segregation, with AURKA regulating spindle formation and CENPO being part of the kinetochore complex. AURKA may phosphorylate CENPO via intermediate proteins such as CENPK or MYC.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCAPD2](https://www.ncbi.nlm.nih.gov/gene?term=NCAPD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.2</br>**STRING score:** 725</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between BIRC5 and NCAPD2 in the TCGA-HNSC cancer type may involve their roles in promoting cell proliferation, preventing apoptosis, and regulating chromosome condensation and segregation during mitosis. BIRC5 is a component of the chromosome passage protein complex (CPC) and is involved in directing its movement to different locations during mitosis and cytokinesis. NCAPD2 is a subunit of the condensin complex, which is required for converting interphase chromatin into mitotic-like condense chromosomes. Both genes have been implicated in neoplasia and may play a role in the maintenance of mitochondrial integrity and function, suggesting potential collaborations in the development and progression of cancer.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 459</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: PLK1 phosphorylates multiple kinetochore proteins and CENPO is a component of the CENPA-CAD complex involved in kinetochore assembly and chromosome segregation, suggesting a role for PLK1 in regulating CENPO function.   Functional descriptions: CENPO is involved in centromere assembly and incorporation of CENPA into centromeres, while PLK1 regulates several aspects of M phase including centrosome maturation, spindle assembly, and mitotic exit and cytokinesis.|
| [AKR1A1](https://www.ncbi.nlm.nih.gov/gene?term=AKR1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCP2](https://www.ncbi.nlm.nih.gov/gene?term=SCP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between AKR1A1 and SCP2 include their involvement in lipid metabolism and detoxification processes. AKR1A1 catalyzes the reduction of carbonyl-containing compounds, including lipid-derived aldehydes, while SCP2 mediates the transfer of phospholipids, cholesterol, and fatty acids. The shortest path between these genes in the BioGRID interaction network suggests they may interact in multiple ways, including through the tumor suppressor TP53 and peroxisomal proteins such as PEX5.|
| [MYCBP2](https://www.ncbi.nlm.nih.gov/gene?term=MYCBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association: MYCBP2 mediates ubiquitination and degradation of TSC2, while NCOA3 is involved in the coactivation of different nuclear receptors, including PPARs, which are known to regulate TSC2 expression. Therefore, these two genes may functionally interact in the regulation of TSC2 in cancer cells, impacting cell growth and proliferation.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.3</br>**STRING score:** 608</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | CCNA2 and TIMELESS are functionally associated in the regulation of DNA replication and maintenance of genome stability. They interact through various protein complexes like CDK1, PCNA and RPA1 to stabilize replication forks during DNA replication. TIMELESS also promotes TIPIN nuclear localization and interacts with PARP1 to promote homologous recombination repair at DNA damage sites.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 392 (min len 4) | Possible functional associations between NR1H2 and RHEB can be inferred through their shortest protein interaction path in the BioGRID database, which involves TSC2. NR1H2 is a nuclear receptor that regulates cholesterol uptake and lipid metabolism, while RHEB activates mTORC1 signaling and plays a role in the regulation of apoptosis. TSC2, which interacts with both NR1H2 and RHEB, acts as a tumor suppressor gene and is involved in the control of cell growth and proliferation.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNA11](https://www.ncbi.nlm.nih.gov/gene?term=GNA11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 586</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | AKT1 and GNA11 have a functional association through protein interactions in the BioGRID network. AKT1 plays a role in regulating many cellular processes, including metabolism, proliferation, cell survival, and angiogenesis, while GNA11 acts as an activator of phospholipase C, modulating various signaling systems.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMELESS](https://www.ncbi.nlm.nih.gov/gene?term=TIMELESS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 619</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CCNB2 and TIMELESS have been shown to interact through a protein interaction network, indicating a functional association in the control of DNA replication and maintenance of genome stability. CCNB2 is essential for cell cycle control and interacts with CDK1 to form a serine/threonine kinase complex, while TIMELESS stabilizes replication forks during DNA replication and promotes homologous recombination repair after DNA damage.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC6](https://www.ncbi.nlm.nih.gov/gene?term=HDAC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.2</br>**STRING score:** 981</br>**BioGRID count:** 6</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: G3BP1 and HDAC6 both play roles in stress granule formation and autophagy. G3BP1 is involved in RNA binding and activation, while HDAC6 facilitates clearance of misfolded proteins and promotes autophagosome-lysosome fusion. Their interaction may play a role in coordinating these processes.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.7</br>**STRING score:** 592</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between BIRC5 and ORC1 are the regulation of cell proliferation and DNA replication respectively. BIRC5 is involved in promoting cell proliferation and preventing apoptosis, while ORC1 is a component of the pre-replication complex necessary to initiate DNA replication. The shortest path between these genes in the BioGrid network suggests that BIRC5 may have a regulatory role in ORC1-mediated DNA replication.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: BIRC5 and USP22 may play roles in cell proliferation and transcription regulation, respectively. It is not clear if they directly interact or collaborate in a specific biological process.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GADD45A](https://www.ncbi.nlm.nih.gov/gene?term=GADD45A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between BAG6 and GADD45A could involve regulation of cellular stress responses and cell cycle progression through interactions with MAP kinases and PCNA. BAG6 may play a role in mediating apoptosis induced by microbial infection, while GADD45A is involved in DNA excision repair and inhibition of S phase entry.|
| [CCNB1](https://www.ncbi.nlm.nih.gov/gene?term=CCNB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.8</br>**STRING score:** 545</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional association between CCNB1 and RFC5 is their involvement in DNA replication and cell cycle regulation. CCNB1 promotes cell cycle progression through G2/M phase by forming a complex with CDC2 protein kinase, while RFC5 is a subunit of the protein complex involved in DNA replication and repair, interacting with proliferating cell nuclear antigen (PCNA). The shortest path in the BioGRID protein interaction network suggests that CCNF and RFC5 also interact, possibly indicating a more direct functional relationship between the two genes.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUV39H1](https://www.ncbi.nlm.nih.gov/gene?term=SUV39H1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.5</br>**STRING score:** 436</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional association between PLK1 and SUV39H1 is the regulation of cell cycle progression and epigenetic modifications in chromatin structure. PLK1 is a serine/threonine-protein kinase involved in many processes including regulation of centrosome maturation, spindle assembly, chromosome segregation, and mitotic exit. SUV39H1 is a histone methyltransferase specifically involved in the trimethylation of histone H3 lysine 9, thereby playing a central role in the establishment of constitutive heterochromatin at pericentric and telomere regions.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 996</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between CDK1 and UBE2C in TCGA-HNSC may involve the APC/C complex, in which UBE2C acts as an essential factor and CDK1 interacts with ANAPC2. CDK1 also plays a role in facilitating cell entry of certain viruses and inhibits the formation of cyclin B1-CDK1 complex, leading to cell cycle arrest in viral infections.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 254</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between CDKN2A and IQGAP1 include regulation of cell cycle progression and cytoskeleton organization. CDKN2A acts as a negative regulator of cell proliferation by interacting with CDK4 and CDK6, while IQGAP1 plays a crucial role in regulating the dynamics and assembly of the actin cytoskeleton. There is currently no direct evidence of functional associations between these two genes.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 557</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CASP3 and CXCR4 could be related to apoptotic processes, as CASP3 is a key effector caspase involved in apoptosis and CXCR4 has been found to act as a coreceptor for some viruses that promote Env-mediated fusion, which can trigger apoptosis. They may also have a role in promoting cell adhesion and neurogenesis through RET cleavage by CASP3.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZW10](https://www.ncbi.nlm.nih.gov/gene?term=ZW10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.4</br>**STRING score:** 641</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There is no clear functional association between LYPLA1 and ZW10 based on their known biological functions in the provided context. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while ZW10 is involved in the mitotic checkpoint and regulation of membrane traffic between the Golgi and the endoplasmic reticulum.|
