| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [GMNN](https://www.ncbi.nlm.nih.gov/gene?term=GMNN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 124.0</br>**STRING score:** 283</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | GMNN inhibits DNA replication and interacts with CDT1. TFDP1 stimulates E2F-dependent transcription and blocks adipocyte differentiation. These genes may have a functional association in the regulation of cell cycle progression and DNA replication, as both are involved in the E2F:DP transcription factor complex and interact with CDT1.|
| [LGALS8](https://www.ncbi.nlm.nih.gov/gene?term=LGALS8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYF2](https://www.ncbi.nlm.nih.gov/gene?term=SYF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 536 (min len 4) | Possible functional associations between LGALS8 and SYF2 may involve pre-mRNA splicing, as SYF2 is involved in the spliceosome, and LGALS8 interacts with SNRPB, which is also a component of the spliceosome. Additionally, LGALS8 is involved in autophagy and detects membrane damage caused by infection, while SYF2 interacts with CCNDBP1. Further investigation is needed to determine the exact nature of any potential functional associations.|
| [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between PARP2 and TFDP1 may include their roles in DNA replication and cell cycle regulation. PARP2 is involved in DNA repair and initiates the repair of DNA breaks, while TFDP1 can stimulate E2F-dependent transcription and bind cooperatively with E2F family members to regulate the cell cycle. The two genes may potentially interact through the E2F:DP transcription factor complex.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST6GALNAC2](https://www.ncbi.nlm.nih.gov/gene?term=ST6GALNAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between CDK2 and ST6GALNAC2 is the regulation of cell proliferation and cell signaling. CDK2 is a serine/threonine protein kinase that controls the cell cycle, while ST6GALNAC2 catalyzes the transfer of N-acetylneuraminyl groups onto glycan chains in glycoproteins. Both genes play significant roles in cell proliferation and signaling pathways.|
| [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.2</br>**STRING score:** 737</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | PARP2 is involved in DNA repair, specifically in the initiation of repair of double-strand DNA breaks, while RFC2 plays a role in DNA replication. There is no known direct interaction or functional association between these two genes in specific biological processes.|
| [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: PARP2 is involved in DNA repair and interacts with nucleosomes, while CETN3 is involved in mRNA export through the nuclear pores and is a component of the TREX-2 complex. These genes may have a role in coordinating the repair of DNA damage with the transcription and export of mRNA.|
| [SLC25A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCD2](https://www.ncbi.nlm.nih.gov/gene?term=SMARCD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 110.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between SLC25A4 and SMARCD2 are unclear based on the provided information. SLC25A4 is involved in ADP:ATP antiporter activity, mitochondrial uncoupling, mitochondrial permeability transition pore activity, and mitophagy. SMARCD2 is involved in chromatin remodeling, transcriptional activation and repression, and myeloid differentiation.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 109.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | CDK2 is a kinase involved in cell cycle control, DNA synthesis, and repair. HES1 is a transcriptional repressor that modulates the expression of genes requiring a bHLH protein. They may functionally interact in the regulation of transcriptional programs involved in cell cycle progression and DNA repair.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIST1H2BK](https://www.ncbi.nlm.nih.gov/gene?term=HIST1H2BK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.2</br>**STRING score:** 680</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | CDK2 is a protein kinase involved in cell cycle control, while HIST1H2BK is a histone involved in DNA packaging. There may be a functional association between the two in regulating DNA synthesis and repair through their roles in chromatin remodeling and gene expression.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.0</br>**STRING score:** 259</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | CDK2 and PARP2 are involved in DNA repair processes. CDK2 is a serine/threonine protein kinase that promotes cell cycle progression and controls DNA synthesis, while PARP2 is a poly-ADP-ribosyltransferase that mediates poly-ADP-ribosylation of proteins and plays a key role in DNA repair. There could be a functional association between these genes in the regulation of DNA repair mechanisms. The shortest path between CDK2 and PARP2 in the BioGRID protein interaction network supports this possibility.|
| [NUP85](https://www.ncbi.nlm.nih.gov/gene?term=NUP85[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.0</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional association between NUP85 and RFC2: NUP85 is involved in DNA replication and cell survival, while RFC2 is a component of the replication factor C complex, which is required for DNA elongation. Both genes are involved in DNA replication and maintenance of the nuclear pore complex, suggesting they may function together in these processes.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP85](https://www.ncbi.nlm.nih.gov/gene?term=NUP85[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.5</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: CCNB2 and NUP85 both have roles in mitosis. CCNB2 is essential for cell cycle control at the G2/M transition and forms a complex with CDK1 to regulate substrate specificity. NUP85 is essential for NPC assembly and maintenance and plays a role in spindle assembly during mitosis.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPP38](https://www.ncbi.nlm.nih.gov/gene?term=RPP38[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.1</br>**STRING score:** 253</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between RFC2 and RPP38 include their involvement in DNA replication and RNA processing. RFC2 is a subunit of a protein complex involved in DNA replication, while RPP38 is a component of ribonuclease P, a complex involved in tRNA and pre-rRNA processing. The shortest path between them in the BioGRID protein interaction network involves several other proteins, including MYC and ESR2, which are involved in transcription and cell survival, respectively.|
| [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STMN1](https://www.ncbi.nlm.nih.gov/gene?term=STMN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.8</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between HES1 and STMN1 is that HES1 could indirectly regulate microtubule dynamics through its interaction with FANCC, which in turn interacts with STMN1. HES1 also plays a role in the DNA damage response, which could potentially affect the stability of microtubules via STMN1 regulation.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.2</br>**STRING score:** 539</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | CASP3 is a major effector caspase involved in the execution phase of apoptosis, while SMAD3 is a receptor-regulated SMAD that activates transcription in response to TGF-beta. One possible functional association between these two genes could be their involvement in regulating cell death and survival pathways. The shortest path between CASP3 and SMAD3 in the BioGRID protein interaction network suggests that they may interact through CFLAR, SREBF2, RBX1, BTRC, CDK2.|
| [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYF2](https://www.ncbi.nlm.nih.gov/gene?term=SYF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.1</br>**STRING score:** 278</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HAT1 and SYF2 are in pre-mRNA splicing, as SYF2 is involved in the spliceosome, and in histone acetylation, as HAT1 is a catalytic subunit of the HAT complex that acetylates histones H4 and H2A.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.0</br>**STRING score:** 164</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are currently no known functional associations between PRKAG2 and PRUNE in specific biological terms based on the provided context. However, PRKAG2 plays a key role in regulating cellular energy metabolism and cellular polarity, while PRUNE functions as a negative regulator of NME1 and is involved in the regulation of cell proliferation, migration, and differentiation.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHKG2](https://www.ncbi.nlm.nih.gov/gene?term=PHKG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.0</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | CDK2 is a serine/threonine-protein kinase involved in cell cycle control and responsible for promoting DNA synthesis and modulating G2 progression. PHKG2 is the catalytic subunit of phosphorylase b kinase, which mediates neural and hormonal regulation of glycogen breakdown. The shortest path between these genes involves several proteins, suggesting a possible functional association in DNA damage checkpoint and phosphorylation events.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.8</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CDK2 and CETN3 in TCGA-HNSC cancer type could be related to centrosome duplication, DNA synthesis, and mRNA export. CDK2 is a serine/threonine-protein kinase involved in the control of the cell cycle, while CETN3 is a component of the TREX-2 complex involved in mRNA export and a component of the NPC-associated TREX-2 complex. The shortest path in the BioGRID protein interaction network suggests that CDK2 may interact with CETN3 indirectly via other proteins.|
| [HMGN1](https://www.ncbi.nlm.nih.gov/gene?term=HMGN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There's currently no known functional association between HMGN1 and PLS1 in TCGA-HNSC. HMGN1 binds to the inside of nucleosomal DNA, inhibiting histone phosphorylation by kinases involved in transcription, while PLS1 is an actin-bundling protein necessary for stereocilia formation in the inner ear.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.1</br>**STRING score:** 454</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | Possible functional associations between CDK2 and RFC2 include their involvement in DNA replication and repair as well as cell cycle control. CDK2 is a serine/threonine kinase involved in the G1-S transition, while RFC2 is part of the RFC complex that loads PCNA onto DNA during replication and repair. The BioGRID interaction network shows that CDK2 interacts with RFC2 through several other proteins, indicating a potential functional association between them.|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.7</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: MNAT1 and MTX2 have a protein interaction network connection and are involved in cellular processes related to transcription, cell cycle control, and mitochondrial transport. There is no known direct biological significance between these two genes.|
