| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [GMNN](https://www.ncbi.nlm.nih.gov/gene?term=GMNN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 124.0</br>**STRING score:** 283</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | GMNN and TFDP1 both play roles in DNA replication and cell cycle regulation. It is possible that they may have functional associations in these processes, such as GMNN inhibiting DNA replication while TFDP1 stimulates E2F-dependent transcription. The shortest path between these two genes in the BioGRID protein interaction network suggests a physical interaction between them.|
| [LGALS8](https://www.ncbi.nlm.nih.gov/gene?term=LGALS8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYF2](https://www.ncbi.nlm.nih.gov/gene?term=SYF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 536 (min len 4) | Possible functional associations between LGALS8 and SYF2 could be related to pre-mRNA splicing, as SYF2 is involved in spliceosome complex C and interacts with CCNDBP1, while LGALS8 is involved in autophagy via interaction with CALCOCO2 and can detect membrane rupture caused by infection. However, more experimental evidence is needed to establish a clear link between these two genes.|
| [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TFDP1](https://www.ncbi.nlm.nih.gov/gene?term=TFDP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 118.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | PARP2 and TFDP1 are not known to have any direct functional associations. PARP2 is involved in DNA repair and ADP-ribosylation of proteins and DNA, while TFDP1 stimulates E2F-dependent transcription and is involved in cell cycle regulation. The shortest path between these two genes in the BioGRID protein interaction network involves several intermediate protein interactions.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST6GALNAC2](https://www.ncbi.nlm.nih.gov/gene?term=ST6GALNAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CDK2 is a kinase involved in cell cycle control and DNA replication, while ST6GALNAC2 catalyzes the transfer of N-acetylneuraminyl groups onto glycan chains in glycoproteins. CDK2 may indirectly regulate ST6GALNAC2 expression or activity through downstream effectors and/or phosphorylation events involved in glycan biosynthesis. Further experimental evidence is needed to confirm this hypothesis.|
| [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.2</br>**STRING score:** 737</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | PARP2 and RFC2 are involved in different biological processes and there is no direct functional association known between them. PARP2 is a key player in DNA repair through poly-ADP-ribosylation of proteins, while RFC2 is part of a complex involved in DNA replication.|
| [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CETN3 and PARP2 include their involvement in DNA repair processes. CETN3 is part of the TREX-2 complex involved in mRNA export, while PARP2 is a poly-ADP-ribosyltransferase that mediates poly-ADP-ribosylation of proteins and DNA. Shortest path analysis shows several interactions involving both genes, suggesting potential co-regulation.|
| [SLC25A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCD2](https://www.ncbi.nlm.nih.gov/gene?term=SMARCD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 110.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between SLC25A4 and SMARCD2 include their involvement in mitochondrial function and metabolism. SLC25A4 is an ADP:ATP antiporter involved in import of ADP into the mitochondria for ATP synthesis, while SMARCD2 is involved in chromatin remodeling and transcriptional regulation. There is no direct biological evidence of their physical interaction, but both genes are involved in important processes that contribute to proper cellular function and development.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 109.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between CDK2 and HES1 may involve regulation of the cell cycle and gene transcription. CDK2 is a kinase involved in cell cycle control and phosphorylation of various proteins including RB1 and BRCA2, while HES1 is a transcriptional repressor that binds to DNA and interacts with corepressor proteins such as TLE1. The shortest path between these two genes in the BioGRID protein interaction network involves RB1 and several other proteins.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIST1H2BK](https://www.ncbi.nlm.nih.gov/gene?term=HIST1H2BK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.2</br>**STRING score:** 680</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | CDK2 is a serine/threonine-protein kinase involved in cell cycle control, while HIST1H2BK is a histone that contributes to the formation of nucleosomes. There is no clear biological process connecting the two genes, as CDK2 is primarily involved in cell cycle regulation and HIST1H2BK is primarily involved in DNA packaging.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 108.0</br>**STRING score:** 259</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | CDK2 and PARP2 are involved in DNA repair processes and cell cycle regulation. CDK2 is a serine/threonine-protein kinase essential for the initiation of DNA synthesis, while PARP2 is a poly-ADP-ribosyltransferase that plays a key role in repairing DNA strand breaks. They may interact in the DNA damage response and repair pathways.|
| [NUP85](https://www.ncbi.nlm.nih.gov/gene?term=NUP85[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 106.0</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional associations between NUP85 and RFC2 are in the maintenance of the nuclear pore complex assembly and in RNA export. NUP85 is essential for the assembly and maintenance of the NPC, while RFC2 plays a role in elongation of primed DNA templates, requiring the action of accessory proteins, and may have cell survival activity.|
| [CCNB2](https://www.ncbi.nlm.nih.gov/gene?term=CCNB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP85](https://www.ncbi.nlm.nih.gov/gene?term=NUP85[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.5</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CCNB2 and NUP85 could involve the regulation of cell cycle progression and nuclear pore complex assembly, respectively. CCNB2 is essential for controlling the cell cycle at the G2/M transition, while NUP85 is an essential component of the nuclear pore complex required for NPC assembly and maintenance. It is possible that CCNB2 and NUP85 may cooperate in ensuring proper timing and regulation of nuclear transport during cell division.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPP38](https://www.ncbi.nlm.nih.gov/gene?term=RPP38[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.1</br>**STRING score:** 253</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations between RFC2 and RPP38 are involvement in DNA replication and cell survival, as RFC2 is involved in DNA polymerase delta and epsilon elongation and interacts with PRKAR1A, while RPP38 is a component of ribonuclease P and MRP complexes involved in tRNA and pre-rRNA processing. There is no known direct biological interaction between RFC2 and RPP38 in these processes.|
| [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STMN1](https://www.ncbi.nlm.nih.gov/gene?term=STMN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.8</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: There is no clear functional association between HES1 and STMN1 based on their known roles in transcription regulation and microtubule destabilization, respectively. However, their physical interaction through FANCC, STAT3, TNIP2, and LMNA may suggest their involvement in common cellular processes, such as DNA damage response and cell cycle control.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.2</br>**STRING score:** 539</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional association: SMAD3 plays a role in TGF-beta signaling, which regulates cell growth and differentiation. CASP3 is activated during apoptosis, which can be triggered by TGF-beta signaling. Thus, there may be a functional relationship between these two genes in the regulation of cell fate.|
| [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SYF2](https://www.ncbi.nlm.nih.gov/gene?term=SYF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.1</br>**STRING score:** 278</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HAT1 and SYF2 could be their involvement in histone modification and pre-mRNA splicing, respectively. HAT1 is a histone acetyltransferase that modifies histones H4 and H2A, while SYF2 is involved in pre-mRNA splicing as a component of the spliceosome. Although there is no direct evidence of their interaction, their roles in chromatin structure and gene expression suggest that they could functionally interact in regulating gene expression.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.0</br>**STRING score:** 164</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between PRKAG2 and PRUNE is their roles in cell proliferation and migration. PRKAG2 regulates cell growth and proliferation through AMPK activation, while PRUNE plays a role in cell proliferation, migration, and differentiation as a negative regulator of NME1. The interaction between PRKAG2 and PRUNE through ELAVL1 may suggest a possible mechanism of coordinating energy metabolism and cytoskeletal dynamics during cellular processes like migration.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PHKG2](https://www.ncbi.nlm.nih.gov/gene?term=PHKG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.0</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between CDK2 and PHKG2 are unclear based on their known roles. CDK2 is a critical regulator of the cell cycle and promotes DNA replication and repair, while PHKG2 is involved in glycogen breakdown. There is limited overlap between their known biological functions.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.8</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | CDK2 is involved in controlling the cell cycle, orchestrating a balance between cell proliferation, death, and DNA repair, while CETN3 is a component of the TREX-2 complex and aids in the export of mRNAs to cytoplasm. There is no immediate clear biological function association between these two genes, but they are connected in a protein interaction network through multiple shortest paths.|
| [HMGN1](https://www.ncbi.nlm.nih.gov/gene?term=HMGN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no obvious functional associations between HMGN1 and PLS1 based on their known biological functions and interaction network. HMGN1 is involved in nucleosome regulation and histone phosphorylation, while PLS1 is involved in actin-bundling and stereocilia formation in the inner ear.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.1</br>**STRING score:** 454</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | Possible functional associations between CDK2 and RFC2 may include their involvement in DNA replication and repair processes. CDK2 is a critical component for promoting the initiation of DNA synthesis and coordinating the activation of cyclin B/CDK1, while RFC2 is an accessory protein required for DNA polymerase delta and epsilon elongation on primed DNA templates. The protein interaction network suggests that there are several potential intermediate targets connecting CDK2 and RFC2, some of which have been reported to be involved in DNA repair processes (e.g. BRCA1 and BRCA2).|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.7</br>**STRING score:** 170</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between MNAT1 and MTX2. MNAT1 is involved in cell cycle control and RNA transcription, while MTX2 is involved in transport of proteins into the mitochondrion. The shortest path between the two genes in the BioGRID protein interaction network involves ESR1 and TRIM5.|
