| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ATF1](https://www.ncbi.nlm.nih.gov/gene?term=ATF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between ATF1 and JMJD6 are their interaction with BRD4 in the regulation of polymerase II promoter-proximal pause release, leading to transcriptional activation of a large cohort of genes. ATF1 binds to the cAMP response element and triggers cell proliferation and transformation, while JMJD6 acts as both an arginine demethylase and a lysyl-hydroxylase, regulating RNA splicing and playing a role in histone code.|
| [IARS2](https://www.ncbi.nlm.nih.gov/gene?term=IARS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB10](https://www.ncbi.nlm.nih.gov/gene?term=PSMB10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between IARS2 and PSMB10 could be their involvement in antigen processing to generate class I binding peptides, as PSMB10 is a subunit of the proteasome which has an ATP-dependent proteolytic activity and is involved in antigen processing, while IARS2 is an adapter protein that plays a role in the activation of MAP kinases and phosphorylation of PIK3R1 in response to ligand-mediated activation of FGFR1. However, there is no direct evidence indicating a functional interaction between these two genes.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FKBP14](https://www.ncbi.nlm.nih.gov/gene?term=FKBP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between FAM20B and FKBP14 are not immediately obvious based on their known functions and interactions. FAM20B is involved in regulating the amount of mature GAG chains while FKBP14 accelerates protein folding, with a preference for certain collagen types. However, the identified protein interaction between FAM20B and FKBP14 via ELAVL1 warrants further investigation into any potential collaborative role in RNA processing or post-transcriptional regulation.|
| [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between PLSCR1 and RBM34 based on their known biological roles. PLSCR1 is involved in calcium-dependent phospholipid scrambling and antiviral response, while RBM34 is involved in EGF signaling and magnesium reabsorption. The shortest path between them in the BioGRID protein interaction network involves multiple intermediate proteins.|
| [CDC25B](https://www.ncbi.nlm.nih.gov/gene?term=CDC25B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CDC25B may indirectly regulate PLS1 through its interaction with MAPK14 and 14-3-3 proteins. Both genes are involved in cellular processes related to cell division and cytokinesis, but in different ways: CDC25B is a phosphatase required for mitotic progression, while PLS1 is an actin-bundling protein required for stereocilia formation.|
| [SH3BP5](https://www.ncbi.nlm.nih.gov/gene?term=SH3BP5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR61](https://www.ncbi.nlm.nih.gov/gene?term=WDR61[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between SH3BP5 and WDR61 based on their known roles in biology and the provided context. SH3BP5 functions as a guanine nucleotide exchange factor and negative regulator of BTK signaling, while WDR61 is a component of the PAF1 complex involved in transcriptional elongation and histone modifications.|
| [PTPRK](https://www.ncbi.nlm.nih.gov/gene?term=PTPRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between PTPRK and RGS2 may involve regulation of cell signaling pathways and G protein-coupled receptor signaling cascades. PTPRK negatively regulates the EGFR signaling pathway and may interact with beta-catenin and gamma-catenin/plakoglobin, while RGS2 inhibits signal transduction by increasing the GTPase activity of G protein alpha subunits and regulates blood pressure and vascular smooth muscle constriction/relaxation. The shortest path between PTPRK and RGS2 in the BioGRID protein interaction network involves the intermediate protein EGFR.|
| [BID](https://www.ncbi.nlm.nih.gov/gene?term=BID[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no known direct functional associations between BID and MAPK1IP1L. BID induces apoptotic processes, interacts with ITCH and humanin, while MAPK1IP1L is involved in transcriptional regulation and interacts with ZNF202.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COASY](https://www.ncbi.nlm.nih.gov/gene?term=COASY[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There are no known direct functional associations between CD44 and COASY in TCGA-LUAD. CD44 is involved in cell adhesion, migration, and signal transduction, while COASY catalyzes steps in CoA biosynthesis.|
| [PHKG2](https://www.ncbi.nlm.nih.gov/gene?term=PHKG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between PHKG2 and PIK3R4 are involvement in autophagy and endocytosis pathways, as PIK3R4 is a regulatory subunit of the PI3K complex and PHKG2 may play a role in glycogenolysis regulation in the testis. The shortest path between these genes involves other proteins, such as MYC and PLEKHA4, suggesting potential indirect interactions or functional pathways.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.8</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 872 (min len 4) | Possible functional associations between ICAM1 and XBP1 include involvement in immunoglobulin and interleukin-6 production during plasma cell differentiation and induction of ER stress response target genes for maintaining ER homeostasis. However, there is no clear biological process linking these two genes.|
| [RELB](https://www.ncbi.nlm.nih.gov/gene?term=RELB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.4</br>**STRING score:** 197</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | RELB and SMARCC1 are both involved in transcriptional regulation and chromatin remodeling. There may be a functional association between these two genes in the context of modulating gene expression and controlling biological processes such as cell growth and differentiation. The shortest path between these two genes in the BioGRID protein interaction network suggests a direct physical interaction between them.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SERPINE1](https://www.ncbi.nlm.nih.gov/gene?term=SERPINE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.1</br>**STRING score:** 260</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CYCS and SERPINE1 include their roles in cellular and replicative senescence. CYCS plays a role in apoptosis by releasing cytochrome c into the cytosol and activating caspases, while SERPINE1 is involved in the regulation of cell migration, as well as being a primary inhibitor of tissue-type plasminogen activator and urokinase-type plasminogen activator. There is no clear evidence of direct functional associations between these two genes.|
| [EXOSC4](https://www.ncbi.nlm.nih.gov/gene?term=EXOSC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between EXOSC4 and HSPA8 could be in the regulation of RNA processing and degradation events, as both genes have been implicated in these processes. EXOSC4 is a component of the RNA exosome complex while HSPA8 is involved in chaperone-mediated autophagy and the protein quality control system. It is possible that these genes work together to ensure proper RNA degradation and protein folding.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLCB3](https://www.ncbi.nlm.nih.gov/gene?term=PLCB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.2</br>**STRING score:** 690</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | Possible functional association: EGFR is a receptor tyrosine kinase involved in regulating cell proliferation, differentiation, and survival, while PLCB3 is a phospholipase C enzyme involved in producing second messenger molecules. They may be functionally associated in signaling pathways that regulate cell growth and survival.|
| [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear biological or functional association between MRPL18 and RASA1 based on their described functions and interactions. MRPL18 is involved in mitochondrial ribosome assembly and RNA chaperoning, while RASA1 is an inhibitory regulator of the Ras-cyclic AMP pathway.|
| [EZH2](https://www.ncbi.nlm.nih.gov/gene?term=EZH2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.7</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 265 (min len 3) | Possible functional associations between EZH2 and HSPA8 include regulation of protein quality control and chaperone-mediated autophagy, as both genes are involved in these processes. EZH2 is a histone methyltransferase that is part of the PRC2/EED-EZH2 complex, which also interacts with HSPA8, and HSPA8 is a molecular chaperone implicated in protein folding and transport.|
