| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ATF1](https://www.ncbi.nlm.nih.gov/gene?term=ATF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between ATF1 and JMJD6 are their interaction with the BRD4:P-TEFb complex in transcriptional activation and regulation of pre-mRNA splicing of U2AF2/U2AF65. ATF1 binds to CRE and mediates PKA-induced stimulation of CRE-reporter genes, while JMJD6 acts as a lysyl-hydroxylase and arginine demethylase, regulating RNA splicing, angiogenic sprouting, and cytokine responses.|
| [IARS2](https://www.ncbi.nlm.nih.gov/gene?term=IARS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB10](https://www.ncbi.nlm.nih.gov/gene?term=PSMB10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between IARS2 and PSMB10 may involve antigen processing and immune response, as PSMB10 is involved in antigen processing and IARS2 may play a role in modulating signaling pathways in response to ligand-mediated activation. However, no direct biological association between these two genes has been reported in the literature.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FKBP14](https://www.ncbi.nlm.nih.gov/gene?term=FKBP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on the provided context and biology, it is difficult to determine any specific functional associations between FAM20B and FKBP14. FAM20B is involved in the regulation of glycosaminoglycan-protein linkage regions of proteoglycans, while FKBP14 is involved in protein folding and has a preference for substrates containing 4-hydroxylproline modifications, including type III collagen. The shortest path in the BioGRID protein interaction network between these two genes involves the mediating factor ELAVL1.|
| [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between PLSCR1 and RBM34. PLSCR1 is involved in lipid scrambling, apoptosis, and antiviral response, while RBM34 is involved in EGF stimulation, magnesium reabsorption, and protein degradation through ERAD. The protein interaction network shows a shortest path of interaction through ILF3, APP, and ESR2.|
| [CDC25B](https://www.ncbi.nlm.nih.gov/gene?term=CDC25B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between CDC25B and PLS1 include their involvement in cell cycle progression and actin filament organization respectively. The shortest path in the BioGRID interaction network suggests a potential physical interaction between the two proteins. However, further research is needed to understand the specific biological relevance of any such interaction.|
| [SH3BP5](https://www.ncbi.nlm.nih.gov/gene?term=SH3BP5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR61](https://www.ncbi.nlm.nih.gov/gene?term=WDR61[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between SH3BP5 and WDR61 based on their known biological roles. SH3BP5 functions as a guanine nucleotide exchange factor and negative regulator of BTK-related cytoplasmic signaling, while WDR61 is a component of the PAF1 complex with functions in transcriptional elongation and RNA decay.|
| [PTPRK](https://www.ncbi.nlm.nih.gov/gene?term=PTPRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | PTPRK is a negative regulator of EGFR signaling and is involved in regulation of processes involving cell contact and adhesion. RGS2 inhibits signal transduction pathways and plays a role in regulating blood pressure. The shortest path between PTPRK and RGS2 in the BioGRID protein interaction network is through EGFR. Possible functional association between PTPRK and RGS2 could be in regulation of signaling pathways and cytoskeletal organization.|
| [BID](https://www.ncbi.nlm.nih.gov/gene?term=BID[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between BID and MAPK1IP1L may involve regulation of apoptosis and transcriptional activity, respectively. However, without further information or experimental data, it is difficult to determine if and how these genes may directly interact or influence each other in a specific biological context.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COASY](https://www.ncbi.nlm.nih.gov/gene?term=COASY[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There is no clear functional association between CD44 and COASY based on their known biology and the provided context. CD44 is a cell-surface receptor involved in cell adhesion and migration, while COASY is a bifunctional enzyme involved in CoA biosynthesis regulation. The shortest path in the BioGRID protein interaction network between CD44 and COASY involves several intermediate proteins including EGFR, SRC, KRAS, HDAC2, and KIF23.|
| [PHKG2](https://www.ncbi.nlm.nih.gov/gene?term=PHKG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between PHKG2 and PIK3R4 might be related to their involvement in the regulation of glycogen breakdown and phosphatidylinositol 3-phosphate formation, respectively. They may interact in the context of PI3K complex II, which is involved in maturation of autophagosomes and endocytosis.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.8</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 872 (min len 4) | ICAM1 and XBP1 are two genes that have a possible functional association through their interaction in the BioGRID protein interaction network. ICAM1 is involved in immune response and microbial infection while XBP1 is a transcriptional activator involved in ER stress and the production of immunoglobulins and interleukin-6. It is possible that ICAM1 and XBP1 may have a role in regulating immune response during ER stress.|
| [RELB](https://www.ncbi.nlm.nih.gov/gene?term=RELB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.4</br>**STRING score:** 197</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | RELB and SMARCC1 are involved in transcriptional regulation and chromatin remodeling, respectively. As both genes are part of the BioGRID protein interaction network and have a short path between them, they may function together in a complex or pathway related to gene expression and chromatin structure.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SERPINE1](https://www.ncbi.nlm.nih.gov/gene?term=SERPINE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.1</br>**STRING score:** 260</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no evident functional association between CYCS and SERPINE1. CYCS plays a role in apoptosis, while SERPINE1 is a serine protease inhibitor involved in blood clot regulation, cell migration, and senescence.|
| [EXOSC4](https://www.ncbi.nlm.nih.gov/gene?term=EXOSC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: EXOSC4 and HSPA8 may work together to ensure proper folding and transport of newly synthesized polypeptides. EXOSC4 is involved in RNA processing and degradation, while HSPA8 is a molecular chaperone that plays a pivotal role in the protein quality control system. There is no direct evidence of physical interaction or functional association between these two proteins based on the provided context.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLCB3](https://www.ncbi.nlm.nih.gov/gene?term=PLCB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.2</br>**STRING score:** 690</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | EGFR is a receptor protein involved in growth factor signaling pathways and is associated with cell proliferation and survival. PLCB3 encodes for protein involved in the regulation of intracellular calcium levels. Based on their roles, these genes could have functional associations in cell signaling pathways and calcium homeostasis.|
| [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between MRPL18 and RASA1 based on the provided context. MRPL18 is involved in mitochondrial ribosome subunits while RASA1 is an inhibitory regulator of the Ras-cyclic AMP pathway.|
| [EZH2](https://www.ncbi.nlm.nih.gov/gene?term=EZH2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.7</br>**STRING score:** 221</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 265 (min len 3) | There are no obvious functional associations between EZH2 and HSPA8. EZH2 is a polycomb group protein involved in histone methylation and gene repression, while HSPA8 is a molecular chaperone involved in protein quality control and mitochondrial import. The shortest path between the two genes in the BioGRID protein interaction network involves multiple intermediate proteins.|
