| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 293 (min len 4) | Possible functional association: Both DUSP4 and XBP1 are involved in regulating stress responses, with DUSP4 regulating MAPK signaling and XBP1 regulating the unfolded protein response in the endoplasmic reticulum. They may play a role in coordinating stress responses in cancer cells.|
| [SOCS2](https://www.ncbi.nlm.nih.gov/gene?term=SOCS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF451](https://www.ncbi.nlm.nih.gov/gene?term=ZNF451[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between SOCS2 and ZNF451 based on the provided context. SOCS2 is involved in negative regulation of cytokine signaling and target protein degradation, while ZNF451 is an E3 SUMO-protein ligase and negative regulator of transcriptional activation mediated by the SMAD4 complex in response to TGF-beta signaling.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ICAM1 and PTPN12 include regulation of cellular signaling cascades, dephosphorylation of ERBB2 and PTK2B/PYK2, and interaction with TGFB1I1, PSTPIP1, LPXN, and SORBS2. These genes may play a role in microbial infection, and ICAM1 specifically interacts with Coxsackievirus A21 capsid proteins.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 117 (min len 3) | Possible functional associations between HSPA8 and PGRMC1 include their interaction in the progesterone-binding protein complex, their roles in heme homeostasis and regulation of heme synthesis, and their involvement in protein quality control through chaperone-mediated autophagy and the ER-associated degradation pathway. They may also have a scaffolding role in the spliceosome assembly and be required for activating pre-mRNA splicing.|
| [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SH3BP5](https://www.ncbi.nlm.nih.gov/gene?term=SH3BP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | PSMD2 is a component of the proteasome regulatory particle complex involved in protein degradation. SH3BP5 functions as a guanine nucleotide exchange factor and interacts with MAPK8. There are no clear functional associations between these two genes in specific biological terms based on the information provided.|
| [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MEF2C](https://www.ncbi.nlm.nih.gov/gene?term=MEF2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.5</br>**STRING score:** 239</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CD44 and MEF2C include regulation of cell adhesion, migration, and survival. CD44 is a cell surface receptor involved in these processes, while MEF2C is a transcription factor that controls gene expression and has been implicated in the regulation of B-cell survival and proliferation. The shortest path between these two proteins in the BioGRID protein interaction network involves EP300, MYC, and phosphorylation pathways involving RhoGTPases and PKN2.|
| [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRAF2](https://www.ncbi.nlm.nih.gov/gene?term=PRAF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association: Both DNAJB6 and PRAF2 are involved in protein transport and may interact in this process; DNAJB6 inhibits huntingtin aggregation while PRAF2 plays a proapoptotic role in apoptosis.|
| [SH3BP5](https://www.ncbi.nlm.nih.gov/gene?term=SH3BP5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP1](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between SH3BP5 and TP53BP1 may involve DNA damage response and repair processes. SH3BP5 may interact with MAPKs and inhibit B-cell cytoplasmic signaling, while TP53BP1 plays a key role in non-homologous end joining-mediated DNA repair and immunoglobulin class-switch recombination during antibody genesis. The shortest path between these two genes in the BioGRID protein interaction network involves intermediary proteins MOV10, NXF1, and TRIM67.|
| [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are currently no known functional associations between NOLC1 and SLC1A4. NOLC1 is a regulator of RNA polymerase I and nucleologenesis, while SLC1A4 is a transporter of amino acids. Their shortest path interactions suggest possible indirect connections through MYC, RNF4, HSPA1A, and TRIM67.|
| [MRPL18](https://www.ncbi.nlm.nih.gov/gene?term=MRPL18[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XBP1](https://www.ncbi.nlm.nih.gov/gene?term=XBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 406 (min len 4) | Possible functional associations between MRPL18 and XBP1 include their interaction through the SRSF1 protein in the BioGRID protein interaction network, as well as XBP1's role in the unfolded protein response and protein folding, while MRPL18 is involved in mitochondrial ribosome assembly and RNA chaperone activity. However, their specific biological connection in TCGA-LUAD requires further investigation.|
| [ADRB2](https://www.ncbi.nlm.nih.gov/gene?term=ADRB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CD44](https://www.ncbi.nlm.nih.gov/gene?term=CD44[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.8</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association between ADRB2 and CD44 could be in cell migration and adhesion processes, as both genes are involved in signaling pathways promoting these processes through interactions with extracellular matrix components and cytoskeleton reorganization. ADRB2 is also involved in the activation of adenylate cyclase through G proteins, while CD44 plays a role in T-cell activation, hematopoiesis, inflammation, and response to bacterial infection.|
| [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCYL3](https://www.ncbi.nlm.nih.gov/gene?term=SCYL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.0</br>**STRING score:** 298</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between COPB2 and SCYL3 may involve vesicular trafficking and cell adhesion/migration complexes. They may interact to regulate these processes and potentially play a role in cancer progression.|
| [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKACA](https://www.ncbi.nlm.nih.gov/gene?term=PRKACA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.1</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 41 (min len 3) | Possible functional associations between HNRNPA2B1 and PRKACA may involve RNA transport and protein phosphorylation. HNRNPA2B1 is involved in transport of HIV-1 genomic RNA, while PRKACA phosphorylates and activates numerous proteins, including ABL1 and regulatory subunits of PKA. The shortest path in the BioGRID protein interaction network suggests that HNRNPA2B1 and PRKACA may interact through other proteins, such as HSPD1, CUL3, and TPR.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FGFR2](https://www.ncbi.nlm.nih.gov/gene?term=FGFR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.4</br>**STRING score:** 477</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 107 (min len 3) | Possible functional association between EGFR and FGFR2 is their interaction in the MAPK/ERK pathway for promoting cell proliferation, differentiation, and migration. EGFR also acts as a receptor for hepatitis C virus in hepatocytes, while FGFR2 is essential for normal embryonic patterning, lung morphogenesis, osteogenesis, and skin development.|
| [FGFR2](https://www.ncbi.nlm.nih.gov/gene?term=FGFR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLCB3](https://www.ncbi.nlm.nih.gov/gene?term=PLCB3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.3</br>**STRING score:** 639</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: FGFR2 and PLCB3 both play a role in the production of diacylglycerol and inositol 1,4,5-trisphosphate, important second messenger molecules involved in intracellular signaling. The shortest path between these two genes in the BioGRID protein interaction network involves PLCG1, which is phosphorylated by FGFR2 and activates PLCB3.|
| [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.9</br>**STRING score:** 190</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 772 (min len 4) | CDKN1B and KDM5A interact with each other through a shortest protein path of 4 proteins. CDKN1B is an inhibitor of cell cycle progression and forms complexes with cyclin-CDK complexes, while KDM5A is a histone demethylase involved in chromatin remodeling and transcriptional regulation. Possible functional associations between these two genes could involve the regulation of gene transcription during cell cycle progression.|
| [CCNE2](https://www.ncbi.nlm.nih.gov/gene?term=CCNE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.7</br>**STRING score:** 453</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CCNE2 is involved in cell cycle regulation, while SMARCA4 is involved in chromatin remodeling and transcriptional regulation. The short protein interaction path between these two genes suggests a potential functional association in regulating gene expression during cell cycle progression and differentiation, through a complex interplay of chromatin remodeling and transcriptional regulation mechanisms.|
| [BNIP3L](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.0</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | BNIP3L induces apoptosis and is involved in mitochondrial quality control, whereas HDAC2 is a histone deacetylase that regulates transcriptional repression. The shortest path between the two genes in the BioGRID interaction network suggests that they may functionally associate through EWSR1, LMNA, BAG3, or CIT in specific biological processes such as chromatin remodeling, transcriptional regulation, or cellular apoptosis.|
| [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.5</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations: CETN3 and HSPA8 are connected in a protein interaction network via several short paths involving different middle proteins, indicating that they may be functionally associated. CETN3 is involved in nuclear export and may interact with HSPA8 as a molecular chaperone, which is involved in protein folding, quality control, and degradation.|
| [CD58](https://www.ncbi.nlm.nih.gov/gene?term=CD58[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HERPUD1](https://www.ncbi.nlm.nih.gov/gene?term=HERPUD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known direct functional associations between CD58 and HERPUD1. CD58 is involved in T-cell interactions, while HERPUD1 is part of the endoplasmic reticulum quality control system. Individually, they do not appear to have any overlapping functions.|
| [SH3BP5](https://www.ncbi.nlm.nih.gov/gene?term=SH3BP5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.0</br>**STRING score:** 342</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between SH3BP5 and STK10 could be involvement in lymphocyte migration and regulation of cytoplasmic signaling in B-cells. The shortest path between these genes in the BioGRID protein interaction network suggests they may interact indirectly through intermediate proteins such as MOV10, NXF1, XPO1, and TRIM67.|
