| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PCM1](https://www.ncbi.nlm.nih.gov/gene?term=PCM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between NMT1 and PCM1 involve centrosome assembly and function, microtubule anchoring, and cilium biogenesis. Specifically, PCM1 is required for centrosome assembly and function, while NMT1 interacts with proteins involved in cilium biogenesis and microtubule anchoring. The shortest path between NMT1 and PCM1 in the BioGRID protein interaction network indicates potential direct interactions between these two genes through intermediary proteins.|
| [NFKBIE](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between NFKBIE and PIK3R4 are unclear based on the provided information. NFKBIE inhibits NF-kappa-B signaling, while PIK3R4 is a regulatory subunit of the PI3K complex involved in various membrane trafficking pathways and cytokinesis. However, both genes have been implicated in cancer through their roles in regulating cellular processes such as inflammation, apoptosis, and cell proliferation.|
| [PIK3R4](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are no clear biological functional associations between PIK3R4 and SLC25A46 based on their known roles in regulation of phosphatidylinositol 3-kinase complex and mitochondrial organization, respectively. The shortest path between these genes in the BioGRID protein interaction network suggests potential indirect interactions through intermediate proteins such as RRBP1 and LAMTOR1.|
| [MAP7](https://www.ncbi.nlm.nih.gov/gene?term=MAP7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR2K](https://www.ncbi.nlm.nih.gov/gene?term=POLR2K[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations: MAP7 and POLR2K may be involved in microtubule organization and RNA polymerase activity, respectively. There may be potential roles for MAP7 in transcriptional regulation or mRNA transport. Alternatively, both genes could play a role in epithelial cell differentiation and intercellular communication through direct or indirect interactions.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIGB](https://www.ncbi.nlm.nih.gov/gene?term=PIGB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 45 (min len 4) | There is no clear functional association between LYPLA1 and PIGB based on the provided context. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while PIGB is involved in glycosylphosphatidylinositol-anchor biosynthesis. However, they both interact with RPN1, a component of the proteasome regulatory particle, indicating a possible connection to protein degradation pathways.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between FRS2 and NFKBIB: FRS2 plays a role in the activation of MAP kinases while NFKBIB inhibits NF-kappa-B. Both genes are important in cellular signaling pathways and could potentially interact in regulating cellular responses. The shortest path in the BioGRID protein interaction network involves EGFR as a mediator.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2L6](https://www.ncbi.nlm.nih.gov/gene?term=UBE2L6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between IDE and UBE2L6 in TCGA-LUAD may involve regulation of protein degradation or modification. IDE plays a role in regulating protein turnover and has been implicated in viral entry, while UBE2L6 catalyzes the covalent attachment of ubiquitin to target proteins for degradation. The protein interaction network suggests a possible link between the two genes via the E3 ubiquitin ligase, PARK2.|
| [CBR3](https://www.ncbi.nlm.nih.gov/gene?term=CBR3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRK](https://www.ncbi.nlm.nih.gov/gene?term=PTPRK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is not enough information to speculate about any functional associations between CBR3 and PTPRK. CBR3 is involved in reducing carbonyl compounds to alcohols while PTPRK acts as a negative regulator of the EGFR signaling pathway and forms complexes with beta-catenin and gamma-catenin/plakoglobin.|
| [ATF1](https://www.ncbi.nlm.nih.gov/gene?term=ATF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KCTD2](https://www.ncbi.nlm.nih.gov/gene?term=KCTD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between ATF1 and KCTD2 could be related to chromatin remodeling and transcriptional regulation, as both proteins are involved in these processes within the SWI/SNF chromatin remodeling complex. Additionally, ATF1 has been shown to activate cell proliferation and transformation, while KCTD2 may play a role in BRD4-mediated gene transcription.|
| [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP2](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is no direct known functional association between GRN and TP53BP2. GRN plays a role in lysosomal targeting and proteolytic activity, while TP53BP2 regulates apoptosis and cell growth via interactions with proteins such as TP53.|
| [BECN1](https://www.ncbi.nlm.nih.gov/gene?term=BECN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NPC1](https://www.ncbi.nlm.nih.gov/gene?term=NPC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.2</br>**STRING score:** 454</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: BECN1 and NPC1 are involved in microbial infection. BECN1 plays a role in inducing mitochondrial translocation of BAX, while NPC1 acts as an endosomal entry receptor for ebolavirus. There is no known direct functional association between these two genes.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.9</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: Both genes are involved in protein folding and may interact through TCP1 complex. GLOD4 may inhibit LPXN nuclear accumulation, while TXNDC9 negatively impacts protein folding, including actin and tubulin.|
| [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FGFR4](https://www.ncbi.nlm.nih.gov/gene?term=FGFR4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: There is no known direct functional association between CTNNAL1 and FGFR4; they are only connected in the BioGRID protein interaction network by a single shortest path.          Brief description of CTNNAL1: CTNNAL1 may modulate the Rho pathway signaling by interacting with the Rho guanine nucleotide exchange factor ARHGEF1, providing a scaffold for its activity.          Brief description of FGFR4: FGFR4 is a cell-surface receptor for fibroblast growth factors and plays a role in regulation of cell proliferation, differentiation, migration, and metabolism. Mutations that impair normal FGFR4 inactivation lead to aberrant signaling.|
| [BECN1](https://www.ncbi.nlm.nih.gov/gene?term=BECN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53BP2](https://www.ncbi.nlm.nih.gov/gene?term=TP53BP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.2</br>**STRING score:** 436</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between BECN1 and TP53BP2 in TCGA-LUAD could be related to the regulation of apoptosis and cell growth, as both genes have been shown to play a role in these processes. Additionally, BECN1 has been shown to interact with the herpes simplex virus 1 protein ICP34.5, while TP53BP2 interacts with proteins such as TP53, BCL2, and RELA NF-kappa-B subunit.|
| [ANXA7](https://www.ncbi.nlm.nih.gov/gene?term=ANXA7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.5</br>**STRING score:** 490</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: TP53 is a tumor suppressor gene involved in cell cycle regulation and apoptosis induction, while ANXA7 promotes membrane fusion and exocytosis. There is no direct biological evidence for functional association between them in the given context.|
| [ANXA7](https://www.ncbi.nlm.nih.gov/gene?term=ANXA7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DHRS7](https://www.ncbi.nlm.nih.gov/gene?term=DHRS7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.9</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between ANXA7 and DHRS7 include involvement in cellular processes such as exocytosis, membrane fusion, and biotransformation of xenobiotics. There is no clear indication of a direct functional association between these two genes.|
| [CCNE2](https://www.ncbi.nlm.nih.gov/gene?term=CCNE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDC25B](https://www.ncbi.nlm.nih.gov/gene?term=CDC25B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 734</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CCNE2 and CDC25B include their involvement in regulating cell cycle progression as part of a kinase/phosphatase complex. CCNE2 as a cyclin subunit imparts substrate specificity to the CDK2 and CDK3 kinases, which in turn can activate CDC25B through direct dephosphorylation of CDK1, promoting the G2/M transition and cytokinesis.|
| [CETN3](https://www.ncbi.nlm.nih.gov/gene?term=CETN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NDUFC1](https://www.ncbi.nlm.nih.gov/gene?term=NDUFC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.2</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 4) | There are no obvious functional associations between CETN3 and NDUFC1 based solely on their roles in mRNA export and mitochondrial respiration, respectively. However, as they are both part of protein interaction networks involving common interactors, future studies may reveal novel connections between these genes in specific biological processes.|
| [GLOD4](https://www.ncbi.nlm.nih.gov/gene?term=GLOD4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL19](https://www.ncbi.nlm.nih.gov/gene?term=MRPL19[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.0</br>**STRING score:** 323</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There isn't any clear biological process that would suggest a functional association between GLOD4 and MRPL19. GLOD4 inhibits PTK2/FAK1 phosphorylation signaling, while MRPL19 acts as a ligand for C-C chemokine receptor CCR2 and is a component of the mitochondrial large ribosomal subunit.|
| [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM2](https://www.ncbi.nlm.nih.gov/gene?term=TMEM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context, there seem to be no specific functional associations between RAC2 and TMEM2. RAC2 is involved in regulating cellular responses and augmenting ROS production, while TMEM2 mediates the cleavage of extracellular hyaluronan and acts as a regulator of angiogenesis and heart morphogenesis.|
