| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [TMCO6](https://www.ncbi.nlm.nih.gov/gene?term=TMCO6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | TMCO6 is involved in the respiratory chain, while UBE2C is a component of the APC/C complex that controls progression through mitosis. There may be a possible association between these two genes in the regulation of cell cycle progression through the degradation of key substrates during mitosis.|
| [HMOX1](https://www.ncbi.nlm.nih.gov/gene?term=HMOX1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UXT](https://www.ncbi.nlm.nih.gov/gene?term=UXT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: Both HMOX1 and UXT have been shown to play a role in cellular responses to stressors such as microbial infection and TNF-alpha-induced apoptosis. It is possible that these genes may interact in some way in these stress responses.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SEC11A](https://www.ncbi.nlm.nih.gov/gene?term=SEC11A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There does not seem to be a clear functional association between APOE and SEC11A based on their known biological roles and the provided context. APOE is involved in microbial infection, particularly with hepatitis C virus, while SEC11A is a catalytic component of the signal peptidase complex involved in cleaving signal sequences from nascent proteins during translocation into the endoplasmic reticulum.|
| [CANT1](https://www.ncbi.nlm.nih.gov/gene?term=CANT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear biological functional association between CANT1 and SKP1 based on their known functions. CANT1 is involved in proteoglycan synthesis and is a calcium-dependent nucleotidase, while SKP1 is an adapter protein in the SCF ubiquitin ligase complex involved in cell cycle progression and signal transduction. The shortest path between them in the BioGRID protein interaction network is ORF6.|
| [HADH](https://www.ncbi.nlm.nih.gov/gene?term=HADH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations: HADH is involved in mitochondrial fatty acid beta-oxidation, while RAE1 functions as an mRNA export factor and spindle formation regulator. There is no known direct functional association between these two genes based on the provided context and biological processes they are involved in.|
| [ALDOC](https://www.ncbi.nlm.nih.gov/gene?term=ALDOC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: There is a protein interaction network shortest path between ALDOC and LSM6 via ZRANB1. ALDOC is involved in modulating G proteins and phospholipase C, while LSM6 plays a role in pre-mRNA splicing and RNA processing, suggesting a potential involvement in regulating gene expression in liver cancer.|
| [EPB41L2](https://www.ncbi.nlm.nih.gov/gene?term=EPB41L2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is not enough information available to reason about any functional associations between these two genes. EPB41L2 is involved in mitotic cell cortex recruitment and interacts with NUMA1 and FCGR1A, while TCFL5 is a putative transcription factor that may play a role in early spermatogenesis.|
| [NFIL3](https://www.ncbi.nlm.nih.gov/gene?term=NFIL3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 154 (min len 4) | Possible functional associations between NFIL3 and UBE2C include UBE2C's role in ubiquitination and degradation of proteins involved in cell cycle regulation, which may impact NFIL3's function as a transcriptional regulator and circadian clock component. NFIL3 has also been shown to interact with DR1, which is involved in transcriptional regulation and DNA binding, and may be connected to UBE2C through shared interaction partners.|
| [ABCF3](https://www.ncbi.nlm.nih.gov/gene?term=ABCF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: ABCF3 displays an antiviral effect, while CD40 is a receptor for TNFSF5/CD40LG, transducing signals that activate ERK in macrophages and B cells, leading to induction of immunoglobulin secretion. No direct biological association is apparent between ABCF3 and CD40.|
| [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 4) | There is no known functional association between HLA-DMA and PUM2. HLA-DMA plays a role in peptide presentation, while PUM2 is a post-transcriptional regulator of mRNA. The only interaction between these two genes is via protein-protein interaction network, and the pathway relevance of this interaction is unknown in terms of a biological process.|
| [RAD9A](https://www.ncbi.nlm.nih.gov/gene?term=RAD9A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.9</br>**STRING score:** 290</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | RAD9A is involved in DNA repair through its association with the 9-1-1 complex, while SKP1 acts as an adapter in the SCF ubiquitin ligase complex, mediating the ubiquitination of proteins involved in cell cycle progression and transcription. The interaction between RAD9A and SKP1 in the BioGRID protein interaction network suggests a possible functional association between these two genes in regulating cell cycle progression and DNA repair.|
| [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.6</br>**STRING score:** 343</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between EIF4EBP1 and PARP1 are unclear based on the provided context and knowledge of their biological functions. EIF4EBP1 is involved in the regulation of protein translation by signaling pathways such as MAP kinase and mTORC1, and PARP1 is involved in DNA repair and apoptosis. While both genes play important roles in cellular processes, their specific functions do not appear to intersect.|
| [GPC1](https://www.ncbi.nlm.nih.gov/gene?term=GPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.2</br>**STRING score:** 276</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context, there is no known functional association between GPC1 and RPS5. GPC1 is involved in cell surface proteoglycan binding, skeletal muscle differentiation, and endolymph production in the inner ear, while RPS5 is a component of the small ribosomal subunit involved in protein synthesis.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.8</br>**STRING score:** 991</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: CCNA2 and SKP1 both play roles in cell cycle progression and ubiquitination of proteins. The shortest path between them in the BioGRID protein interaction network suggests a direct interaction.|
| [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP16](https://www.ncbi.nlm.nih.gov/gene?term=USP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.1</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between UBE2C and USP16 could include a role in regulating chromosome segregation during mitosis, as both genes have been implicated in this process. UBE2C is part of the anaphase promoting complex/cyclosome (APC/C), which is crucial for mitotic exit, while USP16 deubiquitinates histone H2A, a process that is necessary for proper chromosome segregation.|
| [DERA](https://www.ncbi.nlm.nih.gov/gene?term=DERA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.4</br>**STRING score:** 811</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between DERA and HK1 is their involvement in the glycolytic pathway. DERA catalyzes a reversible aldol reaction in the pathway, while HK1 mediates the initial step of glycolysis, phosphorylating hexoses to hexose 6-phosphate. The BioGRID protein interaction network shows a shortest path between DERA and HK1, suggesting they interact in some biological processes.|
| [ADH5](https://www.ncbi.nlm.nih.gov/gene?term=ADH5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.1</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: KDM5B interacts with HDAC1, HDAC4, HDAC5 and HDAC7, which are involved in histone modification and transcriptional regulation. ADH5 is involved in oxidation of long-chain alcohols and fatty acids, as well as the clearance of formaldehyde. It is possible that ADH5 may play a role in regulating the activity or expression of KDM5B's interacting proteins, such as histone deacetylases, in the context of liver cancer.|
| [ATP5B](https://www.ncbi.nlm.nih.gov/gene?term=ATP5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.0</br>**STRING score:** 515</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no direct evidence of functional association between ATP5B and LYPLA1, but they are both involved in mitochondrial function. ATP5B is a subunit of ATP synthase, which produces ATP from ADP using a proton gradient generated by electron transport. LYPLA1 is an acyl-protein thioesterase that hydrolyzes fatty acids from S-acylated cysteine residues in proteins, as well as lysophospholipids.|
| [ABCF3](https://www.ncbi.nlm.nih.gov/gene?term=ABCF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.9</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no clear association between ABCF3 and NVL based on their known functions and protein interactions. ABCF3 displays an antiviral effect and associates with ATP synthase, while NVL participates in telomerase activity, ribosomal subunit biogenesis, and interacts with various protein factors involved in RNA processing.|
