| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [TMCO6](https://www.ncbi.nlm.nih.gov/gene?term=TMCO6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between TMCO6 and UBE2C could involve the regulation of protein degradation during mitosis. TMCO6 is part of the mitochondrial respiratory chain complex, while UBE2C is involved in protein ubiquitination and degradation by the anaphase promoting complex/cyclosome. Their interaction through UBQLN1 may suggest a role in regulating protein turnover during cell division.|
| [HMOX1](https://www.ncbi.nlm.nih.gov/gene?term=HMOX1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UXT](https://www.ncbi.nlm.nih.gov/gene?term=UXT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between HMOX1 and UXT include their roles in cellular response to stress and inflammation, where HMOX1 catalyzes heme degradation to produce anti-inflammatory biliverdin and UXT protects cells from TNF-alpha-induced apoptosis. They may also be involved in autophagy modulation, as HMOX1 interacts with SARS-CoV-2 ORF3A protein to promote autophagy and UXT is part of the TNF-alpha receptor complex I that regulates autophagy induction.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SEC11A](https://www.ncbi.nlm.nih.gov/gene?term=SEC11A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are no clear functional associations between APOE and SEC11A based on the provided context. APOE is involved in microbial infection and interactions with HCV, while SEC11A is involved in catalyzing the cleavage of signal peptides during protein translocation into the endoplasmic reticulum.|
| [CANT1](https://www.ncbi.nlm.nih.gov/gene?term=CANT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between CANT1 and SKP1 is their involvement in proteoglycan synthesis and ubiquitination, respectively. It is unclear whether there are direct biological interactions between these two processes.|
| [HADH](https://www.ncbi.nlm.nih.gov/gene?term=HADH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAE1](https://www.ncbi.nlm.nih.gov/gene?term=RAE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no known direct functional association between HADH and RAE1, as they are involved in distinct biological processes - HADH in fatty acid oxidation and insulin regulation, and RAE1 in mRNA export and spindle formation. The only interaction between them is through NXF1, TXN, M, CS, and METTL17 in the BioGRID protein interaction network.|
| [ALDOC](https://www.ncbi.nlm.nih.gov/gene?term=ALDOC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LSM6](https://www.ncbi.nlm.nih.gov/gene?term=LSM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: Both ALDOC and LSM6 are involved in RNA processing, with LSM6 playing a role in pre-mRNA splicing and ALDOC potentially interacting with PLD2. It is possible that these genes may be involved in regulating gene expression or RNA metabolism in specific biological contexts.|
| [EPB41L2](https://www.ncbi.nlm.nih.gov/gene?term=EPB41L2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | A possible functional association between EPB41L2 and TCFL5 is that EPB41L2 may regulate TCFL5's function as a transcription factor in early spermatogenesis. EPB41L2 is also involved in mitotic cell cortex recruitment and interacts with NUMA1 and FCGR1A, while TCFL5 requires dimerization with other bHLH proteins for efficient DNA binding.|
| [NFIL3](https://www.ncbi.nlm.nih.gov/gene?term=NFIL3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 154 (min len 4) | Possible functional association: UBE2C is involved in the ubiquitination of proteins and is a component of the anaphase promoting complex/cyclosome, while NFIL3 is a transcriptional regulator that binds to specific sequences in promoters and represses transcription. The shortest path in the BioGRID protein interaction network suggests that NFIL3 and UBE2C may interact through the protein ELAVL1, but the exact nature of this interaction is unclear.|
| [ABCF3](https://www.ncbi.nlm.nih.gov/gene?term=ABCF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association between ABCF3 and CD40 is related to the immune response. ABCF3 displays an antiviral effect against flaviviruses, and CD40 is a receptor for TNFSF5/CD40LG, which activates ERK in macrophages and B cells, leading to immunoglobulin secretion. The shortest path between them in the BioGRID protein interaction network involves several proteins related to intracellular transport and signaling, which may play a role in regulating these immune-related functions.|
| [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 4) | There is currently no known functional association between HLA-DMA and PUM2. HLA-DMA plays a critical role in catalyzing the release of class II-associated invariant chain peptide (CLIP) from newly synthesized MHC class II molecules. PUM2 acts as a post-transcriptional repressor by binding to the 3'-UTR of mRNA targets.|
| [RAD9A](https://www.ncbi.nlm.nih.gov/gene?term=RAD9A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 69.9</br>**STRING score:** 290</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | RAD9A and SKP1 are connected in the BioGRID protein interaction network through several interactions. RAD9A plays a crucial role in DNA repair, while SKP1 is an essential component of the SCF ubiquitin ligase complex, which mediates the ubiquitination of various proteins. There could be a possible functional association between these genes in regulating different signaling pathways involved in DNA repair.|
| [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.6</br>**STRING score:** 343</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | There is no clear biological connection between EIF4EBP1 and PARP1. EIF4EBP1 regulates translation initiation by binding to EIF4E, while PARP1 is involved in DNA damage repair and apoptosis. However, there is a shortest path of protein interactions between the two genes, suggesting indirect functional associations.|
| [GPC1](https://www.ncbi.nlm.nih.gov/gene?term=GPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.2</br>**STRING score:** 276</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no clear functional associations between GPC1 and RPS5 based on their known biological roles and interactions. GPC1 is a cell surface proteoglycan involved in various signaling pathways and protein interactions, while RPS5 is a ribosomal protein essential for protein synthesis.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.8</br>**STRING score:** 991</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | CCNA2 and SKP1 are functionally associated through the SCF ubiquitin ligase complex, where SKP1 serves as an adapter linking F-box proteins to CUL1. CCNA2 is a cyclin-dependent protein kinase activator that controls cell cycle progression through G1/S and G2/M phases, while SKP1 is involved in mediating the ubiquitination of proteins involved in cell cycle progression and signaling. They likely have a role in regulating cell cycle and protein degradation pathways.|
| [UBE2C](https://www.ncbi.nlm.nih.gov/gene?term=UBE2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP16](https://www.ncbi.nlm.nih.gov/gene?term=USP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.1</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Both UBE2C and USP16 are involved in ubiquitination processes important for cell cycle regulation. They are also known to interact with each other in the BioGRID protein interaction network, potentially affecting histone H2A deubiquitination and Hox gene expression. This suggests a possible functional association in regulating gene expression and chromatin structure.|
| [DERA](https://www.ncbi.nlm.nih.gov/gene?term=DERA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.4</br>**STRING score:** 811</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: Both DERA and HK1 are involved in glucose metabolism. DERA catalyzes a reversible aldol reaction while HK1 initiates glycolysis by phosphorylating D-glucose. Therefore, they may influence each other's activity in glucose metabolism. However, there is no direct evidence of their functional association.|
| [ADH5](https://www.ncbi.nlm.nih.gov/gene?term=ADH5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.1</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ADH5 and KDM5B could be related to cellular metabolism and gene expression regulation, where ADH5 is involved in the oxidative metabolism of alcohols and fatty acids, and KDM5B is a histone demethylase that plays a central role in regulating histone code and gene expression. The shortest path in the BioGRID protein interaction network connecting these two genes involves the chromatin remodeling factors CHD3 and HDACs, suggesting a potential role in epigenetic regulation of gene expression in liver cancer.|
| [ATP5B](https://www.ncbi.nlm.nih.gov/gene?term=ATP5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.0</br>**STRING score:** 515</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between ATP5B and LYPLA1 include involvement in energy metabolism and lipid modification, respectively. While they may not directly interact in a biological pathway, dysregulation of either gene could impact the function of the other given their key roles in cellular processes.|
| [ABCF3](https://www.ncbi.nlm.nih.gov/gene?term=ABCF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.9</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between ABCF3 and NVL include involvement in ribosome biogenesis and assembly, as well as telomerase activity. ABCF3 plays a role in antiviral defense and associates with ATP synthase, while NVL catalyzes the release of assembly factors from pre-60S ribosomal particles through ATPase activity and interacts with nucleolin and other proteins involved in ribosomal biogenesis. Overall, these genes may work together to regulate ribosome biogenesis and telomerase activity.|
