| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAPB](https://www.ncbi.nlm.nih.gov/gene?term=VAPB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Based on their interaction in the BioGRID network, HEBP1 and VAPB may be functionally associated in the endoplasmic reticulum unfolded protein response and calcium homeostasis regulation. HEBP1 binds to heme and porphyrins, potentially aiding in their removal as toxic compounds, while VAPB interacts with hepatitis C virus and RNA polymerase.|
| [ITGB5](https://www.ncbi.nlm.nih.gov/gene?term=ITGB5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXLNA](https://www.ncbi.nlm.nih.gov/gene?term=TXLNA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ITGB5 and TXLNA are not immediately clear from the provided context. ITGB5 functions as a receptor for adenovirus type C and interacts with other proteins involved in cell adhesion and migration, while TXLNA may be involved in vesicle traffic and exocytosis in neuroendocrine cells through its interactions with syntaxin family members.|
| [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SERPINE1](https://www.ncbi.nlm.nih.gov/gene?term=SERPINE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 121 (min len 4) | Possible functional association: SERPINE1 has been shown to regulate cell migration and DENND2D may activate RAB proteins involved in vesicle trafficking. It is possible that these genes may function together in regulating cellular processes that involve vesicle trafficking and cell migration.|
| [HEBP1](https://www.ncbi.nlm.nih.gov/gene?term=HEBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NNT](https://www.ncbi.nlm.nih.gov/gene?term=NNT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between HEBP1 and NNT in TCGA-LIHC cancer type include detoxification of potentially toxic compounds such as porphyrinogens and ROS, since HEBP1 can bind free porphyrinogens and NNT may play a role in ROS detoxification. They may also be involved in energy production, with NNT functioning as a proton pump across the membrane and HEBP1 potentially facilitating the removal of toxins to support ATP production.|
| [PPP2R5A](https://www.ncbi.nlm.nih.gov/gene?term=PPP2R5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFX5](https://www.ncbi.nlm.nih.gov/gene?term=RFX5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 263 (min len 4) | Possible functional association: PPP2R5A and RFX5 interact in the BioGRID protein interaction network. PPP2R5A is a regulatory subunit of the PP2A enzyme complex involved in signaling and cell cycle regulation while RFX5 is a transcription factor involved in MHC-II gene expression. These genes may have a role in regulating immune response mechanisms through the regulation of gene expression and signal transduction pathways.|
| [NCK2](https://www.ncbi.nlm.nih.gov/gene?term=NCK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PYGL](https://www.ncbi.nlm.nih.gov/gene?term=PYGL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between NCK2 and PYGL could be related to glucose homeostasis regulation and glycogen catabolism. NCK2 interacts with PPP1R15B, PP1, and TGFB1I1, which may affect PYGL through PPP1R3B to deactivate the enzyme in response to glucose levels.|
| [ITGB5](https://www.ncbi.nlm.nih.gov/gene?term=ITGB5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SDHB](https://www.ncbi.nlm.nih.gov/gene?term=SDHB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between ITGB5 and SDHB are unclear based on the provided context and knowledge about their respective biological activities. ITGB5 is involved in cell adhesion and has been implicated in adenovirus infection, while SDHB is a component of the mitochondrial respiratory chain complex II and has been shown to interact with the JC virus small t antigen.|
| [PYGL](https://www.ncbi.nlm.nih.gov/gene?term=PYGL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN3](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between PYGL and TSPAN3 based on their known biological roles and protein interactions. PYGL plays a role in glycogen catabolism and glucose homeostasis, while TSPAN3 is involved in oligodendrocyte proliferation and migration. The protein interaction network shows three shortest paths between the two genes, but the biological significance of these interactions is unclear.|
| [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFP36](https://www.ncbi.nlm.nih.gov/gene?term=ZFP36[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no obvious functional associations between DENND2D and ZFP36 based on their known functions. DENND2D is a guanine nucleotide exchange factor (GEF) that activates RAB9A and RAB9B, while ZFP36 negatively regulates HTLV-1 TAX-dependent transactivation of the viral long terminal repeat promoter.|
| [HTRA1](https://www.ncbi.nlm.nih.gov/gene?term=HTRA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIMM22](https://www.ncbi.nlm.nih.gov/gene?term=TIMM22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 4) | There are no known direct functional associations between HTRA1 and TIMM22. HTRA1 is a serine protease involved in various physiological processes, including degradation of extracellular matrix proteins and regulation of insulin-like growth factors. TIMM22 is essential for the import and insertion of multi-pass transmembrane proteins into the mitochondrial inner membrane.|
| [FOS](https://www.ncbi.nlm.nih.gov/gene?term=FOS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN6](https://www.ncbi.nlm.nih.gov/gene?term=PTPN6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 88.1</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | FOS and PTPN6 have a potential functional association via their protein interaction network. FOS is involved in signal transduction, cell proliferation, and formation of the skeleton. PTPN6 modulates signaling by tyrosine phosphorylated receptors and plays a key role in hematopoiesis. There is no known direct functional association between the two genes.|
| [IFRD2](https://www.ncbi.nlm.nih.gov/gene?term=IFRD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NDUFA3](https://www.ncbi.nlm.nih.gov/gene?term=NDUFA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.3</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 203 (min len 4) | There is no clear functional association between IFRD2 and NDUFA3 based on the provided context and their known biological functions. IFRD2 is a cell surface receptor that plays a role in cell proliferation, differentiation, migration, and apoptosis, while NDUFA3 is an accessory subunit of the respiratory chain NADH dehydrogenase that is involved in electron transfer.|
| [PPP2R5A](https://www.ncbi.nlm.nih.gov/gene?term=PPP2R5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.8</br>**STRING score:** 231</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | PPP2R5A and SHC1 both have protein-protein interactions and are involved in signaling pathways, with PPP2R5A being a regulatory subunit of the PP2A enzyme and SHC1 acting as a signaling adapter that couples growth factor receptors to Ras and oxidative stress pathways. There may be a functional association between these genes in regulating cellular growth and stress response pathways.|
| [PTPN6](https://www.ncbi.nlm.nih.gov/gene?term=PTPN6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFP36](https://www.ncbi.nlm.nih.gov/gene?term=ZFP36[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.6</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between PTPN6 and ZFP36 include regulation of cell signaling pathways and immune responses. PTPN6 modulates signaling by tyrosine phosphorylated cell surface receptors and plays a key role in hematopoiesis, while ZFP36 negatively regulates HTLV-1 TAX-dependent transactivation of viral long terminal repeat promoter. The shortest path between these two genes in the BioGRID protein interaction network suggests they may interact through proteins such as SAMD4B and TRIP6.|
| [NNT](https://www.ncbi.nlm.nih.gov/gene?term=NNT[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUCLG1](https://www.ncbi.nlm.nih.gov/gene?term=SUCLG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between NNT and SUCLG1 could be related to energy metabolism, as both genes are involved in the production of ATP through different pathways. NNT functions as a proton pump across the membrane, coupled with respiration and ATP hydrolysis, while SUCLG1 plays a role in substrate-level phosphorylation in the TCA cycle.|
| [P4HA2](https://www.ncbi.nlm.nih.gov/gene?term=P4HA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN6](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 273 (min len 4) | Possible functional associations: P4HA2 is involved in collagen formation and TSPAN6 plays a role in DNA methylation and transcriptional repression. There is no direct biological process linking these two genes.|
| [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPP2R5A](https://www.ncbi.nlm.nih.gov/gene?term=PPP2R5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.4</br>**STRING score:** 815</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between LPGAT1 and PPP2R5A may include regulation of de novo hepatic lipid synthesis and body fat content, as well as interaction with ADRB2 and SNX27. LPGAT1 is involved in fatty acyl chain remodeling of glycerophospholipids and can acylate lysophosphatidylglycerols, while PPP2R5A is a regulatory subunit of the PP2A enzyme involved in substrate selectivity and catalytic activity.|
| [MYLK](https://www.ncbi.nlm.nih.gov/gene?term=MYLK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.2</br>**STRING score:** 279</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MYLK and NFKBIA include regulation of inflammatory response and cell motility. MYLK is involved in smooth muscle contraction and phosphorylation of myosin light chains, while NFKBIA inhibits the activity of dimeric NF-kappa-B/REL complexes by masking their nuclear localization signals. The interaction between MYLK and NFKBIA may modulate cellular processes related to inflammation and cell motility.|
