| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BACE2](https://www.ncbi.nlm.nih.gov/gene?term=BACE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNP](https://www.ncbi.nlm.nih.gov/gene?term=PNP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between BACE2 and PNP in the given context. BACE2 cleaves a precursor protein involved in the development of Alzheimer's disease, while PNP catalyzes the breakdown of nucleoside molecules.|
| [GNAS](https://www.ncbi.nlm.nih.gov/gene?term=GNAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: There is a shortest path between GNAS and PSMG1 through EGFR, KRAS, PARK2, ZRANB1, and RAF1 in the BioGRID protein interaction network. GNAS plays a critical role in regulation of immune cells and transcription regulation, while PSMG1 promotes assembly of the 20S proteasome as part of a heterodimer with PSMG2. There is no obvious functional association between these two genes in specific biological terms.|
| [DNMT3A](https://www.ncbi.nlm.nih.gov/gene?term=DNMT3A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no direct evidence for functional associations between DNMT3A and PSME1 based on the protein interaction network and provided context. DNMT3A is involved in de novo DNA methylation, while PSME1 is involved in immunoproteasome assembly and antigen processing.|
| [BACE2](https://www.ncbi.nlm.nih.gov/gene?term=BACE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BACE2 is involved in proteolytic processing of APP and PMEL. BIRC5 plays a dual role in promoting cell proliferation and preventing apoptosis, and is a component of the CPC essential for chromosome alignment and segregation during mitosis and cytokinesis. There may not be any direct functional associations known between these two genes in specific biological terms.|
| [RAB11FIP2](https://www.ncbi.nlm.nih.gov/gene?term=RAB11FIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TWF2](https://www.ncbi.nlm.nih.gov/gene?term=TWF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: Both RAB11FIP2 and TWF2 are involved in membrane trafficking and endocytosis processes. RAB11FIP2 also plays a role in cell polarity and phagocytosis, while TWF2 regulates actin polymerization and motility, and may play a role in stereocilia length regulation. There may be interactions between RAB11FIP2 and TWF2 in regulating endocytic organelles or actin dynamics.|
| [AKAP8L](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB1](https://www.ncbi.nlm.nih.gov/gene?term=PSMB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional associations between AKAP8L and PSMB1 include regulation of protein homeostasis and degradation, and involvement in viral infection and reverse transcription. AKAP8L may interact with PSMB1 through its association with other regulatory particles in the proteasome complex.|
| [DUSP11](https://www.ncbi.nlm.nih.gov/gene?term=DUSP11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB10](https://www.ncbi.nlm.nih.gov/gene?term=PSMB10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 504 (min len 4) | Possible functional association: PSMB10 is involved in antigen processing and DUSP11 may participate in nuclear mRNA metabolism. They may play a role in regulating the expression of genes involved in immune response or viral defense.|
| [ATP5G3](https://www.ncbi.nlm.nih.gov/gene?term=ATP5G3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BAMBI](https://www.ncbi.nlm.nih.gov/gene?term=BAMBI[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 4) | There is no direct biological association between ATP5G3 and BAMBI as they belong to different pathways. ATP5G3 encodes ATP synthase subunit c, which plays a role in oxidative phosphorylation, while BAMBI negatively regulates TGF-beta signaling pathway.|
| [ALDOA](https://www.ncbi.nlm.nih.gov/gene?term=ALDOA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCEAL4](https://www.ncbi.nlm.nih.gov/gene?term=TCEAL4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between ALDOA and TCEAL4 may involve regulatory processes in glycolysis and transcriptional regulation, as ALDOA plays a key role in glycolysis and TCEAL4 may be involved in transcriptional regulation. The interaction between these two genes in the BioGRID protein interaction network may suggest a possible scaffolding function or involvement in microbial infection.|
| [EIF5](https://www.ncbi.nlm.nih.gov/gene?term=EIF5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 119.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 68 (min len 4) | There are no obvious functional associations between EIF5 and HLA-DMA based on current literature and the protein interaction network. EIF5 is involved in ribosomal initiation complex hydrolysis, while HLA-DMA plays a critical role in catalyzing the release of class II-associated invariant chain peptide from newly synthesized MHC class II molecules.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.5</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between ATP6V0B and RPA1 based on their known roles in vacuolar acidification and DNA damage response, respectively. However, there is evidence of indirect interaction between these genes through their protein interactions with MOV10, EGFR, and KRAS in the BioGRID network, suggesting potential cross-talk between vacuolar acidification and cellular signaling pathways.|
| [MRPL12](https://www.ncbi.nlm.nih.gov/gene?term=MRPL12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 105.1</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between MRPL12 and ZNF131 include roles in transcriptional regulation and central nervous system function. MRPL12 is involved in mitochondrial translation and ribosomal biogenesis, while ZNF131 is known to interact with transcriptional regulators and receptors involved in endocytosis. It is possible that ZNF131 plays a role in regulating the expression of MRPL12 and other mitochondrial genes.|
| [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.3</br>**STRING score:** 618</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between TMED10 and TRAPPC3 could be related to vesicular transport between the endoplasmic reticulum and Golgi. TMED10 is a cargo receptor involved in protein vesicular trafficking from the ER to Golgi, while TRAPPC3 is a component of the multisubunit transport protein particle complex involved in vesicular transport from ER to Golgi.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [OXA1L](https://www.ncbi.nlm.nih.gov/gene?term=OXA1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.8</br>**STRING score:** 281</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | There are currently no known functional associations between MTX2 and OXA1L in specific biological terms as they are involved in distinct processes. MTX2 is involved in protein transport into the mitochondrion while OXA1L is required for the insertion of integral membrane proteins into the mitochondrial inner membrane.|
| [HINT1](https://www.ncbi.nlm.nih.gov/gene?term=HINT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.2</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between HINT1 and TMED10 could be in the regulation of protein trafficking and quality control in the endoplasmic reticulum and Golgi, as TMED10 is involved in cargo transport through COPI and COPII-coated vesicles, while HINT1 modulates proteasomal degradation of target proteins and interacts with a SCF E3 ubiquitin-protein ligase complex. Additionally, HINT1 has been shown to modulate p53/TP53 levels and apoptosis, while TMED10 is involved in trafficking of amyloid beta A4 protein and soluble APP-beta release.|
| [ELOVL6](https://www.ncbi.nlm.nih.gov/gene?term=ELOVL6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PEX11A](https://www.ncbi.nlm.nih.gov/gene?term=PEX11A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 101.1</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 4) | Possible functional associations between ELOVL6 and PEX11A could be their roles in fatty acid metabolism and peroxisomal function, respectively. Specifically, ELOVL6 catalyzes the elongation of fatty acids, while PEX11A is involved in peroxisomal proliferation and membrane dynamics. There may be a relationship between fatty acid metabolism and peroxisomal function that requires further investigation.|
| [CXCR4](https://www.ncbi.nlm.nih.gov/gene?term=CXCR4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMAIP1](https://www.ncbi.nlm.nih.gov/gene?term=PMAIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.3</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | One possible functional association between CXCR4 and PMAIP1 is in the regulation of apoptosis or cell death. PMAIP1 promotes activation of caspases and apoptosis, while CXCR4 interacts with HIV and Staphylococcus aureus and may play a role in cell survival or death. The shortest paths between these genes in the BioGRID protein interaction network involve several intermediary proteins, suggesting a potential direct or indirect interaction in modulating these processes.|
