| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.5</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | AKAP8 and CREB1 both play important roles in transcriptional regulation and are connected through protein-protein interactions in the BioGRID network. They may be functionally associated in processes such as cell cycle progression and chromatin condensation during mitosis.|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 157.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between AKAP8 and PUM2 include roles in cell cycle regulation, DNA replication, and transcriptional regulation. AKAP8 is involved in anchoring PKA to chromatin, promoting chromatin condensation, and enhancing HMT activity, while PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor. The shortest path between AKAP8 and PUM2 in the BioGRID protein interaction network involves MYC, ELAVL1, ESR1, BMI1, and TNIP2.|
| [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFX](https://www.ncbi.nlm.nih.gov/gene?term=ZFX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MLEC and ZFX are unclear based on the provided context. MLEC is a carbohydrate-binding protein involved in early steps of protein N-glycosylation and interacts with the oligosaccharyltransferase complex, while ZFX is a probable transcriptional activator that interacts with MAP2K3/6 and influences NF-kappa-B activation.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TAOK2](https://www.ncbi.nlm.nih.gov/gene?term=TAOK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between PUM2 and TAOK2 based on the provided context. PUM2 is involved in post-transcriptional repression of mRNA targets and regulation of miRNAs, while TAOK2 is involved in MAPK signaling pathway, DNA damage response, and apoptotic morphological changes. However, there is a protein interaction between them through ESR1 and ZRANB1, which may suggest a possible indirect functional association.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no known direct functional association between IDE and SMNDC1. IDE is involved in membrane-associated isoform entry receptor for varicella-zoster virus, while SMNDC1 is involved in spliceosome assembly.|
| [MICALL1](https://www.ncbi.nlm.nih.gov/gene?term=MICALL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional association: MICALL1 and SNX6 are both involved in intracellular trafficking and endocytosis processes. MICALL1 may act downstream of Rab proteins to regulate membrane tubulation, while SNX6 is a component of the SNX-BAR retromer complex involved in retrograde transport of cargo proteins from endosomes to the trans-Golgi network, as well as endosome-to-plasma membrane transport for cargo protein recycling. It is possible that they have complementary roles in these processes.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.5</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on the protein interaction network and gene functions, FRS2 and PUM2 may be functionally associated in the regulation of post-transcriptional processes such as mRNA degradation and translation inhibition. FRS2 is an adapter protein involved in the activation of MAP kinases and phosphorylation of PIK3R1, while PUM2 is an RNA-binding protein that represses target mRNA via deadenylation and miRNA regulation.|
| [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XPNPEP1](https://www.ncbi.nlm.nih.gov/gene?term=XPNPEP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between SMNDC1 and XPNPEP1 based on the limited information provided. SMNDC1 is involved in spliceosome assembly, while XPNPEP1 is a metalloaminopeptidase that degrades peptides.|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are no known direct functional associations between MNAT1 and PMPCB in the scientific literature. MNAT1 is involved in cell cycle control and RNA transcription, while PMPCB functions as the catalytic subunit of the mitochondrial processing protease.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B1](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CASP3 and PAFAH1B1 involve their involvement in apoptotic processes. CASP3 is a thiol protease involved in the execution phase of apoptosis, while PAFAH1B1 is involved in regulating the activity of dynein, which is required for various types of cell motility including cell migration during wound healing.|
| [CCT7](https://www.ncbi.nlm.nih.gov/gene?term=CCT7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 124.9</br>**STRING score:** 162</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | As both CCT7 and UBQLN2 are involved in protein folding and degradation mechanisms, they may have a functional association in regulating protein homeostasis. The shortest path between these genes in the BioGRID network suggests a possible physical interaction between them in this process.|
| [APBB2](https://www.ncbi.nlm.nih.gov/gene?term=APBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DCUN1D4](https://www.ncbi.nlm.nih.gov/gene?term=DCUN1D4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.8</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 169 (min len 4) | Possible functional associations between APBB2 and DCUN1D4 include a role in neuromuscular junction formation and maintenance of lens transparency, as well as contributing to the neddylation of all cullins. These genes may be involved in regulating protein turnover or degradation pathways in neurons and muscle cells.|
| [CTSD](https://www.ncbi.nlm.nih.gov/gene?term=CTSD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SIRT3](https://www.ncbi.nlm.nih.gov/gene?term=SIRT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.7</br>**STRING score:** 191</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between CTSD and SIRT3 is their involvement in regulating cellular energy metabolism. CTSD is involved in intracellular protein breakdown, and SIRT3 is a NAD-dependent protein deacetylase that activates or deactivates mitochondrial target proteins by deacetylating key lysine residues. Both genes may play a role in regulating the cellular energy metabolism in response to metabolic stress.|
| [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.0</br>**STRING score:** 782</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association between KDELR2 and TMED10 is their involvement in protein trafficking and quality control in the endoplasmic reticulum and Golgi. KDELR2 is a membrane receptor that participates in vesicle-mediated recycling, while TMED10 is a cargo receptor involved in incorporation of secretory cargo molecules into transport vesicles and vesicle coat formation. They interact with each other and with other proteins to mediate protein transport through COPI and COPII-coated vesicles.|
| [KIAA1033](https://www.ncbi.nlm.nih.gov/gene?term=KIAA1033[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.3</br>**STRING score:** 213</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | KIAA1033 and PUM2 are functionally associated through protein-protein interactions in the BioGRID network, specifically via the intermediate proteins NTRK1, MYC, PLEKHA4, PARK2, and ZRANB1. KIAA1033 is involved in actin polymerization and endosome sorting, while PUM2 acts as a post-transcriptional repressor by binding to mRNA targets and promoting accessibility of miRNAs. There may be a possible regulatory role of PUM2 on KIAA1033-mediated processes.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.9</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations: PUM2 is involved in post-transcriptional regulation of mRNA targets and may support proliferation and self-renewal of stem cells. IDE is a membrane-associated isoform that acts as an entry receptor for varicella-zoster virus. There is no clear biological process linking these two genes.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP3K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP3K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.8</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CASP3 and MAP3K4 could include involvement in apoptotic pathways, as CASP3 is a major effector caspase involved in the execution of apoptosis, and MAP3K4 is involved in a protein kinase signal transduction cascade that activates the CSBP2, P38 and JNK MAPK pathways. The shortest interaction path between the two genes involves intermediate proteins such as MAPK8, BAG3, CDC42, TRIM28, and ZRANB1.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.5</br>**STRING score:** 313</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | There are no obvious direct functional associations between CREB1 and IDE. CREB1 is involved in transcription activation and adipose differentiation, while IDE acts as an entry receptor for varicella-zoster virus. However, CREB1 has been linked to the regulation of glucose metabolism, which may indirectly impact IDE's role in insulin-degrading enzyme activity.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IGF2R](https://www.ncbi.nlm.nih.gov/gene?term=IGF2R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 111.3</br>**STRING score:** 193</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association: IGF2R mediates the transport of phosphorylated lysosomal enzymes, and is involved in retrograde trafficking to the Golgi. CREB1 is a transcription factor involved in adipose differentiation and circadian rhythmicity. These genes may be involved in cellular processes related to lysosomal function and adipose differentiation.|
| [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 111.1</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | There is no clear functional association between KIF2C and NRAS based on their known biological roles and the protein interaction network. KIF2C plays a role in microtubule regulation during mitosis, while NRAS is involved in Ras signaling pathways. However, both genes have been found to interact with proteins involved in microtubule organization and regulation.|
