| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.5</br>**STRING score:** 180</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | AKAP8 and CREB1 have a shortest path of interactions in the BioGRID protein interaction network, with potentially functional associations in transcriptional regulation and cell cycle progression. AKAP8 anchors PKA-signaling complex to mitotic chromosomes and recruits histone deacetylase to the vicinity of chromatin, while CREB1 is a phosphorylation-dependent transcription factor that stimulates transcription upon binding to the DNA cAMP response element.|
| [AKAP8](https://www.ncbi.nlm.nih.gov/gene?term=AKAP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 157.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between AKAP8 and PUM2 could be in the regulation of cell cycle progression and DNA replication. AKAP8 is involved in anchoring and recruiting proteins involved in chromatin condensation and histone deacetylation during mitosis. PUM2 is a post-transcriptional repressor that binds to mRNA targets and regulates their degradation, translation, and miRNA regulation. The shortest path between these two genes in the BioGRID protein interaction network involves several proteins related to cell proliferation and transcriptional regulation, suggesting that they may interact in similar pathways.|
| [MLEC](https://www.ncbi.nlm.nih.gov/gene?term=MLEC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZFX](https://www.ncbi.nlm.nih.gov/gene?term=ZFX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 155.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MLEC and ZFX may involve regulation of protein N-glycosylation and transcriptional activation. Both proteins interact with other proteins such as the OST complex and MAP2K3/6, respectively, suggesting possible roles in cellular signaling. However, the specific biological processes where these two genes directly interact require further investigation.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TAOK2](https://www.ncbi.nlm.nih.gov/gene?term=TAOK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 149.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between PUM2 and TAOK2 are unclear as they are involved in different processes. PUM2 is a post-transcriptional repressor of mRNA targets, involved in cytoplasmic sensing of viral infection and regulating miRNA expression. TAOK2 is a serine/threonine-protein kinase involved in membrane blebbing, apoptotic bodies formation, DNA damage response and MAPK14/p38 MAPK stress-activated MAPK cascade.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 145.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of functional associations between IDE and SMNDC1. IDE is involved in membrane-associated viral infection while SMNDC1 is involved in spliceosome assembly and interacts with U2 snRNP and SNRPD3. They may be involved in distinct biological processes.|
| [MICALL1](https://www.ncbi.nlm.nih.gov/gene?term=MICALL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional association between MICALL1 and SNX6 is in endocytic recycling and trafficking pathways. MICALL1 is involved in receptor-mediated endocytosis and slow endocytic recycling of endocytosed proteins back to the plasma membrane, while SNX6 functions as a component of the retromer complex involved in endosome-to-plasma membrane transport for cargo protein recycling. Both genes interact with Rab proteins and may play a role in regulating membrane tubulation and vesicular trafficking.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.5</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FRS2 and PUM2 include FRS2's role in the activation of MAP kinases and phosphorylation of PIK3R1, and PUM2's role in post-transcriptional repression of transcripts. FRS2 has been shown to interact with NTRK1, which is also a binding partner of PUM2, suggesting a potential overlap in signaling pathways.|
| [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XPNPEP1](https://www.ncbi.nlm.nih.gov/gene?term=XPNPEP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 138.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are currently no known functional associations between SMNDC1 and XPNPEP1. SMNDC1 is involved in spliceosome assembly and interacts with snRNPs, while XPNPEP1 functions as a metalloaminopeptidase that contributes to peptide degradation.|
| [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 133.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear functional association between MNAT1 and PMPCB. MNAT1 is involved in cell cycle control and RNA transcription by RNA polymerase II, while PMPCB is the catalytic subunit of the mitochondrial processing protease.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B1](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 131.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CASP3 and PAFAH1B1 could be in the regulation of apoptosis and cytoskeleton dynamics. CASP3 is a thiol protease involved in the execution phase of apoptosis, while PAFAH1B1 regulates actin polymerization and dynein-mediated transport. Interaction between these genes could modulate these processes in the context of cancer.|
| [CCT7](https://www.ncbi.nlm.nih.gov/gene?term=CCT7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 124.9</br>**STRING score:** 162</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: UBQLN2 plays a role in protein degradation pathways including the proteasomal targeting of misfolded or accumulated proteins, whereas CCT7 is a component of the TRiC chaperone complex that assists in protein folding. They may functionally interact to ensure proper protein quality control and folding.  |
| [APBB2](https://www.ncbi.nlm.nih.gov/gene?term=APBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DCUN1D4](https://www.ncbi.nlm.nih.gov/gene?term=DCUN1D4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.8</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 169 (min len 4) | Possible functional association: APBB2 activates transcription of APP and interacts with APP and APLP2, while DCUN1D4 contributes to the neddylation of cullins, including CUL1, CUL2, CUL3, CUL4A, CUL4B and CUL5. These genes are not known to be functionally associated in specific biological processes.|
| [CTSD](https://www.ncbi.nlm.nih.gov/gene?term=CTSD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SIRT3](https://www.ncbi.nlm.nih.gov/gene?term=SIRT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.7</br>**STRING score:** 191</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: SIRT3 and CTSD are protein interaction partners in TCGA-OV.  SIRT3 is a NAD-dependent protein deacetylase involved in regulating cellular energy metabolism and mtDNA transcription, while CTSD is an acid protease active in intracellular protein breakdown and involved in breast cancer pathogenesis and APP processing. It is possible that SIRT3 may regulate the activity or expression of CTSD in some way.|
| [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 120.0</br>**STRING score:** 782</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between KDELR2 and TMED10 include their involvement in protein vesicular trafficking and quality control in the endoplasmic reticulum and Golgi, with KDELR2 serving as a membrane receptor that binds to endoplasmic reticulum resident proteins and participating in their vesicle-mediated recycling back from the Golgi, while TMED10 acts as a cargo receptor involved in vesicle coat formation and mediating cargo transport through COPI and COPII-coated vesicles. Their interaction via TRIM25, TSSC1, ESR1, M, and nsp9 likely contributes to the regulation of these processes.|
| [KIAA1033](https://www.ncbi.nlm.nih.gov/gene?term=KIAA1033[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 117.3</br>**STRING score:** 213</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: KIAA1033 plays a key role in endosome sorting and tubule fission, while PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor. There are no known direct functional associations between these two genes in specific biological terms based on their known functions.|
| [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 116.9</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between IDE and PUM2 may involve regulation of gene expression and potentially have roles in viral infection. IDE may interact with varicella-zoster virus (VZV) while PUM2 is a sequence-specific RNA-binding protein that represses transcripts and plays a role in cytoplasmic sensing of viral infection. The shortest path in the BioGRID protein interaction network suggests potential indirect interactions through intermediate proteins.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP3K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP3K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.8</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CASP3 and MAP3K4 are their involvement in the apoptotic pathway, with CASP3 acting as a major effector caspase and MAP3K4 activating the CSBP2, P38 and JNK MAPK pathways. The shortest path between CASP3 and MAP3K4 in the BioGRID protein interaction network suggests that they may also interact through multiple intermediate proteins, such as MAPK8 and BAG3.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IDE](https://www.ncbi.nlm.nih.gov/gene?term=IDE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.5</br>**STRING score:** 313</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between CREB1 and IDE in the context of TCGA-OV cancer may involve regulation of circadian rhythms, adipocyte differentiation, and viral infections, based on their respective biological roles. A molecular link between these two genes may involve interplay between IDE as an entry receptor and CREB1 as a transcription factor, potentially affecting processes related to viral replication and cellular differentiation.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IGF2R](https://www.ncbi.nlm.nih.gov/gene?term=IGF2R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 111.3</br>**STRING score:** 193</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between CREB1 and IGF2R include the regulation of cellular differentiation, as CREB1 is involved in adipose cell differentiation and IGF2R mediates the transport of lysosomal enzymes. The bioGRID protein interaction network indicates multiple protein-protein interactions between these genes, suggesting potential involvement in cellular processes such as microbial infection, T-cell coactivation, and retrograde trafficking.|
| [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 111.1</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | KIF2C and NRAS may be functionally associated in regulating chromosome segregation during mitosis. KIF2C is involved in microtubule depolymerization and interactions with kinetochores, while NRAS is a GTPase protein that interacts with other proteins in the cell signaling pathway.|
