| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ECD](https://www.ncbi.nlm.nih.gov/gene?term=ECD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SKP1](https://www.ncbi.nlm.nih.gov/gene?term=SKP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 182.7</br>**STRING score:** 214</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between ECD and SKP1 include their interaction in the SCF (SKP1-CUL1-F-box protein) ubiquitin ligase complex, which mediates ubiquitination of proteins involved in cell cycle progression, signal transduction, and transcription. Additionally, ECD may play a role in regulation of pre-mRNA splicing and transcriptional activation of glycolytic genes, while SKP1 may be involved in Wnt signaling and ubiquitination of various proteins.|
| [BID](https://www.ncbi.nlm.nih.gov/gene?term=BID[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAK6](https://www.ncbi.nlm.nih.gov/gene?term=PAK6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 179.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between BID and PAK6 based on the provided context. BID induces apoptosis and interacts with ITCH and MTCH2, while PAK6 is a kinase that regulates gene transcription and interacts with IQGAP1 and GTP-bound CDC42/p21 and RAC1.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM3A](https://www.ncbi.nlm.nih.gov/gene?term=KDM3A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 173.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: KDM3A may be involved in hormone-dependent transcriptional activation by participating in recruitment to androgen-receptor target genes where ATF6 could be a potential target. Additionally, ATF6 has been implicated in ER stress response and KDM3A has been shown to regulate metabolic genes, potentially linking the two in these processes.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.8</br>**STRING score:** 973</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between EGFR and PTK2 include their involvement in cellular signaling pathways such as migration, proliferation, and apoptosis. They are also both involved in the regulation of the cytoskeleton, adhesion, and membrane trafficking. In specific biological terms, EGFR is a receptor tyrosine kinase involved in growth factor signaling, while PTK2 is a non-receptor tyrosine kinase involved in integrin-mediated signaling.|
| [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSC22D3](https://www.ncbi.nlm.nih.gov/gene?term=TSC22D3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: TSC22D3 interacts with HDAC1 which inhibits MYOD1 transcriptional activity, while SRP14 is a component of the SRP complex which mediates the cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum. There is no known direct biological association between these two genes.|
| [KAZALD1](https://www.ncbi.nlm.nih.gov/gene?term=KAZALD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 165.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 91 (min len 4) | Possible functional associations between KAZALD1 and SNX6 in TCGA-OV may involve involvement in intracellular trafficking and endosome-to-TGN transport. KAZALD1 may promote matrix assembly and interact with CEP290, while SNX6 may negatively regulate retrograde transport of BACE1 and promote lysosomal degradation of CDKN1B.  Their interaction may play a role in the movement of retromer-containing vesicles towards the TGN via SNX6's interaction with DCTN1.|
| [ARID5B](https://www.ncbi.nlm.nih.gov/gene?term=ARID5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPP2R5E](https://www.ncbi.nlm.nih.gov/gene?term=PPP2R5E[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 164.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | ARID5B and PPP2R5E are functionally associated in the regulation of transcription and gene expression. ARID5B is a transcription coactivator involved in adipogenesis and liver development, while PPP2R5E is a regulatory subunit of the PP2A enzyme that modulates substrate selectivity and catalytic activity. The shortest path between ARID5B and PPP2R5E in the BioGRID protein interaction network suggests that they may physically interact through other proteins such as ELAVL1, RNF4, and FASN.|
| [DFFA](https://www.ncbi.nlm.nih.gov/gene?term=DFFA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DMTF1](https://www.ncbi.nlm.nih.gov/gene?term=DMTF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 163.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 61 (min len 4) | Possible functional association between DFFA and DMTF1 is their involvement in apoptosis and cell cycle regulation through interactions with TP53 and caspases. DFFA is a caspase inhibitor, while DMTF1 is a transcriptional activator that promotes growth arrest via TP53 activation.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TJP1](https://www.ncbi.nlm.nih.gov/gene?term=TJP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 161.9</br>**STRING score:** 479</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations: HIF1A is involved in promoting glycolysis and inducing pro-inflammatory state upon SARS-CoV-2 infection, while TJP1 plays a role in tight junction formation and epithelial polarization. There may be a potential link between these genes in regulating inflammation and epithelial barrier function in the context of cancer.|
| [ITGB5](https://www.ncbi.nlm.nih.gov/gene?term=ITGB5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRAF2](https://www.ncbi.nlm.nih.gov/gene?term=PRAF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 160.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ITGB5 and PRAF2 could involve vesicular traffic and apoptosis, as PRAF2 is involved in ER/Golgi transport and plays a proapoptotic role, while ITGB5 interacts with proteins involved in vesicular transport and can play a role in adenovirus infection. However, further experimental investigation is needed to establish any specific biological processes linking these two genes.|
| [APP](https://www.ncbi.nlm.nih.gov/gene?term=APP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [INSIG1](https://www.ncbi.nlm.nih.gov/gene?term=INSIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no apparent functional associations between APP and INSIG1 based on their known biological functions and protein interactions. APP is involved in neuronal processes, microtubule binding, and amyloid beta production, while INSIG1 regulates cholesterol synthesis and degradation through interactions with SCAP and HMGCR.|
| [FOXO4](https://www.ncbi.nlm.nih.gov/gene?term=FOXO4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 159.4</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | FOXO4 is involved in the regulation of insulin signaling, negative regulation of cell cycle, and suppression of HIF1A. HIF1A is a transcription factor that is activated in response to hypoxia and is involved in promoting glycolysis and pro-inflammatory state. There may be a possible functional association between FOXO4 and HIF1A in the regulation of cellular metabolism and inflammation.|
| [MYLK](https://www.ncbi.nlm.nih.gov/gene?term=MYLK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [XPNPEP1](https://www.ncbi.nlm.nih.gov/gene?term=XPNPEP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is currently no known functional association between MYLK and XPNPEP1. MYLK is involved in smooth muscle contraction, inflammation, and cell motility, while XPNPEP1 is a metalloaminopeptidase that contributes to the degradation of peptides such as bradykinin.|
| [BNIP3](https://www.ncbi.nlm.nih.gov/gene?term=BNIP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 154.1</br>**STRING score:** 702</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations: BNIP3 and DNM1L both play a role in mitochondrial quality control. BNIP3 interacts with DNM1L in the BioGRID protein interaction network, potentially indicating a role in mitochondrial fission. Both genes are involved in apoptotic pathways and have been linked to cell death processes in various contexts.|
| [KIF2C](https://www.ncbi.nlm.nih.gov/gene?term=KIF2C[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PYCR1](https://www.ncbi.nlm.nih.gov/gene?term=PYCR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.7</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between KIF2C and PYCR1 could be related to mitotic activities, as KIF2C is involved in mitosis and chromosome segregation, while PYCR1 is involved in the response to oxidative stress. There is no clear indication of a direct functional association between these two genes based on the provided context.|
| [ICAM1](https://www.ncbi.nlm.nih.gov/gene?term=ICAM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 151.1</br>**STRING score:** 356</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between ICAM1 and PTK2 include their involvement in cell adhesion and migration processes, as well as their role in signal transduction pathways. ICAM1 plays a role in immune cell function and viral infection, while PTK2 is involved in integrin signaling and is a regulator of actin dynamics. The protein interaction network suggests a potential direct interaction between the two proteins mediated by several other proteins.|
| [IGF1R](https://www.ncbi.nlm.nih.gov/gene?term=IGF1R[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 147.1</br>**STRING score:** 453</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: IGF1R is involved in cell growth and division, while SNX6 plays a role in intracellular trafficking. It is possible that IGF1R may be trafficked by SNX6 and that this interaction may affect cell growth and division.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM3A](https://www.ncbi.nlm.nih.gov/gene?term=KDM3A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 143.7</br>**STRING score:** 891</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both HIF1A and KDM3A are involved in transcriptional regulation and can influence each other's activity through protein-protein interactions. Specifically, HIF1A plays a role in promoting glycolysis and cytokine production in immune cells during microbial infection and can interact with KDM3A to regulate gene expression.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.8</br>**STRING score:** 374</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between ATF6 and HIF1A could involve the regulation of cellular stress response and metabolism. ATF6 is involved in initiating the unfolded protein response (UPR) during endoplasmic reticulum stress, while HIF1A plays a key role in promoting glycolysis in hypoxic conditions and regulating cytokine expression during infection. It is possible that these two genes may interact in regulating metabolic and stress response pathways in cancer.|
| [GNA15](https://www.ncbi.nlm.nih.gov/gene?term=GNA15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.2</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 155 (min len 4) | Possible functional association: GNA15 and NFKBIA have a protein interaction network, with a shortest path involving ADRBK1 and SRC. GNA15 is involved in transmembrane signaling as a modulator or transducer, while NFKBIA inhibits the activity of dimeric NF-kappa-B/REL complexes and promotes ubiquitination and degradation. A possible biological process linking these genes could be the regulation of immune and pro-inflammatory responses.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 141.1</br>**STRING score:** 376</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | ATP6V0B and COPB2 are functionally associated through their participation in vesicular trafficking and protein transport processes in the Golgi apparatus. ATP6V0B encodes a proton transport subunit of the V0 complex of the vacuolar ATPase, which is responsible for acidifying and maintaining the pH of intracellular compartments. COPB2 encodes a coatomer complex protein that binds to Golgi membranes in a GTP-dependent manner and is essential for Golgi budding and vesicular trafficking. Their interaction may facilitate proper Golgi function and protein transport.|
| [PYCR1](https://www.ncbi.nlm.nih.gov/gene?term=PYCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.7</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 45 (min len 3) | PYCR1 is involved in proline biosynthesis and responding to oxidative stress, while SQSTM1 is an autophagy receptor required for selective macroautophagy and involved in the degradation of polyubiquitinated cargo. Based on their roles, these genes may function in the regulation of protein degradation pathways in response to cellular stress. The shortest path in the protein interaction network linking the two genes involves other proteins involved in protein ubiquitination, suggesting a potential functional association between the two genes in this process.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 140.3</br>**STRING score:** 255</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between ARHGAP1 and PTK2 include their involvement in cell signaling pathways and cytoskeleton organization through interactions with SRC, RAC1, and PIK3R1. ARHGAP1 also regulates GTPase activity while PTK2 is a non-receptor tyrosine kinase that influences cell migration and adhesion.|
| [CSK](https://www.ncbi.nlm.nih.gov/gene?term=CSK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MLLT11](https://www.ncbi.nlm.nih.gov/gene?term=MLLT11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no evident functional association between CSK and MLLT11 based on their known biological roles and interactions. CSK plays a role in inhibiting Src-family kinases, while MLLT11 is involved in regulating T cell differentiation and interacting with transcription factor TCF7.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EXT1](https://www.ncbi.nlm.nih.gov/gene?term=EXT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 136.0</br>**STRING score:** 221</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and EXT1 are not known to have direct functional associations. EGFR is a receptor involved in cell signaling, including cell proliferation and migration, while EXT1 is a glycosyltransferase involved in the biosynthesis of heparan sulfate, a component of the extracellular matrix.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP7](https://www.ncbi.nlm.nih.gov/gene?term=USP7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.7</br>**STRING score:** 507</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | PLK1 and USP7 do not have a well-defined functional association in the literature. PLK1 is a serine/threonine-protein kinase that plays a key role in regulating centrosome functions, spindle assembly, and mitotic exit. USP7 is a deubiquitinase that stabilizes and transactivates viral proteins during infection.|
