| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RTN2](https://www.ncbi.nlm.nih.gov/gene?term=RTN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAV3](https://www.ncbi.nlm.nih.gov/gene?term=VAV3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 146 (min len 4) | Possible functional association between RTN2 and VAV3 could be in the regulation of cell migration and adhesion through the RhoA and Rac1 signaling pathways. RTN2 may interact with VAV3 through FANCG, RINT1, and AR proteins to modulate cellular trafficking and migration processes.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP8](https://www.ncbi.nlm.nih.gov/gene?term=RRP8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between BHLHE40 and RRP8 include involvement in regulation of the circadian rhythm and silencing of rDNA in response to intracellular energy status. BHLHE40 is a transcriptional repressor involved in regulation of circadian rhythm by competing for binding to E-box elements, while RRP8 is a component of the eNoSC complex that mediates silencing of rDNA. The shortest path between the two genes in the BioGRID protein interaction network suggests that they interact indirectly through other proteins.|
| [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 107 (min len 4) | Possible functional association: PTPRC (receptor protein tyrosine phosphatase) mediates binding of human cytomegalovirus protein UL11 to T-cells while STX1A plays an essential role in calcium-dependent exocytosis and endocytosis, particularly in neurotransmitter release. There is no currently known biological connection between PTPRC and STX1A.|
| [SATB1](https://www.ncbi.nlm.nih.gov/gene?term=SATB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUCLG1](https://www.ncbi.nlm.nih.gov/gene?term=SUCLG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between these two genes, as SATB1 is a transcriptional repressor involved in chromatin organization and regulation of gene expression, while SUCLG1 is involved in the citric acid cycle and substrate-level phosphorylation. The shortest path in their protein interaction network includes CTNNB1, TIPARP, and TRIM67, but the biological significance of these connections is unknown.|
| [AMDHD2](https://www.ncbi.nlm.nih.gov/gene?term=AMDHD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CBLB](https://www.ncbi.nlm.nih.gov/gene?term=CBLB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between AMDHD2 and CBLB based on their roles in the Neu5Gc degradation pathway and T-cell/B-cell receptor signal transduction pathways, respectively. However, they do interact indirectly through the PARK2 protein in a protein interaction network.|
| [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.1</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between GAPDH and KAT6A include involvement in transcriptional regulation, specifically acetylation of histones and p53/TP53, and potential roles in inflammation and immune responses. There is a short path between GAPDH and KAT6A in the BioGRID protein interaction network, indicating potential physical interaction between these proteins.|
| [MBNL2](https://www.ncbi.nlm.nih.gov/gene?term=MBNL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 88.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 154 (min len 4) | Possible functional associations between MBNL2 and STX1A involve their roles in regulating alternative splicing and exocytosis, respectively. While a direct physical interaction has not been reported, their participation in common pathways suggestive of coordinated biological roles cannot be excluded.|
| [CSNK1E](https://www.ncbi.nlm.nih.gov/gene?term=CSNK1E[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SSBP2](https://www.ncbi.nlm.nih.gov/gene?term=SSBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 88.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | The possible functional association between CSNK1E and SSBP2 is their involvement in the ERK signaling pathway. CSNK1E participates in the pathway by phosphorylating DVL1 and DVL2, while SSBP2 may act downstream of ERK kinases to regulate ERK dephosphorylation. However, there is currently no direct evidence for a functional interaction between these genes.|
| [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPAG4](https://www.ncbi.nlm.nih.gov/gene?term=SPAG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 88.2</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: EBNA1BP2 is involved in RNA processing and interacts with WDR46, while SPAG4 is involved in spermatogenesis and interacts with SUN3 and SYNE1 to form a LINC complex. It is unclear if there is a direct functional association between these two genes in a specific biological process.|
| [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | IKBKE and KAT6A have a potential functional association as they are connected by a shortest path in the BioGRID protein interaction network. IKBKE is involved in regulating inflammatory responses and antiviral genes, while KAT6A is a histone acetyltransferase that may act as a transcriptional coactivator and positively regulates p53 acetylation activity.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.6</br>**STRING score:** 191</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | AKT1 and IKBKE are functionally associated through protein-protein interactions in the BioGRID network and are involved in important signaling pathways related to cell survival, proliferation, and inflammation. AKT1 is a kinase that regulates multiple downstream targets involved in metabolism, proliferation, and survival. IKBKE is a kinase that plays an essential role in regulating inflammatory responses to viral infection and can activate the type I interferon, NF-kappa-B, and STAT signaling pathways. These two genes may be functionally associated in regulating cell growth, survival, and inflammation.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP62](https://www.ncbi.nlm.nih.gov/gene?term=NUP62[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.7</br>**STRING score:** 709</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 3) | FKBP4 and NUP62 may be functionally associated in mitotic cell cycle progression, as NUP62 plays a role in centrosome segregation and FKBP4 regulates microtubule dynamics. Their interaction in promoting protein recruitment to the centrosome after nuclear breakdown may also suggest a possible collaboration in nuclear transport and localization of cellular components.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.2</br>**STRING score:** 553</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | BIRC2 and IKBKE have a direct protein interaction, possibly indicating a functional association. BIRC2 is involved in regulating apoptosis, immunity, inflammation, and cell proliferation. IKBKE regulates inflammatory responses to viral infection, as well as TNFA and cytokine signaling.|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BCL2](https://www.ncbi.nlm.nih.gov/gene?term=BCL2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.1</br>**STRING score:** 995</br>**BioGRID count:** 14</br>**BioGRID shortest paths:** 1 (min len 2) | BAD and BCL2 interact in the protein interaction network and have opposite functions in regulating cell death. BAD promotes cell death by competing for binding to Bcl-X(L), Bcl-2, and Bcl-W, while BCL2 suppresses apoptosis by controlling the mitochondrial membrane permeability. Therefore, these two genes may have a functional association in regulating cell death processes.|
| [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SGCB](https://www.ncbi.nlm.nih.gov/gene?term=SGCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.7</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between GAPDH and SGCB based on the available context. GAPDH is involved in glycolysis, microtubule and membrane associations, translation inhibition, nuclear events, and innate immunity, while SGCB is part of the sarcoglycan complex involved in linking the cytoskeleton to the extracellular matrix in muscle cells.|
| [GABPB1](https://www.ncbi.nlm.nih.gov/gene?term=GABPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.6</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between GABPB1 and GAPDH in TCGA-LUSC could involve regulating transcriptional activity and glycolysis. GABPB1 is involved in transcriptional regulation and interacts with HCFC1 to repress transcriptional activity, while GAPDH plays a key role in glycolysis. The shortest path between these two genes in the BioGRID protein interaction network also suggests potential direct interactions.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.6</br>**STRING score:** 230</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association between CEBPD and EBNA1BP2 is their interaction through the RB1 pathway in the regulation of cell cycle progression and tumor suppression. CEBPD is a transcription factor that regulates immune and inflammatory responses, while EBNA1BP2 is involved in rRNA processing and essential for the stable segregation of EBV episomes during cell division.|
| [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B1](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.6</br>**STRING score:** 218</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | Possible functional associations between GAPDH and PAFAH1B1 include their involvement in actin cytoskeleton organization, maintenance of Golgi integrity, and dynein-mediated microtubule sliding. They also play roles in nuclear events, transcription, RNA transport, DNA replication, apoptosis, and innate immunity. However, there is no direct evidence of functional association between these two genes in the literature.|
| [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.8</br>**STRING score:** 186</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | Possible functional associations between GAPDH and NMT1 include their involvement in glycolysis, nuclear functions, and innate immunity. While neither gene interacts with the other directly in the BioGRID protein interaction network, their shortest paths involve several common proteins such as TP53 and EGFR.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNA15](https://www.ncbi.nlm.nih.gov/gene?term=GNA15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.4</br>**STRING score:** 366</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 693 (min len 4) | CDC42 and GNA15 may have a functional association in regulating cell migration and cytoskeleton organization. CDC42 is a plasma membrane-associated small GTPase involved in the regulation of cellular responses, while GNA15 is a guanine nucleotide-binding protein involved in modulating transmembrane signaling systems. The shortest path between these two genes in the BioGRID protein interaction network includes the protein APPBP2, which has been shown to regulate cytoskeleton organization and cell migration.|
| [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.1</br>**STRING score:** 303</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between STX1A and TM9SF2 could include their involvement in intracellular transport and/or compartmentalization, as well as potential roles in endocytosis and exocytosis processes. STX1A participates in hormone and neurotransmitter calcium-dependent exocytosis and endocytosis, while TM9SF2 may function as a channel or small molecule transporter in intracellular compartments. However, without further experimental evidence, the specific functional relationship between these genes remains unclear.|
| [DECR1](https://www.ncbi.nlm.nih.gov/gene?term=DECR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 73.9</br>**STRING score:** 251</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional association: Both DECR1 and GAPDH play roles in metabolism. DECR1 is involved in beta-oxidation of fatty acids, while GAPDH is a key enzyme in glycolysis. There could be potential crosstalk between these metabolic pathways, but further investigation would be needed to elucidate any specific functional associations.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPAG4](https://www.ncbi.nlm.nih.gov/gene?term=SPAG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 73.8</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between MYC and SPAG4: MYC regulates genes involved in spermatogenesis, including SPAG4. The shortest path in the BioGRID protein interaction network between MYC and SPAG4 involves several intermediate proteins that are also involved in regulating gene expression, such as PTPN9, HSPA5, and DDX58.|
