| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CFLAR and MBNL1 are unclear, as they are involved in different biological processes. CFLAR is an apoptosis regulator while MBNL1 mediates pre-mRNA alternative splicing. However, through the BioGRID protein interaction network, CFLAR and MBNL1 are indirectly linked through the proteins EGFR and KRAS.|
| [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There are no known direct functional associations between ETFB and TMEM97. ETFB is involved in electron transfer and amino acid metabolism, while TMEM97 plays a role in cellular cholesterol homeostasis and drug binding in cancer cells. The shortest path between the two in the BioGRID protein interaction network involves several intermediate proteins, suggesting that any potential functional associations would be indirect.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | NVL and VAT1 are known to interact with each other in the BioGRID protein interaction network, with several intermediate proteins involved in the shortest path. While NVL plays a crucial role in ribosomal subunit biogenesis and in the assembly of the telomerase holoenzyme, VAT1 is an ATPase involved in keratinocyte activation and mitochondrial fusion regulation. Without further experimental evidence, it is difficult to infer any specific biological functional associations between these two genes.|
| [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between EBNA1BP2 and POLE2 could involve their roles in DNA replication and repair. EBNA1BP2 is required for rRNA processing and interacts with EBV EBNA1, while POLE2 is an accessory component of the DNA polymerase epsilon complex involved in chromosomal DNA replication and repair. It's possible that EBNA1BP2 and POLE2 may work together to maintain genome stability and/or facilitate viral replication.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations:          CFLAR and IKBKE are part of a protein interaction network involved in TNF-mediated apoptosis and inflammatory responses, respectively. They could potentially interact through the TNF pathway or in the regulation of inflammatory cytokines.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35F2](https://www.ncbi.nlm.nih.gov/gene?term=SLC35F2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 136 (min len 4) | There is no clear functional association between POLG2 and SLC35F2 based on their known biology and the provided context. POLG2 is involved in DNA synthesis while SLC35F2 is a putative solute transporter that interacts with various proteins involved in signaling pathways.|
| [C2CD2L](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between C2CD2L and TSPAN4 is their roles in calcium and phosphoinositide signaling. C2CD2L transports phosphatidylinositol to the cell membrane and helps maintain phosphoinositide pools, while TSPAN4 mediates signal transduction downstream of a variety of receptors, including those involved in calcium signaling. This suggests that these genes may work together in regulating calcium and phosphoinositide signaling pathways.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Based on their interaction network, CFLAR and MSH6 may be functionally associated in apoptosis and DNA repair pathways through interactions with proteins such as TP53 and BIRC3. CFLAR is an apoptosis regulator that inhibits TNFRSF6-mediated apoptosis, while MSH6 is a component of the post-replicative DNA mismatch repair system.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM50A](https://www.ncbi.nlm.nih.gov/gene?term=TMEM50A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between FRS2 and TMEM50A based on their biological roles. FRS2 is an adapter protein that modulates signaling pathways, while TMEM50A is a subunit of the propionyl-CoA carboxylase enzyme involved in the catabolism of fatty acids and amino acids.|
| [HOXA10](https://www.ncbi.nlm.nih.gov/gene?term=HOXA10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPS2](https://www.ncbi.nlm.nih.gov/gene?term=MRPS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 713 (min len 4) | There is no clear functional association between HOXA10 and MRPS2 based on the available information. HOXA10 is a transcription factor involved in developmental regulation while MRPS2 is involved in mitochondrial ribosome formation and stability. However, there is a protein interaction network between the two genes involving EGFR, MAPK14, and other proteins.|
| [BACE2](https://www.ncbi.nlm.nih.gov/gene?term=BACE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.0</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on the BioGRID protein interaction network and context, BACE2 and RRP1B are functionally associated as they have two shortest paths between them. BACE2 is involved in the proteolytic processing of amyloid precursor protein while RRP1B facilitates viral mRNA transcription following influenza A virus infection. There is no clear biological process linking these two genes.|
| [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.7</br>**STRING score:** 237</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 42 (min len 3) | There is no clear functional association between ETFB and GAPDH based on the provided context and biological knowledge. ETFB is involved in mitochondrial electron transport during fatty acid and amino acid metabolism, while GAPDH is a key enzyme in glycolysis with additional roles in nuclear functions, cytoskeleton organization, innate immunity, and transcription regulation through nitrosylase activity.|
| [PDIA5](https://www.ncbi.nlm.nih.gov/gene?term=PDIA5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RTN2](https://www.ncbi.nlm.nih.gov/gene?term=RTN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.1</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between PDIA5 and RTN2 may include regulation of protein folding and trafficking, as well as involvement in signaling cascades and glucose uptake processes. However, there is not enough information from the provided context to draw clear conclusions about their precise functional relationship.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A13](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | There is currently no known direct functional association between PUM2 and SLC25A13. PUM2 is a post-transcriptional repressor of gene expression and SLC25A13 is a mitochondrial electrogenic antiporter. They could potentially have indirect interactions through pathways related to energy metabolism and RNA processing.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.3</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 77 (min len 3) | There is no clear functional association between MBNL1 and MYC based solely on their known biology and protein interaction network. MBNL1 is involved in alternative splicing regulation and inhibits the pre-mRNA exon inclusion of cardiac troponin-T. MYC is a transcription factor that activates the transcription of growth-related genes.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.6</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association: MYC is a transcription factor that regulates growth-related genes and activates the transcription of VEGFA. PLSCR1 catalyzes calcium-induced phospholipid movement and negatively regulates phagocytosis. Both genes may contribute to cytokine-regulated cell proliferation and differentiation. There is no direct biological process connecting these two genes based on known literature.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZW10](https://www.ncbi.nlm.nih.gov/gene?term=ZW10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | There is no clear functional association between MYC and ZW10 based on the provided context and known biology. MYC is a transcription factor that regulates growth-related genes, while ZW10 is involved in the mitotic checkpoint and membrane traffic between the Golgi and the endoplasmic reticulum.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UGDH](https://www.ncbi.nlm.nih.gov/gene?term=UGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.6</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between DLD and UGDH are related to glucose metabolism and glycosaminoglycan biosynthesis, respectively. However, there is no known direct connection between these two biological processes.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 773</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: BHLHE40 is a transcriptional repressor involved in the regulation of the circadian rhythm, while STAT3 mediates cellular responses to interleukins and is involved in cell cycle regulation. There is no clear biological reason for these two genes to be functionally associated.|
| [CSNK1E](https://www.ncbi.nlm.nih.gov/gene?term=CSNK1E[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CSNK1E is involved in Wnt signaling, phosphorylates DVL1 and DVL2, and interacts with LRP5 and LRP6, while STX1A plays a role in calcium-dependent exocytosis and endocytosis, participates in the SNARE complex, and interacts with STXBP1. There is no known direct functional association between these two genes in TCGA-LUSC, but they may play roles in cellular processes related to intracellular signaling and membrane trafficking in lung squamous cell carcinoma.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.0</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 188 (min len 3) | CDC42 and RAB4A both play a role in vesicular transport, suggesting a potential functional association in this process. CDC42 also regulates cellular responses such as cell migration and spindle microtubule attachment, while RAB4A mediates VEGFR2 endosomal trafficking and platelet alpha-granule release during activation and aggregation.|
| [MOK](https://www.ncbi.nlm.nih.gov/gene?term=MOK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.0</br>**STRING score:** 213</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | The BioGRID protein interaction network shows a potential functional association between MOK and MYC, as they are connected by several short paths. MOK is involved in phosphorylation and negatively regulates cilium length, while MYC is a transcription factor that activates growth-related genes and regulates somatic reprogramming. However, without additional information, it is unclear what the specific biological implications of their association may be.|
