| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: CFLAR is an apoptosis regulator and MBNL1 is involved in pre-mRNA alternative splicing regulation. There is no known direct biological connection between these two genes, but the shortest path between them in the BioGRID protein interaction network suggests a potential indirect interaction through EGFR/KRAS.|
| [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 104.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between ETFB and TMEM97 could be related to cellular cholesterol homeostasis or cytochrome P450 protein activity. ETFB is involved in mitochondrial electron transfer and fatty acid oxidation, while TMEM97 is an intracellular orphan receptor that binds numerous drugs and may be involved in regulating cell survival and morphology. There is no clear biological mechanism linking these two genes.|
| [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 98.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | There are no known direct functional associations between NVL and VAT1. NVL is involved in pre-rRNA processing and assembly of the telomerase holoenzyme, while VAT1 plays a role in calcium-regulated keratinocyte activation and mitochondrial fusion regulation. The shortest path in the BioGRID protein interaction network between these genes involves several intermediate proteins.|
| [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between EBNA1BP2 and POLE2 could be in DNA repair and chromosomal DNA replication, as both genes are involved in these processes. EBNA1BP2 is specifically required for the processing of 27S preribosomal RNA, while POLE2 is an accessory component of the DNA polymerase epsilon complex. There is no clear functional association between these two genes based on the provided information.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 95.0</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CFLAR and IKBKE could involve the regulation of inflammatory responses, specifically through the activation of NF-kappa-B and type I IFN signaling pathways. CFLAR acts as an apoptosis regulator, while IKBKE is involved in the phosphorylation of IRF3 and IRF7, ultimately leading to the transcriptional activation of pro-inflammatory and antiviral genes.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC35F2](https://www.ncbi.nlm.nih.gov/gene?term=SLC35F2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 136 (min len 4) | There is no clear biological pathway connecting POLG2 and SLC35F2. POLG2 is involved in DNA synthesis and exonuclease activities, while SLC35F2 is a putative solute transporter that interacts with SH3 domain-containing proteins and poly-ubiquitinated proteins. Without more information, it is difficult to determine any specific functional associations between these two genes.|
| [C2CD2L](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSPAN4](https://www.ncbi.nlm.nih.gov/gene?term=TSPAN4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no obvious functional association between C2CD2L and TSPAN4 based on their known biological processes and interaction network. C2CD2L is involved in phosphatidylinositol transport and insulin secretion regulation while TSPAN4 is a non-receptor tyrosine kinase involved in signal transduction downstream of various transmembrane receptors and regulates innate and adaptive immunity, cell adhesion, osteoclast maturation, platelet activation, and vascular development.|
| [CFLAR](https://www.ncbi.nlm.nih.gov/gene?term=CFLAR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: CFLAR is involved in apoptosis regulation, while MSH6 is involved in post-replicative DNA mismatch repair. These two genes may be functionally associated through a potential role for MSH6 in the repair of DNA damage caused by apoptotic signals.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM50A](https://www.ncbi.nlm.nih.gov/gene?term=TMEM50A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between FRS2 and TMEM50A based on their known biological roles. FRS2 is an adapter protein involved in signaling pathways, while TMEM50A is a subunit of a mitochondrial enzyme involved in fatty acid catabolism.|
| [HOXA10](https://www.ncbi.nlm.nih.gov/gene?term=HOXA10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPS2](https://www.ncbi.nlm.nih.gov/gene?term=MRPS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 713 (min len 4) | Based on the BioGRID interaction network and context, there may be a functional association between HOXA10 and MRPS2 through their interactions with EGFR and involvement in mitochondrial ribosome biogenesis, respectively. However, further investigation is needed to determine the nature and significance of this association.|
| [BACE2](https://www.ncbi.nlm.nih.gov/gene?term=BACE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.0</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on their interaction network and context, BACE2 and RRP1B may function together in the proteolytic processing of CLTRN in pancreatic beta cells and may both play a role in response to viral infection, with RRP1B promoting viral mRNA transcription following influenza A virus infection.|
| [ETFB](https://www.ncbi.nlm.nih.gov/gene?term=ETFB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.7</br>**STRING score:** 237</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 42 (min len 3) | There could be a possible functional association between ETFB and GAPDH in the mitochondrial respiratory chain via ETF-ubiquinone oxidoreductase.   ETFB is a heterodimeric electron transfer protein that accepts electrons from various mitochondrial dehydrogenases, while GAPDH has both glyceraldehyde-3-phosphate dehydrogenase and nitrosylase activities, playing a role in glycolysis and nuclear functions, respectively.|
| [PDIA5](https://www.ncbi.nlm.nih.gov/gene?term=PDIA5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RTN2](https://www.ncbi.nlm.nih.gov/gene?term=RTN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.1</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between PDIA5 and RTN2 could involve protein trafficking and processing, as well as regulation of signaling cascades. PDIA5 phosphorylates phosphatidylinositols and interacts with CALR, while RTN2 inhibits APP processing and interacts with TMEM33, suggesting they may play roles in related cellular processes. The shortest path in the BioGRID protein interaction network between PDIA5 and RTN2 suggests a direct physical interaction between them via VAPA.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A13](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.4</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Based on the BioGRID network, PUM2 and SLC25A13 may have functional associations related to RNA processing and mitochondrial transport, respectively. However, without additional evidence, it is unclear whether these interactions occur in a biological context or if they are merely coincidental.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.3</br>**STRING score:** 245</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 77 (min len 3) | Possible functional associations between MBNL1 and MYC could include regulation of splicing and transcriptional activation of growth-related genes. MBNL1 mediates pre-mRNA alternative splicing regulation and inhibits/increases exon inclusion while MYC activates the transcription of growth-related genes. They both interact with HNRNPH1 and could potentially compete as they have opposing roles in splicing regulation.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLSCR1](https://www.ncbi.nlm.nih.gov/gene?term=PLSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.6</br>**STRING score:** 372</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 29 (min len 3) | Possible functional association: MYC and PLSCR1 are not known to have direct interactions, but MYC is a transcription factor involved in growth regulation and angiogenesis, while PLSCR1 affects lipid scrambling and apoptosis. It is possible that MYC could regulate PLSCR1 expression or activity in certain contexts, such as cell proliferation or differentiation.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZW10](https://www.ncbi.nlm.nih.gov/gene?term=ZW10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 35 (min len 3) | There is no clear indication of a direct functional association between MYC and ZW10. MYC is a transcription factor involved in the regulation of growth-related genes and somatic reprogramming. ZW10 is an essential component of the mitotic checkpoint and is involved in the regulation of membrane traffic between the Golgi and the endoplasmic reticulum. However, both genes are involved in important cellular processes, and alterations in their expression or function can contribute to the development or progression of cancer.|
| [DLD](https://www.ncbi.nlm.nih.gov/gene?term=DLD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UGDH](https://www.ncbi.nlm.nih.gov/gene?term=UGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.6</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between DLD and UGDH may involve their interaction within the multimeric pyruvate dehydrogenase complex or the 2-oxoglutarate dehydrogenase complex, both involved in the metabolism of energy substrates. DLD is also involved in hyperactivating spermatozoa during capacitation, while UGDH catalyzes the formation of UDP-alpha-D-glucuronate, a constituent of complex glycosaminoglycans necessary for proper development.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 773</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: BHLHE40 negatively regulates circadian rhythm genes and transcription factors, while STAT3 mediates cellular responses to growth factors and regulates inflammatory response. It is possible that these genes may interact in the regulation of immune response and inflammation through circadian rhythm dysregulation, possibly impacting cell cycle regulation and gene expression in inflammatory processes.|
| [CSNK1E](https://www.ncbi.nlm.nih.gov/gene?term=CSNK1E[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | CSNK1E and STX1A may functionally associate in the regulation of circadian rhythm and neurotransmitter release. CSNK1E is a central component of the circadian clock and can regulate PER1 and PER2 nuclear transport and degradation, while STX1A plays an essential role in hormone and neurotransmitter release through its participation in the SNARE complex.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.0</br>**STRING score:** 187</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 188 (min len 3) | Possible functional association: CDC42 is involved in regulating cellular responses, including cell migration and phagocytosis, while RAB4A plays a role in vesicular transport and endosomal recycling. There is no clear direct functional association between these genes, but they may indirectly contribute to cellular processes such as cell adhesion and migration through the regulation of vesicular trafficking.|
| [MOK](https://www.ncbi.nlm.nih.gov/gene?term=MOK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.0</br>**STRING score:** 213</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | MOK and MYC have protein-protein interactions mediated by SDF4, HUWE1, NAA50, CAMK2D, and DGKD. MOK is involved in negative regulation of cilium length and has been shown to interact with bacterial proteins, while MYC is a transcription factor that regulates growth-related genes and is involved in somatic reprogramming. There is no known direct functional association between MOK and MYC based on provided information.|
