| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MAPKAPK5 and PSME2 may include immunoproteasome assembly, efficient antigen processing, and post-transcriptional regulation. MAPKAPK5 is involved in mTORC1 signaling, post-transcriptional regulation of MYC, and acts as a tumor suppressor. PSME2 is required for efficient antigen processing and enhances the generation of class I binding peptides by altering the cleavage pattern of the proteasome. The shortest path between these genes in the BioGRID protein interaction network involves MAPK6, PARKIN, and ZRANB1. There is no direct functional association between these two genes.|
| [PRSS23](https://www.ncbi.nlm.nih.gov/gene?term=PRSS23[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between PRSS23 and SQSTM1 are unclear based on the provided context. PRSS23 is involved in catalyzing reactions in the methionine recycle pathway and down-regulating cell migration. SQSTM1 acts as an autophagy receptor required for the selective degradation of specific protein aggregates.|
| [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CHMP6 and HAT1 are not clear from the provided context as they appear to be involved in different biological processes. CHMP6 is involved in endosomal sorting and multivesicular body formation, while HAT1 is a histone acetyltransferase.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NSA2](https://www.ncbi.nlm.nih.gov/gene?term=NSA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between KDM5B and NSA2 are unclear as they are involved in separate biological processes. KDM5B is a histone demethylase that plays a role in histone code, while NSA2 is involved in ribosomal subunit biogenesis.|
| [GNAI2](https://www.ncbi.nlm.nih.gov/gene?term=GNAI2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.7</br>**STRING score:** 193</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between GNAI2 and KDELR2 in TCGA-LUSC may involve the regulation of intracellular trafficking and localization of proteins by KDELR2, which could affect signaling pathways mediated by GNAI2, potentially influencing cell proliferation and survival. Additionally, GNAI2 may regulate the trafficking or expression levels of KDELR2 or other proteins localized in the endoplasmic reticulum-Golgi complex.|
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: COG7 is required for normal Golgi function while NOTCH1 is involved in angiogenesis and cellular differentiation. It is possible that COG7 may play a role in the trafficking of NOTCH1 within the Golgi complex to facilitate its processing and signaling functions.  COG7 is a component of the conserved oligomeric Golgi complex and is required for normal Golgi morphology and localization. NOTCH1 functions as a receptor for membrane-bound ligands to regulate cell-fate determination, affect differentiation, proliferation, and apoptotic programs. It is involved in angiogenesis, negative regulation of endothelial cell proliferation and migration, maturation of both CD4(+) and CD8(+) cells in the thymus, and important for follicular differentiation. COG7 may play a role in the trafficking of NOTCH1 within the Golgi complex to facilitate its processing and signaling functions.|
| [DDIT4](https://www.ncbi.nlm.nih.gov/gene?term=DDIT4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: The shortest path in the BioGRID interaction network suggests that DDIT4 and TPM1 may interact through PARK2, LRRK2, and EGFR. DDIT4 inhibits the activity of mTORC1, playing a role in responses to cellular energy levels and stress, while TPM1 binds to actin filaments in muscle and non-muscle cells, playing a central role in the calcium-dependent regulation of vertebrate striated muscle contraction. However, without further investigation, it is unclear what specific functional associations may exist between DDIT4 and TPM1.|
| [CLPX](https://www.ncbi.nlm.nih.gov/gene?term=CLPX[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KCNK1](https://www.ncbi.nlm.nih.gov/gene?term=KCNK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between CLPX and KCNK1 is unclear based on the provided information. CLPX is an ATP-dependent specificity component of the Clp protease complex and has chaperone functions. KCNK1 is an ion channel that contributes to passive transmembrane potassium transport and regulation of the resting membrane potential in astrocytes, kidneys, and other tissues.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 162 (min len 4) | There is no clear functional association between PRKAG2 and SLC1A4 based on their known functions and interactions. PRKAG2 is involved in energy metabolism regulation and cellular polarity, while SLC1A4 is a sodium-dependent neutral amino-acid transporter. The shortest path between them in the BioGRID interaction network is mediated by the NEK4 protein, which has diverse functions including cell division and DNA damage response.|
| [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP14A](https://www.ncbi.nlm.nih.gov/gene?term=UTP14A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | HK1 and UTP14A are not known to have a direct functional association in biological processes. However, HK1 is involved in glycolysis and innate immunity, while UTP14A is involved in ribosome biogenesis.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.3</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CEBPD and IKBKE are functionally associated with immune response and inflammation. CEBPD is a transcriptional activator that regulates the expression of genes involved in these processes, while IKBKE is a serine/threonine kinase that plays an essential role in regulating inflammatory responses to viral infection. The shortest path between these two genes in the BioGRID protein interaction network involves several other genes involved in immune and inflammatory responses.|
| [PAK6](https://www.ncbi.nlm.nih.gov/gene?term=PAK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP14](https://www.ncbi.nlm.nih.gov/gene?term=USP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | PAK6 is a serine/threonine kinase that regulates gene transcription and interacts with GTP-bound CDC42/p21 and RAC1. USP14 is a proteasome-associated deubiquitinase that releases ubiquitin from targeted proteins and plays a role in innate immune defense against viruses. There are several shortest paths connecting the two genes in the BioGRID protein interaction network and potential functional associations may involve proteasomal degradation of PAK6. However, more experimental evidence is needed to confirm any specific biological terms.|
| [PRKCD](https://www.ncbi.nlm.nih.gov/gene?term=PRKCD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.5</br>**STRING score:** 676</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no obvious functional association between PRKCD and RHOA based on their known biological functions and interactions. PRKCD plays a role in apoptosis, cell cycle regulation, and immune response, while RHOA is involved in cytoskeletal organization and microtubule dynamics. However, both genes have been implicated in cancer development and progression.|
| [ERBB2](https://www.ncbi.nlm.nih.gov/gene?term=ERBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.6</br>**STRING score:** 478</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 85 (min len 3) | ERBB2 and HSPB1 are functionally associated in TCGA-LUSC through a protein interaction network. ERBB2 is involved in transcriptional regulation and enhances protein synthesis and cell growth, while HSPB1 functions as a molecular chaperone regulating stress resistance and actin organization. They may interact to support protein folding and cell growth in lung cancer.|
| [HTRA1](https://www.ncbi.nlm.nih.gov/gene?term=HTRA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBTPS1](https://www.ncbi.nlm.nih.gov/gene?term=MBTPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between HTRA1 and MBTPS1 in TCGA-LUSC cancer type can be that HTRA1 may regulate the availability of insulin-like growth factors and inhibit signaling mediated by TGF-beta family members, while MBTPS1 plays a key role in the regulation of mega vesicle-mediated collagen trafficking and participates in biogenesis of lysosomes by mediating protein cleavage, including GNPTAB, which bridges GNPTAB to MBTPS1. There is no known direct interaction between these two proteins according to the BioGRID protein interaction network.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLB](https://www.ncbi.nlm.nih.gov/gene?term=POLB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.1</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | The possible functional association between INTS3 and POLB is in DNA repair. INTS3 is a component of the SOSS complex that promotes DNA repair and G2/M checkpoint, while POLB plays a key role in base-excision repair by conducting 'gap-filling' DNA synthesis. The interaction between INTS3 and POLB may be important in the repair of double-strand breaks and maintenance of genomic stability.|
| [AXIN1](https://www.ncbi.nlm.nih.gov/gene?term=AXIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.9</br>**STRING score:** 979</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | AXIN1 and NOTCH1 may have a functional association in regulating cell fate determination and development. AXIN1 is involved in modulating Wnt-signaling and enhancing TGF-beta signaling, while NOTCH1 is a receptor for membrane-bound ligands and regulates cell-fate determination, angiogenesis, and follicular differentiation. The shortest path between AXIN1 and NOTCH1 in the BioGRID protein interaction network involves GSK3B, CTNNB1, MAP3K1, PPP2CA, ARR1, and suggests a potential regulatory relationship.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.7</br>**STRING score:** 263</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between MYC and MYL9 in TCGA-LUSC include regulation of cell contractile activity, cytokinesis, and cell locomotion through MYL9 phosphorylation, as well as MYC activation of growth-related genes and somatic reprogramming. It is not clear from available information whether or not these two genes have direct interactions, but further investigation could clarify their potential association in the context of cancer.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK25](https://www.ncbi.nlm.nih.gov/gene?term=STK25[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.6</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association: STK25 may regulate protein transport events, which could intersect with EED's role in recognizing and repressing gene expression through PRC2/EED-EZH2 complex.|
| [HAX1](https://www.ncbi.nlm.nih.gov/gene?term=HAX1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context, there is no obvious functional association between HAX1 and RPA1. HAX1 is involved in regulating the cortical actin cytoskeleton and processes such as endocytosis and cell survival, while RPA1 plays a crucial role in DNA replication and repair.|
