| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MAPKAPK5](https://www.ncbi.nlm.nih.gov/gene?term=MAPKAPK5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 135.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no known direct functional associations between MAPKAPK5 and PSME2. MAPKAPK5 is a tumor suppressor involved in mTORC1 signaling and post-transcriptional regulation, while PSME2 is implicated in immunoproteasome assembly and required for efficient antigen processing. The shortest path between the two genes in the BioGRID protein interaction network involves intermediaries MAPK6, PARK2, and ZRANB1.|
| [PRSS23](https://www.ncbi.nlm.nih.gov/gene?term=PRSS23[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 134.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association: SQSTM1 is involved in autophagy and PRSS23 is involved in down-regulating cell migration and hepatitis C virus replication, but there is no clear biological connection between these two genes based on the provided information.|
| [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 130.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: None evident based on provided context and known biology of CHMP6 and HAT1.  CHMP6 is a core component of the endosomal sorting required for transport complex III (ESCRT-III) which is involved in multivesicular bodies formation and lysosomal degradation. HAT1 is a histone acetyltransferase complex involved in chromatin remodeling and epigenetic regulation.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NSA2](https://www.ncbi.nlm.nih.gov/gene?term=NSA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 122.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between KDM5B and NSA2 in specific biological terms are unclear based solely on the given context. KDM5B is a histone demethylase and transcriptional corepressor, while NSA2 is involved in the biogenesis of the 60S ribosomal subunit. However, the shortest path in the BioGRID protein interaction network between these two genes suggests indirect interaction via BRD4, CHD3, MYC, KRAS, and MYCN.|
| [GNAI2](https://www.ncbi.nlm.nih.gov/gene?term=GNAI2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.7</br>**STRING score:** 193</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between GNAI2 and KDELR2 in TCGA-LUSC may involve regulation of intracellular signaling and localization. GNAI2 regulates intracellular dopamine receptor levels and interacts with intracellular signaling proteins, while KDELR2 maintains localization of endoplasmic reticulum resident proteins.|
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 115.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | COG7 is involved in normal Golgi function while NOTCH1 acts as a receptor for membrane-bound ligands to regulate cell-fate determination, angiogenesis, maturation of CD4(+) and CD8(+) cells, follicular differentiation, left/right symmetry determination, and somite formation. There is no known functional association between COG7 and NOTCH1 in any specific biological processes.|
| [DDIT4](https://www.ncbi.nlm.nih.gov/gene?term=DDIT4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | One possible functional association between DDIT4 and TPM1 is their involvement in cellular stress response pathways. DDIT4 regulates cell growth and survival during stress and TPM1 stabilizes actin filaments during stress-induced contraction. The shortest path between them in the BioGRID protein interaction network also involves proteins associated with stress response, such as PARK2 and LRRK2.|
| [CLPX](https://www.ncbi.nlm.nih.gov/gene?term=CLPX[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KCNK1](https://www.ncbi.nlm.nih.gov/gene?term=KCNK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear indication of a functional association between CLPX and KCNK1 in the provided context and network information. However, CLPX is involved in ATP-dependent specificity in protein degradation and maintaining mitochondrial nucleoid structure, while KCNK1 contributes to passive transmembrane potassium transport and regulation of membrane potential in various tissues, including the brain and kidney.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC1A4](https://www.ncbi.nlm.nih.gov/gene?term=SLC1A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 114.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 162 (min len 4) | Possible functional association between PRKAG2 and SLC1A4 could be in cellular metabolism and amino acid transport. PRKAG2 plays a role in regulating cellular energy metabolism through AMPK activation, while SLC1A4 is involved in the transport of several amino acids. The interaction network also suggests a possible interaction between NEK4, which regulates centrosome duplication, and both PRKAG2 and SLC1A4.|
| [HK1](https://www.ncbi.nlm.nih.gov/gene?term=HK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP14A](https://www.ncbi.nlm.nih.gov/gene?term=UTP14A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 113.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | There is no obvious functional association between HK1 and UTP14A based on current context and their known biology. HK1 is involved in catalyzing phosphorylation of hexose and acts as a pattern recognition receptor in innate immunity. UTP14A is needed for ribosome biogenesis and interacts with DHX37.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.3</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CEBPD and IKBKE are likely functionally associated in the regulation of immune and inflammatory responses. CEBPD is a transcription factor that enhances the transcription of genes involved in immune and inflammatory responses, while IKBKE plays an essential role in regulating inflammatory responses to viral infection by activating the NF-kappa-B and STAT signaling pathways. Their shortest path of interaction suggests they both play a role in regulating IKBKE.|
| [PAK6](https://www.ncbi.nlm.nih.gov/gene?term=PAK6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP14](https://www.ncbi.nlm.nih.gov/gene?term=USP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 112.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association: PAK6 and USP14 may be involved in regulating protein degradation and proteasome activity based on their interactions with CDH1, MDM2, and the proteasome, and their respective roles in phosphorylation and deubiquitination.   PAK6 is a serine/threonine protein kinase that regulates gene transcription, cytoskeleton regulation, and cell survival by interacting with and phosphorylating various proteins. USP14 is a proteasome-associated deubiquitinase that regulates protein degradation, endoplasmic reticulum-associated degradation, and synaptic development. The shortest paths between PAK6 and USP14 suggests possible interactions with the androgen receptor, MAP2K6/MAPKK6, CDC42/p21, RAC1, and CXCR4 that may modulate cellular processes such as transcription, signaling, migration, and chemotaxis. However, further studies are needed to determine the precise roles and functional associations between these genes in the context of TCGA-LUSC and cancer.|
| [PRKCD](https://www.ncbi.nlm.nih.gov/gene?term=PRKCD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.5</br>**STRING score:** 676</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PRKCD and RHOA include regulation of actin dynamics and cell migration. PRKCD is known to inhibit filopodia formation and actin polymerization by negatively regulating VASP phosphorylation, while RHOA is involved in regulating actin cytoskeleton dynamics and cell migration.|
| [ERBB2](https://www.ncbi.nlm.nih.gov/gene?term=ERBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.6</br>**STRING score:** 478</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 85 (min len 3) | There is no clear functional association between ERBB2 and HSPB1 based on the provided information. ERBB2 is involved in transcriptional regulation and cell growth, while HSPB1 functions as a molecular chaperone and is important for stress resistance and actin organization. However, the shortest path in the BioGRID interaction network suggests that these genes may interact through SHC1, SH3BGRL, GRB2, CTNNB1, and PAK1.|
| [HTRA1](https://www.ncbi.nlm.nih.gov/gene?term=HTRA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MBTPS1](https://www.ncbi.nlm.nih.gov/gene?term=MBTPS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 100.1</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no known functional associations between HTRA1 and MBTPS1 in specific biological terms.   HTRA1 functions as a serine protease with a variety of targets, including extracellular matrix proteins and proteoglycans. MBTPS1 is also a serine protease, which cleaves substrates after hydrophobic or small residues and is involved in the proteolytic activation of SREBPs and ATF-6, protein cleavage of GNPTAB, and M6P-dependent Golgi-to-lysosome trafficking of lysosomal enzymes.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLB](https://www.ncbi.nlm.nih.gov/gene?term=POLB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 99.1</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | INTS3 and POLB are functionally associated in DNA repair, as they are involved in different steps of the process. INTS3 is a component of the SOSS complex, required for efficient homologous recombination-dependent repair of double-strand breaks, while POLB is a repair polymerase that plays a key role in base-excision repair. The interaction between the two proteins may facilitate efficient repair of DNA damage.|
| [AXIN1](https://www.ncbi.nlm.nih.gov/gene?term=AXIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOTCH1](https://www.ncbi.nlm.nih.gov/gene?term=NOTCH1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.9</br>**STRING score:** 979</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations: There are no known direct functional associations between AXIN1 and NOTCH1, however they both play important roles in regulating intracellular signaling pathways. AXIN1 is a key component of the beta-catenin destruction complex in the Wnt signaling pathway, while NOTCH1 is a transmembrane receptor involved in regulating cell-fate determination, angiogenesis, and thymus development.|
| [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.7</br>**STRING score:** 263</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between MYC and MYL9 are unclear from the given context as they do not have direct protein-protein interaction in the BioGRID network. MYC is a transcription factor that activates the transcription of growth-related genes and regulates somatic reprogramming, while MYL9 is a myosin regulatory subunit that plays an important role in regulation of contractile activity in smooth muscle and nonmuscle cells.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK25](https://www.ncbi.nlm.nih.gov/gene?term=STK25[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.6</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association: EED and STK25 may be involved in the regulation of gene expression and cell adhesion. The shortest path between them in the BioGRID network involves proteins associated with cell division and protein transport, suggesting potential roles in these processes.|
| [HAX1](https://www.ncbi.nlm.nih.gov/gene?term=HAX1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPA1](https://www.ncbi.nlm.nih.gov/gene?term=RPA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no clear functional associations between HAX1 and RPA1. HAX1 is involved in actin cytoskeleton regulation, cell migration, endocytosis, and inhibition of apoptosis, while RPA1 is involved in DNA replication, repair, and damage response.|
