| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [P4HA2](https://www.ncbi.nlm.nih.gov/gene?term=P4HA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 864 (min len 4) | Possible functional associations between HMG20B and P4HA2 could involve regulation of cell cycle progression and collagen synthesis, respectively. The shortest path in the BioGRID protein interaction network shows that they may interact indirectly through other proteins involved in splicing, transcription, and translation.|
| [DNTTIP2](https://www.ncbi.nlm.nih.gov/gene?term=DNTTIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFATC4](https://www.ncbi.nlm.nih.gov/gene?term=NFATC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | DNTTIP2 regulates transcriptional activity and is involved in chromatin remodeling, as well as ribosomal subunit assembly. NFATC4 is a Ca(2+)-regulated transcription factor involved in T cell activation and various physiological processes. There is a shortest path of one protein interaction between DNTTIP2 and NFATC4 through APEX1, but no clear functional association between these two genes is known in the literature.|
| [P4HA2](https://www.ncbi.nlm.nih.gov/gene?term=P4HA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRSF4](https://www.ncbi.nlm.nih.gov/gene?term=SRSF4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between P4HA2 and SRSF4 include involvement in alternative splice site selection during pre-mRNA splicing and potential regulation of MAPT/Tau exon 10 splicing. P4HA2 catalyzes post-translational formation of 4-hydroxyproline in collagen, while SRSF4 plays a role in selecting alternate splice sites during pre-mRNA splicing and interacts with PNN.|
| [DPH2](https://www.ncbi.nlm.nih.gov/gene?term=DPH2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [P4HA2](https://www.ncbi.nlm.nih.gov/gene?term=P4HA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: P4HA2 catalyzes the post-translational formation of 4-hydroxyproline in collagen, while DPH2 is involved in histidine biosynthesis and interacts with DPH1. There is no direct biological process linking these two genes.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM109](https://www.ncbi.nlm.nih.gov/gene?term=TMEM109[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no known functional association between MTX2 and TMEM109. MTX2 is involved in mitochondrial protein transport and interacts with the MICOS complex, while TMEM109 may protect against DNA damage and form voltage-gated channels.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPS16](https://www.ncbi.nlm.nih.gov/gene?term=MRPS16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association: APOE is involved in viral infection, while MRPS16 plays a role in cell survival and signaling. There is no known direct interaction between them, but they may have indirect effects on each other through their involvement in cellular processes such as autophagy and PI3K-AKT signaling.|
| [GRB10](https://www.ncbi.nlm.nih.gov/gene?term=GRB10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are no known functional associations between GRB10 and HMG20B in terms of biological processes. GRB10 is an adapter protein involved in modulating signaling pathways of tyrosine kinase receptors, while HMG20B is involved in G2 and mitosis phase progression and regulation of gene expression through a histone deacetylase complex.|
| [SQSTM1](https://www.ncbi.nlm.nih.gov/gene?term=SQSTM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.7</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional association:      SQSTM1 is involved in autophagy and protein degradation while USP1 is a negative regulator of DNA damage repair and involved in PCNA-mediated translesion synthesis. It is possible that USP1 could regulate SQSTM1-mediated selective macroautophagy by modifying or regulating the ubiquitination status of target proteins.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2E1](https://www.ncbi.nlm.nih.gov/gene?term=UBE2E1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1428 (min len 4) | Possible functional associations between APOE and UBE2E1 could be their involvement in protein degradation and clearance pathways, as UBE2E1 is an E2 ubiquitin-conjugating enzyme and APOE has been linked to autophagy pathways during viral infections.|
| [MFSD10](https://www.ncbi.nlm.nih.gov/gene?term=MFSD10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX13](https://www.ncbi.nlm.nih.gov/gene?term=SNX13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: There is currently no known direct functional association between MFSD10 and SNX13 based on available literature and protein interaction data.   MFSD10: Likely functions as a transporter for certain non-steroidal anti-inflammatory drugs and other organic anions across the luminal membranes of renal proximal tubules for their excretion into urine. It may also interact with Herpes simplex virus 1 protein UL12.   SNX13: May be involved in various stages of intracellular trafficking, particularly in endosome homeostasis. It acts as a GAP for Galphas. It may also interact with Sindbis virus non-structural protein 3, which inhibits the formation of host stress granules on viral mRNAs and orchestrate the assembly of viral replication complexes.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBQLN2](https://www.ncbi.nlm.nih.gov/gene?term=UBQLN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.3</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations between APOE and UBQLN2 are in the regulation of autophagy and protein degradation mechanisms. APOE can interact with HCV envelope glycoprotein E2 to promote viral particle assembly and may up-regulate cellular autophagy. UBQLN2 plays an important role in ubiquitin-proteasome system, autophagy, and endoplasmic reticulum-associated protein degradation pathways.|
| [IGFBP3](https://www.ncbi.nlm.nih.gov/gene?term=IGFBP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MUC1](https://www.ncbi.nlm.nih.gov/gene?term=MUC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.8</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: Both genes are involved in tumor progression. IGFBP3 inhibits growth promoting effects, induces apoptosis, and interacts with TMEM219. MUC1 promotes tumor progression, influences cell signaling, and interacts with various proteins such as ERBB2, ERBB3, ERBB4, and P53, among others. There may be some potential crosstalk between them through the ERK, SRC, NF-kappa-B, or RAS signaling pathways.|
| [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.6</br>**STRING score:** 912</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 88 (min len 3) | Possible functional associations between RFC5 and TP53 could be their involvement in DNA repair pathways and cell cycle regulation. RFC5 is an accessory protein required for DNA elongation, while TP53 acts as a tumor suppressor and negatively regulates cell division. Their interaction may suggest a role in maintaining genome stability and preventing cancer development.|
| [PNKP](https://www.ncbi.nlm.nih.gov/gene?term=PNKP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAD51C](https://www.ncbi.nlm.nih.gov/gene?term=RAD51C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.0</br>**STRING score:** 349</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between PNKP and RAD51C are their roles in DNA repair pathways, specifically in non-homologous end-joining (NHEJ) and homologous recombination (HR), respectively. PNKP removes 3'-phosphates and phosphorylates 5'-hydroxyl groups on DNA backbone, while RAD51C is involved in the HR pathway of double-stranded DNA breaks.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYC](https://www.ncbi.nlm.nih.gov/gene?term=MYC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.9</br>**STRING score:** 297</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 72 (min len 3) | Possible functional associations between APOE and MYC include regulation of growth-related genes, activation of target gene expression through RNA polymerase II pause release, and transcriptional activity enhancement through interaction with TRIM6 or ABI1. APOE participates in the attachment and infectivity of hepatitis C virus and may be upregulated via viral promoted autophagy. MYC is a transcription factor that binds DNA in a non-specific manner and positively regulates the transcription of HNRNPA1, HNRNPA2, and PTBP1.|
| [FBXL12](https://www.ncbi.nlm.nih.gov/gene?term=FBXL12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KLHL21](https://www.ncbi.nlm.nih.gov/gene?term=KLHL21[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.5</br>**STRING score:** 250</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 133 (min len 4) | Possible functional association between these two genes is that they both play a role in E3 ubiquitin ligase complex. FBXL12 mediates polyubiquitination and proteasomal degradation of CAMK1, leading to G1 cell cycle arrest, while KLHL21 is a substrate-specific adapter of a BCR E3 ubiquitin-protein ligase complex that regulates chromosomal passenger complex localization and mediates AURKB ubiquitination.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAFAH1B2](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.3</br>**STRING score:** 222</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between APOE and PAFAH1B2 in TCGA-LGG cancer type include their involvement in lipid metabolism and potentially in the Reelin pathway, as well as their interaction in the BioGRID protein interaction network. APOE plays a role in microbial infection and interacts with HCV envelope glycoprotein E2, while PAFAH1B2 is an alpha2 catalytic subunit of the cytosolic type I platelet-activating factor (PAF) acetylhydrolase and may play a role in male germ cell meiosis and early spermiogenesis.|
