| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDS5A](https://www.ncbi.nlm.nih.gov/gene?term=PDS5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional association between KDM5B and PDS5A could involve regulating sister chromatid cohesion in mitosis and DNA replication. KDM5B acts as a transcriptional corepressor for FOXG1B and PAX9 and favors breast cancer cell proliferation, whereas PDS5A interacts with the cohesin complex and TP63, playing a role in accurate chromosome partitioning in both meiotic and mitotic cells.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: KDM5B and RRP1B have a shortest path of protein interactions that go through MYC, which is a transcription factor involved in cell growth and proliferation. KDM5B is a histone demethylase that controls gene expression, while RRP1B plays a role in promoting viral mRNA transcription and interacts with the influenza A virus nucleoprotein. There is no direct biological evidence for a functional association between these two genes.|
| [PIP4K2B](https://www.ncbi.nlm.nih.gov/gene?term=PIP4K2B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations: PIP4K2B and RRP1B may be functionally associated in the context of influenza A virus infection as RRP1B interacts with the viral nucleoprotein NP and with RNA-directed RNA polymerase subunits PB1 and PB2, while PIP4K2B negatively regulates insulin signaling and interacts with PIP5Ks to suppress PIP5K-mediated conversion to PtdIns(3,4,5)P3. However, further experimental evidence is required to establish a definitive functional association between these two genes.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.7</br>**STRING score:** 258</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HDAC2 is involved in transcriptional regulation through histone deacetylation, while ELAVL1 stabilizes mRNAs. They may both play a role in regulating gene expression and potentially interact in the regulation of specific genes important in cancer development or progression.|
| [NNT](https://www.ncbi.nlm.nih.gov/gene?term=NNT[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | Possible functional associations between NNT and NR3C1 may involve NNT's role in reactive oxygen species detoxification in the adrenal gland and NR3C1's role in transcriptional activation and repression, suggesting a potential connection between oxidative stress and gene expression regulation.|
| [CDK4](https://www.ncbi.nlm.nih.gov/gene?term=CDK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CDK4 and CHMP6 may be related to cell cycle regulation and endosomal protein sorting, respectively. As CDK4 is a key regulator of G1/S transition and CHMP6 is a core component of the ESCRT-III complex involved in multivesicular body formation and sorting of endosomal cargo proteins, they may play roles in regulating protein trafficking and degradation during cell cycle progression.|
| [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.5</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MAPK1IP1L and TMED10 are not evident based on the provided context. MAPK1IP1L may regulate transcriptional activity and interact with ZNF202, while TMED10 is involved in protein vesicular trafficking and quality control in the endoplasmic reticulum and Golgi.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: ABL1 is a non-receptor tyrosine kinase involved in cytoskeleton remodeling, cell motility and adhesion, and DNA damage response, while CTNNAL1 is a scaffold protein that modulates Rho pathway signaling. It's possible that CTNNAL1 provides a scaffold for ABL1 to regulate Rho pathway signaling.|
| [BBS9](https://www.ncbi.nlm.nih.gov/gene?term=BBS9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Due to their interactions with the BBSome complex and ciliogenesis, BBS9 and REEP5 may have a functional association in protein sorting and transport. BBS9 plays a role in assembly of the BBSome complex and ciliary localization, while REEP5 regulates the organization and function of the sarcoplasmic reticulum in cardiomyocytes.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX13](https://www.ncbi.nlm.nih.gov/gene?term=SNX13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | It is unclear based on the provided context if PUM2 and SNX13 have any direct functional association. However, PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor, while SNX13 may be involved in intracellular trafficking and acts as a GAP for Galphas during microbial infection.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ELAVL1 and KDM5B do not have any known functional associations in the literature. ELAVL1 is an RNA-binding protein that stabilizes mRNAs, while KDM5B is a histone demethylase that plays a central role in the histone code. ELAVL1 is involved in embryonic stem cell differentiation and MYC mRNA stability, while KDM5B is implicated in the proliferation of breast cancer cells and may act as a tumor suppressor in melanoma.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.1</br>**STRING score:** 281</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | ABL1 and MSH6 are involved in distinct biological processes. ABL1 is a non-receptor tyrosine kinase involved in cell growth, survival, and DNA damage response, while MSH6 is a component of the DNA mismatch repair system. There may be functional associations in the context of DNA damage response and potentially in homologous recombination repair.|
| [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.0</br>**STRING score:** 561</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | KDM5A and KDM5B are both histone demethylases that play roles in histone code and gene transcription regulation. There are several interactions between KDM5A and KDM5B in the BioGRID protein interaction network, suggesting that they may functionally associate with each other in the regulation of specific genes.|
| [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 333</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | MSH6 and SMARCC1 may have a functional association in DNA repair and chromatin remodeling. MSH6 is involved in DNA mismatch repair, while SMARCC1 is part of the SWI/SNF chromatin remodeling complex, which alters DNA-histone contacts. In addition, SMARCC1 has been shown to interact with MMR proteins, suggesting a possible role in coordinating chromatin remodeling with DNA repair.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.9</br>**STRING score:** 251</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between APOE and NFKBIB could involve regulation of immune response and inflammation. APOE plays a role in viral infection while NFKBIB inhibits NF-kappa-B, a transcription factor involved in inflammation and immune response. The shortest path in the protein interaction network suggests possible protein-protein interactions between APOE and NFKBIB through other proteins involved in transcription or signaling processes.|
| [FOXJ3](https://www.ncbi.nlm.nih.gov/gene?term=FOXJ3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their known functions, there are no obvious functional associations between FOXJ3 and NR3C1. FOXJ3 is involved in muscle fiber type identity, spermatogenesis, and interacts with G-actin and capping protein. NR3C1 has transcriptional repression and activation activity, binds to DNA, and interacts with several coactivator complexes and chaperones. However, it is possible that they may indirectly interact through shared pathways or protein partners.|
| [EZH2](https://www.ncbi.nlm.nih.gov/gene?term=EZH2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOX4](https://www.ncbi.nlm.nih.gov/gene?term=SOX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.5</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | EZH2 and SOX4 have a shortest path of protein interactions in the BioGRID network, suggesting a potential functional association in cellular processes. EZH2 is a Polycomb group protein involved in histone methylation and transcriptional repression, while SOX4 is a transcriptional activator required for Vgamma2-positive gamma-delta T-cell maturation and skeletal myoblast differentiation.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 437</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: KDM5B is a histone demethylase that plays a role in the histone code and as a transcriptional corepressor. ZNF131 is involved in transcriptional regulation and interacts with various receptors. Both genes have been found to interact with other genes and proteins involved in transcriptional regulation and signaling pathways. It's possible that KDM5B and ZNF131 may work together in these processes.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP7](https://www.ncbi.nlm.nih.gov/gene?term=USP7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.9</br>**STRING score:** 467</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: BIRC2 and USP7 both play a role in regulating apoptosis/cell death and inflammation signaling. BIRC2 is an E3 ubiquitin-protein ligase that regulates NF-kappa-B signaling, while USP7 stabilizes proteins involved in antiviral and innate immune signaling. There may be a potential regulatory interaction between these genes through their involvement in these pathways.|
| [CEP57](https://www.ncbi.nlm.nih.gov/gene?term=CEP57[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDS5A](https://www.ncbi.nlm.nih.gov/gene?term=PDS5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 240</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Both CEP57 and PDS5A are involved in mitosis and contribute to accurate chromosome partitioning. Their interaction in the BioGRID protein interaction network suggests they may have a role in regulating sister chromatid cohesion.|
| [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 994</br>**BioGRID count:** 15</br>**BioGRID shortest paths:** 1 (min len 2) | PSME1 and PSME2 are involved in immunoproteasome assembly and required for efficient antigen processing. As a heterodimer, they form a hexameric ring. There is a direct interaction between these two genes, suggesting they may work together in these processes.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCERG1](https://www.ncbi.nlm.nih.gov/gene?term=TCERG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 468</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: ELAVL1 binds to mRNA, increasing their stability, while TCERG1 regulates transcription elongation of RNA polymerase II. It is possible that ELAVL1 may stabilize TCERG1 mRNA, leading to an increase in TCERG1 expression and potentially affecting its transcriptional regulation role.|
| [CCNE2](https://www.ncbi.nlm.nih.gov/gene?term=CCNE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | CCNE2 and KDM5B are functionally associated as they are connected in the BioGRID protein interaction network through KIF14. CCNE2 controls the cell cycle at late G1 and early S phase through interaction with CDK2 and CDK3, while KDM5B is a histone demethylase that plays a central role in the histone code and acts as a transcriptional corepressor and/or tumor suppressor.|
| [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMF1](https://www.ncbi.nlm.nih.gov/gene?term=PSMF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.6</br>**STRING score:** 260</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are no known functional associations between CHMP6 and PSMF1. CHMP6 is involved in the endosomal sorting required for transport complex III, while PSMF1 plays a role in proteasome function inhibition.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.2</br>**STRING score:** 317</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | BAF155, SMARCC2/BAF170, PHF10/BAF45A, BRD7, BCL7B/BOB1, BCL11A/CTIP1, and BCL11B/CTIP2. Interacts with RB1, JUN, MYC and NR3C1. Interacts with NR2C1; the interaction leaves SMARCA4/BAF190A unable to remodel chromatin and represses transcription. Interacts with SMARCE1/BAF57, SMARCB1/SNF5/INI1, SMARCC1/BAF155, SMARCC2/BAF170, SMARCD3/BAF60C, ARID1A/BAF250A and ARID2/BAF200. Interacts with ZEB1. Interacts with DLX1; the interaction is direct. Interacts with lncRNA Evf2; the interaction is RNA-dependent and leads to inhibition of SMARCA4 ATPase and chromatin remodeling activities. Interacts with KDM5A. Interacts with DPF2/BAF45D; the interaction is direct and activates SMARCA4 ATPase activity and chromatin remodeling activity by inducing conformational changes in SMARCA4. Interacts with FAM156A; the interaction is direct. Interacts with KIAA0317. Interacts with DCTN4 through its helicase SD2 domain (PubMed:31031098). Interacts with SALL1; the interaction is direct in a calcium-independent manner and mediates SMARCA4 recruitment to SALL1-bound enhancers to activate gene expression involved in early nephron development (PubMed:33938409). Interacts via the HMG box domain with the forkhead transcription factor FOXM1 to co-regulate gene expression during cell cycle progression (By similarity). In vitro, interacts with the heat shock transcription factor HSF1 (PubMed:15661742). Interacts with HNRNPU; this interaction induces phosphorylation of HNRNPU in mitosis (PubMed:25986610). Interacts with KLHL22; the interaction regulates its nuclear translocation and ubiquitination (PubMed:24067371). Interacts with BANF1; this interaction is direct and is required for chromatin remodeling by the|
| [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.0</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CTNNAL1 and RNMT are unclear based on the provided information. CTNNAL1 may modulate the Rho pathway signaling and interact with ARHGEF1 while RNMT is a catalytic subunit of the mRNA-capping methyltransferase RNMT:RAMAC complex that methylates the N7 position of added guanosine to the 5'-cap structure of mRNAs, binds RNA, and interacts with importin alpha and elongating form of polymerase II.|
