| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDS5A](https://www.ncbi.nlm.nih.gov/gene?term=PDS5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between KDM5B and PDS5A could be their involvement in regulating chromatin structure during DNA replication and cell division. KDM5B is a histone demethylase that regulates transcription and PDS5A is a regulator of sister chromatid cohesion, both key processes in DNA replication and cell division.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: KDM5B is a histone demethylase that acts as a corepressor for genes like MYC and interacts with MYC as well as RRP1B. RRP1B plays a role in facilitating viral mRNA transcription and interacts with influenza A virus nucleoprotein NP and RNA-directed RNA polymerase subunits PB1 and PB2. There is no clear biological process that connects KDM5B and RRP1B.|
| [PIP4K2B](https://www.ncbi.nlm.nih.gov/gene?term=PIP4K2B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRP1B](https://www.ncbi.nlm.nih.gov/gene?term=RRP1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | As there are no direct interactions between PIP4K2B and RRP1B in the BioGRID protein interaction network, a possible functional association between these two genes could occur through a shared involvement in the regulation of phospholipid metabolism. PIP4K2B participates in the biosynthesis of phosphatidylinositol 4,5-bisphosphate, while RRP1B facilitates the binding of RNA polymerase to capped mRNA. These processes could potentially intersect in the regulation of transcription.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.7</br>**STRING score:** 258</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between ELAVL1 and HDAC2 in TCGA-LGG cancer type are unclear based on the provided context. ELAVL1 is an RNA-binding protein involved in mRNA stability and ESC differentiation, while HDAC2 is a histone deacetylase involved in transcriptional regulation and developmental events. There is no evident overlap in the biological processes where these two genes are involved.|
| [NNT](https://www.ncbi.nlm.nih.gov/gene?term=NNT[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | Possible functional associations: NNT is involved in ROS detoxification and ATP hydrolysis, while NR3C1 is a transcription factor with transcriptional activation and repression activities. There is no direct known functional association between these two genes based on the provided information.|
| [CDK4](https://www.ncbi.nlm.nih.gov/gene?term=CDK4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: CDK4 is a kinase that regulates the cell-cycle during G(1)/S transition, while CHMP6 is a component of the endosomal sorting required for transport complex III (ESCRT-III) which is involved in multivesicular bodies (MVBs) formation and sorting of endosomal cargo proteins into MVBs. It is possible that CHMP6 could be involved in the regulation of CDK4-mediated cell cycle progression.|
| [MAPK1IP1L](https://www.ncbi.nlm.nih.gov/gene?term=MAPK1IP1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMED10](https://www.ncbi.nlm.nih.gov/gene?term=TMED10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.5</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between MAPK1IP1L and TMED10 could be in the vesicular trafficking and quality control of proteins in the endoplasmic reticulum and Golgi, as both genes are involved in the transport of cargo through COPI and COPII-coated vesicles. MAPK1IP1L may act as a cargo receptor and TMED10 as a cargo receptor and protein channel, facilitating the entry of proteins into the ERGIC and secretion of leaderless cargos.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between ABL1 and CTNNAL1 could involve modulating the Rho pathway signaling by providing a scaffold for the Lbc Rho guanine nucleotide exchange factor (CTNNAL1) and regulating multiple processes linked to cell growth and survival such as cytoskeleton remodeling, cell motility and adhesion, receptor endocytosis, autophagy, DNA damage response, and apoptosis (ABL1). ABL1 is also involved in DNA-damage response and apoptosis, while CTNNAL1 interacts with ARHGEF1 and may modulate the Rho pathway signaling.|
| [BBS9](https://www.ncbi.nlm.nih.gov/gene?term=BBS9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [REEP5](https://www.ncbi.nlm.nih.gov/gene?term=REEP5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between BBS9 and REEP5 is their involvement in ciliary trafficking. BBS9 is part of the BBSome complex, which is required for ciliogenesis and interacts with Rab8 to promote extension of the ciliary membrane. REEP5, on the other hand, has been shown to regulate the organization and function of the sarcoplasmic reticulum in cardiomyocytes as well as interact with ATL1, ATL2, ATL3, CKAP4, RTN4, and ZFYVE27, which may play a role in ciliary trafficking and/or membrane dynamics.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX13](https://www.ncbi.nlm.nih.gov/gene?term=SNX13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no obvious functional association between PUM2 and SNX13 based on their known biological roles. PUM2 is involved in post-transcriptional regulation of mRNA targets, while SNX13 is involved in intracellular trafficking and may play a role in endosome homeostasis. The shortest path between these two genes in the BioGRID protein interaction network is through TRIM25, which has a role in the innate immune response and may be involved in regulating protein ubiquitination.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | ELAVL1 is an RNA-binding protein that stabilizes mRNAs, while KDM5B is a histone demethylase that plays a central role in the histone code. There is currently no known functional association between these two genes in specific biological terms based on the provided context.|
| [ABL1](https://www.ncbi.nlm.nih.gov/gene?term=ABL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.1</br>**STRING score:** 281</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations: ABL1 is a non-receptor tyrosine kinase involved in DNA damage response, apoptosis, and regulation of cell adhesion and motility. MSH6 is a component of the DNA mismatch repair system. They may function together in DNA repair processes.|
| [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.0</br>**STRING score:** 561</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | KDM5A and KDM5B are histone demethylases that regulate specific gene transcription by removing methyl groups from histone H3 at the lysine-4 position. They also interact with other proteins, such as RB1, MYC, HDAC1, and FOXG1B, to regulate transcriptional activity and proliferation of cancer cells. Their shortest interaction path is through LMO2, RB1, EZH2, HDAC2, and HDAC1. There are possible functional associations between these two genes in the regulation of histone code and the circadian rhythm, as KDM5A acts as a coactivator of CLOCK-BMAL1-mediated transcriptional activation of PER1/2 genes and increases histone acetylation at PER1/2 promoters by inhibiting HDAC1 activity. Meanwhile, KDM5B represses the CLOCK-BMAL1 heterodimer-mediated transcriptional activation of the core clock component PER2.|
| [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCC1](https://www.ncbi.nlm.nih.gov/gene?term=SMARCC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 333</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | MSH6 is involved in DNA mismatch repair while SMARCC1 is a component of the SWI/SNF chromatin remodeling complex. There is no clear biological reason to suggest a functional association between these two genes based on their known roles and interactions.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.9</br>**STRING score:** 251</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association between APOE and NFKBIB is in the regulation of inflammation. APOE plays a role in innate immune response and is involved in microbial infection, while NFKBIB inhibits NF-kappa-B, a transcription factor that regulates inflammation.|
| [FOXJ3](https://www.ncbi.nlm.nih.gov/gene?term=FOXJ3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.0</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between FOXJ3 and NR3C1. FOXJ3 is involved in muscle fiber type identity, spermatogenesis, and meiosis, while NR3C1 is a transcriptional regulator involved in transport to the nucleus and DNA binding as a homodimer or heterodimer.|
| [EZH2](https://www.ncbi.nlm.nih.gov/gene?term=EZH2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SOX4](https://www.ncbi.nlm.nih.gov/gene?term=SOX4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.5</br>**STRING score:** 908</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | EZH2 is involved in transcriptional repression through methylation of histone H3. SOX4 is a transcriptional activator that plays a role in T-cell development and skeletal myoblast differentiation. Given their roles in transcriptional regulation, it is possible that they may have opposing effects on gene expression and potentially regulate similar pathways in different ways.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.4</br>**STRING score:** 437</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: KDM5B is a histone demethylase involved in transcriptional regulation and is known for its role in cancer cell proliferation. ZNF131 is involved in transcriptional regulation and may act as a repressor of ESR1/ER-alpha signaling, while also playing a role in development and organogenesis. It is possible that these two genes may function together in the regulation of gene expression. The BioGRID protein interaction network suggests they may interact indirectly through other proteins, such as CBX4, TRIM25, KMT2A, and KDM4A.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP7](https://www.ncbi.nlm.nih.gov/gene?term=USP7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.9</br>**STRING score:** 467</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is a possible functional association between BIRC2 and USP7 in innate immune signaling pathways as both genes are involved in the regulation of NF-kappa-B signaling. BIRC2 acts as an E3 ubiquitin-protein ligase and regulates NF-kappa-B signaling while USP7 interacts with TRAF3 and TRAF6 which are innate immune signaling proteins.|
| [CEP57](https://www.ncbi.nlm.nih.gov/gene?term=CEP57[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDS5A](https://www.ncbi.nlm.nih.gov/gene?term=PDS5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 240</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CEP57 and PDS5A may be in regulating sister chromatid cohesion in mitosis and coupling sister chromatid cohesion during mitosis to DNA replication. They may also play roles in DNA repair and interact with the cohesin complex. There is no direct evidence of a specific biological process where both genes are involved in working together.|
| [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME2](https://www.ncbi.nlm.nih.gov/gene?term=PSME2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 994</br>**BioGRID count:** 15</br>**BioGRID shortest paths:** 1 (min len 2) | PSME1 and PSME2 form a heterodimeric ring and are involved in immunoproteasome assembly and efficient antigen processing. They are both members of the PA28 activator complex, which enhances class I binding peptide generation by altering the proteasome cleavage pattern. Therefore, it is likely that they work together to regulate antigen processing.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCERG1](https://www.ncbi.nlm.nih.gov/gene?term=TCERG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 468</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between ELAVL1 and TCERG1 are unknown based on the provided context. ELAVL1 is an RNA-binding protein that stabilizes mRNAs, including MYC and p53, and is involved in ESC differentiation. TCERG1 is a transcription factor that inhibits elongation of transcripts and is necessary for TAT-dependent activation of HIV-1 promoter.|
| [CCNE2](https://www.ncbi.nlm.nih.gov/gene?term=CCNE2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.7</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between CCNE2 and KDM5B may involve cell cycle regulation and epigenetic modifications. CCNE2 controls the cell cycle at the late G1 and early S phases, while KDM5B is a histone demethylase that modifies histone H3, playing a central role in the histone code. The shortest path between them in the BioGRID protein interaction network involves KIF14, a protein that is involved in cytokinesis during cell division.|
| [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMF1](https://www.ncbi.nlm.nih.gov/gene?term=PSMF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.6</br>**STRING score:** 260</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CHMP6 and PSMF1 include regulation of protein degradation through the proteasome and lysosomal pathways, and involvement in cellular processes such as multivesicular body formation, cytokinesis, and viral budding. CHMP6 is a core component of the endosomal sorting required for transport complex III (ESCRT-III), while PSMF1 inhibits the hydrolysis of protein and peptide substrates by the 20S proteasome. Interactions between these two genes may play a role in coordinating these cellular processes.|
| [PLK1](https://www.ncbi.nlm.nih.gov/gene?term=PLK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.2</br>**STRING score:** 317</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 54 (min len 3) | BAF155, SMARCC2/BAF170, ARID2/BAF200, PBRM1/BAF180, BRD7, BRD9, and/or GLTSCR1/BAF57L1 (PubMed:22952240, PubMed:26601204). Interacts with ATR (PubMed:16216318). Interacts with ZEB1 (PubMed:22981593). Interacts with DLX1 (PubMed:24656865). Interacts with HCFC1 (PubMed:15526032). Interacts with LRIG1 (via NPXY motif) (PubMed:19822762). Interacts with MYOD1 (PubMed:20061329). Interacts with POGZ; the interaction is direct and is required for embryonic development (PubMed:22711834). Interacts with POU5F1 (PubMed:26496184). Interacts with SMARCC1/BAF155, SMARCC2/BAF170, and SMARCC3/BAF155B (PubMed:21846716). Interacts with INO80 (PubMed:31581170). Interacts with GBP1; the interaction reduces ATPase activity and chromatin remodeling by the SMARCA4-containing BAF complex (PubMed:32354956). Interacts with GID4; the interaction is required for PLK1-dependent stabilization and transcriptional activation of preloaded PRC1-E2F4 complex (PubMed:31554790). Interacts with RPS7, leading to regulate transcription (PubMed:32024787). Interacts with GTF2F1 in a ATP-dependent manner (PubMed:34351444). Interacts with WDR5 (PubMed:22184123). Interacts with DIXDC1 (PubMed:24344115). Interacts with AKAP8L (PubMed:30093433). Interacts with HNRNPA1BP1 and DDX21 (PubMed:33172939). Interacts with the SWI/SNF-B related complexes (PubMed:29602467). Interacts with SMARCA2/BRM (PubMed:33181303). Interacts with ACVR2A (PubMed:33707408). Interacts with KHSRP (PubMed:31869534). Interacts with INO80C (PubMed:33612356). Interacts with W|
| [CTNNAL1](https://www.ncbi.nlm.nih.gov/gene?term=CTNNAL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.0</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no clear functional associations between CTNNAL1 and RNMT based on the provided context. CTNNAL1 may modulate Rho pathway signaling while RNMT is involved in mRNA capping and methylation.|
