| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | EIF4EBP1 and EPN2 do not have any known direct functional associations. EIF4EBP1 is involved in the regulation of protein translation through interactions with EIF4E and signaling pathways, while EPN2 is involved in clathrin-coated invaginations and endocytosis. The shortest path between them in the BioGRID protein interaction network suggests they may interact indirectly through other proteins.|
| [CHERP](https://www.ncbi.nlm.nih.gov/gene?term=CHERP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: CHERP and GMEB2 may interact through the NR2C2 pathway and play a role in regulating calcium homeostasis, growth, and proliferation. Alternatively, GMEB2 may act as an auxiliary factor for parvovirus replication and may interact with CREB-binding protein (CBP), potentially impacting CHERP's interaction with the SARS-CoV nucleoprotein.|
| [SPR](https://www.ncbi.nlm.nih.gov/gene?term=SPR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSKU](https://www.ncbi.nlm.nih.gov/gene?term=TSKU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 4) | Possible functional association between SPR and TSKU is their involvement in regulating signaling pathways and developmental events. SPR catalyzes the final steps in tetra-hydrobiopterin biosynthesis while TSKU contributes to processes such as wound healing, cholesterol homeostasis, and is a Wnt signaling inhibitor. They may interact in pathways related to neural development, bone elongation, and cholesterol metabolism.|
| [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAF1](https://www.ncbi.nlm.nih.gov/gene?term=PAF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations: NMT1 adds a myristoyl group to the N-terminus of certain proteins, while PAF1 is a component of the PAF1 complex involved in transcription regulation and histone modifications. There is no clear biological process linking these two genes.|
| [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between LPGAT1 and VAT1 include their interaction through RNF4 and involvement in the regulation of cellular lipid metabolism and mitochondrial function, respectively. It is unclear whether there are direct biological processes linking the activities of these two genes.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NT5DC2](https://www.ncbi.nlm.nih.gov/gene?term=NT5DC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear functional association between EPN2 and NT5DC2, as they are involved in different cellular processes. EPN2 is involved in endocytosis and binds to several proteins, while NT5DC2 acts as a sensor of membrane damage and initiates autophagy via interaction with CALCOCO2/NDP52.|
| [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRSF4](https://www.ncbi.nlm.nih.gov/gene?term=SRSF4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between NMT1 and SRSF4 are not clear from the provided information, as they do not have any direct protein-protein interactions. NMT1 is involved in myristoylation of various cellular and viral proteins, while SRSF4 plays a role in alternative splice site selection during pre-mRNA splicing.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.5</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association between MTX2 and PMPCB is that they are both involved in mitochondrial protein processing. MTX2 is a component of the MICOS complex that helps organize mitochondrial cristae, while PMPCB is a catalytic subunit of the essential mitochondrial processing protease involved in cleaving off mitochondrial sequences from newly imported precursor proteins.|
| [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between PAFAH1B3 and TMEM97 based on the provided context and their known biological roles. PAFAH1B3 encodes an enzyme involved in modulating PAF activity and plays a role in brain development, while TMEM97 is an intracellular orphan receptor with potential diagnostic and therapeutic uses in cancer cells. The shortest path between them in the BioGRID protein interaction network involves RNF4 and TRIM67. However, the biological significance of this interaction is unclear.|
| [GNB5](https://www.ncbi.nlm.nih.gov/gene?term=GNB5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: LGMN may play a role in the regulation of cell proliferation via its role in EGFR degradation, while GNB5 may contribute to the deactivation of G protein signaling initiated by D(2) dopamine receptors. Based on their roles, it is possible that these genes may be involved in the regulation of cell growth and proliferation pathways. However, further experiments would be needed to confirm any specific functional associations between the two genes.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATG3](https://www.ncbi.nlm.nih.gov/gene?term=ATG3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.2</br>**STRING score:** 577</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | AKT1 is involved in regulating many processes including metabolism, proliferation, cell survival, growth, and angiogenesis, while ATG3 is a key enzyme required for autophagy and mitochondrial homeostasis. There is no direct functional association between these two genes, but they may be involved in regulating different aspects of cellular physiology.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPEN](https://www.ncbi.nlm.nih.gov/gene?term=SPEN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 247</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Based on their protein interaction network, EPN2 and SPEN may have a functional association in endocytosis and nuclear transcriptional regulation. EPN2 is involved in clathrin-coated invaginations and endocytosis, while SPEN may serve as a nuclear matrix platform that regulates transcriptional responses and represses differentiation.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLE1](https://www.ncbi.nlm.nih.gov/gene?term=TLE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.4</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | One possible functional association between NR1H2 and TLE1 is that NR1H2 may regulate TLE1's transcriptional activity through interaction with RUNX1, FOXA2, or possibly other transcription factors. Both genes are involved in regulating gene expression and have been implicated in liver metabolism and lipid homeostasis.|
| [DMTF1](https://www.ncbi.nlm.nih.gov/gene?term=DMTF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMNN](https://www.ncbi.nlm.nih.gov/gene?term=GMNN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.5</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 89 (min len 4) | Possible functional association: DMTF1 can interact with TP53, CDKN2A, and SMARCA4 to activate or inhibit transcription of downstream genes. GMNN inhibits DNA replication by preventing pre-replication complex formation. There is no known direct functional association between DMTF1 and GMNN, but their indirect association through TP53 and SMARCA4 in transcription regulation suggests a potential role in cell cycle control.|
| [DNTTIP2](https://www.ncbi.nlm.nih.gov/gene?term=DNTTIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between DNTTIP2 and PARP1 include their involvement in DNA repair processes through interaction with DNA and chromatin. DNTTIP2 also plays a role in ribosomal subunit assembly, while PARP1 promotes DNA damage-induced apoptosis when auto-poly-ADP-ribosylated.|
