| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EIF4EBP1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4EBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between EIF4EBP1 and EPN2 could involve regulation of endocytosis or clathrin-coated invaginations as EPN2 is involved in these processes and EIF4EBP1 has been shown to interact with RPTOR and the mTORC1 pathway, which regulates protein translation in response to growth factors and other stimuli. However, further research is needed to determine the specific nature of any functional association between these two genes.|
| [CHERP](https://www.ncbi.nlm.nih.gov/gene?term=CHERP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CHERP and GMEB2 are unclear based on their known biological processes and protein interactions. CHERP is involved in calcium homeostasis, growth and proliferation, while GMEB2 is a trans-acting factor that binds to glucocorticoid modulatory elements and is essential for parvovirus replication. The shortest path between these two genes in the BioGRID protein interaction network involves multiple intermediaries and does not provide clear biological insight into their potential interactions.|
| [SPR](https://www.ncbi.nlm.nih.gov/gene?term=SPR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSKU](https://www.ncbi.nlm.nih.gov/gene?term=TSKU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 4) | Possible functional associations between SPR and TSKU are unclear based on their known biological functions and the context provided. SPR catalyzes the final steps in tetrahydrobiopterin biosynthesis, while TSKU is involved in multiple developmental processes and signaling pathways. However, it is worth investigating if there are any indirect effects of SPR on TSKU through the pathways they are involved in.|
| [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAF1](https://www.ncbi.nlm.nih.gov/gene?term=PAF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between NMT1 and PAF1 include involvement in transcriptional regulation and mRNA processing. NMT1 has been shown to add myristoyl groups to certain proteins, while PAF1 is part of a complex involved in RNA polymerase II transcription and histone modifications. Their interaction in the BioGRID network suggests they may be co-regulated in some contexts.|
| [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VAT1](https://www.ncbi.nlm.nih.gov/gene?term=VAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations: LPGAT1 and VAT1 are connected in the BioGRID protein interaction network through the shortest path of several intermediary proteins. LPGAT1 plays a role in phospholipid metabolism while VAT1 functions in epidermal repair mechanisms and negatively regulates mitochondrial fusion. There is no known specific biological relationship between these two genes based on their functions.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NT5DC2](https://www.ncbi.nlm.nih.gov/gene?term=NT5DC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | It is not immediately clear what functional associations exist between EPN2 and NT5DC2 based on the provided information. However, EPN2 has been shown to play a role in endocytosis and binding to clathrin. NT5DC2 acts as a sensor for membrane damage and restricts pathogen proliferation by initiating autophagy.|
| [NMT1](https://www.ncbi.nlm.nih.gov/gene?term=NMT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRSF4](https://www.ncbi.nlm.nih.gov/gene?term=SRSF4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Both NMT1 and SRSF4 are involved in protein modification and splicing, respectively. There may be a functional association between the two genes through their common interactions with NXF1, FBXW7, MAPT, USP14, and BRD7 in the BioGRID protein interaction network, but further investigation is needed to determine the specific biological processes involved.|
| [MTX2](https://www.ncbi.nlm.nih.gov/gene?term=MTX2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.5</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There may be a functional association between MTX2 and PMPCB in regulating mitochondrial protein import and processing. MTX2 is part of the mitochondrial outer membrane sorting assembly machinery (SAM) complex, which interacts with PMPCB as the catalytic subunit of the essential mitochondrial processing protease (MPP) that cleaves the mitochondrial sequences off precursors proteins.|
| [PAFAH1B3](https://www.ncbi.nlm.nih.gov/gene?term=PAFAH1B3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | TMEM97 is involved in regulating cell survival and may play a role in cellular cholesterol homeostasis, while PAFAH1B3 is important for brain development and modulates the action of platelet-activating factor. The protein interaction network suggests the possibility of regulation through RNF4 and TRIM67, but the exact functional association between these two genes remains unclear.|
| [GNB5](https://www.ncbi.nlm.nih.gov/gene?term=GNB5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LGMN](https://www.ncbi.nlm.nih.gov/gene?term=LGMN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between GNB5 and LGMN is that GNB5 may play a role in the regulation of cell proliferation via its involvement in EGFR degradation, which is a process that may also involve LGMN, a lysosomal endopeptidase known to play a role in the processing of proteins for MHC class II antigen presentation.|
| [AKT1](https://www.ncbi.nlm.nih.gov/gene?term=AKT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ATG3](https://www.ncbi.nlm.nih.gov/gene?term=ATG3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.2</br>**STRING score:** 577</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | AKT1 and ATG3 are not known to have direct functional associations. AKT1 is a serine/threonine kinase involved in regulating processes such as cell survival, growth, and metabolism, while ATG3 is an E2 conjugating enzyme required for autophagy, mitochondrial homeostasis, and the cytoplasm to vacuole transport. The shortest path between these two genes in the BioGRID protein interaction network involves several intermediate proteins.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPEN](https://www.ncbi.nlm.nih.gov/gene?term=SPEN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 247</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between EPN2 and SPEN could be related to endocytosis and transcriptional regulation, as EPN2 is involved in clathrin-coated invaginations and endocytosis while SPEN plays a role in transcriptional responses and regulation of the Notch pathway. However, there is currently no direct evidence linking the two genes in a specific biological process.|
| [NR1H2](https://www.ncbi.nlm.nih.gov/gene?term=NR1H2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TLE1](https://www.ncbi.nlm.nih.gov/gene?term=TLE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.4</br>**STRING score:** 198</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | A possible functional association between NR1H2 and TLE1 is through their protein interaction network where NR1H2 activates SIRT1, which then interacts with TLE1. NR1H2 is a nuclear receptor involved in regulating cholesterol uptake and phospholipid remodeling in hepatocytes, while TLE1 is a transcriptional corepressor that inhibits NF-kappa-B-regulated gene expression and enhances FOXG1/BF-1-mediated transcriptional repression.|
| [DMTF1](https://www.ncbi.nlm.nih.gov/gene?term=DMTF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GMNN](https://www.ncbi.nlm.nih.gov/gene?term=GMNN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.5</br>**STRING score:** 173</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 89 (min len 4) | DMTF1 and GMNN are functionally associated through the TP53 pathway and cell cycle regulation. DMTF1 activates the CDKN2A/ARF locus, promoting p53-dependent growth arrest, while GMNN inhibits DNA replication and is degraded during mitosis. Their interaction suggests a potential role in regulating replication and cell cycle progression.|
| [DNTTIP2](https://www.ncbi.nlm.nih.gov/gene?term=DNTTIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.1</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear indication of a functional association between DNTTIP2 and PARP1. DNTTIP2 plays a role in regulating transcriptional activity and chromatin remodeling, while PARP1 is involved in DNA repair and apoptosis. However, both genes have been shown to interact with other proteins involved in DNA repair, suggesting they may play complementary roles in this process.|
