| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: LOXL1 and NOLC1 may be involved in neural crest specification and ribosomal processing/modification, as they interact with TCOF1 and RNA polymerase I.   LOXL1 functions as an enzyme involved in the modification of elastin and collagen substrates, while NOLC1 is a nucleolar protein that acts as a regulator of RNA polymerase I. These genes may have a role in the maintenance of the nucleolar structure or in the regulation of translation.|
| [CHIC2](https://www.ncbi.nlm.nih.gov/gene?term=CHIC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MACF1](https://www.ncbi.nlm.nih.gov/gene?term=MACF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no known direct functional associations between CHIC2 and MACF1. CHIC2 is involved in cell cycle regulation and phosphorylation of RB1, while MACF1 plays contrasting roles in cell survival and apoptosis, and is involved in tumor suppression and platelet functional responses. The closest interaction between the two is a protein interaction network shortest path from CHIC2 to MACF1 via MTUS2.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between ATF6 and PUM2 could involve the regulation of gene expression during cellular stress response. ATF6 is known to activate transcription of genes involved in the unfolded protein response during endoplasmic reticulum stress, while PUM2 acts as a post-transcriptional repressor of mRNA targets. The shortest path in the BioGRID protein interaction network between the two genes involves TRIM25, which may suggest a potential role in ubiquitin-mediated proteolysis.|
| [ADNP2](https://www.ncbi.nlm.nih.gov/gene?term=ADNP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | One possible functional association between ADNP2 and PUM2 is in transcriptional regulation and mRNA post-transcriptional repression. ADNP2 may be involved in transcriptional regulation and could interact with PUM2 in the post-transcriptional regulation of mRNA targets.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: EVL and PUM2 may be involved in regulating cytoskeleton remodeling, cell polarity, and cell migration, as EVL enhances actin nucleation and polymerization while PUM2 is a sequence-specific RNA-binding protein involved in post-transcriptional regulation of gene expression. The shortest path between them in the BioGRID protein interaction network suggests a possible interaction through TP53, which is a well-known tumor suppressor involved in regulating these cellular processes.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFIL3](https://www.ncbi.nlm.nih.gov/gene?term=NFIL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 4) | There is no clear functional association between LOXL1 and NFIL3 based on their known biological roles and the protein interaction network. LOXL1 is involved in elastin and collagen metabolism, while NFIL3 is a transcriptional regulator with various functions, including roles in circadian rhythm and natural killer cell development.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 178</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations: PUM2 is a post-transcriptional repressor that can bind to miRNAs and regulate gene expression, while SMAD3 is a transcriptional modulator activated by TGF-beta. It is possible that PUM2 can regulate SMAD3 expression or activity through its post-transcriptional repression mechanisms.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | BHLHE40 and TPM1 are connected in a protein interaction network via several short paths. BHLHE40 is a transcriptional repressor involved in circadian rhythm regulation, while TPM1 plays a central role in calcium-dependent muscle contraction and stabilizing actin filaments. It is unclear how these two genes could be functionally associated.|
| [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 646 (min len 4) | RBM6 and RGS2 are functionally associated through a protein interaction network involving common binding partner DDB1. RBM6 is involved in RNA binding, while RGS2 regulates G protein-coupled receptor signaling, blood pressure regulation, and vascular smooth muscle constriction. It is possible that RBM6 could regulate RGS2 expression or activity through its RNA binding function.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MACF1](https://www.ncbi.nlm.nih.gov/gene?term=MACF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: There is no direct evidence of a functional association between LOXL1 and MACF1 in TCGA-READ. However, both genes have roles in cancer, with LOXL1 potentially promoting tumor survival and MACF1 having contrasting roles in cell death and survival. The shortest path between these genes in the BioGRID protein interaction network involves an intermediate interaction with SOX2.|
| [LRP10](https://www.ncbi.nlm.nih.gov/gene?term=LRP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.5</br>**STRING score:** 258</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | LRP10 is involved in lipophilic molecule internalization and signal transduction, and interacts with NOLC1, which acts as a regulator of RNA polymerase I and is required for neural crest specification. It is possible that their interaction may play a role in the regulation and modification of RNA processing and translational programs during neural crest specification or nucleologenesis.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPTAN1](https://www.ncbi.nlm.nih.gov/gene?term=SPTAN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.3</br>**STRING score:** 641</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | There are no clear functional associations between EVL and SPTAN1, other than their physical interaction as identified in the BioGRID protein interaction network. EVL is involved in actin nucleation and polymerization, while SPTAN1 is important for cytoskeleton movement at the membrane and can form dimers and tetramers.|
| [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2J1](https://www.ncbi.nlm.nih.gov/gene?term=UBE2J1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.2</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations: It's unclear if FIS1 and UBE2J1 have direct functional associations. FIS1 is involved in mitochondrial and peroxisomal fission, while UBE2J1 is part of the HRD1 complex involved in ubiquitination.|
| [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.2</br>**STRING score:** 211</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between PAN2 and RBM6 are related to mRNA turnover and stability, as PAN2 is involved in deadenylation and degradation of mRNA, while RBM6 binds RNA and may play a role in RNA stability. The shortest path in the BioGRID protein interaction network between PAN2 and RBM6 suggests a possible indirect functional association through intermediate proteins involved in RNA processing, such as PLEC, EEF1G, PPP1CB, XRN2, and ELAVL1.|
| [GLTSCR1](https://www.ncbi.nlm.nih.gov/gene?term=GLTSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 961</br>**BioGRID count:** 5</br>**BioGRID shortest paths:** 1 (min len 2) | GLTSCR1 and SMARCA4 both participate in chromatin remodeling through different complexes. They are directly connected in the BioGRID protein interaction network and may interact in the SWI/SNF (GBAF) subcomplex. There is a possibility that they cooperate in regulating gene transcription, possibly during neural development or cancer progression.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 150</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 277 (min len 4) | Possible functional association: PUM2 may regulate the expression of POLG2 at a post-transcriptional level through binding to its mRNA 3'-UTR.  POLG2 is involved in regulating the polymerase and exonuclease activities during DNA synthesis, while PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor.|
| [ERBB2](https://www.ncbi.nlm.nih.gov/gene?term=ERBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations: ERBB2 is involved in transcriptional regulation and enhances protein synthesis and cell growth, while PUM2 is a sequence-specific RNA-binding protein that acts as a post-transcriptional repressor. There may be a regulatory relationship between the two genes through the shortest path protein interactions network, potentially affecting mitotic, DNA repair, and DNA replication factors.|
| [PPARG](https://www.ncbi.nlm.nih.gov/gene?term=PPARG[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 184</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | PPARG is involved in adipocyte differentiation and transcriptional regulation, and is activated by agonists. TRIB1 is an adapter protein involved in protein degradation, serving to alter COP1 substrate specificity. The two genes have a known protein interaction, suggesting a possible regulatory or signaling role for TRIB1 in PPARG-mediated processes.|
| [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: Both KAT6A and RBM6 have been shown to interact in a protein interaction network, and RBM6 has been shown to bind poly(G) RNA homopolymers. KAT6A is a histone acetyltransferase that may act as a transcriptional coactivator, and has been shown to interact with transcription factors such as RUNX1 and p53/TP53.   It is possible that the interaction between KAT6A and RBM6 may be related to RNA processing and/or transcriptional regulation, as mediated by KAT6A's histone acetyltransferase activity and RBM6's RNA binding ability. However, further investigation would be necessary to determine the specific nature of this potential functional association.|
| [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSD17B11](https://www.ncbi.nlm.nih.gov/gene?term=HSD17B11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 58 (min len 3) | There is no clear functional association between FIS1 and HSD17B11 based on the provided information.  FIS1 is involved in mitochondrial fission and peroxisomal fission, while HSD17B11 is involved in androgen metabolism during steroidogenesis and is a tumor-associated antigen in cutaneous T-cell lymphoma.|
