| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: LOXL1 is involved in substrate modification while NOLC1 is a nucleolar protein that regulates RNA polymerase I and plays a role in neural crest specification. It is possible that LOXL1 could modify substrates important for neural crest specification, thus potentially affecting the function of NOLC1. However, further investigation is needed to determine if there is a direct functional association between these two genes.|
| [CHIC2](https://www.ncbi.nlm.nih.gov/gene?term=CHIC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MACF1](https://www.ncbi.nlm.nih.gov/gene?term=MACF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: CHIC2 and MACF1 are both involved in the regulation of cell cycle and cytoskeletal organization. MACF1, as an actin-binding protein, plays a role in stabilizing microtubules and regulating focal adhesions, while CHIC2 is a regulatory component of the cyclin D1-CDK4 complex that phosphorylates members of the RB protein family and regulates G1/S transition. It's possible that these two genes work together to coordinate cell cycle control and cytoskeletal organization in specific contexts, such as during proliferation or migration.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | ATF6 is involved in endoplasmic reticulum stress response and activation of transcription of genes. PUM2 is a post-transcriptional repressor that binds to RNA targets and mediates mRNA degradation. The shortest path between ATF6 and PUM2 in the BioGRID protein interaction network is TRIM25, suggesting a possible functional association between these genes in regulating mRNA stability and degradation.|
| [ADNP2](https://www.ncbi.nlm.nih.gov/gene?term=ADNP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: Both ADNP2 and PUM2 are involved in transcriptional regulation and have been shown to interact in a protein interaction network, suggesting they may function together in post-transcriptional processes related to mRNA degradation and miRNA regulation. ADNP2 may also play a role in neuronal function and erythroid differentiation, while PUM2 is known to be involved in repressing genes necessary for maintaining genomic stability.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: EVL, a protein involved in actin dynamics and cell polarity, and PUM2, a post-transcriptional repressor, may have a role in regulating mRNA translation and degradation. The shortest path between them in the BioGRID protein interaction network involves the tumor suppressor TP53.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFIL3](https://www.ncbi.nlm.nih.gov/gene?term=NFIL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 4) | Possible functional association: NFIL3 may regulate LOXL1 expression or activity, as NFIL3 has been shown to regulate transcriptional activity and LOXL1 is known to interact with EFEMP2, which is involved in elastin production.          LOXL1 is involved in the cross-linking of extracellular matrix proteins, particularly elastin and collagen, while NFIL3 is a transcriptional regulator that binds to various promoters. It is possible that NFIL3 may regulate the transcription or activity of LOXL1 in specific cell types, potentially impacting extracellular matrix production and organization.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMAD3](https://www.ncbi.nlm.nih.gov/gene?term=SMAD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 178</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations between PUM2 and SMAD3 involve transcriptional regulation and signal transduction pathways in cancer. PUM2 is a post-transcriptional repressor of transcripts and may regulate DCUN1D3 mRNA levels, while SMAD3 is a transcriptional modulator activated by TGF-beta and activin type 1 receptor kinases. The shortest path between these genes in the BioGRID protein interaction network involves several intermediary proteins, suggesting their potential interaction in various signaling pathways.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between BHLHE40 and TPM1 include BHLHE40's regulation of chondrocyte differentiation via cAMP pathway and transcriptional repression of ATP1B1 in cardiovascular system, which may affect TPM1's involvement in stabilizing cytoskeleton actin filaments. TPM1 also plays a central role, in association with the troponin complex, in calcium dependent regulation of muscle contraction.|
| [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RGS2](https://www.ncbi.nlm.nih.gov/gene?term=RGS2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 646 (min len 4) | Possible functional associations between RBM6 and RGS2: RBM6 is involved in RNA binding and may interact with FAM168B. RGS2 is involved in regulating G protein-coupled receptor signaling, blood pressure, and vascular smooth muscle constriction/relaxation, and may interact with EIF2B5. There is no clear biological process connecting these two genes.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MACF1](https://www.ncbi.nlm.nih.gov/gene?term=MACF1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between LOXL1 and MACF1 involve their role in cell migration, specifically in regulating focal adhesion size and dynamics. LOXL1 interacts with EFEMP2 and MACF1 interacts with CAMSAP3, both of which play a role in coordinating microtubule dynamics and regulating actin filaments. Additionally, MACF1 is involved in stabilizing microtubules, which is essential for efficient upward migration of bulge cells in response to wound healing, a process that requires coordination with focal adhesions. Overall, these genes may work together to regulate the cytoskeleton during cell migration and wound healing processes.|
| [LRP10](https://www.ncbi.nlm.nih.gov/gene?term=LRP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOLC1](https://www.ncbi.nlm.nih.gov/gene?term=NOLC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.5</br>**STRING score:** 258</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | LRP10 and NOLC1 may have a functional association in neural crest specification through their interaction with TCOF1 and RNA polymerase I. LRP10 is involved in receptor-mediated internalization and interacts with multiple proteins involved in ciliogenesis, while NOLC1 regulates and interacts with RNA polymerase I to maintain ribosome processing in the nucleolus.|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPTAN1](https://www.ncbi.nlm.nih.gov/gene?term=SPTAN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.3</br>**STRING score:** 641</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: EVL and SPTAN1 may be involved in cytoskeleton remodeling and cell polarity, as they both interact with proteins associated with these processes. The shortest path between them in the BioGRID protein interaction network suggests a potential direct interaction or cooperation in this context.|
| [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2J1](https://www.ncbi.nlm.nih.gov/gene?term=UBE2J1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.2</br>**STRING score:** 185</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between FIS1 and UBE2J1 include their involvement in protein degradation pathways as part of the HRD1 complex, and FIS1's role in mitochondrial fission and apoptosis. However, there is no known direct interaction or functional association between these two genes based on the provided context.|
| [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.2</br>**STRING score:** 211</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between PAN2 and RBM6 include mRNA processing and regulation. PAN2 is involved in mRNA deadenylation and decay, while RBM6 is a poly(G) RNA binding protein. The BioGRID protein interaction network suggests they may interact through EEF1G, PPP1CB, XRN2, ELAVL1, and PLEC.|
| [GLTSCR1](https://www.ncbi.nlm.nih.gov/gene?term=GLTSCR1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 961</br>**BioGRID count:** 5</br>**BioGRID shortest paths:** 1 (min len 2) | GLTSCR1 and SMARCA4 are both components of the SWI/SNF chromatin remodeling complexes and interact with each other in the BioGRID protein interaction network. They both play a role in altering chromatin structure through ATP-dependent mechanisms and are involved in gene transcription regulation. Possible functional associations could be related to their roles in chromatin remodeling and gene regulation in cancer development and progression.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 150</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 277 (min len 4) | Possible functional associations between POLG2 and PUM2 could involve regulation of DNA replication and RNA post-transcriptional repression. The shortest path between POLG2 and PUM2 in the BioGRID protein interaction network involves the tumor suppressor TP53, which is involved in DNA damage response.|
| [ERBB2](https://www.ncbi.nlm.nih.gov/gene?term=ERBB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between ERBB2 and PUM2 include regulation of transcriptional activation and post-transcriptional repression. ERBB2 is involved in transcriptional regulation by activating PTGS2/COX-2 promoter and enhancing protein synthesis and cell growth, while PUM2 acts as a post-transcriptional repressor by binding to the 3'-UTR of mRNA targets, leading to translational inhibition and mRNA degradation. The BioGRID protein interaction network shows a shortest path between ERBB2 and PUM2 via intermediate proteins, suggesting potential functional associations between them through protein-protein interactions.|
| [PPARG](https://www.ncbi.nlm.nih.gov/gene?term=PPARG[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 184</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between PPARG and TRIB1 include regulation of macrophage, eosinophil and neutrophil differentiation via the COP1-binding domain, and control of myeloid cell differentiation by altering the expression of CEBPA in a COP1-dependent manner. PPARG is a nuclear receptor that interacts with coactivators and ligands to regulate transcription of target genes, while TRIB1 is an adapter protein involved in protein degradation and MAPK signaling. The shortest path between PPARG and TRIB1 in the BioGRID protein interaction network is PPARG-CEBPB-TRIB1.|
| [KAT6A](https://www.ncbi.nlm.nih.gov/gene?term=KAT6A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: Both KAT6A and RBM6 have been shown to interact with transcriptional regulators, such as RUNX1 and p53/TP53, suggesting a potential role in regulating gene expression.   KAT6A: a histone acetyltransferase that acetylates lysine residues in histones, and acts as a coactivator for transcription factors. It also interacts with PML, a protein involved in regulating cellular responses to DNA damage.   RBM6: specifically binds poly(G) RNA homopolymers in vitro, with potential roles in RNA processing and regulation. It may also interact with FAM168B, a protein involved in DNA repair.   Overall, there is currently no direct evidence of functional association between KAT6A and RBM6, but both genes appear to play roles in transcriptional regulation and could potentially intersect in this context.|
| [FIS1](https://www.ncbi.nlm.nih.gov/gene?term=FIS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSD17B11](https://www.ncbi.nlm.nih.gov/gene?term=HSD17B11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.1</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 58 (min len 3) | There are no known direct functional associations between FIS1 and HSD17B11. FIS1 plays a role in mitochondrial fission and peroxisomal fission, and is involved in apoptosis. HSD17B11 may participate in androgen metabolism during steroidogenesis and acts as a tumor-associated antigen in cutaneous T-cell lymphoma.|
