| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 7.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between BHLHE40 and PUM2:     These two genes have a protein interaction network and are connected by a shortest path of three proteins. BHLHE40 is involved in the regulation of the circadian rhythm and has been shown to negatively regulate the transcriptional activator CLOCK-BMAL1. PUM2 is a post-transcriptional repressor that binds to mRNA targets. Therefore, it is possible that BHLHE40 and PUM2 could interact to regulate the circadian rhythm at the post-transcriptional level.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 7.3</br>**STRING score:** 336</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association between CIRBP and PUM2 is their regulatory role in translation. CIRBP stabilizes transcripts of genes involved in cell survival, and PUM2 acts as a post-transcriptional repressor by binding to the 3'-UTR of mRNA targets. Additionally, PUM2 binds to miRNA MIR199A precursor and regulates miRNA MIR199A expression, which might impact the regulation of CIRBP.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 7.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between BHLHE40 and CIRBP include a role in the regulation of the circadian rhythm and response to genotoxic stress. They may interact in stress granule assembly and translational regulation, as CIRBP binds to stress-responsive transcripts and promotes SG formation, while BHLHE40 is a transcriptional repressor involved in the regulation of circadian rhythm and corepressor of RXR and the RXR-LXR heterodimers. The shortest path between these two proteins in the BioGRID protein interaction network includes several hub proteins involved in important biological processes, such as TP53BP1 and DDX58.|
| [BID](https://www.ncbi.nlm.nih.gov/gene?term=BID[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CAMSAP2](https://www.ncbi.nlm.nih.gov/gene?term=CAMSAP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BID induces apoptosis and interacts with ITCH and humanin, while CAMSAP2 is a microtubule-organizing protein that regulates microtubule dynamics and organization. There does not seem to be a direct functional association between BID and CAMSAP2 based on their known biological roles.|
| [CAMSAP2](https://www.ncbi.nlm.nih.gov/gene?term=CAMSAP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.8</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | As CAMSAP2 is involved in microtubule organization and LPGAT1 in fatty acid remodeling of glycerophospholipids, it is possible that these two genes may have a functional association in the regulation of cellular organelle organization, specifically in the Golgi. However, further experimentation would be needed to confirm any specific interactions or relationships between the two genes.|
| [MBNL1](https://www.ncbi.nlm.nih.gov/gene?term=MBNL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C2B](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C2B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between MBNL1 and PIK3C2B are not immediately clear. MBNL1 is involved in alternative splicing regulation, while PIK3C2B is involved in phosphorylation of lipids and may be involved in signaling cascades. Both genes have been identified in a protein interaction network with several other proteins but do not share any direct interaction partners.|
| [CAMSAP2](https://www.ncbi.nlm.nih.gov/gene?term=CAMSAP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CAMSAP2 and CIRBP include regulating cellular organelle organization, autophagosome transport, neuronal polarity, and stabilizing stress-responsive transcripts. However, there is no direct evidence of interaction between these two genes and their functions overlap in different processes of the cell.|
| [BID](https://www.ncbi.nlm.nih.gov/gene?term=BID[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear biological function linking BID and LPGAT1. BID is involved in apoptosis and interacts with ITCH and MTCH2, while LPGAT1 is involved in fatty acid chain remodeling in the endoplasmic reticulum and interacts with NHERF1 and several other proteins. The shortest path between the two genes in the BioGRID network involves several intermediate proteins and it is unclear how or if they are functionally related.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between BHLHE40 and DDX42 include involvement in RNA processing and splicing. BHLHE40 is a transcriptional repressor involved in regulating the circadian rhythm, while DDX42 is an ATP-dependent RNA helicase and a component of the splicing factor SF3B complex. The shortest path in the BioGRID protein interaction network between these two genes also includes interactions with several other proteins involved in RNA processing.|
| [ARHGEF12](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: Both genes are involved in regulation of gene expression. ARHGEF12 acts as a guanine nucleotide exchange factor and GTPase-activating protein, while BHLHE40 is a transcriptional repressor. There may be a regulatory relationship between the two genes, potentially through the circadian rhythm pathway as BHLHE40 is involved in negative regulation of clock genes and ARHGEF12 interacts with IGF1R beta subunit.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GPATCH8](https://www.ncbi.nlm.nih.gov/gene?term=GPATCH8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.1</br>**STRING score:** 281</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | There are no evident functional associations between CIRBP and GPATCH8 based on their known biological roles and the provided context. CIRBP plays a role in mRNA binding and stress granule formation, while GPATCH8 is involved in mitochondrial protein import and chaperoning. However, they do interact indirectly through several proteins in the BioGRID protein interaction network.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 435</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 49 (min len 3) | Possible functional associations between CIRBP and EPRS may be in stress response and translation regulation. CIRBP stabilizes transcripts of genes involved in cell survival and promotes assembly of stress granules, while EPRS is involved in aminoacyl-tRNA synthetase multienzyme complex, translation inhibition, and mTORC1 signaling pathway. It is possible that these two genes may interact in regulating translation during cellular stress responses.|
| [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 303</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association between PUM2 and SHC1 is that PUM2 may mediate post-transcriptional repression of SHC1 mRNA. PUM2 is a post-transcriptional repressor that binds to the 3'-UTR of mRNA targets, and its shortest path to SHC1 in the BioGRID protein interaction network supports this hypothesis. Additionally, both genes are involved in signal transduction pathways, with PUM2 being involved in cytoplasmic sensing of viral infection and SHC1 playing a role in signaling downstream of growth factor receptors and angiopoietin receptor TEK/TIE2.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUM2](https://www.ncbi.nlm.nih.gov/gene?term=PUM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 184</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between INTS3 and PUM2 could involve DNA repair and maintenance of genomic stability, as INTS3 is a component of the SOSS complex involved in these processes, while PUM2 represses a program of genes necessary for genomic stability. The shortest path in the BioGRID protein interaction network suggests the possibility of direct interaction between INTS3 and PUM2 through other proteins such as ELAVL1 or ESR2.|
| [DNAJB2](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 175</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between DNAJB2 and EPRS include involvement in protein quality control and regulation of translation. DNAJB2 acts as a co-chaperone, regulating substrate binding and proteasomal degradation of misfolded proteins. EPRS is part of the aminoacyl-tRNA synthetase multienzyme complex, and also plays a role in translation inhibition in response to interferon-gamma. It is possible that these genes may interact in the regulation of protein folding and translation.|
| [ARID4B](https://www.ncbi.nlm.nih.gov/gene?term=ARID4B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 203</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is no known functional association between ARID4B and CIRBP based on the provided context. ARID4B is involved in transcriptional regulation and may function as a leukemia suppressor, while CIRBP plays a protective role in the genotoxic stress response by stabilizing transcripts of genes involved in cell survival and promotes assembly of stress granules.|
| [CLIC4](https://www.ncbi.nlm.nih.gov/gene?term=CLIC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP1](https://www.ncbi.nlm.nih.gov/gene?term=PARP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no direct evidence of functional associations between CLIC4 and PARP1. CLIC4 plays a role in membrane insertion and ion transport, as well as angiogenesis and endothelial morphogenesis, while PARP1 is involved in DNA repair and apoptosis.|
| [ARID4B](https://www.ncbi.nlm.nih.gov/gene?term=ARID4B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BAG3](https://www.ncbi.nlm.nih.gov/gene?term=BAG3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 184</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | ARID4B and BAG3 are not directly functionally associated, but they both play roles in transcriptional regulation and protein chaperone activity, respectively. ARID4B acts as a transcriptional repressor and regulates the expression of genes involved in spermatogenesis, myeloid homeostasis, and leukemia suppression, while BAG3 acts as a co-chaperone for HSP70 and HSC70 chaperone proteins, promoting client protein release and playing a role in nucleocytoplasmic shuttling.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 239</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There are no known functional associations between CIRBP and ZNF131. CIRBP is involved in stress response and stabilization of transcripts, while ZNF131 is involved in transcriptional regulation and signaling.|
| [CIRBP](https://www.ncbi.nlm.nih.gov/gene?term=CIRBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DDX42](https://www.ncbi.nlm.nih.gov/gene?term=DDX42[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | CIRBP and DDX42 are functionally associated as they both play a role in RNA binding and processing: CIRBP stabilizes transcripts and promotes assembly of stress granules, while DDX42 is an ATP-dependent RNA helicase involved in splicing and RNA folding. The shortest path between them in the BioGRID interaction network involves multiple proteins, suggesting they may act in a complex or pathway together.|
| [BHLHE40](https://www.ncbi.nlm.nih.gov/gene?term=BHLHE40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.5</br>**STRING score:** 379</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: BHLHE40 is a transcriptional repressor involved in the regulation of circadian rhythm while FOXO3 is a transcriptional activator that regulates apoptosis and autophagy. These two genes may be functionally associated in regulating cellular processes that involve circadian rhythm and/or apoptosis/autophagy. The protein interaction network suggests that they may even physically interact.|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.5</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between EPRS and RBM6 include RNA processing/stability, as RBM6 binds poly(G) RNA homopolymers and EPRS is involved in translation inhibition in response to interferon-gamma. There is also a potential interaction between these two genes in the context of the GAIT complex, which includes EPRS, RPL13A, and GAPDH and is involved in the regulation of inflammatory mRNAs.|
