| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between COPB2 and NFKBIB in TCGA-GBM involve regulation of Golgi vesicular transport and inhibition of the NF-kappa-B signaling pathway respectively. There is no direct evidence of a biological interaction or functional relationship between these two genes in the context of TCGA-GBM.|
| [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 73.9</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between HMG20B and TXNDC9 in TCGA-GBM could involve regulation of the cell cycle and protein folding, which are both important processes in cancer development. HMG20B is required for correct progression through G2 phase of the cell cycle and entry into mitosis, while TXNDC9 negatively impacts protein folding. Additionally, they both interact with protein complexes containing other important cancer-related proteins such as HDAC1, BRCA2, and the chaperonin TCP1 complex.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKCH](https://www.ncbi.nlm.nih.gov/gene?term=PRKCH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 77 (min len 4) | Possible functional associations between ATP6V0B and PRKCH could include regulation of intracellular pH and cell differentiation, as well as potential involvement in tight junction assembly and maintenance. The shortest path in the BioGRID protein interaction network suggests that these genes could interact through several intermediate proteins, including MTCH2 and GLP1R.|
| [SUPV3L1](https://www.ncbi.nlm.nih.gov/gene?term=SUPV3L1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 70.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between SUPV3L1 and UBL5 may involve mitochondrial RNA metabolism and protein folding, as SUPV3L1 is a helicase player in mitochondrial RNA metabolism and a component of the mitochondrial degradosome complex, while UBL5 is a component of the chaperonin-containing T-complex and potentially interacts with CLK1, CLK3, and CLK4. There is no evidence of a direct interaction between the two proteins.|
| [NFATC3](https://www.ncbi.nlm.nih.gov/gene?term=NFATC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 70.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 309 (min len 4) | Possible functional associations between NFATC3 and UBL5 could involve the regulation of cytokine gene expression in T-cells, telomere maintenance, actin and tubulin folding, and protein folding through TRiC chaperone complex. As NFATC3 interacts with MYC and UBL5 interacts with CLK1, CLK3, and CLK4, it is possible that NFATC3 and UBL5 may have a synergistic effect on gene expression and protein folding.|
| [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.6</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | A possible functional association between NFKBIB and YME1L1 is that NFKBIB may regulate the expression or activity of YME1L1 in the mitochondrial intermembrane region, impacting mitochondrial morphology and function, complex I respiration activity, and antiapoptotic activity.|
| [SMC1A](https://www.ncbi.nlm.nih.gov/gene?term=SMC1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations: SMC1A is involved in chromosome cohesion during cell cycle and DNA repair, while TXNDC9 negatively impacts protein folding. There is no known direct functional association between these two genes in specific biological terms based on available information.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MICALL1](https://www.ncbi.nlm.nih.gov/gene?term=MICALL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association: MICALL1 may indirectly regulate endocytic recycling of endocytosed proteins by interacting with ECH1 which is involved in isomerization of 3-trans,5-cis-dienoyl-CoA to 2-trans,4-trans-dienoyl-CoA.|
| [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: TM9SF3 is involved in DNA methylation and imprinting, while TMEM5 is involved in the biosynthesis of a carbohydrate structure present in alpha-dystroglycan that is required for binding laminin G-like domain-containing extracellular proteins with high affinity. There is currently no known direct functional association between these two genes in specific biological terms.|
| [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE3B](https://www.ncbi.nlm.nih.gov/gene?term=UBE3B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TXNDC9 may affect the folding of actin or tubulin, which are downstream targets of UBE3B's E3 ubiquitin-protein ligase activity. Additionally, UBE3B is a component of the PRC2/EED-EZH2 complex, which is involved in histone methylation and transcriptional regulation, and may interact with TXNDC9 via MYC.|
| [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 338</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: UBL5 and YME1L1 are involved in protein folding and degradation, respectively, and could potentially interact in regulating mitochondrial function and protein metabolism. UBL5 is a component of the TRiC chaperonin complex and interacts with several kinases, while YME1L1 is an ATP-dependent metalloprotease that cleaves OPA1 and substrates in the mitochondrial intermembrane space.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.0</br>**STRING score:** 302</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between HIF1A and PIN1 could include regulation of cell cycle, DNA repair and oncogenesis. HIF1A is involved in oxygen sensing and regulation of glycolysis, while PIN1 is a PPIase enzyme that mediates protein conformational changes. PIN1 may regulate HIF1A stability or activity via post-translational modifications.|
| [GTF2E2](https://www.ncbi.nlm.nih.gov/gene?term=GTF2E2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.4</br>**STRING score:** 188</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | GTF2E2 and IKBKE are both involved in regulating gene expression, with GTF2E2 recruiting TFIIH to the initiation complex and stimulating RNA polymerase, and IKBKE playing a role in activating inflammatory responses. They have a possible functional association through their interaction with SND1 in the BioGRID protein interaction network.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | FRS2 and PIN1 may have a functional association in the regulation of cell growth and proliferation, as FRS2 is involved in downstream signaling pathways, including MAP kinase activation, while PIN1 regulates multiple cellular processes through conformational changes in phosphorylated proteins. However, more research is needed to determine the specific biological processes where these two genes may interact.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 334</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations: BIRC2 is known to regulate apoptosis and inflammation signaling, and also acts as an E3 ubiquitin-protein ligase. CDKN2A may be involved in the regulation of autophagy and caspase-independent cell death. There is no known direct interaction between the two genes, but their shortest path in the BioGRID protein interaction network involves intermediates involved in cell cycle regulation and control, such as RAF1 and E2F1. It is possible that BIRC2 and CDKN2A may have indirect functional associations through shared pathways involved in cell proliferation and survival.|
