| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [COPB2](https://www.ncbi.nlm.nih.gov/gene?term=COPB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations: The protein interaction network indicates that COPB2 and NFKBIB can interact through multiple pathways. COPB2 is involved in vesicular trafficking and interacts with Golgi membranes, while NFKBIB inhibits NF-kappa-B by trapping it in the cytoplasm. There may be a functional association between COPB2 and NFKBIB in regulating intracellular trafficking processes involving NF-kappa-B.|
| [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 73.9</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between HMG20B and TXNDC9 could involve neuronal gene repression, cell cycle progression, protein folding, and chaperonin complex formation. The shortest path between these two genes in the BioGRID protein interaction network suggests direct protein-protein interactions, potentially influencing one another's functions in some contexts.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKCH](https://www.ncbi.nlm.nih.gov/gene?term=PRKCH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 77 (min len 4) | Possible functional association: Both genes are involved in intracellular processes - ATP6V0B in proton transport and maintenance of pH in intracellular compartments, while PRKCH plays a role in regulation of protein kinase activity. The shortest path between the two genes in the BioGRID protein interaction network suggests possible indirect association through intermediary proteins involved in various intracellular processes.|
| [SUPV3L1](https://www.ncbi.nlm.nih.gov/gene?term=SUPV3L1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 70.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: Both SUPV3L1 and UBL5 are involved in protein folding and regulation. SUPV3L1 plays a role in RNA surveillance and mitochondrial RNA metabolism, while UBL5 is a component of the chaperonin-containing T-complex that assists in protein folding. There is no known specific biological relationship between these two genes.|
| [NFATC3](https://www.ncbi.nlm.nih.gov/gene?term=NFATC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 70.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 309 (min len 4) | NFATC3 is involved in transcriptional activation and cytokine gene expression in T-cells, while UBL5 is a component of the chaperonin-containing T-complex that assists protein folding. There is a possible functional association between these two genes through their interaction with MYC in the BioGRID protein interaction network, which suggests a role in regulating gene expression and protein folding in cancer cells.|
| [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.6</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no apparent functional association between NFKBIB and YME1L1. NFKBIB inhibits NF-kappa-B and interacts with THRB, RELA and REL. YME1L1 is a metalloprotease that regulates mitochondrial morphology and function and interacts with AFG1L.|
| [SMC1A](https://www.ncbi.nlm.nih.gov/gene?term=SMC1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional associations between SMC1A and TXNDC9 involve their roles in protein folding and DNA repair. SMC1A is involved in DNA repair and spindle assembly during mitosis, while TXNDC9 negatively impacts protein folding. It is possible that SMC1A and TXNDC9 interact within the cohesin complex and chaperonin TCP1 complex, which both involve protein folding and may play a role in DNA repair.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MICALL1](https://www.ncbi.nlm.nih.gov/gene?term=MICALL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations: ECH1 and MICALL1 interact with several common proteins involved in endocytic recycling and membrane trafficking suggesting a potential role in these processes.      ECH1 is involved in the isomerization of 3-trans,5-cis-dienoyl-CoA to 2-trans,4-trans-dienoyl-CoA. MICALL1 may play a role in receptor-mediated endocytosis and slow endocytic recycling of endocytosed proteins back to the plasma membrane, potentially through interactions with RAB proteins, EHD1, and ACAP2.|
| [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: TM9SF3 is involved in DNA methylation and transcriptional corepression, while TMEM5 participates in the biosynthesis of the O-mannosyl trisaccharide required for binding laminin G-like domain-containing extracellular proteins with high affinity. There are no direct biological processes linking these two genes.|
| [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE3B](https://www.ncbi.nlm.nih.gov/gene?term=UBE3B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between TXNDC9 and UBE3B in TCGA-GBM are not immediately clear from their individual functions and interactions. TXNDC9 negatively impacts protein folding, while UBE3B is involved in ubiquitin-mediated protein degradation and forms part of the PRC2 complex implicated in transcriptional regulation. Further investigation is necessary to determine any specific interactions or pathways that may link these two genes in the GBM cancer context.|
| [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.0</br>**STRING score:** 338</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association between UBL5 and YME1L1 could be related to mitochondrial function. UBL5 is involved in assisting protein folding in the chaperonin-containing T-complex while YME1L1 is responsible for regulating mitochondrial morphology and function by cleaving OPA1. The protein interaction network suggests that these two genes may be connected through a pathway involving factors such as PLEKHA4, CCNF, KRAS, MYC, and ZRANB1.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.0</br>**STRING score:** 302</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HIF1A and PIN1 are both involved in regulating cell responses to various stress factors such as hypoxia and inflammation. PIN1 mediates conformational changes in phosphorylated proteins, while HIF1A acts as a transcription factor that responds to hypoxic stress. They may interact in signaling pathways regulating inflammation and metabolic adaptation to stress.|
| [GTF2E2](https://www.ncbi.nlm.nih.gov/gene?term=GTF2E2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKE](https://www.ncbi.nlm.nih.gov/gene?term=IKBKE[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.4</br>**STRING score:** 188</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between GTF2E2 and IKBKE in TCGA-GBM could be related to the regulation of inflammatory responses to viral infection and transcriptional activation of pro-inflammatory and antiviral genes through NF-kB and STAT signaling pathways, as well as energy balance regulation through sustaining chronic low-grade inflammation in obesity. However, there is no direct evidence of interaction between these two genes in the provided context.|
| [FRS2](https://www.ncbi.nlm.nih.gov/gene?term=FRS2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: FRS2 and PIN1 may interact in the regulation of signaling pathways and cellular processes that involve phosphorylation and isomerization of specific motifs, possibly contributing to the development and progression of cancer. However, further research is needed to establish the exact mechanisms and functional implications of this association.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 56.5</br>**STRING score:** 334</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | BIRC2 and CDKN2A have a shortest path interaction via RAF1, UBE2S, UBC, BIRC3, and E2F1. BIRC2 is a multi-functional protein that regulates apoptosis, inflammatory signaling, and immunity, while CDKN2A acts as a negative regulator of cell proliferation by inhibiting the phosphorylation of retinoblastoma protein. There are no known functional associations between these two genes.|
