| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CYB561](https://www.ncbi.nlm.nih.gov/gene?term=CYB561[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMCO1](https://www.ncbi.nlm.nih.gov/gene?term=TMCO1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: CYB561 and TMCO1 may be involved in calcium homeostasis and trafficking, as well as in the proper folding and membrane insertion of transmembrane proteins. They might also interact with MAPK1/ERK2, suggesting a role in signaling pathways.|
| [H2AFV](https://www.ncbi.nlm.nih.gov/gene?term=H2AFV[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between H2AFV and JMJD6:     H2AFV is a variant histone involved in chromatin regulation and chromosome segregation, while JMJD6 is a lysyl-hydroxylase and arginine demethylase involved in RNA splicing and transcription regulation. They both interact with other proteins in the BioGRID network. There is no obvious direct functional association between these two genes.|
| [ADNP2](https://www.ncbi.nlm.nih.gov/gene?term=ADNP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between ADNP2 and JMJD6 include involvement in transcriptional regulation and neuronal function. ADNP2 may also play a role in erythroid differentiation and interact with SMARCA4/BRG1, while JMJD6 acts as an arginine demethylase and lysyl-hydroxylase and regulates RNA splicing. No direct functional associations between the two genes are known.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL4R](https://www.ncbi.nlm.nih.gov/gene?term=IL4R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between BIRC2 and IL4R are unclear based on the provided context. BIRC2 regulates caspases and apoptosis, and modulates immune signaling and cell proliferation, while IL4R is a receptor involved in IL4-mediated cell proliferation and immune response. They interact indirectly through a protein interaction network.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ARHGAP1 and HMG20B could be their involvement in cell cycle progression and gene expression regulation as indicated by their respective interactions with RCOR1/CoREST and BRCA2. ARHGAP1 is a GTPase activator for Rho, Rac, and Cdc42 proteins, while HMG20B is required for correct progression through G2 phase of the cell cycle and entry into mitosis.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Functional association: JMJD6 and NCOA3 both have roles in regulating gene transcription, with JMJD6 acting as a histone demethylase and RNA hydroxylase, and NCOA3 as a nuclear receptor coactivator and histone acetyltransferase. The shortest path between these two genes in the BioGRID protein interaction network involves BRD4, suggesting that they may interact with BRD4 to regulate transcription.|
| [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between HPRT1 and NET1 may involve the regulation of purine nucleotide generation and RhoA GTPase activation pathways, as suggested by their known biological functions and interactions in the literature. HPRT1 is involved in the generation of purine nucleotides through the salvage pathway, while NET1 functions as a guanine nucleotide exchange factor for RhoA GTPase and may be involved in activation of the SAPK/JNK pathway.|
| [SLC25A22](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A22[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIAM1](https://www.ncbi.nlm.nih.gov/gene?term=TIAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between SLC25A22 and TIAM1 could involve regulation of glucose-stimulated insulin secretion and cell migration. SLC25A22 is a mitochondrial glutamate/H(+) symporter while TIAM1 is a guanyl-nucleotide exchange factor that activates RHO-like proteins and connects extracellular signals to cytoskeletal activities. These genes may interact in processes such as insulin secretion and cell migration.|
| [TBC1D9B](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D9B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2L6](https://www.ncbi.nlm.nih.gov/gene?term=UBE2L6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between TBC1D9B and UBE2L6 are related to their roles in the regulation of protein ubiquitination and subsequent degradation via the proteasome. TBC1D9B may act as a GTPase-activating protein, while UBE2L6 catalyzes the covalent attachment of ubiquitin or ISG15 to other proteins, including p53/TP53 and FLT3.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF395](https://www.ncbi.nlm.nih.gov/gene?term=ZNF395[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 4) | Possible functional associations between ATMIN and ZNF395 could be in the regulation of transcription and DNA damage response, based on the roles of ATMIN in cell survival and RAD51 foci formation, and ZNF395 in papillomavirus genes transcription and interaction with repression-mediating E2 binding site P2 of HPV8. However, further experimental validation is necessary to confirm any direct functional relationship between these two genes.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.9</br>**STRING score:** 420</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between JMJD6 and KDM5B is their involvement in regulation of histone modification. JMJD6 acts as an arginine demethylase and a lysyl-hydroxylase, while KDM5B is a histone demethylase that demethylates 'Lys-4' of histone H3. Both genes are involved in histone code regulation and transcriptional activity, which suggests a possible interaction between them in gene expression regulation.|
| [OGFOD1](https://www.ncbi.nlm.nih.gov/gene?term=OGFOD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [P4HTM](https://www.ncbi.nlm.nih.gov/gene?term=P4HTM[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.8</br>**STRING score:** 683</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: P4HTM hydroxylates HIF1A which is involved in the regulation of gene expression in response to low oxygen levels. OGFOD1 is involved in stress granule formation which plays a role in cellular response to stress. Both genes may play a role in regulating cellular response to hypoxia.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIT1](https://www.ncbi.nlm.nih.gov/gene?term=NIT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.6</br>**STRING score:** 267</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between ECH1 and NIT1 is that they may interact to regulate cancer cell growth and apoptosis, as loss of NIT1 expression promotes cell growth and resistance to DNA damage stress while ECH1 is involved in isomerization of 3-trans,5-cis-dienoyl-CoA to 2-trans,4-trans-dienoyl-CoA which is important for fatty acid metabolism, a process that is often altered in cancer.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.5</br>**STRING score:** 565</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CRKL and JMJD6 may involve regulation of RNA splicing and transcriptional activation through interactions with U2AF2/U2AF65 and the P-TEFb complex, respectively. CRKL may also have a role in signal transduction and interactions with tyrosine-phosphorylated receptors and downstream effectors, while JMJD6 can act as an arginine demethylase and lysyl-hydroxylase involved in histone modification and differentiation during embryogenesis and angiogenic sprouting.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 226</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CREB1 and JMJD6 may involve the regulation of gene transcription and RNA splicing. CREB1 is a transcription factor that binds to the cAMP response element to stimulate transcription, while JMJD6 can regulate RNA splicing by mediating hydroxylation of U2AF2/U2AF65. The shortest path in the BioGRID interaction network suggests potential interactions between the two genes.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB8](https://www.ncbi.nlm.nih.gov/gene?term=PSMB8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.3</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | One possible functional association between CCL2 and PSMB8 is their involvement in the inflammatory response pathway. CCL2 acts as a chemotactic factor for monocytes, while PSMB8 is involved in antigen processing and generation of spliced peptides, both of which are important in the immune response. Additionally, PSMB8 has been shown to be involved in apoptosis and may play a role in cancer.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.2</br>**STRING score:** 390</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 49 (min len 3) | Possible functional associations: CRKL and CTTN are both involved in actin cytoskeletal rearrangement, cell migration, and receptor-mediated endocytosis. They also form a complex with NEDD9, which is required for their formation.   These genes may work together in regulating the formation of neuronal growth cones and in the invasiveness of cancer cells, particularly in the formation of metastases. They may also contribute to the stabilization of potassium channels at the cell membrane.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKB2](https://www.ncbi.nlm.nih.gov/gene?term=NFKB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.9</br>**STRING score:** 182</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are potential functional associations between ATF6 and NFKB2 in immune response and endoplasmic reticulum stress. ATF6 is involved in the unfolded protein response, while NFKB2 is a transcription factor involved in inflammation, immunity, and apoptosis. The shortest path between the two genes in the BioGRID protein interaction network involves GTF2I, which is involved in the regulation of inflammatory pathways and immune response.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.6</br>**STRING score:** 469</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of functional association between HIF1A and SHC1, but HIF1A is involved in hypoxia response and immune response to viral infection, while SHC1 is a signaling adapter that plays a role in cell growth and oxidative stress response. HIF1A may promote pro-inflammatory state in monocytes upon SARS-CoV-2 infection and SHC1 interacts with herpes simplex virus 1 UL46.|
