| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CYB561](https://www.ncbi.nlm.nih.gov/gene?term=CYB561[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMCO1](https://www.ncbi.nlm.nih.gov/gene?term=TMCO1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: CYB561 and TMCO1 may be involved in calcium homeostasis and regulation, as well as in the multi-pass translocon complex mediating the insertion of membrane proteins into lipid bilayers. There is no known direct interaction between these two genes, but they may have overlapping roles in maintaining proper cellular function.|
| [H2AFV](https://www.ncbi.nlm.nih.gov/gene?term=H2AFV[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between H2AFV and JMJD6 may include their role in chromatin remodeling and transcription regulation. H2AFV is a variant histone involved in the formation of heterochromatin while JMJD6 acts as an arginine demethylase and lysyl-hydroxylase that regulates RNA splicing and transcription elongation through interactions with P-TEFb complex. The shortest path between them in the BioGRID protein interaction network suggests that they may interact indirectly through other proteins.|
| [ADNP2](https://www.ncbi.nlm.nih.gov/gene?term=ADNP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: JMJD6 can regulate RNA splicing by 5-hydroxylating U2AF2/U2AF65, while ADNP2 is involved in transcriptional regulation and may interact with SMARCA4/BRG1. These could suggest a potential role for ADNP2 in the regulation of JMJD6's activity, or vice versa. However, further research would be needed to confirm any direct functional associations between these two genes.|
| [BIRC2](https://www.ncbi.nlm.nih.gov/gene?term=BIRC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL4R](https://www.ncbi.nlm.nih.gov/gene?term=IL4R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no direct evidence of a functional association between BIRC2 and IL4R. BIRC2 functions as an E3 ubiquitin-protein ligase regulating NF-kappa-B signaling and the modulation of the cell cycle, while IL4R forms functional receptors that respond to IL4 and IL13 and play essential roles in immune modulation. However, both are part of a shortest path in the BioGRID protein interaction network that includes CD40, suggesting a possible indirect association through this protein.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMG20B](https://www.ncbi.nlm.nih.gov/gene?term=HMG20B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ARHGAP1 and HMG20B might include involvement in cell cycle regulation and gene expression, as HMG20B is required for correct progression through G2 phase and is a component of a histone deacetylase complex, while ARHGAP1 is a GTPase activator and has been found in a complex with other proteins involved in mRNA translation and splicing. However, further research would be needed to determine any specific interactions between these genes and their contribution to cancer development in TCGA-GBM.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NCOA3](https://www.ncbi.nlm.nih.gov/gene?term=NCOA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between JMJD6 and NCOA3 is that they both play a role in the regulation of transcription. JMJD6 acts as a lysyl-hydroxylase and arginine demethylase, and interacts with BRD4 to regulate pause release of polymerase II and transcriptional activation of certain genes. NCOA3 is a nuclear receptor coactivator that directly binds to nuclear receptors and stimulates transcriptional activities in a hormone-dependent manner.|
| [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NET1](https://www.ncbi.nlm.nih.gov/gene?term=NET1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between HPRT1 and NET1 based on the provided context and their known biology. HPRT1 plays a central role in the generation of purine nucleotides through the purine salvage pathway, while NET1 lacks sodium- and chloride-dependent glycine transporter activity and is involved in DNA binding complexes.|
| [SLC25A22](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A22[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIAM1](https://www.ncbi.nlm.nih.gov/gene?term=TIAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is currently no evident functional association between SLC25A22 and TIAM1. SLC25A22 is a mitochondrial transporter for glutamate, while TIAM1 is a guanyl-nucleotide exchange factor that activates RHO-like proteins.|
| [TBC1D9B](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D9B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2L6](https://www.ncbi.nlm.nih.gov/gene?term=UBE2L6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TBC1D9B and UBE2L6 may be involved in the regulation of protein degradation and ubiquitination pathways, as they both interact with the proteasome and catalyze the attachment of ubiquitin to other proteins. The shortest path between them in the BioGRID network includes PARK2, which is also involved in protein degradation.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF395](https://www.ncbi.nlm.nih.gov/gene?term=ZNF395[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 25 (min len 4) | There is no known direct functional association between ATMIN and ZNF395 in specific biological terms. ATMIN plays a role in cell survival and DNA damage response through ATM activation, while ZNF395 is involved in papillomavirus gene transcription through binding to HPV8.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 68.9</br>**STRING score:** 420</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | JMJD6 and KDM5B may have functional associations in histone modification and transcriptional regulation. JMJD6 can act as both an arginine demethylase and a lysyl-hydroxylase, while KDM5B is a histone demethylase that specifically targets H3K4me3. They may cooperate in regulating gene expression and promoting cancer cell proliferation.|
| [OGFOD1](https://www.ncbi.nlm.nih.gov/gene?term=OGFOD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [P4HTM](https://www.ncbi.nlm.nih.gov/gene?term=P4HTM[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.8</br>**STRING score:** 683</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is a possible functional association between OGFOD1 and P4HTM via TRIM67, which is involved in protein turnover. OGFOD1 is involved in stress granule formation, while P4HTM catalyzes the formation of 4-hydroxyproline in HIF alpha proteins, targeting them for proteasomal degradation.|
| [ECH1](https://www.ncbi.nlm.nih.gov/gene?term=ECH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIT1](https://www.ncbi.nlm.nih.gov/gene?term=NIT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.6</br>**STRING score:** 267</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between ECH1 and NIT1 is unclear based on the provided context and interaction network. ECH1 is involved in isomerization of fatty acid membrane lipids, while NIT1 is involved in the repair of deaminated glutathione. Both have been implicated in tumor suppression and may have roles in cell growth and apoptosis.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.5</br>**STRING score:** 565</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations:  CRKL is involved in signal transduction, while JMJD6 is a lysyl-hydroxylase and arginine demethylase. They are connected in the BioGRID protein interaction network via several intermediate proteins. It is not clear what biological processes might connect these genes, if any.|
| [CREB1](https://www.ncbi.nlm.nih.gov/gene?term=CREB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.3</br>**STRING score:** 226</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CREB1 and JMJD6 include roles in gene transcription and RNA splicing. CREB1 is a phosphorylation-dependent transcription factor that stimulates transcription, while JMJD6 is involved in regulating RNA splicing and the pre-mRNA splicing activity of U2AF2/U2AF65. There is no direct evidence of a functional association between the two genes, but they both play important roles in regulating gene expression.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB8](https://www.ncbi.nlm.nih.gov/gene?term=PSMB8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.3</br>**STRING score:** 242</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between CCL2 and PSMB8 include antigen processing and inflammation pathways. CCL2 is involved in chemotaxis for monocytes and has a potential role in the recruitment of monocytes into the arterial wall during the disease process of atherosclerosis. PSMB8 plays a key role in antigen processing to generate class I binding peptides and may be involved in the inflammatory response pathway. The shortest path between the two genes in the BioGRID protein interaction network involves DDX58, which is involved in the innate immune response to viral infection.|
| [CRKL](https://www.ncbi.nlm.nih.gov/gene?term=CRKL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 61.2</br>**STRING score:** 390</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 49 (min len 3) | Possible functional association: Both CRKL and CTTN are involved in the regulation of cell shape, migration, and adhesion through their interactions with other proteins in the signaling pathways. They may be involved in cancer cell invasiveness and metastasis. The shortest path between CRKL and CTTN in the BioGRID protein interaction network suggests that they may interact with each other through intermediate proteins, such as NEDD9 and ABL1, which are also involved in cell adhesion and signaling.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKB2](https://www.ncbi.nlm.nih.gov/gene?term=NFKB2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.9</br>**STRING score:** 182</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: Both ATF6 and NFKB2 are transcription factors involved in regulating cellular responses to stress and inflammation. They may have a functional association in mediating the cellular response to endoplasmic reticulum stress and inflammation.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.6</br>**STRING score:** 469</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no apparent functional association between HIF1A and SHC1 based on their known biological roles and protein interactions. HIF1A is a transcription factor involved in regulating responses to hypoxia, while SHC1 is a signaling adapter involved in propagation of mitogenic signals. However, they may both be involved in cellular responses to microbial infection based on the provided context.|
