| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MBOAT7](https://www.ncbi.nlm.nih.gov/gene?term=MBOAT7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCRN1](https://www.ncbi.nlm.nih.gov/gene?term=SCRN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between MBOAT7 and SCRN1 could be related to membrane remodeling and repair processes, as MBOAT7 is involved in phospholipid remodeling pathways and SCRN1 plays a role in base excision repair complex. However, without further information or experiments, the specific biological interaction between these two genes is unclear.|
| [H2AFV](https://www.ncbi.nlm.nih.gov/gene?term=H2AFV[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSD17B10](https://www.ncbi.nlm.nih.gov/gene?term=HSD17B10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.0</br>**STRING score:** 228</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | Possible functional associations between H2AFV and HSD17B10 could involve regulation of chromatin structure and tRNA processing in mitochondria, respectively. However, based on the provided information, there is no clear biological process that directly links these two genes.|
| [GPC1](https://www.ncbi.nlm.nih.gov/gene?term=GPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is no clear biological process linking GPC1 and TM9SF2, but they are indirectly connected through several intermediate proteins in the BioGRID protein interaction network. GPC1 is involved in cell surface proteoglycan function, while TM9SF2 functions as a channel or transporter in intracellular compartments.|
| [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between STAT3 and STK10 include regulation of lymphocyte migration and cell cycle regulation. STAT3 is a signal transducer and transcription activator involved in cellular responses to growth factors and has been implicated in regulation of inflammatory response. STK10 is a serine/threonine protein kinase involved in regulation of lymphocyte migration and cell cycle regulation and may act as a polo kinase kinase. The shortest path between these two genes in the BioGRID protein interaction network suggests they may interact through RAC1, LYN, SHMT2, MOV10, and/or NXF1.|
| [CHMP6](https://www.ncbi.nlm.nih.gov/gene?term=CHMP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBX2](https://www.ncbi.nlm.nih.gov/gene?term=TBX2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 87.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 55 (min len 4) | There are currently no known functional associations between CHMP6 and TBX2 in specific biological terms. CHMP6 is involved in multivesicular bodies formation and sorting of endosomal cargo proteins, while TBX2 is a transcription factor involved in various biological processes such as embryonic development, melanocyte proliferation, and cellular senescence.|
| [RPS6](https://www.ncbi.nlm.nih.gov/gene?term=RPS6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional association between RPS6 and STK10 could be in regulating cell growth and proliferation. RPS6 is a component of the ribosomal subunit and regulates translation of specific classes of mRNA, while STK10 is involved in phosphorylation of ERM proteins in regulation of lymphocyte migration and acts as a cell cycle regulator. How they may directly interact still needs further investigation.|
| [ARID5B](https://www.ncbi.nlm.nih.gov/gene?term=ARID5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STK10](https://www.ncbi.nlm.nih.gov/gene?term=STK10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 85.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: ARID5B and STK10 may be involved in regulation of lymphocyte migration and cell cycle regulation, respectively.      Reasoning: STK10 is a serine/threonine-protein kinase involved in regulation of lymphocyte migration, while ARID5B has been shown to induce smooth muscle marker genes, suggesting that it may also act as a regulator of smooth muscle cell differentiation and proliferation. Additionally, STK10 has been shown to act as a polo kinase kinase, while ARID5B is a transcription coactivator that regulates gene expression. These different functions suggest that these two genes may have distinct roles but could exert influence on each other in specific biological processes.|
| [BCL7B](https://www.ncbi.nlm.nih.gov/gene?term=BCL7B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CALU](https://www.ncbi.nlm.nih.gov/gene?term=CALU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 84.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no clear functional association between BCL7B and CALU based on their individual context and known interactions. BCL7B is involved in apoptosis regulation and Wnt signaling pathway, while CALU plays a role in vitamin K-dependent carboxylation. Their shortest path in the BioGRID protein interaction network involves other proteins, indicating potential indirect interactions.|
| [GRB10](https://www.ncbi.nlm.nih.gov/gene?term=GRB10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HPRT1](https://www.ncbi.nlm.nih.gov/gene?term=HPRT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between GRB10 and HPRT1 are currently unknown. GRB10 is an adapter protein that modulates the signaling pathways of various cell surface receptors and can negatively regulate Wnt signaling. HPRT1 plays a central role in generating purine nucleotides through the salvage pathway.|
| [COG4](https://www.ncbi.nlm.nih.gov/gene?term=COG4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIM2](https://www.ncbi.nlm.nih.gov/gene?term=TRIM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between COG4 and TRIM2 based on their known roles. COG4 is involved in Golgi function and retrograde transport, while TRIM2 is a ubiquitin ligase with a neuroprotective function and a role in antiviral immunity.|
| [TES](https://www.ncbi.nlm.nih.gov/gene?term=TES[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.9</br>**STRING score:** 424</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear evidence of a direct functional association between TES and TPM1. TES is involved in cell adhesion, cell spreading, and actin cytoskeleton organization, while TPM1 plays a role in regulating muscle contraction and stabilizing actin filaments.|
| [GPC1](https://www.ncbi.nlm.nih.gov/gene?term=GPC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHOA](https://www.ncbi.nlm.nih.gov/gene?term=RHOA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 220</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 3) | Possible functional associations between GPC1 and RHOA could involve regulation of cell signaling pathways related to cell differentiation, myelination and migration, as well as modulation of virus-induced syncytium formation. GPC1 is a transmembrane proteoglycan involved in myelination and differentiation, while RHOA is a small GTPase that regulates cell migration and cytoskeletal rearrangements.|
| [BECN1](https://www.ncbi.nlm.nih.gov/gene?term=BECN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLEKHM1](https://www.ncbi.nlm.nih.gov/gene?term=PLEKHM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.1</br>**STRING score:** 513</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between BECN1 and PLEKHM1 is their involvement in autophagy and pathogen response. BECN1 (Beclin-1) promotes autophagy and interacts with viral proteins to antagonize pathogen response, while PLEKHM1 contributes to pathogen pathogenesis and interacts with bacterial proteins. The shortest path in the BioGRID protein interaction network between BECN1 and PLEKHM1 involves proteins involved in autophagy and protein synthesis, indicating a potential regulation of these processes by BECN1 and PLEKHM1.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.6</br>**STRING score:** 233</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between CASP3 and JMJD6 in TCGA-GBM include their involvement in regulating apoptosis and RNA splicing, respectively. JMJD6 may induce programmed cell death by regulating the pre-mRNA splicing activity of U2AF2/U2AF65, which is also a target for 5-hydroxylation by the same protein. Given that CASP3 functions as one of the major effector caspases involved in cellular apoptosis and JMJD6 is known to regulate related splicing processes, it is possible that they may interact within the same pathway leading to apoptosis in TCGA-GBM.|
| [RAI14](https://www.ncbi.nlm.nih.gov/gene?term=RAI14[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIAM1](https://www.ncbi.nlm.nih.gov/gene?term=TIAM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.4</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | Possible functional associations between RAI14 and TIAM1 include their involvement in cell adhesion and cytoskeletal regulation. RAI14 plays a role in regulating actin at the ectoplasmic specialization in testicular cells, while TIAM1 activates RHO-like proteins to regulate cell adhesion and migration. The protein interaction network suggests they may interact directly with each other and other proteins such as PAR polarity complex components.|
| [EDEM1](https://www.ncbi.nlm.nih.gov/gene?term=EDEM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNDC9](https://www.ncbi.nlm.nih.gov/gene?term=TXNDC9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.6</br>**STRING score:** 234</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Considering the shortest path in the BioGRID interaction network, EDEM1 and TXNDC9 may form a complex, potentially impacting protein folding. EDEM1 extracts misfolded glycoproteins for degradation while TXNDC9 negatively impacts protein folding, forming ternary complexes with chaperonins.|
