| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 103 (min len 3) | One possible functional association between EPRS and HSPB1 is their interaction in protein folding and stress response. EPRS is involved in aminoacylation of tRNA and can be recruited to the GAIT complex to suppress translation, while HSPB1 functions as a molecular chaperone and may maintain denatured proteins in a folding-competent state. The shortest path between EPRS and HSPB1 in the BioGRID protein interaction network suggests potential physical interactions between these two proteins.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between FAM20B and HSPB1 may be related to stress response and protein folding, as HSPB1 is a chaperone protein involved in these processes and FAM20B may influence catalytic activity and protein function through 2-O-phosphorylation. However, further research is needed to determine any direct or indirect interactions between these genes in specific biological pathways.|
| [CAMSAP2](https://www.ncbi.nlm.nih.gov/gene?term=CAMSAP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional associations between CAMSAP2 and HSPB1 may involve regulation of microtubule dynamics and organization, as well as stress response and protein folding through molecular chaperone activity. However, no direct interaction has been reported between these two genes.|
| [GPATCH8](https://www.ncbi.nlm.nih.gov/gene?term=GPATCH8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between GPATCH8 and HSPB1 may be related to protein folding and stress response, as they are both involved in chaperone activity. GPATCH8 is a mitochondrial chaperonin that facilitates proper folding of imported proteins, while HSPB1 serves as a molecular chaperone that maintains denatured proteins in a folding-competent state and plays a role in stress resistance. The shortest path between these genes in the BioGRID protein interaction network suggests potential indirect interactions through other proteins such as ATXN1, TP53, BTRC, EPB41L3, and PHF12.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | One possible functional association between HSPB1 and YME1L1 is their involvement in maintaining cellular proteostasis. HSPB1 acts as a chaperone to maintain denatured proteins in a folding-competent state, while YME1L1 is an ATP-dependent metalloprotease that catalyzes protein degradation in the mitochondria. Both genes play important roles in cellular stress response and in maintaining cellular homeostasis.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 149 (min len 4) | There is no clear functional association between FAM20B and UBL5 based on the context provided. FAM20B is responsible for 2-O-phosphorylation of xylose in glycosaminoglycan-protein linkage and interacts with enterovirus protein 2B. UBL5 is a component of the chaperonin-containing T-complex, mediates the folding of WRAP53/TCAB1 and interacts with CLK1, CLK3, and CLK4.|
| [DUSP22](https://www.ncbi.nlm.nih.gov/gene?term=DUSP22[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIPSNAP1](https://www.ncbi.nlm.nih.gov/gene?term=NIPSNAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between DUSP22 and NIPSNAP1 could be related to endocytosis and neurite outgrowth processes. DUSP22 activates the Jnk signaling pathway while NIPSNAP1 is involved in receptor-mediated endocytosis and regulating slow endocytic recycling, and can indirectly play a role in neurite outgrowth. The interaction network shows they are connected by proteins involved in membrane trafficking, suggesting they may play a role in coordinating endocytic processes.|
| [PHKB](https://www.ncbi.nlm.nih.gov/gene?term=PHKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRSF4](https://www.ncbi.nlm.nih.gov/gene?term=SRSF4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations: PHKB is involved in regulating phosphorylation and activity of enzymes, while SRSF4 plays a role in alternative splicing during pre-mRNA processing. Shortest paths in the BioGRID network suggest they may interact through proteins involved in splicing regulation and ubiquitination. Therefore, it is possible they may have a role in regulating alternative splicing of genes involved in metabolic processes.|
| [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional association: PSMG1 and PIH1D1 may play a role in proteasome assembly and transcription regulation. They interact indirectly through POLR3E and CAPZA1 in the shortest protein interaction path.|
| [CBLB](https://www.ncbi.nlm.nih.gov/gene?term=CBLB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST7](https://www.ncbi.nlm.nih.gov/gene?term=ST7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear biological association between CBLB and ST7 based on the information provided. CBLB acts as an E3 ubiquitin-protein ligase, negatively regulating TCR, BCR, and FCER1 signal transduction pathways. ST7 may act as a tumor suppressor and forms homotetramers.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Both HSPB1 and MSH6 are involved in DNA repair mechanisms, suggesting a possible functional association. However, they act in different stages of DNA repair, with HSPB1 functioning as a molecular chaperone to maintain folding-competency of denatured proteins and MSH6 being a component of the post-replicative DNA mismatch repair system.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KHDRBS1](https://www.ncbi.nlm.nih.gov/gene?term=KHDRBS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.3</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 88 (min len 3) | Possible functional association: Both genes are involved in RNA binding and regulation, with KHDRBS1 also being involved in protein-protein interactions and signal transduction.   Biological process involvement: HSPB1 is a chaperone protein involved in maintaining protein folding and stress resistance, while KHDRBS1 is involved in RNA splicing and alternative splicing regulation, signal transduction, and apoptosis. There may be a possible role for KHDRBS1 in regulating HSPB1-mediated protein folding through its RNA binding activity.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOR1A](https://www.ncbi.nlm.nih.gov/gene?term=TOR1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.6</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between HSPB1 and TOR1A include their roles in protein folding and quality control, as well as their involvement in cellular trafficking and organization. Specifically, HSPB1 functions as a molecular chaperone while TOR1A regulates protein folding and stability. Additionally, both genes have been implicated in the regulation of neurotransmitter transportation and synaptic vesicle recycling.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.5</br>**STRING score:** 207</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between G3BP1 and HSPB1 may involve stress response and protein quality control, as both genes are involved in stress granule formation and maintenance of folded protein states through chaperone activity and regulation of protein phosphorylation and transport. The shortest path between these two genes in the BioGRID protein interaction network suggests they may directly interact.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.6</br>**STRING score:** 328</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between ATF6 and HSPB1 include the regulation of stress response and protein folding. ATF6 is involved in the unfolded protein response and transcriptional activation, while HSPB1 functions as a molecular chaperone and helps maintain denatured proteins in a folding-competent state. The shortest path between the two genes in the BioGRID protein interaction network suggests a potential interaction through multiple intermediates, including MAPK14 and TRIM25.|
| [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.2</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between NFKBIB and PIK3C3 could include regulation of NF-kappa-B signaling and autophagy. NFKBIB inhibits NF-kappa-B by trapping it in the cytoplasm, while PIK3C3 is a component of the PI3K complex involved in autophagy. The shortest path in the BioGRID protein interaction network between these two genes includes common interacting proteins such as CSNK2A1 and HSP90AA1.|
| [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.2</br>**STRING score:** 373</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between FOXO3 and HSPB1 include regulation of apoptosis, autophagy, and stress response, as well as maintenance of protein folding and actin organization. FOXO3 is a transcriptional activator that binds to DNA sequences and regulates processes such as apoptosis and autophagy, while HSPB1 functions as a molecular chaperone and likely maintains denatured proteins in a folding-competent state. The shortest path between these two genes in the BioGRID protein interaction network involves intermediates YWHAZ and FKBP4.|
| [RPS6](https://www.ncbi.nlm.nih.gov/gene?term=RPS6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.8</br>**STRING score:** 429</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 62 (min len 3) | Possible functional association: RPS6 and SRC are involved in cell growth and proliferation. RPS6 is a component of the small ribosomal subunit and plays a role in selective translation of mRNA. SRC is a non-receptor protein tyrosine kinase involved in neurite elongation and interacts with HCV non-structural protein 5A. Their interaction via the shortest path in the BioGRID protein interaction network suggests possible involvement in protein synthesis and signaling pathways related to these processes.|
| [TERF2IP](https://www.ncbi.nlm.nih.gov/gene?term=TERF2IP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TJP1](https://www.ncbi.nlm.nih.gov/gene?term=TJP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between TERF2IP and TJP1 is the regulation of cell adhesion and migration, as TJP1 is involved in junctional retention and stability of transcription factor DBPA, in podosome formation, and targets CDC42BPB to the leading edge of migrating cells while TERF2IP interacts with SLX4 /BTBD12 and is involved in the regulation of NF-kappa-B signaling by promoting IKK-mediated phosphorylation of RELA/p65. They may also contribute to the formation of tight junctions and the regulation of gene expression.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | These genes do not have a known functional association. HSPB1 is a small heat shock protein which functions as a molecular chaperone and plays a role in stress resistance and actin organization. USP1 is a negative regulator of DNA damage repair and specifically deubiquitinates monoubiquitinated FANCD2.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP14A](https://www.ncbi.nlm.nih.gov/gene?term=UTP14A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PHGDH and UTP14A could involve ribosome biogenesis. UTP14A has been implicated in ribosome biogenesis while PHGDH catalyzes the first step of the phosphorylated L-serine biosynthesis pathway, which is a crucial amino acid needed for protein synthesis.|
| [GALE](https://www.ncbi.nlm.nih.gov/gene?term=GALE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM110](https://www.ncbi.nlm.nih.gov/gene?term=TMEM110[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.6</br>**STRING score:** 209</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 4) | TMEM110 may regulate the activity of GALE, since they interact through DDX39A in the protein interaction network. GALE catalyzes the reversible epimerization of UDP-glucose to UDP-galactose and contributes to glycoprotein and glycolipid synthesis, while TMEM110 acts as a regulator of store-operated Ca(2+) entry at the ER-PM junction and interacts with STIM1.|
| [PHKB](https://www.ncbi.nlm.nih.gov/gene?term=PHKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCAND1](https://www.ncbi.nlm.nih.gov/gene?term=SCAND1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.2</br>**STRING score:** 323</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 67 (min len 4) | There are no clear functional associations between PHKB and SCAND1 based on their known biology and the provided context. PHKB is involved in phosphorylation of certain substrates, including troponin I, while SCAND1 may regulate transcriptional activity and interacts with ZNF202.|
| [SORBS3](https://www.ncbi.nlm.nih.gov/gene?term=SORBS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC6A](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC6A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.5</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on the BioGRID protein interaction network, SORBS3 and TRAPPC6A may be functionally associated through the transport protein particle (TRAPP) complex. SORBS3 is involved in actin stress fiber formation and interacts with MAPK1/ERK2, while TRAPPC6A plays a role in vesicular transport during the biogenesis of melanosomes and interacts with TRAPPC2L.|
