| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 103 (min len 3) | Possible functional associations between EPRS and HSPB1 in TCGA-KIRP could be related to their roles in stress response and protein folding. EPRS is part of the aminoacyl-tRNA synthetase multienzyme complex, while HSPB1 functions as a molecular chaperone. It is possible that EPRS and HSPB1 work together to ensure proper protein folding and stress resistance in TCGA-KIRP.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: FAM20B is responsible for 2-O-phosphorylation of xylose in the glycosaminoglycan-protein linkage region of proteoglycans, while HSPB1 functions as a molecular chaperone and plays a role in stress resistance and actin organization. Given their roles in protein modification and folding, respectively, it's possible that these genes may be functionally associated in regulating proteoglycan synthesis or degradation.|
| [CAMSAP2](https://www.ncbi.nlm.nih.gov/gene?term=CAMSAP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 86.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 21 (min len 3) | Possible functional association: CAMSAP2 and HSPB1 may have a role in regulating microtubule dynamics and organization. CAMSAP2 specifically binds to the minus-end of non-centrosomal microtubules and regulates their stabilization, organization of cellular organelles, neuronal polarity and development. HSPB1 functions as a molecular chaperone in maintaining denatured proteins in a folding-competent state and may regulate numerous biological processes, including the axonal transport of neurofilament proteins.|
| [GPATCH8](https://www.ncbi.nlm.nih.gov/gene?term=GPATCH8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 82.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between GPATCH8 and HSPB1 include the facilitation of correct folding and assembly of proteins, stress resistance, and regulation of biological processes such as phosphorylation and axonal transport. While there is no direct evidence of these genes interacting in specific biological processes, they both play a role in protein folding and may indirectly affect each other's function in the cell.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 80.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between HSPB1 and YME1L1 may be related to mitochondrial function and response to stress. HSPB1 functions as a molecular chaperone and is important in stress resistance and actin organization. YME1L1 is an ATP-dependent metalloprotease that plays an important role in regulating mitochondrial morphology and function, as well as proteolysis of mitochondrial proteins. Both genes are involved in mitochondrial activity and may play a role in maintaining the quality of proteins and organelles in response to stress.|
| [FAM20B](https://www.ncbi.nlm.nih.gov/gene?term=FAM20B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBL5](https://www.ncbi.nlm.nih.gov/gene?term=UBL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 78.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 149 (min len 4) | Possible functional associations between FAM20B and UBL5 include their involvement in protein folding and regulation of cellular processes such as glycosaminoglycan-protein linkage and telomere maintenance. While there is no direct evidence of their interaction, they both interact with the protein DDX39B, suggesting a potential involvement in RNA metabolism.|
| [DUSP22](https://www.ncbi.nlm.nih.gov/gene?term=DUSP22[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NIPSNAP1](https://www.ncbi.nlm.nih.gov/gene?term=NIPSNAP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between DUSP22 and NIPSNAP1 may involve the regulation of endocytosis and recycling of endocytosed proteins, as well as the role in neurite outgrowth. DUSP22 activates the Jnk signaling pathway, while NIPSNAP1 is a lipid-binding protein that recruits cytosolic proteins to regulate membrane tubulation on endosome membranes.|
| [PHKB](https://www.ncbi.nlm.nih.gov/gene?term=PHKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRSF4](https://www.ncbi.nlm.nih.gov/gene?term=SRSF4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between PHKB and SRSF4 include alternative splicing regulation and phosphorylation modulation, as SRSF4 is involved in alternative splice site selection and PHKB acts as a regulatory unit and modulates activity of the holoenzyme in response to phosphorylation.|
| [PIH1D1](https://www.ncbi.nlm.nih.gov/gene?term=PIH1D1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMG1](https://www.ncbi.nlm.nih.gov/gene?term=PSMG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional association: PIH1D1 is involved in the assembly and regulation of protein complexes, including the mTORC1 complex, while PSMG1 promotes assembly and prevents dimerization of the proteasome alpha subunits. These genes may play roles in regulating protein degradation and/or cellular signaling pathways.          Specific gene function: PIH1D1 mediates interaction of TELO2 with the R2TP complex, enhances pre-rRNA transcription, and recruits the SWI/SNF complex to the core promoter of rRNA genes. PSMG1 promotes assembly of the 20S proteasome as a heterodimer with PSMG2.|
| [CBLB](https://www.ncbi.nlm.nih.gov/gene?term=CBLB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST7](https://www.ncbi.nlm.nih.gov/gene?term=ST7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: CBLB is involved in ubiquitination, negatively regulates signal transduction, and interacts with EGFR. ST7 may act as a tumor suppressor and may interact with RACK1 and other proteins involved in signal transduction pathways. There is no clear biological process linking these two genes based on the currently available information.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 72.9</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: MSH6 is involved in DNA mismatch repair and HSPB1 plays a role in stress resistance and actin organization. It is possible that HSPB1 could aid in MSH6-mediated repair processes under stress conditions.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KHDRBS1](https://www.ncbi.nlm.nih.gov/gene?term=KHDRBS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.3</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 88 (min len 3) | One possible functional association between HSPB1 and KHDRBS1 may be in their roles in stress response and cellular organization. Specifically, HSPB1 is a molecular chaperone that maintains denatured proteins in a folding-competent state, while KHDRBS1 has been shown to self-associate and form homooligomers in response to stress and play a role in RNA substrate specificity modulation. Their interaction via a shortest path in the BioGRID protein interaction network suggests a potential regulatory relationship in these processes.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOR1A](https://www.ncbi.nlm.nih.gov/gene?term=TOR1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.6</br>**STRING score:** 155</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association between HSPB1 and TOR1A could be in protein folding and quality control. HSPB1 is a molecular chaperone that maintains denatured proteins in a folding-competent state, while TOR1A is involved in the quality control of protein folding, clearance of misfolded proteins and endoplasmic reticulum stress response.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 65.5</br>**STRING score:** 207</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between G3BP1 and HSPB1 could involve stress response and protein folding. G3BP1 plays a role in stress granule formation and HSPB1 functions as a molecular chaperone, maintaining denatured proteins in a folding-competent state.|
| [ATF6](https://www.ncbi.nlm.nih.gov/gene?term=ATF6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 64.6</br>**STRING score:** 328</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations: ATF6 is a transcription factor involved in the unfolded protein response during ER stress, while HSPB1 is a small heat shock protein that functions as a molecular chaperone. There could be a possible association between these genes in maintaining protein folding competence during stress response.|
| [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3C3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3C3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.2</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: NFKBIB inhibits NF-kappa-B by trapping it in the cytoplasm, while PIK3C3 is a component of the PI3K complex involved in autophagy and microbial infection. It's possible that NFKBIB may regulate autophagy through PIK3C3, or that PIK3C3 may affect NF-kappa-B signaling through NFKBIB. However, there is no direct evidence to suggest a functional association between these two genes.|
| [FOXO3](https://www.ncbi.nlm.nih.gov/gene?term=FOXO3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 63.2</br>**STRING score:** 373</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional association: FOXO3 activates autophagy genes, while HSPB1 functions as a molecular chaperone and regulates stress resistance. They both have a role in cellular response to stress, which suggests they may function together in protein homeostasis. The BioGRID protein interaction network shows a possible interaction through several intermediary proteins.|
| [RPS6](https://www.ncbi.nlm.nih.gov/gene?term=RPS6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.8</br>**STRING score:** 429</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 62 (min len 3) | Possible functional associations between RPS6 and SRC in TCGA-KIRP cancer type could be related to translation regulation and cell growth/proliferation control (RPS6) and non-receptor protein tyrosine kinase activity and neurite elongation (SRC). The shortest path in the BioGRID protein interaction network suggests that RPS6 and SRC could interact through common protein partners such as HSP90AA1, RPS6KB1, and ARRB2, among others, playing potential roles in cancer progression in TCGA-KIRP.|
| [TERF2IP](https://www.ncbi.nlm.nih.gov/gene?term=TERF2IP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TJP1](https://www.ncbi.nlm.nih.gov/gene?term=TJP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.8</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between TERF2IP and TJP1 include their involvement in cell adhesion and matrix remodeling, as well as in regulation of gene expression. While TERF2IP acts both as a regulator of telomere function and a transcription regulator, TJP1 is a scaffolding protein that links tight junction proteins to the actin cytoskeleton, regulating barrier formation and cell migration. These genes may be involved in processes related to the maintenance of the integrity and polarity of epithelial and endothelial cells.|
| [HSPB1](https://www.ncbi.nlm.nih.gov/gene?term=HSPB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | HSPB1 and USP1 have no known direct functional associations. HSPB1 is a small heat shock protein that functions as a molecular chaperone and regulates numerous biological processes, while USP1 is a negative regulator of DNA damage repair and is involved in PCNA-mediated translesion synthesis.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP14A](https://www.ncbi.nlm.nih.gov/gene?term=UTP14A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between PHGDH and UTP14A. PHGDH catalyzes steps in the serine biosynthesis pathway while UTP14A is involved in ribosome biogenesis.|
| [GALE](https://www.ncbi.nlm.nih.gov/gene?term=GALE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM110](https://www.ncbi.nlm.nih.gov/gene?term=TMEM110[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.6</br>**STRING score:** 209</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 4) | GALE and TMEM110 are not known to have direct functional associations. GALE catalyzes interconversions of UDP-sugars important in the synthesis of glycoproteins and glycolipids, while TMEM110 regulates store-operated Ca(2+) entry at ER-PM junctions. The shortest path between them in the BioGRID protein interaction network involves several intermediate proteins, including DDX39A, which is involved in RNA metabolism.|
| [PHKB](https://www.ncbi.nlm.nih.gov/gene?term=PHKB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCAND1](https://www.ncbi.nlm.nih.gov/gene?term=SCAND1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.2</br>**STRING score:** 323</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 67 (min len 4) | There is no clear functional association between PHKB and SCAND1 based on their known biological roles. PHKB acts as a regulatory subunit of phosphorylase b kinase, while SCAND1 may regulate transcriptional activity. The shortest path between PHKB and SCAND1 in the BioGRID protein interaction network suggests that they may interact via common binding partners such as SNRPA.|
| [SORBS3](https://www.ncbi.nlm.nih.gov/gene?term=SORBS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC6A](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC6A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.5</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: SORBS3 is involved in actin stress fiber formation and interacts with MAPK1/ERK2 and EGF. TRAPPC6A is involved in vesicular transport and part of the TRAPP complex. There are currently no known direct biological connections between these two genes.|
