| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [STXBP1](https://www.ncbi.nlm.nih.gov/gene?term=STXBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMCO6](https://www.ncbi.nlm.nih.gov/gene?term=TMCO6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 63 (min len 4) | There is no clear functional association between STXBP1 and TMCO6 in specific biological terms, as they have very different roles. STXBP1 is involved in regulation of synaptic vesicle docking and fusion, while TMCO6 is an accessory subunit of the mitochondrial membrane respiratory chain NADH dehydrogenase.|
| [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 118 (min len 4) | Possible functional associations between HLA-DMA and SRP14 could be in protein folding and transport. HLA-DMA plays a critical role in releasing peptides from MHC class II molecules while SRP14 is a component of the signal recognition particle complex that mediates the cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum, suggesting a connection in protein processing.|
| [EDN1](https://www.ncbi.nlm.nih.gov/gene?term=EDN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LAP3](https://www.ncbi.nlm.nih.gov/gene?term=LAP3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 4) | Possible functional associations between EDN1 and LAP3 include the metabolism of glutathione and degradation of glutathione S-conjugates. EDN1 is involved in vasoconstriction and the activation of PTK2B, BCAR1, BCAR3, RAP1, and RHOA cascades. LAP3 is a metallopeptidase that catalyzes the removal of unsubstituted N-terminal hydrophobic amino acids from various peptides.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.9</br>**STRING score:** 329</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | CCL2 and GRN are functionally associated through protein-protein interaction. CCL2 acts as a chemokine ligand involved in monocyte recruitment, while GRN stabilizes CTSD through interaction leading to maintain its aspartic-type peptidase activity and mediates lysosomal delivery in stressed neurons.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPIC](https://www.ncbi.nlm.nih.gov/gene?term=PPIC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no evidence for a direct functional association between INTS3 and PPIC in terms of their biological processes. INTS3 is involved in DNA repair and checkpoint activation, while PPIC is an isomerase involved in protein folding. The interaction between them could be mediated by other proteins in the pathways they function in.|
| [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | PGRMC1 and RALA have a possible functional association through their interaction in the BioGRID protein interaction network. PGRMC1 is involved in heme homeostasis while RALA is involved in gene expression, cell migration, and membrane trafficking.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKACA](https://www.ncbi.nlm.nih.gov/gene?term=PRKACA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between INTS3 and PRKACA may involve DNA repair processes and cellular signaling pathways involved in maintaining genomic stability. INTS3 is a component of the SOSS complex involved in homologous recombination-dependent repair of double-strand breaks and ATM-dependent signaling pathways, while PRKACA is involved in phosphorylation and activation of various targets in the context of cellular signaling, such as ABL1 in sperm flagellum and protein kinase A (PKA) holoenzymes.|
| [MEST](https://www.ncbi.nlm.nih.gov/gene?term=MEST[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR1C](https://www.ncbi.nlm.nih.gov/gene?term=POLR1C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no obvious or known functional association between MEST and POLR1C in specific biological terms. MEST is involved in initiating chromosomal DNA replication, while POLR1C is a subunit of RNA polymerases I and III. However, the BioGRID protein interaction network shows a shortest path between MEST and POLR1C through ESR1 and TRIM67, suggesting potential protein-protein interactions.|
| [BRPF1](https://www.ncbi.nlm.nih.gov/gene?term=BRPF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [S100A13](https://www.ncbi.nlm.nih.gov/gene?term=S100A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 128 (min len 4) | Possible functional associations between BRPF1 and S100A13 could be related to protein export regulation or copper-dependent stress response. However, based on the limited information, it is difficult to pinpoint a specific biological process where these two genes interact. BRPF1 is a scaffold subunit of various histone acetyltransferase complexes, while S100A13 plays a role in the export of proteins that lack a signal peptide.|
| [ARID4B](https://www.ncbi.nlm.nih.gov/gene?term=ARID4B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CANT1](https://www.ncbi.nlm.nih.gov/gene?term=CANT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 310 (min len 4) | There is likely no direct functional association between ARID4B and CANT1 based on their respective roles in transcriptional regulation and nucleotide metabolism. ARID4B acts as a transcriptional repressor and coactivator in spermatogenesis and myeloid homeostasis, while CANT1 is involved in proteoglycan synthesis through calcium-dependent nucleotidase activity.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.2</br>**STRING score:** 511</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CASP3 and LYN include regulation of apoptosis and immune response, as both genes are involved in these processes. The BioGRID protein interaction network shows a shortest path of interactions between the two genes.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.8</br>**STRING score:** 203</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional associations between HSPA4 and NVL can be inferred from their shared involvement in ribosomal subunit biogenesis and RNA processing pathways. HSPA4 is a kinase component of the chromosomal passenger complex regulating mitosis, while NVL participates in telomerase activity and interacts with nucleolin. No direct links between their functions have been reported.|
| [ACAT2](https://www.ncbi.nlm.nih.gov/gene?term=ACAT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LBR](https://www.ncbi.nlm.nih.gov/gene?term=LBR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.4</br>**STRING score:** 371</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between ACAT2 and LBR could involve cholesterol biosynthesis, as ACAT2 is involved in this pathway and LBR catalyzes a step in it. Additionally, LBR anchors the lamina and heterochromatin to the inner nuclear membrane, which could potentially interact with ACAT2's role in maintaining critical levels of cholesterol required for membrane lipid raft formation during neutrophil differentiation.|
| [ALDH7A1](https://www.ncbi.nlm.nih.gov/gene?term=ALDH7A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 272</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | There are no clear functional associations between ALDH7A1 and SRC based on the provided context and known biology of the genes. ALDH7A1 is a multifunctional enzyme involved in lysine catabolism and cellular osmoregulation, while SRC is a tyrosine kinase involved in neurite elongation and may interact with viral proteins.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC1A](https://www.ncbi.nlm.nih.gov/gene?term=SMC1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.6</br>**STRING score:** 216</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between EED and SMC1A include their involvement in chromatin organization and DNA repair. EED is a component of the PRC2/EED-EZH2 complex which methylates histones, leading to transcriptional repression of target genes, while SMC1A is a central component of the cohesin complex which is required for chromosome cohesion during cell cycle and in DNA repair.|
| [ARHGEF2](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.9</br>**STRING score:** 803</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | There is currently no clear biological basis for the association between ARHGEF2 and TUBB6. ARHGEF2 is involved in Rho-GTPase signaling and innate immune response, while TUBB6 is a major component of microtubules.|
| [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are several possible functional associations between CDKN1B and RALA based on their interactions in the BioGRID protein interaction network. CDKN1B is an important regulator of cell cycle progression, while RALA is a multifunctional GTPase involved in various cellular processes, including cell proliferation and membrane trafficking. The shortest path between these genes involves several intermediates, suggesting they may have indirect effects on each other's functions.|
| [DNAJB1](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.4</br>**STRING score:** 749</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 3) | Possible functional associations between DNAJB1 and FKBP4 are related to protein folding and regulation of cellular stress responses. DNAJB1 stimulates protein folding mediated by HSP70 and regulates the activity of heat shock factor 1 (HSF1) during the recovery phase of the heat shock response, while FKBP4 is involved in the intracellular trafficking of steroid hormone receptors and regulates neuronal growth cones via regulation of TRPC1 channel opening. The shortest path between these two genes in the BioGRID protein interaction network involves chaperone proteins and suggests a potential role in protein folding.|
| [LBR](https://www.ncbi.nlm.nih.gov/gene?term=LBR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.4</br>**STRING score:** 190</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | Possible functional association between LBR and SUZ12: LBR is involved in cholesterol biosynthesis and anchoring lamina and heterochromatin to the inner nuclear membrane while SUZ12 is part of the PRC2 complex that can methylate histones leading to transcriptional repression. The shortest path between LBR and SUZ12 in the BioGRID protein interaction network involves several proteins including CBX3, CBX5, and AKAP8L, suggesting possible indirect interactions between LBR and SUZ12.|
| [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.9</br>**STRING score:** 308</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional associations: RALA is a GTPase involved in cellular processes such as gene expression, cell migration, and oncogenic transformation, while TUBB6 is a major constituent of microtubules. It is possible that RALA plays a role in regulating microtubule dynamics through its interaction with TUBB6.|
