| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [STXBP1](https://www.ncbi.nlm.nih.gov/gene?term=STXBP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMCO6](https://www.ncbi.nlm.nih.gov/gene?term=TMCO6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 102.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 63 (min len 4) | There are no known functional associations between STXBP1 and TMCO6. STXBP1 is involved in neurotransmitter release and interacts with syntaxin and SNARE complex components, while TMCO6 is an accessory subunit of the mitochondrial membrane respiratory chain NADH dehydrogenase.|
| [HLA-DMA](https://www.ncbi.nlm.nih.gov/gene?term=HLA-DMA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 97.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 118 (min len 4) | Possible functional associations between HLA-DMA and SRP14 may involve the cotranslational targeting of secretory and membrane proteins to the endoplasmic reticulum, as well as the processing of peptides for antigen presentation. Both genes are involved in the function of the signal recognition particle complex.|
| [EDN1](https://www.ncbi.nlm.nih.gov/gene?term=EDN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LAP3](https://www.ncbi.nlm.nih.gov/gene?term=LAP3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 96.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 4) | Possible functional associations between EDN1 and LAP3 could involve the metabolism of glutathione and degradation of glutathione S-conjugates, as LAP3 is involved in these processes. EDN1, on the other hand, is involved in vasoconstriction, activating PTK2B, BCAR1, BCAR3, GTPases RAP1 and RHOA cascade, and binding DEAR/FBXW7-AS1 receptor.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRN](https://www.ncbi.nlm.nih.gov/gene?term=GRN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 94.9</br>**STRING score:** 329</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Both CCL2 and GRN have been shown to be involved in inflammatory processes and immune cell recruitment. They may be functionally associated in the context of inflammation and potentially contribute to the regulation of immune responses. The shortest path between the two genes via RNF4 suggests a potential direct interaction between them, but this requires further investigation.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPIC](https://www.ncbi.nlm.nih.gov/gene?term=PPIC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 93.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: Both genes are involved in DNA repair and maintenance of genomic stability. INTS3 is a component of the SOSS complex, while PPIC may assist protein folding. Since protein folding and DNA repair are essential processes for cellular health, it is possible that they have indirect interactions in maintaining cellular homeostasis.|
| [PGRMC1](https://www.ncbi.nlm.nih.gov/gene?term=PGRMC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | Possible functional associations between PGRMC1 and RALA in TCGA-KIRP cancer may involve regulation of cell proliferation and oncogenic transformation. PGRMC1 plays a role in progesterone binding, heme homeostasis and acts as a heme donor for at least some hemoproteins, while RALA is multifunctional GTPase involved in various cellular processes like cell migration, proliferation, and gene expression. The BioGRID protein interaction network suggests that PGRMC1 and RALA have direct interactions, indicating possible regulatory effects on downstream pathways.|
| [INTS3](https://www.ncbi.nlm.nih.gov/gene?term=INTS3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKACA](https://www.ncbi.nlm.nih.gov/gene?term=PRKACA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 92.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between INTS3 and PRKACA may relate to DNA damage repair and cellular signaling pathways. INTS3 is a component of the SOSS complex involved in homologous recombination-dependent repair of double-strand breaks, while PRKACA phosphorylates and activates proteins involved in a range of cellular processes including spermatozoa capacitation and regulation of tight junction assembly. The short protein interaction network suggests they may interact indirectly through other proteins, potentially affecting cellular processes such as DNA repair and signaling.|
| [MEST](https://www.ncbi.nlm.nih.gov/gene?term=MEST[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLR1C](https://www.ncbi.nlm.nih.gov/gene?term=POLR1C[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 91.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between MEST and POLR1C may exist in the regulation and initiation of DNA replication and transcription processes. MEST is a core component of the CDC45-MCM-GINS helicase, which unwinds template DNA during replication, while POLR1C is a common component of RNA polymerases I and III, which synthesize ribosomal RNA precursors and small RNAs. The interaction network suggests a potential indirect connection between MEST and POLR1C via ESR1 and TRIM67.|
| [BRPF1](https://www.ncbi.nlm.nih.gov/gene?term=BRPF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [S100A13](https://www.ncbi.nlm.nih.gov/gene?term=S100A13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 90.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 128 (min len 4) | There is no clear functional association between BRPF1 and S100A13 based on their known biological processes and interaction partners. BRPF1 is involved in histone acetylation and transcription regulation, while S100A13 is involved in protein secretion and binds calcium and copper ions. Their shortest interaction path involves the protein RPN2.|
| [ARID4B](https://www.ncbi.nlm.nih.gov/gene?term=ARID4B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CANT1](https://www.ncbi.nlm.nih.gov/gene?term=CANT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 89.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 310 (min len 4) | Possible functional associations: ARID4B and CANT1 have a shortest path of interactions in the BioGRID protein interaction network, suggesting a potential direct interaction. ARID4B is involved in transcriptional regulation and myeloid homeostasis, while CANT1 is involved in proteoglycan synthesis. There is currently no known biological process in which both genes are directly involved.|
| [CASP3](https://www.ncbi.nlm.nih.gov/gene?term=CASP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 83.2</br>**STRING score:** 511</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between CASP3 and LYN can include their involvement in the regulation of apoptosis and immune responses. CASP3 is a protease that mediates the execution phase of apoptosis, while LYN is a tyrosine kinase that plays a role in the regulation of innate and adaptive immune responses. They are linked via the BioGRID protein interaction network through a few shortest paths, suggesting potential physical interactions.|
| [HSPA4](https://www.ncbi.nlm.nih.gov/gene?term=HSPA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NVL](https://www.ncbi.nlm.nih.gov/gene?term=NVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 81.8</br>**STRING score:** 203</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 28 (min len 3) | Possible functional association between HSPA4 and NVL: NVL is involved in ribosome biogenesis and interacts with NCL, while HSPA4 is involved in mitosis regulation and interacts with TJP1. There is no clear biological relationship or overlap between these two functions.|
| [ACAT2](https://www.ncbi.nlm.nih.gov/gene?term=ACAT2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LBR](https://www.ncbi.nlm.nih.gov/gene?term=LBR[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 79.4</br>**STRING score:** 371</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no apparent functional association between ACAT2 and LBR based on their known biology and protein interactions. ACAT2 is involved in cholesterol biosynthesis, while LBR catalyzes the reduction of lanosterol in the same pathway and also anchors the lamina and heterochromatin to the inner nuclear membrane.|
| [ALDH7A1](https://www.ncbi.nlm.nih.gov/gene?term=ALDH7A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.7</br>**STRING score:** 272</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Based on the BioGRID protein interaction network, there are several possible functional associations between ALDH7A1 and SRC, including processes related to cell signaling, cytoskeletal remodeling, and lysine catabolism. However, without further evidence, it is difficult to determine the specific biological context in which these associations occur. ALDH7A1 is a multifunctional enzyme involved in lysine catabolism and can protect cells from oxidative stress. On the other hand, SRC is a non-receptor protein tyrosine kinase that plays a role in neurite elongation and interacts with HCV non-structural protein 5A.|
| [EED](https://www.ncbi.nlm.nih.gov/gene?term=EED[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC1A](https://www.ncbi.nlm.nih.gov/gene?term=SMC1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 77.6</br>**STRING score:** 216</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: Both EED and SMC1A are involved in DNA repair processes. EED is part of the PRC2/EED-EZH2 complex, which may recruit DNA methyltransferases for epigenetic repression, while SMC1A is involved in chromosome cohesion after DNA replication. EED may also interact with the MA protein of HIV-1. There is no known direct interaction between these two genes, but they may be involved in related biological processes.|
| [ARHGEF2](https://www.ncbi.nlm.nih.gov/gene?term=ARHGEF2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.9</br>**STRING score:** 803</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 44 (min len 3) | Possible functional association between ARHGEF2 and TUBB6 may involve regulation of microtubule dynamics during cell division and migration. ARHGEF2 activates Rho-GTPases involved in cell motility and polarization, while TUBB6 is a major constituent of microtubules essential for cell division and migration.|
| [CDKN1B](https://www.ncbi.nlm.nih.gov/gene?term=CDKN1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.5</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between CDKN1B and RALA include regulation of cell cycle progression and anchorage-independent proliferation of transformed cells, which are both important processes in cancer development. The shortest path between these genes in the BioGRID protein interaction network suggests possible direct interactions through proteins such as ELAVL1, TSC2, and RHOA.|
| [DNAJB1](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 76.4</br>**STRING score:** 749</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 40 (min len 3) | Possible functional associations between DNAJB1 and FKBP4 are the regulation of protein folding and trafficking, modulation of steroid hormone receptor complex, and control of microtubule dynamics. They interact in the HSP90 complex through HSF1. DNAJB1 inhibits HSF1 transcriptional activity during the attenuation and recovery phase period of the heat shock response. FKBP4 has PPIase and co-chaperone activities.|
| [LBR](https://www.ncbi.nlm.nih.gov/gene?term=LBR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SUZ12](https://www.ncbi.nlm.nih.gov/gene?term=SUZ12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 75.4</br>**STRING score:** 190</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | The shortest path protein interaction network between LBR and SUZ12 suggests possible functional associations between the two genes. LBR is involved in cholesterol biosynthesis and anchoring the lamina and heterochromatin to the inner nuclear membrane, while SUZ12 is a component of the PRC2 complex that methylates histones and represses gene transcription. It is possible that LBR plays a role in regulating SUZ12-mediated gene repression.|
| [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TUBB6](https://www.ncbi.nlm.nih.gov/gene?term=TUBB6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 74.9</br>**STRING score:** 308</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 13 (min len 3) | Possible functional association between RALA and TUBB6 could be in regulating cytoskeleton dynamics and mitotic spindle formation during cell division since RALA is involved in membrane trafficking, while TUBB6 is a major constituent of microtubules. Additionally, RALA is also involved in mitochondria fission, which could potentially affect microtubule dynamics indirectly.|
