| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between PIN1 and TM9SF2 include their interaction within the Clp complex and their involvement in cellular processes such as protein isomerization, cell division regulation, and proteasomal degradation. However, further experimentation is required to determine the extent and specificity of their functional relationship.|
| [DHRS7](https://www.ncbi.nlm.nih.gov/gene?term=DHRS7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL4R](https://www.ncbi.nlm.nih.gov/gene?term=IL4R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 469 (min len 4) | There is no clear functional association between DHRS7 and IL4R, as they are involved in different biological processes. DHRS7 is involved in the reduction of carbonyl compounds, while IL4R has a role in cytokine signaling and immune response.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 4) | Possible functional association between LOXL1 and TRAPPC3 in TCGA-STAD could be related to cellular transport processes. LOXL1 is involved in extracellular matrix remodeling, while TRAPPC3 is a component of the TRAPP complex, which plays a role in intracellular vesicular transport. It is possible that LOXL1 and TRAPPC3 could have overlapping functions in trafficking extracellular matrix components within cells.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association:      Both C2CD2 and RPS5 are involved in the process of DNA damage response through different mechanisms.      Brief description of gene function:     C2CD2 is involved in binding to DNA breaks and promoting DNA damage-induced apoptosis, while RPS5 is a component of the small ribosomal subunit and is involved in protein synthesis within the cell.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSTA3](https://www.ncbi.nlm.nih.gov/gene?term=TSTA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between PACSIN3 and TSTA3 based on their known functions. PACSIN3 is involved in endocytosis and regulation of internalization of plasma membrane proteins, while TSTA3 catalyzes the conversion of GDP-4-dehydro-6-deoxy-D-mannose to GDP-fucose.|
| [EIF4A3](https://www.ncbi.nlm.nih.gov/gene?term=EIF4A3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TESK1](https://www.ncbi.nlm.nih.gov/gene?term=TESK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: EIF4A3 is involved in pre-mRNA splicing and associates with TESK1 through BIRC3 in the BioGRID protein interaction network. TESK1 is a dual specificity protein kinase that regulates the cytoskeleton and plays a role in spermatogenesis. There is no clear indication of direct functional interaction between these two genes given the available context.|
| [GNAI2](https://www.ncbi.nlm.nih.gov/gene?term=GNAI2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 214</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association between GNAI2 and RHEB is through the process of mTORC1 signaling pathway. GNAI2 regulates the activity of mTORC1 through its interaction with CCDC88C/DAPLE and CCDC8A/GIV, while RHEB activates the protein kinase activity of mTORC1.|
| [GNA11](https://www.ncbi.nlm.nih.gov/gene?term=GNA11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between GNA11 and PACSIN3 include their interactions in endocytosis and internalization of plasma membrane proteins. GNA11 plays a role in transmembrane signaling systems and acts as an activator of phospholipase C, while PACSIN3 regulates internalization and inhibits TRPV4 calcium channel activity.|
| [FOXO4](https://www.ncbi.nlm.nih.gov/gene?term=FOXO4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IFRD2](https://www.ncbi.nlm.nih.gov/gene?term=IFRD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between FOXO4 and IFRD2 based on the provided context and their known biology. FOXO4 is a transcription factor involved in insulin signaling and cell cycle regulation, while IFRD2 is a tyrosine-protein kinase that acts as a cell-surface receptor for fibroblast growth factors and plays a role in cell proliferation, differentiation, and apoptosis.  The two genes are connected by two short paths in the BioGRID protein interaction network.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.1</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between PACSIN3 and SPDEF include the regulation of internalization of plasma membrane proteins and the transcriptional activation of SERPINB5 promoter, respectively. However, there is no direct biological evidence to suggest a functional association between these two genes.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.1</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | PIN1 and YME1L1 do not have any known direct functional associations. PIN1 is a peptidyl-prolyl cis/trans isomerase that regulates mitosis and transactivates oncogenes, while YME1L1 is an ATP-dependent metalloprotease that degrades folded and unfolded proteins in the mitochondrial intermembrane region and regulates mitochondrial morphology and function.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.3</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ATP6V0B and PTPRF are their intermediate interaction nodes EGFR and KRAS that may play a role in signal transduction and regulation of cell division. ATP6V0B codes for a subunit of vacuolar ATPase that pumps protons across the membranes, while PTPRF codes for a protein tyrosine phosphatase that influences substrate specificity and interacts with other proteins involved in insulin signaling and protein trafficking.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL12](https://www.ncbi.nlm.nih.gov/gene?term=MRPL12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.5</br>**STRING score:** 203</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | One possible functional association between KDM5B and MRPL12 is that KDM5B may play a role in mitochondrial translation through its interaction with MRPL12, which is a component of the mitochondrial large ribosomal subunit. KDM5B has also been implicated in cancer, particularly in breast cancer proliferation, while MRPL12 is involved in mitochondrial ribosome biogenesis and function.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.8</br>**STRING score:** 708</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association between CCNH and RNMT is their involvement in RNA transcription and processing. CCNH regulates CDK7, which activates RNA polymerase II by serine phosphorylation of its large subunit, while RNMT is the catalytic subunit of the mRNA-capping methyltransferase complex that methylates the N7 position of the added guanosine to the 5'-cap structure of mRNAs. The shortest path between them in the protein interaction network involves several genes that are involved in transcriptional regulation, suggesting a possible co-regulatory role.|
| [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST6GALNAC2](https://www.ncbi.nlm.nih.gov/gene?term=ST6GALNAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.5</br>**STRING score:** 473</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: SPDEF and ST6GALNAC2 have a protein interaction through CSK, and both genes play a role in prostate cancer. SPDEF functions as a transcriptional activator of PSA and SERPINB5, while ST6GALNAC2 catalyzes the transfer of N-acetylneuraminyl groups onto glycan chains in glycoproteins.|
| [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FZD7](https://www.ncbi.nlm.nih.gov/gene?term=FZD7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.0</br>**STRING score:** 299</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There might not be a direct functional association between COL4A1 and FZD7. COL4A1 is involved in regulating angiogenesis and tumor formation, while FZD7 is a receptor for toxins and is involved in microbial infection.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DPH2](https://www.ncbi.nlm.nih.gov/gene?term=DPH2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.9</br>**STRING score:** 516</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between ATP6V0B and DPH2 include their interaction through a shortest protein interaction path in the BioGRID network, which involves several key signaling proteins. ATP6V0B is involved in acidifying and maintaining the pH of intracellular compartments, while DPH2 is required for the first step of diphthamide biosynthesis, a post-translational modification of histidine which occurs in elongation factor 2.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 365</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | AURKA and IKBKB have no known direct functional association. AURKA is a mitotic kinase involved in spindle formation, microtubule regulation, and p53 pathway regulation. IKBKB is a serine kinase involved in NF-kappaB signaling pathway activation.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPARG](https://www.ncbi.nlm.nih.gov/gene?term=PPARG[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.1</br>**STRING score:** 377</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: None evident from the provided context and BioGRID network. PIN1 is a PPIase involved in regulating mitosis, transactivating oncogenes, and degrading proteins like PML/BCL6/JUN. PPARG is a nuclear receptor involved in transcriptional regulation, interacting with coactivators NCOA6/PDPK1/PRDM16, and also MAPK p38 and IL-6 in response to M. tuberculosis.|
| [CASP10](https://www.ncbi.nlm.nih.gov/gene?term=CASP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 26.9</br>**STRING score:** 255</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CASP10 and GAPDH could involve regulation of apoptosis and inflammation processes. CASP10 functions as a protease in the apoptotic pathway and interacts with proteins involved in inflammation, while GAPDH is involved in glycolysis, microtubule binding, and regulation of nuclear events, as well as in promoting TNF-induced NF-kappa-B activation and type I interferon production. The short path between these genes in the protein interaction network may indicate a possible direct or indirect interaction.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 26.8</br>**STRING score:** 451</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Both PIN1 and SMC4 are involved in cell division. PIN1 regulates mitosis and can induce centrosome amplification and cell transformation, while SMC4 is a central component of the condensin complex required for the conversion of interphase chromatin into mitotic-like condensed chromosomes. The shortest path in the protein interaction network suggests that PIN1 and SMC4 interact through multiple proteins involved in various biological processes including mitosis, chromosome stability, and DNA repair. Therefore, there may be functional associations between these two genes in regulating various aspects of cell division.|
