| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There are no obvious functional associations between PIN1 and TM9SF2 based on their known roles in biology and the provided context. PIN1 is a peptidyl-prolyl cis/trans isomerase that regulates multiple cellular processes, while TM9SF2 functions as a channel or small molecule transporter in intracellular compartments.|
| [DHRS7](https://www.ncbi.nlm.nih.gov/gene?term=DHRS7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL4R](https://www.ncbi.nlm.nih.gov/gene?term=IL4R[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 469 (min len 4) | There is no evidence of a direct functional association between DHRS7 and IL4R, but both genes have been linked to cancer. DHRS7 is a potential tumor suppressor, while IL4R plays a role in T-cell function and has been implicated in the progression of gastric cancer.|
| [LOXL1](https://www.ncbi.nlm.nih.gov/gene?term=LOXL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 4) | There is no clear functional association between LOXL1 and TRAPPC3 based on their known biological roles and interactions. LOXL1 is involved in elastin and collagen substrates activity, while TRAPPC3 may play a role in vesicular transport between endoplasmic reticulum and Golgi.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between C2CD2 and RPS5 are unclear based solely on the provided context. C2CD2 is involved in DNA repair and apoptosis, while RPS5 is a component of the small ribosomal subunit involved in protein synthesis. Without further information, it is difficult to determine a specific biological relationship between these genes.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TSTA3](https://www.ncbi.nlm.nih.gov/gene?term=TSTA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no obvious functional associations between PACSIN3 and TSTA3 based on their known roles, one in endocytosis and the other in GDP-fucose conversion. The shortest path between them in BioGRID protein interaction network involves intermediate proteins HSPA8, XPO1, HNRNPH1, suggesting possible indirect interactions.|
| [EIF4A3](https://www.ncbi.nlm.nih.gov/gene?term=EIF4A3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TESK1](https://www.ncbi.nlm.nih.gov/gene?term=TESK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between EIF4A3 and TESK1 is related to the regulation of gene expression. EIF4A3 is involved in splicing and mRNA metabolism, while TESK1 regulates cytoskeletal dynamics and integrin-mediated cell spreading. Both genes can impact gene expression by modulating mRNA localization, translation efficiency, and stability. However, further studies are required to uncover any direct interaction or regulatory relationship between these genes.|
| [GNAI2](https://www.ncbi.nlm.nih.gov/gene?term=GNAI2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 214</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between GNAI2 and RHEB could be in the regulation of cell proliferation and apoptosis through the mTORC1 signaling pathway. GNAI2 is involved in G protein-coupled receptor signaling and can interact with RHEB in the BioGRID protein interaction network. RHEB activates mTORC1 and plays a role in the regulation of apoptosis.|
| [GNA11](https://www.ncbi.nlm.nih.gov/gene?term=GNA11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional association: GNA11 activates phospholipase C and transduces signaling in response to long-chain fatty acids, while PACSIN3 regulates endocytosis and internalization of plasma membrane proteins, inhibiting TRPV4 calcium channel activity. Thus, they may function together in regulating cellular signaling and protein internalization processes.|
| [FOXO4](https://www.ncbi.nlm.nih.gov/gene?term=FOXO4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IFRD2](https://www.ncbi.nlm.nih.gov/gene?term=IFRD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FOXO4 and IFRD2 could be related to cell proliferation and differentiation. FOXO4 is involved in regulating the insulin signaling pathway and negative regulation of cell cycle, while IFRD2 plays essential roles in the regulation of cell proliferation, differentiation, migration, and apoptosis. The shortest path between these two genes in the BioGRID protein interaction network involves ESR1, OGT, and PLCG1, which could suggest a potential interaction in these processes.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.1</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Based on the provided context, there are no clear functional associations between PACSIN3 and SPDEF. PACSIN3 is involved in endocytosis and regulates internalization of plasma membrane proteins, while SPDEF may function as an androgen-independent transactivator of the prostate-specific antigen (PSA) promoter and may play a role in regulation of the prostate gland and/or prostate cancer development.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.1</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between PIN1 and YME1L1 could be related to the regulation of mitochondrial function and morphology. PIN1 is involved in multiple cellular processes, including the regulation of mitosis and the JNK cascade, while YME1L1 is an ATP-dependent metalloprotease that regulates mitochondrial function and degradation of unfolded proteins. Their interaction could play a role in maintaining normal mitochondrial structure and protein metabolism.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.3</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between ATP6V0B and PTPRF could be related to proton transport and cell signaling regulation, as ATP6V0B is involved in proton translocation and PTPRF has enzymatic activity and interacts with various proteins, including those involved in signal transduction.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MRPL12](https://www.ncbi.nlm.nih.gov/gene?term=MRPL12[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.5</br>**STRING score:** 203</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There are no known functional associations between KDM5B and MRPL12, as they are involved in distinct processes. KDM5B is a histone demethylase that regulates gene expression, while MRPL12 is a component of the mitochondrial large ribosomal subunit involved in mitochondrial translation.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.8</br>**STRING score:** 708</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CCNH and RNMT in TCGA-STAD could be related to RNA transcription and processing, given that CCNH is involved in cell cycle control and RNA transcription by RNA polymerase II, while RNMT is a catalytic subunit of the mRNA-capping methyltransferase complex that interacts with elongating forms of polymerase II. The shortest path between these genes in the BioGRID protein interaction network also involves several transcription-associated proteins, including AR, MYC, BRCA1, BRD4, and EGFR.|
| [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST6GALNAC2](https://www.ncbi.nlm.nih.gov/gene?term=ST6GALNAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.5</br>**STRING score:** 473</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | A possible functional association between SPDEF and ST6GALNAC2 could be in the regulation and modification of glycoproteins. SPDEF may act as a transcriptional activator for ST6GALNAC2, which catalyzes the transfer of N-acetylneuraminyl groups onto glycan chains in glycoproteins.|
| [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FZD7](https://www.ncbi.nlm.nih.gov/gene?term=FZD7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.0</br>**STRING score:** 299</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no obvious functional association between COL4A1 and FZD7 based on the provided context. COL4A1 is involved in the anti-angiogenic activity and interacts with EFEMP2, while FZD7 acts as a receptor for C. difficile toxin TcdB in the colonic epithelium and interacts with it.|
| [ATP6V0B](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V0B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DPH2](https://www.ncbi.nlm.nih.gov/gene?term=DPH2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.9</br>**STRING score:** 516</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There are currently no known functional associations between ATP6V0B and DPH2. ATP6V0B is involved in proton transport and pH maintenance, while DPH2 is necessary for diphthamide biosynthesis, a post-translational modification of histidine. The shortest path in the BioGRID protein interaction network between these two genes involves intermediary genes involved in RNA processing and protein synthesis.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IKBKB](https://www.ncbi.nlm.nih.gov/gene?term=IKBKB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.8</br>**STRING score:** 365</br>**BioGRID count:** 3</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between AURKA and IKBKB could be in the regulation of cell cycle progression and the activation of the NF-kappa-B signaling pathway respectively, as both genes are involved in phosphorylation of key proteins in these pathways. However, there is no direct known interaction between AURKA and IKBKB in the literature.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPARG](https://www.ncbi.nlm.nih.gov/gene?term=PPARG[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 27.1</br>**STRING score:** 377</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between PIN1 and PPARG could include regulation of cell transformation and transcriptional activity. PIN1 is involved in isomerization of phosphorylated proteins and acts as a molecular switch in cellular processes, while PPARG is a nuclear receptor that interacts with coactivators to increase transcription of target genes. It is possible that PIN1 may regulate PPARG activity or vice versa in certain cellular processes.|
| [CASP10](https://www.ncbi.nlm.nih.gov/gene?term=CASP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 26.9</br>**STRING score:** 255</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: None were stated in the context or suggested by their known functions.  Specific functions: CASP10 is a protease involved in apoptosis, interacting with FADD and CASP8 in the Fas-signaling complex. GAPDH is a glycolytic enzyme and nitrosylase involved in cytoskeleton organization, innate immunity, and nuclear events such as transcription and RNA transport. It interacts with TRAF2 and TRAF3 for NF-kappa-B activation and type I interferon production.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 26.8</br>**STRING score:** 451</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | PIN1 and SMC4 are involved in different biological processes, but the shortest path between them in the BioGRID protein interaction network suggests a potential functional association. PIN1 is a peptidyl-prolyl cis/trans isomerase that regulates mitosis, transactivates oncogenes, and targets proteins for degradation, while SMC4 is a central component of the condensin complex that converts interphase chromatin into mitotic-like condensed chromosomes. The potential association between these two genes may suggest a regulatory role of PIN1 in mitotic chromosome condensation through interaction with the condensin complex.|
