| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [MLLT11](https://www.ncbi.nlm.nih.gov/gene?term=MLLT11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: The shortest path interaction between MLLT11 and PACSIN3 is through HSPA8, which is involved in chaperone-mediated autophagy. MLLT11 is a cofactor for the transcription factor TCF7 and involved in regulation of lymphoid development, while PACSIN3 plays a role in endocytosis and regulates internalization of plasma membrane proteins. There is no clear biological process that directly links the two genes.|
| [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between EBNA1BP2 and PSME1 may involve their roles in protein processing and antigen presentation. EBNA1BP2 is required for pre-rRNA processing and interacts with EBV and WDR46, while PSME1 is involved in immunoproteasome assembly and generation of class I binding peptides. The shortest path between the two genes in the BioGRID protein interaction network involves several other proteins, suggesting potential regulatory mechanisms.|
| [DCTD](https://www.ncbi.nlm.nih.gov/gene?term=DCTD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between DCTD and TM9SF2 in specific biological terms include their involvement in nucleotide metabolism and transport. DCTD supplies nucleotide substrate for thymidylate synthetase, while TM9SF2 functions as a channel or small molecule transporter homohexamer. Additionally, TM9SF2 interacts with TFAM and is a component of the Clp complex.|
| [CENPO](https://www.ncbi.nlm.nih.gov/gene?term=CENPO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association between CENPO and PMPCB is unclear based on the limited information provided, as they are involved in separate biological processes: CENPO is involved in kinetochore assembly and chromosome segregation during mitosis, while PMPCB is the catalytic subunit of the mitochondrial processing protease involved in protein maturation in the mitochondrial matrix.|
| [SPDEF](https://www.ncbi.nlm.nih.gov/gene?term=SPDEF[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UGDH](https://www.ncbi.nlm.nih.gov/gene?term=UGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between SPDEF and UGDH is involved in the biosynthesis of glycosaminoglycans, including chondroitin sulfate and heparan sulfate, which are essential components of extracellular matrix in various tissues including the brain. SPDEF may play a regulatory role in UGDH expression, and their interaction may contribute to proper brain and neuronal development.|
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between COG7 and PIK3R3 may be involved in Golgi function and insulin signaling. COG7 is a component of the Golgi complex required for normal morphology and localization while PIK3R3 binds to activated protein-tyrosine kinases and regulates their kinase activity, including in insulin stimulation. Interaction between these two genes could contribute to the proper functioning of these processes.|
| [HDGFRP3](https://www.ncbi.nlm.nih.gov/gene?term=HDGFRP3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 638 (min len 4) | Possible functional associations between HDGFRP3 and PIN1 involve regulation of cell proliferation, DNA synthesis, and mitosis. HDGFRP3 enhances DNA synthesis and may play a role in cell proliferation, while PIN1 regulates mitosis and induces centrosome amplification and cell transformation. The shortest path between the two genes in the BioGRID protein interaction network suggests that they may interact indirectly through XRCC6, a DNA repair protein involved in double-strand break repair, and TERF2IP, a telomere-binding protein involved in telomere maintenance.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | ORC1 and PMPCB are not directly associated in biological processes. ORC1 is involved in DNA replication, whereas PMPCB is involved in mitochondrial protein processing. The shortest path in the BioGRID interaction network reflects physical interactions that are not necessarily biologically relevant.|
| [PIN1](https://www.ncbi.nlm.nih.gov/gene?term=PIN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCEA2](https://www.ncbi.nlm.nih.gov/gene?term=TCEA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.2</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 36 (min len 3) | PIN1 and TCEA2 may have functional associations in RNA polymerase II transcription. PIN1 is involved in regulating mitosis, transactivation of oncogenes, and the ubiquitination and proteasomal degradation of RBBP8/CtIP. TCEA2 is necessary for efficient RNA polymerase II transcription elongation. The shortest path between these genes in the BioGRID protein interaction network includes several proteins involved in RNA metabolism and processing, suggesting a possible functional relationship in this biological process.|
| [ECD](https://www.ncbi.nlm.nih.gov/gene?term=ECD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT5B](https://www.ncbi.nlm.nih.gov/gene?term=STAT5B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is currently no known functional association between ECD and STAT5B in specific biological terms. ECD is involved in regulation of p53 and cell cycle progression, while STAT5B mediates cellular responses to growth factors and activates transcription.|
| [DAXX](https://www.ncbi.nlm.nih.gov/gene?term=DAXX[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETS1](https://www.ncbi.nlm.nih.gov/gene?term=ETS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 32.2</br>**STRING score:** 987</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between DAXX and ETS1 could involve regulating transcription, as DAXX represses transcriptional potential and inhibits transcriptional activation of ETS1 through direct protein-protein interactions. Additionally, DAXX may influence TNFRSF6-dependent apoptosis and modulate PAX5 activity. ETS1 acts as a dominant negative for isoform c-ETS-1A and binds DNA as a homodimer, with homodimerization required for transcription activation.|
| [HSPD1](https://www.ncbi.nlm.nih.gov/gene?term=HSPD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSME1](https://www.ncbi.nlm.nih.gov/gene?term=PSME1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.2</br>**STRING score:** 201</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 60 (min len 3) | Possible functional association between HSPD1 and PSME1 is their involvement in protein quality control. HSPD1 is a chaperonin that facilitates the proper folding of proteins, while PSME1 is implicated in immunoproteasome assembly and efficient antigen processing. This suggests a potential role for HSPD1 in assisting PSME1 in generating class I binding peptides by providing correctly folded substrate proteins to the proteasome.|
| [PRKACA](https://www.ncbi.nlm.nih.gov/gene?term=PRKACA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.1</br>**STRING score:** 212</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between PRKACA and TGFBR2 may involve the regulation of phosphorylation events and downstream signaling pathways. PRKACA is a catalytic subunit of cAMP-dependent protein kinase, which interacts with TGFBR2 to potentially regulate TGFB1 signaling and SMAD2/3 activation.|
| [NFKB2](https://www.ncbi.nlm.nih.gov/gene?term=NFKB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PMAIP1](https://www.ncbi.nlm.nih.gov/gene?term=PMAIP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 31.0</br>**STRING score:** 218</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is a possible functional association between NFKB2 and PMAIP1 in apoptosis. NFKB2 is a transcription factor involved in immune response, inflammation, and apoptosis, while PMAIP1 promotes activation of caspases and apoptosis and contributes to p53-dependent apoptosis.|
| [PMPCB](https://www.ncbi.nlm.nih.gov/gene?term=PMPCB[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.6</br>**STRING score:** 584</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | PMPCB and YME1L1 are both involved in mitochondrial protein processing and degradation. PMPCB is a catalytic subunit of the mitochondrial processing protease (MPP) while YME1L1 is an ATP-dependent metalloprotease involved in regulating mitochondrial morphology and function. There is a possible functional association between both genes through their involvement in mitochondrial protein turnover and cristae morphology.|
| [ORC1](https://www.ncbi.nlm.nih.gov/gene?term=ORC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB8](https://www.ncbi.nlm.nih.gov/gene?term=PSMB8[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 30.3</br>**STRING score:** 902</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is no clear functional association between ORC1 and PSMB8 based on the provided context. ORC1 is involved in mitochondrial transport and the urea cycle, while PSMB8 is a proteasome subunit involved in antigen processing and apoptosis regulation. The shortest path between them in the BioGRID protein interaction network involves an intermediate protein, HNRNPM.|
| [CCND3](https://www.ncbi.nlm.nih.gov/gene?term=CCND3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETS1](https://www.ncbi.nlm.nih.gov/gene?term=ETS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.9</br>**STRING score:** 172</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | The possible functional association between CCND3 and ETS1 is the regulation of the cell cycle during G1/S transition. CCND3 is a regulatory component of the cyclin D3-CDK4 complex that phosphorylates and inhibits members of the retinoblastoma protein family, while ETS1 acts as a dominant-negative for isoform c-ETS-1A and binds DNA as a homodimer.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPS5](https://www.ncbi.nlm.nih.gov/gene?term=RPS5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between AURKA and RPS5 could be that AURKA plays a critical role in regulation of cell cycle progression and spindle assembly, while RPS5 is a ribosomal protein important for protein synthesis. They may both be involved in processes related to cell proliferation and growth. However, it is unclear from the provided information if there are any direct interactions or regulatory relationships between these two genes.|
| [SMC3](https://www.ncbi.nlm.nih.gov/gene?term=SMC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP16](https://www.ncbi.nlm.nih.gov/gene?term=USP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.6</br>**STRING score:** 265</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between SMC3 and USP16 could involve epigenetic regulation of gene expression, as USP16 deubiquitinates histone H2A, which is a tag for transcriptional repression. SMC3 is a central component of the cohesin complex involved in chromosome cohesion, DNA repair, and spindle pole assembly during mitosis. It is possible that USP16 could regulate SMC3's function in these processes.|
| [CDK7](https://www.ncbi.nlm.nih.gov/gene?term=CDK7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ETS1](https://www.ncbi.nlm.nih.gov/gene?term=ETS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.5</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 14 (min len 3) | Possible functional association: CDK7 is a kinase involved in RNA polymerase II-mediated transcription and activation of cyclin-dependent kinases (CDKs), while ETS1 is a transcription factor that binds DNA as a homodimer. They interact directly via the AR protein and are both involved in cell cycle regulation, suggesting they may play a role together in controlling gene expression during the cell cycle in gastric cancer.|
| [CCNA2](https://www.ncbi.nlm.nih.gov/gene?term=CCNA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGA2](https://www.ncbi.nlm.nih.gov/gene?term=HMGA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.4</br>**STRING score:** 427</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between these two genes are their involvement in cell cycle regulation through CCNA2 and HMGA2's role as a transcriptional regulator. Additionally, they interact with common proteins such as TP53 and RB1, suggesting they may have related functions in tumorigenesis.|
| [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.4</br>**STRING score:** 282</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | PTK2 and TGFBR2 have a known protein interaction with a shortest path of PTK2->STAT1->TGFBR2 in TCGA-STAD cancer. In terms of function, PTK2 is involved in integrin signaling, cell migration, and proliferation, while TGFBR2 is involved in transforming growth factor beta (TGFB) signaling, cell differentiation, and immunoregulation. There is a possible functional association in cancer development and progression, as both genes are involved in cell migration and proliferation processes.|
| [USP16](https://www.ncbi.nlm.nih.gov/gene?term=USP16[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.3</br>**STRING score:** 314</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | There is no apparent functional association between USP16 and YME1L1 based on their known biological roles. USP16 is involved in histone deubiquitination and regulation of gene expression, while YME1L1 is an ATP-dependent metalloprotease that plays a critical role in mitochondrial function and cell proliferation.|
| [MBNL2](https://www.ncbi.nlm.nih.gov/gene?term=MBNL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF2](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.2</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There may not be a direct functional association between MBNL2 and TM9SF2, as their known actions are in different cellular processes - pre-mRNA alternative splicing regulation and intracellular transport/channel function, respectively. However, their interaction through PARK2 in the BioGRID protein interaction network suggests a possible indirect association.|
| [AURKA](https://www.ncbi.nlm.nih.gov/gene?term=AURKA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.1</br>**STRING score:** 426</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between AURKA and RALA include their interaction through the BioGRID protein interaction network and their roles in cellular processes such as cell cycle progression, cell proliferation, and membrane trafficking. AURKA is a mitotic kinase involved in various mitotic events, while RALA is a multifunctional GTPase involved in a variety of cellular processes including gene expression, cell migration, and cell proliferation.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLE2](https://www.ncbi.nlm.nih.gov/gene?term=POLE2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 29.0</br>**STRING score:** 615</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between BIRC5 and POLE2 include a role in chromosomal DNA replication and DNA repair, as both genes are involved in these processes. BIRC5 is a regulator of the localization of the chromosome passage protein complex (CPC) during mitosis and cytokinesis, while POLE2 is an accessory component of the DNA polymerase epsilon complex that participates in DNA repair and replication.|
| [SMC4](https://www.ncbi.nlm.nih.gov/gene?term=SMC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCERG1](https://www.ncbi.nlm.nih.gov/gene?term=TCERG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.8</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional association:  SMC4 is involved in chromosome condensation during mitosis, while TCERG1 is a transcription factor that inhibits elongation of transcripts. They may have a role in regulating gene expression during mitosis. The BioGRID protein interaction network shows possible interaction paths that include MYC and RPA, which have also been implicated in regulation of gene expression.|
| [SSBP2](https://www.ncbi.nlm.nih.gov/gene?term=SSBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between SSBP2 and STAT3 could be their involvement in the ERK signaling pathway and survival regulation in cancer. SSBP2 inhibits ERK dephosphorylation while STAT3 mediates cellular responses to growth factors and regulates inflammatory responses. There is no direct evidence of an interaction between these two genes, but they may play complementary roles in cancer progression.|
| [CDK19](https://www.ncbi.nlm.nih.gov/gene?term=CDK19[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 28.5</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on their interaction in the BioGRID protein network, CDK19 and KDM5A may be functionally associated in regulating gene transcription. CDK19 is a nucleolar protein that connects RNA polymerase I with enzymes responsible for ribosomal processing and modification, while KDM5A specifically demethylates 'Lys-4' of histone H3, playing a central role in histone code and transcriptional regulation.|
