| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BUB1B](https://www.ncbi.nlm.nih.gov/gene?term=BUB1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.7</br>**STRING score:** 191</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between BUB1B and TMEM97 based on their known biology and protein interaction network. BUB1B is involved in mitotic checkpoint regulation and tumor suppression, while TMEM97 is an intracellular receptor involved in regulating cell survival and cholesterol homeostasis.|
| [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NSA2](https://www.ncbi.nlm.nih.gov/gene?term=NSA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association between MAP2K5 and NSA2 is involved in cell survival and development through the MAPK signaling pathway and ribosomal biogenesis, respectively. The shortest path between these two proteins in the BioGRID network suggests they may be involved in regulating protein translation and synthesis.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDGFA](https://www.ncbi.nlm.nih.gov/gene?term=PDGFA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: COPS5 is a protease subunit of the COP9 signalosome complex while PDGFA is a growth factor involved in cell proliferation and migration. There are no known direct functional associations between these two genes in specific biological terms based on the provided context.|
| [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST7](https://www.ncbi.nlm.nih.gov/gene?term=ST7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 391 (min len 4) | Possible functional associations between DENND2D and ST7 remain unclear based on their known functions and interactions in the context of TCGA-STAD. DENND2D is a guanine nucleotide exchange factor that activates Rab GTPases, while ST7 may act as a tumor suppressor. It is possible that these genes may interact indirectly through common downstream pathways involved in cancer progression.|
| [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional association: TRAPPC3 is a component of the transport protein particle (TRAPP) complex, which is involved in vesicular transport and may interact with KIF20A to transport Golgi membranes and associated vesicles along microtubules. KIF20A, a mitotic kinesin, is required for CPC-mediated cytokinesis and has microtubule plus end-directed motility, which may be involved in transporting TRAPP complexes.|
| [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | TM9SF3 is involved in DNA methylation and transcriptional corepression, while TMEM5 participates in the biosynthesis of O-mannosyl glycan. These genes are not functionally associated based on the provided information, as they are involved in distinct molecular processes.|
| [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.2</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | RAB4A is involved in vesicular traffic and mediates VEGFR2 endosomal trafficking, while STX1A plays an essential role in hormone and neurotransmitter exocytosis. Based on their roles, they may be functionally associated in facilitating the transport and release of specific molecules, such as hormones or neurotransmitters, through coordinated vesicular trafficking and exocytosis processes.|
| [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 102 (min len 3) | Possible functional associations between GRWD1 and HNRNPA2B1 could include regulation of chromatin dynamics and minichromosome maintenance at replication origins, as well as transport of HIV-1 genomic RNA out of the nucleus to the cytoplasm. However, no direct functional association has been reported between these two genes in the provided context.|
| [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZDHHC6](https://www.ncbi.nlm.nih.gov/gene?term=ZDHHC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between PRUNE and ZDHHC6 include involvement in cell proliferation, migration, and differentiation, as well as regulation of neurogenesis and microtubule polymerization. Given that both genes interact with various cellular structures and proteins, it is possible that they may have shared roles in these processes.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MEST](https://www.ncbi.nlm.nih.gov/gene?term=MEST[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between EPN2 and MEST according to the provided context. EPN2 plays a role in endocytosis and interacts with clathrin and AP-2, while MEST is involved in chromosomal DNA replication and is a core component of the CDC45-MCM-GINS helicase.|
| [HMGA2](https://www.ncbi.nlm.nih.gov/gene?term=HMGA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.2</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 195 (min len 4) | There may not be a specific functional association between HMGA2 and PTPRC based on their known roles and the provided context.  HMGA2 functions as a transcriptional regulator and has a role in chromosome condensation during meiosis and postnatal myogenesis. PTPRC is a receptor involved in T-cell signaling and has a role in mediating binding of human cytomegalovirus protein UL11 to T-cells.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.0</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: SLC25A46 regulates mitochondrial organization and could potentially interact with CCNH, which regulates cell cycle control and RNA transcription. It is possible that CCNH plays a role in regulating mitochondrial organization through its interaction with SLC25A46. Further studies are needed to determine the extent of their functional association.|
| [BDH1](https://www.ncbi.nlm.nih.gov/gene?term=BDH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.8</br>**STRING score:** 250</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between BDH1 and GRWD1 could involve regulation of chromatin dynamics, minichromosome maintenance, and origin binding during replication. BDH1 plays a role in controlling the cell cycle at the late G1 and early S phase, while GRWD1 binds to histone complexes and interacts with METTL18, CDT1, and CDC6 in promoting chromatin openness during replication.|
| [DUSP11](https://www.ncbi.nlm.nih.gov/gene?term=DUSP11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.4</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between DUSP11 and GAPDH are unclear based on the provided context and their known biological functions. DUSP11 possesses RNA phosphatase activities and may participate in nuclear mRNA metabolism, while GAPDH plays a role in glycolysis, nuclear functions, and innate immunity. As they seem to be involved in different biological processes, any functional associations between them would need to be further investigated.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.4</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | There are no known direct functional associations between ELAVL1 and RNMT. ELAVL1 is an RNA-binding protein that stabilizes mRNA, while RNMT is the catalytic subunit of a mRNA-capping methyltransferase complex. However, both genes have been implicated in post-transcriptional regulation of gene expression.|
| [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: Both TMEM97 and TOP2A are involved in regulating cell proliferation and differentiation in cancer cells. TMEM97 may also play a role in cellular cholesterol homeostasis and alter cytochrome P450 activity, while TOP2A is a key decatenating enzyme that alters DNA topology and interacts with various proteins for its function. There is no clear biological process linking these two genes.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.5</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Both CDK1 and TRIB1 are involved in regulating cell cycle and differentiation processes. There is a shortest path between them in the BioGRID protein interaction network through MYC. It is possible that CDK1 and TRIB1 may interact to regulate cell cycle and differentiation processes through the MYC pathway.|
| [FGFR2](https://www.ncbi.nlm.nih.gov/gene?term=FGFR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAT6B](https://www.ncbi.nlm.nih.gov/gene?term=KAT6B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.3</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 243 (min len 4) | FGFR2 and KAT6B may be functionally associated in regulating transcription, as KAT6B is a histone acetyltransferase that may be involved in positive and negative transcriptional regulation, and FGFR2 plays an essential role in the regulation of cell proliferation, differentiation, and apoptosis. The shortest path between these genes in the BioGRID protein interaction network suggests that they may interact through the protein KIAA1429.|
| [IL1B](https://www.ncbi.nlm.nih.gov/gene?term=IL1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIPA](https://www.ncbi.nlm.nih.gov/gene?term=LIPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: IL1B is a pro-inflammatory cytokine involved in inducing inflammation and promoting Th17 differentiation of T-cells, while LIPA catalyzes the deacylation of lipids. It is possible that IL1B could induce lipolysis through LIPA to provide energy for immune cells during inflammation.|
| [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [INSIG1](https://www.ncbi.nlm.nih.gov/gene?term=INSIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.8</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association: There is currently no evidence suggesting a direct functional association between DNAJB6 and INSIG1 in TCGA-STAD.   DNAJB6: Isoform B inhibits aggregation of huntingtin, and interacts with HDACs to require antiaggregation activity.   INSIG1: Acts as a negative regulator of cholesterol biosynthesis by mediating retention of the SCAP-SREBP complex in the endoplasmic reticulum, thereby blocking SREBP processing and translocation to the nucleus. Additionally, INSIG1 initiates the sterol-mediated ubiquitin-mediated endoplasmic reticulum-associated degradation (ERAD) of HMGCR via recruitment of ubiquitin ligases AMFR/gp78 and/or RNF139, and initiates the ubiquitin-mediated degradation of SOAT2/ACAT2 via recruitment of AMFR/gp78 when lipid levels are low.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD4](https://www.ncbi.nlm.nih.gov/gene?term=PSMD4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.7</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between PACSIN3 and PSMD4 may involve a role in endocytosis and protein degradation, respectively. These processes are important for maintaining cellular homeostasis, suggesting that these genes may work together to regulate protein turnover within cells.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.7</br>**STRING score:** 999</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the BioGRID protein interaction network and context, there is a direct interaction between RFC2 and RFC5. Both genes are involved in DNA replication as part of the RFC complex, which is required for the elongation of primed DNA templates during DNA synthesis. It is possible that the interaction between RFC2 and RFC5 is necessary for the formation and activity of the RFC complex in DNA replication.|
| [CASP10](https://www.ncbi.nlm.nih.gov/gene?term=CASP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL1B](https://www.ncbi.nlm.nih.gov/gene?term=IL1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.5</br>**STRING score:** 424</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Both CASP10 and IL1B are involved in inflammation and immune responses. There may be a functional association between CASP10 and IL1B in the inflammasome, as CASP10 can be activated by inflammasome complexes and IL1B is a pro-inflammatory cytokine that is released through the inflammasome pathway.|
| [CALM3](https://www.ncbi.nlm.nih.gov/gene?term=CALM3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.3</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CALM3 and COL1A1 include involvement in cell signaling and extracellular matrix production, considering their interactions with proteins such as EGFR, DDR2, and MRC2. Additionally, COL1A1 is known to be involved in type I collagen formation, while CALM3 plays a role in calcium signaling.|
