| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BUB1B](https://www.ncbi.nlm.nih.gov/gene?term=BUB1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.7</br>**STRING score:** 191</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear biological link between the two genes BUB1B and TMEM97, as they perform completely distinct functions. BUB1B is involved in the mitotic checkpoint, ensuring proper chromosome attachment during mitosis, while TMEM97 is an orphan receptor that regulates cellular cholesterol homeostasis and plays a role in drug binding.|
| [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NSA2](https://www.ncbi.nlm.nih.gov/gene?term=NSA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 45.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between MAP2K5 and NSA2 include involvement in stress-induced apoptosis, neuronal survival, cardiac development, and angiogenesis via the MAP3K2/MAP3K3-MAP3K5-MAPK7 signaling complex, and in biogenesis and quality control of the 60S ribosomal subunit. However, there is no direct biological evidence suggesting a functional association between these two genes.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PDGFA](https://www.ncbi.nlm.nih.gov/gene?term=PDGFA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: PDGFA is a growth factor involved in cell proliferation and migration, while COPS5 is a protease subunit involved in regulation of the ubiquitin conjugation pathway. There is no known direct functional association between these two genes.|
| [DENND2D](https://www.ncbi.nlm.nih.gov/gene?term=DENND2D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST7](https://www.ncbi.nlm.nih.gov/gene?term=ST7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 391 (min len 4) | Possible functional associations between DENND2D and ST7 include involvement in signal transduction and potential roles as tumor suppressors. DENND2D may activate Rab proteins and ST7 may act as a receptor for lipophilic molecules. There is currently no known direct interaction between the two genes.|
| [KIF20A](https://www.ncbi.nlm.nih.gov/gene?term=KIF20A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between KIF20A and TRAPPC3 may involve vesicular transport of Golgi membranes and associated vesicles along microtubules, as well as regulation of cytokinesis and chromosome segregation during mitosis. KIF20A is involved in these processes as a mitotic kinesin, while TRAPPC3 is a component of the TRAPP complex involved in vesicular transport from endoplasmic reticulum to Golgi.|
| [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM5](https://www.ncbi.nlm.nih.gov/gene?term=TMEM5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TM9SF3 is involved in DNA methylation and imprinting, while TMEM5 participates in the biosynthesis of the phosphorylated O-mannosyl trisaccharide required for binding laminin G-like domain-containing extracellular proteins with high affinity. It's possible that TM9SF3 may affect the expression of genes involved in laminin binding through its role in DNA methylation and imprinting, potentially impacting TMEM5's function indirectly.|
| [RAB4A](https://www.ncbi.nlm.nih.gov/gene?term=RAB4A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STX1A](https://www.ncbi.nlm.nih.gov/gene?term=STX1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.2</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 22 (min len 3) | Possible functional associations between RAB4A and STX1A are related to vesicular traffic and exocytosis. RAB4A is involved in protein transport and VEGFR2 endosomal trafficking, while STX1A plays an essential role in hormone and neurotransmitter exocytosis and endocytosis. Therefore, RAB4A may regulate the transport of STX1A-containing vesicles, or STX1A may be involved in the exocytosis of RAB4A-regulated proteins.|
| [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HNRNPA2B1](https://www.ncbi.nlm.nih.gov/gene?term=HNRNPA2B1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.1</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 102 (min len 3) | Possible functional associations between GRWD1 and HNRNPA2B1 could be in chromatin regulation and RNA transport, respectively. GRWD1 interacts with histones and regulates chromatin dynamics, while HNRNPA2B1 is involved in transporting HIV-1 genomic RNA and interacts with proteins involved in RNA processing and transport. However, no direct interactions between these two genes have been reported in the literature.|
| [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZDHHC6](https://www.ncbi.nlm.nih.gov/gene?term=ZDHHC6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between PRUNE and ZDHHC6 could be in the regulation of neurogenesis and microtubule polymerization. PRUNE's negative regulation of NME1 and interaction with tubulin may point to a role in cytoskeletal organization, potentially linking it with ZDHHC6's function as a palmitoyl transferase that mediates protein localization to the peripheral endoplasmic reticulum.|
| [EPN2](https://www.ncbi.nlm.nih.gov/gene?term=EPN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MEST](https://www.ncbi.nlm.nih.gov/gene?term=MEST[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between EPN2 and MEST based on literature and the protein interaction network. EPN2 is involved in endocytosis and binds to proteins such as EPS15 and ITSN1, while MEST is involved in chromosomal DNA replication as part of the CMG helicase complex.|
| [HMGA2](https://www.ncbi.nlm.nih.gov/gene?term=HMGA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.2</br>**STRING score:** 206</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 195 (min len 4) | There is no clear functional association between HMGA2 and PTPRC based on their known biological roles. HMGA2 is a transcriptional regulator involved in cell cycle regulation, while PTPRC acts as a receptor for viral proteins and moderates T-cell signaling. The observed protein-protein interactions between these genes in the BioGRID network may not have a physiological relevance.|
| [CCNH](https://www.ncbi.nlm.nih.gov/gene?term=CCNH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A46](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A46[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 38.0</br>**STRING score:** 179</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between CCNH and SLC25A46 is their involvement in regulation of cell cycle control and mitochondrial organization, respectively. However, based on the limited information given, it is difficult to reason about any direct interaction between these genes at a molecular level.|
| [BDH1](https://www.ncbi.nlm.nih.gov/gene?term=BDH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GRWD1](https://www.ncbi.nlm.nih.gov/gene?term=GRWD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.8</br>**STRING score:** 250</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between BDH1 and GRWD1 include regulation of chromatin dynamics, minichromosome maintenance loading at replication origins, and control of the cell cycle at the late G1 and early S phase. There may be potential interactions between the two genes in these processes.|
| [DUSP11](https://www.ncbi.nlm.nih.gov/gene?term=DUSP11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GAPDH](https://www.ncbi.nlm.nih.gov/gene?term=GAPDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 36.4</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional associations between DUSP11 and GAPDH may involve regulation of mRNA metabolism, as DUSP11 participates in nuclear mRNA metabolism and GAPDH has been shown to play a role in RNA transport and translation inhibition. Additionally, GAPDH has been shown to interact with proteins involved in innate immunity, suggesting a potential connection between these genes in immune response pathways.|
| [ELAVL1](https://www.ncbi.nlm.nih.gov/gene?term=ELAVL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RNMT](https://www.ncbi.nlm.nih.gov/gene?term=RNMT[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 35.4</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 39 (min len 3) | Possible functional associations between ELAVL1 and RNMT are yet unclear. ELAVL1 is an RNA-binding protein that increases stability by binding to the 3'-UTR region of mRNAs, while RNMT is a catalytic subunit of the mRNA-capping methyltransferase complex that methylates the N7 position of the added guanosine to the 5'-cap structure of mRNAs.|
| [TMEM97](https://www.ncbi.nlm.nih.gov/gene?term=TMEM97[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOP2A](https://www.ncbi.nlm.nih.gov/gene?term=TOP2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.7</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: TMEM97 is a sigma-2 receptor that regulates cell survival and differentiation, and may alter the activity of some cytochrome P450 proteins. TOP2A is a key enzyme that regulates DNA topology by binding to two double-stranded DNA molecules, generating a double-stranded break, and re-ligating the broken strand. There is no clear biological process that directly links the functions of TMEM97 and TOP2A.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.5</br>**STRING score:** 171</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between CDK1 and TRIB1 is that TRIB1 alters COP1 substrate specificity by directing the activity of COP1 toward CEBPA. CDK1 is involved in cell cycle regulation and can be inhibited by severe fever with thrombocytopenia syndrome virus (SFTSV) NSs, leading to cell cycle arrest.|
| [FGFR2](https://www.ncbi.nlm.nih.gov/gene?term=FGFR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAT6B](https://www.ncbi.nlm.nih.gov/gene?term=KAT6B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 34.3</br>**STRING score:** 189</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 243 (min len 4) | FGFR2 and KAT6B are functionally associated as they both play a role in regulating cell differentiation. FGFR2 is a receptor for fibroblast growth factors and is required for embryonic development, osteoblast differentiation, and proliferation. KAT6B is a histone acetyltransferase involved in transcriptional regulation and may be required for cerebral cortex development.|
| [IL1B](https://www.ncbi.nlm.nih.gov/gene?term=IL1B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LIPA](https://www.ncbi.nlm.nih.gov/gene?term=LIPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.9</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: IL1B is a pro-inflammatory cytokine that induces inflammation and angiogenesis, while LIPA catalyzes the deacylation of lipoproteins. These genes may be involved in the regulation of lipid metabolism and inflammation in cancer. The shortest path between IL1B and LIPA in the BioGRID protein interaction network includes ELAVL1, a RNA binding protein, suggesting a possible post-transcriptional interaction between IL1B and LIPA.|
| [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [INSIG1](https://www.ncbi.nlm.nih.gov/gene?term=INSIG1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.8</br>**STRING score:** 159</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | There is no clear functional association between DNAJB6 and INSIG1 based on the available information. DNAJB6 inhibits aggregation of huntingtin protein and interacts with histone deacetylases, while INSIG1 is involved in feedback regulation of cholesterol synthesis and degradation of HMGCR.|
| [PACSIN3](https://www.ncbi.nlm.nih.gov/gene?term=PACSIN3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD4](https://www.ncbi.nlm.nih.gov/gene?term=PSMD4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.7</br>**STRING score:** 169</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between PACSIN3 and PSMD4 include regulation of protein homeostasis and degradation through the proteasome complex, involvement in endocytosis and internalization of plasma membrane proteins, and potential interactions with other proteins in these processes.|
| [RFC2](https://www.ncbi.nlm.nih.gov/gene?term=RFC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RFC5](https://www.ncbi.nlm.nih.gov/gene?term=RFC5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.7</br>**STRING score:** 999</br>**BioGRID count:** 7</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between RFC2 and RFC5 is that they both belong to the RFC complex, which is involved in the elongation of primed DNA templates by DNA polymerase delta and epsilon, requiring the action of the accessory proteins proliferating cell nuclear antigen (PCNA) and activator 1. RFC2 also interacts with PRKAR1A, which may be involved in cell survival.|
| [CASP10](https://www.ncbi.nlm.nih.gov/gene?term=CASP10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IL1B](https://www.ncbi.nlm.nih.gov/gene?term=IL1B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.5</br>**STRING score:** 424</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is evidence of a possible functional association between CASP10 and IL1B. CASP10 is involved in the Fas signaling complex along with CASP8, while IL1B is a pro-inflammatory cytokine that promotes T-cell activation and cytokine production. The bioGRID protein interaction network also shows a shortest path between these two genes through CASP8, suggesting they may have overlapping roles in certain biological processes.|
| [CALM3](https://www.ncbi.nlm.nih.gov/gene?term=CALM3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 33.3</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CALM3 and COL1A1 genes are their involvement in extracellular matrix (ECM) assembly and remodeling. CALM3 codes for calmodulin, a calcium-binding protein that regulates various processes including ECM remodeling, while COL1A1 codes for alpha 1 chain of type I collagen, a major component of ECM. The shortest path in the protein interaction network suggests that CALM3 and COL1A1 may interact through EGFR, CDC42, DDR2, or ELAVL1 proteins in ECM signaling or remodeling.|
