| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLOD3](https://www.ncbi.nlm.nih.gov/gene?term=PLOD3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 125 (min len 4) | Possible functional associations:      The shortest path in the BioGRID protein interaction network suggests a possible direct physical interaction between these two genes. ATP11B encodes a P4-ATPase flippase complex and PLOD3 encodes a multifunctional enzyme that catalyzes post-translational modifications on Lys residues in procollagen. It is possible that PLOD3 may modify procollagen proteins whose translocation is facilitated by ATP11B.|
| [CDH3](https://www.ncbi.nlm.nih.gov/gene?term=CDH3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMCO6](https://www.ncbi.nlm.nih.gov/gene?term=TMCO6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | A possible functional association between CDH3 and TMCO6 is that CDH3 may contribute to the sorting of heterogeneous cell types, potentially impacting the composition of cells with TMCO6 expression. However, based on their biological functions described in the context, there is no clear indication of a direct functional relationship between these two genes.|
| [STK25](https://www.ncbi.nlm.nih.gov/gene?term=STK25[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: STK25 and TOPBP1 may be involved in DNA replication and checkpoint control, possibly through their interaction with each other and other proteins in the network.  STK25 is an oxidant stress-activated kinase that regulates protein transport events, cell adhesion, and polarity complexes involved in cell migration. TOPBP1 is required for DNA replication and plays a role in the rescue of stalled replication forks and checkpoint control. Their interaction in the BioGRID protein interaction network may suggest a functional association in DNA replication and checkpoint control processes.|
| [ID2](https://www.ncbi.nlm.nih.gov/gene?term=ID2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPOX](https://www.ncbi.nlm.nih.gov/gene?term=PPOX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 325 (min len 4) | Possible functional associations between ID2 and PPOX are unclear based solely on the protein interaction network and provided context. ID2 is a transcriptional regulator involved in cellular processes including growth, differentiation, and neoplastic transformation. PPOX catalyzes the oxidation of protoporphyrinogen-IX to protoporphyrin-IX.|
| [CDH3](https://www.ncbi.nlm.nih.gov/gene?term=CDH3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPL39L](https://www.ncbi.nlm.nih.gov/gene?term=RPL39L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 4) | There is no clear functional association between CDH3 and RPL39L based on their known biological functions. CDH3 is a cell adhesion protein while RPL39L is a ribosomal protein required for the formation of sperm in male germ cells. The shortest path between them in the BioGRID protein interaction network involves CTNNB1 and NRAS.|
| [C2CD2L](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SEC11A](https://www.ncbi.nlm.nih.gov/gene?term=SEC11A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Based on their known functions, there is no obvious direct functional association between C2CD2L and SEC11A. C2CD2L is involved in lipid transport and calcium signaling, while SEC11A is a catalytic component of the signal peptidase complex. However, both genes are involved in processes taking place in the endoplasmic reticulum, suggesting a possible indirect functional association.|
| [ENOPH1](https://www.ncbi.nlm.nih.gov/gene?term=ENOPH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBKS](https://www.ncbi.nlm.nih.gov/gene?term=RBKS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 48 (min len 4) | Possible functional associations between ENOPH1 and RBKS could involve the pentose phosphate pathway, as RBKS is involved in ribose phosphorylation, a process linked to nucleotide synthesis and this pathway, and ENOPH1 is involved in the enolization of 2,3-diketo-5-methylthiopentyl-1-phosphate, an intermediate in this pathway. Both genes may be part of a metabolic pathway that generates energy.|
| [KAZALD1](https://www.ncbi.nlm.nih.gov/gene?term=KAZALD1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPOX](https://www.ncbi.nlm.nih.gov/gene?term=PPOX[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 33 (min len 4) | There is no clear biological functional association between KAZALD1 and PPOX based on their known functions. KAZALD1 is involved in bone formation while PPOX is involved in protoporphyrin-IX synthesis. However, they have some common protein interaction partners, such as HNRNPH1, PARK2, and EGFR, which may suggest a potential indirect interaction or shared regulatory pathway.|
| [MFSD10](https://www.ncbi.nlm.nih.gov/gene?term=MFSD10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPL39L](https://www.ncbi.nlm.nih.gov/gene?term=RPL39L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no clear functional associations between MFSD10 and RPL39L based on their known biological processes. MFSD10 is a transporter for some organic anions across renal proximal tubules while RPL39L is a component of the male germ cell-specific ribosome, specifically involved in the regulation of biosynthesis and folding of male germ-cell-specific proteins.|
| [PLOD3](https://www.ncbi.nlm.nih.gov/gene?term=PLOD3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPL39L](https://www.ncbi.nlm.nih.gov/gene?term=RPL39L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 4) | Possible functional associations:     There is currently no known direct functional association between PLOD3 and RPL39L. PLOD3 is involved in collagen biosynthesis and basement membrane formation, while RPL39L is a component of the ribosome in male germ cells and is essential for formation of sperm.|
| [CDKN2A](https://www.ncbi.nlm.nih.gov/gene?term=CDKN2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 43.9</br>**STRING score:** 205</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: CDKN2A interacts with MYC, a transcription factor that is also a known oncogene, and HIF1A, a factor involved in hypoxia-inducible gene expression. USP22 is a member of the SAGA transcription coactivator-HAT complex and is recruited to specific gene promoters by activators such as MYC.  These findings suggest that CDKN2A and USP22 may cooperate in promoting cell cycle progression and gene expression in cancer cells, possibly through MYC-mediated pathways. However, further studies are needed to validate this hypothesis.|
| [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PUF60](https://www.ncbi.nlm.nih.gov/gene?term=PUF60[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 42.0</br>**STRING score:** 231</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | G3BP1 and PUF60 are involved in different biological processes but their interaction may play a role in regulating MYC expression and transcriptional regulation. There is currently no known direct functional association between these two genes.|
| [COG7](https://www.ncbi.nlm.nih.gov/gene?term=COG7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAPPC3](https://www.ncbi.nlm.nih.gov/gene?term=TRAPPC3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 41.9</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations: COG7 and TRAPPC3 are both involved in vesicular transport, with COG7 playing a role in Golgi function and TRAPPC3 being part of the TRAPP complex involved in transport from the ER to the Golgi. The shortest path between these genes in the BioGRID protein interaction network suggests they may interact in this process.|
| [BIRC5](https://www.ncbi.nlm.nih.gov/gene?term=BIRC5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FAT1](https://www.ncbi.nlm.nih.gov/gene?term=FAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.2</br>**STRING score:** 156</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | BIRC5 (also known as survivin) is involved in promoting cell proliferation, preventing apoptosis, and maintaining mitochondrial integrity, while FAT1 plays a role in cellular polarization, directed cell migration, and modulating cell-cell contact. There is no known direct functional association between these two genes at the moment.|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPL39L](https://www.ncbi.nlm.nih.gov/gene?term=RPL39L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 40.2</br>**STRING score:** 195</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 4) | There is no immediately apparent functional association between DUSP4 and RPL39L. DUSP4 regulates mitogenic signal transduction by dephosphorylating MAP kinases, while RPL39L is a component of the ribosome required for the formation of sperm and male fertility. The shortest path between the two genes in the BioGRID protein interaction network involves multiple intermediate proteins, suggesting they may not directly interact.|
| [ATP11B](https://www.ncbi.nlm.nih.gov/gene?term=ATP11B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GATA2](https://www.ncbi.nlm.nih.gov/gene?term=GATA2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.9</br>**STRING score:** 210</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 155 (min len 4) | One possible functional association between ATP11B and GATA2 could be related to membrane lipid asymmetry and endothelial cell gene expression, respectively. ATP11B is involved in the maintenance of membrane lipid asymmetry in endosome compartment while GATA2 is a transcriptional activator which regulates endothelin-1 gene expression in endothelial cells. There is no direct evidence for a physical interaction between these two genes but they may play complementary roles in regulating cell membrane and gene expression.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRAF2](https://www.ncbi.nlm.nih.gov/gene?term=PRAF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 39.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: HSPA8 and PRAF2 may be involved in ER/Golgi transport and vesicular traffic based on PRAF2's known function and HSPA8's role in mitochondrial import and ER-associated degradation (ERAD) quality control pathway. Additionally, HSPA8 participates in chaperone-mediated autophagy, which could potentially intersect with PRAF2's proapoptotic role in cerulenin-induced neuroblastoma apoptosis.|
