| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | There is currently no known functional association between LYN and TM9SF3. LYN is a tyrosine kinase involved in signaling pathways for immune responses, hematopoiesis, and integrin signaling, while TM9SF3 is essential for establishment of DNA methylation patterns during development and acts as a transcriptional corepressor.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations:      LIG1 is involved in DNA repair, replication, and recombination, while PIK3R3 regulates protein-tyrosine kinase activity and interacts with IRS-1. They may be involved in signaling pathways related to DNA damage response or insulin signaling.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IFNAR1](https://www.ncbi.nlm.nih.gov/gene?term=IFNAR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | HDAC2 and IFNAR1 are involved in the regulation of gene expression and the immune response, respectively. There may be a functional association between these genes through the HDAC NuRD complex, which participates in chromatin remodeling, and the downstream effect on transcription activation or repression of IFNAR1-related genes.|
| [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIM2](https://www.ncbi.nlm.nih.gov/gene?term=TRIM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between TRIB1 and TRIM2 may involve regulation of protein degradation and immune responses. TRIB1 is involved in protein degradation and regulation of myeloid cell differentiation, while TRIM2 plays a role in antiviral immunity and neuroprotection. The shortest path between these genes in the BioGRID protein interaction network involves intermediate proteins related to RNA processing, suggesting a potential regulatory role in RNA metabolism.|
| [ATP6V1D](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between ATP6V1D and HAT1 include regulation of intracellular pH, cilium biogenesis, and histone acetylation. However, there is no direct biological evidence linking these two genes together.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.7</br>**STRING score:** 170</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between CDC42 and MELK could involve regulation of cell proliferation and carcinogenesis, as both genes play a role in these processes. CDC42 also regulates spindle microtubules and cell migration, while MELK is involved in cell cycle regulation and self-renewal of stem cells. There is no known direct interaction between these two genes based on the provided protein interaction network.|
| [CALU](https://www.ncbi.nlm.nih.gov/gene?term=CALU[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between CALU and DYNLT3 is their involvement in intracellular transport. CALU interacts with GGCX to regulate carboxylation while DYNLT3 acts as a non-catalytic accessory component of dynein complex involved in cytoplasmic retrograde motility, and both genes are connected through protein-protein interaction in BioGRID network.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP18](https://www.ncbi.nlm.nih.gov/gene?term=UTP18[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | CDC42 is a plasma membrane-associated GTPase involved in cell migration, cell polarization, and regulation of the cytoskeleton. UTP18 is part of the SSU processome involved in ribosome biogenesis. Based on their roles, it is possible that CDC42 could regulate UTP18 in processes involving cytoskeletal changes during ribosome biogenesis. However, there is no direct evidence of functional association between these two genes.|
| [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is no clear functional association between TBC1D15 and TOPBP1. TBC1D15 acts as a GTPase activating protein for RAB7A, while TOPBP1 plays a role in DNA replication, checkpoint control, and E2F1-dependent apoptosis inhibition.|
| [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAM2](https://www.ncbi.nlm.nih.gov/gene?term=TRAM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: PTPN12 and TRAM2 may be involved in regulating cellular signaling cascades and collagen synthesis, respectively.   Based on the provided context, there is no clear indication of a direct functional association between PTPN12 and TRAM2. However, PTPN12 is involved in the dephosphorylation of cellular tyrosine kinases, including ERBB2 which is involved in signaling cascades, while TRAM2 is necessary for collagen type I synthesis and may be involved in the proper insertion of the first transmembrane helix of TM4SF20 into the ER lumen.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGN1](https://www.ncbi.nlm.nih.gov/gene?term=HMGN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.0</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | HDAC2 and HMGN1 both have roles in chromatin remodeling and transcriptional regulation. Given their interactions through multiple proteins, they may work together in forming repressor complexes and regulating histone deacetylase activity on various gene targets.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.1</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association between NRAS and TBC1D15 may be that NRAS regulates the GTPase activity of RAB7A, which is a substrate for TBC1D15. Both genes are involved in regulating intracellular vesicle trafficking and membrane dynamics.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is currently no known functional association between LYN and RBM34. LYN encodes a tyrosine kinase involved in immune responses, while RBM34 is involved in growth factor signaling and magnesium reabsorption in the kidney.|
| [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.9</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between PIK3R3 and PTPRF could be related to insulin signaling and regulation of protein-tyrosine kinases through their interaction with IRS-1 and GRIP1, respectively. These interactions suggest a potential role for these genes in controlling cell growth and differentiation processes involved in cancer.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.7</br>**STRING score:** 389</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 317 (min len 3) | CDC42 and NRAS both play roles in regulating cell migration, with CDC42 also involved in phagocytosis and dendritic spine structural plasticity and NRAS interacting with RGS14 and RASSF7. There may be a possible functional association between these two genes in regulating cell migration, but further studies would be needed to confirm this.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.5</br>**STRING score:** 972</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: The NRAS and SRC genes interact at the protein level in the shortest path of the BioGRID network. NRAS is involved in binding GDP/GTP and possesses intrinsic GTPase activity, while SRC is a non-receptor protein tyrosine kinase involved in neurite elongation and microbial infection. It is possible that these genes play a role in regulating each other's functions in these processes or potentially in other biological processes.|
| [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.3</br>**STRING score:** 226</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between NRIP1 and TBP include their involvement in transcriptional regulation. NRIP1 modulates activation and repression of steroid and nuclear hormone receptors, while TBP is a core component of the TFIID complex that initiates RNA polymerase II-dependent transcription. They both interact with common partners such as HDAC1. However, there is no clear evidence of a direct interaction or shared pathway between NRIP1 and TBP in the literature.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [S100A4](https://www.ncbi.nlm.nih.gov/gene?term=S100A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.9</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between NFKBIA and S100A4 in TCGA-UCEC may be related to immune response and cancer progression. NFKBIA inhibits NF-kappa-B/REL complexes and activates transcription upon cellular stimulation, while S100A4 is involved in various cellular processes including cell differentiation, apoptosis, and autophagy, and promotes cell motility and invasiveness. Both genes have been associated with cancer progression and interact with TP53, suggesting a potential role in tumor suppression.|
| [ILK](https://www.ncbi.nlm.nih.gov/gene?term=ILK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.8</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | ILK and MAP2K5 could be functionally associated in the regulation of cell motility and adhesion, as they both interact with integrins and are involved in signaling pathways that regulate these processes. The shortest path between these two genes in the BioGRID protein interaction network involves multiple proteins involved in these processes, supporting this possible association.|
| [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.2</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on the provided context, there is no clear indication of a functional association between KDM5A and MYL9. KDM5A is a histone demethylase involved in regulating gene transcription, while MYL9 plays a role in cellular contractile activity. However, it is possible that KDM5A could regulate the transcription of MYL9 in certain contexts.|
| [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.1</br>**STRING score:** 325</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 56 (min len 3) | Possible functional associations between TP53 and USP22: TP53 is a tumor suppressor gene involved in cell cycle regulation, apoptosis induction, and oxidative stress-induced necrosis. USP22 is a histone deubiquitinase that acts as a coactivator for transcription and is required for cell cycle progression. Both genes are part of the same SAGA transcription coactivator-HAT complex and may have a role in regulating gene expression.|
