| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 34 (min len 3) | Possible functional associations: LYN is involved in immune responses, including B-cell differentiation and regulation, cytokine release, and integrin signaling. TM9SF3 is required for de novo DNA methylation and is essential for establishing DNA methylation patterns during development. There is no obvious biological relationship between these two genes.|
| [LIG1](https://www.ncbi.nlm.nih.gov/gene?term=LIG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations: LIG1 may regulate the activity of the PI3K/AKT signaling pathway through PIK3R3. LIG1 may also interact with EGFR/ERBB1, ERBB2, ERBB3 and ERBB4, which are upstream of PIK3R3 in this pathway.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [IFNAR1](https://www.ncbi.nlm.nih.gov/gene?term=IFNAR1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | There is no obvious functional association between the two genes HDAC2 and IFNAR1 based on their known biological roles. HDAC2 is a histone deacetylase involved in transcriptional regulation, while IFNAR1 forms a receptor for type I interferons and activates the JAK-STAT signaling cascade. However, the protein interaction network shows a potential indirect relationship through several intermediary proteins, indicating a possible regulatory relationship between these two genes.|
| [TRIB1](https://www.ncbi.nlm.nih.gov/gene?term=TRIB1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRIM2](https://www.ncbi.nlm.nih.gov/gene?term=TRIM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There is currently no known functional association between TRIB1 and TRIM2 in specific biological terms. TRIB1 is involved in protein degradation and regulation of myeloid cell differentiation, while TRIM2 is an E3 ubiquitin-protein ligase that plays a neuroprotective function and may play a role in antiviral immunity.|
| [ATP6V1D](https://www.ncbi.nlm.nih.gov/gene?term=ATP6V1D[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HAT1](https://www.ncbi.nlm.nih.gov/gene?term=HAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between ATP6V1D and HAT1 include their interaction in the BioGRID protein network, suggesting a possible regulatory relationship. ATP6V1D is involved in acidification of intracellular compartments and cilium biogenesis, while HAT1 is a histone acetyltransferase complex involved in regulating gene expression through histone modification.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MELK](https://www.ncbi.nlm.nih.gov/gene?term=MELK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.7</br>**STRING score:** 170</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear indication of a functional association between CDC42 and MELK based on the given context. CDC42 is involved in cell polarization, regulation of spindle microtubules, cell migration, and phagocytosis. MELK is a kinase involved in cell cycle regulation, self-renewal of stem cells, apoptosis, and splicing regulation.|
| [CALU](https://www.ncbi.nlm.nih.gov/gene?term=CALU[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.5</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association: DYNLT3 is involved in binding to transport cargo, and CALU is involved in the regulation of vitamin K-dependent carboxylation. It is possible that DYNLT3 may transport CALU to its target location for its activity, or CALU may affect the cargo binding ability of DYNLT3. However, further experiments are needed to confirm any specific biological association between these genes.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UTP18](https://www.ncbi.nlm.nih.gov/gene?term=UTP18[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 32 (min len 3) | There is no clear information indicating a direct functional association between CDC42 and UTP18. CDC42 is involved in cell polarization, regulation of cellular responses, and spine formation in neurons. UTP18 is part of the small subunit processome involved in ribosomal RNA processing.|
| [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOPBP1](https://www.ncbi.nlm.nih.gov/gene?term=TOPBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between TBC1D15 and TOPBP1 are unclear based on the provided context. However, TBC1D15 acts as a GTPase activating protein for RAB7A and interacts with non-phosphorylated form of RAB8A. TOPBP1 is required for DNA replication, plays a role in the rescue of stalled replication forks, and down-regulates E2F1 activity.|
| [PTPN12](https://www.ncbi.nlm.nih.gov/gene?term=PTPN12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TRAM2](https://www.ncbi.nlm.nih.gov/gene?term=TRAM2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: PTPN12 is involved in regulating cellular signaling cascades by dephosphorylating proteins, including tyrosine kinases such as ERBB2 and PTK2B/PYK2. TRAM2 is necessary for collagen type I synthesis and may couple the activity of the ER Ca(2+) pump SERCA2B with the activity of the translocon, which may increase the local Ca(2+) concentration at the site of collagen synthesis.  The shortest path in the BioGRID protein interaction network between PTPN12 and TRAM2 involves ZRANB1, and it is not clear what the functional implications of this interaction might be. However, PTPN12 and TRAM2 appear to be involved in different biological processes and are not directly related in terms of their known functions.|
| [HDAC2](https://www.ncbi.nlm.nih.gov/gene?term=HDAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HMGN1](https://www.ncbi.nlm.nih.gov/gene?term=HMGN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.0</br>**STRING score:** 154</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | HDAC2 and HMGN1 are both involved in regulation of chromatin structure and transcriptional repression. They may functionally associate in the formation of the NuRD complex, which remodels chromatin and mediates transcriptional repression. The short path between them in the BioGRID protein interaction network suggests they may interact directly.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBC1D15](https://www.ncbi.nlm.nih.gov/gene?term=TBC1D15[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 48.1</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There is no known direct functional association between NRAS and TBC1D15. NRAS encodes a protein involved in regulation of cell growth and division, while TBC1D15 functions as a GTPase activating protein for RAB7A, a protein involved in intracellular transport and autophagy.|
| [LYN](https://www.ncbi.nlm.nih.gov/gene?term=LYN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 47.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between LYN and RBM34 based on the available information. LYN is a tyrosine kinase involved in immune responses, while RBM34 is involved in EGF signaling and magnesium reabsorption in the kidney. Further investigation is needed to determine any possible interactions between these genes.|
| [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.9</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between PIK3R3 and PTPRF could be in regulating insulin signaling and substrate specificity in protein-tyrosine phosphatase activity. PIK3R3 binds to activated protein-tyrosine kinases and regulates their activity while PTPRF contains two PTPase domains, where the second one affects the specificity of the first one.|
| [CDC42](https://www.ncbi.nlm.nih.gov/gene?term=CDC42[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.7</br>**STRING score:** 389</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 317 (min len 3) | Possible functional association: CDC42 and NRAS may be involved in cell migration and filopodia formation, as well as regulation of effector proteins and synaptic plasticity. The shortest path between them in the BioGRID protein interaction network suggests they may interact indirectly through common proteins like DOCK8 and BAIAP2.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRC](https://www.ncbi.nlm.nih.gov/gene?term=SRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.5</br>**STRING score:** 972</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: NRAS and SRC may interact in the regulation of cell growth and proliferation, as both proteins have been shown to be involved in signaling pathways that control these processes. NRAS is a Ras protein that binds GDP/GTP and activates downstream signaling pathways, while SRC is a non-receptor protein tyrosine kinase that plays a role in neurite elongation and has been implicated in cancer. The interaction between NRAS and SRC may be relevant in cancer progression, particularly in the context of TCGA-UCEC.|
| [NRIP1](https://www.ncbi.nlm.nih.gov/gene?term=NRIP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 46.3</br>**STRING score:** 226</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 11 (min len 3) | Possible functional associations between NRIP1 and TBP may involve transcriptional activation and regulation of gene expression. NRIP1 may modulate transcriptional activation by steroid receptors and act as a coactivator for RORA and RORC, while TBP is a general transcription factor that functions at the core of the TFIID complex, promoting assembly of the pre-initiation complex and regulating transcription mediated by RNA polymerase II and III. The shortest path between NRIP1 and TBP in the BioGRID protein interaction network suggests they may interact indirectly via CTBP1, NR3C1, RXRA, ESR1, and/or AHR.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [S100A4](https://www.ncbi.nlm.nih.gov/gene?term=S100A4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.9</br>**STRING score:** 241</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between NFKBIA and S100A4 include regulation of cellular motility, apoptosis, and cytokine production. NFKBIA inhibits NF-kappa-B/REL complexes and can interact with TP53, while S100A4 is a calcium-binding protein that interacts with non-muscle myosin heavy chain and modulates the pro-apoptotic function of TP53. The shortest path between these genes in the BioGRID protein interaction network includes several other genes that may have additional functional links.|
| [ILK](https://www.ncbi.nlm.nih.gov/gene?term=ILK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP2K5](https://www.ncbi.nlm.nih.gov/gene?term=MAP2K5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.8</br>**STRING score:** 158</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 26 (min len 3) | Possible functional associations: ILK regulates integrin-mediated signaling and cell motility, while MAP2K5 is a scaffold for MAP kinase signaling involved in cellular stress response and survival. There could be crosstalk between these pathways in regulating cell behavior in the context of cancer.|
| [KDM5A](https://www.ncbi.nlm.nih.gov/gene?term=KDM5A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYL9](https://www.ncbi.nlm.nih.gov/gene?term=MYL9[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.2</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: It is unclear based on the information provided whether KDM5A and MYL9 have a direct functional association.   KDM5A: Histone demethylase that plays a role in histone code and gene transcription regulation, including Hox proteins and circadian rhythm. May also participate in transcriptional repression of cytokines and favor proliferation of cancer cells.  MYL9: Myosin regulatory subunit that plays an important role in regulation of contractile activity in smooth and nonmuscle cells, including cytokinesis, receptor capping, and cell locomotion. May also regulate PIEZO1-dependent cortical actomyosin assembly involved in myotube formation in myoblasts.|
| [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP22](https://www.ncbi.nlm.nih.gov/gene?term=USP22[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 44.1</br>**STRING score:** 325</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 56 (min len 3) | Possible functional associations: TP53 acts as a tumor suppressor, induces growth arrest or apoptosis, and regulates cell cycle by controlling genes required for the process. USP22 is a histone deubiquitinating component of the transcription regulatory histone acetylation complex, required for transcription and cell cycle progression. These genes can both be involved in cell cycle regulation, growth suppression, and/or apoptosis induction, suggesting some possible functional associations.|
