| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ATF5](https://www.ncbi.nlm.nih.gov/gene?term=ATF5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COG4](https://www.ncbi.nlm.nih.gov/gene?term=COG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 79 (min len 4) | Possible functional associations between ATF5 and COG4 in TCGA-UCEC include their involvement in cell survival, proliferation, and differentiation, as well as their roles in Golgi function and SNARE-pin assembly. However, there is currently no known direct biological interaction between these two genes.|
| [BLCAP](https://www.ncbi.nlm.nih.gov/gene?term=BLCAP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: BLCAP and EML3 do not have a direct interaction but are connected through SKAP1 in the BioGRID network. BLCAP may regulate cell proliferation and apoptosis, while EML3 plays a role in mitotic spindle assembly and chromosome separation. There is no known direct relationship between these two genes in specific biological terms.|
| [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 157</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association between RBM34 and YME1L1 is their interaction through NR3C1, ESR2, RECQL4, MYC, and BRD7 in the BioGRID protein interaction network. RBM34 is involved in EGF stimulation of growth and can induce neurite outgrowth, while YME1L1 is an ATP-dependent metalloprotease that catalyzes the degradation of folded and unfolded proteins with a suitable degron sequence in the mitochondrial intermembrane region and is important in regulating mitochondrial morphology and function.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.4</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between GADD45B and TGFBR2 is their involvement in the regulation of growth and apoptosis through their respective pathways - GADD45B through MTK1/MEKK4 MAPKKK and TGFBR2 through SMAD2/3 activation. The shortest path identified in the BioGRID protein interaction network suggests a potential direct physical interaction between the two proteins through the mediator CCNB1.|
| [ST3GAL5](https://www.ncbi.nlm.nih.gov/gene?term=ST3GAL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP6NL](https://www.ncbi.nlm.nih.gov/gene?term=USP6NL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on the context, there is no clear functional association between ST3GAL5 and USP6NL. ST3GAL5 is involved in the biosynthesis of ganglioside GM3, while USP6NL is involved in receptor trafficking, retrograde transport, and structural integrity of the Golgi complex. The shortest path between these genes in the BioGRID protein interaction network involves an intermediate protein, ORF3a, suggesting a possible indirect association.|
| [BPHL](https://www.ncbi.nlm.nih.gov/gene?term=BPHL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FPGS](https://www.ncbi.nlm.nih.gov/gene?term=FPGS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between BPHL and FPGS include involvement in drug metabolism and detoxification processes, as well as the conversion and concentration of folate compounds in cells for use in one-carbon transfer reactions. Given their interaction through the PARK2 and TRIM67 proteins in the BioGRID protein interaction network, they may also act together in specific biological processes related to these functions.|
| [CAPN1](https://www.ncbi.nlm.nih.gov/gene?term=CAPN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX7](https://www.ncbi.nlm.nih.gov/gene?term=SNX7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 186 (min len 4) | There is no clear functional association between CAPN1 and SNX7 based on their individual roles in protein cleavage and endocytosis regulation, respectively. However, they are part of a protein interaction network with several intermediary proteins involved in cytoskeletal remodeling and signal transduction which could suggest they participate in common signaling pathways or biological processes.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRAF2](https://www.ncbi.nlm.nih.gov/gene?term=PRAF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: Both genes are involved in cellular signaling and response to immune and pro-inflammatory stimuli. NFKBIA inhibits REL dimers while PRAF2 may be involved in ER/Golgi transport and vesicular traffic, playing a proapoptotic role. There is a protein interaction between NFKBIA and PRAF2 via COPB2. This suggests a possible role in cellular trafficking and regulation of pro-inflammatory signaling pathways.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST3GAL5](https://www.ncbi.nlm.nih.gov/gene?term=ST3GAL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | HIF1A is involved in hypoxia response and regulation of glycolysis and cytokine expression during microbial infection. ST3GAL5 transfers sialyl groups to glycolipids, mainly involved in the biosynthesis of ganglioside GM3. The shortest path in the BioGRID protein interaction network between HIF1A and ST3GAL5 involves SP1 and GBF1, suggesting a potential association in ganglioside metabolism.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAZALD1](https://www.ncbi.nlm.nih.gov/gene?term=KAZALD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between CEBPD and KAZALD1 is involved in bone formation and regeneration as KAZALD1 is involved in osteoblast proliferation and matrix assembly while CEBPD regulates the expression of genes involved in immune and inflammatory responses. There is no clear biological mechanism for a direct interaction between CEBPD and KAZALD1.|
| [CASP7](https://www.ncbi.nlm.nih.gov/gene?term=CASP7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 177</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CASP7 and SHC1 may involve regulation of cell death and stress response pathways, as CASP7 has enzymatic activity in apoptotic pathways and SHC1 is involved in the cellular response to oxidative stress and plays a role in apoptosis. The BioGRID protein interaction network suggests that the shortest path between these two genes involves interactions with chaperone proteins and regulatory proteins like RB1, which may modulate their activity in the relevant pathways.|
| [LPAR2](https://www.ncbi.nlm.nih.gov/gene?term=LPAR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.5</br>**STRING score:** 223</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between LPAR2 and RAC2 may involve regulating cellular responses such as secretion, phagocytosis, and epithelial cell polarization. The shortest path between LPAR2 and RAC2 in the BioGRID protein interaction network suggests that LPAR2 may interact with PLCB3 to activate RAC2, and that LPAR2 may also interact with EGFR, THEM6, SLC39A6, and CCDC115 to activate RAC2 through other signaling pathways.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNP](https://www.ncbi.nlm.nih.gov/gene?term=PNP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: There is no clear biological process linking CDK2 and PNP. CDK2 is involved in cell cycle control, DNA replication and repair, and chromatin remodeling, while PNP is a hormone regulator and component of mitochondrial ribosomes.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN1](https://www.ncbi.nlm.nih.gov/gene?term=RPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 278</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | HSPA8 and RPN1 do not have any known direct functional associations in the literature. HSPA8 is involved in protein folding and quality control, chaperone-mediated autophagy, mitochondrial import, and transcriptional repression. RPN1 is a subunit of the oligosaccharyl transferase (OST) complex, which catalyzes the initial transfer of a defined glycan in protein N-glycosylation.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIPARP](https://www.ncbi.nlm.nih.gov/gene?term=TIPARP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional association: CCL2 is a chemokine that recruits monocytes into the arterial wall during atherosclerosis. TIPARP is a negative regulator of AHR transcriptional activity. There is no currently known specific functional association between these two genes.|
| [APPBP2](https://www.ncbi.nlm.nih.gov/gene?term=APPBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNP](https://www.ncbi.nlm.nih.gov/gene?term=PNP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: There is no direct evidence for a functional association between APPBP2 and PNP. However, based on their individual functions, it is possible that APPBP2 plays a role in the transport of APP while PNP is involved in mitochondrial ribosomal subunit assembly.|
| [ASCC3](https://www.ncbi.nlm.nih.gov/gene?term=ASCC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.9</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional associations between ASCC3 and RBM34 include involvement in DNA repair, ribosome quality control, and transcriptional regulation through the ASC-1/TRIP4 complex. There is no direct evidence of a specific biological interaction between these two genes.|
| [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.5</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There is no obvious functional association between KDELR2 and TPM1, as they participate in distinct biological processes. KDELR2 is a membrane receptor involved in maintaining protein localization, while TPM1 is a cytoskeletal protein involved in muscle contraction. The shortest path between them in the BioGRID protein interaction network involves intermediate proteins that may act as bridges between distant biological processes.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.1</br>**STRING score:** 265</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | Possible functional associations between CDK2 and NFKBIB include CDK2 phosphorylating NFKBIB, leading to its degradation and subsequent nuclear transport of NF-kappa-B, as well as their interaction in the bioGRID protein interaction network involving CDC25A, REL, IKBKG, SKP1, and HSP90AB1. CDK2 plays a crucial role in cell cycle control, while NFKBIB inhibits NF-kappa-B by trapping it in the cytoplasm, however, it can also bind NF-kappa-B after cell stimulation and transport it to the nucleus.|
| [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between DNAJB6 and RBM34 based on the provided context and existing literature. DNAJB6 is involved in inhibiting protein aggregation and interacts with histone deacetylases, while RBM34 is involved in signaling through growth factor receptors and magnesium transport. Both genes may play roles in regulating cellular processes, but these processes appear to be distinct.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between KDM5B and RBM34 may involve transcriptional regulation and cell growth. KDM5B is a histone demethylase that acts as a transcriptional corepressor, while RBM34 interacts with EGFR and promotes cell growth. The shortest path in the BioGRID protein interaction network suggests potential physical interaction between these two proteins via CHD3, MYC, MYCN, or PLEKHA4.|
| [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.8</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations: PIK3R3 and PTPRF both interact with insulin receptor substrate (IRS-1) and insulin receptor (INSR), indicating a shared involvement in insulin signaling pathways.   PIK3R3 encodes a protein that binds to and regulates the activity of protein-tyrosine kinases, including those involved in insulin signaling. PTPRF encodes a protein with two protein tyrosine phosphatase domains that may interact with PIK3R3 and modulate its substrate specificity. Together, these genes may play a role in modulating insulin signaling pathways.|
| [PAF1](https://www.ncbi.nlm.nih.gov/gene?term=PAF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2A](https://www.ncbi.nlm.nih.gov/gene?term=UBE2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.7</br>**STRING score:** 671</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between PAF1 and UBE2A in specific biological terms could be through PAF1C's recruitment of the RNF20/40 E3 ubiquitin-protein ligase complex and the E2 enzyme UBE2A in monoubiquitination of 'Lys-120' of histone H2B. This modification gives a specific tag for epigenetic transcriptional activation, elongation by RNA polymerase II, and is proposed to be coupled to transcription.|
| [MAT2A](https://www.ncbi.nlm.nih.gov/gene?term=MAT2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UGDH](https://www.ncbi.nlm.nih.gov/gene?term=UGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.2</br>**STRING score:** 597</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: MAT2A and UGDH may have a role in the biosynthesis of complex glycosaminoglycans, specifically chondroitin sulfate and heparan sulfate required for embryonic and neuronal development. The shortest path between them in the BioGRID protein interaction network suggests a direct interaction between the two proteins.|
