| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [ATF5](https://www.ncbi.nlm.nih.gov/gene?term=ATF5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COG4](https://www.ncbi.nlm.nih.gov/gene?term=COG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 71.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 79 (min len 4) | Possible functional association: Both genes interact through a protein interaction network and are involved in cellular survival, proliferation, and differentiation, as well as Golgi function. However, their specific biological roles and potential interactions remain unclear and would require further investigation.|
| [BLCAP](https://www.ncbi.nlm.nih.gov/gene?term=BLCAP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EML3](https://www.ncbi.nlm.nih.gov/gene?term=EML3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 67.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Based on their protein interaction network, BLCAP and EML3 may function together in regulating mitotic spindle assembly and proper chromosomal alignment during cell division. Additionally, BLCAP has a potential role in regulating cell proliferation and activating apoptosis.|
| [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [YME1L1](https://www.ncbi.nlm.nih.gov/gene?term=YME1L1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 66.0</br>**STRING score:** 157</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | One possible functional association between RBM34 and YME1L1 is they both play roles in maintaining mitochondrial function. RBM34 interacts with EGFR to promote EGFR dimerization, while YME1L1 is an ATP-dependent metalloprotease that regulates mitochondrial morphology and function by cleaving OPA1.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 62.4</br>**STRING score:** 176</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: GADD45B and TGFBR2 are connected in a protein interaction network with one intermediate node, CCNB1. GADD45B is involved in the regulation of growth and apoptosis, while TGFBR2 blocks activation of SMAD2 and SMAD3 by TGFB1 homodimer. It is possible that GADD45B and TGFBR2 are involved in regulating cell growth and apoptosis through the TGFB1/SMAD signaling pathway.|
| [ST3GAL5](https://www.ncbi.nlm.nih.gov/gene?term=ST3GAL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP6NL](https://www.ncbi.nlm.nih.gov/gene?term=USP6NL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 60.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: There is no direct biological evidence suggesting a functional association between ST3GAL5 and USP6NL.   ST3GAL5: It transfers sialyl group to the non-reducing terminal galactose of glycosphingolipids, mainly involved in the biosynthesis of ganglioside GM3, but can also use different glycolipids as substrate acceptors.  USP6NL: It acts as a GTPase-activating protein for RAB5A and RAB43, involved in receptor trafficking, retrograde transport from the endocytic pathway to the Golgi apparatus, and transport of Shiga toxin from early and recycling endosomes to the trans-Golgi network.|
| [BPHL](https://www.ncbi.nlm.nih.gov/gene?term=BPHL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FPGS](https://www.ncbi.nlm.nih.gov/gene?term=FPGS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between BPHL and FPGS include their involvement in drug metabolism and activation, with BPHL hydrolyzing amino acid ester prodrugs and FPGS converting folates to polyglutamate derivatives. The shortest path between BPHL and FPGS in the BioGRID protein interaction network includes PARK2 and TRIM67, which have been implicated in ubiquitination and degradation pathways, suggesting a possible role in regulating the activity of these two enzymes.|
| [CAPN1](https://www.ncbi.nlm.nih.gov/gene?term=CAPN1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX7](https://www.ncbi.nlm.nih.gov/gene?term=SNX7[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 186 (min len 4) | There is no obvious functional association between CAPN1 and SNX7 based on their known biological roles and protein interactions. CAPN1 is involved in proteolysis and signal transduction, while SNX7 is involved in endocytosis and intracellular trafficking. However, further research may reveal a potential connection between these processes.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRAF2](https://www.ncbi.nlm.nih.gov/gene?term=PRAF2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: NFKBIA inhibits RELA transcription factor complex activity, while PRAF2 may be involved in ER/Golgi transport and have a proapoptotic role. There is no directly known biological association between these two genes.|
| [HIF1A](https://www.ncbi.nlm.nih.gov/gene?term=HIF1A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ST3GAL5](https://www.ncbi.nlm.nih.gov/gene?term=ST3GAL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 59.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between HIF1A and ST3GAL5 may involve HIF1A's role in glycolysis induction and pro-inflammatory cytokine expression during coronavirus infection, and ST3GAL5's involvement in ganglioside biosynthesis and regulation of cellular processes. However, there is no direct biological link between these two genes in known literature or in the shortest protein interaction path between them.|
| [CEBPD](https://www.ncbi.nlm.nih.gov/gene?term=CEBPD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [KAZALD1](https://www.ncbi.nlm.nih.gov/gene?term=KAZALD1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 58.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | CEBPD and KAZALD1 have a known functional association as indicated by the shortest path in the BioGRID network. CEBPD is a transcriptional activator important for immune and inflammatory responses, while KAZALD1 is involved in osteoblast proliferation during bone formation and regeneration. There may be a potential link between these two genes in the regulation of immune responses and bone regeneration.|
| [CASP7](https://www.ncbi.nlm.nih.gov/gene?term=CASP7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SHC1](https://www.ncbi.nlm.nih.gov/gene?term=SHC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 57.2</br>**STRING score:** 177</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations between CASP7 and SHC1 could involve apoptosis and signal transduction pathways. CASP7 is a key mediator in apoptosis, while SHC1 is a signaling adapter that couples activated growth factor receptors to signaling pathways. The shortest path between these two genes in the BioGRID protein interaction network also suggests potential direct interactions between them.|
| [LPAR2](https://www.ncbi.nlm.nih.gov/gene?term=LPAR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.5</br>**STRING score:** 223</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | LPAR2 and RAC2 are functionally associated via the BioGRID protein interaction network as they are connected by several shortest paths. LPAR2 is a receptor for lysophosphatidic acid that plays a key role in phospholipase C-beta signaling pathway while RAC2 is a plasma membrane-associated small GTPase that regulates cellular responses and augments the production of reactive oxygen species. It is possible that LPAR2 and RAC2 interact to regulate phagocytosis of apoptotic cells and production of ROS.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNP](https://www.ncbi.nlm.nih.gov/gene?term=PNP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 55.2</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CDK2 is a serine/threonine-protein kinase involved in the control of the cell cycle, while PNP is a enzyme catalyzing the breakdown of nucleoside molecules. There is no known direct functional association between these two genes based on the provided context and knowledge about biology.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RPN1](https://www.ncbi.nlm.nih.gov/gene?term=RPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 278</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: both HSPA8 and RPN1 are involved in protein quality control and folding. HSPA8 acts as a molecular chaperone and plays a role in chaperone-mediated autophagy and ER-associated degradation. RPN1 functions in the initial step of protein N-glycosylation. These genes may work together in ensuring proper protein folding and degradation. The BioGRID interaction network shows a direct interaction between the two proteins.|
| [CCL2](https://www.ncbi.nlm.nih.gov/gene?term=CCL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TIPARP](https://www.ncbi.nlm.nih.gov/gene?term=TIPARP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 54.2</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between CCL2 and TIPARP are currently unknown based on the available information. CCL2 acts as a chemotactic ligand for CCR2, involved in monocyte recruitment, while TIPARP mediates mono-ADP-ribosylation of target proteins and negatively regulates AHR. However, their interaction in specific biological processes remains unclear.|
| [APPBP2](https://www.ncbi.nlm.nih.gov/gene?term=APPBP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PNP](https://www.ncbi.nlm.nih.gov/gene?term=PNP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 53.5</br>**STRING score:** 229</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between APPBP2 and PNP are unclear based on the provided context, as they are involved in different biological processes. APPBP2 is a substrate-recognition component of a CRL2 E3 ubiquitin-protein ligase complex and may play a role in intracellular protein transport, while PNP catalyzes the phosphorolytic breakdown of N-glycosidic bond in beta-(deoxy)ribonucleoside molecules.|
| [ASCC3](https://www.ncbi.nlm.nih.gov/gene?term=ASCC3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.9</br>**STRING score:** 248</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | There are currently no known functional associations between ASCC3 and RBM34. ASCC3 is involved in DNA repair, ribosome quality control, and transcriptional transactivation, while RBM34 is involved in EGF signaling and ion transport regulation.|
| [KDELR2](https://www.ncbi.nlm.nih.gov/gene?term=KDELR2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPM1](https://www.ncbi.nlm.nih.gov/gene?term=TPM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.5</br>**STRING score:** 194</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | There are no obvious functional associations between KDELR2 and TPM1 based on their known roles. KDELR2 is a membrane receptor involved in the recycling of endoplasmic reticulum resident proteins, while TPM1 binds to actin filaments and plays a central role in muscle contraction regulation.|
| [CDK2](https://www.ncbi.nlm.nih.gov/gene?term=CDK2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIB](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 52.1</br>**STRING score:** 265</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 19 (min len 3) | CDK2 is a serine/threonine protein kinase involved in controlling the cell cycle, while NFKBIB inhibits NF-kappa-B by trapping it in the cytoplasm. Possible functional associations between the two genes include CDK2's phosphorylation of NFKBIB or their interaction in DNA damage checkpoint regulation.|
| [DNAJB6](https://www.ncbi.nlm.nih.gov/gene?term=DNAJB6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 51.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations could include a role in protein aggregation and interaction with histone deacetylases for DNAJB6, and a role in growth stimulation and interaction with EGFR for RBM34. There is no direct evidence of functional association between these two genes, but they may share common pathways related to growth regulation or protein quality control.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM34](https://www.ncbi.nlm.nih.gov/gene?term=RBM34[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 50.0</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between KDM5B and RBM34 are unclear based on their individual known functions. KDM5B is a histone demethylase, while RBM34 is a growth factor that promotes magnesium reabsorption. However, the shortest path in the protein interaction network suggests they may have indirect interactions through several proteins, including MYC and MYCN, which are known to regulate cellular proliferation and apoptosis.|
| [PIK3R3](https://www.ncbi.nlm.nih.gov/gene?term=PIK3R3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.8</br>**STRING score:** 151</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 18 (min len 3) | Possible functional associations between PIK3R3 and PTPRF may be related to insulin signaling and regulation of protein tyrosine kinases. PIK3R3 binds to activated protein-tyrosine kinases and regulates their kinase activity, while PTPRF has enzymatic activity and affects substrate specificity. Additionally, PIK3R3 interacts with IRS-1, and PTPRF interacts with INSR, both of which are involved in insulin signaling.|
| [PAF1](https://www.ncbi.nlm.nih.gov/gene?term=PAF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UBE2A](https://www.ncbi.nlm.nih.gov/gene?term=UBE2A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.7</br>**STRING score:** 671</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations between PAF1 and UBE2A include their involvement in histone modifications, specifically ubiquitination of histone H2B, and their cooperation in transcriptional regulation through monoubiquitination of 'Lys-120' of histone H2B, leading to transcriptional elongation and activation. PAF1 is also involved in mRNA 3' end formation, while UBE2A is required for postreplication repair of UV-damaged DNA.|
| [MAT2A](https://www.ncbi.nlm.nih.gov/gene?term=MAT2A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UGDH](https://www.ncbi.nlm.nih.gov/gene?term=UGDH[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 49.2</br>**STRING score:** 597</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between MAT2A and UGDH could include their involvement in the biosynthesis of complex glycosaminoglycans, such as chondroitin sulfate and heparan sulfate, which are important components of the extracellular matrix. MAT2A catalyzes the formation of S-adenosylmethionine, a precursor for the synthesis of glycosaminoglycans, while UGDH catalyzes the formation of UDP-alpha-D-glucuronate, a constituent of these same glycosaminoglycans.|
