| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [BAG3](https://www.ncbi.nlm.nih.gov/gene?term=BAG3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ID2](https://www.ncbi.nlm.nih.gov/gene?term=ID2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | There are no known direct functional associations between BAG3 and ID2 in TCGA-THCA. BAG3 acts as a co-chaperone for HSP70 and HSC70 chaperone proteins and has anti-apoptotic activity. ID2 is a transcriptional regulator that negatively regulates the basic helix-loop-helix (bHLH) transcription factors.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TMEM50A](https://www.ncbi.nlm.nih.gov/gene?term=TMEM50A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no evident functional associations between BAG6 and TMEM50A. BAG6 mediates ricin-induced apoptosis and interacts with L. pneumophila Lpg2160 and LegU1 proteins, while TMEM50A is one of the 2 subunits of the mitochondrial enzyme biotin-dependent propionyl-CoA carboxylase involved in the catabolism of odd chain fatty acids and branched-chain amino acids.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UQCR10](https://www.ncbi.nlm.nih.gov/gene?term=UQCR10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: BAG6 has been found to interact with UQCR10 in the BioGRID protein interaction network. UQCR10 is involved in the mitochondrial electron transport chain, while BAG6 mediates ricin-induced apoptosis and interacts with various proteins, including L. pneumophila Lpg2160 and LegU1. There may be a functional association between UQCR10 and BAG6 in regulating apoptosis or in mitochondrial function.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NDUFA3](https://www.ncbi.nlm.nih.gov/gene?term=NDUFA3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between EGFR and NDUFA3 are unclear based solely on the provided context. EGFR acts as a receptor for hepatitis C virus and mediates cell entry, while NDUFA3 is an accessory subunit of the mitochondrial respiratory chain NADH dehydrogenase. They may possibly be involved in cellular metabolism and energy production, but further investigation is needed to determine any concrete functional associations.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNAS](https://www.ncbi.nlm.nih.gov/gene?term=GNAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 3) | There is no clear functional association between BAG6 and GNAS in the context provided. BAG6 is involved in apoptosis, while GNAS plays a crucial role in B cell development, differentiation, and signaling. The shortest path between the two genes in the BioGRID protein interaction network suggests several intermediary proteins that could potentially connect these two genes in specific biological processes.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | Possible functional associations: BAG6 mediates apoptosis and interacts with microbial proteins, while SRP14 plays a role in targeting proteins to the endoplasmic reticulum. There is no direct known functional association between these two genes.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 766</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 48 (min len 3) | Possible functional association: TP53 is a tumor suppressor gene involved in cell cycle regulation and apoptosis, while CEBPA enhances ribosomal DNA transcription and induces G1 cell cycle arrest. Both genes are involved in regulating cell division and growth.|
| [PDLIM1](https://www.ncbi.nlm.nih.gov/gene?term=PDLIM1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations: PDLIM1 is involved in cytoskeletal organization, while SLC25A14 is involved in mitochondrial proton leak and cell motility. It is possible that PDLIM1 may play a role in regulating the activity or localization of SLC25A14 in the cytoskeleton, potentially impacting mitochondrial function or cell motility. The interaction between the two proteins through PLEKHA4 supports this possibility.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [VPS28](https://www.ncbi.nlm.nih.gov/gene?term=VPS28[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional associations: VPS28 and BAG6 have been found to interact in the BioGRID protein interaction network and both are involved in vesicular trafficking processes.   VPS28 is a component of the ESCRT-I complex which regulates vesicular trafficking, while BAG6 mediates apoptosis and interacts with microbial proteins. It is possible that BAG6 may play a role in regulating the ESCRT-I complex or vesicular trafficking through its interaction with VPS28.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CCDC92](https://www.ncbi.nlm.nih.gov/gene?term=CCDC92[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | There are no known functional associations between BAG6 and CCDC92 in the literature, but BAG6 is involved in mediating ricin-induced apoptosis and has interactions with L. pneumophila Lpg2160 and LegU1 proteins, while CCDC92 is an interferon-stimulated protein that plays a role in innate immunity and inhibits ebolavirus transcription and replication by forming a complex with viral RNA-bound nucleocapsid NP, preventing its transport to the cell surface.|
| [EIF4G1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4G1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SRP14](https://www.ncbi.nlm.nih.gov/gene?term=SRP14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between EIF4G1 and SRP14 could be related to protein translation and localization, as EIF4G1 is a component of the eIF4F complex involved in mRNA recognition and translation, and SRP14 is a component of the signal recognition particle (SRP) complex that mediates cotranslational targeting of proteins to the endoplasmic reticulum. However, additional experiments would be needed to confirm any direct interactions or functional associations between these two genes.|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMB1](https://www.ncbi.nlm.nih.gov/gene?term=PSMB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 55 (min len 3) | Possible functional associations between EPRS and PSMB1 can be inferred from their interaction in the BioGRID protein interaction network, where they are connected by a shortest path of protein interactions. EPRS is involved in protein synthesis and translation inhibition, and PSMB1 is involved in protein degradation, suggesting that they may be part of a regulatory mechanism that controls protein turnover.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GNAS](https://www.ncbi.nlm.nih.gov/gene?term=GNAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.5</br>**STRING score:** 696</br>**BioGRID count:** 4</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and GNAS have no known direct functional association. EGFR acts as a receptor for hepatitis C virus in hepatocytes, and mediates downstream signaling pathways involved in cell proliferation, migration, and NF-kappa-B activation. GNAS is a non-receptor tyrosine kinase that plays important roles in B lymphocyte development and differentiation, as well as being involved in the function of immune cells and regulation of transcription.|
