| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SSBP2](https://www.ncbi.nlm.nih.gov/gene?term=SSBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between RBM6 and SSBP2 are in the regulation of ERK signaling pathway and in survival processes. RBM6 is known to bind poly(G) RNA homopolymers and may interact with FAM168B, while SSBP2 inhibits ERK dephosphorylation by PP2A-PPP2R5C holoenzyme and regulates ERK kinases, thus playing a downstream effector mediating survival. The shortest path between RBM6 and SSBP2 in the BioGRID protein interaction network involves ARHGAP22 as an intermediary.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations between ARHGAP1 and BAG6 could be their involvement in mediating apoptosis and protein degradation pathways. ARHGAP1 acts as a GTPase activator for Rho and Rac proteins, while BAG6 interacts with L. pneumophila Lpg2160 and LegU1 proteins to mediate ricin-induced apoptosis. Furthermore, both proteins have been implicated in protein degradation pathways through interactions with other proteins in the BioGRID network.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 331</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between BAG6 and TP53 is that BAG6 mediates ricin-induced apoptosis and interacts with TP53BP2/ASPP2 which activates TP53's pro-apoptotic activity. TP53 acts as a tumor suppressor by inducing growth arrest or apoptosis depending on the physiological circumstances and cell type and is involved in cell cycle regulation and Notch signaling cross-over.|
| [ADAT1](https://www.ncbi.nlm.nih.gov/gene?term=ADAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF4G1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4G1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no clear functional associations between ADAT1 and EIF4G1 based on the provided context and biology knowledge. ADAT1 specifically deaminates adenosine to inosine in tRNA-Ala while EIF4G1 is a component of the eIF4F complex involved in mRNA recognition and ribosome recruitment. The protein interaction network between ADAT1 and EIF4G1 involves several intermediary proteins.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBBP6](https://www.ncbi.nlm.nih.gov/gene?term=RBBP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 477 (min len 4) | Possible functional associations between C2CD2 and RBBP6 include DNA repair, apoptosis promotion, and viral restriction through their interactions with AIFM1 and ebolavirus VP30, respectively. However, there is no known direct biological process linking these two genes.|
| [DAG1](https://www.ncbi.nlm.nih.gov/gene?term=DAG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP4K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP4K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional associations: DAG1 is involved in microbial infection as a receptor for viruses and bacteria, while MAP4K4 plays a role in response to environmental stress and cytokines. There is no known direct biological interaction or functional association between DAG1 and MAP4K4.|
| [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 4) | Possible functional associations between PRUNE and SLC25A14 may involve their roles in cell migration and regulation of energy metabolism in mitochondria, as well as their interactions with GSK3B and PPFIBP1. However, further studies and experiments are needed to confirm any specific biological mechanisms linking these two genes.|
| [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TES](https://www.ncbi.nlm.nih.gov/gene?term=TES[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations: Both PLS1 and TES are involved in the regulation of actin cytoskeleton organization and cell adhesion, making it possible that they are functionally associated in these biological processes.   PLS1 is an actin-bundling protein that helps form stereocilia in the inner ear. Meanwhile, TES is a scaffold protein that plays a role in cell adhesion and the reorganization of the actin cytoskeleton and may act as a tumor suppressor. Through their shared involvement in actin cytoskeleton organization and cell adhesion, PLS1 and TES have potential functional associations that could impact cell growth and tumor suppression.|
| [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNL4B](https://www.ncbi.nlm.nih.gov/gene?term=TXNL4B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no known functional associations between TCFL5 and TXNL4B. TCFL5 is a putative transcription factor involved in early spermatogenesis, while TXNL4B is involved in pre-mRNA splicing and cell cycle progression. The shortest path between these two genes in the BioGRID protein interaction network includes the EGFR gene.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is currently no known functional association between POLG2 and TCFL5. POLG2 is involved in mitochondrial DNA synthesis while TCFL5 is a putative transcription factor that may play a role in spermatogenesis. The shortest path between these two genes in the BioGRID protein interaction network involves a third gene, EGFR.|
| [CASK](https://www.ncbi.nlm.nih.gov/gene?term=CASK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional association: Both CASK and PTK2 are involved in signaling pathways and can interact with other proteins to regulate cellular processes. CASK has a role in stabilizing milk micelle formation, while PTK2 has a variety of functions, including interacting with components of the extracellular matrix and mediating responses to growth factors. The shortest path in the BioGRID protein interaction network suggests that there are multiple intermediary proteins that may connect CASK and PTK2 in a signaling pathway.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CALU](https://www.ncbi.nlm.nih.gov/gene?term=CALU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no apparent functional association between BAG6 and CALU based on their known interactions and functions. BAG6 mediates ricin-induced apoptosis and interacts with bacterial proteins, while CALU is involved in regulating vitamin K-dependent carboxylation.|
| [CAST](https://www.ncbi.nlm.nih.gov/gene?term=CAST[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 37 (min len 3) | Possible functional association between CAST and TP53 is related to apoptosis regulation. CAST forms homodimers and heterodimers with BAX, BAD, BAK and Bcl-X(L) to play a key role in postmortem tenderization of meat and muscle protein degradation. TP53 acts as a tumor suppressor gene that induces growth arrest or apoptosis depending on the physiological circumstances and cell type. Both genes interact with other proteins involved in apoptotic pathways and mitochondrial function.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 469</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between ARHGAP1 and CTTN is their involvement in the regulation of actin cytoskeleton organization and cell migration. ARHGAP1 is a GTPase activator for Rho, Rac, and Cdc42 proteins, while CTTN contributes to the organization of the actin cytoskeleton and cell shape, plays a role in the formation of lamellipodia, and is involved in focal adhesion assembly and turnover.|
| [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 761</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 42 (min len 3) | IQGAP1 and NRAS are functionally associated through direct protein interactions in the BioGRID network. IQGAP1 regulates actin cytoskeleton dynamics and may play a role in cell cycle progression, while NRAS is a Ras protein involved in GDP/GTP binding and GTPase activity. These genes may be involved in regulating signal transduction pathways and downstream cellular processes, potentially contributing to cancer development and progression in TCGA-THCA.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 215</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | CEBPA and HES1 have a shortest path of interactions that include HDAC1, RB1, PARP1, and ELAVL1. CEBPA enhances ribosomal DNA transcription and interacts with Epstein-Barr virus lytic switch protein BZLF1, while HES1 acts as a transcriptional repressor of genes that require a bHLH protein. There are no clear functional associations between these two genes in specific biological terms based on the current context.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TARBP1](https://www.ncbi.nlm.nih.gov/gene?term=TARBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear functional association between NR3C1 and TARBP1. NR3C1 is a transcriptional regulator that binds to DNA and coactivators, while TARBP1 is involved in binding TAR RNA during HIV-1 infection.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TES](https://www.ncbi.nlm.nih.gov/gene?term=TES[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 270</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between BAG6 and TES in specific biological terms based on current information. BAG6 is involved in mediating ricin-induced apoptosis and interacts with bacterial proteins, while TES is a scaffold protein involved in cell adhesion and regulation of cell proliferation.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 987</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: NRAS and PIK3CA are both involved in cellular signaling pathways related to cell growth, survival, and proliferation through their interactions with other proteins, including RAS and AKT1. There may be a potential functional association between these two genes in regulating these cellular processes.|
| [RRP12](https://www.ncbi.nlm.nih.gov/gene?term=RRP12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPD52L2](https://www.ncbi.nlm.nih.gov/gene?term=TPD52L2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 160</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | Possible functional associations: based on the protein interaction network and contextual information, RRP12 and TPD52L2 may both be involved in intracellular trafficking and may play a role in endosome homeostasis. Additionally, RRP12 may play a role in regulating the dynamics and assembly of the actin cytoskeleton, while TPD52L2 may act as a GAP for Galphas. The specific processes where these genes are involved suggest possible contributions to cell cycle regulation, neurite outgrowth, and possibly in response to microbial infection. However, more research is needed to determine any direct functional associations between these two genes.|
| [AKR7A2](https://www.ncbi.nlm.nih.gov/gene?term=AKR7A2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between AKR7A2 and SLC25A14 could involve regulating cell motility and chemoattraction, as SLC25A14 interacts with proteins involved in these processes and AKR7A2 has been implicated in protecting liver against carcinogenic effects. However, more studies are needed to determine the exact relationship between these two genes.|
| [BAG3](https://www.ncbi.nlm.nih.gov/gene?term=BAG3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 353</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 174 (min len 3) | BAG3 and TP53 have a functional association through protein-protein interactions, as there are multiple shortest paths between the two genes in the BioGRID protein interaction network. BAG3 is a co-chaperone for HSP70/HSC70 proteins and has anti-apoptotic activity, while TP53 is a tumor suppressor involved in cell cycle regulation, transcription, and apoptosis.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 999</br>**BioGRID count:** 11</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and PTPN1 interact directly, and PTPN1 may regulate EGFR downstream signaling in cell proliferation. PTPN1 can also regulate the hepatocyte growth factor receptor signaling pathway through dephosphorylation of MET.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 176</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations: None described in the given context.          EGFR is a receptor involved in cellular signaling and regulation of cell proliferation. It also acts as a receptor for hepatitis C virus. G3BP1 is a helicase involved in innate immunity and stress granule formation. There are no known direct functional associations between these two genes.|
| [EIF4G1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4G1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRS1](https://www.ncbi.nlm.nih.gov/gene?term=RRS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 55 (min len 3) | There is no clear biological process that links EIF4G1 and RRS1. However, based on their interactions, we can speculate that they might be involved in regulating translation initiation and/or ribosomal subunit assembly. EIF4G1 is a component of the protein complex, eIF4F, that recognizes the mRNA cap, and ATP-dependent unwinding of 5'-terminal secondary structure. RRS1, on the other hand, is involved in ribosomal large subunit assembly and localization of the 5S RNP/5S ribonucleoprotein particle to the nucleolus.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP4K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP4K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 209</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: There is a direct protein interaction between EGFR and MAP4K4. EGFR plays a role in cellular signaling and proliferation, while MAP4K4 may play a role in the response to environmental stress and cytokines. It is possible that MAP4K4 is involved in regulating EGFR signaling in response to stress or cytokines.|
| [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 997</br>**BioGRID count:** 10</br>**BioGRID shortest paths:** 1 (min len 2) | Both genes are involved in transcriptional regulation and chromatin remodeling. TP53 is a tumor suppressor gene that induces growth arrest or apoptosis, while SMARCA4 is a component of SWI/SNF chromatin remodeling complexes that alters DNA-histone contacts in ATP-dependent manner. There may be functional associations between these two genes in regulating gene expression in cancer cells, especially those involving proliferation and apoptosis.|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: DUSP4 regulates MAPK signaling, while PTPRF is a protein phosphatase that interacts with insulin receptor and affects substrate specificity. They may function together in regulating cellular signaling pathways involving MAPK and insulin receptor pathways.|
| [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 392</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 146 (min len 3) | Possible functional associations between CTTN and TP53 could be regulation of cell migration and invasiveness of cancer cells, as CTTN is involved in the organization of the actin cytoskeleton and cell shape, and TP53 is a tumor suppressor involved in inducing growth arrest or apoptosis in many tumor types. CTTN may also play a role in regulating the levels of potassium channels present at the cell membrane, and interact with TP53 in forming a protein interaction network involving the shortest path between the two genes.|
| [CASK](https://www.ncbi.nlm.nih.gov/gene?term=CASK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 208</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CASK and PLS1 are related to actin bundling and stereocilia formation. CASK is involved in stabilizing micelle formation, while PLS1 is an actin-bundling protein that is required for stereocilia formation. The shortest path between these two genes in the BioGRID protein interaction network suggests they may interact in the inner ear to mediate liquid packaging of actin filaments necessary for stereocilia to grow to their proper dimensions.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 964</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between NRAS and RASA1 is in the regulation of the Ras-cyclic AMP pathway. NRAS is a Ras protein that binds GDP/GTP and possesses intrinsic GTPase activity, while RASA1 is an inhibitory regulator of the pathway that stimulates the GTPase of normal but not oncogenic Ras p21.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 985</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and NRAS are both involved in the regulation of cell signaling pathways. They may interact in the MAPK/ERK pathway, which is important for cell proliferation and differentiation. The shortest path between EGFR and NRAS in the BioGRID protein interaction network suggests a direct physical interaction between these two proteins.|
