| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SSBP2](https://www.ncbi.nlm.nih.gov/gene?term=SSBP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between RBM6 and SSBP2 include regulation of RNA processing and ERK signaling pathway, with RBM6 binding to poly(G) RNA homopolymers and SSBP2 inhibiting the dephosphorylation of ERK by phosphatase PP2A-PPP2R5C holoenzyme and mediating survival. While there is no direct evidence of physical interaction between these genes in literature, the shortest path in the BioGRID protein interaction network suggests possible indirect interaction through ARHGAP22.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations: ARHGAP1 is a GTPase activator for Rho, Rac and Cdc42 proteins, while BAG6 mediates ricin-induced apoptosis and interacts with L. pneumophila Lpg2160 and LegU1 proteins. Based on these functions, it is possible that ARHGAP1 and BAG6 may be involved in regulating cellular signaling pathways related to apoptosis and microbial infection.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 331</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between BAG6 and TP53 include BAG6's ability to mediate ricin-induced apoptosis and its interaction with L. pneumophila Lpg2160 and LegU1 proteins, while TP53 acts as a tumor suppressor and induces growth arrest or apoptosis depending on the physiological circumstances and cell type, involved in cell cycle regulation, and cooperates with mitochondrial PPIF to activate oxidative stress-induced necrosis. However, there is currently no direct evidence of a functional association between these two genes.|
| [ADAT1](https://www.ncbi.nlm.nih.gov/gene?term=ADAT1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EIF4G1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4G1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ADAT1 and EIF4G1 are involved in translation initiation, given that EIF4G1 is a component of the eIF4F complex required for mRNA cap recognition and ribosome recruitment, while ADAT1 may affect translation initiation by specifically deaminating adenosine in tRNA. However, without further evidence, the precise nature and significance of these associations remain unclear.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBBP6](https://www.ncbi.nlm.nih.gov/gene?term=RBBP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 477 (min len 4) | There is currently no known functional association between C2CD2 and RBBP6. C2CD2 plays a role in DNA repair and may promote apoptosis, while RBBP6 restricts Ebola virus replication and interacts with VP30.|
| [DAG1](https://www.ncbi.nlm.nih.gov/gene?term=DAG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP4K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP4K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 24 (min len 3) | Possible functional associations between DAG1 and MAP4K4 involve pathways related to environmental stress response and cytokine signaling. DAG1 also acts as a receptor for certain microbial infections, while MAP4K4 phosphorylates SMAD1 and interacts with adapter proteins NCK and ATL1. They may play a role in immune response and cellular signaling.|
| [PRUNE](https://www.ncbi.nlm.nih.gov/gene?term=PRUNE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 4) | Possible functional association: PRUNE is involved in cell proliferation, migration, and neurogenesis, while SLC25A14 participates in mitochondrial proton leak, promotes lymphocyte movement and increases cell motility. These genes may have a role in cancer cell growth and metastasis. The shortest path between them suggests that they may interact at the level of microtubule polymerization or through the regulation of GSK3.|
| [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TES](https://www.ncbi.nlm.nih.gov/gene?term=TES[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 15 (min len 3) | Possible functional associations between PLS1 and TES include their involvement in the regulation of the actin cytoskeleton, cell adhesion, and cell proliferation. The shortest path in the BioGRID interaction network between PLS1 and TES involves several proteins that also play a role in actin cytoskeleton regulation and cell adhesion, suggesting a possible functional relationship between these genes in these processes.|
| [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TXNL4B](https://www.ncbi.nlm.nih.gov/gene?term=TXNL4B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: TCFL5 and TXNL4B may interact in pre-mRNA splicing and cell cycle progression.   TCFL5 is a transcription factor involved in early spermatogenesis while TXNL4B plays an essential role in pre-mRNA splicing and interacts with the spliceosome U5-102 kDa protein subunit. The shortest path between TCFL5 and TXNL4B in the BioGRID protein interaction network involves EGFR, which suggests a possible role in cell growth and differentiation.|
| [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TCFL5](https://www.ncbi.nlm.nih.gov/gene?term=TCFL5[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between POLG2 and TCFL5 may be in DNA synthesis and transcription regulation during spermatogenesis, as both are involved in DNA binding and polymerase activity. The shortest path in the BioGRID protein interaction network suggests that EGFR may also play a role in the potential interaction between these two genes.|
| [CASK](https://www.ncbi.nlm.nih.gov/gene?term=CASK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTK2](https://www.ncbi.nlm.nih.gov/gene?term=PTK2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 174</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional association: Both CASK and PTK2 are involved in neural development and play roles in synaptic protein anchoring and ion channel trafficking. CASK interacts with transcription factors and cell-surface proteins, while PTK2 interacts with proteins involved in cell adhesion and signaling. It is possible that CASK and PTK2 could interact or modulate each other's activity in neural processes related to synaptic signaling and adhesion.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CALU](https://www.ncbi.nlm.nih.gov/gene?term=CALU[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no strong evidence to suggest a direct functional association between BAG6 and CALU, as they only have a single interaction in the BioGRID network. Individually, BAG6 plays a role in mediating ricin-induced apoptosis, while CALU is involved in the regulation of vitamin K-dependent carboxylation.|
| [CAST](https://www.ncbi.nlm.nih.gov/gene?term=CAST[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 37 (min len 3) | Possible functional associations between CAST and TP53 could be related to the regulation of apoptosis. CAST is involved in the anti-apoptotic activity of Bcl-2 and interacts with TP53. TP53 is a tumor suppressor gene that induces apoptosis and negatively regulates cell division. The shortest path in the BioGRID protein interaction network between CAST and TP53 includes other genes involved in apoptosis, such as CAPN1 and BAX, suggesting a potential connection in this biological process.|
| [ARHGAP1](https://www.ncbi.nlm.nih.gov/gene?term=ARHGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 469</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: ARHGAP1 is involved in regulating the activity of Rho, Rac, and Cdc42 proteins, while CTTN contributes to the organization of the actin cytoskeleton and cell shape. These processes are interconnected, as Rho GTPases, including Cdc42, regulate actin cytoskeleton organization and dynamics. Therefore, these two genes could potentially interact in controlling cellular functions related to cytoskeletal dynamics.|
| [IQGAP1](https://www.ncbi.nlm.nih.gov/gene?term=IQGAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 761</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 42 (min len 3) | Possible functional associations between IQGAP1 and NRAS are in the regulation of actin cytoskeleton dynamics and cell cycle progression as both genes are involved in these processes. IQGAP1 also plays a role in promoting neurite outgrowth while NRAS interacts with RGS14 and RASSF7.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HES1](https://www.ncbi.nlm.nih.gov/gene?term=HES1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 215</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between CEBPA and HES1 could involve transcriptional regulation and cell cycle control. CEBPA directly enhances ribosomal DNA transcription and interacts with RNA polymerase I-specific cofactors, while HES1 is a transcriptional repressor that may inhibit myogenesis, but also plays a role in the response to DNA cross-link damage. The shortest path between the two genes in the BioGRID protein interaction network involves HDAC1, RB1, PARP1, and ELAVL1, suggesting a potential regulatory or signaling interaction between them.|
| [NR3C1](https://www.ncbi.nlm.nih.gov/gene?term=NR3C1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TARBP1](https://www.ncbi.nlm.nih.gov/gene?term=TARBP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between NR3C1 and TARBP1 is unclear based on the context provided. NR3C1 encodes a transcription factor involved in regulation of gene expression, while TARBP1 is involved in RNA processing.|
| [BAG6](https://www.ncbi.nlm.nih.gov/gene?term=BAG6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TES](https://www.ncbi.nlm.nih.gov/gene?term=TES[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 270</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between BAG6 and TES. BAG6 mediates apoptosis and interacts with microbial proteins, while TES is a scaffold protein that may play a role in cell adhesion and inhibit tumor cell growth.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PIK3CA](https://www.ncbi.nlm.nih.gov/gene?term=PIK3CA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 987</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: NRAS and PIK3CA are both involved in the regulation of cell growth, survival, and proliferation through the activation of signaling cascades. In addition, they both interact with downstream effectors in these pathways, including AKT1 and RAS, suggesting a potential crosstalk between these two genes in driving cancer progression.|
| [RRP12](https://www.ncbi.nlm.nih.gov/gene?term=RRP12[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TPD52L2](https://www.ncbi.nlm.nih.gov/gene?term=TPD52L2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 160</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 17 (min len 3) | RRP12 and TPD52L2 both have roles in intracellular trafficking and could potentially interact in this process. RRP12 also regulates actin cytoskeleton organization and may contribute to cell cycle progression, while TPD52L2 acts as a GAP for Galphas and may play a role in endosome homeostasis.|
| [AKR7A2](https://www.ncbi.nlm.nih.gov/gene?term=AKR7A2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 161</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between AKR7A2 and SLC25A14 is related to the regulation of cell motility and potential involvement in protecting the liver against carcinogenic effects. AKR7A2 acts as an aldehyde reductase, while SLC25A14 participates in mitochondrial proton leak and interacts with MYH9 to increase cell motility.|
| [BAG3](https://www.ncbi.nlm.nih.gov/gene?term=BAG3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 353</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 174 (min len 3) | BAG3 and TP53 are functionally associated in TCGA-THCA. BAG3 is involved in anti-apoptotic activity and nucleotide-exchange factor promotion while TP53 is a tumor suppressor involved in cell cycle regulation and inducing growth arrest or apoptosis. There is a shortest path of protein interactions between BAG3 and TP53 in the BioGRID interaction network.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPN1](https://www.ncbi.nlm.nih.gov/gene?term=PTPN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 999</br>**BioGRID count:** 11</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association: EGFR is a receptor for growth factors that mediates cell proliferation and differentiation, while PTPN1 is a phosphatase that regulates signaling pathways by dephosphorylating tyrosine residues in target proteins. It is possible that PTPN1 regulates EGFR signaling by dephosphorylating EGFR or its downstream effectors.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [G3BP1](https://www.ncbi.nlm.nih.gov/gene?term=G3BP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 176</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on their roles in stress granule formation and innate immunity, it is possible that EGFR and G3BP1 may functionally associate in response to cellular stress and infection. EGFR acts as a receptor in HCV infection and mediates downstream signaling, while G3BP1 is an ATP- and magnesium-dependent helicase that plays a role in stress granule formation and participates in the DNA-triggered cGAS/STING pathway.|
| [EIF4G1](https://www.ncbi.nlm.nih.gov/gene?term=EIF4G1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RRS1](https://www.ncbi.nlm.nih.gov/gene?term=RRS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 166</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 55 (min len 3) | Possible functional associations between EIF4G1 and RRS1 include roles in mRNA translation and ribosomal subunit assembly, respectively. While there is not enough information to definitively determine a specific interaction between these two genes, both genes play important roles in protein synthesis, suggesting a potential functional relationship.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAP4K4](https://www.ncbi.nlm.nih.gov/gene?term=MAP4K4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 209</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional association between EGFR and MAP4K4 is their involvement in the response to environmental stress and cytokines, such as TNF-alpha. EGFR acts as a receptor for hepatitis C virus and is essential for HCV entry, while MAP4K4 appears to act upstream of the JUN N-terminal pathway and phosphorylates SMAD1. No direct functional association between these two genes is known, but both play important roles in cellular signaling and response to stress and inflammation.|
| [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 997</br>**BioGRID count:** 10</br>**BioGRID shortest paths:** 1 (min len 2) | Both SMARCA4 and TP53 have been implicated in regulation of transcription and cell cycle control. Possible functional associations between the two genes could include cooperation in the control of cell proliferation, apoptosis, or DNA damage response.|
| [DUSP4](https://www.ncbi.nlm.nih.gov/gene?term=DUSP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRF](https://www.ncbi.nlm.nih.gov/gene?term=PTPRF[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 204</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: Both DUSP4 and PTPRF are involved in regulating signal transduction pathways. DUSP4 dephosphorylates MAP kinases, while PTPRF is a protein tyrosine phosphatase that interacts with insulin receptor substrate. There may be a functional association in the regulation of cellular signaling.|
| [CTTN](https://www.ncbi.nlm.nih.gov/gene?term=CTTN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 392</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 146 (min len 3) | Possible functional associations between CTTN and TP53 include their roles in cancer, cell shape and migration, and endocytosis. TP53 acts as a tumor suppressor while CTTN has a role in the invasiveness of cancer cells and the formation of metastases. Both genes play a role in cell shape and migration, as well as in endocytosis via clathrin-coated pits.|
| [CASK](https://www.ncbi.nlm.nih.gov/gene?term=CASK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PLS1](https://www.ncbi.nlm.nih.gov/gene?term=PLS1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 208</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between CASK and PLS1 include a role in actin cytoskeleton organization and ion channel trafficking in the inner ear. CASK is a scaffolding protein involved in synaptic transmembrane protein anchoring and interacts with PLS1, which is an actin-bundling protein required for stereocilia formation in the inner ear.|
| [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RASA1](https://www.ncbi.nlm.nih.gov/gene?term=RASA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 964</br>**BioGRID count:** 2</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between NRAS and RASA1 include their involvement in the Ras-cyclic AMP pathway and their ability to interact with each other as well as with other proteins such as NCK1 and PDGFRB. Both genes play roles in regulating cellular signaling and proliferation, making their interaction relevant in cancer biology.|
| [EGFR](https://www.ncbi.nlm.nih.gov/gene?term=EGFR[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NRAS](https://www.ncbi.nlm.nih.gov/gene?term=NRAS[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.6</br>**STRING score:** 985</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | EGFR and NRAS both play important roles in cell signaling pathways that regulate cell growth and division. There is a strong possibility that they may interact with each other to promote these processes in specific biological contexts, such as cancer. The shortest path between EGFR and NRAS in the BioGRID protein interaction network indicates a direct physical association between the two proteins.|
