| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional associations: TBP is involved in RNA polymerase II-dependent transcription, while FASTKD5 plays a role in processing non-canonical mitochondrial mRNA precursors. There is a protein interaction between FASTKD5 and TBP via multiple intermediary proteins. The specific biological context in which these two genes may function together is currently unclear.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 4) | Possible functional associations between ADO and SLC25A14 could involve regulation of metabolism and mitochondrial proton leakage. ADO plays a role in regulating thiol metabolism, while SLC25A14 participates in the mitochondrial proton leak. They also have separate protein interactions, but no direct interaction with each other.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1309 (min len 4) | Possible functional association: COX8A is a component of the mitochondrial electron transport chain while C2CD2 is involved in DNA repair and promotes DNA damage-induced apoptosis. There may be an indirect connection between these genes through SLC38A2, a common protein interacting partner.  |
| [ITFG1](https://www.ncbi.nlm.nih.gov/gene?term=ITFG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations between ITFG1 and RBM6 could involve regulation of mRNA processing and splicing, as RBM6 is involved in RNA binding and ITFG1 interacts with proteins involved in mRNA splicing. Additionally, ITFG1 has a protective effect in graft versus host disease and interacts with proteins involved in microtubule dynamics.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 209 (min len 4) | Possible functional associations: ADO plays a role in regulating thiol metabolism and oxidizes cysteamine to hypotaurine, while TBP is a general transcription factor that binds to and bends promoter DNA, irrespective of whether the promoter contains a TATA box. There is no obvious direct functional association between these two genes based on their known biological functions.|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCYL3](https://www.ncbi.nlm.nih.gov/gene?term=SCYL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | There are no known functional associations between FASTKD5 and SCYL3 based on the provided context. FASTKD5 is involved in processing non-canonical mitochondrial mRNA precursors, while SCYL3 may play a role in regulating cell adhesion/migration complexes in migrating cells.|
| [LAGE3](https://www.ncbi.nlm.nih.gov/gene?term=LAGE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCRN1](https://www.ncbi.nlm.nih.gov/gene?term=SCRN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 375 (min len 4) | Possible functional associations between LAGE3 and SCRN1 include involvement in base excision repair and exocytosis regulation in mast cells. They are connected in the BioGRID protein interaction network through multiple paths involving the TRIM67 protein, suggesting a potential role in protein degradation or signaling pathways.|
| [RAB27A](https://www.ncbi.nlm.nih.gov/gene?term=RAB27A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 4) | Possible functional associations between RAB27A and SLC25A14 include their involvement in mitochondrial function and regulation of cellular trafficking and secretion. RAB27A cycles between active and inactive states to regulate endosomal positioning, maturation, and secretion, and is required for granule maturation and docking in lymphocytes. SLC25A14 participates in mitochondrial proton leak and interacts with PPFIBP1 in a calcium-dependent mode. There is no known direct interaction between RAB27A and SLC25A14, but they both could be involved in intracellular transport and secretion, which are processes that require proper mitochondrial function.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between ADO and RBM6 can be related to RNA binding and regulation of gene expression, as RBM6 is known to bind RNA and ADO plays a role in regulating thiol metabolism. However, further experimental evidence would be needed to confirm any direct interactions or relationships between these two genes.|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between BAD and JMJD6 is through their roles in regulating cell death and apoptosis. BAD promotes cell death and competes for binding to anti-apoptotic proteins, while JMJD6 regulates RNA splicing and is necessary for the regulation of macrophage cytokine responses. However, there is currently no known direct biological interaction between the two genes.|
| [CPSF4](https://www.ncbi.nlm.nih.gov/gene?term=CPSF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional association: CPSF4 and TBP are both involved in transcription. CPSF4 is part of a complex that recognizes and cleaves pre-mRNA while TBP is a general transcription factor that binds to promoter DNA and promotes assembly of the pre-initiation complex. It is possible that CPSF4 and TBP cooperate in regulating gene expression.|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 391</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association between FASTKD5 and POLG2 may be their involvement in mitochondrial RNA processing and regulation of mitochondrial DNA synthesis, respectively. They both play important roles in mitochondrial function, suggesting potential interactions in cellular processes related to energy production.|
| [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 919</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | TBP and TBPL1 are involved in transcriptional regulation. TBP is a general transcription factor that binds to promoters to recruit and assemble the pre-initiation complex, while TBPL1 is involved in specialized transcription of ribosomal proteins and may also mediate transcription of NF1. There is a direct protein interaction between TBP and TBPL1, suggesting they may cooperatively regulate transcription in some biological contexts.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBBP6](https://www.ncbi.nlm.nih.gov/gene?term=RBBP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 277</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 236 (min len 4) | Possible functional associations between ATMIN and RBBP6 may involve their role in DNA damage response and transcriptional regulation. ATMIN is involved in cell survival and RAD51 foci formation in response to methylating DNA damage, while RBBP6 interacts with ebolavirus VP30 and may restrict viral transcription. However, without further experimental evidence, it is difficult to make specific claims regarding their interactions in biological systems.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between CEBPA and JMJD6 could be in regulating RNA splicing and histone modifications, as well as in promoting differentiation of multiple organs during embryogenesis and regulating cytokine responses in macrophages. In specific biological terms, CEBPA enhances ribosomal DNA transcription and induces histone acetylation, while JMJD6 can act as an arginine demethylase and a lysyl-hydroxylase, regulating RNA splicing and histone code, as well as being necessary for the regulation of macrophage cytokine responses. The two genes may work together in controlling these biological processes.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 806</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 59 (min len 3) | Possible functional associations between JMJD6 and TP53 include regulation of gene expression, cell cycle control, and apoptosis. JMJD6 acts as an RNA and histone demethylase, and is involved in pre-mRNA splicing and transcriptional activation. TP53 is a tumor suppressor that induces growth arrest or apoptosis, and controls a set of genes required for cell cycle regulation. The protein interaction network shows that JMJD6 and TP53 are connected via multiple pathways, including BRD4 and CCNL1, suggesting potential co-regulation of target genes.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 438</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | Both JMJD6 and SMARCA4 are involved in chromatin remodeling and transcriptional regulation. The BioGRID protein interaction network shows that they can interact through multiple intermediate proteins, suggesting a potential functional association in these processes.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between PRKAG2 and TBP. PRKAG2 is involved in regulating energy metabolism and cellular polarity, while TBP plays a major role in the initiation of RNA polymerase II-dependent transcription. However, they both interact with other proteins in their respective pathways and may have overlapping roles in cellular processes.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 298</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between APOE and CEBPA: APOE interacts with HCV envelope glycoprotein E2 and is involved in the assembly, maturation, and infectivity of HCV viral particles, while CEBPA enhances ribosomal DNA transcription and interacts with Epstein-Barr virus lytic switch protein BZLF1 to induce G1 cell cycle arrest. There is no known direct functional relationship between APOE and CEBPA based on the provided context, but both genes are involved in microbial infection processes.|
