| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Both FASTKD5 and TBP are involved in transcription regulation. The shortest path between them in the BioGRID protein interaction network suggests that they may have a functional association through other intermediary proteins, such as RNF4, CLPP, CD3EAP, WDR5, and MYC.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 4) | Possible functional associations between ADO and SLC25A14 are not immediately apparent. ADO oxidizes sulfur-containing molecules and regulates thiol metabolism. SLC25A14 interacts with various proteins and participates in mitochondrial proton leak and lymphocyte movement.|
| [C2CD2](https://www.ncbi.nlm.nih.gov/gene?term=C2CD2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COX8A](https://www.ncbi.nlm.nih.gov/gene?term=COX8A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1309 (min len 4) | Possible functional association: C2CD2 interacts with AIFM1, promoting DNA damage-induced apoptosis. COX8A is a component of cytochrome c oxidase, the last enzyme in the mitochondrial electron transport chain. There is no known direct functional association between these two genes, as they function in different biological processes.|
| [ITFG1](https://www.ncbi.nlm.nih.gov/gene?term=ITFG1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 16 (min len 3) | Possible functional associations: ITFG1 and RBM6 interact through various proteins involved in RNA splicing and modulating T-cell function, suggesting potential involvement in these processes in the context of THCA.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 209 (min len 4) | There is no known functional association between ADO and TBP. ADO is involved in regulating thiol metabolism and preserving oxygen homeostasis, while TBP is a component of the TFIID basal transcription factor complex that plays a major role in the initiation of RNA polymerase II-dependent transcription.|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCYL3](https://www.ncbi.nlm.nih.gov/gene?term=SCYL3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 6.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: There is no known functional association between FASTKD5 and SCYL3 based on available literature and protein interaction networks.  FASTKD5 functions in the processing of non-canonical mitochondrial mRNA precursors while SCYL3 may play a role in regulating cell adhesion/migration complexes.|
| [LAGE3](https://www.ncbi.nlm.nih.gov/gene?term=LAGE3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SCRN1](https://www.ncbi.nlm.nih.gov/gene?term=SCRN1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 375 (min len 4) | Based on the BioGRID protein interaction network, LAGE3 and SCRN1 are functionally associated through TRIM67 and may be involved in regulating exocytosis and/or base excision repair processes. LAGE3 is a component of the EKC/KEOPS complex required for tRNA modification, while SCRN1 is a regulator of exocytosis and a component of a base excision repair complex.|
| [RAB27A](https://www.ncbi.nlm.nih.gov/gene?term=RAB27A[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SLC25A14](https://www.ncbi.nlm.nih.gov/gene?term=SLC25A14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 69 (min len 4) | Possible functional associations between RAB27A and SLC25A14 include regulation of endosomal positioning, maturation, and secretion, as well as participation in mitochondrial proton leak. There is no direct biological mechanism linking the two genes, but both are involved in cellular processes such as granule exocytosis and motility that are important for immune response and survival.|
| [ADO](https://www.ncbi.nlm.nih.gov/gene?term=ADO[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBM6](https://www.ncbi.nlm.nih.gov/gene?term=RBM6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations: ADO plays a role in regulating thiol metabolism and oxidizing cysteine-containing proteins, while RBM6 binds to poly(G) RNA homopolymers. It is possible that RBM6 may play a role in regulating the expression or translation of ADO in the context of RNA metabolism.|
| [BAD](https://www.ncbi.nlm.nih.gov/gene?term=BAD[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between BAD and JMJD6 is in the regulation of apoptosis and RNA splicing. BAD promotes cell death by affecting the level of heterodimerization of proteins with BAX and interacts with NOL3, whereas JMJD6 mediates RNA splicing by 5-hydroxylation of U2AF2/U2AF65. There is currently no known direct interaction between these genes, but they may play complementary roles in regulating cellular processes.|
| [CPSF4](https://www.ncbi.nlm.nih.gov/gene?term=CPSF4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 165</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between CPSF4 and TBP may involve regulation and initiation of transcription. CPSF4 is involved in pre-mRNA 3'-end formation and interacts with factors to bring about cleavage and poly(A) addition, while TBP is a general transcription factor that binds to and bends promoter DNA to promote assembly of the pre-initiation complex and regulate RNA polymerase II-dependent transcription.|
| [FASTKD5](https://www.ncbi.nlm.nih.gov/gene?term=FASTKD5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [POLG2](https://www.ncbi.nlm.nih.gov/gene?term=POLG2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 391</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between FASTKD5 and POLG2 may involve their roles in mitochondrial RNA processing and DNA synthesis, respectively, as indicated by their context. POLG2 is involved in mitochondrial DNA replication while FASTKD5 plays a role in processing non-canonical mitochondrial mRNA precursors. The two genes may interact through a shared function in mitochondrial metabolism.|
| [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBPL1](https://www.ncbi.nlm.nih.gov/gene?term=TBPL1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.9</br>**STRING score:** 919</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between TBP and TBPL1 may involve their roles in transcription regulation and complex assembly. TBP is a general transcription factor that binds DNA and promotes assembly of the pre-initiation complex, while TBPL1 is involved in mediating transcription of certain genes through specialized promoter elements. It is possible that they may interact to form a complex involved in specific transcriptional regulation processes.|
| [ATMIN](https://www.ncbi.nlm.nih.gov/gene?term=ATMIN[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RBBP6](https://www.ncbi.nlm.nih.gov/gene?term=RBBP6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 277</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 236 (min len 4) | Possible functional association: ATMIN and RBBP6 interact via TP53 in the protein interaction network, suggesting they may work together in DNA damage response pathways. ATMIN plays a role in cell survival and DNA damage repair, while RBBP6 restricts viral replication and interacts with viral proteins, suggesting a possible role in antiviral defense mechanisms.|
| [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.8</br>**STRING score:** 167</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | CEBPA and JMJD6 may both play a role in transcriptional regulation, with CEBPA specifically enhancing ribosomal DNA transcription and JMJD6 interacting with the P-TEFb complex to regulate polymerase II promoter-proximal pause release for transcriptional activation. It is possible that CEBPA and JMJD6 function together to regulate gene expression in a coordinated manner. However, further functional studies would be needed to confirm any direct functional associations between these two genes.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TP53](https://www.ncbi.nlm.nih.gov/gene?term=TP53[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 806</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 59 (min len 3) | Possible functional associations between JMJD6 and TP53 are related to their involvement in apoptosis, cell cycle regulation, and RNA splicing. TP53 acts as a tumor suppressor and induces growth arrest or apoptosis, while JMJD6 regulates RNA splicing and plays a role in hematopoietic differentiation. Their shortest path in the BioGRID protein interaction network involves intermediary interactions with BRD4, a positive transcription elongation factor, and CCNL1, a cyclin-like protein involved in cell cycle regulation.|
| [JMJD6](https://www.ncbi.nlm.nih.gov/gene?term=JMJD6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMARCA4](https://www.ncbi.nlm.nih.gov/gene?term=SMARCA4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 438</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 23 (min len 3) | JMJD6 is involved in RNA splicing and histone demethylation, while SMARCA4 is involved in chromatin remodeling. The shortest path in the protein interaction network suggests that these genes may function together in regulating transcription elongation and promoter-proximal pause release.|
| [PRKAG2](https://www.ncbi.nlm.nih.gov/gene?term=PRKAG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TBP](https://www.ncbi.nlm.nih.gov/gene?term=TBP[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between PRKAG2 and TBP could involve regulation of gene expression and metabolism. PRKAG2 plays a role in regulating cellular energy metabolism, while TBP is involved in initiating RNA polymerase II-dependent transcription. It is possible that PRKAG2 may regulate the activity of TBP and affect transcription of certain genes involved in metabolism.|
| [APOE](https://www.ncbi.nlm.nih.gov/gene?term=APOE[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CEBPA](https://www.ncbi.nlm.nih.gov/gene?term=CEBPA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 5.7</br>**STRING score:** 298</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are currently no known functional associations between APOE and CEBPA in specific biological terms. APOE is involved in lipid transport and viral infection while CEBPA enhances ribosomal DNA transcription and interacts with Epstein-Barr virus lytic switch protein BZLF1 to induce G1 cell cycle arrest.|
