| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 229.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: One possible functional association between EVL and FBXO11 could be related to cell migration and cytoskeleton remodeling, as both genes are involved in these processes based on their described functions. Specifically, EVL enhances actin nucleation and polymerization, while FBXO11 is involved in the regulation of TGF-beta signaling and cell migration through degradation of DTL and its interaction with p53/TP53.|
| [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 221.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no clear functional association between RAC2 and ZNF131 based on their known processes and interactions. RAC2 is involved in regulating cellular responses and production of reactive oxygen species, while ZNF131 plays a role in transcriptional regulation and interacts with various receptors. The shortest path between them in the BioGRID network involves an intermediate protein, LGALS1 and KIAA1109, suggesting indirect interactions.|
| [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 211.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: DYNLT3 and MNAT1 are involved in cell cycle control and RNA transcription. DYNLT3 links dynein to cargos and adapter proteins, while MNAT1 stabilizes the cyclin H-CDK7 complex to activate RNA polymerase II. There is no clear biological interaction between these two genes in this cancer type.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 208.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | CD40 is a receptor for TNFSF5/CD40L and activates ERK signaling in macrophages and B cells. SNX6 is involved in intracellular trafficking processes, participates in endosome-to-TGN transport, and acts as a component of the retromer membrane-deforming SNX-BAR subcomplex. Given their roles, a possible functional association between these genes could involve CD40-mediated signaling affecting SNX6-mediated intracellular trafficking.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 205.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1243 (min len 4) | Possible functional association between CD40 and SMNDC1 could be related to the regulation of RNA splicing. CD40 is known to activate ERK in macrophages and B cells, leading to induction of immunoglobulin secretion, while SMNDC1 is involved in spliceosome assembly and interacts with U2 and U4/U5/U6 tri-snRNPs. It is possible that CD40 may regulate splicing by interacting with SMNDC1.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 204.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 218 (min len 4) | Possible functional association: PSMD10 is involved in regulating the proteasomal degradation of RB1, while GADD45B is involved in growth regulation and interacting with stress-responsive MAPKKKs. Both genes are involved in cell growth regulation and could potentially interact in this process. However, this hypothesis requires further investigation.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 202.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations: PHGDH catalyzes the reversible oxidation of 3-phospho-D-glycerate to 3-phosphonooxypyruvate, the first step of the phosphorylated L-serine biosynthesis pathway, while RALA is a multifunctional GTPase involved in various cellular processes. There may be a functional association between these genes in regulating cell proliferation and growth signaling. The bioGRID protein interaction network shows a possible interaction between PHGDH and RALA via CUL7, CCDC8, NTRK1, TRIM25, and HDAC6.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOXJ3](https://www.ncbi.nlm.nih.gov/gene?term=FOXJ3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 370 (min len 4) | Based on their shortest path in the BioGRID protein interaction network, CD40 and FOXJ3 could potentially functionally associate through ESR2, a transcription factor involved in regulating gene expression in both cancer and non-cancer cells. CD40 is a receptor involved in B-cell activation and immunoglobulin secretion, while FOXJ3 is a transcriptional activator important for skeletal muscle regeneration and spermatogenesis.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between COPS5 and GADD45B is their involvement in the regulation of growth and apoptosis. COPS5 acts as a protease subunit of the COP9 signalosome complex, which is involved in various cellular and developmental processes. GADD45B mediates activation of stress-responsive MAPKKK. The shortest path between these two genes in the BioGRID protein interaction network involves an intermediate protein, GADD45GIP1.|
| [DRAP1](https://www.ncbi.nlm.nih.gov/gene?term=DRAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FEZ2](https://www.ncbi.nlm.nih.gov/gene?term=FEZ2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.8</br>**STRING score:** 404</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided context and my knowledge of biology, there is no clear functional association between DRAP1 and FEZ2. DRAP1 is involved in transcriptional regulation and can bind DNA, while FEZ2 is involved in axonal outgrowth and fasciculation. Their roles are not directly related and there is no evidence of interaction or shared pathways between them.|
| [FEZ2](https://www.ncbi.nlm.nih.gov/gene?term=FEZ2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 187.9</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | RAC2 is involved in cellular responses such as phagocytosis and epithelial cell polarization, and interacts with FEZ2 through the shortest path in the BioGRID protein interaction network. There may be a functional association between these genes in regulating cellular processes related to cell polarity and phagocytosis.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 182.2</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 47 (min len 3) | Possible functional associations between COPS5 and NFKBIA include regulation of the ubiquitin conjugation pathway and phosphorylation of key transcription factors such as TP53, JUN, and NFKBIA itself. COPS5, as a protease subunit of the COP9 signalosome complex, is involved in deneddylation of cullin subunits and may contribute to the proteasomal degradation of BRSK2. NFKBIA inhibits dimeric NF-kappa-B/REL complexes by masking their nuclear localization signals, and its phosphorylation promotes ubiquitination and degradation, leading to activation of transcription.|
| [DSG2](https://www.ncbi.nlm.nih.gov/gene?term=DSG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are currently no known functional associations between DSG2 and PAN2. DSG2 is involved in cell-cell adhesion while PAN2 is a retinol dehydrogenase. Their biological functions do not overlap.|
| [DUSP6](https://www.ncbi.nlm.nih.gov/gene?term=DUSP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.4</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are currently no known functional associations between DUSP6 and PTPRC in TCGA-SKCM. DUSP6 is involved in cell differentiation by regulating MAPK1/MAPK3 activity and interacts with MAPK1/ERK2, while PTPRC acts as a receptor for human cytomegalovirus protein UL11 and mediates binding of UL11 to T-cells.|
| [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 169.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between RAC2 and TM9SF3 may involve regulation of cellular processes such as secretory processes, phagocytosis of apoptotic cells, and epithelial cell polarization, as RAC2 interacts with DOCK2 and S100A8/9 while TM9SF3 is required for genome-wide de novo methylation and histone modification. However, more information is needed to determine the exact nature of their interaction.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 167.1</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between NFKBIA and USP1 may involve DNA damage repair and regulation of transcription. NFKBIA inhibits the activity of dimeric NF-kappa-B/REL complexes and USP1 is a negative regulator of DNA damage repair and interacts with FANCD2, PCNA, and WDR48. Together, they may play a role in regulating cell growth and survival in response to oxidative stress and DNA damage.|
| [CDK7](https://www.ncbi.nlm.nih.gov/gene?term=CDK7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There are no known functional associations between CDK7 and EVL in specific biological terms. CDK7 is a kinase involved in cell cycle control and RNA transcription, while EVL promotes actin nucleation and polymerization in processes such as cell polarity and migration.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 165.9</br>**STRING score:** 188</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | Possible functional associations: FKBP4 is involved in protein folding and trafficking, while PSMD2 is part of the proteasome complex. They may interact in the degradation of misfolded proteins, which could play a role in cancer development.|
