| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 229.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between EVL and FBXO11 may involve the regulation of cytoskeleton organization and protein degradation through the SCF (SKP1-CUL1-F-box protein) E3 ubiquitin-protein ligase complex. EVL enhances actin nucleation and polymerization, while FBXO11 mediates ubiquitination and subsequent proteasomal degradation of target proteins such as DTL/CDT2 and BCL6. The shortest path between EVL and FBXO11 in the BioGRID protein interaction network involves TP53, which is involved in cell cycle regulation and apoptosis.|
| [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF131](https://www.ncbi.nlm.nih.gov/gene?term=ZNF131[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 221.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There is no obvious functional association between RAC2 and ZNF131 based on the information provided. RAC2 is involved in plasma membrane-associated processes such as secretion and phagocytosis, while ZNF131 plays a role in transcriptional regulation and development. The shortest path between the two genes in the BioGRID protein interaction network involves LGALS1 and KIAA1109.|
| [DYNLT3](https://www.ncbi.nlm.nih.gov/gene?term=DYNLT3[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MNAT1](https://www.ncbi.nlm.nih.gov/gene?term=MNAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 211.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | DYNLT3 is involved in transporting cellular cargo, while MNAT1 is involved in activating cyclin-associated kinases and RNA polymerase II. There is no known direct functional association between the two genes, but both are involved in crucial cellular processes necessary for proper cell division and transcription.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX6](https://www.ncbi.nlm.nih.gov/gene?term=SNX6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 208.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Based on their known functions and interactions, it is not immediately clear what specific functional associations these two genes may have in TCGA-SKCM. However, CD40 is involved in immune activation and signaling pathways, while SNX6 plays a role in intracellular trafficking and sorting. These genes may be involved in different aspects of cancer development and progression.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SMNDC1](https://www.ncbi.nlm.nih.gov/gene?term=SMNDC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 205.8</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1243 (min len 4) | Possible functional association between CD40 and SMNDC1 is spliceosome assembly. CD40 is involved in transducing signals that activate ERK, leading to induction of immunoglobulin secretion, while SMNDC1 associates with spliceosomes and interacts with U4/U5/U6 tri-snRNP and U2 snRNP in spliceosome assembly.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD10](https://www.ncbi.nlm.nih.gov/gene?term=PSMD10[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 204.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 218 (min len 4) | Possible functional associations between GADD45B and PSMD10 include their involvement in the regulation of cell growth and apoptosis, as well as their roles in the regulation of RB1 and p53/TP53. The interaction between these two genes may play a role in the regulation of the 26S proteasome and NF-kappa-B.|
| [PHGDH](https://www.ncbi.nlm.nih.gov/gene?term=PHGDH[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RALA](https://www.ncbi.nlm.nih.gov/gene?term=RALA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 202.3</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 20 (min len 3) | Possible functional associations between PHGDH and RALA could be their involvement in regulating cell proliferation and growth signaling. PHGDH is involved in the phosphorylated L-serine biosynthesis pathway and the reversible oxidation of 2-hydroxyglutarate. RALA is a multifunctional GTPase involved in various processes including gene expression, cell migration, and membrane trafficking. The shortest path between these genes in the BioGRID protein interaction network involves Cul7, CCDC8, NTRK1, Trim25, and HDAC6.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FOXJ3](https://www.ncbi.nlm.nih.gov/gene?term=FOXJ3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 370 (min len 4) | There are currently no known functional associations between CD40 and FOXJ3 in the context of TCGA-SKCM. CD40 is a receptor involved in immunoglobulin secretion, while FOXJ3 is a transcriptional activator involved in muscle fiber type identity and spermatogenesis.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There is a protein interaction between COPS5 and GADD45B mediated by GADD45GIP1. COPS5 is involved in protein degradation through the COP9 signalosome complex, while GADD45B regulates growth and apoptosis as well as activates stress-responsive MAPKKK. They may function together in the regulation of cellular processes such as cell growth and stress response.|
| [DRAP1](https://www.ncbi.nlm.nih.gov/gene?term=DRAP1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [FEZ2](https://www.ncbi.nlm.nih.gov/gene?term=FEZ2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.8</br>**STRING score:** 404</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between DRAP1 and FEZ2 are unclear based on their known functions and interactions. DRAP1 is involved in transcriptional regulation and can bind to DNA, while FEZ2 is involved in axonal outgrowth and interacts with synaptotagmin. Further investigations are needed to determine any possible biological connections between these two genes.|
| [FEZ2](https://www.ncbi.nlm.nih.gov/gene?term=FEZ2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 187.9</br>**STRING score:** 163</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association between FEZ2 and RAC2 is through the interaction with KRAS, which can regulate cellular processes such as cell proliferation, differentiation, and survival. FEZ2 is involved in axonal outgrowth, while RAC2 is a plasma membrane-associated small GTPase that regulates cellular responses such as phagocytosis and epithelial cell polarization.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 182.2</br>**STRING score:** 380</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 47 (min len 3) | Possible functional association between COPS5 and NFKBIA is that COPS5, as a component of the COP9 signalosome complex, participates in the phosphorylation of NFKBIA, regulating its activity as an inhibitor of NF-ÎºB transcription factors. Additionally, the shortest path between these genes in the BioGRID protein interaction network suggests they may directly interact and participate in processes such as p53 and ubiquitin-mediated protein degradation.|
| [DSG2](https://www.ncbi.nlm.nih.gov/gene?term=DSG2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.4</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the available information, there is no clear evidence of any functional association between DSG2 and PAN2. DSG2 is involved in cell-cell adhesion, while PAN2 is a catalytic subunit of the poly(A)-nuclease complex involved in mRNA turnover and HIF1A-mediated hypoxic response.|
| [DUSP6](https://www.ncbi.nlm.nih.gov/gene?term=DUSP6[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PTPRC](https://www.ncbi.nlm.nih.gov/gene?term=PTPRC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.4</br>**STRING score:** 227</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | There are no known functional associations between DUSP6 and PTPRC in specific biological terms. DUSP6 is involved in inactivating MAP kinases and promoting cell differentiation, while PTPRC acts as a receptor for cytomegalovirus and affects T-cell signaling and proliferation.|
| [RAC2](https://www.ncbi.nlm.nih.gov/gene?term=RAC2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TM9SF3](https://www.ncbi.nlm.nih.gov/gene?term=TM9SF3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 169.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between RAC2 and TM9SF3 may involve regulation of cellular responses and phagocytosis. RAC2 is a plasma membrane-associated small GTPase while TM9SF3 is necessary for genome-wide de novo methylation and is essential for the establishment of DNA methylation patterns during development. There is no evident direct interaction between RAC2 and TM9SF3 based on shortest path analysis through BioGRID.|
| [NFKBIA](https://www.ncbi.nlm.nih.gov/gene?term=NFKBIA[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [USP1](https://www.ncbi.nlm.nih.gov/gene?term=USP1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 167.1</br>**STRING score:** 192</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | Possible functional associations between NFKBIA and USP1 in TCGA-SKCM could involve their roles in regulating DNA damage response and repair. NFKBIA inhibits dimeric NF-kappa-B/REL complexes, while USP1 is a negative regulator of DNA damage repair and is involved in PCNA-mediated translesion synthesis. Their interaction in the BioGRID protein interaction network may suggest a potential role in modulating each other's functions in regulating DNA damage response.|
| [CDK7](https://www.ncbi.nlm.nih.gov/gene?term=CDK7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EVL](https://www.ncbi.nlm.nih.gov/gene?term=EVL[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 166.7</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | CDK7 is involved in cell cycle control and RNA transcription, while EVL is involved in actin-associated processes such as cell polarity and migration. There is no obvious functional association between these two genes in specific biological terms based on their known functions and provided context.|
| [FKBP4](https://www.ncbi.nlm.nih.gov/gene?term=FKBP4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSMD2](https://www.ncbi.nlm.nih.gov/gene?term=PSMD2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 165.9</br>**STRING score:** 188</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 51 (min len 3) | Possible functional association: FKBP4 and PSMD2 could be involved in the regulation of protein degradation by the proteasome based on their interactions in the BioGRID protein interaction network. FKBP4 is involved in steroid receptors heterocomplexes and regulation of microtubule dynamics, while PSMD2 is a component of the 19S proteasome regulatory particle complex.|
