| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CGRRF1](https://www.ncbi.nlm.nih.gov/gene?term=CGRRF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 238.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CGRRF1 and CRK are regulation of cell adhesion, spreading, and migration, as well as involvement in the EFNA5-EPHA3 signaling pathway. CGRRF1 inhibits growth in several cell lines and interacts with EIF proteins, while CRK interacts with PDGFRA/B, EPHA3, and KIT, and is involved in membrane ruffling, phagocytosis, and activation of MAPK8 via Rac-dependent mechanisms.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 225.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 105 (min len 4) | Possible functional associations: CD40 is a receptor for TNFSF5/CD40LG and is involved in immunoglobulin secretion. COL4A1 plays a role in inhibiting angiogenesis and endothelial cell proliferation, migration, and tube formation. There is no known direct biological association between CD40 and COL4A1.|
| [CDK7](https://www.ncbi.nlm.nih.gov/gene?term=CDK7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 210.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | CDK7 is involved in cell cycle control and RNA transcription, while CRK plays a role in cell adhesion, spreading, and migration. These two genes may potentially interact in regulating cellular growth and RNA transcription initiation processes.|
| [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UFM1](https://www.ncbi.nlm.nih.gov/gene?term=UFM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 206.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Based on the provided information, there does not seem to be a direct functional association between CRK and UFM1. CRK is involved in regulating cell adhesion and migration, as well as phagocytosis of apoptotic cells, while UFM1 is a ubiquitin-like modifier involved in reticulophagy. However, they both interact with other proteins involved in cellular processes such as transcription, receptor signaling, and protein modification.|
| [DHX29](https://www.ncbi.nlm.nih.gov/gene?term=DHX29[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 205.2</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | DHX29 and EBNA1BP2 interact in the protein interaction network and both genes are involved in RNA processes. DHX29 is an ATP-binding RNA helicase involved in translation initiation, while EBNA1BP2 is required for the processing of pre-rRNA and interacts with EBV EBNA1. It is possible that these genes may function together in RNA processing and potentially have a role in viral infection.|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX13](https://www.ncbi.nlm.nih.gov/gene?term=SNX13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 362 (min len 4) | Possible functional associations: EPRS is a multifunctional protein involved in protein synthesis and translation regulation. SNX13 is involved in intracellular trafficking and may play a role in endosome homeostasis. There is no known direct biological connection between these two genes based on the given information.|
| [MPZL1](https://www.ncbi.nlm.nih.gov/gene?term=MPZL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 199.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional association between MPZL1 and TGFBR2 is that MPZL1 may recruit PTPN11/SHP-2 to the cell membrane and be a substrate for PTPN11/SHP-2. On the other hand, TGFBR2 binds TGFB1, TGFB2, and TGFB3 and blocks activation of SMAD2 and SMAD3 by TGFB1. Both genes may play roles in cellular signaling pathways and regulation of protein phosphorylation.|
| [COPZ1](https://www.ncbi.nlm.nih.gov/gene?term=COPZ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPAG4](https://www.ncbi.nlm.nih.gov/gene?term=SPAG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between COPZ1 and SPAG4 are difficult to infer based on their known biological functions, as they appear to be involved in distinct cellular processes. COPZ1 is involved in protein transport, while SPAG4 plays a role in spermatogenesis and nuclear envelope maintenance. However, the BioGRID protein interaction network shows that the two proteins interact indirectly through DDX58, suggesting a possible regulatory or signaling relationship.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOP16](https://www.ncbi.nlm.nih.gov/gene?term=NOP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | LYPLA1 and NOP16 are not known to have a direct functional association. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while NOP16 is a subunit of succinyl-CoA synthetase, involved in substrate-level phosphorylation in the citric acid cycle.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOR1A](https://www.ncbi.nlm.nih.gov/gene?term=TOR1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 197.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between LYPLA1 and TOR1A in TCGA-SKCM could involve regulation of protein folding/stability, synaptic vesicle recycling, and control of nuclear envelope integrity and polarity. LYPLA1 acts as an acyl-protein thioesterase and lysophospholipase, while TOR1A has chaperone functions important for protein processing and regulation of subcellular protein location. There is no clear biological process linking the two genes.|
| [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 182.2</br>**STRING score:** 694</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between FBXO11 and MSH6 could be related to DNA repair mechanisms. FBXO11 is involved in ubiquitination and proteasomal degradation of proteins including those involved in TGF-beta signaling, cell migration and cell-cycle progression. MSH6 is a component of the post-replicative DNA mismatch repair system and may also play a role in homologous recombination repair.|
| [DNTTIP2](https://www.ncbi.nlm.nih.gov/gene?term=DNTTIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DPH2](https://www.ncbi.nlm.nih.gov/gene?term=DPH2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 180.9</br>**STRING score:** 190</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: DPH2 is required for diphthamide biosynthesis, which occurs in elongation factor 2. DNTTIP2 is involved in ribosome biogenesis and works with many ribosome biogenesis factors, an RNA chaperone, and ribosomal proteins to generate RNA folding and modifications. There is no known direct functional association between DNTTIP2 and DPH2.|
| [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 180.3</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: MAPK13 can interact with COL1A1 through VCP in the protein interaction network.   COL1A1 encodes type I collagen, a fibrillar forming collagen, which is a major component of the extracellular matrix. MAPK13 is a member of the MAP kinase family and is involved in various cellular responses to stimuli such as stress. While their exact functional association is unclear, it is possible that MAPK13 may play a role in regulating collagen production through its interaction with COL1A1.|
| [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAI14](https://www.ncbi.nlm.nih.gov/gene?term=RAI14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 177.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Based on the provided information, there is no clear indication of a functional association between PAN2 and RAI14 in specific biological terms. PAN2 is involved in retinol metabolism and interacts with human cytomegalovirus, while RAI14 plays a role in actin regulation in testis cells. Further analysis and research would be needed to determine any potential association between these genes.|
| [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.8</br>**STRING score:** 325</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | STAT3 is a signal transducer that mediates responses to growth factors, while COL1A1 encodes for type I collagen, which is involved in the extracellular matrix. The shortest path between these genes in the BioGRID protein interaction network suggests possible interactions in the context of cell adhesion and migration.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYBL2](https://www.ncbi.nlm.nih.gov/gene?term=MYBL2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.2</br>**STRING score:** 315</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional associations between GADD45B and MYBL2 include regulation of growth, apoptosis, cell survival, proliferation, and differentiation. GADD45B may interact with MYBL2 in the stress-responsive MTK1/MEKK4 MAPKKK pathway, while MYBL2 may be involved in the DREAM/LINC complex that regulates cell cycle-dependent genes.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.0</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | KDM5B and PARP2 have a shortest path interaction through HDAC1 and HIST2H2BE. KDM5B is a histone demethylase that represses the tumor suppressor gene BRCA1, while PARP2 plays a key role in initiating the repair of double-strand breaks in DNA. Based on this, they may be functionally associated in regulating DNA damage repair.|
| [CSK](https://www.ncbi.nlm.nih.gov/gene?term=CSK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.0</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | Possible functional associations between CSK and MAPK13 are in regulating cell growth, differentiation, migration, and immune response through phosphorylation and inhibition of Src-family kinases and downstream cascades of cellular responses evoked by extracellular stimuli such as pro-inflammatory cytokines or physical stress. The shortest path between these genes in the BioGRID protein interaction network suggests a direct or indirect link via various proteins involved in recruiting CSK to the plasma membrane and in cytoskeletal remodeling through phosphorylation of MAPT and STMN1.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | HSPA8 is a molecular chaperone that plays a role in protein quality control, mitochondrial import, and autophagy. SACM1L is a phosphoinositide phosphatase involved in the hydrolysis of phosphatidylinositol phosphates. A functional association between these two genes is not immediately clear based on their known roles and interactions.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOP16](https://www.ncbi.nlm.nih.gov/gene?term=NOP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | Possible functional associations between COPS5 and NOP16 may involve regulation of the ubiquitin conjugation pathway and the phosphorylation of certain proteins involved in cellular and developmental processes. However, there is no known direct interaction or shared biological function between these two genes in specific terms.|
| [CDC20](https://www.ncbi.nlm.nih.gov/gene?term=CDC20[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSRC1](https://www.ncbi.nlm.nih.gov/gene?term=PSRC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 399</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional associations between CDC20 and PSRC1 include their roles in mitosis, chromosome segregation, and regulation of the mitotic spindle. CDC20 is involved in the ubiquitin ligase activity of the anaphase promoting complex/cyclosome, while PSRC1 is required for normal progression through mitosis and helps regulate mitotic spindle dynamics. There may be synergistic effects between these genes in these processes.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.3</br>**STRING score:** 361</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations: CDK1 is involved in cell cycle regulation and interacts with multiple proteins, including CCNF and TRIM25. LYPLA1 has thioesterase and lysophospholipase activity and is involved in lipid metabolism. There is no clear overlap in their biological functions, suggesting they may not directly interact.|
| [NFATC4](https://www.ncbi.nlm.nih.gov/gene?term=NFATC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSRC1](https://www.ncbi.nlm.nih.gov/gene?term=PSRC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.2</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between NFATC4 and PSRC1 could involve regulation of mitotic spindle dynamics, progression through mitosis, and congress of chromosomes at the metaphase plate. These genes are not known to interact directly, but could potentially play a role in the regulation of cell division. Other functions of NFATC4 include regulation of adult hippocampal neurogenesis, involvement in BDNF-driven pro-survival signaling, and stimulation of cytokine gene transcription, while PSRC1 is involved in the turnover of microtubules on metaphase spindles and contributes to tension across sister kinetochores.|
