| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [CGRRF1](https://www.ncbi.nlm.nih.gov/gene?term=CGRRF1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 238.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional associations between CGRRF1 and CRK include their interaction with PDGFRB, which is involved in regulating cell growth and migration, and their roles in cell adhesion and motility through interaction with SRC and DOCK1/4. Both genes may also be involved in signaling pathways related to growth factor receptors such as EPHA3 and KIT.|
| [CD40](https://www.ncbi.nlm.nih.gov/gene?term=CD40[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [COL4A1](https://www.ncbi.nlm.nih.gov/gene?term=COL4A1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 225.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 105 (min len 4) | Possible functional association between CD40 and COL4A1 is regulation of angiogenesis and tumor formation through MAP3K8-mediated signals leading to induction of immunoglobulin secretion and inhibition of endothelial cell proliferation, migration and tube formation. CD40 is a receptor for TNFSF5/CD40LG and COL4A1 interacts with EFEMP2, forming a type IV collagen network.|
| [CDK7](https://www.ncbi.nlm.nih.gov/gene?term=CDK7[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 210.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 31 (min len 3) | There is no clear functional association between CDK7 and CRK based on their known biological functions. CDK7 is a kinase involved in cell cycle control and RNA transcription, while CRK regulates cell adhesion and motility. However, they do interact in the BioGRID protein interaction network, potentially suggesting a physical association between the proteins.|
| [CRK](https://www.ncbi.nlm.nih.gov/gene?term=CRK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [UFM1](https://www.ncbi.nlm.nih.gov/gene?term=UFM1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 206.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | There is no obvious functional association between CRK and UFM1 based on their known biological roles. CRK plays a role in cell adhesion, migration, and phagocytosis, while UFM1 is involved in protein ufmylation, which is part of a cellular stress response. However, the protein interaction network suggests potential indirect interactions between the two genes, possibly mediated by other proteins in the network.|
| [DHX29](https://www.ncbi.nlm.nih.gov/gene?term=DHX29[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [EBNA1BP2](https://www.ncbi.nlm.nih.gov/gene?term=EBNA1BP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 205.2</br>**STRING score:** 202</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 30 (min len 3) | Possible functional associations: DHX29 is involved in translation initiation as part of the 43S pre-initiation complex, while EBNA1BP2 is required for the processing of pre-rRNA and interacts with EBV EBNA1. It is unclear if there is a direct functional association between these two genes, but they may both play a role in protein synthesis and/or viral processes involving EBV.|
| [EPRS](https://www.ncbi.nlm.nih.gov/gene?term=EPRS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SNX13](https://www.ncbi.nlm.nih.gov/gene?term=SNX13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 200.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 362 (min len 4) | Possible functional associations between EPRS and SNX13 may involve intracellular trafficking and endosome homeostasis. EPRS is involved in protein synthesis and mTORC1 signaling pathway, while SNX13 acts as a GAP for Galphas and may interact with viral replication complexes. However, there is currently no known direct biological interaction between these two genes.|
| [MPZL1](https://www.ncbi.nlm.nih.gov/gene?term=MPZL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TGFBR2](https://www.ncbi.nlm.nih.gov/gene?term=TGFBR2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 199.1</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | Possible functional associations between MPZL1 and TGFBR2 may involve cell signaling, as MPZL1 is a cell surface receptor involved in signal transduction processes and TGFBR2 is a receptor for TGFB1, TGFB2, and TGFB3. Additionally, TGFBR2 is involved in blocking activation of SMAD2 and SMAD3 by TGFB1, indicating a role in regulating cellular responses to growth factors.|
| [COPZ1](https://www.ncbi.nlm.nih.gov/gene?term=COPZ1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPAG4](https://www.ncbi.nlm.nih.gov/gene?term=SPAG4[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: None identified based on known biological roles of COPZ1 and SPAG4.  COPZ1 is involved in protein transport and is a subunit of the coatomer complex while SPAG4 plays a role in spermatogenesis, particularly in the formation and organization of the manchette and anchoring of the SUN3 protein. There are no known functional associations between these two genes based on their respective roles in different cellular processes.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOP16](https://www.ncbi.nlm.nih.gov/gene?term=NOP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 198.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between LYPLA1 and NOP16 could be related to lipid metabolism and energy production. LYPLA1 is involved in the hydrolysis of fatty acids and lysophospholipids, while NOP16 plays a role in the citric acid cycle for energy production. It is possible that the interaction between these genes relates to the regulation of lipid and energy metabolism.|
| [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TOR1A](https://www.ncbi.nlm.nih.gov/gene?term=TOR1A[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 197.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association between LYPLA1 and TOR1A could involve regulation of protein folding, stability, and localization, as well as modulation of neurotransmission and cytoskeleton-nuclear envelope interactions. LYPLA1 has thioesterase and lysophospholipase activities, while TOR1A acts as a chaperone and is involved in synaptic vesicle recycling. There could be potential interactions in the quality control of protein folding and trafficking between the two genes.|
| [FBXO11](https://www.ncbi.nlm.nih.gov/gene?term=FBXO11[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MSH6](https://www.ncbi.nlm.nih.gov/gene?term=MSH6[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 182.2</br>**STRING score:** 694</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations: None apparent based on their known functions and biological processes.          FBXO11 is a component of a SCF E3 ubiquitin-protein ligase complex that targets proteins for degradation, including BCL6 and DTL. MSH6 is a component of the post-replicative DNA mismatch repair system, which recognizes and repairs DNA mismatches. These genes are not known to have any direct functional associations based on their known functions and biological processes.|
| [DNTTIP2](https://www.ncbi.nlm.nih.gov/gene?term=DNTTIP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [DPH2](https://www.ncbi.nlm.nih.gov/gene?term=DPH2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 180.9</br>**STRING score:** 190</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | DNTTIP2 and DPH2 have a possible functional association in ribosome biogenesis, as DNTTIP2 is a component of the small subunit processome and DPH2 is required for the first step of diphthamide biosynthesis, a post-translational modification of histidine which occurs in elongation factor 2.|
| [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 180.3</br>**STRING score:** 162</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: MAPK13 may regulate the expression of COL1A1 as part of its role in the regulation of epidermal keratinocyte differentiation and skin tumor development.   COL1A1 encodes type I collagen, which forms fibrils and interacts with other proteins such as MRC2, TRAM2, and MFAP4. MAPK13, a p38 MAPK isoform, plays a role in signaling pathways activated by pro-inflammatory cytokines and physical stress, as well as in cytoskeletal remodeling and the regulation of gene expression, differentiation, and apoptosis in skin cells. The shortest path between these two genes in the BioGRID protein interaction network involves VCP as an intermediate protein.|
| [PAN2](https://www.ncbi.nlm.nih.gov/gene?term=PAN2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RAI14](https://www.ncbi.nlm.nih.gov/gene?term=RAI14[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 177.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | It is not clear from the provided information whether PAN2 and RAI14 have any direct functional associations. PAN2 is involved in mRNA turnover and regulation of HIF1A-mediated hypoxic response, while RAI14 plays a role in actin regulation at the ectoplasmic specialization in testis and sperm polarity.|
| [COL1A1](https://www.ncbi.nlm.nih.gov/gene?term=COL1A1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAT3](https://www.ncbi.nlm.nih.gov/gene?term=STAT3[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 175.8</br>**STRING score:** 325</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 7 (min len 3) | COL1A1 and STAT3 are functionally associated through protein interactions as shown in the BioGRID network. COL1A1 encodes for Type I collagen, a fibrillar-forming collagen, which interacts with various proteins. STAT3, on the other hand, is involved in signal transduction and transcriptional regulation in response to various growth factors and cytokines. The network suggests that these genes are potentially involved in processes such as extracellular matrix remodeling, cell migration, and immune response.|
| [GADD45B](https://www.ncbi.nlm.nih.gov/gene?term=GADD45B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MYBL2](https://www.ncbi.nlm.nih.gov/gene?term=MYBL2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.2</br>**STRING score:** 315</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | Possible functional association: MYBL2 and GADD45B may be involved in cell cycle regulation and stress response pathways. Both genes interact with other proteins involved in these processes and may contribute to the coordination of cellular responses to stress and growth signals.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PARP2](https://www.ncbi.nlm.nih.gov/gene?term=PARP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 174.0</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 3 (min len 3) | Possible functional association: KDM5B and PARP2 interact in protein networks and may be involved in DNA repair and chromatin remodeling processes in the context of cancer. KDM5B is a histone demethylase that represses tumor suppressor genes, while PARP2 is a poly-ADP ribosyltransferase that initiates repair of DNA strand breaks and plays a key role in base excision repair.|
| [CSK](https://www.ncbi.nlm.nih.gov/gene?term=CSK[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [MAPK13](https://www.ncbi.nlm.nih.gov/gene?term=MAPK13[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 172.0</br>**STRING score:** 243</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 5 (min len 3) | CSK is a non-receptor tyrosine kinase that phosphorylates and maintains inactivation of Src-family kinases, while MAPK13 is a serine/threonine kinase that plays a role in the regulation of protein translation, cytoskeletal remodeling, epidermal keratinocyte differentiation, and skin tumor development. Based on the BioGRID protein interaction network, CSK and MAPK13 may have a functional association as they are linked by multiple shortest paths, potentially suggesting a regulatory or signaling relationship between the two proteins.|
| [HSPA8](https://www.ncbi.nlm.nih.gov/gene?term=HSPA8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SACM1L](https://www.ncbi.nlm.nih.gov/gene?term=SACM1L[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.6</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | HSPA8 and SACM1L do not have an obvious functional association based on their known biological roles. HSPA8 is a molecular chaperone involved in protein folding and quality control, while SACM1L is a phosphoinositide phosphatase. The shortest path between them in the BioGRID protein interaction network indicates that they may physically interact, but further investigation is needed to determine the functional significance of this interaction.|
| [COPS5](https://www.ncbi.nlm.nih.gov/gene?term=COPS5[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NOP16](https://www.ncbi.nlm.nih.gov/gene?term=NOP16[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 171.3</br>**STRING score:** 0</br>**BioGRID count:** 1</br>**BioGRID shortest paths:** 1 (min len 2) | There is no clear biological or functional association between COPS5 and NOP16 based on their known functions. COPS5 is a protease subunit involved in regulating the ubiquitin conjugation pathway, while NOP16 is a subunit of succinyl-CoA synthetase involved in the citric acid cycle.|
| [CDC20](https://www.ncbi.nlm.nih.gov/gene?term=CDC20[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSRC1](https://www.ncbi.nlm.nih.gov/gene?term=PSRC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.4</br>**STRING score:** 399</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 8 (min len 3) | Possible functional association between CDC20 and PSRC1 is that they both play a role in regulating mitosis and the mitotic spindle dynamics. CDC20 is required for full ubiquitin ligase activity of the anaphase promoting complex/cyclosome (APC/C), while PSRC1 is required for normal progression through mitosis and for normal congress of chromosomes at the metaphase plate.|
| [CDK1](https://www.ncbi.nlm.nih.gov/gene?term=CDK1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LYPLA1](https://www.ncbi.nlm.nih.gov/gene?term=LYPLA1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.3</br>**STRING score:** 361</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations: CDK1 is involved in cell cycle control, while LYPLA1 functions as an acyl-protein thioesterase and lysophospholipase. They may regulate common signaling pathways and have intersecting roles in cellular processes such as proliferation and differentiation. The shortest path between CDK1 and LYPLA1 in the BioGRID protein interaction network suggests potential direct interactions between the two proteins, which warrant further investigation.|
| [NFATC4](https://www.ncbi.nlm.nih.gov/gene?term=NFATC4[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PSRC1](https://www.ncbi.nlm.nih.gov/gene?term=PSRC1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 170.2</br>**STRING score:** 152</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional associations between NFATC4 and PSRC1 could be related to mitosis, as PSRC1 is involved in normal progression through mitosis and regulation of mitotic spindle dynamics, while NFATC4 is involved in cardiovascular system development and function, T-cell activation, and transcription activation. However, further investigation is needed to determine any direct biological links between these two genes.|
