| Genes&nbsp;&nbsp;&nbsp;&nbsp;| Data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Summary |
|:---|:---|:---|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HINT1](https://www.ncbi.nlm.nih.gov/gene?term=HINT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 274.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 27 (min len 3) | Possible functional association: DNM1L and HINT1 are connected through various protein interactions involved in mitochondrial fission and regulation of transcriptional activation. Both proteins are involved in regulating apoptosis and can modulate proteasomal degradation of target proteins. It is possible they work together to regulate these processes.|
| [MBNL2](https://www.ncbi.nlm.nih.gov/gene?term=MBNL2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAP2](https://www.ncbi.nlm.nih.gov/gene?term=STAP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 268.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 1 (min len 3) | There are no clear functional associations between MBNL2 and STAP2 based on their known roles in pre-mRNA alternative splicing and acute-phase response, respectively. However, they do interact indirectly through HNRNPL, which may indicate shared pathways or regulatory mechanisms.|
| [MFSD10](https://www.ncbi.nlm.nih.gov/gene?term=MFSD10[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [RHEB](https://www.ncbi.nlm.nih.gov/gene?term=RHEB[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 264.0</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | Possible functional association: MFSD10 and RHEB may be involved in the regulation of cellular signaling pathways related to apoptosis and viral infection, due to their interactions with MTORC1 and herpes simplex virus 1 protein UL12, respectively.   The specific biological processes where these genes are involved include organic anion transport in renal proximal tubules for MFSD10, and activation of mTORC1 signaling and regulation of apoptosis for RHEB. It is unclear if there are any direct functional associations between these two genes beyond their protein interaction network.|
| [CYCS](https://www.ncbi.nlm.nih.gov/gene?term=CYCS[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PRKCD](https://www.ncbi.nlm.nih.gov/gene?term=PRKCD[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 261.3</br>**STRING score:** 607</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | Possible functional association: Both CYCS and PRKCD are involved in apoptosis. CYCS plays a role in the mitochondrial pathway of apoptosis by releasing cytochrome c, while PRKCD has contrasting roles in cell death and survival as it functions as a pro-apoptotic protein during DNA damage-induced apoptosis but as an anti-apoptotic protein during cytokine receptor-initiated cell death. They may have interaction in regulating apoptosis in SKCM cancer cells.|
| [IGF2BP2](https://www.ncbi.nlm.nih.gov/gene?term=IGF2BP2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STUB1](https://www.ncbi.nlm.nih.gov/gene?term=STUB1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 260.7</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 38 (min len 3) | Possible functional associations between IGF2BP2 and STUB1 include IGF2BP2's role in mRNA stability and transport and STUB1's role in protein degradation through ubiquitination. It is possible that STUB1 could target IGF2BP2 for degradation, but further research would be needed to confirm this hypothesis.|
| [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 258.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | GMEB2 is a transcription factor involved in glucocorticoid response and parvovirus replication, while LPGAT1 is a lysophospholipid acyltransferase involved in lipid metabolism. The shortest protein interaction paths between GMEB2 and LPGAT1 in BioGRID suggest a possible indirect functional association through proteins like CUL3 and TIPARP, but the biological significance of such an association is unclear without further experimental evidence.|
| [RNPS1](https://www.ncbi.nlm.nih.gov/gene?term=RNPS1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SERPINE1](https://www.ncbi.nlm.nih.gov/gene?term=SERPINE1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 253.4</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 3) | Possible functional association: RNPS1 is involved in mRNA splicing and regulation of alternative splicing, while SERPINE1 plays a role in regulating cell migration and senescence. There is no clear biological mechanism connecting these two genes. However, their interaction via protein network may suggest a potential indirect association.|
| [CTSL1](https://www.ncbi.nlm.nih.gov/gene?term=CTSL1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [TESK1](https://www.ncbi.nlm.nih.gov/gene?term=TESK1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 252.6</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 0 (min len 0) | There is no known functional association between CTSL1 and TESK1 based on the BioGRID protein interaction network. CTSL1 is involved in regulating cell cycle progression through proteolytic processing of the CUX1 transcription factor, while TESK1 is a dual-specificity protein kinase that regulates the cytoskeleton by enhancing actin stress fiber formation and inhibiting microtubule breakdown.|
| [GTDC1](https://www.ncbi.nlm.nih.gov/gene?term=GTDC1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [STAP2](https://www.ncbi.nlm.nih.gov/gene?term=STAP2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 251.2</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 6 (min len 4) | Possible functional associations between GTDC1 and STAP2 could be related to regulation of transcriptional activation and inflammation response. GTDC1 may affect NF-kappa-B activation and interact with PCDH8, while STAP2 may modulate STAT3 activity and interact with PTK6 and CSF1R. Their shortest protein interaction path involves APP, PTBP1, PTK2, PTK6, EGFR, and KEAP1, suggesting they may be involved in signaling pathways related to cancer progression.|
| [CAB39](https://www.ncbi.nlm.nih.gov/gene?term=CAB39[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [LPGAT1](https://www.ncbi.nlm.nih.gov/gene?term=LPGAT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 249.9</br>**STRING score:** 0</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | The possible functional association between CAB39 and LPGAT1 is unclear based solely on their known biology and protein interactions. CAB39 is involved in activating the STK11/LKB1 complex, while LPGAT1 is involved in fatty acid remodeling of phospholipids in the endoplasmic reticulum membrane. There may be indirect interactions between these processes, but it is not clear from the available information what those might be.|
| [DNM1L](https://www.ncbi.nlm.nih.gov/gene?term=DNM1L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [SPTLC2](https://www.ncbi.nlm.nih.gov/gene?term=SPTLC2[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 242.7</br>**STRING score:** 186</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 9 (min len 3) | Possible functional associations between DNM1L and SPTLC2 are unclear based on the provided information. DNM1L is involved in mitochondrial fission, while SPTLC2 plays a role in de novo sphingolipid biosynthesis crucial for adipogenesis. The shortest path in the BioGRID protein interaction network suggests potential indirect interactions between these two genes through other proteins.|
| [ENOPH1](https://www.ncbi.nlm.nih.gov/gene?term=ENOPH1[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [HINT1](https://www.ncbi.nlm.nih.gov/gene?term=HINT1[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 237.2</br>**STRING score:** 234</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 4 (min len 3) | There is no clear functional association between ENOPH1 and HINT1 in specific biological terms. ENOPH1 is involved in the biosynthesis of methionine via the transsulfuration pathway, while HINT1 is an enzyme that hydrolyzes purine nucleotide phosphoramidates. The shortest path between these two genes in the BioGRID protein interaction network involves intermediate proteins such as PRDX6, ERBB3, ZRANB1, and TRIM67.|
| [ASH2L](https://www.ncbi.nlm.nih.gov/gene?term=ASH2L[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [H2AFV](https://www.ncbi.nlm.nih.gov/gene?term=H2AFV[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 227.2</br>**STRING score:** 751</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 12 (min len 3) | ASH2L and H2AFV are both involved in histone modification and regulation of gene transcription. ASH2L is a component of several histone methyltransferase complexes, including the Set1/Ash2 histone methyltransferase complex, while H2AFV is a variant histone H2A involved in the formation of constitutive heterochromatin. It is possible that ASH2L and H2AFV work together to regulate chromatin structure and gene expression during cell division and hematopoiesis.|
| [KDM5B](https://www.ncbi.nlm.nih.gov/gene?term=KDM5B[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [ZNF274](https://www.ncbi.nlm.nih.gov/gene?term=ZNF274[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 226.5</br>**STRING score:** 262</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 153 (min len 4) | Possible functional association: KDM5B may act as a transcriptional corepressor for ZNF274 by interacting with chromatin-modifying proteins, such as HDACs, and by repressing tumor suppressor genes, such as BRCA1 and HOXA5.   KDM5B is involved in histone demethylation, while ZNF274 is a transcriptional repressor that specifically binds to the 3'-end of zinc-finger coding genes. It is possible that KDM5B and ZNF274 may work together to regulate gene expression, particularly in the context of chromatin modifications and transcriptional repression.|
| [GTPBP8](https://www.ncbi.nlm.nih.gov/gene?term=GTPBP8[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [PPIC](https://www.ncbi.nlm.nih.gov/gene?term=PPIC[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 224.0</br>**STRING score:** 216</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 52 (min len 4) | There are no clear functional associations between GTPBP8 and PPIC based on the protein interaction network and context. GTPBP8 is involved in GH signaling and interacts with ADAM17, while PPIC catalyzes cis-trans isomerization of proline imidic peptide bonds and interacts with EGFR/ERBB1-4.|
| [HMGA2](https://www.ncbi.nlm.nih.gov/gene?term=HMGA2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [NUP85](https://www.ncbi.nlm.nih.gov/gene?term=NUP85[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 215.9</br>**STRING score:** 181</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 10 (min len 3) | Possible functional association: NUP85 may play a role in regulating the expression of HMGA2, which functions as a transcriptional regulator. Both genes are involved in nuclear processes, including chromosome condensation and assembly of the nuclear pore complex.|
| [GMEB2](https://www.ncbi.nlm.nih.gov/gene?term=GMEB2[Gene+Name]+AND+9606[Taxonomy+ID]) </br> and </br> [WDR61](https://www.ncbi.nlm.nih.gov/gene?term=WDR61[Gene+Name]+AND+9606[Taxonomy+ID]) |  **RMST differance:** 211.5</br>**STRING score:** 183</br>**BioGRID count:** 0</br>**BioGRID shortest paths:** 2 (min len 3) | GMEB2 and WDR61 have a possible functional association through their interaction with the parvovirus initiator complex (PIF). GMEB2 is involved in the replication of parvoviruses and forms the PIF complex with GMEB1, while WDR61 is a component of the PAF1 complex which associates with the PIF complex.|
